{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## NOTES on Licensing\n",
    "\n",
    "Neo4j may be end up being pretty expensive, prices are not reported in the webpage, see: https://www.reddit.com/r/Database/comments/jpmnxp/freecheaper_alternatives_to_neo4j_for_a/\n",
    "\n",
    "Ne4j python repo: https://github.com/neo4j/neo4j-python-driver\n",
    "\n",
    "__Alternatives:__\n",
    "\n",
    "* OngDB (GPL v3, forces code to be open): https://github.com/graphfoundation/ongdb\n",
    "* MemGRAPH (BSL license, compatible with neo4j): https://memgraph.com/blog/neo4j-alternative-what-are-my-open-source-db-options"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "43774"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import json \n",
    "\n",
    "\n",
    "file_path = \"/home/robaina/Documents/NewAtlantis/enzyme_activity/notebooks/data/annotations/modelseed/reactions.json\"\n",
    "\n",
    "with open(file_path, \"r\") as file:\n",
    "    data = json.load(file)\n",
    "\n",
    "len(data)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'1.1.1.1': {'ec_num': '1.1.1.1',\n",
       "  'accepted_name': 'alcohol dehydrogenase',\n",
       "  'reaction': '(1) a primary alcohol + NAD+ = an aldehyde + NADH + H+;;(2) a secondary alcohol + NAD+ = a ketone + NADH + H+',\n",
       "  'other_names': 'aldehyde reductase; ADH; alcohol dehydrogenase (NAD); aliphatic alcohol dehydrogenase; ethanol dehydrogenase; NAD-dependent alcohol dehydrogenase; NAD-specific aromatic alcohol dehydrogenase; NADH-alcohol dehydrogenase; NADH-aldehyde dehydrogenase; primary alcohol dehydrogenase; yeast alcohol dehydrogenase',\n",
       "  'sys_name': 'alcohol:NAD+ oxidoreductase',\n",
       "  'comments': 'A zinc protein. Acts on primary or secondary alcohols or hemi-acetals with very broad specificity; however the enzyme oxidizes methanol much more poorly than ethanol. The animal, but not the yeast, enzyme acts also on cyclic secondary alcohols.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.1\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.1\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.1\" target=\"new\">EXPASY</a>, <a href=\"https://randr.nist.gov/enzyme/Default.aspx\" target=\"new\">GTD</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.1\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.1.1.1%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:30:59'},\n",
       " '1.1.1.2': {'ec_num': '1.1.1.2',\n",
       "  'accepted_name': 'alcohol dehydrogenase (NADP+)',\n",
       "  'reaction': 'an alcohol + NADP+ = an aldehyde + NADPH + H+',\n",
       "  'other_names': 'aldehyde reductase (NADPH2); NADP-alcohol dehydrogenase; NADP+-aldehyde reductase; NADP+-dependent aldehyde reductase; NADPH-aldehyde reductase; NADPH-dependent aldehyde reductase; nonspecific succinic semialdehyde reductase; ALR 1; low-Km aldehyde reductase; high-Km aldehyde reductase; alcohol dehydrogenase (NADP)',\n",
       "  'sys_name': 'alcohol:NADP+ oxidoreductase',\n",
       "  'comments': 'A zinc protein. Some members of this group oxidize only primary alcohols; others act also on secondary alcohols. May be identical with EC 1.1.1.19 (L-glucuronate reductase), EC 1.1.1.33 [mevaldate reductase (NADPH)] and EC 1.1.1.55 [lactaldehyde reductase (NADPH)]. Re-specific with respect to NADPH.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.3': {'ec_num': '1.1.1.3',\n",
       "  'accepted_name': 'homoserine dehydrogenase',\n",
       "  'reaction': 'L-homoserine + NAD(P)+ = L-aspartate 4-semialdehyde + NAD(P)H + H+',\n",
       "  'other_names': 'HSDH; HSD',\n",
       "  'sys_name': 'L-homoserine:NAD(P)+ oxidoreductase',\n",
       "  'comments': 'The yeast enzyme acts most rapidly with NAD+; the Neurospora enzyme with NADP+. The enzyme from Escherichia coli is a multi-functional protein, which also catalyses the reaction of EC 2.7.2.4 (aspartate kinase).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.4': {'ec_num': '1.1.1.4',\n",
       "  'accepted_name': '(R,R)-butanediol dehydrogenase',\n",
       "  'reaction': '(R,R)-butane-2,3-diol + NAD+ = (R)-acetoin + NADH + H+',\n",
       "  'other_names': 'butyleneglycol dehydrogenase; D-butanediol dehydrogenase; D-(–)-butanediol dehydrogenase; butylene glycol dehydrogenase; diacetyl (acetoin) reductase; D-aminopropanol dehydrogenase; 1-amino-2-propanol dehydrogenase; 2,3-butanediol dehydrogenase; D-1-amino-2-propanol dehydrogenase; (R)-diacetyl reductase; (R)-2,3-butanediol dehydrogenase; D-1-amino-2-propanol:NAD+ oxidoreductase; 1-amino-2-propanol oxidoreductase; aminopropanol oxidoreductase',\n",
       "  'sys_name': '(R,R)-butane-2,3-diol:NAD+ oxidoreductase',\n",
       "  'comments': 'Also converts diacetyl into acetoin with NADH as reductant.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.5': {'ec_num': '1.1.1.5',\n",
       "  'accepted_name': 'acetoin dehydrogenase',\n",
       "  'reaction': 'acetoin + NAD+ = diacetyl + NADH + H+',\n",
       "  'other_names': 'diacetyl reductase',\n",
       "  'sys_name': 'acetoin:NAD+ oxidoreductase',\n",
       "  'comments': 'NADP+ also acts as acceptor.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.6': {'ec_num': '1.1.1.6',\n",
       "  'accepted_name': 'glycerol dehydrogenase',\n",
       "  'reaction': 'glycerol + NAD+ = glycerone + NADH + H+',\n",
       "  'other_names': 'glycerin dehydrogenase; NAD-linked glycerol dehydrogenase',\n",
       "  'sys_name': 'glycerol:NAD+ 2-oxidoreductase',\n",
       "  'comments': 'Also acts on propane-1,2-diol.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.7': {'ec_num': '1.1.1.7',\n",
       "  'accepted_name': 'propanediol-phosphate dehydrogenase',\n",
       "  'reaction': 'propane-1,2-diol 1-phosphate + NAD+ = hydroxyacetone phosphate + NADH + H+',\n",
       "  'other_names': 'PDP dehydrogenase; 1,2-propanediol-1-phosphate:NAD+ oxidoreductase; propanediol phosphate dehydrogenase',\n",
       "  'sys_name': 'propane-1,2-diol-1-phosphate:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.8': {'ec_num': '1.1.1.8',\n",
       "  'accepted_name': 'glycerol-3-phosphate dehydrogenase (NAD+)',\n",
       "  'reaction': 'sn-glycerol 3-phosphate + NAD+ = glycerone phosphate + NADH + H+',\n",
       "  'other_names': 'α-glycerol phosphate dehydrogenase (NAD+); α-glycerophosphate dehydrogenase (NAD+); glycerol 1-phosphate dehydrogenase; glycerol phosphate dehydrogenase (NAD+); glycerophosphate dehydrogenase (NAD+); hydroglycerophosphate dehydrogenase; L-α-glycerol phosphate dehydrogenase; L-α-glycerophosphate dehydrogenase; L-glycerol phosphate dehydrogenase; L-glycerophosphate dehydrogenase (ambiguous); NAD+-α-glycerophosphate dehydrogenase; NAD+-dependent glycerol phosphate dehydrogenase; NAD+-dependent glycerol-3-phosphate dehydrogenase; NAD+-L-glycerol-3-phosphate dehydrogenase; NAD+-linked glycerol 3-phosphate dehydrogenase; NADH-dihydroxyacetone phosphate reductase; glycerol-3-phosphate dehydrogenase (NAD+); L-glycerol-3-phosphate dehydrogenase (ambiguous)',\n",
       "  'sys_name': 'sn-glycerol-3-phosphate:NAD+ 2-oxidoreductase',\n",
       "  'comments': 'Also acts on propane-1,2-diol phosphate and glycerone sulfate (but with a much lower affinity).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': 'glycerone phosphate = dihydroxyacetone phosphate = 3-hydroxy-2-oxopropyl phosphate',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.9': {'ec_num': '1.1.1.9',\n",
       "  'accepted_name': 'D-xylulose reductase',\n",
       "  'reaction': 'xylitol + NAD+ = D-xylulose + NADH + H+',\n",
       "  'other_names': 'NAD+-dependent xylitol dehydrogenase; xylitol dehydrogenase (ambiguous); erythritol dehydrogenase; 2,3-cis-polyol(DPN) dehydrogenase (C3-5); pentitol-DPN dehydrogenase (ambiguous); xylitol-2-dehydrogenase',\n",
       "  'sys_name': 'xylitol:NAD+ 2-oxidoreductase (D-xylulose-forming)',\n",
       "  'comments': 'Also acts as an L-erythrulose reductase.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.10': {'ec_num': '1.1.1.10',\n",
       "  'accepted_name': 'L-xylulose reductase',\n",
       "  'reaction': 'xylitol + NADP+ = L-xylulose + NADPH + H+',\n",
       "  'other_names': 'xylitol dehydrogenase (ambiguous)',\n",
       "  'sys_name': 'xylitol:NADP+ 4-oxidoreductase (L-xylulose-forming)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.11': {'ec_num': '1.1.1.11',\n",
       "  'accepted_name': 'D-arabinitol 4-dehydrogenase',\n",
       "  'reaction': 'D-arabinitol + NAD+ = D-xylulose + NADH + H+',\n",
       "  'other_names': 'D-arabitol dehydrogenase; arabitol dehydrogenase',\n",
       "  'sys_name': 'D-arabinitol:NAD+ 4-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.12': {'ec_num': '1.1.1.12',\n",
       "  'accepted_name': 'L-arabinitol 4-dehydrogenase',\n",
       "  'reaction': 'L-arabinitol + NAD+ = L-xylulose + NADH + H+',\n",
       "  'other_names': 'pentitol-DPN dehydrogenase (ambiguous); L-arabitol dehydrogenase',\n",
       "  'sys_name': 'L-arabinitol:NAD+ 4-oxidoreductase (L-xylulose-forming)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.13': {'ec_num': '1.1.1.13',\n",
       "  'accepted_name': 'L-arabinitol 2-dehydrogenase',\n",
       "  'reaction': 'L-arabinitol + NAD+ = L-ribulose + NADH + H+',\n",
       "  'other_names': 'L-arabinitol dehydrogenase (ribulose-forming); L-arabinitol (ribulose-forming) dehydrogenase',\n",
       "  'sys_name': 'L-arabinitol:NAD+ 2-oxidoreductase (L-ribulose-forming)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.14': {'ec_num': '1.1.1.14',\n",
       "  'accepted_name': 'L-iditol 2-dehydrogenase',\n",
       "  'reaction': 'L-iditol + NAD+ = L-sorbose + NADH + H+',\n",
       "  'other_names': 'polyol dehydrogenase; sorbitol dehydrogenase; L-iditol:NAD+ 5-oxidoreductase; L-iditol (sorbitol) dehydrogenase; glucitol dehydrogenase; L-iditol:NAD+ oxidoreductase; NAD+-dependent sorbitol dehydrogenase; NAD+-sorbitol dehydrogenase',\n",
       "  'sys_name': 'L-iditol:NAD+ 2-oxidoreductase',\n",
       "  'comments': 'This enzyme is widely distributed and has been described in archaea, bacteria, yeast, plants and animals. It acts on a number of sugar alcohols, including (but not limited to) L-iditol, D-glucitol, D-xylitol, and D-galactitol. Enzymes from different organisms or tissues display different substrate specificity. The enzyme is specific to NAD+ and can not use NADP+.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.15': {'ec_num': '1.1.1.15',\n",
       "  'accepted_name': 'D-iditol 2-dehydrogenase',\n",
       "  'reaction': 'D-iditol + NAD+ = D-sorbose + NADH + H+',\n",
       "  'other_names': 'D-sorbitol dehydrogenase',\n",
       "  'sys_name': 'D-iditol:NAD+ 2-oxidoreductase',\n",
       "  'comments': 'Also converts xylitol into L-xylulose and L-glucitol into L-fructose.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.16': {'ec_num': '1.1.1.16',\n",
       "  'accepted_name': 'galactitol 2-dehydrogenase',\n",
       "  'reaction': 'galactitol + NAD+ = D-tagatose + NADH + H+',\n",
       "  'other_names': 'dulcitol dehydrogenase; AtuSorbD (gene name); galactitol:NAD+ 2-oxidoreductase',\n",
       "  'sys_name': 'galactitol:NAD+ 2-oxidoreductase (D-tagatose-forming)',\n",
       "  'comments': 'Also converts other alditols containing an L-threo-configuration adjacent to a primary alcohol group into the corresponding sugars. The enzyme from Agrobacterium fabrum C58 is part of D-altritol and galactitol degradation pathways.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.17': {'ec_num': '1.1.1.17',\n",
       "  'accepted_name': 'mannitol-1-phosphate 5-dehydrogenase',\n",
       "  'reaction': 'D-mannitol 1-phosphate + NAD+ = D-fructose 6-phosphate + NADH + H+',\n",
       "  'other_names': 'hexose reductase; mannitol 1-phosphate dehydrogenase; D-mannitol-1-phosphate dehydrogenase; fructose 6-phosphate reductase',\n",
       "  'sys_name': 'D-mannitol-1-phosphate:NAD+ 5-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.18': {'ec_num': '1.1.1.18',\n",
       "  'accepted_name': 'inositol 2-dehydrogenase',\n",
       "  'reaction': 'myo-inositol + NAD+ = 2,4,6/3,5-pentahydroxycyclohexanone + NADH + H+',\n",
       "  'other_names': 'myo-inositol 2-dehydrogenase; myo-inositol:NAD+ oxidoreductase; inositol dehydrogenase; myo-inositol dehydrogenase',\n",
       "  'sys_name': 'myo-inositol:NAD+ 2-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.19': {'ec_num': '1.1.1.19',\n",
       "  'accepted_name': 'glucuronate reductase',\n",
       "  'reaction': 'L-gulonate + NADP+ = D-glucuronate + NADPH + H+',\n",
       "  'other_names': 'L-hexonate:NADP dehydrogenase; TPN-L-gulonate dehydrogenase; NADP-L-gulonate dehydrogenase; D-glucuronate dehydrogenase; D-glucuronate reductase; L-glucuronate reductase (incorrect)',\n",
       "  'sys_name': 'L-gulonate:NADP+ 6-oxidoreductase',\n",
       "  'comments': 'Also reduces D-galacturonate. May be identical with EC 1.1.1.2 [alcohol dehydrogenase (NADP+)].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.20': {'ec_num': '1.1.1.20',\n",
       "  'accepted_name': 'glucuronolactone reductase',\n",
       "  'reaction': 'L-gulono-1,4-lactone + NADP+ = D-glucurono-3,6-lactone + NADPH + H+',\n",
       "  'other_names': 'GRase; gulonolactone dehydrogenase',\n",
       "  'sys_name': 'L-gulono-1,4-lactone:NADP+ 1-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.21': {'ec_num': '1.1.1.21',\n",
       "  'accepted_name': 'aldose reductase',\n",
       "  'reaction': 'alditol + NAD(P)+ = aldose + NAD(P)H + H+',\n",
       "  'other_names': 'polyol dehydrogenase (NADP+); ALR2; alditol:NADP+ oxidoreductase; alditol:NADP+ 1-oxidoreductase; NADPH-aldopentose reductase; NADPH-aldose reductase; aldehyde reductase (misleading)',\n",
       "  'sys_name': 'alditol:NAD(P)+ 1-oxidoreductase',\n",
       "  'comments': 'Has wide specificity.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.22': {'ec_num': '1.1.1.22',\n",
       "  'accepted_name': 'UDP-glucose 6-dehydrogenase',\n",
       "  'reaction': 'UDP-α-D-glucose + 2 NAD+ + H2O = UDP-α-D-glucuronate + 2 NADH + 2 H+',\n",
       "  'other_names': 'UDP-glucose dehydrogenase; uridine diphosphoglucose dehydrogenase; UDPG dehydrogenase; UDPG:NAD oxidoreductase; UDP-α-D-glucose:NAD oxidoreductase; UDP-glucose:NAD+ oxidoreductase; uridine diphosphate glucose dehydrogenase; UDP-D-glucose dehydrogenase; uridine diphosphate D-glucose dehydrogenase',\n",
       "  'sys_name': 'UDP-α-D-glucose:NAD+ 6-oxidoreductase',\n",
       "  'comments': 'Also acts on UDP-α-D-2-deoxyglucose.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.23': {'ec_num': '1.1.1.23',\n",
       "  'accepted_name': 'histidinol dehydrogenase',\n",
       "  'reaction': 'L-histidinol + 2 NAD+ + H2O = L-histidine + 2 NADH + 3 H+',\n",
       "  'other_names': 'L-histidinol dehydrogenase',\n",
       "  'sys_name': 'L-histidinol:NAD+ oxidoreductase',\n",
       "  'comments': 'Also oxidizes L-histidinal. The Neurospora enzyme also catalyses the reactions of EC 3.5.4.19 (phosphoribosyl-AMP cyclohydrolase) and EC 3.6.1.31 (phosphoribosyl-ATP diphosphatase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.24': {'ec_num': '1.1.1.24',\n",
       "  'accepted_name': 'quinate/shikimate dehydrogenase (NAD+)',\n",
       "  'reaction': 'L-quinate + NAD+ = 3-dehydroquinate + NADH + H+',\n",
       "  'other_names': 'quinate dehydrogenase (ambiguous); quinic dehydrogenase (ambiguous); quinate:NAD oxidoreductase; quinate 5-dehydrogenase (ambiguous); quinate:NAD+ 5-oxidoreductase',\n",
       "  'sys_name': 'L-quinate:NAD+ 3-oxidoreductase',\n",
       "  'comments': 'The enzyme, found mostly in bacteria (mostly, but not exclusively in Gram-positive bacteria), fungi, and plants, participates in the degradation of quinate and shikimate with a strong preference for NAD+ as a cofactor. While the enzyme can act on both quinate and shikimate, activity is higher with the former. cf. EC 1.1.5.8, quinate/shikimate dehydrogenase (quinone), EC 1.1.1.282, quinate/shikimate dehydrogenase [NAD(P)+], and EC 1.1.1.25, shikimate dehydrogenase (NADP+).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': \"quinate = (1R,3R,4R,5R)-1,3,4,5-tetrahydroxycyclohexanecarboxylic acid and is a cyclitol carboxylate\\nThe numbering system used for the 3-dehydroquinate is that of the recommendations on cyclitols, sections I-8 and I-9: and is shown in the reaction diagram. The use of the term '5-dehydroquinate' for this compound is based on an earlier system of numbering.\",\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.25': {'ec_num': '1.1.1.25',\n",
       "  'accepted_name': 'shikimate dehydrogenase (NADP+)',\n",
       "  'reaction': 'shikimate + NADP+ = 3-dehydroshikimate + NADPH + H+',\n",
       "  'other_names': 'shikimate dehydrogenase; dehydroshikimic reductase; shikimate oxidoreductase; shikimate:NADP+ oxidoreductase; 5-dehydroshikimate reductase; shikimate 5-dehydrogenase; 5-dehydroshikimic reductase; DHS reductase; shikimate:NADP+ 5-oxidoreductase; AroE',\n",
       "  'sys_name': 'shikimate:NADP+ 3-oxidoreductase',\n",
       "  'comments': 'NAD+ cannot replace NADP+ [3]. In higher organisms, this enzyme forms part of a multienzyme complex with EC 4.2.1.10, 3-dehydroquinate dehydratase [4]. cf. EC 1.1.1.24, quinate/shikimate dehydrogenase (NAD+), EC 1.1.5.8, quinate/shikimate dehydrogenase (quinone), and EC 1.1.1.282, quinate/shikimate dehydrogenase [NAD(P)+].',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.26': {'ec_num': '1.1.1.26',\n",
       "  'accepted_name': 'glyoxylate reductase',\n",
       "  'reaction': 'glycolate + NAD+ = glyoxylate + NADH + H+',\n",
       "  'other_names': 'NADH-glyoxylate reductase; glyoxylic acid reductase; NADH-dependent glyoxylate reductase',\n",
       "  'sys_name': 'glycolate:NAD+ oxidoreductase',\n",
       "  'comments': 'Reduces glyoxylate to glycolate or hydroxypyruvate to D-glycerate.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.27': {'ec_num': '1.1.1.27',\n",
       "  'accepted_name': 'L-lactate dehydrogenase',\n",
       "  'reaction': '(S)-lactate + NAD+ = pyruvate + NADH + H+',\n",
       "  'other_names': 'lactic acid dehydrogenase; L(+)-nLDH; L-(+)-lactate dehydrogenase; L-lactic dehydrogenase; L-lactic acid dehydrogenase; lactate dehydrogenase; lactate dehydrogenase NAD-dependent; lactic dehydrogenase; NAD-lactate dehydrogenase',\n",
       "  'sys_name': '(S)-lactate:NAD+ oxidoreductase',\n",
       "  'comments': 'Also oxidizes other (S)-2-hydroxymonocarboxylic acids. NADP+ also acts, more slowly, with the animal, but not the bacterial, enzyme.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.28': {'ec_num': '1.1.1.28',\n",
       "  'accepted_name': 'D-lactate dehydrogenase',\n",
       "  'reaction': '(R)-lactate + NAD+ = pyruvate + NADH + H+',\n",
       "  'other_names': 'lactic acid dehydrogenase; lactic acid dehydrogenase; D-specific lactic dehydrogenase; D-(-)-lactate dehydrogenase (NAD); D-lactic acid dehydrogenase; D-lactic dehydrogenase',\n",
       "  'sys_name': '(R)-lactate:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.29': {'ec_num': '1.1.1.29',\n",
       "  'accepted_name': 'glycerate dehydrogenase',\n",
       "  'reaction': 'D-glycerate + NAD+ = hydroxypyruvate + NADH + H+',\n",
       "  'other_names': 'D-glycerate dehydrogenase; hydroxypyruvate reductase; (R)-glycerate:NAD+ oxidoreductase',\n",
       "  'sys_name': 'D-glycerate:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.30': {'ec_num': '1.1.1.30',\n",
       "  'accepted_name': '3-hydroxybutyrate dehydrogenase',\n",
       "  'reaction': '(R)-3-hydroxybutanoate + NAD+ = acetoacetate + NADH + H+',\n",
       "  'other_names': 'NAD-β-hydroxybutyrate dehydrogenase; hydroxybutyrate oxidoreductase; β-hydroxybutyrate dehydrogenase; D-β-hydroxybutyrate dehydrogenase; D-3-hydroxybutyrate dehydrogenase; D-(-)-3-hydroxybutyrate dehydrogenase; β-hydroxybutyric acid dehydrogenase; 3-D-hydroxybutyrate dehydrogenase; β-hydroxybutyric dehydrogenase',\n",
       "  'sys_name': '(R)-3-hydroxybutanoate:NAD+ oxidoreductase',\n",
       "  'comments': 'Also oxidizes other 3-hydroxymonocarboxylic acids.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.31': {'ec_num': '1.1.1.31',\n",
       "  'accepted_name': '3-hydroxyisobutyrate dehydrogenase',\n",
       "  'reaction': '3-hydroxy-2-methylpropanoate + NAD+ = 2-methyl-3-oxopropanoate + NADH + H+',\n",
       "  'other_names': 'β-hydroxyisobutyrate dehydrogenase',\n",
       "  'sys_name': '3-hydroxy-2-methylpropanoate:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.32': {'ec_num': '1.1.1.32',\n",
       "  'accepted_name': 'mevaldate reductase',\n",
       "  'reaction': '(R)-mevalonate + NAD+ = mevaldate + NADH + H+',\n",
       "  'other_names': 'mevalonic dehydrogenase',\n",
       "  'sys_name': '(R)-mevalonate:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-07-10 14:08:31'},\n",
       " '1.1.1.33': {'ec_num': '1.1.1.33',\n",
       "  'accepted_name': 'mevaldate reductase (NADPH)',\n",
       "  'reaction': '(R)-mevalonate + NADP+ = mevaldate + NADPH + H+',\n",
       "  'other_names': 'mevaldate (reduced nicotinamide adenine dinucleotide phosphate) reductase',\n",
       "  'sys_name': '(R)-mevalonate:NADP+ oxidoreductase',\n",
       "  'comments': 'May be identical with EC 1.1.1.2 [alcohol dehydrogenase (NADP+)].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-07-10 14:08:33'},\n",
       " '1.1.1.34': {'ec_num': '1.1.1.34',\n",
       "  'accepted_name': 'hydroxymethylglutaryl-CoA reductase (NADPH)',\n",
       "  'reaction': '(R)-mevalonate + CoA + 2 NADP+ = (S)-3-hydroxy-3-methylglutaryl-CoA + 2 NADPH + 2 H+',\n",
       "  'other_names': 'hydroxymethylglutaryl coenzyme A reductase (reduced nicotinamide adenine dinucleotide phosphate); 3-hydroxy-3-methylglutaryl-CoA reductase (ambiguous); β-hydroxy-β-methylglutaryl coenzyme A reductase (ambiguous); hydroxymethylglutaryl CoA reductase (NADPH); S-3-hydroxy-3-methylglutaryl-CoA reductase (ambiguous); NADPH-hydroxymethylglutaryl-CoA reductase; HMGCoA reductase-mevalonate:NADP-oxidoreductase (acetylating-CoA); 3-hydroxy-3-methylglutaryl CoA reductase (NADPH); hydroxymethylglutaryl-CoA reductase (NADPH2)',\n",
       "  'sys_name': '(R)-mevalonate:NADP+ oxidoreductase (CoA-acylating)',\n",
       "  'comments': 'The enzyme is inactivated by EC 2.7.11.31 {[hydroxymethylglutaryl-CoA reductase (NADPH)] kinase} and reactivated by EC 3.1.3.47 {[hydroxymethylglutaryl-CoA reductase (NADPH)]-phosphatase}.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.35': {'ec_num': '1.1.1.35',\n",
       "  'accepted_name': '3-hydroxyacyl-CoA dehydrogenase',\n",
       "  'reaction': '(S)-3-hydroxyacyl-CoA + NAD+ = 3-oxoacyl-CoA + NADH + H+',\n",
       "  'other_names': 'β-hydroxyacyl dehydrogenase; β-keto-reductase; 3-keto reductase; 3-hydroxyacyl coenzyme A dehydrogenase; β-hydroxyacyl-coenzyme A synthetase; β-hydroxyacylcoenzyme A dehydrogenase; β-hydroxybutyrylcoenzyme A dehydrogenase; 3-hydroxyacetyl-coenzyme A dehydrogenase; L-3-hydroxyacyl coenzyme A dehydrogenase; L-3-hydroxyacyl CoA dehydrogenase; β-hydroxyacyl CoA dehydrogenase; 3β-hydroxyacyl coenzyme A dehydrogenase; 3-hydroxybutyryl-CoA dehydrogenase; β-ketoacyl-CoA reductase; β-hydroxy acid dehydrogenase; 3-L-hydroxyacyl-CoA dehydrogenase; 3-hydroxyisobutyryl-CoA dehydrogenase; 1-specific DPN-linked β-hydroxybutyric dehydrogenase',\n",
       "  'sys_name': '(S)-3-hydroxyacyl-CoA:NAD+ oxidoreductase',\n",
       "  'comments': 'Also oxidizes S-3-hydroxyacyl-N-acylthioethanolamine and S-3-hydroxyacyl-hydrolipoate. Some enzymes act, more slowly, with NADP+. Broad specificity to acyl chain-length (cf. EC 1.1.1.211 [long-chain-3-hydroxyacyl-CoA dehydrogenase]).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.35\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.35\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.35\" target=\"new\">EXPASY</a>, <a href=\"https://randr.nist.gov/enzyme/Default.aspx\" target=\"new\">GTD</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.35\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.1.1.35%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.36': {'ec_num': '1.1.1.36',\n",
       "  'accepted_name': 'acetoacetyl-CoA reductase',\n",
       "  'reaction': '(R)-3-hydroxyacyl-CoA + NADP+ = 3-oxoacyl-CoA + NADPH + H+',\n",
       "  'other_names': 'acetoacetyl coenzyme A reductase; hydroxyacyl coenzyme-A dehydrogenase; NADP-linked acetoacetyl CoA reductase; NADPH:acetoacetyl-CoA reductase; D(–)-β-hydroxybutyryl CoA-NADP oxidoreductase; short chain β-ketoacetyl(acetoacetyl)-CoA reductase; β-ketoacyl-CoA reductase; D-3-hydroxyacyl-CoA reductase; (R)-3-hydroxyacyl-CoA dehydrogenase',\n",
       "  'sys_name': '(R)-3-hydroxyacyl-CoA:NADP+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.36\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.36\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.36\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.36\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.1.1.36%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.37': {'ec_num': '1.1.1.37',\n",
       "  'accepted_name': 'malate dehydrogenase',\n",
       "  'reaction': '(S)-malate + NAD+ = oxaloacetate + NADH + H+',\n",
       "  'other_names': 'malic dehydrogenase; L-malate dehydrogenase; NAD-L-malate dehydrogenase; malic acid dehydrogenase; NAD-dependent malic dehydrogenase; NAD-malate dehydrogenase; NAD-malic dehydrogenase; malate (NAD) dehydrogenase; NAD-dependent malate dehydrogenase; NAD-specific malate dehydrogenase; NAD-linked malate dehydrogenase; MDH (ambiguous); L-malate-NAD+ oxidoreductase',\n",
       "  'sys_name': '(S)-malate:NAD+ oxidoreductase',\n",
       "  'comments': 'There are several forms of malate dehydrogenases that differ by their use of substrate and cofactors. This NAD+-dependent enzyme forms oxaloacetate and unlike EC 1.1.1.38, malate dehydrogenase (oxaloacetate-decarboxylating), is unable to convert it to pyruvate. Also oxidizes some other 2-hydroxydicarboxylic acids. cf. EC 1.1.1.82, malate dehydrogenase (NADP+); EC 1.1.1.299, malate dehydrogenase [NAD(P)+]; and EC 1.1.5.4, malate dehydrogenase (quinone).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.38': {'ec_num': '1.1.1.38',\n",
       "  'accepted_name': 'malate dehydrogenase (oxaloacetate-decarboxylating)',\n",
       "  'reaction': '(1) (S)-malate + NAD+ = pyruvate + CO2 + NADH;;(2) oxaloacetate = pyruvate + CO2',\n",
       "  'other_names': '’malic’ enzyme (ambiguous); pyruvic-malic carboxylase (ambiguous); NAD+-specific malic enzyme; NAD+-malic enzyme; NAD+-linked malic enzyme',\n",
       "  'sys_name': '(S)-malate:NAD+ oxidoreductase (oxaloacetate-decarboxylating)',\n",
       "  'comments': 'Unlike EC 1.1.1.39, malate dehydrogenase (decarboxylating), this enzyme can also decarboxylate oxaloacetate. cf. EC 1.1.1.40, malate dehydrogenase (oxaloacetate-decarboxylating) (NADP+).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.39': {'ec_num': '1.1.1.39',\n",
       "  'accepted_name': 'malate dehydrogenase (decarboxylating)',\n",
       "  'reaction': '(S)-malate + NAD+ = pyruvate + CO2 + NADH',\n",
       "  'other_names': '’malic’ enzyme (ambiguous); pyruvic-malic carboxylase (ambiguous); NAD-specific malic enzyme (ambiguous); NAD-malic enzyme (ambiguous); malate dehydrogenase (decarboxylating) (ambiguous)',\n",
       "  'sys_name': '(S)-malate:NAD+ oxidoreductase (decarboxylating)',\n",
       "  'comments': 'There are several forms of malate dehydrogenases that differ in their use of substrates and cofactors. This particular form is found only in the plant kingdom. Unlike EC 1.1.1.38, which catalyses a similar reaction, this enzyme can not bind oxaloacetate, and thus does not decarboxylate exogeneously-added oxaloacetate. cf. EC 1.1.1.37, malate dehydrogenase; EC 1.1.1.38, malate dehydrogenase (oxaloacetate-decarboxylating); and EC 1.1.1.83, D-malate dehydrogenase (decarboxylating).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.40': {'ec_num': '1.1.1.40',\n",
       "  'accepted_name': 'malate dehydrogenase (oxaloacetate-decarboxylating) (NADP+)',\n",
       "  'reaction': '(1) (S)-malate + NADP+ = pyruvate + CO2 + NADPH;;(2) oxaloacetate = pyruvate + CO2',\n",
       "  'other_names': '’malic’ enzyme (ambiguous); pyruvic-malic carboxylase (ambiguous); malate dehydrogenase (decarboxylating, NADP+); NADP+-linked decarboxylating malic enzyme; NADP+-malic enzyme; NADP+-specific malic enzyme; NADP+-specific malate dehydrogenase; malate dehydrogenase (NADP+, decarboxylating); L-malate:NADP+ oxidoreductase',\n",
       "  'sys_name': '(S)-malate:NADP+ oxidoreductase (oxaloacetate-decarboxylating)',\n",
       "  'comments': 'The enzyme catalyses the oxidative decarboxylation of (S)-malate in the presence of NADP+ and divalent metal ions, and the decarboxylation of oxaloacetate. cf. EC 1.1.1.38, malate dehydrogenase (oxaloacetate-decarboxylating), and EC 1.1.1.39, malate dehydrogenase (decarboxylating).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.41': {'ec_num': '1.1.1.41',\n",
       "  'accepted_name': 'isocitrate dehydrogenase (NAD+)',\n",
       "  'reaction': 'isocitrate + NAD+ = 2-oxoglutarate + CO2 + NADH',\n",
       "  'other_names': 'isocitric dehydrogenase; β-ketoglutaric-isocitric carboxylase; isocitric acid dehydrogenase; NAD dependent isocitrate dehydrogenase; NAD isocitrate dehydrogenase; NAD-linked isocitrate dehydrogenase; NAD-specific isocitrate dehydrogenase; NAD isocitric dehydrogenase; isocitrate dehydrogenase (NAD); IDH (ambiguous); nicotinamide adenine dinucleotide isocitrate dehydrogenase',\n",
       "  'sys_name': 'isocitrate:NAD+ oxidoreductase (decarboxylating)',\n",
       "  'comments': 'Requires Mn2+ or Mg2+ for activity. Unlike EC 1.1.1.42, isocitrate dehydrogenase (NADP+), oxalosuccinate cannot be used as a substrate. In eukaryotes, isocitrate dehydrogenase exists in two forms: an NAD+-linked enzyme found only in mitochondria and displaying allosteric properties, and a non-allosteric, NADP+-linked enzyme that is found in both mitochondria and cytoplasm [7]. The enzyme from some species can also use NADP+ but much more slowly [9].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'isocitrate = (1R,2S)-1-hydroxypropane-1,2,3-tricarboxylate (previously known as threo-Ds-isocitrate)',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.42': {'ec_num': '1.1.1.42',\n",
       "  'accepted_name': 'isocitrate dehydrogenase (NADP+)',\n",
       "  'reaction': 'isocitrate + NADP+ = 2-oxoglutarate + CO2 + NADPH + H+ (overall reaction);;(1a) isocitrate + NADP+ = oxalosuccinate + NADPH + H+;;(1b) oxalosuccinate = 2-oxoglutarate + CO2',\n",
       "  'other_names': 'oxalosuccinate decarboxylase; oxalsuccinic decarboxylase; isocitrate (NADP) dehydrogenase; isocitrate (nicotinamide adenine dinucleotide phosphate) dehydrogenase; NADP-specific isocitrate dehydrogenase; NADP-linked isocitrate dehydrogenase; NADP-dependent isocitrate dehydrogenase; NADP isocitric dehydrogenase; isocitrate dehydrogenase (NADP-dependent); NADP-dependent isocitric dehydrogenase; triphosphopyridine nucleotide-linked isocitrate dehydrogenase-oxalosuccinate carboxylase; NADP+-linked isocitrate dehydrogenase; IDH (ambiguous); dual-cofactor-specific isocitrate dehydrogenase; NADP+-ICDH; NADP+-IDH; IDP; IDP1; IDP2; IDP3',\n",
       "  'sys_name': 'isocitrate:NADP+ oxidoreductase (decarboxylating)',\n",
       "  'comments': 'Requires Mn2+ or Mg2+ for activity. Unlike EC 1.1.1.41, isocitrate dehydrogenase (NAD+), oxalosuccinate can be used as a substrate. In eukaryotes, isocitrate dehydrogenase exists in two forms: an NAD+-linked enzyme found only in mitochondria and displaying allosteric properties, and a non-allosteric, NADP+-linked enzyme that is found in both mitochondria and cytoplasm [6]. The enzyme from some species can also use NAD+ but much more slowly [6,7].',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': ' isocitrate = (1R,2S)-1-hydroxypropane-1,2,3-tricarboxylate (previously known as threo-Ds-isocitrate)\\noxalosuccinate = 1-oxopropane-1,2,3-tricarboxylate',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.43': {'ec_num': '1.1.1.43',\n",
       "  'accepted_name': 'phosphogluconate 2-dehydrogenase',\n",
       "  'reaction': '6-phospho-D-gluconate + NAD(P)+ = 6-phospho-2-dehydro-D-gluconate + NAD(P)H + H+',\n",
       "  'other_names': '6-phosphogluconic dehydrogenase; phosphogluconate dehydrogenase; gluconate 6-phosphate dehydrogenase; 6-phosphogluconate dehydrogenase (NAD); 2-keto-6-phosphogluconate reductase',\n",
       "  'sys_name': '6-phospho-D-gluconate:NAD(P)+ 2-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.44': {'ec_num': '1.1.1.44',\n",
       "  'accepted_name': 'phosphogluconate dehydrogenase (NADP+-dependent, decarboxylating)',\n",
       "  'reaction': '6-phospho-D-gluconate + NADP+ = D-ribulose 5-phosphate + CO2 + NADPH + H+',\n",
       "  'other_names': 'phosphogluconic acid dehydrogenase; 6-phosphogluconic dehydrogenase; 6-phosphogluconic carboxylase; 6-phosphogluconate dehydrogenase (decarboxylating); 6-phospho-D-gluconate dehydrogenase',\n",
       "  'sys_name': '6-phospho-D-gluconate:NADP+ 2-oxidoreductase (decarboxylating)',\n",
       "  'comments': 'The enzyme participates in the oxidative branch of the pentose phosphate pathway, whose main purpose is to produce NADPH and pentose for biosynthetic reactions. Highly specific for NADP+. cf. EC 1.1.1.343, phosphogluconate dehydrogenase (NAD+-dependent, decarboxylating).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.45': {'ec_num': '1.1.1.45',\n",
       "  'accepted_name': 'L-gulonate 3-dehydrogenase',\n",
       "  'reaction': 'L-gulonate + NAD+ = 3-dehydro-L-gulonate + NADH + H+',\n",
       "  'other_names': 'L-3-aldonate dehydrogenase; L-3-aldonic dehydrogenase; L-gulonic acid dehydrogenase; L-β-hydroxyacid dehydrogenase; L-β-hydroxy-acid-NAD-oxidoreductase; L-3-hydroxyacid dehydrogenase',\n",
       "  'sys_name': 'L-gulonate:NAD+ 3-oxidoreductase',\n",
       "  'comments': 'Also oxidizes other L-3-hydroxyacids.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.46': {'ec_num': '1.1.1.46',\n",
       "  'accepted_name': 'L-arabinose 1-dehydrogenase',\n",
       "  'reaction': 'L-arabinose + NAD+ = L-arabinono-1,4-lactone + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'L-arabinose:NAD+ 1-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.47': {'ec_num': '1.1.1.47',\n",
       "  'accepted_name': 'glucose 1-dehydrogenase [NAD(P)+]',\n",
       "  'reaction': 'D-glucose + NAD(P)+ = D-glucono-1,5-lactone + NAD(P)H + H+',\n",
       "  'other_names': 'D-glucose dehydrogenase (NAD(P)+); hexose phosphate dehydrogenase; β-D-glucose:NAD(P)+ 1-oxidoreductase; glucose 1-dehydrogenase',\n",
       "  'sys_name': 'D-glucose:NAD(P)+ 1-oxidoreductase',\n",
       "  'comments': 'This enzyme has similar activity with either NAD+ or NADP+. cf. EC 1.1.1.118, glucose 1-dehydrogenase (NAD+) and EC 1.1.1.119, glucose 1-dehydrogenase (NADP+).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.48': {'ec_num': '1.1.1.48',\n",
       "  'accepted_name': 'D-galactose 1-dehydrogenase',\n",
       "  'reaction': 'D-galactose + NAD+ = D-galactono-1,4-lactone + NADH + H+',\n",
       "  'other_names': 'D-galactose dehydrogenase; β-galactose dehydrogenase (ambiguous); NAD+-dependent D-galactose dehydrogenase',\n",
       "  'sys_name': 'D-galactose:NAD+ 1-oxidoreductase',\n",
       "  'comments': 'This enzyme is part of the De Ley-Doudoroff pathway, which is used by some bacteria during growth on D-galactose.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.49': {'ec_num': '1.1.1.49',\n",
       "  'accepted_name': 'glucose-6-phosphate dehydrogenase (NADP+)',\n",
       "  'reaction': 'D-glucose 6-phosphate + NADP+ = 6-phospho-D-glucono-1,5-lactone + NADPH + H+',\n",
       "  'other_names': 'NADP-glucose-6-phosphate dehydrogenase; Zwischenferment; D-glucose 6-phosphate dehydrogenase; glucose 6-phosphate dehydrogenase (NADP); NADP-dependent glucose 6-phosphate dehydrogenase; 6-phosphoglucose dehydrogenase; Entner-Doudoroff enzyme; glucose-6-phosphate 1-dehydrogenase; G6PDH; GPD; glucose-6-phosphate dehydrogenase',\n",
       "  'sys_name': 'D-glucose-6-phosphate:NADP+ 1-oxidoreductase',\n",
       "  'comments': 'The enzyme catalyses a step of the pentose phosphate pathway. The enzyme is specific for NADP+. cf. EC 1.1.1.363, glucose-6-phosphate dehydrogenase [NAD(P)+] and EC 1.1.1.388, glucose-6-phosphate dehydrogenase (NAD+).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.50': {'ec_num': '1.1.1.50',\n",
       "  'accepted_name': '3α-hydroxysteroid 3-dehydrogenase (Si-specific)',\n",
       "  'reaction': 'a 3α-hydroxysteroid + NAD(P)+ = a 3-oxosteroid + NAD(P)H + H+',\n",
       "  'other_names': 'hydroxyprostaglandin dehydrogenase; 3α-hydroxysteroid oxidoreductase; sterognost 3α; 3α-hydroxysteroid dehydrogenase (B-specific); 3α-hydroxysteroid 3-dehydrogenase (B-specific); 3α-hydroxysteroid:NAD(P)+ 3-oxidoreductase (B-specific)',\n",
       "  'sys_name': '3α-hydroxysteroid:NAD(P)+ 3-oxidoreductase (Si-specific)',\n",
       "  'comments': 'The enzyme acts on androsterone and other 3α-hydroxysteroids and on 9-, 11- and 15-hydroxyprostaglandin. Si-specific with respect to NAD+ or NADP+. cf. EC 1.1.1.213, 3α-hydroxysteroid 3-dehydrogenase (Re-specific).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.51': {'ec_num': '1.1.1.51',\n",
       "  'accepted_name': '3(or 17)β-hydroxysteroid dehydrogenase',\n",
       "  'reaction': 'testosterone + NAD(P)+ = androstenedione + NAD(P)H + H+',\n",
       "  'other_names': 'β-hydroxy steroid dehydrogenase; 17-ketoreductase; 17β-hydroxy steroid dehydrogenase; 3β-hydroxysteroid dehydrogenase; 3β-hydroxy steroid dehydrogenase',\n",
       "  'sys_name': '3(or 17)β-hydroxysteroid:NAD(P)+ oxidoreductase',\n",
       "  'comments': 'Also acts on other 3β- or 17β-hydroxysteroids. cf. EC 1.1.1.209 3(or 17)α-hydroxysteroid dehydrogenase.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.51\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.51\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.51\" target=\"new\">EXPASY</a>, <a href=\"https://randr.nist.gov/enzyme/Default.aspx\" target=\"new\">GTD</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.51\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.1.1.51%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': 'androstenedione = androst-4-ene-3,17-dione',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.52': {'ec_num': '1.1.1.52',\n",
       "  'accepted_name': '3α-hydroxycholanate dehydrogenase (NAD+)',\n",
       "  'reaction': 'lithocholate + NAD+ = 3-oxo-5β-cholan-24-oate + NADH + H+',\n",
       "  'other_names': 'α-hydroxy-cholanate dehydrogenase; lithocholate:NAD+ oxidoreductase; 3α-hydroxycholanate dehydrogenase',\n",
       "  'sys_name': 'lithocholate:NAD+ 3-oxidoreductase',\n",
       "  'comments': 'Also acts on other 3α-hydroxysteroids with an acidic side-chain. cf. EC 1.1.1.392, 3α-hydroxycholanate dehydrogenase (NADP+).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'lithocholate = 3α-hydroxy-5β-cholan-24-oate',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.53': {'ec_num': '1.1.1.53',\n",
       "  'accepted_name': '3α(or 20β)-hydroxysteroid dehydrogenase',\n",
       "  'reaction': 'androstan-3α,17β-diol + NAD+ = 17β-hydroxyandrostan-3-one + NADH + H+',\n",
       "  'other_names': 'cortisone reductase; (R)-20-hydroxysteroid dehydrogenase; 20β-hydroxy steroid dehydrogenase; Δ4-3-ketosteroid hydrogenase; 20β-hydroxysteroid dehydrogenase; 3α,20β-hydroxysteroid:NAD+-oxidoreductase; NADH-20β-hydroxysteroid dehydrogenase; 20β-HSD',\n",
       "  'sys_name': '3α(or 20β)-hydroxysteroid:NAD+ oxidoreductase',\n",
       "  'comments': 'The 3α-hydroxy group or 20β-hydroxy group of pregnane and androstane steroids can act as donor.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.54': {'ec_num': '1.1.1.54',\n",
       "  'accepted_name': 'allyl-alcohol dehydrogenase',\n",
       "  'reaction': 'allyl alcohol + NADP+ = acrolein + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'allyl-alcohol:NADP+ oxidoreductase',\n",
       "  'comments': 'Also acts on saturated primary alcohols.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.55': {'ec_num': '1.1.1.55',\n",
       "  'accepted_name': 'lactaldehyde reductase (NADPH)',\n",
       "  'reaction': 'propane-1,2-diol + NADP+ = L-lactaldehyde + NADPH + H+',\n",
       "  'other_names': 'lactaldehyde (reduced nicotinamide adenine dinucleotide phosphate) reductase; NADP-1,2-propanediol dehydrogenase; propanediol dehydrogenase; 1,2-propanediol:NADP+ oxidoreductase; lactaldehyde reductase (NADPH2)',\n",
       "  'sys_name': 'propane-1,2-diol:NADP+ oxidoreductase',\n",
       "  'comments': 'May be identical with EC 1.1.1.2 alcohol dehydrogenase (NADP+).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.56': {'ec_num': '1.1.1.56',\n",
       "  'accepted_name': 'ribitol 2-dehydrogenase',\n",
       "  'reaction': 'ribitol + NAD+ = D-ribulose + NADH + H+',\n",
       "  'other_names': 'adonitol dehydrogenase; ribitol dehydrogenase A (wild type); ribitol dehydrogenase B (mutant enzyme with different properties); ribitol dehydrogenase D (mutant enzyme with different properties)',\n",
       "  'sys_name': 'ribitol:NAD+ 2-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.57': {'ec_num': '1.1.1.57',\n",
       "  'accepted_name': 'fructuronate reductase',\n",
       "  'reaction': 'D-mannonate + NAD+ = D-fructuronate + NADH + H+',\n",
       "  'other_names': 'mannonate oxidoreductase; mannonic dehydrogenase; D-mannonate dehydrogenase; D-mannonate:NAD oxidoreductase',\n",
       "  'sys_name': 'D-mannonate:NAD+ 5-oxidoreductase',\n",
       "  'comments': 'Also reduces D-tagaturonate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.58': {'ec_num': '1.1.1.58',\n",
       "  'accepted_name': 'tagaturonate reductase',\n",
       "  'reaction': 'D-altronate + NAD+ = D-tagaturonate + NADH + H+',\n",
       "  'other_names': 'altronic oxidoreductase; altronate oxidoreductase; TagUAR; altronate dehydrogenase; D-tagaturonate reductase',\n",
       "  'sys_name': 'D-altronate:NAD+ 3-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.59': {'ec_num': '1.1.1.59',\n",
       "  'accepted_name': '3-hydroxypropionate dehydrogenase',\n",
       "  'reaction': '3-hydroxypropanoate + NAD+ = 3-oxopropanoate + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '3-hydroxypropanoate:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.60': {'ec_num': '1.1.1.60',\n",
       "  'accepted_name': '2-hydroxy-3-oxopropionate reductase',\n",
       "  'reaction': 'D-glycerate + NAD(P)+ = 2-hydroxy-3-oxopropanoate + NAD(P)H + H+',\n",
       "  'other_names': 'tartronate semialdehyde reductase; (R)-glycerate:NAD(P)+ oxidoreductase',\n",
       "  'sys_name': 'D-glycerate:NAD(P)+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.61': {'ec_num': '1.1.1.61',\n",
       "  'accepted_name': '4-hydroxybutyrate dehydrogenase',\n",
       "  'reaction': '4-hydroxybutanoate + NAD+ = succinate semialdehyde + NADH + H+',\n",
       "  'other_names': 'γ-hydroxybutyrate dehydrogenase',\n",
       "  'sys_name': '4-hydroxybutanoate:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.62': {'ec_num': '1.1.1.62',\n",
       "  'accepted_name': '17β-estradiol 17-dehydrogenase',\n",
       "  'reaction': '17β-estradiol + NAD(P)+ = estrone + NAD(P)H + H+',\n",
       "  'other_names': '20α-hydroxysteroid dehydrogenase; 17β,20α-hydroxysteroid dehydrogenase; 17β-estradiol dehydrogenase; estradiol dehydrogenase; estrogen 17-oxidoreductase; 17β-HSD; HSD17B7',\n",
       "  'sys_name': '17β-estradiol:NAD(P)+ 17-oxidoreductase',\n",
       "  'comments': 'The enzyme oxidizes or reduces the hydroxy/keto group on C17 of estrogens and androgens in mammals and regulates the biological potency of these steroids. The mammalian enzyme is bifunctional and also catalyses EC 1.1.1.270, 3β-hydroxysteroid 3-dehydrogenase [3]. The enzyme also acts on (S)-20-hydroxypregn-4-en-3-one and related compounds, oxidizing the (S)-20-group, but unlike EC 1.1.1.149, 20α-hydroxysteroid dehydrogenase, it is Si-specific with respect to NAD(P)+.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.63': {'ec_num': '1.1.1.63',\n",
       "  'accepted_name': 'testosterone 17β-dehydrogenase',\n",
       "  'reaction': 'testosterone + NAD+ = androstenedione + NADH + H+',\n",
       "  'other_names': '17-ketoreductase; 17β-HSD',\n",
       "  'sys_name': '17β-hydroxysteroid:NAD+ 17-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'androstenedione = androst-4-ene-3,17-dione',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.64': {'ec_num': '1.1.1.64',\n",
       "  'accepted_name': 'testosterone 17β-dehydrogenase (NADP+)',\n",
       "  'reaction': 'testosterone + NADP+ = androstenedione + NADPH + H+',\n",
       "  'other_names': '17-ketoreductase; NADP-dependent testosterone-17β-oxidoreductase; testosterone 17β-dehydrogenase (NADP)',\n",
       "  'sys_name': '17β-hydroxysteroid:NADP+ 17-oxidoreductase',\n",
       "  'comments': 'Also oxidizes 3-hydroxyhexobarbital to 3-oxohexobarbital.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'androstenedione = androst-4-ene-3,17-dione',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.65': {'ec_num': '1.1.1.65',\n",
       "  'accepted_name': 'pyridoxine 4-dehydrogenase',\n",
       "  'reaction': 'pyridoxine + NADP+ = pyridoxal + NADPH + H+',\n",
       "  'other_names': 'pyridoxin dehydrogenase; pyridoxol dehydrogenase; pyridoxine dehydrogenase',\n",
       "  'sys_name': 'pyridoxine:NADP+ 4-oxidoreductase',\n",
       "  'comments': 'Also oxidizes pyridoxine phosphate.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.65\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.65\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.65\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.65\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.66': {'ec_num': '1.1.1.66',\n",
       "  'accepted_name': 'ω-hydroxydecanoate dehydrogenase',\n",
       "  'reaction': '10-hydroxydecanoate + NAD+ = 10-oxodecanoate + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '10-hydroxydecanoate:NAD+ 10-oxidoreductase',\n",
       "  'comments': 'Also acts, more slowly, on 9-hydroxynonanoate and 11-hydroxyundecanoate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.67': {'ec_num': '1.1.1.67',\n",
       "  'accepted_name': 'mannitol 2-dehydrogenase',\n",
       "  'reaction': 'D-mannitol + NAD+ = D-fructose + NADH + H+',\n",
       "  'other_names': 'D-mannitol dehydrogenase; mannitol dehydrogenase',\n",
       "  'sys_name': 'D-mannitol:NAD+ 2-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.69': {'ec_num': '1.1.1.69',\n",
       "  'accepted_name': 'gluconate 5-dehydrogenase',\n",
       "  'reaction': 'D-gluconate + NAD(P)+ = 5-dehydro-D-gluconate + NAD(P)H + H+',\n",
       "  'other_names': '5-keto-D-gluconate 5-reductase; 5-keto-D-gluconate 5-reductase; 5-ketogluconate 5-reductase; 5-ketogluconate reductase; 5-keto-D-gluconate reductase',\n",
       "  'sys_name': 'D-gluconate:NAD(P)+ 5-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.71': {'ec_num': '1.1.1.71',\n",
       "  'accepted_name': 'alcohol dehydrogenase [NAD(P)+]',\n",
       "  'reaction': 'an alcohol + NAD(P)+ = an aldehyde + NAD(P)H + H+',\n",
       "  'other_names': 'retinal reductase (ambiguous); aldehyde reductase (NADPH/NADH); alcohol dehydrogenase [NAD(P)]',\n",
       "  'sys_name': 'alcohol:NAD(P)+ oxidoreductase',\n",
       "  'comments': 'Reduces aliphatic aldehydes of carbon chain length from 2 to 14, with greatest activity on C4, C6 and C8 aldehydes; also reduces retinal to retinol.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.71\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.71\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.71\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.71\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.1.1.71%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.72': {'ec_num': '1.1.1.72',\n",
       "  'accepted_name': 'glycerol dehydrogenase (NADP+)',\n",
       "  'reaction': 'glycerol + NADP+ = D-glyceraldehyde + NADPH + H+',\n",
       "  'other_names': 'glycerol dehydrogenase (NADP)',\n",
       "  'sys_name': 'glycerol:NADP+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.73': {'ec_num': '1.1.1.73',\n",
       "  'accepted_name': 'octanol dehydrogenase',\n",
       "  'reaction': 'octan-1-ol + NAD+ = octanal + NADH + H+',\n",
       "  'other_names': '1-octanol dehydrogenase; octanol:NAD+ oxidoreductase',\n",
       "  'sys_name': 'octan-1-ol:NAD+ oxidoreductase',\n",
       "  'comments': 'Acts, less rapidly, on other long-chain alcohols.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.75': {'ec_num': '1.1.1.75',\n",
       "  'accepted_name': '(R)-aminopropanol dehydrogenase',\n",
       "  'reaction': '(R)-1-aminopropan-2-ol + NAD+ = aminoacetone + NADH + H+',\n",
       "  'other_names': 'L-aminopropanol dehydrogenase; 1-aminopropan-2-ol-NAD+ dehydrogenase; L(+)-1-aminopropan-2-ol:NAD+ oxidoreductase; 1-aminopropan-2-ol-dehydrogenase; DL-1-aminopropan-2-ol: NAD+ dehydrogenase; L(+)-1-aminopropan-2-ol-NAD/NADP oxidoreductase',\n",
       "  'sys_name': '(R)-1-aminopropan-2-ol:NAD+ oxidoreductase',\n",
       "  'comments': 'Requires K+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.76': {'ec_num': '1.1.1.76',\n",
       "  'accepted_name': '(S,S)-butanediol dehydrogenase',\n",
       "  'reaction': '(2S,3S)-butane-2,3-diol + NAD+ = (S)-acetoin + NADH + H+',\n",
       "  'other_names': 'L-butanediol dehydrogenase; L-BDH; L(+)-2,3-butanediol dehydrogenase (L-acetoin forming); (S)-acetoin reductase [(S,S)-butane-2,3-diol forming]',\n",
       "  'sys_name': '(S,S)-butane-2,3-diol:NAD+ oxidoreductase',\n",
       "  'comments': 'This enzyme catalyses the reversible reduction of (S)-acetoin to (S,S)-butane-2,3-diol. It can also catalyse the irreversible reduction of diacetyl to (S)-acetoin.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.77': {'ec_num': '1.1.1.77',\n",
       "  'accepted_name': 'lactaldehyde reductase',\n",
       "  'reaction': '(R)[or (S)]-propane-1,2-diol + NAD+ = (R)[or (S)]-lactaldehyde + NADH + H+',\n",
       "  'other_names': 'propanediol:nicotinamide adenine dinucleotide (NAD) oxidoreductase; L-lactaldehyde:propanediol oxidoreductase',\n",
       "  'sys_name': '(R)[or (S)]-propane-1,2-diol:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.78': {'ec_num': '1.1.1.78',\n",
       "  'accepted_name': 'methylglyoxal reductase (NADH)',\n",
       "  'reaction': '(R)-lactaldehyde + NAD+ = 2-oxopropanal + NADH + H+',\n",
       "  'other_names': 'methylglyoxal reductase; D-lactaldehyde dehydrogenase; methylglyoxal reductase (NADH-dependent)',\n",
       "  'sys_name': '(R)-lactaldehyde:NAD+ oxidoreductase',\n",
       "  'comments': 'This mammalian enzyme differs from the yeast enzyme, EC 1.1.1.283, methylglyoxal reductase (NADPH), by its cosubstrate requirement, reaction direction, and enantiomeric preference.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '2-oxopropanal = methylglyoxal',\n",
       "  'last_change': '2023-05-19 09:09:01'},\n",
       " '1.1.1.79': {'ec_num': '1.1.1.79',\n",
       "  'accepted_name': 'glyoxylate reductase (NADP+)',\n",
       "  'reaction': 'glycolate + NADP+ = glyoxylate + NADPH + H+',\n",
       "  'other_names': 'NADPH-glyoxylate reductase; glyoxylate reductase (NADP)',\n",
       "  'sys_name': 'glycolate:NADP+ oxidoreductase',\n",
       "  'comments': 'Also reduces hydroxypyruvate to glycerate; has some affinity for NAD+.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.80': {'ec_num': '1.1.1.80',\n",
       "  'accepted_name': 'isopropanol dehydrogenase (NADP+)',\n",
       "  'reaction': 'propan-2-ol + NADP+ = acetone + NADPH + H+',\n",
       "  'other_names': 'isopropanol dehydrogenase (NADP)',\n",
       "  'sys_name': 'propan-2-ol:NADP+ oxidoreductase',\n",
       "  'comments': 'Also acts on other short-chain secondary alcohols and, slowly, on primary alcohols.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.81': {'ec_num': '1.1.1.81',\n",
       "  'accepted_name': 'hydroxypyruvate reductase',\n",
       "  'reaction': 'D-glycerate + NAD(P)+ = hydroxypyruvate + NAD(P)H + H+',\n",
       "  'other_names': 'β-hydroxypyruvate reductase; NADH:hydroxypyruvate reductase; D-glycerate dehydrogenase',\n",
       "  'sys_name': 'D-glycerate:NADP+ 2-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.82': {'ec_num': '1.1.1.82',\n",
       "  'accepted_name': 'malate dehydrogenase (NADP+)',\n",
       "  'reaction': '(S)-malate + NADP+ = oxaloacetate + NADPH + H+',\n",
       "  'other_names': 'NADP-malic enzyme; NADP-malate dehydrogenase; malic dehydrogenase (nicotinamide adenine dinucleotide phosphate); malate NADP dehydrogenase; NADP malate dehydrogenase; NADP-linked malate dehydrogenase; malate dehydrogenase (NADP)',\n",
       "  'sys_name': '(S)-malate:NADP+ oxidoreductase',\n",
       "  'comments': 'Activated by light.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.83': {'ec_num': '1.1.1.83',\n",
       "  'accepted_name': 'D-malate dehydrogenase (decarboxylating)',\n",
       "  'reaction': '(R)-malate + NAD+ = pyruvate + CO2 + NADH',\n",
       "  'other_names': 'D-malate dehydrogenase; D-malic enzyme; bifunctional L(+)-tartrate dehydrogenase-D(+)-malate (decarboxylating)',\n",
       "  'sys_name': '(R)-malate:NAD+ oxidoreductase (decarboxylating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.84': {'ec_num': '1.1.1.84',\n",
       "  'accepted_name': 'dimethylmalate dehydrogenase',\n",
       "  'reaction': '(R)-3,3-dimethylmalate + NAD+ = 3-methyl-2-oxobutanoate + CO2 + NADH',\n",
       "  'other_names': 'β,β-dimethylmalate dehydrogenase',\n",
       "  'sys_name': '(R)-3,3-dimethylmalate:NAD+ oxidoreductase (decarboxylating)',\n",
       "  'comments': 'Requires K+ or NH4+ and Mn2+ or Co2+; also acts on (R)-malate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.85': {'ec_num': '1.1.1.85',\n",
       "  'accepted_name': '3-isopropylmalate dehydrogenase',\n",
       "  'reaction': '(2R,3S)-3-isopropylmalate + NAD+ = 4-methyl-2-oxopentanoate + CO2 + NADH + H+ (overall reaction);;(1a) (2R,3S)-3-isopropylmalate + NAD+ = (2S)-2-isopropyl-3-oxosuccinate + NADH + H+;;(1b) (2S)-2-isopropyl-3-oxosuccinate = 4-methyl-2-oxopentanoate + CO2 (spontaneous)',\n",
       "  'other_names': 'β-isopropylmalic enzyme; β-isopropylmalate dehydrogenase; threo-Ds-3-isopropylmalate dehydrogenase; 3-carboxy-2-hydroxy-4-methylpentanoate:NAD+ oxidoreductase',\n",
       "  'sys_name': '(2R,3S)-3-isopropylmalate:NAD+ oxidoreductase',\n",
       "  'comments': 'The product decarboxylates spontaneously to yield 4-methyl-2-oxopentanoate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.86': {'ec_num': '1.1.1.86',\n",
       "  'accepted_name': 'ketol-acid reductoisomerase (NADP+)',\n",
       "  'reaction': '(2R)-2,3-dihydroxy-3-methylbutanoate + NADP+ = (2S)-2-hydroxy-2-methyl-3-oxobutanoate + NADPH + H+',\n",
       "  'other_names': 'dihydroxyisovalerate dehydrogenase (isomerizing); acetohydroxy acid isomeroreductase; ketol acid reductoisomerase; α-keto-β-hydroxylacyl reductoisomerase; 2-hydroxy-3-keto acid reductoisomerase; acetohydroxy acid reductoisomerase; acetolactate reductoisomerase; dihydroxyisovalerate (isomerizing) dehydrogenase; isomeroreductase; reductoisomerase; ketol-acid reductoisomerase; (R)-2,3-dihydroxy-3-methylbutanoate:NADP+ oxidoreductase (isomerizing)',\n",
       "  'sys_name': '(2R)-2,3-dihydroxy-3-methylbutanoate:NADP+ oxidoreductase (isomerizing)',\n",
       "  'comments': 'Also catalyses the reduction of 2-ethyl-2-hydroxy-3-oxobutanoate to 2,3-dihydroxy-3-methylpentanoate. The enzyme, found in many bacteria and archaea, is specific for NADPH (cf. EC 1.1.1.382, ketol-acid reductoisomerase (NAD+) and EC 1.1.1.383, ketol-acid reductoisomerase [NAD(P)+]).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '(2S)-2-hydroxy-2-methyl-3-oxobutanoate = (2S)-2-acetolactate',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.87': {'ec_num': '1.1.1.87',\n",
       "  'accepted_name': 'homoisocitrate dehydrogenase',\n",
       "  'reaction': '(1R,2S)-1-hydroxybutane-1,2,4-tricarboxylate + NAD+ = 2-oxoadipate + CO2 + NADH + H+',\n",
       "  'other_names': '2-hydroxy-3-carboxyadipate dehydrogenase; 3-carboxy-2-hydroxyadipate dehydrogenase; homoisocitric dehydrogenase; (-)-1-hydroxy-1,2,4-butanetricarboxylate:NAD+ oxidoreductase (decarboxylating); 3-carboxy-2-hydroxyadipate:NAD+ oxidoreductase (decarboxylating); HICDH',\n",
       "  'sys_name': '(1R,2S)-1-hydroxybutane-1,2,4-tricarboxylate:NAD+ oxidoreductase (decarboxylating)',\n",
       "  'comments': 'Forms part of the lysine biosynthesis pathway in fungi [3].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'homoisocitrate = (-)-threo-homoisocitrate = (1R,2S)-1-hydroxybutane-1,2,4-tricarboxylate',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.88': {'ec_num': '1.1.1.88',\n",
       "  'accepted_name': 'hydroxymethylglutaryl-CoA reductase',\n",
       "  'reaction': '(R)-mevalonate + CoA + 2 NAD+ = 3-hydroxy-3-methylglutaryl-CoA + 2 NADH + 2 H+',\n",
       "  'other_names': 'β-hydroxy-β-methylglutaryl coenzyme A reductase (ambiguous); β-hydroxy-β-methylglutaryl CoA-reductase (ambiguous); 3-hydroxy-3-methylglutaryl coenzyme A reductase (ambiguous); hydroxymethylglutaryl coenzyme A reductase (ambiguous)',\n",
       "  'sys_name': '(R)-mevalonate:NAD+ oxidoreductase (CoA-acylating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.90': {'ec_num': '1.1.1.90',\n",
       "  'accepted_name': 'aryl-alcohol dehydrogenase',\n",
       "  'reaction': 'an aromatic alcohol + NAD+ = an aromatic aldehyde + NADH + H+',\n",
       "  'other_names': 'p-hydroxybenzyl alcohol dehydrogenase; benzyl alcohol dehydrogenase; coniferyl alcohol dehydrogenase',\n",
       "  'sys_name': 'aryl-alcohol:NAD+ oxidoreductase',\n",
       "  'comments': 'A group of enzymes with broad specificity towards primary alcohols with an aromatic or cyclohex-1-ene ring, but with low or no activity towards short-chain aliphatic alcohols.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.90\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.90\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.90\" target=\"new\">EXPASY</a>, <a href=\"https://randr.nist.gov/enzyme/Default.aspx\" target=\"new\">GTD</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.90\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.1.1.90%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.91': {'ec_num': '1.1.1.91',\n",
       "  'accepted_name': 'aryl-alcohol dehydrogenase (NADP+)',\n",
       "  'reaction': 'an aromatic alcohol + NADP+ = an aromatic aldehyde + NADPH + H+',\n",
       "  'other_names': 'aryl alcohol dehydrogenase (nicotinamide adenine dinucleotide phosphate); coniferyl alcohol dehydrogenase; NADPH-linked benzaldehyde reductase; aryl-alcohol dehydrogenase (NADP)',\n",
       "  'sys_name': 'aryl-alcohol:NADP+ oxidoreductase',\n",
       "  'comments': 'Also acts on some aliphatic aldehydes, but cinnamaldehyde was the best substrate found.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.92': {'ec_num': '1.1.1.92',\n",
       "  'accepted_name': 'oxaloglycolate reductase (decarboxylating)',\n",
       "  'reaction': 'D-glycerate + NAD(P)+ + CO2 = 2-hydroxy-3-oxosuccinate + NAD(P)H + 2 H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'D-glycerate:NAD(P)+ oxidoreductase (carboxylating)',\n",
       "  'comments': 'Also reduces hydroxypyruvate to D-glycerate and glyoxylate to glycolate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.93': {'ec_num': '1.1.1.93',\n",
       "  'accepted_name': 'tartrate dehydrogenase',\n",
       "  'reaction': 'tartrate + NAD+ = oxaloglycolate + NADH + H+',\n",
       "  'other_names': 'mesotartrate dehydrogenase',\n",
       "  'sys_name': 'tartrate:NAD+ oxidoreductase',\n",
       "  'comments': 'meso-tartrate and (R,R)-tartrate act as substrates. Requires Mn2+ and a monovalent cation.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.94': {'ec_num': '1.1.1.94',\n",
       "  'accepted_name': 'glycerol-3-phosphate dehydrogenase [NAD(P)+]',\n",
       "  'reaction': 'sn-glycerol 3-phosphate + NAD(P)+ = glycerone phosphate + NAD(P)H + H+',\n",
       "  'other_names': 'L-glycerol-3-phosphate:NAD(P) oxidoreductase; glycerol phosphate dehydrogenase (nicotinamide adenine dinucleotide (phosphate)); glycerol 3-phosphate dehydrogenase (NADP); glycerol-3-phosphate dehydrogenase [NAD(P)]',\n",
       "  'sys_name': 'sn-glycerol-3-phosphate:NAD(P)+ 2-oxidoreductase',\n",
       "  'comments': 'The enzyme from Escherichia coli shows specificity for the B side of NADPH.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'glycerone phosphate = dihydroxyacetone phosphate = 3-hydroxy-2-oxopropyl phosphate',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.95': {'ec_num': '1.1.1.95',\n",
       "  'accepted_name': 'phosphoglycerate dehydrogenase',\n",
       "  'reaction': '3-phospho-D-glycerate + NAD+ = 3-phosphooxypyruvate + NADH + H+',\n",
       "  'other_names': 'PHGDH (gene name); D-3-phosphoglycerate:NAD+ oxidoreductase; α-phosphoglycerate dehydrogenase; 3-phosphoglycerate dehydrogenase; 3-phosphoglyceric acid dehydrogenase; D-3-phosphoglycerate dehydrogenase; glycerate 3-phosphate dehydrogenase; glycerate-1,3-phosphate dehydrogenase; phosphoglycerate oxidoreductase; phosphoglyceric acid dehydrogenase; SerA; 3-phosphoglycerate:NAD+ 2-oxidoreductase; SerA 3PG dehydrogenase; 3PHP reductase',\n",
       "  'sys_name': '3-phospho-D-glycerate:NAD+ 2-oxidoreductase',\n",
       "  'comments': 'This enzyme catalyses the first committed and rate-limiting step in the phosphoserine pathway of serine biosynthesis. The reaction occurs predominantly in the direction of reduction. The enzyme from the bacterium Escherichia coli also catalyses the activity of EC 1.1.1.399, 2-oxoglutarate reductase [6].',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.3.98.4': {'ec_num': '1.3.98.4',\n",
       "  'accepted_name': '5a,11a-dehydrotetracycline reductase',\n",
       "  'reaction': ' tetracycline + oxidized coenzyme F420 = 5a,11a-dehydrotetracycline + reduced coenzyme F420',\n",
       "  'other_names': 'oxyR (gene name); 12-dehydrotetracycline dehydrogenase; dehydrooxytetracycline dehydrogenase; 12-dehydrotetracycline reductase',\n",
       "  'sys_name': 'tetracycline:coenzyme F420 dehydrogenase',\n",
       "  'comments': 'The enzyme, characterized from the bacteria Streptomyces aureofaciens and Streptomyces rimosus, catalyses the last step in the biosynthesis of the tetracycline antibiotics tetracycline and oxytetracycline.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.1.1.96': {'ec_num': '1.1.1.96',\n",
       "  'accepted_name': 'diiodophenylpyruvate reductase',\n",
       "  'reaction': '3-(3,5-diiodo-4-hydroxyphenyl)lactate + NAD+ = 3-(3,5-diiodo-4-hydroxyphenyl)pyruvate + NADH + H+',\n",
       "  'other_names': 'aromatic α-keto acid; KAR; 2-oxo acid reductase',\n",
       "  'sys_name': '3-(3,5-diiodo-4-hydroxyphenyl)lactate:NAD+ oxidoreductase',\n",
       "  'comments': 'Substrates contain an aromatic ring with a pyruvate side chain. The most active substrates are halogenated derivatives. Compounds with hydroxy or amino groups in the 3 or 5 position are inactive.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.97': {'ec_num': '1.1.1.97',\n",
       "  'accepted_name': '3-hydroxybenzyl-alcohol dehydrogenase',\n",
       "  'reaction': '3-hydroxybenzyl alcohol + NADP+ = 3-hydroxybenzaldehyde + NADPH + H+',\n",
       "  'other_names': 'm-hydroxybenzyl alcohol dehydrogenase; m-hydroxybenzyl alcohol (NADP) dehydrogenase; m-hydroxybenzylalcohol dehydrogenase',\n",
       "  'sys_name': '3-hydroxybenzyl-alcohol:NADP+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.98': {'ec_num': '1.1.1.98',\n",
       "  'accepted_name': '(R)-2-hydroxy-fatty-acid dehydrogenase',\n",
       "  'reaction': '(R)-2-hydroxystearate + NAD+ = 2-oxostearate + NADH + H+',\n",
       "  'other_names': 'D-2-hydroxy fatty acid dehydrogenase; 2-hydroxy fatty acid oxidase',\n",
       "  'sys_name': '(R)-2-hydroxystearate:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.99': {'ec_num': '1.1.1.99',\n",
       "  'accepted_name': '(S)-2-hydroxy-fatty-acid dehydrogenase',\n",
       "  'reaction': '(S)-2-hydroxystearate + NAD+ = 2-oxostearate + NADH + H+',\n",
       "  'other_names': 'dehydrogenase, L-2-hydroxy fatty acid; L-2-hydroxy fatty acid dehydrogenase; 2-hydroxy fatty acid oxidase',\n",
       "  'sys_name': '(S)-2-hydroxystearate:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.100': {'ec_num': '1.1.1.100',\n",
       "  'accepted_name': '3-oxoacyl-[acyl-carrier-protein] reductase',\n",
       "  'reaction': 'a (3R)-3-hydroxyacyl-[acyl-carrier protein] + NADP+ = a 3-oxoacyl-[acyl-carrier protein] + NADPH + H+',\n",
       "  'other_names': 'β-ketoacyl-[acyl-carrier protein](ACP) reductase; β-ketoacyl acyl carrier protein (ACP) reductase; β-ketoacyl reductase; β-ketoacyl thioester reductase; β-ketoacyl-ACP reductase; β-ketoacyl-acyl carrier protein reductase; 3-ketoacyl acyl carrier protein reductase; NADPH-specific 3-oxoacyl-[acylcarrier protein]reductase; 3-oxoacyl-[ACP]reductase; (3R)-3-hydroxyacyl-[acyl-carrier-protein]:NADP+ oxidoreductase',\n",
       "  'sys_name': '(3R)-3-hydroxyacyl-[acyl-carrier protein]:NADP+ oxidoreductase',\n",
       "  'comments': 'Exhibits a marked preference for acyl-carrier-protein derivatives over CoA derivatives as substrates.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.101': {'ec_num': '1.1.1.101',\n",
       "  'accepted_name': 'acylglycerone-phosphate reductase',\n",
       "  'reaction': '1-palmitoylglycerol 3-phosphate + NADP+ = palmitoylglycerone phosphate + NADPH + H+',\n",
       "  'other_names': 'palmitoyldihydroxyacetone-phosphate reductase; palmitoyl dihydroxyacetone phosphate reductase; palmitoyl-dihydroxyacetone-phosphate reductase; acyldihydroxyacetone phosphate reductase; palmitoyl dihydroxyacetone phosphate reductase',\n",
       "  'sys_name': '1-palmitoylglycerol-3-phosphate:NADP+ oxidoreductase',\n",
       "  'comments': 'Also acts on alkylglycerone 3-phosphate and alkylglycerol 3-phosphate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.102': {'ec_num': '1.1.1.102',\n",
       "  'accepted_name': '3-dehydrosphinganine reductase',\n",
       "  'reaction': 'sphinganine + NADP+ = 3-dehydrosphinganine + NADPH + H+',\n",
       "  'other_names': 'D-3-dehydrosphinganine reductase; D-3-oxosphinganine reductase; DSR; 3-oxosphinganine reductase; 3-oxosphinganine:NADPH oxidoreductase; D-3-oxosphinganine:B-NADPH oxidoreductase',\n",
       "  'sys_name': 'D-erythro-dihydrosphingosine:NADP+ 3-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.103': {'ec_num': '1.1.1.103',\n",
       "  'accepted_name': 'L-threonine 3-dehydrogenase',\n",
       "  'reaction': 'L-threonine + NAD+ = L-2-amino-3-oxobutanoate + NADH + H+',\n",
       "  'other_names': 'L-threonine dehydrogenase; threonine 3-dehydrogenase; threonine dehydrogenase; TDH',\n",
       "  'sys_name': 'L-threonine:NAD+ oxidoreductase',\n",
       "  'comments': 'This enzyme acts in concert with EC 2.3.1.29, glycine C-acetyltransferase, in the degradation of threonine to glycine. This threonine-degradation pathway is common to prokaryotic and eukaryotic cells and the two enzymes involved form a complex [2]. In aqueous solution, the product L-2-amino-3-oxobutanoate can spontaneously decarboxylate to form aminoacetone.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.104': {'ec_num': '1.1.1.104',\n",
       "  'accepted_name': '4-oxoproline reductase',\n",
       "  'reaction': 'cis-4-hydroxy-L-proline + NAD+ = 4-oxo-L-proline + NADH + H+',\n",
       "  'other_names': 'cis-hydroxy-L-proline oxidase',\n",
       "  'sys_name': 'cis-4-hydroxy-L-proline:NAD+ oxidoreductase (4-oxo-L-proline forming)',\n",
       "  'comments': 'The enzyme, isolated from animals, is specific for 4-oxo-L-proline and cis-4-hydroxy-L-proline. It has no activity with trans-4-hydroxy-L-proline.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.105': {'ec_num': '1.1.1.105',\n",
       "  'accepted_name': 'all-trans-retinol dehydrogenase (NAD+)',\n",
       "  'reaction': 'all-trans-retinol—[cellular-retinol-binding-protein] + NAD+ = all-trans-retinal—[cellular-retinol-binding-protein] + NADH + H+',\n",
       "  'other_names': 'retinol (vitamin A1) dehydrogenase; MDR; microsomal retinol dehydrogenase; retinol dehydrogenase (misleading); retinal reductase (ambiguous); retinene reductase; epidermal retinol dehydrogenase 2; SDR16C5 (gene name); RDH16 (gene name)',\n",
       "  'sys_name': 'all-trans retinol:NAD+ oxidoreductase',\n",
       "  'comments': 'The enzyme recognizes all-trans-retinol and all-trans-retinal as substrates and exhibits a strong preference for NAD+/NADH as cofactors. Recognizes the substrate both in free form and when bound to cellular-retinol-binding-protein (CRBP1), but has higher affinity for the bound form [2]. No activity with 11-cis-retinol or 11-cis-retinal (cf. EC 1.1.1.315, 11-cis retinol dehydrogenase). Also active with 3α-hydroxysteroids [2].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.106': {'ec_num': '1.1.1.106',\n",
       "  'accepted_name': 'pantoate 4-dehydrogenase',\n",
       "  'reaction': '(R)-pantoate + NAD+ = (R)-4-dehydropantoate + NADH + H+',\n",
       "  'other_names': 'pantoate dehydrogenase; pantothenase; D-pantoate:NAD+ 4-oxidoreductase',\n",
       "  'sys_name': '(R)-pantoate:NAD+ 4-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'pantoate = 2,4-dihydroxy-3,3-dimethylbutanoate',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.107': {'ec_num': '1.1.1.107',\n",
       "  'accepted_name': 'pyridoxal 4-dehydrogenase',\n",
       "  'reaction': 'pyridoxal + NAD+ = 4-pyridoxolactone + NADH + H+',\n",
       "  'other_names': 'pyridoxal dehydrogenase',\n",
       "  'sys_name': 'pyridoxal:NAD+ 4-oxidoreductase',\n",
       "  'comments': 'The enzyme acts on the hemiacetal form of the substrate.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.107\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.107\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.107\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.107\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.1.1.107%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.108': {'ec_num': '1.1.1.108',\n",
       "  'accepted_name': 'carnitine 3-dehydrogenase',\n",
       "  'reaction': 'carnitine + NAD+ = 3-dehydrocarnitine + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'carnitine:NAD+ 3-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.110': {'ec_num': '1.1.1.110',\n",
       "  'accepted_name': 'aromatic 2-oxoacid reductase',\n",
       "  'reaction': '(1) (R)-3-(phenyl)lactate + NAD+ = 3-phenylpyruvate + NADH + H+;;(2) (R)-3-(4-hydroxyphenyl)lactate + NAD+ = 3-(4-hydroxyphenyl)pyruvate + NADH + H+;;(3) (R)-(indol-3-yl)lactate + NAD+ = (indol-3-yl)pyruvate + NADH + H+',\n",
       "  'other_names': '(R)-aromatic lactate dehydrogenase; (R)-4-hydroxyphenyllactate dehydrogenase; indolelactate:NAD+ oxidoreductase; indolelactate dehydrogenase; fldH (gene name); (indol-3-yl)lactate:NAD+ oxidoreductase',\n",
       "  'sys_name': 'aromatic 2-oxoacid:NAD+ oxidoreductase',\n",
       "  'comments': 'The enzymes from anaerobic bacteria such as Clostridium sporogenes participate in the fermentation pathways of L-phenylalanine, L-tyrosine and L-tryptophan. The enzyme from the yeast Candida maltosa has similar activity, but, unlike the bacterial enzyme, requires Mn2+ and can also use NADPH with lower activity.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG',\n",
       "  'glossary': '3-phenylpyruvate = 2-oxo-3-phenylpropanoate',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.111': {'ec_num': '1.1.1.111',\n",
       "  'accepted_name': '3-(imidazol-5-yl)lactate dehydrogenase',\n",
       "  'reaction': '(S)-3-(imidazol-5-yl)lactate + NAD(P)+ = 3-(imidazol-5-yl)pyruvate + NAD(P)H + H+',\n",
       "  'other_names': 'imidazol-5-yl lactate dehydrogenase',\n",
       "  'sys_name': '(S)-3-(imidazol-5-yl)lactate:NAD(P)+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.112': {'ec_num': '1.1.1.112',\n",
       "  'accepted_name': 'indanol dehydrogenase',\n",
       "  'reaction': 'indan-1-ol + NAD(P)+ = indanone + NAD(P)H + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'indan-1-ol:NAD(P)+ 1-oxidoreductase',\n",
       "  'comments': '3(20)α-Hydroxysteroids are also oxidized, more slowly.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.113': {'ec_num': '1.1.1.113',\n",
       "  'accepted_name': 'L-xylose 1-dehydrogenase',\n",
       "  'reaction': 'L-xylose + NADP+ = L-xylono-1,4-lactone + NADPH + H+',\n",
       "  'other_names': 'L-xylose dehydrogenase; NADPH-xylose reductase',\n",
       "  'sys_name': 'L-xylose:NADP+ 1-oxidoreductase',\n",
       "  'comments': 'Also oxidizes D-arabinose and D-lyxose.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.114': {'ec_num': '1.1.1.114',\n",
       "  'accepted_name': 'apiose 1-reductase',\n",
       "  'reaction': 'D-apiitol + NAD+ = D-apiose + NADH + H+',\n",
       "  'other_names': 'D-apiose reductase; D-apiitol reductase',\n",
       "  'sys_name': 'D-apiitol:NAD+ 1-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.115': {'ec_num': '1.1.1.115',\n",
       "  'accepted_name': 'ribose 1-dehydrogenase (NADP+)',\n",
       "  'reaction': 'D-ribose + NADP+ + H2O = D-ribonate + NADPH + H+',\n",
       "  'other_names': 'D-ribose dehydrogenase (NADP+); NADP-pentose-dehydrogenase; ribose 1-dehydrogenase (NADP)',\n",
       "  'sys_name': 'D-ribose:NADP+ 1-oxidoreductase',\n",
       "  'comments': 'Also acts, more slowly, on D-xylose and other pentoses.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.116': {'ec_num': '1.1.1.116',\n",
       "  'accepted_name': 'D-arabinose 1-dehydrogenase (NAD+)',\n",
       "  'reaction': 'D-arabinose + NAD+ = D-arabinono-1,4-lactone + NADH + H+',\n",
       "  'other_names': 'NAD+-pentose-dehydrogenase; arabinose(fucose)dehydrogenase',\n",
       "  'sys_name': 'D-arabinose:NAD+ 1-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.117': {'ec_num': '1.1.1.117',\n",
       "  'accepted_name': 'D-arabinose 1-dehydrogenase [NAD(P)+]',\n",
       "  'reaction': 'D-arabinose + NAD(P)+ = D-arabinono-1,4-lactone + NAD(P)H + H+',\n",
       "  'other_names': 'D-arabinose 1-dehydrogenase [NAD(P)]',\n",
       "  'sys_name': 'D-arabinose:NAD(P)+ 1-oxidoreductase',\n",
       "  'comments': 'Also acts on L-galactose, 6-deoxy- and 3,6-dideoxy-L-galactose.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.118': {'ec_num': '1.1.1.118',\n",
       "  'accepted_name': 'glucose 1-dehydrogenase (NAD+)',\n",
       "  'reaction': 'D-glucose + NAD+ = D-glucono-1,5-lactone + NADH + H+',\n",
       "  'other_names': 'D-glucose:NAD oxidoreductase; D-aldohexose dehydrogenase; glucose 1-dehydrogenase (NAD)',\n",
       "  'sys_name': 'D-glucose:NAD+ 1-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.119': {'ec_num': '1.1.1.119',\n",
       "  'accepted_name': 'glucose 1-dehydrogenase (NADP+)',\n",
       "  'reaction': 'D-glucose + NADP+ = D-glucono-1,5-lactone + NADPH + H+',\n",
       "  'other_names': 'nicotinamide adenine dinucleotide phosphate-linked aldohexose dehydrogenase; NADP-linked aldohexose dehydrogenase; NADP-dependent glucose dehydrogenase; glucose 1-dehydrogenase (NADP)',\n",
       "  'sys_name': 'D-glucose:NADP+ 1-oxidoreductase',\n",
       "  'comments': 'Also oxidizes D-mannose, 2-deoxy-D-glucose and 2-amino-2-deoxy-D-mannose.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.120': {'ec_num': '1.1.1.120',\n",
       "  'accepted_name': 'galactose 1-dehydrogenase (NADP+)',\n",
       "  'reaction': 'D-galactose + NADP+ = D-galactono-1,5-lactone + NADPH + H+',\n",
       "  'other_names': 'D-galactose dehydrogenase (NADP+); galactose 1-dehydrogenase (NADP)',\n",
       "  'sys_name': 'D-galactose:NADP+ 1-oxidoreductase',\n",
       "  'comments': 'Also acts on L-arabinose, 6-deoxy- and 2-deoxy-D-galactose.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.121': {'ec_num': '1.1.1.121',\n",
       "  'accepted_name': 'aldose 1-dehydrogenase (NAD+)',\n",
       "  'reaction': 'D-aldose + NAD+ = D-aldonolactone + NADH + H+',\n",
       "  'other_names': 'aldose dehydrogenase; D-aldohexose dehydrogenase; aldose 1-dehydrogenase',\n",
       "  'sys_name': 'D-aldose:NAD+ 1-oxidoreductase',\n",
       "  'comments': 'Acts on D-glucose, 2-deoxy- and 6-deoxy-D-glucose, D-galactose, 6-deoxy-D-galactose, 2-deoxy-L-arabinose and D-xylose.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.122': {'ec_num': '1.1.1.122',\n",
       "  'accepted_name': 'D-threo-aldose 1-dehydrogenase',\n",
       "  'reaction': 'a D-threo-aldose + NAD+ = a D-threo-aldono-1,5-lactone + NADH + H+',\n",
       "  'other_names': 'L-fucose dehydrogenase; (2S,3R)-aldose dehydrogenase; dehydrogenase, L-fucose; L-fucose (D-arabinose) dehydrogenase',\n",
       "  'sys_name': 'D-threo-aldose:NAD+ 1-oxidoreductase',\n",
       "  'comments': 'Acts on L-fucose, D-arabinose and L-xylose; the animal enzyme was also shown to act on L-arabinose, and the enzyme from Pseudomonas caryophylli on L-glucose.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.123': {'ec_num': '1.1.1.123',\n",
       "  'accepted_name': 'sorbose 5-dehydrogenase (NADP+)',\n",
       "  'reaction': 'L-sorbose + NADP+ = 5-dehydro-D-fructose + NADPH + H+',\n",
       "  'other_names': '5-ketofructose reductase; 5-keto-D-fructose reductase; sorbose (nicotinamide adenine dinucleotide phosphate) dehydrogenase; reduced nicotinamide adenine dinucleotide phosphate-linked reductase; sorbose 5-dehydrogenase (NADP+)',\n",
       "  'sys_name': 'L-sorbose:NADP+ 5-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.124': {'ec_num': '1.1.1.124',\n",
       "  'accepted_name': 'fructose 5-dehydrogenase (NADP+)',\n",
       "  'reaction': 'D-fructose + NADP+ = 5-dehydro-D-fructose + NADPH + H+',\n",
       "  'other_names': '5-ketofructose reductase (NADP); 5-keto-D-fructose reductase (NADP+); fructose 5-(nicotinamide adenine dinucleotide phosphate) dehydrogenase; D-(-)fructose:(NADP+) 5-oxidoreductase; fructose 5-dehydrogenase (NADP)',\n",
       "  'sys_name': 'D-fructose:NADP+ 5-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.125': {'ec_num': '1.1.1.125',\n",
       "  'accepted_name': '2-deoxy-D-gluconate 3-dehydrogenase',\n",
       "  'reaction': '2-deoxy-D-gluconate + NAD+ = 3-dehydro-2-deoxy-D-gluconate + NADH + H+',\n",
       "  'other_names': '2-deoxygluconate dehydrogenase',\n",
       "  'sys_name': '2-deoxy-D-gluconate:NAD+ 3-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.126': {'ec_num': '1.1.1.126',\n",
       "  'accepted_name': '2-dehydro-3-deoxy-D-gluconate 6-dehydrogenase',\n",
       "  'reaction': '2-dehydro-3-deoxy-D-gluconate + NADP+ = (4S,5S)-4,5-dihydroxy-2,6-dioxohexanoate + NADPH + H+',\n",
       "  'other_names': '2-keto-3-deoxy-D-gluconate dehydrogenase (ambiguous); 2-keto-3-deoxygluconate dehydrogenase (ambiguous)',\n",
       "  'sys_name': '2-dehydro-3-deoxy-D-gluconate:NADP+ 6-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.127': {'ec_num': '1.1.1.127',\n",
       "  'accepted_name': '2-dehydro-3-deoxy-D-gluconate 5-dehydrogenase',\n",
       "  'reaction': '2-dehydro-3-deoxy-D-gluconate + NAD+ = (4S)-4,6-dihydroxy-2,5-dioxohexanoate + NADH + H+',\n",
       "  'other_names': '2-keto-3-deoxygluconate 5-dehydrogenase; 2-keto-3-deoxy-D-gluconate dehydrogenase (ambiguous); 2-keto-3-deoxygluconate (nicotinamide adenine dinucleotide (phosphate)) dehydrogenase; 2-keto-3-deoxy-D-gluconate (3-deoxy-D-glycero-2,5-hexodiulosonic acid) dehydrogenase (ambiguous)',\n",
       "  'sys_name': '2-dehydro-3-deoxy-D-gluconate:NAD+ 5-oxidoreductase',\n",
       "  'comments': 'The enzyme from Pseudomonas acts equally well on NAD+ or NADP+, while that from Erwinia chrysanthemi and Escherichia coli is more specific for NAD+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.128': {'ec_num': '1.1.1.128',\n",
       "  'accepted_name': 'L-idonate 2-dehydrogenase',\n",
       "  'reaction': 'L-idonate + NADP+ = 5-dehydro-D-gluconate + NADPH + H+',\n",
       "  'other_names': '5-ketogluconate 2-reductase; 5-keto-D-gluconate 2-reductase; L-idonate dehydrogenase; reductase, 5-ketogluconate 5- (L-idonate-forming); 5KGR; 5-ketoglucono-idono-reductase',\n",
       "  'sys_name': 'L-idonate:NADP+ 2-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.129': {'ec_num': '1.1.1.129',\n",
       "  'accepted_name': 'L-threonate 3-dehydrogenase',\n",
       "  'reaction': 'L-threonate + NAD+ = 3-dehydro-L-erythronate + NADH + H+',\n",
       "  'other_names': 'threonate dehydrogenase; L-threonic acid dehydrogenase',\n",
       "  'sys_name': 'L-threonate:NAD+ 3-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.130': {'ec_num': '1.1.1.130',\n",
       "  'accepted_name': '3-dehydro-L-gulonate 2-dehydrogenase',\n",
       "  'reaction': '3-dehydro-L-gulonate + NAD(P)+ = (4R,5S)-4,5,6-trihydroxy-2,3-dioxohexanoate + NAD(P)H + H+',\n",
       "  'other_names': '3-keto-L-gulonate dehydrogenase; 3-ketogulonate dehydrogenase; 3-keto-L-gulonate dehydrogenase; 3-ketogulonate dehydrogenase',\n",
       "  'sys_name': '3-dehydro-L-gulonate:NAD(P)+ 2-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.131': {'ec_num': '1.1.1.131',\n",
       "  'accepted_name': 'mannuronate reductase',\n",
       "  'reaction': 'D-mannonate + NAD(P)+ = D-mannuronate + NAD(P)H + H+',\n",
       "  'other_names': 'mannonate dehydrogenase; mannonate (nicotinamide adenine dinucleotide (phosphate))dehydrogenase; mannonate dehydrogenase; mannuronate reductase; mannonate dehydrogenase (NAD(P)+); D-mannonate:nicotinamide adenine dinucleotide (phosphate oxidoreductase (D-mannuronate-forming))',\n",
       "  'sys_name': 'D-mannonate:NAD(P)+ 6-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.132': {'ec_num': '1.1.1.132',\n",
       "  'accepted_name': 'GDP-mannose 6-dehydrogenase',\n",
       "  'reaction': 'GDP-D-mannose + 2 NAD+ + H2O = GDP-D-mannuronate + 2 NADH + 2 H+',\n",
       "  'other_names': 'guanosine diphosphomannose dehydrogenase; GDP-mannose dehydrogenase; guanosine diphosphomannose dehydrogenase; guanosine diphospho-D-mannose dehydrogenase',\n",
       "  'sys_name': 'GDP-D-mannose:NAD+ 6-oxidoreductase',\n",
       "  'comments': 'Also acts on the corresponding deoxynucleoside diphosphate derivative as a substrate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.133': {'ec_num': '1.1.1.133',\n",
       "  'accepted_name': 'dTDP-4-dehydrorhamnose reductase',\n",
       "  'reaction': 'dTDP-β-L-rhamnose + NADP+ = dTDP-4-dehydro-β-L-rhamnose + NADPH + H+',\n",
       "  'other_names': 'dTDP-4-keto-L-rhamnose reductase; dTDP-4-ketorhamnose reductase; TDP-4-keto-rhamnose reductase; thymidine diphospho-4-ketorhamnose reductase; dTDP-6-deoxy-L-mannose:NADP+ 4-oxidoreductase; dTDP-6-deoxy-β-L-mannose:NADP+ 4-oxidoreductase',\n",
       "  'sys_name': 'dTDP-β-L-rhamnose:NADP+ 4-oxidoreductase',\n",
       "  'comments': 'In the reverse direction, reduction on the 4-position of the hexose moiety takes place only while the substrate is bound to another enzyme that catalyses epimerization at C-3 and C-5; the complex has been referred to as dTDP-L-rhamnose synthase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'dTDP-4-dehydro-β-L-rhamnose = dTDP-4-dehydro-6-deoxy-β-L-mannose\\ndTDP-4-β-L-rhamnose = dTDP-6-deoxy-β-L-mannose',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.134': {'ec_num': '1.1.1.134',\n",
       "  'accepted_name': 'dTDP-6-deoxy-L-talose 4-dehydrogenase (NADP+)',\n",
       "  'reaction': 'dTDP-6-deoxy-β-L-talose + NADP+ = dTDP-4-dehydro-β-L-rhamnose + NADPH + H+',\n",
       "  'other_names': 'thymidine diphospho-6-deoxy-L-talose dehydrogenase; TDP-6-deoxy-L-talose dehydrogenase; dTDP-6-deoxy-L-talose dehydrogenase (4-reductase); dTDP-6-deoxy-L-talose:NADP+ 4-oxidoreductase',\n",
       "  'sys_name': 'dTDP-6-deoxy-β-L-talose:NADP+ 4-oxidoreductase',\n",
       "  'comments': 'Oxidation on the 4-position of the hexose moiety takes place only while the substrate is bound to another enzyme that catalyses epimerization at C-3 and C-5.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'dTDP-4-dehydro-β-L-rhamnose = dTDP-4-dehydro-6-deoxy-β-L-mannose\\ndTDP-6-deoxy-β-L-talose = dTDP-β-L-pneumose\\n',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.135': {'ec_num': '1.1.1.135',\n",
       "  'accepted_name': 'GDP-6-deoxy-D-talose 4-dehydrogenase',\n",
       "  'reaction': 'GDP-6-deoxy-α-D-talose + NAD(P)+ = GDP-4-dehydro-α-D-rhamnose + NAD(P)H + H+',\n",
       "  'other_names': 'guanosine diphospho-6-deoxy-D-talose dehydrogenase; GDP-6-deoxy-D-talose:NAD(P)+ 4-oxidoreductase',\n",
       "  'sys_name': 'GDP-6-deoxy-α-D-talose:NAD(P)+ 4-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'GDP-4-dehydro-α-D-rhamnose = GDP-4-dehydro-6-deoxy-α-D-mannose\\nGDP-6-deoxy-α-D-talose = GDP-α-D-pneumose\\n',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.136': {'ec_num': '1.1.1.136',\n",
       "  'accepted_name': 'UDP-N-acetylglucosamine 6-dehydrogenase',\n",
       "  'reaction': 'UDP-N-acetyl-α-D-glucosamine + 2 NAD+ + H2O = UDP-2-acetamido-2-deoxy-α-D-glucuronate + 2 NADH + 2 H+',\n",
       "  'other_names': 'uridine diphosphoacetylglucosamine dehydrogenase; UDP-acetylglucosamine dehydrogenase; UDP-2-acetamido-2-deoxy-D-glucose:NAD oxidoreductase; UDP-GlcNAc dehydrogenase; WbpA; WbpO',\n",
       "  'sys_name': 'UDP-N-acetyl-α-D-glucosamine:NAD+ 6-oxidoreductase',\n",
       "  'comments': 'This enzyme participates in the biosynthetic pathway for UDP-α-D-ManNAc3NAcA (UDP-2,3-diacetamido-2,3-dideoxy-α-D-mannuronic acid), an important precursor of B-band lipopolysaccharide.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.137': {'ec_num': '1.1.1.137',\n",
       "  'accepted_name': 'ribitol-5-phosphate 2-dehydrogenase',\n",
       "  'reaction': 'D-ribitol 5-phosphate + NAD(P)+ = D-ribulose 5-phosphate + NAD(P)H + H+',\n",
       "  'other_names': 'ribitol 5-phosphate dehydrogenase',\n",
       "  'sys_name': 'D-ribitol-5-phosphate:NAD(P)+ 2-oxidoreductase',\n",
       "  'comments': 'The enzyme, characterized from the bacterium Lactobacillus plantarum, can use both NAD+ and NADP+ as electron acceptor [cf. EC 1.1.1.405, ribitol-5-phosphate 2-dehydrogenase (NADP+)].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.138': {'ec_num': '1.1.1.138',\n",
       "  'accepted_name': 'mannitol 2-dehydrogenase (NADP+)',\n",
       "  'reaction': 'D-mannitol + NADP+ = D-fructose + NADPH + H+',\n",
       "  'other_names': 'mannitol 2-dehydrogenase (NADP)',\n",
       "  'sys_name': 'D-mannitol:NADP+ 2-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.140': {'ec_num': '1.1.1.140',\n",
       "  'accepted_name': 'sorbitol-6-phosphate 2-dehydrogenase',\n",
       "  'reaction': 'D-sorbitol 6-phosphate + NAD+ = D-fructose 6-phosphate + NADH + H+',\n",
       "  'other_names': 'ketosephosphate reductase; ketosephosphate reductase; D-sorbitol 6-phosphate dehydrogenase; D-sorbitol-6-phosphate dehydrogenase; sorbitol-6-P-dehydrogenase; D-glucitol-6-phosphate dehydrogenase',\n",
       "  'sys_name': 'D-sorbitol-6-phosphate:NAD+ 2-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.141': {'ec_num': '1.1.1.141',\n",
       "  'accepted_name': '15-hydroxyprostaglandin dehydrogenase (NAD+)',\n",
       "  'reaction': '(5Z,13E,15S)-11α,15-dihydroxy-9-oxoprost-5,13-dienoate + NAD+ = (5Z,13E)-11α-hydroxy-9,15-dioxoprost-5,13-dienoate + NADH + H+',\n",
       "  'other_names': 'NAD+-dependent 15-hydroxyprostaglandin dehydrogenase (type I); PGDH; 11α,15-dihydroxy-9-oxoprost-13-enoate:NAD+ 15-oxidoreductase; 15-OH-PGDH; 15-hydroxyprostaglandin dehydrogenase; 15-hydroxyprostanoic dehydrogenase; NAD+-specific 15-hydroxyprostaglandin dehydrogenase; prostaglandin dehydrogenase; 15-hydroxyprostaglandin dehydrogenase (NAD+); (5Z,13E)-(15S)-11α,15-dihydroxy-9-oxoprost-13-enoate:NAD+ 15-oxidoreductase',\n",
       "  'sys_name': '(5Z,13E,15S)-11α,15-dihydroxy-9-oxoprost-5,13-dienoate:NAD+ 15-oxidoreductase',\n",
       "  'comments': 'Acts on prostaglandin E2, F2α and B1, but not on prostaglandin D2. cf. EC 1.1.1.196 15-hydroxyprostaglandin-D dehydrogenase (NADP+) and EC 1.1.1.197 15-hydroxyprostaglandin dehydrogenase (NADP+).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.142': {'ec_num': '1.1.1.142',\n",
       "  'accepted_name': 'D-pinitol dehydrogenase',\n",
       "  'reaction': '1D-3-O-methyl-chiro-inositol + NADP+ = 2D-5-O-methyl-2,3,5/4,6-pentahydroxycyclohexanone + NADPH + H+',\n",
       "  'other_names': '5D-5-O-methyl-chiro-inositol:NADP+ oxidoreductase',\n",
       "  'sys_name': '1D-3-O-methyl-chiro-inositol:NADP+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.143': {'ec_num': '1.1.1.143',\n",
       "  'accepted_name': 'sequoyitol dehydrogenase',\n",
       "  'reaction': '5-O-methyl-myo-inositol + NAD+ = 2D-5-O-methyl-2,3,5/4,6-pentahydroxycyclohexanone + NADH + H+',\n",
       "  'other_names': 'D-pinitol dehydrogenase',\n",
       "  'sys_name': '5-O-methyl-myo-inositol:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.144': {'ec_num': '1.1.1.144',\n",
       "  'accepted_name': 'perillyl-alcohol dehydrogenase',\n",
       "  'reaction': 'perillyl alcohol + NAD+ = perillyl aldehyde + NADH + H+',\n",
       "  'other_names': 'perillyl alcohol dehydrogenase',\n",
       "  'sys_name': 'perillyl-alcohol:NAD+ oxidoreductase',\n",
       "  'comments': 'Oxidizes a number of primary alcohols with the alcohol group allylic to an endocyclic double bond and a 6-membered ring, either aromatic or hydroaromatic.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.144\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.144\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.144\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.144\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.145': {'ec_num': '1.1.1.145',\n",
       "  'accepted_name': '3β-hydroxy-Δ5-steroid dehydrogenase',\n",
       "  'reaction': 'a 3β-hydroxy-Δ5-steroid + NAD+ = a 3-oxo-Δ5-steroid + NADH + H+',\n",
       "  'other_names': 'progesterone reductase; Δ5-3β-hydroxysteroid dehydrogenase; 3β-hydroxy-5-ene steroid dehydrogenase; 3β-hydroxy steroid dehydrogenase/isomerase; 3β-hydroxy-Δ5-C27-steroid dehydrogenase/isomerase; 3β-hydroxy-Δ5-C27-steroid oxidoreductase; 3β-hydroxy-5-ene-steroid oxidoreductase; steroid-Δ5-3β-ol dehydrogenase; 3β-HSDH; 5-ene-3-β-hydroxysteroid dehydrogenase; 3β-hydroxy-5-ene-steroid dehydrogenase',\n",
       "  'sys_name': '3β-hydroxy-Δ5-steroid:NAD+ 3-oxidoreductase',\n",
       "  'comments': 'This activity is found in several bifunctional enzymes that catalyse the oxidative conversion of Δ5-3-hydroxy steroids to a Δ4-3-oxo configuration. This conversion is carried out in two separate, sequential reactions; in the first reaction, which requires NAD+, the enzyme catalyses the dehydrogenation of the 3β-hydroxy steroid to a 3-oxo intermediate. In the second reaction the reduced cosubstrate, which remains attached to the enzyme, activates the isomerization of the Δ5 form to a Δ4 form (cf. EC 5.3.3.1, steroid Δ-isomerase). Substrates include dehydroepiandrosterone (which is converted into androst-5-ene-3,17-dione), pregnenolone (converted to progesterone) and cholest-5-en-3-one, an intermediate of cholesterol degradation.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-05-19 09:16:29'},\n",
       " '1.1.1.146': {'ec_num': '1.1.1.146',\n",
       "  'accepted_name': '11β-hydroxysteroid dehydrogenase',\n",
       "  'reaction': 'an 11β-hydroxysteroid + NADP+ = an 11-oxosteroid + NADPH + H+',\n",
       "  'other_names': 'corticosteroid 11β-dehydrogenase; β-hydroxysteroid dehydrogenase; 11β-hydroxy steroid dehydrogenase; corticosteroid 11-reductase; dehydrogenase, 11β-hydroxy steroid',\n",
       "  'sys_name': '11β-hydroxysteroid:NADP+ 11-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.147': {'ec_num': '1.1.1.147',\n",
       "  'accepted_name': '16α-hydroxysteroid dehydrogenase',\n",
       "  'reaction': 'a 16α-hydroxysteroid + NAD(P)+ = a 16-oxosteroid + NAD(P)H + H+',\n",
       "  'other_names': '16α-hydroxy steroid dehydrogenase',\n",
       "  'sys_name': '16α-hydroxysteroid:NAD(P)+ 16-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.148': {'ec_num': '1.1.1.148',\n",
       "  'accepted_name': 'estradiol 17α-dehydrogenase',\n",
       "  'reaction': 'estradiol-17α + NAD(P)+ = estrone + NAD(P)H + H+',\n",
       "  'other_names': '17α-estradiol dehydrogenase; 17α-hydroxy steroid dehydrogenase; 17α-hydroxy steroid oxidoreductase; 17α-hydroxysteroid oxidoreductase; estradiol 17α-oxidoreductase',\n",
       "  'sys_name': '17α-hydroxysteroid:NAD(P)+ 17-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.149': {'ec_num': '1.1.1.149',\n",
       "  'accepted_name': '20α-hydroxysteroid dehydrogenase',\n",
       "  'reaction': '17α,20α-dihydroxypregn-4-en-3-one + NAD(P)+ = 17α-hydroxyprogesterone + NAD(P)H + H+',\n",
       "  'other_names': '20α-hydroxy steroid dehydrogenase; 20α-HSD; 20α-HSDH',\n",
       "  'sys_name': '20α-hydroxysteroid:NAD(P)+ 20-oxidoreductase',\n",
       "  'comments': 'Re-specific with respect to NAD(P)+ (cf. EC 1.1.1.62 17β-estradiol 17-dehydrogenase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.150': {'ec_num': '1.1.1.150',\n",
       "  'accepted_name': '21-hydroxysteroid dehydrogenase (NAD+)',\n",
       "  'reaction': 'pregnan-21-ol + NAD+ = pregnan-21-al + NADH + H+',\n",
       "  'other_names': '21-hydroxysteroid dehydrogenase (NAD)',\n",
       "  'sys_name': '21-hydroxysteroid:NAD+ 21-oxidoreductase',\n",
       "  'comments': 'Acts on a number of 21-hydroxycorticosteroids.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.151': {'ec_num': '1.1.1.151',\n",
       "  'accepted_name': '21-hydroxysteroid dehydrogenase (NADP+)',\n",
       "  'reaction': 'pregnan-21-ol + NADP+ = pregnan-21-al + NADPH + H+',\n",
       "  'other_names': '21-hydroxy steroid dehydrogenase; 21-hydroxy steroid (nicotinamide adenine dinucleotide phosphate) dehydrogenase; 21-hydroxy steroid dehydrogenase (nicotinamide adenine dinucleotide phosphate); NADP-21-hydroxysteroid dehydrogenase; 21-hydroxysteroid dehydrogenase (NADP)',\n",
       "  'sys_name': '21-hydroxysteroid:NADP+ 21-oxidoreductase',\n",
       "  'comments': 'Acts on a number of 21-hydroxycorticosteroids.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.152': {'ec_num': '1.1.1.152',\n",
       "  'accepted_name': '3α-hydroxy-5β-androstane-17-one 3α-dehydrogenase',\n",
       "  'reaction': '3α-hydroxy-5β-androstane-17-one + NAD+ = 5β-androstane-3,17-dione + NADH + H+',\n",
       "  'other_names': 'etiocholanolone 3α-dehydrogenase; etiocholanolone 3α-dehydrogenase; 3α-hydroxy-5β-steroid dehydrogenase',\n",
       "  'sys_name': '3α-hydroxy-5β-steroid:NAD+ 3-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.153': {'ec_num': '1.1.1.153',\n",
       "  'accepted_name': 'sepiapterin reductase (L-erythro-7,8-dihydrobiopterin forming)',\n",
       "  'reaction': '(1) L-erythro-7,8-dihydrobiopterin + NADP+ = sepiapterin + NADPH + H+;;(2) L-erythro-tetrahydrobiopterin + 2 NADP+ = 6-pyruvoyl-5,6,7,8-tetrahydropterin + 2 NADPH + 2 H+',\n",
       "  'other_names': 'SR',\n",
       "  'sys_name': 'L-erythro-7,8-dihydrobiopterin:NADP+ oxidoreductase',\n",
       "  'comments': 'This enzyme catalyses the final step in the de novo synthesis of tetrahydrobiopterin from GTP. The enzyme, which is found in higher animals and some fungi and bacteria, produces the erythro form of tetrahydrobiopterin. cf. EC 1.1.1.325, sepiapterin reductase (L-threo-7,8-dihydrobiopterin forming).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': 'sepiapterin = 2-amino-6-lactoyl-7,8-dihydropteridin-4(3H)-one \\ntetrahydrobiopterin = 5,6,7,8-tetrahydrobiopterin = 2-amino-6-(1,2-dihydroxypropyl)-5,6,7,8-tetrahydropteridin-4(3H)-one',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.154': {'ec_num': '1.1.1.154',\n",
       "  'accepted_name': 'ureidoglycolate dehydrogenase',\n",
       "  'reaction': '(S)-ureidoglycolate + NAD(P)+ = oxalureate + NAD(P)H + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '(S)-ureidoglycolate:NAD(P)+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.156': {'ec_num': '1.1.1.156',\n",
       "  'accepted_name': 'glycerol 2-dehydrogenase (NADP+)',\n",
       "  'reaction': 'glycerol + NADP+ = glycerone + NADPH + H+',\n",
       "  'other_names': 'dihydroxyacetone reductase; dihydroxyacetone (reduced nicotinamide adenine dinucleotide phosphate) reductase; dihydroxyacetone reductase (NADPH); DHA oxidoreductase; glycerol 2-dehydrogenase (NADP)',\n",
       "  'sys_name': 'glycerol:NADP+ 2-oxidoreductase (glycerone-forming)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.157': {'ec_num': '1.1.1.157',\n",
       "  'accepted_name': '3-hydroxybutyryl-CoA dehydrogenase',\n",
       "  'reaction': '(S)-3-hydroxybutanoyl-CoA + NADP+ = 3-acetoacetyl-CoA + NADPH + H+',\n",
       "  'other_names': 'β-hydroxybutyryl coenzyme A dehydrogenase; L(+)-3-hydroxybutyryl-CoA dehydrogenase; BHBD; dehydrogenase, L-3-hydroxybutyryl coenzyme A (nicotinamide adenine dinucleotide phosphate); L-(+)-3-hydroxybutyryl-CoA dehydrogenase; β-hydroxybutyryl-CoA dehydrogenase',\n",
       "  'sys_name': '(S)-3-hydroxybutanoyl-CoA:NADP+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.158': {'ec_num': '1.1.1.158',\n",
       "  'accepted_name': 'UDP-N-acetylmuramate dehydrogenase',\n",
       "  'reaction': 'UDP-N-acetylmuramate + NADP+ = UDP-N-acetyl-3-O-(1-carboxyvinyl)-D-glucosamine + NADPH + H+',\n",
       "  'other_names': 'MurB reductase; UDP-N-acetylenolpyruvoylglucosamine reductase; UDP-N-acetylglucosamine-enoylpyruvate reductase; UDP-GlcNAc-enoylpyruvate reductase; uridine diphosphoacetylpyruvoylglucosamine reductase; uridine diphospho-N-acetylglucosamine-enolpyruvate reductase; uridine-5′-diphospho-N-acetyl-2-amino-2-deoxy-3-O-lactylglucose:NADP-oxidoreductase',\n",
       "  'sys_name': 'UDP-N-acetylmuramate:NADP+ oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD). Sodium dithionite, sodium borohydride and, to a lesser extent, NADH, can replace NADPH.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.159': {'ec_num': '1.1.1.159',\n",
       "  'accepted_name': '7α-hydroxysteroid dehydrogenase',\n",
       "  'reaction': 'cholate + NAD+ = 3α,12α-dihydroxy-7-oxo-5β-cholan-24-oate + NADH + H+',\n",
       "  'other_names': '7α-hydroxy steroid dehydrogenase; 7α-HSDH',\n",
       "  'sys_name': '7α-hydroxysteroid:NAD+ 7-oxidoreductase',\n",
       "  'comments': 'Catalyses the oxidation of the 7α-hydroxy group of bile acids and alcohols both in their free and conjugated forms. The Bacteroides fragilis and Clostridium enzymes can also utilize NADP+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'cholate = 3α,7α,12α-trihydroxy-5β-cholan-24-oate',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.160': {'ec_num': '1.1.1.160',\n",
       "  'accepted_name': 'dihydrobunolol dehydrogenase',\n",
       "  'reaction': '(±)-5-[(tert-butylamino)-2′-hydroxypropoxy]-1,2,3,4-tetrahydro-1-naphthol + NADP+ = (±)-5-[(tert-butylamino)-2′-hydroxypropoxy]-3,4-dihydro-1(2H)-naphthalenone + NADPH + H+',\n",
       "  'other_names': 'bunolol reductase',\n",
       "  'sys_name': '(±)-5-[(tert-butylamino)-2′-hydroxypropoxy]-1,2,3,4-tetrahydro-1-naphthol:NADP+ oxidoreductase',\n",
       "  'comments': 'Also acts, more slowly, with NAD+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.161': {'ec_num': '1.1.1.161',\n",
       "  'accepted_name': 'cholestanetetraol 26-dehydrogenase',\n",
       "  'reaction': '(25R)-5β-cholestane-3α,7α,12α,26-tetraol + NAD+ = (25R)-3α,7α,12α-trihydroxy-5β-cholestan-26-al + NADH + H+',\n",
       "  'other_names': 'cholestanetetrol 26-dehydrogenase; 5β-cholestane-3α,7α,12α,26-tetrol dehydrogenase; TEHC-NAD oxidoreductase; 5β-cholestane-3α,7α,12α,26-tetraol:NAD+ 26-oxidoreductase',\n",
       "  'sys_name': '(25R)-5β-cholestane-3α,7α,12α,26-tetraol:NAD+ 26-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.162': {'ec_num': '1.1.1.162',\n",
       "  'accepted_name': 'erythrulose reductase',\n",
       "  'reaction': 'D-threitol + NADP+ = D-erythrulose + NADPH + H+',\n",
       "  'other_names': 'D-erythrulose reductase; erythritol:NADP+ oxidoreductase',\n",
       "  'sys_name': 'D-threitol:NADP+ oxidoreductase',\n",
       "  'comments': 'NAD+ is also utilized, but more slowly.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.163': {'ec_num': '1.1.1.163',\n",
       "  'accepted_name': 'cyclopentanol dehydrogenase',\n",
       "  'reaction': 'cyclopentanol + NAD+ = cyclopentanone + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'cyclopentanol:NAD+ oxidoreductase',\n",
       "  'comments': '4-Methylcyclohexanol and cyclohexanol can also act as substrates.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.164': {'ec_num': '1.1.1.164',\n",
       "  'accepted_name': 'hexadecanol dehydrogenase',\n",
       "  'reaction': 'hexadecanol + NAD+ = hexadecanal + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'hexadecanol:NAD+ oxidoreductase',\n",
       "  'comments': 'The liver enzyme acts on long-chain alcohols from C8 to C16. The Euglena enzyme also oxidizes the corresponding aldehydes to fatty acids.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.165': {'ec_num': '1.1.1.165',\n",
       "  'accepted_name': '2-alkyn-1-ol dehydrogenase',\n",
       "  'reaction': '2-butyne-1,4-diol + NAD+ = 4-hydroxy-2-butynal + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '2-butyne-1,4-diol:NAD+ 1-oxidoreductase',\n",
       "  'comments': 'Acts on a variety of 2-alkyn-1-ols, and also on 1,4-butanediol. NADP+ also acts as acceptor, but more slowly.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.166': {'ec_num': '1.1.1.166',\n",
       "  'accepted_name': 'hydroxycyclohexanecarboxylate dehydrogenase',\n",
       "  'reaction': '(1S,3R,4S)-3,4-dihydroxycyclohexane-1-carboxylate + NAD+ = (1S,4S)-4-hydroxy-3-oxocyclohexane-1-carboxylate + NADH + H+',\n",
       "  'other_names': 'dihydroxycyclohexanecarboxylate dehydrogenase; (-)t-3,t-4-dihydroxycyclohexane-c-1-carboxylate-NAD+ oxidoreductase',\n",
       "  'sys_name': '(1S,3R,4S)-3,4-dihydroxycyclohexane-1-carboxylate:NAD+ 3-oxidoreductase',\n",
       "  'comments': 'Acts on hydroxycyclohexanecarboxylates that have an equatorial carboxy group at C-1, an axial hydroxy group at C-3 and an equatorial hydroxy or carbonyl group at C-4, including (-)-quinate and (-)-shikimate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.167': {'ec_num': '1.1.1.167',\n",
       "  'accepted_name': 'hydroxymalonate dehydrogenase',\n",
       "  'reaction': 'hydroxymalonate + NAD+ = oxomalonate + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'hydroxymalonate:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.168': {'ec_num': '1.1.1.168',\n",
       "  'accepted_name': '2-dehydropantolactone reductase (Re-specific)',\n",
       "  'reaction': '(R)-pantolactone + NADP+ = 2-dehydropantolactone + NADPH + H+',\n",
       "  'other_names': '2-oxopantoyl lactone reductase; ketopantoyl lactone reductase; 2-ketopantoyl lactone reductase; 2-dehydropantoyl-lactone reductase (A-specific); (R)-pantolactone:NADP+ oxidoreductase (A-specific); 2-dehydropantolactone reductase (A-specific)',\n",
       "  'sys_name': '(R)-pantolactone:NADP+ oxidoreductase (Re-specific)',\n",
       "  'comments': 'The yeast enzyme differs from that from Escherichia coli [EC 1.1.1.214 2-dehydropantolactone reductase (Si-specific)], which is specific for the Si-face of NADP+, and in receptor requirements from EC 1.1.99.26 3-hydroxycyclohexanone dehydrogenase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.169': {'ec_num': '1.1.1.169',\n",
       "  'accepted_name': '2-dehydropantoate 2-reductase',\n",
       "  'reaction': '(R)-pantoate + NADP+ = 2-dehydropantoate + NADPH + H+',\n",
       "  'other_names': '2-oxopantoate reductase; 2-ketopantoate reductase; 2-ketopantoic acid reductase; ketopantoate reductase; ketopantoic acid reductase',\n",
       "  'sys_name': '(R)-pantoate:NADP+ 2-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'pantoate = 2,4-dihydroxy-3,3-dimethylbutanoate',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.170': {'ec_num': '1.1.1.170',\n",
       "  'accepted_name': '3β-hydroxysteroid-4α-carboxylate 3-dehydrogenase (decarboxylating)',\n",
       "  'reaction': 'a 3β-hydroxysteroid-4α-carboxylate + NAD(P)+ = a 3-oxosteroid + CO2 + NAD(P)H',\n",
       "  'other_names': '3β-hydroxy-4β-methylcholestenecarboxylate 3-dehydrogenase (decarboxylating); 3β-hydroxy-4β-methylcholestenoate dehydrogenase; sterol 4α-carboxylic decarboxylase; sterol-4α-carboxylate 3-dehydrogenase (decarboxylating) (ambiguous); ERG26 (gene name); NSDHL (gene name)',\n",
       "  'sys_name': '3β-hydroxysteroid-4α-carboxylate:NAD(P)+ 3-oxidoreductase (decarboxylating)',\n",
       "  'comments': 'The enzyme participates in the biosynthesis of several important sterols such as ergosterol and cholesterol. It is part of a three enzyme system that removes methyl groups from the C-4 position of steroid molecules. The first enzyme, EC 1.14.18.9, 4α-methylsterol monooxygenase, catalyses three successive oxidations of the methyl group, resulting in a carboxyl group; the second enzyme, EC 1.1.1.170, catalyses an oxidative decarboxylation that results in a reduction of the 3β-hydroxy group at the C-3 carbon to an oxo group; and the last enzyme, EC 1.1.1.270, 3β-hydroxysteroid 3-dehydrogenase, reduces the 3-oxo group back to a 3β-hydroxyl. If a second methyl group remains at the C-4 position, this enzyme also catalyses its epimerization from 4β to 4α orientation, so it could serve as a substrate for a second round of demethylation. cf. EC 1.1.1.418, plant 3β-hydroxysteroid-4α-carboxylate 3-dehydrogenase (decarboxylating).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.172': {'ec_num': '1.1.1.172',\n",
       "  'accepted_name': '2-oxoadipate reductase',\n",
       "  'reaction': '2-hydroxyadipate + NAD+ = 2-oxoadipate + NADH + H+',\n",
       "  'other_names': '2-ketoadipate reductase; α-ketoadipate reductase; 2-ketoadipate reductase',\n",
       "  'sys_name': '2-hydroxyadipate:NAD+ 2-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.173': {'ec_num': '1.1.1.173',\n",
       "  'accepted_name': 'L-rhamnose 1-dehydrogenase',\n",
       "  'reaction': 'L-rhamnofuranose + NAD+ = L-rhamno-1,4-lactone + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'L-rhamnofuranose:NAD+ 1-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.174': {'ec_num': '1.1.1.174',\n",
       "  'accepted_name': 'cyclohexane-1,2-diol dehydrogenase',\n",
       "  'reaction': 'trans-cyclohexane-1,2-diol + NAD+ = 2-hydroxycyclohexan-1-one + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'trans-cyclohexane-1,2-diol:NAD+ 1-oxidoreductase',\n",
       "  'comments': 'Also oxidizes, more slowly, the cis isomer and 2-hydroxycyclohexanone.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.175': {'ec_num': '1.1.1.175',\n",
       "  'accepted_name': 'D-xylose 1-dehydrogenase',\n",
       "  'reaction': 'D-xylose + NAD+ = D-xylonolactone + NADH + H+',\n",
       "  'other_names': 'NAD-D-xylose dehydrogenase; D-xylose dehydrogenase; (NAD)-linked D-xylose dehydrogenase',\n",
       "  'sys_name': 'D-xylose:NAD+ 1-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.176': {'ec_num': '1.1.1.176',\n",
       "  'accepted_name': '12α-hydroxysteroid dehydrogenase',\n",
       "  'reaction': 'cholate + NADP+ = 3α,7α-dihydroxy-12-oxo-5β-cholan-24-oate + NADPH + H+',\n",
       "  'other_names': '12α-hydroxy steroid dehydrogenase; NAD+-dependent 12α-hydroxysteroid dehydrogenase; NADP+-12α-hydroxysteroid dehydrogenase',\n",
       "  'sys_name': '12α-hydroxysteroid:NADP+ 12-oxidoreductase',\n",
       "  'comments': 'Catalyses the oxidation of the 12α-hydroxy group of bile acids, both in their free and conjugated form. Also acts on bile alcohols.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'cholate = 3α,7α,12α-trihydroxy-5β-cholan-24-oate\\n',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.177': {'ec_num': '1.1.1.177',\n",
       "  'accepted_name': 'glycerol-3-phosphate 1-dehydrogenase (NADP+)',\n",
       "  'reaction': 'sn-glycerol 3-phosphate + NADP+ = D-glyceraldehyde 3-phosphate + NADPH + H+',\n",
       "  'other_names': 'glycerol phosphate (nicotinamide adenine dinucleotide phosphate) dehydrogenase; L-glycerol 3-phosphate:NADP+ oxidoreductase; glycerin-3-phosphate dehydrogenase; NADPH-dependent glycerin-3-phosphate dehydrogenase; NADP-specific glycerol 3-phosphate 1-dehydrogenase',\n",
       "  'sys_name': 'sn-glycerol-3-phosphate:NADP+ 1-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.178': {'ec_num': '1.1.1.178',\n",
       "  'accepted_name': '3-hydroxy-2-methylbutyryl-CoA dehydrogenase',\n",
       "  'reaction': '(2S,3S)-3-hydroxy-2-methylbutanoyl-CoA + NAD+ = 2-methylacetoacetyl-CoA + NADH + H+',\n",
       "  'other_names': '2-methyl-3-hydroxybutyryl coenzyme A dehydrogenase; 2-methyl-3-hydroxybutyryl coenzyme A dehydrogenase; 2-methyl-3-hydroxy-butyryl CoA dehydrogenase',\n",
       "  'sys_name': '(2S,3S)-3-hydroxy-2-methylbutanoyl-CoA:NAD+ oxidoreductase',\n",
       "  'comments': 'Also acts, more slowly, on (2S,3S)-2-hydroxy-3-methylpentanoyl-CoA.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.179': {'ec_num': '1.1.1.179',\n",
       "  'accepted_name': 'D-xylose 1-dehydrogenase (NADP+, D-xylono-1,5-lactone-forming)',\n",
       "  'reaction': 'D-xylose + NADP+ = D-xylono-1,5-lactone + NADPH + H+',\n",
       "  'other_names': 'D-xylose (nicotinamide adenine dinucleotide phosphate) dehydrogenase (ambiguous); D-xylose-NADP dehydrogenase (ambiguous); D-xylose:NADP+ oxidoreductase (ambiguous); D-xylose 1-dehydrogenase (NADP) (ambiguous)',\n",
       "  'sys_name': 'D-xylose:NADP+ 1-oxidoreductase (D-xylono-1,5-lactone-forming)',\n",
       "  'comments': 'The enzyme, characterized from pig arterial vessels and eye lens, also acts, more slowly, on L-arabinose and D-ribose. cf. EC 1.1.1.424, D-xylose 1-dehydrogenase (NADP+, D-xylono-1,4-lactone-forming).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.181': {'ec_num': '1.1.1.181',\n",
       "  'accepted_name': 'cholest-5-ene-3β,7α-diol 3β-dehydrogenase',\n",
       "  'reaction': 'cholest-5-ene-3β,7α-diol + NAD+ = 7α-hydroxycholest-4-en-3-one + NADH + H+',\n",
       "  'other_names': '3β-hydroxy-Δ5-C27-steroid oxidoreductase (ambiguous)',\n",
       "  'sys_name': 'cholest-5-ene-3β,7α-diol:NAD+ 3-oxidoreductase',\n",
       "  'comments': 'Highly specific for 3β,7α-dihydroxy-C27-steroids with Δ5-double bond.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.183': {'ec_num': '1.1.1.183',\n",
       "  'accepted_name': 'geraniol dehydrogenase (NADP+)',\n",
       "  'reaction': 'geraniol + NADP+ = geranial + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'geraniol:NADP+ oxidoreductase',\n",
       "  'comments': 'Also acts, more slowly on farnesol but not on nerol. The enzyme produces a mixture known as citral, which includes geranial and neral. It is still not known whether neral is produced directly by the enzyme, or by isomerization of geranial.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.183\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.183\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.183\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.183\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.184': {'ec_num': '1.1.1.184',\n",
       "  'accepted_name': 'carbonyl reductase (NADPH)',\n",
       "  'reaction': 'R-CHOH-R′ + NADP+ = R-CO-R′ + NADPH + H+',\n",
       "  'other_names': 'aldehyde reductase 1; prostaglandin 9-ketoreductase; xenobiotic ketone reductase; NADPH-dependent carbonyl reductase; ALR3; carbonyl reductase; nonspecific NADPH-dependent carbonyl reductase; carbonyl reductase (NADPH2)',\n",
       "  'sys_name': 'secondary-alcohol:NADP+ oxidoreductase',\n",
       "  'comments': 'Acts on a wide range of carbonyl compounds, including quinones, aromatic aldehydes, ketoaldehydes, daunorubicin and prostaglandins E and F, reducing them to the corresponding alcohol. Si-specific with respect to NADPH [cf. EC 1.1.1.2 alcohol dehydrogenase (NADP+)].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.185': {'ec_num': '1.1.1.185',\n",
       "  'accepted_name': 'L-glycol dehydrogenase',\n",
       "  'reaction': 'an L-glycol + NAD(P)+ = a 2-hydroxycarbonyl compound + NAD(P)H + H+',\n",
       "  'other_names': 'glycol (nicotinamide adenine dinucleotide (phosphate)) dehydrogenase; L-(+)-glycol:NAD(P) oxidoreductase; L-glycol:NAD(P) dehydrogenase',\n",
       "  'sys_name': 'L-glycol:NAD(P)+ oxidoreductase',\n",
       "  'comments': 'The 2-hydroxycarbonyl compound formed can be further oxidized to a vicinal dicarbonyl compound. In the reverse direction, vicinal diketones, glyceraldehyde, glyoxal, methylglyoxal, 2-oxo-hydroxyketones and 2-ketoacid esters can be reduced.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.186': {'ec_num': '1.1.1.186',\n",
       "  'accepted_name': 'dTDP-galactose 6-dehydrogenase',\n",
       "  'reaction': 'dTDP-D-galactose + 2 NADP+ + H2O = dTDP-D-galacturonate + 2 NADPH + 2 H+',\n",
       "  'other_names': 'thymidine-diphosphate-galactose dehydrogenase',\n",
       "  'sys_name': 'dTDP-D-galactose:NADP+ 6-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.187': {'ec_num': '1.1.1.187',\n",
       "  'accepted_name': 'GDP-4-dehydro-D-rhamnose reductase',\n",
       "  'reaction': '(1) GDP-α-D-rhamnose + NAD(P)+ = GDP-4-dehydro-α-D-rhamnose + NAD(P)H + H+;;(2) GDP-6-deoxy-α-D-talose + NAD(P)+ = GDP-4-dehydro-α-D-rhamnose + NAD(P)H + H+',\n",
       "  'other_names': 'GDP-4-keto-6-deoxy-D-mannose reductase; GDP-4-keto-D-rhamnose reductase; guanosine diphosphate-4-keto-D-rhamnose reductase; GDP-6-deoxy-D-mannose:NAD(P)+ 4-oxidoreductase; GDP-6-deoxy-α-D-mannose:NAD(P)+ 4-oxidoreductase',\n",
       "  'sys_name': 'GDP-4-dehydro-α-D-rhamnose:NAD(P)+ 4-oxidoreductase',\n",
       "  'comments': 'The enzyme, which operates in the opposite direction to that shown, forms a mixture of GDP-α-D-rhamnose and its C-4 epimer, GDP-6-deoxy-α-D-talose. cf. EC 1.1.1.281, GDP-4-dehydro-6-deoxy-D-mannose reductase and EC 1.1.1.135, GDP-6-deoxy-D-talose 4-dehydrogenase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'GDP-α-D-rhamnose = GDP-6-deoxy-α-D-mannose\\nGDP-4-dehydro-α-D-rhamnose = GDP-4-dehydro-6-deoxy-α-D-mannose\\nGDP-6-deoxy-α-D-talose = GDP-α-D-pneumose\\n',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.188': {'ec_num': '1.1.1.188',\n",
       "  'accepted_name': 'prostaglandin-F synthase',\n",
       "  'reaction': '(5Z,13E)-(15S)-9α,11α,15-trihydroxyprosta-5,13-dienoate + NADP+ = (5Z,13E)-(15S)-9α,15-dihydroxy-11-oxoprosta-5,13-dienoate + NADPH + H+',\n",
       "  'other_names': 'prostaglandin-D2 11-reductase; reductase, 15-hydroxy-11-oxoprostaglandin; PGD2 11-ketoreductase; PGF2α synthetase; prostaglandin 11-ketoreductase; prostaglandin D2-ketoreductase; prostaglandin F synthase; prostaglandin F synthetase; synthetase, prostaglandin F2α; PGF synthetase; NADPH-dependent prostaglandin D2 11-keto reductase; prostaglandin 11-keto reductase',\n",
       "  'sys_name': '(5Z,13E)-(15S)-9α,11α,15-trihydroxyprosta-5,13-dienoate:NADP+ 11-oxidoreductase',\n",
       "  'comments': 'Reduces prostaglandin D2 and prostaglandin H2 to prostaglandin F2; prostaglandin D2 is not an intermediate in the reduction of prostaglandin H2. Also catalyses the reduction of a number of carbonyl compounds, such as 9,10-phenanthroquinone and 4-nitroacetophenone.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.189': {'ec_num': '1.1.1.189',\n",
       "  'accepted_name': 'prostaglandin-E2 9-reductase',\n",
       "  'reaction': '(5Z,13E)-(15S)-9α,11α,15-trihydroxyprosta-5,13-dienoate + NADP+ = (5Z,13E)-(15S)-11α,15-dihydroxy-9-oxoprosta-5,13-dienoate + NADPH + H+',\n",
       "  'other_names': 'PGE2-9-OR; reductase, 15-hydroxy-9-oxoprostaglandin; 9-keto-prostaglandin E2 reductase; 9-ketoprostaglandin reductase; PGE-9-ketoreductase; PGE2 9-oxoreductase; PGE2-9-ketoreductase; prostaglandin 9-ketoreductase; prostaglandin E 9-ketoreductase; prostaglandin E2-9-oxoreductase',\n",
       "  'sys_name': '(5Z,13E)-(15S)-9α,11α,15-trihydroxyprosta-5,13-dienoate:NADP+ 9-oxidoreductase',\n",
       "  'comments': 'Reduces prostaglandin E2 to prostaglandin F2α. A number of other 9-oxo- and 15-oxo-prostaglandin derivatives can also be reduced to the corresponding hydroxy compounds. May be identical with EC 1.1.1.197 15-hydroxyprostaglandin dehydrogenase (NADP+).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.190': {'ec_num': '1.1.1.190',\n",
       "  'accepted_name': 'indole-3-acetaldehyde reductase (NADH)',\n",
       "  'reaction': '(indol-3-yl)ethanol + NAD+ = (indol-3-yl)acetaldehyde + NADH + H+',\n",
       "  'other_names': 'indoleacetaldehyde reductase; indole-3-acetaldehyde reductase (NADH); indole-3-ethanol:NAD+ oxidoreductase',\n",
       "  'sys_name': '(indol-3-yl)ethanol:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.191': {'ec_num': '1.1.1.191',\n",
       "  'accepted_name': 'indole-3-acetaldehyde reductase (NADPH)',\n",
       "  'reaction': '(indol-3-yl)ethanol + NADP+ = (indol-3-yl)acetaldehyde + NADPH + H+',\n",
       "  'other_names': 'indoleacetaldehyde (reduced nicotinamide adenine dinucleotide phosphate) reductase; indole-3-acetaldehyde reductase (NADPH); indole-3-ethanol:NADP+ oxidoreductase',\n",
       "  'sys_name': '(indol-3-yl)ethanol:NADP+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.192': {'ec_num': '1.1.1.192',\n",
       "  'accepted_name': 'long-chain-alcohol dehydrogenase',\n",
       "  'reaction': 'a long-chain alcohol + 2 NAD+ + H2O = a long-chain carboxylate + 2 NADH + 2 H+',\n",
       "  'other_names': 'long-chain alcohol dehydrogenase; fatty alcohol oxidoreductase',\n",
       "  'sys_name': 'long-chain-alcohol:NAD+ oxidoreductase',\n",
       "  'comments': 'Hexadecanol is a good substrate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.193': {'ec_num': '1.1.1.193',\n",
       "  'accepted_name': '5-amino-6-(5-phosphoribosylamino)uracil reductase',\n",
       "  'reaction': '5-amino-6-(5-phospho-D-ribitylamino)uracil + NADP+ = 5-amino-6-(5-phospho-D-ribosylamino)uracil + NADPH + H+',\n",
       "  'other_names': 'aminodioxyphosphoribosylaminopyrimidine reductase',\n",
       "  'sys_name': '5-amino-6-(5-phospho-D-ribitylamino)uracil:NADP+ 1′-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.194': {'ec_num': '1.1.1.194',\n",
       "  'accepted_name': 'coniferyl-alcohol dehydrogenase',\n",
       "  'reaction': 'coniferyl alcohol + NADP+ = coniferyl aldehyde + NADPH + H+',\n",
       "  'other_names': 'CAD (ambiguous)',\n",
       "  'sys_name': 'coniferyl-alcohol:NADP+ oxidoreductase',\n",
       "  'comments': 'Specific for coniferyl alcohol; does not act on cinnamyl alcohol, 4-coumaryl alcohol or sinapyl alcohol.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.195': {'ec_num': '1.1.1.195',\n",
       "  'accepted_name': 'cinnamyl-alcohol dehydrogenase',\n",
       "  'reaction': 'cinnamyl alcohol + NADP+ = cinnamaldehyde + NADPH + H+',\n",
       "  'other_names': 'cinnamyl alcohol dehydrogenase; CAD (ambiguous)',\n",
       "  'sys_name': 'cinnamyl-alcohol:NADP+ oxidoreductase',\n",
       "  'comments': 'Acts on coniferyl alcohol, sinapyl alcohol, 4-coumaryl alcohol and cinnamyl alcohol (cf. EC 1.1.1.194 coniferyl-alcohol dehydrogenase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.196': {'ec_num': '1.1.1.196',\n",
       "  'accepted_name': '15-hydroxyprostaglandin-D dehydrogenase (NADP+)',\n",
       "  'reaction': '(5Z,13E)-(15S)-9α,15-dihydroxy-11-oxoprosta-5,13-dienoate + NADP+ = (5Z,13E)-9α-hydroxy-11,15-dioxoprosta-5,13-dienoate + NADPH + H+',\n",
       "  'other_names': 'prostaglandin-D 15-dehydrogenase (NADP); dehydrogenase, prostaglandin D2; NADP-PGD2 dehydrogenase; dehydrogenase, 15-hydroxyprostaglandin (nicotinamide adenine dinucleotide phosphate); 15-hydroxy PGD2 dehydrogenase; 15-hydroxyprostaglandin dehydrogenase (NADP); NADP-dependent 15-hydroxyprostaglandin dehydrogenase; prostaglandin D2 dehydrogenase; NADP-linked 15-hydroxyprostaglandin dehydrogenase; NADP-specific 15-hydroxyprostaglandin dehydrogenase; NADP-linked prostaglandin D2 dehydrogenase; 15-hydroxyprostaglandin-D dehydrogenase (NADP)',\n",
       "  'sys_name': '(5Z,13E)-(15S)-9α,15-dihydroxy-11-oxoprosta-5,13-dienoate:NADP+ 15-oxidoreductase',\n",
       "  'comments': 'Specific for prostaglandins D [cf. EC 1.1.1.141 15-hydroxyprostaglandin dehydrogenase (NAD+) and EC 1.1.1.197 15-hydroxyprostaglandin dehydrogenase (NADP+)].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.197': {'ec_num': '1.1.1.197',\n",
       "  'accepted_name': '15-hydroxyprostaglandin dehydrogenase (NADP+)',\n",
       "  'reaction': '(13E)-(15S)-11α,15-dihydroxy-9-oxoprost-13-enoate + NADP+ = (13E)-11α-hydroxy-9,15-dioxoprost-13-enoate + NADPH + H+',\n",
       "  'other_names': 'NADP-dependent 15-hydroxyprostaglandin dehydrogenase; NADP-linked 15-hydroxyprostaglandin dehydrogenase; NADP-specific 15-hydroxyprostaglandin dehydrogenase; type II 15-hydroxyprostaglandin dehydrogenase; 15-hydroxyprostaglandin dehydrogenase (NADP)',\n",
       "  'sys_name': '(13E)-(15S)-11α,15-dihydroxy-9-oxoprost-13-enoate:NADP+ 15-oxidoreductase',\n",
       "  'comments': 'Acts on prostaglandins E2, F2α and B1, but not on prostaglandin D2 [cf. EC 1.1.1.141 15-hydroxyprostaglandin dehydrogenase (NAD+) and EC 1.1.1.196 15-hydroxyprostaglandin-D dehydrogenase (NADP+)]. May be identical with EC 1.1.1.189 prostaglandin-E2 9-reductase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.198': {'ec_num': '1.1.1.198',\n",
       "  'accepted_name': '(+)-borneol dehydrogenase',\n",
       "  'reaction': '(+)-borneol + NAD+ = (+)-camphor + NADH + H+',\n",
       "  'other_names': 'bicyclic monoterpenol dehydrogenase',\n",
       "  'sys_name': '(+)-borneol:NAD+ oxidoreductase',\n",
       "  'comments': 'NADP+ can also act, but more slowly.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.199': {'ec_num': '1.1.1.199',\n",
       "  'accepted_name': '(S)-usnate reductase',\n",
       "  'reaction': '(6R)-2-acetyl-6-(3-acetyl-2,4,6-trihydroxy-5-methylphenyl)-3-hydroxy-6-methyl-2,4-cyclohexadien-1-one + NAD+ = (S)-usnate + NADH + H+',\n",
       "  'other_names': 'L-usnic acid dehydrogenase',\n",
       "  'sys_name': 'reduced-(S)-usnate:NAD+ oxidoreductase (ether-bond-forming)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'reduced (S)-usnate = reduced (-)-usnate = (6R)-2-acetyl-6-(3-acetyl-2,4,6-trihydroxy-5-methylphenyl)-3-hydroxy-6-methyl-2,4-cyclohexadien-1-one',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.200': {'ec_num': '1.1.1.200',\n",
       "  'accepted_name': 'aldose-6-phosphate reductase (NADPH)',\n",
       "  'reaction': 'D-sorbitol 6-phosphate + NADP+ = D-glucose 6-phosphate + NADPH + H+',\n",
       "  'other_names': 'aldose 6-phosphate reductase; NADP-dependent aldose 6-phosphate reductase; A6PR; aldose-6-P reductase; aldose-6-phosphate reductase; alditol 6-phosphate:NADP 1-oxidoreductase; aldose-6-phosphate reductase (NADPH2)',\n",
       "  'sys_name': 'D-aldose-6-phosphate:NADP+ 1-oxidoreductase',\n",
       "  'comments': 'In the reverse reaction, acts also on D-galactose 6-phosphate and, more slowly, on D-mannose 6-phosphate and 2-deoxy-D-glucose 6-phosphate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.201': {'ec_num': '1.1.1.201',\n",
       "  'accepted_name': '7β-hydroxysteroid dehydrogenase (NADP+)',\n",
       "  'reaction': 'a 7β-hydroxysteroid + NADP+ = a 7-oxosteroid + NADPH + H+',\n",
       "  'other_names': 'NADP-dependent 7β-hydroxysteroid dehydrogenase; 7β-hydroxysteroid dehydrogenase (NADP)',\n",
       "  'sys_name': '7β-hydroxysteroid:NADP+ 7-oxidoreductase',\n",
       "  'comments': 'Catalyses the oxidation of the 7β-hydroxy group of bile acids such as ursodeoxycholate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.202': {'ec_num': '1.1.1.202',\n",
       "  'accepted_name': '1,3-propanediol dehydrogenase',\n",
       "  'reaction': 'propane-1,3-diol + NAD+ = 3-hydroxypropanal + NADH + H+',\n",
       "  'other_names': '3-hydroxypropionaldehyde reductase; 1,3-PD:NAD+ oxidoreductase; 1,3-propanediol:NAD+ oxidoreductase; 1,3-propanediol dehydrogenase',\n",
       "  'sys_name': 'propane-1,3-diol:NAD+ 1-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.203': {'ec_num': '1.1.1.203',\n",
       "  'accepted_name': 'uronate dehydrogenase',\n",
       "  'reaction': '(1) β-D-galacturonate + NAD+ = D-galactaro-1,5-lactone + NADH + H+;;(2) β-D-glucuronate + NAD+ = D-glucaro-1,5-lactone + NADH + H+',\n",
       "  'other_names': 'uronate:NAD-oxidoreductase; uronic acid dehydrogenase',\n",
       "  'sys_name': 'uronate:NAD+ 1-oxidoreductase',\n",
       "  'comments': 'Requires Mg2+. The enzyme, characterized from the bacterium Agrobacterium fabrum, participates in oxidative degradation pathways for galacturonate and glucuronate. The enzyme can only accept the β anomeric form of the substrate [4]. The 1,5-lactone product is rather stable at cytosolic pH and does not hydrolyse spontaneously at a substantial rate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.205': {'ec_num': '1.1.1.205',\n",
       "  'accepted_name': 'IMP dehydrogenase',\n",
       "  'reaction': 'IMP + NAD+ + H2O = XMP + NADH + H+',\n",
       "  'other_names': 'inosine-5′-phosphate dehydrogenase; inosinic acid dehydrogenase; inosinate dehydrogenase; inosine 5′-monophosphate dehydrogenase; inosine monophosphate dehydrogenase; IMP oxidoreductase; inosine monophosphate oxidoreductase',\n",
       "  'sys_name': 'IMP:NAD+ oxidoreductase',\n",
       "  'comments': 'The enzyme acts on the hydroxy group of the hydrated derivative of the substrate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'IMP = inosine 5′-phosphate\\nXMP = xanthosine 5′-phosphate',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.206': {'ec_num': '1.1.1.206',\n",
       "  'accepted_name': 'tropinone reductase I',\n",
       "  'reaction': 'tropine + NADP+ = tropinone + NADPH + H+',\n",
       "  'other_names': 'tropine dehydrogenase; tropinone reductase (ambiguous); TR-I',\n",
       "  'sys_name': 'tropine:NADP+ 3α-oxidoreductase',\n",
       "  'comments': 'Also oxidizes other tropan-3α-ols, but not the corresponding β-derivatives [1]. This enzyme along with EC 1.1.1.236, tropinone reductase II, represents a branch point in tropane alkaloid metabolism [4].  Tropine (the product of EC 1.1.1.206) is incorporated into hyoscyamine and scopolamine whereas pseudotropine (the product of EC 1.1.1.236) is the first specific metabolite on the pathway to the calystegines [4]. Both enzymes are always found together in any given tropane-alkaloid-producing species, have a common substrate, tropinone, and are strictly stereospecific [3].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'tropine = 3α-hydroxytropane = tropan-3-endo-ol',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.207': {'ec_num': '1.1.1.207',\n",
       "  'accepted_name': '(-)-menthol dehydrogenase',\n",
       "  'reaction': '(-)-menthol + NADP+ = (-)-menthone + NADPH + H+',\n",
       "  'other_names': 'monoterpenoid dehydrogenase',\n",
       "  'sys_name': '(-)-menthol:NADP+ oxidoreductase',\n",
       "  'comments': 'Not identical with EC 1.1.1.208 (+)-neomenthol dehydrogenase. Acts also on a number of other cyclohexanols and cyclohexenols.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.208': {'ec_num': '1.1.1.208',\n",
       "  'accepted_name': '(+)-neomenthol dehydrogenase',\n",
       "  'reaction': '(+)-neomenthol + NADP+ = (-)-menthone + NADPH + H+',\n",
       "  'other_names': 'monoterpenoid dehydrogenase',\n",
       "  'sys_name': '(+)-neomenthol:NADP+ oxidoreductase',\n",
       "  'comments': 'Not identical with EC 1.1.1.207 (-)-menthol dehydrogenase. Acts also on a number of other cyclohexanols and cyclohexenols.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.209': {'ec_num': '1.1.1.209',\n",
       "  'accepted_name': '3(or 17)α-hydroxysteroid dehydrogenase',\n",
       "  'reaction': 'androsterone + NAD(P)+ = 5α-androstane-3,17-dione + NAD(P)H + H+',\n",
       "  'other_names': '3(17)α-hydroxysteroid dehydrogenase',\n",
       "  'sys_name': '3(or 17)α-hydroxysteroid:NAD(P)+ oxidoreductase',\n",
       "  'comments': 'Acts on the 3α-hydroxy group of androgens of the 5α-androstane series; and also, more slowly, on the 17α-hydroxy group of both androgenic and estrogenic substrates (cf. EC 1.1.1.51 3(or 17)β-hydroxysteroid dehydrogenase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.210': {'ec_num': '1.1.1.210',\n",
       "  'accepted_name': '3β(or 20α)-hydroxysteroid dehydrogenase',\n",
       "  'reaction': '5α-androstan-3β,17β-diol + NADP+ = 17β-hydroxy-5α-androstan-3-one + NADPH + H+',\n",
       "  'other_names': 'progesterone reductase; dehydrogenase, 3β,20α-hydroxy steroid; 3β,20α-hydroxysteroid oxidoreductase',\n",
       "  'sys_name': '3β(or 20α)-hydroxysteroid:NADP+ oxidoreductase',\n",
       "  'comments': 'Also acts on 20α-hydroxysteroids.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.211': {'ec_num': '1.1.1.211',\n",
       "  'accepted_name': 'long-chain-3-hydroxyacyl-CoA dehydrogenase',\n",
       "  'reaction': 'a long-chain (S)-3-hydroxyacyl-CoA + NAD+ = a long-chain 3-oxoacyl-CoA + NADH + H+',\n",
       "  'other_names': 'β-hydroxyacyl-CoA dehydrogenase; long-chain 3-hydroxyacyl coenzyme A dehydrogenase; 3-hydroxyacyl-CoA dehydrogenase; LCHAD',\n",
       "  'sys_name': 'long-chain-(S)-3-hydroxyacyl-CoA:NAD+ oxidoreductase',\n",
       "  'comments': 'This enzyme was purified from the mitochondrial inner membrane. The enzyme has a preference for long-chain substrates, and activity with a C16 substrate was 6- to 15-fold higher than with a C4 substrate (cf. EC 1.1.1.35 3-hydroxyacyl-CoA dehydrogenase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'a long-chain acyl-CoA = an acyl-CoA thioester where the acyl chain contains 13 to 22 carbon atoms.',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.212': {'ec_num': '1.1.1.212',\n",
       "  'accepted_name': '3-oxoacyl-[acyl-carrier-protein] reductase (NADH)',\n",
       "  'reaction': 'a (3R)-3-hydroxyacyl-[acyl-carrier protein] + NAD+ = a 3-oxoacyl-[acyl-carrier protein] + NADH + H+',\n",
       "  'other_names': '3-oxoacyl-[acyl carrier protein] (reduced nicotinamide adenine dinucleotide) reductase; 3-oxoacyl-[acyl-carrier-protein] reductase (NADH); (3R)-3-hydroxyacyl-[acyl-carrier-protein]:NAD+ oxidoreductase',\n",
       "  'sys_name': '(3R)-3-hydroxyacyl-[acyl-carrier protein]:NAD+ oxidoreductase',\n",
       "  'comments': 'Forms part of the fatty acid synthase system in plants. Can be separated from EC 1.1.1.100, 3-oxoacyl-[acyl-carrier-protein] reductase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.213': {'ec_num': '1.1.1.213',\n",
       "  'accepted_name': '3α-hydroxysteroid 3-dehydrogenase (Re-specific)',\n",
       "  'reaction': 'a 3α-hydroxysteroid + NAD(P)+ = a 3-oxosteroid + NAD(P)H + H+',\n",
       "  'other_names': '3α-hydroxysteroid dehydrogenase; 3α-hydroxysteroid:NAD(P)+ 3-oxidoreductase (A-specific); 3α-hydroxysteroid 3-dehydrogenase (A-specific)',\n",
       "  'sys_name': '3α-hydroxysteroid:NAD(P)+ 3-oxidoreductase (Re-specific)',\n",
       "  'comments': 'The enzyme acts on multiple 3α-hydroxysteroids. Re-specific with respect to NAD+ or NADP+ [cf. EC 1.1.1.50, 3α-hydroxysteroid 3-dehydrogenase (Si-specific)]. Enzymes whose stereo-specificity with respect to NAD+ or NADP+ is not known are described by EC 1.1.1.357, 3α-hydroxysteroid 3-dehydrogenase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.214': {'ec_num': '1.1.1.214',\n",
       "  'accepted_name': '2-dehydropantolactone reductase (Si-specific)',\n",
       "  'reaction': '(R)-pantolactone + NADP+ = 2-dehydropantolactone + NADPH + H+',\n",
       "  'other_names': '2-oxopantoyl lactone reductase; 2-ketopantoyl lactone reductase; ketopantoyl lactone reductase; 2-dehydropantoyl-lactone reductase (B-specific); (R)-pantolactone:NADP+ oxidoreductase (B-specific); 2-dehydropantolactone reductase (B-specific)',\n",
       "  'sys_name': '(R)-pantolactone:NADP+ oxidoreductase (Si-specific)',\n",
       "  'comments': 'The Escherichia coli enzyme differs from that from yeast [EC 1.1.1.168 2-dehydropantolactone reductase (Re-specific)], which is specific for the Re-face of NADP+, and in receptor requirements from EC 1.1.99.26 3-hydroxycyclohexanone dehydrogenase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.215': {'ec_num': '1.1.1.215',\n",
       "  'accepted_name': 'gluconate 2-dehydrogenase',\n",
       "  'reaction': 'D-gluconate + NADP+ = 2-dehydro-D-gluconate + NADPH + H+',\n",
       "  'other_names': '2-keto-D-gluconate reductase; 2-ketogluconate reductase',\n",
       "  'sys_name': 'D-gluconate:NADP+ oxidoreductase',\n",
       "  'comments': 'Also acts on L-idonate, D-galactonate and D-xylonate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.216': {'ec_num': '1.1.1.216',\n",
       "  'accepted_name': 'farnesol dehydrogenase (NADP+)',\n",
       "  'reaction': '(2E,6E)-farnesol + NADP+ = (2E,6E)-farnesal + NADPH + H+',\n",
       "  'other_names': 'NADP+-farnesol dehydrogenase; farnesol (nicotinamide adenine dinucleotide phosphate) dehydrogenase',\n",
       "  'sys_name': '(2E,6E)-farnesol:NADP+ 1-oxidoreductase',\n",
       "  'comments': 'Also acts, more slowly, on (2Z,6E)-farnesol, geraniol, citronerol and nerol.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.217': {'ec_num': '1.1.1.217',\n",
       "  'accepted_name': 'benzyl-2-methyl-hydroxybutyrate dehydrogenase',\n",
       "  'reaction': 'benzyl (2R,3S)-2-methyl-3-hydroxybutanoate + NADP+ = benzyl 2-methyl-3-oxobutanoate + NADPH + H+',\n",
       "  'other_names': 'benzyl 2-methyl-3-hydroxybutyrate dehydrogenase',\n",
       "  'sys_name': 'benzyl-(2R,3S)-2-methyl-3-hydroxybutanoate:NADP+ 3-oxidoreductase',\n",
       "  'comments': 'Also acts on benzyl (2S,3S)-2-methyl-3-hydroxybutanoate; otherwise highly specific.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.218': {'ec_num': '1.1.1.218',\n",
       "  'accepted_name': 'morphine 6-dehydrogenase',\n",
       "  'reaction': 'morphine + NAD(P)+ = morphinone + NAD(P)H + H+',\n",
       "  'other_names': 'naloxone reductase',\n",
       "  'sys_name': 'morphine:NAD(P)+ 6-oxidoreductase',\n",
       "  'comments': 'Also acts on some other alkaloids, including codeine, normorphine and ethylmorphine, but only very slowly on 7,8-saturated derivatives such as dihydromorphine and dihydrocodeine. In the reverse direction, also reduces naloxone to the 6α-hydroxy analogue. Activated by 2-sulfanylethan-1-ol (2-mercaptoethanol).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.219': {'ec_num': '1.1.1.219',\n",
       "  'accepted_name': 'dihydroflavonol 4-reductase',\n",
       "  'reaction': 'a (2R,3S,4S)-leucoanthocyanidin + NADP+ = a (2R,3R)-dihydroflavonol + NADPH + H+',\n",
       "  'other_names': 'dihydrokaempferol 4-reductase; dihydromyricetin reductase; NADPH-dihydromyricetin reductase; dihydroquercetin reductase; DFR (gene name); cis-3,4-leucopelargonidin:NADP+ 4-oxidoreductase; dihydroflavanol 4-reductase (incorrect)',\n",
       "  'sys_name': '(2R,3S,4S)-leucoanthocyanidin:NADP+ 4-oxidoreductase',\n",
       "  'comments': 'This plant enzyme, involved in the biosynthesis of anthocyanidins, is known to act on (+)-dihydrokaempferol, (+)-taxifolin, and (+)-dihydromyricetin, although some enzymes may act only on a subset of these compounds. Each dihydroflavonol is reduced to the corresponding cis-flavan-3,4-diol. NAD+ can act instead of NADP+, but more slowly.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.220': {'ec_num': '1.1.1.220',\n",
       "  'accepted_name': '6-pyruvoyltetrahydropterin 2′-reductase',\n",
       "  'reaction': '6-lactoyl-5,6,7,8-tetrahydropterin + NADP+ = 6-pyruvoyltetrahydropterin + NADPH + H+',\n",
       "  'other_names': '6-pyruvoyltetrahydropterin reductase; 6PPH4(2′-oxo) reductase; 6-pyruvoyl tetrahydropterin (2′-oxo)reductase; 6-pyruvoyl-tetrahydropterin 2′-reductase; pyruvoyl-tetrahydropterin reductase',\n",
       "  'sys_name': '6-lactoyl-5,6,7,8-tetrahydropterin:NADP+ 2′-oxidoreductase',\n",
       "  'comments': 'Not identical with EC 1.1.1.153 sepiapterin reductase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.221': {'ec_num': '1.1.1.221',\n",
       "  'accepted_name': 'vomifoliol dehydrogenase',\n",
       "  'reaction': '(6S,9R)-6-hydroxy-3-oxo-α-ionol + NAD+ = (6S)-6-hydroxy-3-oxo-α-ionone + NADH + H+',\n",
       "  'other_names': 'vomifoliol 4′-dehydrogenase; vomifoliol:NAD+ 4′-oxidoreductase',\n",
       "  'sys_name': '(6S,9R)-6-hydroxy-3-oxo-α-ionol:NAD+ oxidoreductase',\n",
       "  'comments': 'Oxidizes vomifoliol to dehydrovomifoliol; involved in the metabolism of abscisic acid in Corynebacterium sp.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '(6S,9R)-6-hydroxy-3-oxo-α-ionol = vomifoliol = (4S)-4-hydroxy-4-[(1E,3R)-3-hydroxybut-1-en-1-yl]-3,5,5-trimethylcyclohex-2-en-1-one\\n(6S)-6-hydroxy-3-oxo-α-ionone = dehydrovomifoliol = (4S)-4-hydroxy-3,5,5-trimethyl-4-[(1E)-3-oxobut-1-en-1-yl]cyclohex-2-en-1-one',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.222': {'ec_num': '1.1.1.222',\n",
       "  'accepted_name': '(R)-4-hydroxyphenyllactate dehydrogenase',\n",
       "  'reaction': '(R)-3-(4-hydroxyphenyl)lactate + NAD(P)+ = 3-(4-hydroxyphenyl)pyruvate + NAD(P)H + H+',\n",
       "  'other_names': '(R)-aromatic lactate dehydrogenase; D-aryllactate D-hydrogenase',\n",
       "  'sys_name': '(R)-3-(4-hydroxyphenyl)lactate:NAD(P)+ 2-oxidoreductase',\n",
       "  'comments': 'Also acts, more slowly, on (R)-3-phenyllactate, (R)-3-(indol-3-yl)lactate and (R)-lactate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.223': {'ec_num': '1.1.1.223',\n",
       "  'accepted_name': 'isopiperitenol dehydrogenase',\n",
       "  'reaction': '(-)-trans-isopiperitenol + NAD+ = (-)-isopiperitenone + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '(-)-trans-isopiperitenol:NAD+ oxidoreductase',\n",
       "  'comments': 'Acts on (-)-trans-isopiperitenol, (+)-trans-piperitenol and (+)-trans-pulegol. Involved in the biosynthesis of menthol and related monoterpenes in peppermint (Mentha piperita) leaves.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.223\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.223\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.223\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.223\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.224': {'ec_num': '1.1.1.224',\n",
       "  'accepted_name': 'mannose-6-phosphate 6-reductase',\n",
       "  'reaction': 'D-mannitol 1-phosphate + NADP+ = D-mannose 6-phosphate + NADPH + H+',\n",
       "  'other_names': 'NADPH-dependent mannose 6-phosphate reductase; mannose-6-phosphate reductase; 6-phosphomannose reductase; NADP-dependent mannose-6-P:mannitol-1-P oxidoreductase; NADPH-dependent M6P reductase; NADPH-mannose-6-P reductase',\n",
       "  'sys_name': 'D-mannitol-1-phosphate:NADP+ 6-oxidoreductase',\n",
       "  'comments': 'Involved in the biosynthesis of mannitol in celery (Apium graveolens) leaves.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.225': {'ec_num': '1.1.1.225',\n",
       "  'accepted_name': 'chlordecone reductase',\n",
       "  'reaction': 'chlordecone alcohol + NADP+ = chlordecone + NADPH + H+',\n",
       "  'other_names': 'CDR',\n",
       "  'sys_name': 'chlordecone-alcohol:NADP+ 2-oxidoreductase',\n",
       "  'comments': 'Chlordecone is an organochlorine pesticide.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.226': {'ec_num': '1.1.1.226',\n",
       "  'accepted_name': '4-hydroxycyclohexanecarboxylate dehydrogenase',\n",
       "  'reaction': 'trans-4-hydroxycyclohexanecarboxylate + NAD+ = 4-oxocyclohexanecarboxylate + NADH + H+',\n",
       "  'other_names': 'trans-4-hydroxycyclohexanecarboxylate dehydrogenase',\n",
       "  'sys_name': 'trans-4-hydroxycyclohexanecarboxylate:NAD+ 4-oxidoreductase',\n",
       "  'comments': 'The enzyme from Corynebacterium cyclohexanicum is highly specific for the trans-4-hydroxy derivative.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.227': {'ec_num': '1.1.1.227',\n",
       "  'accepted_name': '(-)-borneol dehydrogenase',\n",
       "  'reaction': '(-)-borneol + NAD+ = (-)-camphor + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '(-)-borneol:NAD+ oxidoreductase',\n",
       "  'comments': 'NADP+ can also act, but more slowly.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.228': {'ec_num': '1.1.1.228',\n",
       "  'accepted_name': '(+)-sabinol dehydrogenase',\n",
       "  'reaction': '(+)-cis-sabinol + NAD+ = (+)-sabinone + NADH + H+',\n",
       "  'other_names': '(+)-cis-sabinol dehydrogenase',\n",
       "  'sys_name': '(+)-cis-sabinol:NAD+ oxidoreductase',\n",
       "  'comments': 'NADP+ can also act, but more slowly. Involved in the biosynthesis of (+)-3-thujone and (–)-3-isothujone.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.229': {'ec_num': '1.1.1.229',\n",
       "  'accepted_name': 'diethyl 2-methyl-3-oxosuccinate reductase',\n",
       "  'reaction': 'diethyl (2R,3R)-2-methyl-3-hydroxysuccinate + NADP+ = diethyl 2-methyl-3-oxosuccinate + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'diethyl-(2R,3R)-2-methyl-3-hydroxysuccinate:NADP+ 3-oxidoreductase',\n",
       "  'comments': 'Also acts on diethyl (2S,3R)-2-methyl-3-hydroxysuccinate; and on the corresponding dimethyl esters.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.230': {'ec_num': '1.1.1.230',\n",
       "  'accepted_name': '3α-hydroxyglycyrrhetinate dehydrogenase',\n",
       "  'reaction': '3α-hydroxyglycyrrhetinate + NADP+ = 3-oxoglycyrrhetinate + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '3α-hydroxyglycyrrhetinate:NADP+ 3-oxidoreductase',\n",
       "  'comments': 'Highly specific to 3α-hydroxy derivatives of glycyrrhetinate and its analogues. Not identical to EC 1.1.1.50 3α-hydroxysteroid dehydrogenase (Si-specific).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.231': {'ec_num': '1.1.1.231',\n",
       "  'accepted_name': '15-hydroxyprostaglandin-I dehydrogenase (NADP+)',\n",
       "  'reaction': '(5Z,13E)-(15S)-6,9α-epoxy-11α,15-dihydroxyprosta-5,13-dienoate + NADP+ = (5Z,13E)-6,9α-epoxy-11α-hydroxy-15-oxoprosta-5,13-dienoate + NADPH + H+',\n",
       "  'other_names': 'prostacyclin dehydrogenase; PG I2 dehydrogenase; prostacyclin dehydrogenase; NADP-linked 15-hydroxyprostaglandin (prostacyclin) dehydrogenase; NADP+-dependent PGI2-specific 15-hydroxyprostaglandin dehydrogenase; 15-hydroxyprostaglandin-I dehydrogenase (NADP)',\n",
       "  'sys_name': '(5Z,13E)-(15S)-6,9α-epoxy-11α,15-dihydroxyprosta-5,13-dienoate:NADP+ 15-oxidoreductase',\n",
       "  'comments': 'Specific for prostaglandin I2.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.232': {'ec_num': '1.1.1.232',\n",
       "  'accepted_name': '15-hydroxyicosatetraenoate dehydrogenase',\n",
       "  'reaction': '(15S)-15-hydroxy-5,8,11-cis-13-trans-icosatetraenoate + NAD(P)+ = 15-oxo-5,8,11-cis-13-trans-icosatetraenoate + NAD(P)H + H+',\n",
       "  'other_names': '15-hydroxyeicosatetraenoate dehydrogenase',\n",
       "  'sys_name': '(15S)-15-hydroxy-5,8,11-cis-13-trans-icosatetraenoate:NAD(P)+ 15-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.233': {'ec_num': '1.1.1.233',\n",
       "  'accepted_name': 'N-acylmannosamine 1-dehydrogenase',\n",
       "  'reaction': 'N-acyl-D-mannosamine + NAD+ = N-acyl-D-mannosaminolactone + NADH + H+',\n",
       "  'other_names': 'N-acylmannosamine dehydrogenase; N-acetyl-D-mannosamine dehydrogenase; N-acyl-D-mannosamine dehydrogenase; N-acylmannosamine dehydrogenase',\n",
       "  'sys_name': 'N-acyl-D-mannosamine:NAD+ 1-oxidoreductase',\n",
       "  'comments': 'Acts on acetyl-D-mannosamine and glycolyl-D-mannosamine. Highly specific.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.234': {'ec_num': '1.1.1.234',\n",
       "  'accepted_name': 'flavanone 4-reductase',\n",
       "  'reaction': '(2S)-flavan-4-ol + NADP+ = (2S)-flavanone + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '(2S)-flavan-4-ol:NADP+ 4-oxidoreductase',\n",
       "  'comments': 'Involved in the biosynthesis of 3-deoxyanthocyanidins from flavanones such as naringenin or eriodictyol.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.235': {'ec_num': '1.1.1.235',\n",
       "  'accepted_name': '8-oxocoformycin reductase',\n",
       "  'reaction': 'coformycin + NADP+ = 8-oxocoformycin + NADPH + H+',\n",
       "  'other_names': '8-ketodeoxycoformycin reductase',\n",
       "  'sys_name': 'coformycin:NADP+ 8-oxidoreductase',\n",
       "  'comments': 'Si-specific with respect to NADPH. Also reduces 8-oxodeoxy-coformycin to the nucleoside antibiotic deoxycoformycin.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.236': {'ec_num': '1.1.1.236',\n",
       "  'accepted_name': 'tropinone reductase II',\n",
       "  'reaction': 'pseudotropine + NADP+ = tropinone + NADPH + H+',\n",
       "  'other_names': 'tropinone (ψ-tropine-forming) reductase; pseudotropine forming tropinone reductase; tropinone reductase (ambiguous); TR-II',\n",
       "  'sys_name': 'pseudotropine:NADP+ 3-oxidoreductase',\n",
       "  'comments': 'This enzyme along with EC 1.1.1.206, tropine dehydrogenase, represents a branch point in tropane alkaloid metabolism [3].  Tropine (the product of EC 1.1.1.206) is incorporated into hyoscyamine and scopolamine whereas pseudotropine (the product of EC 1.1.1.236) is the first specific metabolite on the pathway to the calystegines [3]. Both enzymes are always found together in any given tropane-alkaloid-producing species, have a common substrate, tropinone, and are strictly stereospecific [2].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'pseudotropine = ψ-tropine = 3β-hydroxytropane = tropan-3-exo-ol',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.237': {'ec_num': '1.1.1.237',\n",
       "  'accepted_name': 'hydroxyphenylpyruvate reductase',\n",
       "  'reaction': '(1) (R)-3-(4-hydroxyphenyl)lactate + NAD(P)+ = 3-(4-hydroxyphenyl)pyruvate + NAD(P)H + H+;;(2) (R)-3-(3,4-dihydroxyphenyl)lactate + NAD(P)+ = 3-(3,4-dihydroxyphenyl)pyruvate + NAD(P)H + H+;;',\n",
       "  'other_names': 'HPPR',\n",
       "  'sys_name': '(R)-3-(4-hydroxyphenyl)lactate:NAD(P)+ oxidoreductase',\n",
       "  'comments': 'The enzyme participates in the biosynthesis of rosmarinic acid. It belongs to the family of D-isomer-specific 2-hydroxyacid dehydrogenases, and prefers NADPH to NADH.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.238': {'ec_num': '1.1.1.238',\n",
       "  'accepted_name': '12β-hydroxysteroid dehydrogenase',\n",
       "  'reaction': '3α,7α,12β-trihydroxy-5β-cholan-24-oate + NADP+ = 3α,7α-dihydroxy-12-oxo-5β-cholan-24-oate + NADPH + H+',\n",
       "  'other_names': '12β-hydroxy steroid (nicotinamide adenine dinucleotide phosphate) dehydrogenase',\n",
       "  'sys_name': '12β-hydroxysteroid:NADP+ 12-oxidoreductase',\n",
       "  'comments': 'Acts on a number of bile acids, both in their free and conjugated forms.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.239': {'ec_num': '1.1.1.239',\n",
       "  'accepted_name': '3α(17β)-hydroxysteroid dehydrogenase (NAD+)',\n",
       "  'reaction': 'testosterone + NAD+ = androstenedione + NADH + H+',\n",
       "  'other_names': '3α,17β-hydroxy steroid dehydrogenase; 3α(17β)-HSD; 17-ketoreductase (ambiguous); 17β-HSD (ambiguous); HSD17B6 (gene name); HSD17B8 (gene name)',\n",
       "  'sys_name': '3α(or 17β)-hydroxysteroid:NAD+ oxidoreductase',\n",
       "  'comments': 'Also acts on other 17β-hydroxysteroids and on the 3α-hydroxy group of pregnanes and bile acids. Different from EC 1.1.1.50 3α-hydroxysteroid dehydrogenase (Si-specific) or EC 1.1.1.213 3α-hydroxysteroid dehydrogenase (Re-specific).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'androstenedione = androst-4-ene-3,17-dione',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.240': {'ec_num': '1.1.1.240',\n",
       "  'accepted_name': 'N-acetylhexosamine 1-dehydrogenase',\n",
       "  'reaction': 'N-acetyl-α-D-glucosamine + NAD+ = N-acetyl-D-glucosaminate + NADH + H+',\n",
       "  'other_names': 'N-acetylhexosamine dehydrogenase; N-acetyl-D-hexosamine dehydrogenase',\n",
       "  'sys_name': 'N-acetyl-D-hexosamine:NAD+ 1-oxidoreductase',\n",
       "  'comments': 'Also acts on N-acetylgalactosamine and, more slowly, on N-acetylmannosamine. Anomeric specificity was tested with N-acetyl-D-glucosamine, and it was shown that the enzyme is specific for the α anomer.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.241': {'ec_num': '1.1.1.241',\n",
       "  'accepted_name': '6-endo-hydroxycineole dehydrogenase',\n",
       "  'reaction': '6-endo-hydroxycineole + NAD+ = 6-oxocineole + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '6-endo-hydroxycineole:NAD+ 6-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.243': {'ec_num': '1.1.1.243',\n",
       "  'accepted_name': 'carveol dehydrogenase',\n",
       "  'reaction': '(–)-trans-carveol + NADP+ = (–)-carvone + NADPH + H+',\n",
       "  'other_names': '(–)-trans-carveol dehydrogenase',\n",
       "  'sys_name': '(–)-trans-carveol:NADP+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.243\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.243\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.243\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.243\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.244': {'ec_num': '1.1.1.244',\n",
       "  'accepted_name': 'methanol dehydrogenase',\n",
       "  'reaction': 'methanol + NAD+ = formaldehyde + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'methanol:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.245': {'ec_num': '1.1.1.245',\n",
       "  'accepted_name': 'cyclohexanol dehydrogenase',\n",
       "  'reaction': 'cyclohexanol + NAD+ = cyclohexanone + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'cyclohexanol:NAD+ oxidoreductase',\n",
       "  'comments': 'Also oxidizes some other alicyclic alcohols and diols.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.245\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.245\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.245\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.245\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.246': {'ec_num': '1.1.1.246',\n",
       "  'accepted_name': 'vestitone reductase',\n",
       "  'reaction': '7,2′-dihydroxy-4′-methoxyisoflavanol + NADP+ = vestitone + NADPH + H+',\n",
       "  'other_names': 'pterocarpin synthase (incorrect); pterocarpan synthase (incorrect); medicarpin:NADP+ 2′-oxidoreductase',\n",
       "  'sys_name': '7,2′-dihydroxy-4′-methoxyisoflavanol:NADP+ 2′-oxidoreductase',\n",
       "  'comments': 'The reaction is catalysed in the opposite direction to that shown. This plant enzyme catalyses the penultimate step in the biosynthesis of the pterocarpin phytoalexins medicarpin and maackiain.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.247': {'ec_num': '1.1.1.247',\n",
       "  'accepted_name': 'codeinone reductase (NADPH)',\n",
       "  'reaction': 'codeine + NADP+ = codeinone + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'codeine:NADP+ oxidoreductase',\n",
       "  'comments': 'Catalyses the reversible reduction of codeinone to codeine, which is a direct precursor of morphine in the opium poppy plant, Papaver somniferum.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.248': {'ec_num': '1.1.1.248',\n",
       "  'accepted_name': 'salutaridine reductase (NADPH)',\n",
       "  'reaction': 'salutaridinol + NADP+ = salutaridine + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'salutaridinol:NADP+ 7-oxidoreductase',\n",
       "  'comments': 'Catalyses the reversible reduction of salutaridine to salutaridinol, which is a direct precursor of morphinan alkaloids in the poppy plant.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.250': {'ec_num': '1.1.1.250',\n",
       "  'accepted_name': 'D-arabinitol 2-dehydrogenase',\n",
       "  'reaction': 'D-arabinitol + NAD+ = D-ribulose + NADH + H+',\n",
       "  'other_names': 'D-arabinitol 2-dehydrogenase (ribulose-forming)',\n",
       "  'sys_name': 'D-arabinitol:NAD+ 2-oxidoreductase (D-ribulose-forming)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.251': {'ec_num': '1.1.1.251',\n",
       "  'accepted_name': 'galactitol-1-phosphate 5-dehydrogenase',\n",
       "  'reaction': 'galactitol 1-phosphate + NAD+ = D-tagatose 6-phosphate + NADH + H+',\n",
       "  'other_names': 'gatD (gene name)',\n",
       "  'sys_name': 'galactitol-1-phosphate:NAD+ oxidoreductase',\n",
       "  'comments': 'The enzyme from the bacterium Escherichia coli is involved in a galactitol degradation pathway. It contains two zinc atoms per subunit.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.252': {'ec_num': '1.1.1.252',\n",
       "  'accepted_name': 'tetrahydroxynaphthalene reductase',\n",
       "  'reaction': 'scytalone + NADP+ = 1,3,6,8-tetrahydroxynaphthalene + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'scytalone:NADP+ Δ5-oxidoreductase',\n",
       "  'comments': 'Reduces 1,3,6,8-tetrahydroxynaphthalene to scytalone and also reduces 1,3,8-trihydroxynaphthalene to vermelone. Involved with EC 4.2.1.94 scytalone dehydratase in the biosynthesis of melanin in pathogenic fungi.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.254': {'ec_num': '1.1.1.254',\n",
       "  'accepted_name': '(S)-carnitine 3-dehydrogenase',\n",
       "  'reaction': '(S)-carnitine + NAD+ = 3-dehydrocarnitine + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '(S)-carnitine:NAD+ oxidoreductase',\n",
       "  'comments': 'Specific for the (S)-enantiomer of carnitine, i.e., the enantiomer of the substrate of EC 1.1.1.108 carnitine 3-dehydrogenase',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.255': {'ec_num': '1.1.1.255',\n",
       "  'accepted_name': 'mannitol dehydrogenase',\n",
       "  'reaction': 'D-mannitol + NAD+ = D-mannose + NADH + H+',\n",
       "  'other_names': 'MTD; NAD-dependent mannitol dehydrogenase',\n",
       "  'sys_name': 'mannitol:NAD+ 1-oxidoreductase',\n",
       "  'comments': 'The enzyme from Apium graveolens (celery) oxidizes alditols with a minimum requirement of 2R chirality at the carbon adjacent to the primary carbon undergoing the oxidation. The enzyme is specific for NAD+ and does not use NADP+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.256': {'ec_num': '1.1.1.256',\n",
       "  'accepted_name': 'fluoren-9-ol dehydrogenase',\n",
       "  'reaction': 'fluoren-9-ol + NAD(P)+ = fluoren-9-one + NAD(P)H + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'fluoren-9-ol:NAD(P)+ oxidoreductase',\n",
       "  'comments': 'Involved in the pathway for fluorene metabolism in Arthrobacter sp.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.256\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.256\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.256\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.256\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.257': {'ec_num': '1.1.1.257',\n",
       "  'accepted_name': '4-(hydroxymethyl)benzenesulfonate dehydrogenase',\n",
       "  'reaction': '4-(hydroxymethyl)benzenesulfonate + NAD+ = 4-formylbenzenesulfonate + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '4-(hydroxymethyl)benzenesulfonate:NAD+ oxidoreductase',\n",
       "  'comments': 'Involved in the toluene-4-sulfonate degradation pathway in Comamonas testosteroni.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.257\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.257\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.257\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.257\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.258': {'ec_num': '1.1.1.258',\n",
       "  'accepted_name': '6-hydroxyhexanoate dehydrogenase',\n",
       "  'reaction': '6-hydroxyhexanoate + NAD+ = 6-oxohexanoate + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '6-hydroxyhexanoate:NAD+ oxidoreductase',\n",
       "  'comments': 'Involved in the cyclohexanol degradation pathway in Acinetobacter NCIB 9871.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.258\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.258\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.258\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.258\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.259': {'ec_num': '1.1.1.259',\n",
       "  'accepted_name': '3-hydroxypimeloyl-CoA dehydrogenase',\n",
       "  'reaction': '3-hydroxypimeloyl-CoA + NAD+ = 3-oxopimeloyl-CoA + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '3-hydroxypimeloyl-CoA:NAD+ oxidoreductase',\n",
       "  'comments': 'Involved in the anaerobic pathway of benzoate degradation in bacteria.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.259\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.259\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.259\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.259\" target=\"new\">KEGG</a>',\n",
       "  'glossary': 'pimelic acid = heptanedioic acid',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.260': {'ec_num': '1.1.1.260',\n",
       "  'accepted_name': 'sulcatone reductase',\n",
       "  'reaction': 'sulcatol + NAD+ = sulcatone + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'sulcatol:NAD+ oxidoreductase',\n",
       "  'comments': 'Studies on the effects of growth-stage and nutrient supply on the stereochemistry of sulcatone reduction in Clostridia pasteurianum, C. tyrobutyricum and Lactobacillus brevis suggest that there may be at least two sulcatone reductases with different stereospecificities.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'sulcatone = 6-methylhept-5-en-2-one\\nsulcatol = 6-methylhept-5-en-2-ol ',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.261': {'ec_num': '1.1.1.261',\n",
       "  'accepted_name': 'sn-glycerol-1-phosphate dehydrogenase',\n",
       "  'reaction': 'sn-glycerol 1-phosphate + NAD(P)+ = glycerone phosphate + NAD(P)H + H+',\n",
       "  'other_names': 'glycerol-1-phosphate dehydrogenase [NAD(P)+]; sn-glycerol-1-phosphate:NAD+ oxidoreductase; G-1-P dehydrogenase; Gro1PDH; AraM',\n",
       "  'sys_name': 'sn-glycerol-1-phosphate:NAD(P)+ 2-oxidoreductase',\n",
       "  'comments': 'This enzyme is found primarily as a Zn2+-dependent form in archaea but a Ni2+-dependent form has been found in Gram-positive bacteria [6]. The Zn2+-dependent metalloenzyme is responsible for the formation of archaea-specific sn-glycerol-1-phosphate, the first step in the biosynthesis of polar lipids in archaea. It is the enantiomer of sn-glycerol 3-phosphate, the form of glycerophosphate found in bacteria and eukaryotes. The other enzymes involved in the biosynthesis of polar lipids in archaea are EC 2.5.1.41 (phosphoglycerol geranylgeranyltransferase) and EC 2.5.1.42 (geranylgeranylglycerol-phosphate geranylgeranyltransferase), which together alkylate the hydroxy groups of glycerol 1-phosphate to give unsaturated archaetidic acid, which is acted upon by EC 2.7.7.67 (CDP-archaeol synthase) to form CDP-unsaturated archaeol. The final step in the pathway involves the addition of L-serine, with concomitant removal of CMP, leading to the production of unsaturated archaetidylserine [4]. Activity of the enzyme is stimulated by K+ [2].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'glycerone phosphate = dihydroxyacetone phosphate = 3-hydroxy-2-oxopropyl phosphate',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.262': {'ec_num': '1.1.1.262',\n",
       "  'accepted_name': '4-hydroxythreonine-4-phosphate dehydrogenase',\n",
       "  'reaction': '4-phosphooxy-L-threonine + NAD+ = 3-amino-2-oxopropyl phosphate + CO2 + NADH + H+',\n",
       "  'other_names': 'NAD+-dependent threonine 4-phosphate dehydrogenase; L-threonine 4-phosphate dehydrogenase; 4-(phosphohydroxy)-L-threonine dehydrogenase; PdxA; 4-(phosphonooxy)-L-threonine:NAD+ oxidoreductase; 4-phosphooxy-L-threonine:NAD+ oxidoreductase',\n",
       "  'sys_name': '4-phosphooxy-L-threonine:NAD+ 3-oxidoreductase (decarboxylating)',\n",
       "  'comments': ' The enzyme is part of the biosynthesis pathway of the cofactor pyridoxal 5′-phosphate found in anaerobic bacteria.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-07-27 09:35:05'},\n",
       " '1.1.1.263': {'ec_num': '1.1.1.263',\n",
       "  'accepted_name': '1,5-anhydro-D-fructose reductase',\n",
       "  'reaction': '1,5-anhydro-D-glucitol + NADP+ = 1,5-anhydro-D-fructose + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '1,5-anhydro-D-glucitol:NADP+ oxidoreductase',\n",
       "  'comments': 'Also reduces pyridine-3-aldehyde and 2,3-butanedione. Acetaldehyde, 2-dehydroglucose (glucosone) and glucuronate are poor substrates, but there is no detectable action on glucose, mannose and fructose.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.263\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.263\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.263\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.263\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.1.1.263%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.264': {'ec_num': '1.1.1.264',\n",
       "  'accepted_name': 'L-idonate 5-dehydrogenase',\n",
       "  'reaction': 'L-idonate + NAD(P)+ = 5-dehydro-D-gluconate + NAD(P)H + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'L-idonate:NAD(P)+ oxidoreductase',\n",
       "  'comments': 'The enzyme from the bacterium Escherichia coli is specific for 5-dehydro-D-gluconate. cf. EC 1.1.1.366, L-idonate 5-dehydrogenase (NAD+).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.265': {'ec_num': '1.1.1.265',\n",
       "  'accepted_name': '3-methylbutanal reductase',\n",
       "  'reaction': '3-methylbutanol + NAD(P)+ = 3-methylbutanal + NAD(P)H + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '3-methylbutanol:NAD(P)+ oxidoreductase',\n",
       "  'comments': 'The enzyme purified from Saccharomyces cerevisiae catalyses the reduction of a number of straight-chain and branched aldehydes, as well as some aromatic aldehydes.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.266': {'ec_num': '1.1.1.266',\n",
       "  'accepted_name': 'dTDP-4-dehydro-6-deoxyglucose reductase',\n",
       "  'reaction': 'dTDP-α-D-fucopyranose + NAD(P)+ = dTDP-4-dehydro-6-deoxy-α-D-glucose + NAD(P)H + H+',\n",
       "  'other_names': 'dTDP-4-keto-6-deoxyglucose reductase; dTDP-D-fucose:NADP+ oxidoreductase; Fcf1; dTDP-6-deoxy-D-xylo-hex-4-ulopyranose reductase',\n",
       "  'sys_name': 'dTDP-α-D-fucopyranose:NAD(P)+ oxidoreductase',\n",
       "  'comments': 'The enzymes from the Gram-negative bacteria Aggregatibacter actinomycetemcomitans and Escherichia coli O52 are involved in activation of fucose for incorporation into capsular polysaccharide O-antigens [1,3]. The enzyme from the Gram-positive bacterium Anoxybacillus tepidamans (Geobacillus tepidamans) is involved in activation of fucose for incorporation into the organism’s S-layer [2]. The enzyme from Escherichia coli O52 has a higher catalytic efficiency with NADH than with NADPH [3].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'dTDP-4-dehydro-6-deoxy-α-D-glucose = dTDP-6-deoxy-α-D-xylo-hexopyranos-4-ulose = thymidine 5′-[3-(6-deoxy--D-xylo-hexopyranosyl-4-ulose) diphosphate]',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.267': {'ec_num': '1.1.1.267',\n",
       "  'accepted_name': '1-deoxy-D-xylulose-5-phosphate reductoisomerase',\n",
       "  'reaction': '2-C-methyl-D-erythritol 4-phosphate + NADP+ = 1-deoxy-D-xylulose 5-phosphate + NADPH + H+',\n",
       "  'other_names': 'DXP-reductoisomerase; 1-deoxy-D-xylulose-5-phosphate isomeroreductase; 2-C-methyl-D-erythritol 4-phosphate (MEP) synthase',\n",
       "  'sys_name': '2-C-methyl-D-erythritol-4-phosphate:NADP+ oxidoreductase (isomerizing)',\n",
       "  'comments': 'The enzyme requires Mn2+, Co2+ or Mg2+ for activity, with the first being most effective. The enzyme from several eubacteria, including Escherichia coli, forms part of an alternative nonmevalonate pathway for terpenoid biosynthesis (for diagram, click here). The mechanism has been shown to be a retroaldol/aldol reaction [2].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.3.1.88': {'ec_num': '1.3.1.88',\n",
       "  'accepted_name': 'tRNA-dihydrouridine16/17 synthase [NAD(P)+]',\n",
       "  'reaction': '(1) 5,6-dihydrouracil16 in tRNA + NAD(P)+ = uracil16 in tRNA + NAD(P)H + H+;;(2) 5,6-dihydrouracil17 in tRNA + NAD(P)+ = uracil17 in tRNA + NAD(P)H + H+',\n",
       "  'other_names': 'Dus1p; tRNA-dihydrouridine synthase 1 ',\n",
       "  'sys_name': 'tRNA-5,6-dihydrouracil16/17:NAD(P)+ oxidoreductase',\n",
       "  'comments': 'A flavoprotein. The enzyme specifically modifies uracil16 and uracil17 in tRNA. ',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.1.268': {'ec_num': '1.1.1.268',\n",
       "  'accepted_name': '2-(R)-hydroxypropyl-CoM dehydrogenase',\n",
       "  'reaction': '2-(R)-hydroxypropyl-CoM + NAD+ = 2-oxopropyl-CoM + NADH + H+',\n",
       "  'other_names': '2-(2-(R)-hydroxypropylthio)ethanesulfonate dehydrogenase; 2-[2-(R)-hydroxypropylthio]ethanesulfonate:NAD+ oxidoreductase',\n",
       "  'sys_name': '2-{[(2R)-2-hydroxypropyl]sulfanyl}ethane-1-sulfonate:NAD+ oxidoreductase',\n",
       "  'comments': 'The enzyme is highly specific for (R)-2-hydroxyalkyl thioethers of CoM, in contrast to EC 1.1.1.269, 2-(S)-hydroxypropyl-CoM dehydrogenase, which is highly specific for the (S)-enantiomer. This enzyme forms component III of a four-component enzyme system (comprising EC 4.4.1.23 [2-hydroxypropyl-CoM lyase; component I], EC 1.8.1.5 [2-oxopropyl-CoM reductase (carboxylating); component II], EC 1.1.1.268 [2-(R)-hydroxypropyl-CoM dehydrogenase; component III] and EC 1.1.1.269 [2-(S)-hydroxypropyl-CoM dehydrogenase; component IV]) that is involved in epoxyalkane carboxylation in Xanthobacter sp. strain Py2.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.268\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.268\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.268\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.268\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.1.1.268%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': 'coenzyme M (CoM) = 2-sulfanylethane-1-sulfonate = 2-mercaptoethanesulfonate (deprecated)',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.269': {'ec_num': '1.1.1.269',\n",
       "  'accepted_name': '2-(S)-hydroxypropyl-CoM dehydrogenase',\n",
       "  'reaction': '(2S)-2-hydroxypropyl-CoM + NAD+ = 2-oxopropyl-CoM + NADH + H+',\n",
       "  'other_names': '2-(2-(S)-hydroxypropylthio)ethanesulfonate dehydrogenase; 2-[2-(S)-hydroxypropylthio]ethanesulfonate:NAD+ oxidoreductase',\n",
       "  'sys_name': '2-{[(2S)-2-hydroxypropyl]sulfanyl}ethanesulfonate:NAD+ oxidoreductase',\n",
       "  'comments': 'The enzyme is highly specific for (2<em>S</em>)-2-hydroxyalkyl thioethers of CoM, in contrast to <a href=\"query.php?ec=1.1.1.268\" target=\"new\">EC 1.1.1.268</a>, 2-(<em>R</em>)-hydroxypropyl-CoM dehydrogenase, which is highly specific for the (<em>R</em>)-enantiomer. This enzyme forms component IV of a four-component enzyme system <a target=\"new\" href=\"reaction/comprising <a href=\"query.php?ec=4.4.1.23\" target=\"new\">EC 4.4.1.23</a> (2-hydroxypropyl-CoM lyase; component I), <a href=\"query.php?ec=1.8.1.5\" target=\"new\">EC 1.8.1.5</a> [2-oxopropyl-CoM reductase (carboxylating); component II], <a href=\"query.php?ec=1.1.1.268\" target=\"new\">EC 1.1.1.268</a> [2-(<em>R</em>)-hydroxypropyl-CoM dehydrogenase; component III] and EC 1.1.1.269 [2-(<em>S</em>)-hydroxypropyl-CoM dehydrogenase; component IV].html\">click here</a> that is involved in epoxyalkane carboxylation in <em>Xanthobacter</em> sp. strain Py2.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.269\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.269\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.269\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.269\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.1.1.269%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': 'coenzyme M (CoM) = 2-sulfanylethane-1-sulfonate = 2-mercaptoethanesulfonate (deprecated)',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.270': {'ec_num': '1.1.1.270',\n",
       "  'accepted_name': '3β-hydroxysteroid 3-dehydrogenase',\n",
       "  'reaction': 'a 3β-hydroxysteroid + NADP+ = a 3-oxosteroid + NADPH + H+',\n",
       "  'other_names': '3-keto-steroid reductase; 3-KSR; HSD17B7 (gene name); ERG27 (gene name)',\n",
       "  'sys_name': '3β-hydroxysteroid:NADP+ 3-oxidoreductase',\n",
       "  'comments': 'The enzyme acts on multiple 3β-hydroxysteroids. Participates in the biosynthesis of zemosterol and cholesterol, where it catalyses the reaction in the opposite direction to that shown. The mammalian enzyme is bifunctional and also catalyses EC 1.1.1.62, 17β-estradiol 17-dehydrogenase [4].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.271': {'ec_num': '1.1.1.271',\n",
       "  'accepted_name': 'GDP-L-fucose synthase',\n",
       "  'reaction': 'GDP-β-L-fucose + NADP+ = GDP-4-dehydro-α-D-rhamnose + NADPH + H+',\n",
       "  'other_names': 'GDP-4-keto-6-deoxy-D-mannose-3,5-epimerase-4-reductase; GDP-L-fucose:NADP+ 4-oxidoreductase (3,5-epimerizing)',\n",
       "  'sys_name': 'GDP-β-L-fucose:NADP+ 4-oxidoreductase (3,5-epimerizing)',\n",
       "  'comments': 'Both human and Escherichia coli enzymes can use NADH in place of NADPH to a slight extent.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'GDP-4-dehydro-α-D-rhamnose = GDP-4-dehydro-6-deoxy-α-D-mannose',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.272': {'ec_num': '1.1.1.272',\n",
       "  'accepted_name': 'D-2-hydroxyacid dehydrogenase (NADP+)',\n",
       "  'reaction': 'an (R)-2-hydroxycarboxylate + NADP+ = a 2-oxocarboxylate + NADPH + H+',\n",
       "  'other_names': 'ddh (gene name)',\n",
       "  'sys_name': '(R)-2-hydroxycarboxylate:NADP+ oxidoreductase',\n",
       "  'comments': 'This enzyme, characterized from the halophilic archaeon Haloferax mediterranei and the mold Aspergillus oryzae, catalyses a stereospecific reduction of 2-oxocarboxylic acids into the corresponding D-2-hydroxycarboxylic acids. The enzyme prefers substrates with a main chain of 5 carbons (such as 4-methyl-2-oxopentanoate) to those with a shorter chain, and can use NADH with much lower efficiency. cf. EC 1.1.1.345, (D)-2-hydroxyacid dehydrogenase (NAD+).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.273': {'ec_num': '1.1.1.273',\n",
       "  'accepted_name': 'vellosimine dehydrogenase',\n",
       "  'reaction': '10-deoxysarpagine + NADP+ = vellosimine + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '10-deoxysarpagine:NADP+ oxidoreductase',\n",
       "  'comments': 'Also acts on related alkaloids with an endo-aldehyde group as vellosimine (same stereochemistry at C-16) but only slight activity with exo-aldehydes. Detected in many cell suspension cultures of plants from the family Apocynaceae.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.274': {'ec_num': '1.1.1.274',\n",
       "  'accepted_name': '2,5-didehydrogluconate reductase (2-dehydro-D-gluconate-forming)',\n",
       "  'reaction': '2-dehydro-D-gluconate + NADP+ = 2,5-didehydro-D-gluconate + NADPH + H+',\n",
       "  'other_names': '2,5-diketo-D-gluconate reductase (ambiguous)',\n",
       "  'sys_name': '2-dehydro-D-gluconate:NADP+ 2-oxidoreductase (2-dehydro-D-gluconate-forming)',\n",
       "  'comments': 'The enzyme is involved in the catabolism of 2,5-didehydrogluconate. cf. EC 1.1.1.346, 2,5-didehydrogluconate reductase (2-dehydro-L-gulonate-forming).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.343': {'ec_num': '1.1.1.343',\n",
       "  'accepted_name': 'phosphogluconate dehydrogenase (NAD+-dependent, decarboxylating)',\n",
       "  'reaction': '6-phospho-D-gluconate + NAD+ = D-ribulose 5-phosphate + CO2 + NADH + H+',\n",
       "  'other_names': '6-PGDH (ambiguous); gntZ (gene name); GNDl',\n",
       "  'sys_name': '6-phospho-D-gluconate:NAD+ 2-oxidoreductase (decarboxylating)',\n",
       "  'comments': 'Highly specific for NAD+. The enzyme catalyses both the oxidation and decarboxylation of 6-phospho-D-gluconate. In the bacterium Methylobacillus flagellatus the enzyme participates in a formaldehyde oxidation pathway [4]. cf. EC 1.1.1.44, phosphogluconate dehydrogenase (NADP+-dependent, decarboxylating).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.275': {'ec_num': '1.1.1.275',\n",
       "  'accepted_name': '(+)-trans-carveol dehydrogenase',\n",
       "  'reaction': '(+)-trans-carveol + NAD+ = (+)-(S)-carvone + NADH + H+',\n",
       "  'other_names': 'carveol dehydrogenase',\n",
       "  'sys_name': '(+)-trans-carveol:NAD+ oxidoreductase',\n",
       "  'comments': 'NADP+ cannot replace NAD+. Forms part of the monoterpenoid biosynthesis pathway in Carum carvi (caraway) seeds.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.276': {'ec_num': '1.1.1.276',\n",
       "  'accepted_name': 'serine 3-dehydrogenase (NADP+)',\n",
       "  'reaction': 'L-serine + NADP+ = 2-aminoacetaldehyde + CO2 + NADPH + H+ (overall reaction);; (1a) L-serine + NADP+ = 2-aminomalonate semialdehyde + NADPH + H+;; (1b) 2-aminomalonate semialdehyde = 2-aminoacetaldehyde + CO2 (spontaneous)',\n",
       "  'other_names': 'serine 3-dehydrogenase',\n",
       "  'sys_name': 'L-serine:NADP+ 3-oxidoreductase',\n",
       "  'comments': 'NAD+ cannot replace NADP+ [cf. EC 1.1.1.387, serine 3-dehydrogenase (NAD+)].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.277': {'ec_num': '1.1.1.277',\n",
       "  'accepted_name': '3β-hydroxy-5β-steroid dehydrogenase',\n",
       "  'reaction': '3β-hydroxy-5β-pregnane-20-one + NADP+ = 5β-pregnan-3,20-dione + NADPH + H+',\n",
       "  'other_names': '3β-hydroxysteroid 5β-oxidoreductase; 3β-hydroxysteroid 5β-progesterone oxidoreductase',\n",
       "  'sys_name': '3β-hydroxy-5β-steroid:NADP+ 3-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.278': {'ec_num': '1.1.1.278',\n",
       "  'accepted_name': '3β-hydroxy-5α-steroid dehydrogenase',\n",
       "  'reaction': '3β-hydroxy-5α-pregnane-20-one + NADP+ = 5α-pregnan-3,20-dione + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '3β-hydroxy-5α-steroid:NADP+ 3-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.279': {'ec_num': '1.1.1.279',\n",
       "  'accepted_name': '(R)-3-hydroxyacid-ester dehydrogenase',\n",
       "  'reaction': 'ethyl (R)-3-hydroxyhexanoate + NADP+ = ethyl 3-oxohexanoate + NADPH + H+',\n",
       "  'other_names': '3-oxo ester (R)-reductase',\n",
       "  'sys_name': 'ethyl-(R)-3-hydroxyhexanoate:NADP+ 3-oxidoreductase',\n",
       "  'comments': 'Also acts on ethyl (R)-3-oxobutanoate and some other (R)-3-hydroxy acid esters. The (R)- symbol is allotted on the assumption that no substituents change the order of priority from O-3 > C-2 > C-4. A subunit of yeast fatty acid synthase EC 2.3.1.86, fatty-acyl-CoA synthase system. cf. EC 1.1.1.280, (S)-3-hydroxyacid ester dehydrogenase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.280': {'ec_num': '1.1.1.280',\n",
       "  'accepted_name': '(S)-3-hydroxyacid-ester dehydrogenase',\n",
       "  'reaction': 'ethyl (S)-3-hydroxyhexanoate + NADP+ = ethyl 3-oxohexanoate + NADPH + H+',\n",
       "  'other_names': '3-oxo ester (S)-reductase',\n",
       "  'sys_name': 'ethyl-(S)-3-hydroxyhexanoate:NADP+ 3-oxidoreductase',\n",
       "  'comments': 'Also acts on 4-oxo- and 5-oxo-fatty acids and their esters. cf. EC 1.1.1.279 (R)-3-hydroxyacid-ester dehydrogenase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.281': {'ec_num': '1.1.1.281',\n",
       "  'accepted_name': 'GDP-4-dehydro-6-deoxy-D-mannose reductase',\n",
       "  'reaction': 'GDP-α-D-rhamnose + NAD(P)+ = GDP-4-dehydro-α-D-rhamnose + NAD(P)H + H+',\n",
       "  'other_names': 'GDP-4-keto-6-deoxy-D-mannose reductase [ambiguous]; GDP-6-deoxy-D-lyxo-4-hexulose reductase; Rmd; GDP-6-deoxy-D-mannose:NAD(P)+ 4-oxidoreductase (D-rhamnose-forming); GDP-6-deoxy-α-D-mannose:NAD(P)+ 4-oxidoreductase (D-rhamnose-forming)',\n",
       "  'sys_name': 'GDP-α-D-rhamnose:NAD(P)+ 4-oxidoreductase',\n",
       "  'comments': 'This enzyme differs from EC 1.1.1.187, GDP-4-dehydro-D-rhamnose reductase, in that the only product formed is GDP-α-D-rhamnose. D-Rhamnose is a constituent of lipopolysaccharides of Gram-negative plant and human pathogenic bacteria.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'GDP-α-D-rhamnose = GDP-6-deoxy-α-D-mannose\\nGDP-4-dehydro-α-D-rhamnose = GDP-4-dehydro-6-deoxy-α-D-mannose',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.282': {'ec_num': '1.1.1.282',\n",
       "  'accepted_name': 'quinate/shikimate dehydrogenase [NAD(P)+]',\n",
       "  'reaction': '(1) L-quinate + NAD(P)+ = 3-dehydroquinate + NAD(P)H + H+;;(2) shikimate + NAD(P)+ = 3-dehydroshikimate + NAD(P)H + H+',\n",
       "  'other_names': 'YdiB; quinate/shikimate dehydrogenase (ambiguous)',\n",
       "  'sys_name': 'L-quinate:NAD(P)+ 3-oxidoreductase',\n",
       "  'comments': 'This is the second shikimate dehydrogenase enzyme found in Escherichia coli. It can use both quinate and shikimate as substrates and either NAD+ or NADP+ as acceptor. The low catalytic efficiency with both quinate and shikimate suggests that neither may be the physiological substrate. cf. EC 1.1.1.24, quinate/shikimate dehydrogenase (NAD+), EC 1.1.5.8, quinate/shikimate dehydrogenase (quinone), and EC 1.1.1.25, shikimate dehydrogenase (NADP+).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': \"quinate = (1R,3R,4R,5R)-1,3,4,5-tetrahydroxycyclohexanecarboxylic acid and is a cyclitol carboxylate\\nThe numbering system used for the 3-dehydroquinate is that of the recommendations on cyclitols, sections I-8 and I-9: and is shown in the reaction diagram. The use of the term '5-dehydroquinate' for this compound is based on an earlier system of numbering.\",\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.283': {'ec_num': '1.1.1.283',\n",
       "  'accepted_name': 'methylglyoxal reductase (NADPH)',\n",
       "  'reaction': '(S)-lactaldehyde + NADP+ = 2-oxopropanal + NADPH + H+',\n",
       "  'other_names': 'lactaldehyde dehydrogenase (NADP+); GRE2 (gene name); methylglyoxal reductase (NADPH-dependent); lactaldehyde:NADP+ oxidoreductase',\n",
       "  'sys_name': '(S)-lactaldehyde:NADP+ oxidoreductase',\n",
       "  'comments': 'The enzyme from the yeast Saccharomyces cerevisiae catalyses the reduction of a keto group in a number of compounds, forming enantiopure products. Among the substrates are methylglyoxal (which is reduced to (S)-lactaldehyde) [1,2], 3-methylbutanal [3], hexane-2,5-dione [4] and 3-chloro-1-phenylpropan-1-one [5]. The enzyme differs from EC 1.1.1.78, methylglyoxal reductase (NADH), which is found in mammals, by its cosubstrate requirement, reaction direction, and enantiomeric preference.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '2-oxopropanal = methylglyoxal\\n',\n",
       "  'last_change': '2023-05-19 09:09:00'},\n",
       " '1.1.1.284': {'ec_num': '1.1.1.284',\n",
       "  'accepted_name': 'S-(hydroxymethyl)glutathione dehydrogenase',\n",
       "  'reaction': 'S-(hydroxymethyl)glutathione + NAD(P)+ = S-formylglutathione + NAD(P)H + H+',\n",
       "  'other_names': 'NAD-linked formaldehyde dehydrogenase (incorrect); formaldehyde dehydrogenase (incorrect); formic dehydrogenase (incorrect); class III alcohol dehydrogenase; ADH3; χ-ADH; FDH (incorrect); formaldehyde dehydrogenase (glutathione) (incorrect); GS-FDH (incorrect); glutathione-dependent formaldehyde dehydrogenase (incorrect); GD-FALDH; NAD- and glutathione-dependent formaldehyde dehydrogenase; NAD-dependent formaldehyde dehydrogenase (incorrect)',\n",
       "  'sys_name': 'S-(hydroxymethyl)glutathione:NAD+ oxidoreductase',\n",
       "  'comments': 'The substrate, S-(hydroxymethyl)glutathione, forms spontaneously from glutathione and formaldehyde; its rate of formation is increased in some bacteria by the presence of EC 4.4.1.22, S-(hydroxymethyl)glutathione synthase. This enzyme forms part of the pathway that detoxifies formaldehyde, since the product is hydrolysed by EC 3.1.2.12, S-formylglutathione hydrolase. The human enzyme belongs to the family of zinc-dependent alcohol dehydrogenases. Also specifically reduces S-nitrosylglutathione.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.1.284\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.1.284\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.1.284\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.1.284\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.1.1.284%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.285': {'ec_num': '1.1.1.285',\n",
       "  'accepted_name': '3′′-deamino-3′′-oxonicotianamine reductase',\n",
       "  'reaction': '2′-deoxymugineic acid + NAD(P)+ = 3′′-deamino-3′′-oxonicotianamine + NAD(P)H + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '2′-deoxymugineic acid:NAD(P)+ 3′′-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.286': {'ec_num': '1.1.1.286',\n",
       "  'accepted_name': 'isocitrate—homoisocitrate dehydrogenase',\n",
       "  'reaction': '(1) isocitrate + NAD+ = 2-oxoglutarate + CO2 + NADH;;(2) (1R,2S)-1-hydroxybutane-1,2,4-tricarboxylate + NAD+ = 2-oxoadipate + CO2 + NADH + H+',\n",
       "  'other_names': 'homoisocitrate—isocitrate dehydrogenase; PH1722',\n",
       "  'sys_name': 'isocitrate(homoisocitrate):NAD+ oxidoreductase (decarboxylating)',\n",
       "  'comments': 'Requires Mn2+ and K+ or NH4+ for activity. Unlike EC 1.1.1.41, isocitrate dehydrogenase (NAD+) and EC 1.1.1.87, homoisocitrate dehydrogenase, this enzyme, from Pyrococcus horikoshii, can use both isocitrate and homoisocitrate as substrates. The enzyme may play a role in both the lysine and glutamate biosynthesis pathways.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'isocitrate = (1R,2S)-1-hydroxypropane-1,2,3-tricarboxylate (previously known as threo-Ds-isocitrate)\\nhomoisocitrate = (-)-threo-homoisocitrate = (1R,2S)-1-hydroxybutane-1,2,4-tricarboxylate',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.287': {'ec_num': '1.1.1.287',\n",
       "  'accepted_name': 'D-arabinitol dehydrogenase (NADP+)',\n",
       "  'reaction': '(1) D-arabinitol + NADP+ = D-xylulose + NADPH + H+;;(2) D-arabinitol + NADP+ = D-ribulose + NADPH + H+',\n",
       "  'other_names': 'NADP+-dependent D-arabitol dehydrogenase; ARD1p; D-arabitol dehydrogenase 1',\n",
       "  'sys_name': 'D-arabinitol:NADP+ oxidoreductase',\n",
       "  'comments': 'The enzyme from the rust fungus Uromyces fabae can use D-arabinitol and D-mannitol as substrates in the forward direction and D-xylulose, D-ribulose and, to a lesser extent, D-fructose as substrates in the reverse direction. This enzyme carries out the reactions of both EC 1.1.1.11, D-arabinitol 4-dehydrogenase and EC 1.1.1.250, D-arabinitol 2-dehydrogenase, but unlike them, uses NADP+ rather than NAD+ as cofactor. D-Arabinitol is capable of quenching reactive oxygen species involved in defense reactions of the host plant.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.1.288': {'ec_num': '1.1.1.288',\n",
       "  'accepted_name': 'xanthoxin dehydrogenase',\n",
       "  'reaction': 'xanthoxin + NAD+ = abscisic aldehyde + NADH + H+',\n",
       "  'other_names': 'xanthoxin oxidase; ABA2',\n",
       "  'sys_name': 'xanthoxin:NAD+ oxidoreductase',\n",
       "  'comments': 'Requires a molybdenum cofactor for activity. NADP+ cannot replace NAD+ and short-chain alcohols such as ethanol, isopropanol, butanol and cyclohexanol cannot replace xanthoxin as substrate [3]. Involved in the abscisic-acid biosynthesis pathway in plants, along with EC 1.2.3.14 (abscisic-aldehyde oxidase), EC 1.13.11.51 (9-cis-epoxycarotenoid dioxygenase) and EC 1.14.13.93 [(+)-abscisic acid 8′-hydroxylase]. Abscisic acid is a sesquiterpenoid plant hormone that is involved in the control of a wide range of essential physiological processes, including seed development, germination and responses to stress [3].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:17'},\n",
       " '1.1.2.2': {'ec_num': '1.1.2.2',\n",
       "  'accepted_name': 'mannitol dehydrogenase (cytochrome)',\n",
       "  'reaction': 'D-mannitol + a ferricytochrome c = D-fructose + a ferrocytochrome c + 2 H+',\n",
       "  'other_names': 'polyol dehydrogenase',\n",
       "  'sys_name': 'D-mannitol:cytochrome-c 2-oxidoreductase',\n",
       "  'comments': 'The enzyme from the bacterium Gluconobacter oxydans acts on polyols with a D-lyxo configuration, such as D-mannitol and D-sorbitol, with preference towards the former.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.2.3': {'ec_num': '1.1.2.3',\n",
       "  'accepted_name': 'L-lactate dehydrogenase (cytochrome)',\n",
       "  'reaction': '(S)-lactate + 2 ferricytochrome c = pyruvate + 2 ferrocytochrome c + 2 H+',\n",
       "  'other_names': 'lactic acid dehydrogenase; cytochrome b2 (flavin-free derivative of flavocytochrome b2); flavocytochrome b2; L-lactate cytochrome c reductase; L(+)-lactate:cytochrome c oxidoreductase; dehydrogenase, lactate (cytochrome); L-lactate cytochrome c oxidoreductase; lactate dehydrogenase (cytochrome); lactic cytochrome c reductase',\n",
       "  'sys_name': '(S)-lactate:ferricytochrome-c 2-oxidoreductase',\n",
       "  'comments': 'Identical with cytochrome b2; a flavohemoprotein (FMN).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.2.4': {'ec_num': '1.1.2.4',\n",
       "  'accepted_name': 'D-lactate dehydrogenase (cytochrome)',\n",
       "  'reaction': '(R)-lactate + 2 ferricytochrome c = pyruvate + 2 ferrocytochrome c + 2 H+',\n",
       "  'other_names': 'lactic acid dehydrogenase; D-lactate (cytochrome) dehydrogenase; cytochrome-dependent D-(-)-lactate dehydrogenase; D-lactate-cytochrome c reductase; D-(-)-lactic cytochrome c reductase',\n",
       "  'sys_name': '(R)-lactate:cytochrome-c 2-oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.2.5': {'ec_num': '1.1.2.5',\n",
       "  'accepted_name': 'D-lactate dehydrogenase (cytochrome c-553)',\n",
       "  'reaction': '(R)-lactate + 2 ferricytochrome c-553 = pyruvate + 2 ferrocytochrome c-553 + 2 H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '(R)-lactate:cytochrome-c-553 2-oxidoreductase',\n",
       "  'comments': 'The enzyme from the sulfate-reducing bacterium Desulfovibrio vulgaris can also act on (R)-2-hydroxybutanoate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.3': {'ec_num': '1.1.3.3',\n",
       "  'accepted_name': 'malate oxidase',\n",
       "  'reaction': '(S)-malate + O2 = oxaloacetate + H2O2',\n",
       "  'other_names': 'FAD-dependent malate oxidase; malic oxidase; malic dehydrogenase II',\n",
       "  'sys_name': '(S)-malate:oxygen oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.4': {'ec_num': '1.1.3.4',\n",
       "  'accepted_name': 'glucose oxidase',\n",
       "  'reaction': 'β-D-glucose + O2 = D-glucono-1,5-lactone + H2O2',\n",
       "  'other_names': 'glucose oxyhydrase; corylophyline; penatin; glucose aerodehydrogenase; microcid; β-D-glucose oxidase; D-glucose oxidase; D-glucose-1-oxidase; β-D-glucose:quinone oxidoreductase; glucose oxyhydrase; deoxin-1; GOD',\n",
       "  'sys_name': 'β-D-glucose:oxygen 1-oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.5': {'ec_num': '1.1.3.5',\n",
       "  'accepted_name': 'hexose oxidase',\n",
       "  'reaction': 'D-glucose + O2 = D-glucono-1,5-lactone + H2O2',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'D-hexose:oxygen 1-oxidoreductase',\n",
       "  'comments': 'A copper glycoprotein. Also oxidizes D-galactose, D-mannose, maltose, lactose and cellobiose.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.6': {'ec_num': '1.1.3.6',\n",
       "  'accepted_name': 'cholesterol oxidase',\n",
       "  'reaction': 'cholesterol + O2 = cholest-5-en-3-one + H2O2',\n",
       "  'other_names': 'cholesterol- O2 oxidoreductase; 3β-hydroxy steroid oxidoreductase; 3β-hydroxysteroid:oxygen oxidoreductase',\n",
       "  'sys_name': 'cholesterol:oxygen oxidoreductase',\n",
       "  'comments': 'Contains FAD. Cholesterol oxidases are secreted bacterial bifunctional enzymes that catalyse the first two steps in the degradation of cholesterol. The enzyme catalyses the oxidation of the 3β-hydroxyl group to a keto group, and the isomerization of the double bond in the oxidized steroid ring system from the Δ5 position to Δ6 position (cf. EC 5.3.3.1, steroid Δ-isomerase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.7': {'ec_num': '1.1.3.7',\n",
       "  'accepted_name': 'aryl-alcohol oxidase',\n",
       "  'reaction': 'an aromatic primary alcohol + O2 = an aromatic aldehyde + H2O2',\n",
       "  'other_names': 'aryl alcohol oxidase; veratryl alcohol oxidase; arom. alcohol oxidase',\n",
       "  'sys_name': 'aryl-alcohol:oxygen oxidoreductase',\n",
       "  'comments': 'Oxidizes many primary alcohols containing an aromatic ring; best substrates are (2-naphthyl)methanol and 3-methoxybenzyl alcohol.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.8': {'ec_num': '1.1.3.8',\n",
       "  'accepted_name': 'L-gulonolactone oxidase',\n",
       "  'reaction': 'L-gulono-1,4-lactone + O2 = L-ascorbate + H2O2 (overall reaction);;(1a) L-gulono-1,4-lactone + O2 = L-xylo-hex-2-ulono-1,4-lactone + H2O2;;(1b) L-xylo-hex-2-ulono-1,4-lactone = L-ascorbate (spontaneous)',\n",
       "  'other_names': 'L-gulono-γ-lactone: O2 oxidoreductase; L-gulono-γ-lactone oxidase; L-gulono-γ-lactone:oxidoreductase; GLO',\n",
       "  'sys_name': 'L-gulono-1,4-lactone:oxygen 3-oxidoreductase',\n",
       "  'comments': 'A microsomal flavoprotein (FAD). The product spontaneously isomerizes to L-ascorbate. While most higher animals can synthesize asborbic acid, primates and guinea pigs cannot [3].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.9': {'ec_num': '1.1.3.9',\n",
       "  'accepted_name': 'galactose oxidase',\n",
       "  'reaction': 'D-galactose + O2 = D-galacto-hexodialdose + H2O2',\n",
       "  'other_names': 'D-galactose oxidase; β-galactose oxidase',\n",
       "  'sys_name': 'D-galactose:oxygen 6-oxidoreductase',\n",
       "  'comments': 'A copper protein.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.10': {'ec_num': '1.1.3.10',\n",
       "  'accepted_name': 'pyranose oxidase',\n",
       "  'reaction': 'D-glucose + O2 = 2-dehydro-D-glucose + H2O2',\n",
       "  'other_names': 'glucose 2-oxidase; pyranose-2-oxidase',\n",
       "  'sys_name': 'pyranose:oxygen 2-oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD). Also oxidizes D-xylose, L-sorbose and D-glucono-1,5-lactone, which have the same ring conformation and configuration at C-2, C-3 and C-4.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.11': {'ec_num': '1.1.3.11',\n",
       "  'accepted_name': 'L-sorbose oxidase',\n",
       "  'reaction': 'L-sorbose + O2 = 5-dehydro-D-fructose + H2O2',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'L-sorbose:oxygen 5-oxidoreductase',\n",
       "  'comments': 'Also acts on D-glucose, D-galactose and D-xylose, but not on D-fructose. 2,6-Dichloroindophenol can act as acceptor.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.12': {'ec_num': '1.1.3.12',\n",
       "  'accepted_name': 'pyridoxine 4-oxidase',\n",
       "  'reaction': 'pyridoxine + O2 = pyridoxal + H2O2',\n",
       "  'other_names': 'pyridoxin 4-oxidase; pyridoxol 4-oxidase',\n",
       "  'sys_name': 'pyridoxine:oxygen 4-oxidoreductase',\n",
       "  'comments': 'A flavoprotein. Can also use 2,6-dichloroindophenol as an acceptor.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.3.12\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.3.12\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.3.12\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.3.12\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.1.3.12%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.13': {'ec_num': '1.1.3.13',\n",
       "  'accepted_name': 'alcohol oxidase',\n",
       "  'reaction': 'a primary alcohol + O2 = an aldehyde + H2O2',\n",
       "  'other_names': 'ethanol oxidase; alcohol:oxygen oxidoreductase',\n",
       "  'sys_name': 'alcohol:oxygen oxidoreductase (H2O2-forming)',\n",
       "  'comments': 'The enzymes from the fungi Candida methanosorbosa and several Basidiomycetes species contain an FAD cofactor [1,3]. The enzyme from the phytopathogenic fungi Colletotrichum graminicola and Colletotrichum gloeosporioides utilize a mononuclear copper-radical mechanism [4]. The enzyme acts on primary alcohols and unsaturated alcohols, and has much lower activity with branched-chain and secondary alcohols.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.14': {'ec_num': '1.1.3.14',\n",
       "  'accepted_name': 'catechol oxidase (dimerizing)',\n",
       "  'reaction': '4 catechol + 3 O2 = 2 dibenzo[1,4]dioxin-2,3-dione + 6 H2O',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'catechol:oxygen oxidoreductase (dimerizing)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.15': {'ec_num': '1.1.3.15',\n",
       "  'accepted_name': '(S)-2-hydroxy-acid oxidase',\n",
       "  'reaction': 'an (S)-2-hydroxy carboxylate + O2 = a 2-oxo carboxylate + H2O2',\n",
       "  'other_names': 'hydroxy-acid oxidase A; hydroxy-acid oxidase B; glycolate oxidase; L-2-hydroxy acid oxidase; hydroxyacid oxidase A; L-α-hydroxy acid oxidase',\n",
       "  'sys_name': '(S)-2-hydroxy carboxylate:oxygen 2-oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FMN). Exists as two major isoenzymes; the A form preferentially oxidizes short-chain aliphatic hydroxy acids, and was previously listed as EC 1.1.3.1, glycolate oxidase; the B form preferentially oxidizes long-chain and aromatic hydroxy acids. The rat isoenzyme B also acts as EC 1.4.3.2, L-amino-acid oxidase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.16': {'ec_num': '1.1.3.16',\n",
       "  'accepted_name': 'ecdysone oxidase',\n",
       "  'reaction': 'ecdysone + O2 = 3-dehydroecdysone + H2O2',\n",
       "  'other_names': 'β-ecdysone oxidase',\n",
       "  'sys_name': 'ecdysone:oxygen 3-oxidoreductase',\n",
       "  'comments': '2,6-Dichloroindophenol can act as an acceptor.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.17': {'ec_num': '1.1.3.17',\n",
       "  'accepted_name': 'choline oxidase',\n",
       "  'reaction': 'choline + 2 O2 + H2O = betaine + 2 H2O2 (overall reaction);;(1a) choline + O2 = betaine aldehyde + H2O2;;(1b) betaine aldehyde + O2 + H2O = betaine + H2O2',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'choline:oxygen 1-oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD). In many bacteria, plants and animals, the osmoprotectant betaine is synthesized using different enzymes to catalyse the conversion of (1) choline into betaine aldehyde and (2) betaine aldehyde into betaine. In plants, the first reaction is catalysed by EC 1.14.15.7, choline monooxygenase, whereas in animals and many bacteria, it is catalysed by either membrane-bound choline dehydrogenase (EC 1.1.99.1) or soluble choline oxidase (EC 1.1.3.17) [6]. The enzyme involved in the second step, EC 1.2.1.8, betaine-aldehyde dehydrogenase, appears to be the same in those plants, animals and bacteria that use two separate enzymes.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'choline = (2-hydroxyethyl)trimethylammonium\\nbetaine aldehyde = N,N,N-trimethyl-2-oxoethylammonium\\nbetaine = glycine betaine = N,N,N-trimethylglycine = N,N,N-trimethylammonioacetate',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.18': {'ec_num': '1.1.3.18',\n",
       "  'accepted_name': 'secondary-alcohol oxidase',\n",
       "  'reaction': 'a secondary alcohol + O2 = a ketone + H2O2',\n",
       "  'other_names': 'polyvinyl alcohol oxidase; secondary alcohol oxidase',\n",
       "  'sys_name': 'secondary-alcohol:oxygen oxidoreductase',\n",
       "  'comments': 'Acts on secondary alcohols with five or more carbons, and polyvinyl alcohols with molecular mass over 300 Da. The Pseudomonas enzyme contains one atom of non-heme iron per molecule.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.19': {'ec_num': '1.1.3.19',\n",
       "  'accepted_name': '4-hydroxymandelate oxidase (decarboxylating)',\n",
       "  'reaction': '(S)-4-hydroxymandelate + O2 = 4-hydroxybenzaldehyde + CO2 + H2O2',\n",
       "  'other_names': 'L-4-hydroxymandelate oxidase (decarboxylating); (S)-2-hydroxy-2-(4-hydroxyphenyl)acetate:oxygen 1-oxidoreductase; (S)-4-hydroxymandelate:oxygen 1-oxidoreductase; 4-hydroxymandelate oxidase',\n",
       "  'sys_name': '(S)-4-hydroxymandelate:oxygen 1-oxidoreductase (decarboxylating)',\n",
       "  'comments': 'A flavoprotein (FAD), requires Mn2+. The enzyme from the bacterium Pseudomonas putida is involved in the degradation of mandelate. ',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '(S)-4-hydroxymandelate = (S)-2-hydroxy-2-(4-hydroxyphenyl)acetate',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.20': {'ec_num': '1.1.3.20',\n",
       "  'accepted_name': 'long-chain-alcohol oxidase',\n",
       "  'reaction': 'a long-chain alcohol + O2 = a long-chain aldehyde + H2O2',\n",
       "  'other_names': 'long-chain fatty alcohol oxidase; fatty alcohol oxidase; fatty alcohol:oxygen oxidoreductase; long-chain fatty acid oxidase',\n",
       "  'sys_name': 'long-chain-alcohol:oxygen oxidoreductase',\n",
       "  'comments': 'Oxidizes long-chain fatty alcohols; best substrate is dodecyl alcohol.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.21': {'ec_num': '1.1.3.21',\n",
       "  'accepted_name': 'glycerol-3-phosphate oxidase',\n",
       "  'reaction': 'sn-glycerol 3-phosphate + O2 = glycerone phosphate + H2O2',\n",
       "  'other_names': 'glycerol phosphate oxidase; glycerol-1-phosphate oxidase; glycerol phosphate oxidase; L-α-glycerophosphate oxidase; α-glycerophosphate oxidase; L-α-glycerol-3-phosphate oxidase',\n",
       "  'sys_name': 'sn-glycerol-3-phosphate:oxygen 2-oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'glycerone phosphate = dihydroxyacetone phosphate = 3-hydroxy-2-oxopropyl phosphate',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.23': {'ec_num': '1.1.3.23',\n",
       "  'accepted_name': 'thiamine oxidase',\n",
       "  'reaction': 'thiamine + 2 O2 + H2O = thiamine acetic acid + 2 H2O2',\n",
       "  'other_names': 'thiamin dehydrogenase; thiamine dehydrogenase; thiamin:oxygen 5-oxidoreductase',\n",
       "  'sys_name': 'thiamine:oxygen 5-oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD). The product differs from thiamine in replacement of -CH2.CH2.OH by -CH2.COOH; the two-step oxidation proceeds without the release of the intermediate aldehyde from the enzyme.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.24': {'ec_num': '1.1.3.24',\n",
       "  'accepted_name': 'L-galactonolactone oxidase',\n",
       "  'reaction': 'L-galactono-1,4-lactone + O2 = L-ascorbate + H2O2',\n",
       "  'other_names': 'L-galactono-1,4-lactone oxidase',\n",
       "  'sys_name': 'L-galactono-1,4-lactone:oxygen 3-oxidoreductase',\n",
       "  'comments': 'A flavoprotein. Acts on the 1,4-lactones of L-galactonic, D-altronic, L-fuconic, D-arabinic and D-threonic acids; not identical with EC 1.1.3.8 L-gulonolactone oxidase. (cf. EC 1.3.2.3 galactonolactone dehydrogenase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.27': {'ec_num': '1.1.3.27',\n",
       "  'accepted_name': 'hydroxyphytanate oxidase',\n",
       "  'reaction': 'L-2-hydroxyphytanate + O2 = 2-oxophytanate + H2O2',\n",
       "  'other_names': 'L-2-hydroxyphytanate:oxygen 2-oxidoreductase',\n",
       "  'sys_name': 'L-2-hydroxyphytanate:oxygen 2-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.28': {'ec_num': '1.1.3.28',\n",
       "  'accepted_name': 'nucleoside oxidase',\n",
       "  'reaction': 'inosine + O2 = 9-riburonosylhypoxanthine + H2O;; (1a) 2 inosine + O2 = 2 5′-dehydroinosine + 2 H2O;;(1b) 2 5′-dehydroinosine + O2 = 2 9-riburonosylhypoxanthine',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'nucleoside:oxygen 5′-oxidoreductase',\n",
       "  'comments': 'Other purine and pyrimidine nucleosides (as well as 2′-deoxyribonucleosides) are substrates, but ribose and nucleotides are not substrates. The overall reaction takes place in two separate steps, with the 5′-dehydro nucleoside being released from the enzyme to serve as substrate for the second reaction. This enzyme differs from EC 1.1.3.39, nucleoside oxidase (H2O2-forming), as it produces water rather than hydrogen peroxide.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.29': {'ec_num': '1.1.3.29',\n",
       "  'accepted_name': 'N-acylhexosamine oxidase',\n",
       "  'reaction': '(1) N-acetyl-D-glucosamine + O2 + H2O = N-acetyl-D-glucosaminate + H2O2 (overall reaction);;(1a) N-acetyl-D-glucosamine + O2 = N-acetyl-D-glucosamino-1,5-lactone + H2O2;;(1b) N-acetyl-D-glucosamino-1,5-lactone + H2O = N-acetyl-D-glucosaminate (spontaneous);;(2) N-acetyl-D-galactosamine + O2 + H2O = N-acetyl-D-galacotsaminate + H2O2 (overall reaction);;(2a) N-acetyl-D-galactosamine + O2 = N-acetyl-D-galactosamino-1,5-lactone + H2O2;;(2b) N-acetyl-D-galactosamino-1,5-lactone + H2O = N-acetyl-D-galactosaminate (spontaneous)',\n",
       "  'other_names': 'N-acyl-D-hexosamine oxidase; N-acyl-β-D-hexosamine:oxygen 1-oxidoreductase',\n",
       "  'sys_name': 'N-acyl-D-hexosamine:oxygen 1-oxidoreductase',\n",
       "  'comments': 'The enzyme, found in bacteria, also acts more slowly on N-acetyl-D-mannosamine.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.30': {'ec_num': '1.1.3.30',\n",
       "  'accepted_name': 'polyvinyl-alcohol oxidase',\n",
       "  'reaction': 'polyvinyl alcohol + O2 = oxidized polyvinyl alcohol + H2O2',\n",
       "  'other_names': 'dehydrogenase, polyvinyl alcohol; PVA oxidase',\n",
       "  'sys_name': 'polyvinyl-alcohol:oxygen oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.37': {'ec_num': '1.1.3.37',\n",
       "  'accepted_name': 'D-arabinono-1,4-lactone oxidase',\n",
       "  'reaction': 'D-arabinono-1,4-lactone + O2 = dehydro-D-arabinono-1,4-lactone + H2O2',\n",
       "  'other_names': 'D-arabinono-γ-lactone oxidase; ALO',\n",
       "  'sys_name': 'D-arabinono-1,4-lactone:oxygen oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD). L-Galactono-1,4-lactone, L-gulono-1,4-lactone and L-xylono-1,4-lactone can also act as substrates but D-glucono-1,5-lactone, L-arabinono-1,4-lactone, D-galactono-1,4-lactone and D-gulono-1,4-lactone cannot [1]. With L-galactono-1,4-lactone as substrate, the product is L-ascorbate [3]. The product dehydro-D-arabinono-1,4-lactone had previously been referred to erroneously as D-erythroascorbate (CAS no.: 5776-48-7; formula: C6H8O6), although it was referred to as a five-carbon compound [1].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'dehydro-D-arabinono-1,4-lactone = (5R)-3,4-dihydroxy-5-(hydroxymethyl)furan-2(5H)-one',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.38': {'ec_num': '1.1.3.38',\n",
       "  'accepted_name': 'vanillyl-alcohol oxidase',\n",
       "  'reaction': 'vanillyl alcohol + O2 = vanillin + H2O2',\n",
       "  'other_names': '4-hydroxy-2-methoxybenzyl alcohol oxidase',\n",
       "  'sys_name': 'vanillyl alcohol:oxygen oxidoreductase',\n",
       "  'comments': 'Vanillyl-alcohol oxidase from Penicillium simplicissimum contains covalently bound FAD. It converts a wide range of 4-hydroxybenzyl alcohols and 4-hydroxybenzylamines into the corresponding aldehydes. The allyl group of 4-allylphenols is also converted into the -CH=CH-CH2OH group.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.3.38\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.3.38\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.3.38\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.3.38\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.1.3.38%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.39': {'ec_num': '1.1.3.39',\n",
       "  'accepted_name': 'nucleoside oxidase (H2O2-forming)',\n",
       "  'reaction': 'adenosine + 2 O2 + H2O = 9-riburonosyladenine + 2 H2O2 (overall reaction);;(1a) adenosine + O2 = 5′-dehydroadenosine + H2O2;;(1b) 5′-dehydroadenosine + O2 + H2O = 9-riburonosyladenine + H2O2',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'nucleoside:oxygen 5′-oxidoreductase (H2O2-forming)',\n",
       "  'comments': 'A heme-containing flavoprotein (FAD). Other purine and pyrimidine nucleosides (as well as 2′-deoxyribonucleosides and arabinosides) are substrates, but ribose and nucleotides are not substrates. The overall reaction takes place in two separate steps, with the 5′-dehydro nucleoside being released from the enzyme to serve as substrate for the second reaction. This enzyme differs from EC 1.1.3.28, nucleoside oxidase, as it produces hydrogen peroxide rather than water.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.40': {'ec_num': '1.1.3.40',\n",
       "  'accepted_name': 'D-mannitol oxidase',\n",
       "  'reaction': 'D-mannitol + O2 = D-mannose + H2O2',\n",
       "  'other_names': 'mannitol oxidase; D-arabitol oxidase',\n",
       "  'sys_name': 'mannitol:oxygen oxidoreductase (cyclizing)',\n",
       "  'comments': 'Also catalyses the oxidation of D-arabinitol and, to a lesser extent, D-glucitol (sorbitol), whereas L-arabinitol is not a good substrate. The enzyme from the snails Helix aspersa and Arion ater is found in a specialised tubular organelle that has been termed the mannosome.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.3.41': {'ec_num': '1.1.3.41',\n",
       "  'accepted_name': 'alditol oxidase',\n",
       "  'reaction': 'an alditol + O2 = an aldose + H2O2',\n",
       "  'other_names': 'xylitol oxidase; xylitol:oxygen oxidoreductase; AldO',\n",
       "  'sys_name': 'alditol:oxygen oxidoreductase',\n",
       "  'comments': 'The enzyme from Streptomyces sp. IKD472 and from Streptomyces coelicolor is a monomeric oxidase containing one molecule of FAD per molecule of protein [1,2]. While xylitol (five carbons) and sorbitol (6 carbons) are the preferred substrates, other alditols, including L-threitol (four carbons), D-arabinitol (five carbons), D-galactitol (six carbons) and D-mannitol (six carbons) can also act as substrates, but more slowly [1,2]. Belongs in the vanillyl-alcohol-oxidase family of enzymes [2].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.4.1': {'ec_num': '1.1.4.1',\n",
       "  'accepted_name': 'vitamin-K-epoxide reductase (warfarin-sensitive)',\n",
       "  'reaction': '2-methyl-3-phytyl-1,4-naphthoquinone + oxidized dithiothreitol = 2,3-epoxy-2,3-dihydro-2-methyl-3-phytyl-1,4-naphthoquinone + 1,4-dithiothreitol',\n",
       "  'other_names': '',\n",
       "  'sys_name': '2-methyl-3-phytyl-1,4-naphthoquinone:oxidized-dithiothreitol oxidoreductase',\n",
       "  'comments': 'In the reverse reaction, vitamin K 2,3-epoxide is reduced to vitamin K and possibly to vitamin K hydroquinone by 1,4-dithiothreitol, which is oxidized to a disulfide; some other dithiols and 4-butanethiol can also act. Inhibited strongly by warfarin [cf. EC 1.1.4.2 vitamin-K-epoxide reductase (warfarin-insensitive)].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.4.2': {'ec_num': '1.1.4.2',\n",
       "  'accepted_name': 'vitamin-K-epoxide reductase (warfarin-insensitive)',\n",
       "  'reaction': '3-hydroxy-2-methyl-3-phytyl-2,3-dihydronaphthoquinone + oxidized dithiothreitol = 2,3-epoxy-2,3-dihydro-2-methyl-3-phytyl-1,4-naphthoquinone + 1,4-dithiothreitol',\n",
       "  'other_names': '',\n",
       "  'sys_name': '3-hydroxy-2-methyl-3-phytyl-2,3-dihydronaphthoquinone:oxidized-dithiothreitol oxidoreductase',\n",
       "  'comments': 'In the reverse reaction, vitamin K 2,3-epoxide is reduced to 3-hydroxy- (and 2-hydroxy-) vitamin K by 1,4-dithiothreitol, which is oxidized to a disulfide. Not inhibited by warfarin [cf. EC 1.1.4.1 vitamin-K-epoxide reductase (warfarin-sensitive)].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.5.2': {'ec_num': '1.1.5.2',\n",
       "  'accepted_name': 'glucose 1-dehydrogenase (PQQ, quinone)',\n",
       "  'reaction': 'D-glucose + ubiquinone = D-glucono-1,5-lactone + ubiquinol',\n",
       "  'other_names': 'quinoprotein glucose dehydrogenase; membrane-bound glucose dehydrogenase; mGDH; glucose dehydrogenase (PQQ-dependent); glucose dehydrogenase (pyrroloquinoline-quinone); quinoprotein D-glucose dehydrogenase ',\n",
       "  'sys_name': 'D-glucose:ubiquinone oxidoreductase',\n",
       "  'comments': 'Integral membrane protein containing PQQ as a cofactor. It also contains bound ubiquinone and Mg2+ or Ca2+. Electron acceptor is membrane ubiquinone but usually assayed with phenazine methosulfate. Like in all other quinoprotein alcohol dehydrogenases the catalytic domain has an 8-bladed propeller structure. It occurs in a wide range of bacteria. Catalyses a direct oxidation of the pyranose form of D-glucose to the lactone and thence to D-gluconate in the periplasm. Oxidizes other monosaccharides including the pyranose forms of pentoses.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-05-19 09:09:04'},\n",
       " '1.1.99.1': {'ec_num': '1.1.99.1',\n",
       "  'accepted_name': 'choline dehydrogenase',\n",
       "  'reaction': 'choline + acceptor = betaine aldehyde + reduced acceptor',\n",
       "  'other_names': 'choline oxidase; choline-cytochrome c reductase; choline:(acceptor) oxidoreductase; choline:(acceptor) 1-oxidoreductase',\n",
       "  'sys_name': 'choline:acceptor 1-oxidoreductase',\n",
       "  'comments': 'A quinoprotein. In many bacteria, plants and animals, the osmoprotectant betaine is synthesized using different enzymes to catalyse the conversion of (1) choline into betaine aldehyde and (2) betaine aldehyde into betaine. In plants, the first reaction is catalysed by EC 1.14.15.7, choline monooxygenase, whereas in animals and many bacteria, it is catalysed by either membrane-bound choline dehydrogenase (EC 1.1.99.1) or soluble choline oxidase (EC 1.1.3.17) [4]. The enzyme involved in the second step, EC 1.2.1.8, betaine-aldehyde dehydrogenase, appears to be the same in plants, animals and bacteria.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'betaine aldehyde = N,N,N-trimethyl-2-oxoethylammonium\\ncholine = (2-hydroxyethyl)trimethylammonium',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.2': {'ec_num': '1.1.99.2',\n",
       "  'accepted_name': 'L-2-hydroxyglutarate dehydrogenase',\n",
       "  'reaction': '(S)-2-hydroxyglutarate + acceptor = 2-oxoglutarate + reduced acceptor',\n",
       "  'other_names': 'α-ketoglutarate reductase; α-hydroxyglutarate dehydrogenase; L-α-hydroxyglutarate dehydrogenase; hydroxyglutaric dehydrogenase; α-hydroxyglutarate oxidoreductase; L-α-hydroxyglutarate:NAD+ 2-oxidoreductase; α-hydroxyglutarate dehydrogenase (NAD+ specific); (S)-2-hydroxyglutarate:(acceptor) 2-oxidoreductase',\n",
       "  'sys_name': '(S)-2-hydroxyglutarate:acceptor 2-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.3': {'ec_num': '1.1.99.3',\n",
       "  'accepted_name': 'gluconate 2-dehydrogenase (acceptor)',\n",
       "  'reaction': 'D-gluconate + acceptor = 2-dehydro-D-gluconate + reduced acceptor',\n",
       "  'other_names': 'gluconate oxidase; gluconate dehydrogenase; gluconic dehydrogenase; D-gluconate dehydrogenase; gluconic acid dehydrogenase; 2-ketogluconate reductase; D-gluconate dehydrogenase, 2-keto-D-gluconate-yielding; D-gluconate:(acceptor) 2-oxidoreductase',\n",
       "  'sys_name': 'D-gluconate:acceptor 2-oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.4': {'ec_num': '1.1.99.4',\n",
       "  'accepted_name': 'dehydrogluconate dehydrogenase',\n",
       "  'reaction': '2-dehydro-D-gluconate + acceptor = 2,5-didehydro-D-gluconate + reduced acceptor',\n",
       "  'other_names': 'ketogluconate dehydrogenase; α-ketogluconate dehydrogenase; 2-keto-D-gluconate dehydrogenase; 2-oxogluconate dehydrogenase',\n",
       "  'sys_name': '2-dehydro-D-gluconate:acceptor 2-oxidoreductase',\n",
       "  'comments': 'A flavoprotein.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.5': {'ec_num': '1.1.99.5',\n",
       "  'accepted_name': 'glycerol-3-phosphate dehydrogenase',\n",
       "  'reaction': 'sn-glycerol 3-phosphate + acceptor = glycerone phosphate + reduced acceptor',\n",
       "  'other_names': 'sn-glycerol-3-phosphate dehydrogenase; L-glycerol-3-phosphate dehydrogenase; α-glycerophosphate dehydrogenase; L-3-glycerophosphate-ubiquinone oxidoreductase; sn-glycerol 3-phosphate oxidase; dehydrogenase, glycerol phosphate (acceptor); glycerophosphate dehydrogenase; glycerol phosphate dehydrogenase; NAD+-independent glycerol phosphate dehydrogenase; glycerol 3-phosphate cytochrome c reductase; flavoprotein-linked L-glycerol 3-phosphate dehydrogenase; α-glycerophosphate dehydrogenase (acceptor); FAD-dependent glycerol-3-phosphate dehydrogenase; FAD-linked glycerol 3-phosphate dehydrogenase; glycerol phosphate dehydrogenase (FAD); glycerol phosphate dehydrogenase (acceptor); FAD-dependent sn-glycerol-3-phosphate dehydrogenase; L-glycerophosphate dehydrogenase; glycerol-3-phosphate dehydrogenase (flavin-linked); flavin-linked glycerol-3-phosphate dehydrogenase; pyridine nucleotide-independent L-glycerol 3-phosphate dehydrogenase; DL-glycerol 3-phosphate oxidase; FAD-linked L-glycerol-3-phosphate dehydrogenase; sn-glycerol-3-phosphate:(acceptor) 2-oxidoreductase',\n",
       "  'sys_name': 'sn-glycerol-3-phosphate:acceptor 2-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.6': {'ec_num': '1.1.99.6',\n",
       "  'accepted_name': 'D-lactate dehydrogenase (acceptor)',\n",
       "  'reaction': '(R)-lactate + acceptor = pyruvate + reduced acceptor',\n",
       "  'other_names': 'D-2-hydroxy acid dehydrogenase; D-2-hydroxy-acid dehydrogenase; (R)-2-hydroxy-acid:acceptor 2-oxidoreductase',\n",
       "  'sys_name': '(R)-lactate:acceptor 2-oxidoreductase',\n",
       "  'comments': 'The zinc flavoprotein (FAD) from the archaeon Archaeoglobus fulgidus cannot utilize NAD+, cytochrome c, methylene blue or dimethylnaphthoquinone as acceptors. In vitro it is active with artificial electron acceptors such as 2,6-dichlorophenolindophenol, but the physiological acceptor is not yet known.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.7': {'ec_num': '1.1.99.7',\n",
       "  'accepted_name': 'lactate—malate transhydrogenase',\n",
       "  'reaction': '(S)-lactate + oxaloacetate = pyruvate + malate',\n",
       "  'other_names': 'malate-lactate transhydrogenase',\n",
       "  'sys_name': '(S)-lactate:oxaloacetate oxidoreductase',\n",
       "  'comments': 'Catalyses hydrogen transfer from C3 or C4 (S)-2-hydroxy acids to 2-oxo acids. It contains tightly bound nicotinamide nucleotide in its active centre. This cofactor cannot be removed without denaturation of the protein.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-08-04 17:12:20'},\n",
       " '1.1.99.8': {'ec_num': '1.1.99.8',\n",
       "  'accepted_name': 'alcohol dehydrogenase (acceptor)',\n",
       "  'reaction': 'a primary alcohol + acceptor = an aldehyde + reduced acceptor',\n",
       "  'other_names': 'primary alcohol dehydrogenase; MDH; quinohemoprotein alcohol dehydrogenase; quinoprotein alcohol dehydrogenase; quinoprotein ethanol dehydrogenase; alcohol:(acceptor) oxidoreductase',\n",
       "  'sys_name': 'alcohol:acceptor oxidoreductase',\n",
       "  'comments': 'A quinoprotein. Acts on a wide range of primary alcohols, including methanol [cf. EC 1.1.99.20 alkan-1-ol dehydrogenase (acceptor)].',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.99.8\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.99.8\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.99.8\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.99.8\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.1.99.8%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.9': {'ec_num': '1.1.99.9',\n",
       "  'accepted_name': 'pyridoxine 5-dehydrogenase',\n",
       "  'reaction': 'pyridoxine + acceptor = isopyridoxal + reduced acceptor',\n",
       "  'other_names': 'pyridoxal-5-dehydrogenase; pyridoxol 5-dehydrogenase; pyridoxin 5-dehydrogenase; pyridoxine dehydrogenase; pyridoxine 5′-dehydrogenase; pyridoxine:(acceptor) 5-oxidoreductase',\n",
       "  'sys_name': 'pyridoxine:acceptor 5-oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.99.9\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.1.99.9\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.1.99.9\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.1.99.9\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.10': {'ec_num': '1.1.99.10',\n",
       "  'accepted_name': 'glucose dehydrogenase (acceptor)',\n",
       "  'reaction': 'D-glucose + acceptor = D-glucono-1,5-lactone + reduced acceptor',\n",
       "  'other_names': 'glucose dehydrogenase (Aspergillus); glucose dehydrogenase (decarboxylating); D-glucose:(acceptor) 1-oxidoreductase',\n",
       "  'sys_name': 'D-glucose:acceptor 1-oxidoreductase',\n",
       "  'comments': 'A glycoprotein containing one mole of FAD per mole of enzyme. 2,6-Dichloroindophenol can act as acceptor',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.11': {'ec_num': '1.1.99.11',\n",
       "  'accepted_name': 'fructose 5-dehydrogenase',\n",
       "  'reaction': 'D-fructose + acceptor = 5-dehydro-D-fructose + reduced acceptor',\n",
       "  'other_names': 'fructose 5-dehydrogenase (acceptor); D-fructose dehydrogenase; D-fructose:(acceptor) 5-oxidoreductase',\n",
       "  'sys_name': 'D-fructose:acceptor 5-oxidoreductase',\n",
       "  'comments': '2,6-Dichloroindophenol can act as acceptor.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.12': {'ec_num': '1.1.99.12',\n",
       "  'accepted_name': 'sorbose dehydrogenase',\n",
       "  'reaction': 'L-sorbose + acceptor = 5-dehydro-D-fructose + reduced acceptor',\n",
       "  'other_names': 'L-sorbose:(acceptor) 5-oxidoreductase',\n",
       "  'sys_name': 'L-sorbose:acceptor 5-oxidoreductase',\n",
       "  'comments': '2,6-Dichloroindophenol can act as acceptor.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.13': {'ec_num': '1.1.99.13',\n",
       "  'accepted_name': 'glucoside 3-dehydrogenase (acceptor)',\n",
       "  'reaction': 'sucrose + acceptor = 3-dehydro-α-D-glucosyl-β-D-fructofuranoside + reduced acceptor',\n",
       "  'other_names': 'D-glucoside 3-dehydrogenase (ambiguous); D-aldohexopyranoside dehydrogenase (ambiguous); D-aldohexoside:(acceptor) 3-oxidoreductase; thuA (gene name); thuB (gene name); glucoside 3-dehydrogenase',\n",
       "  'sys_name': 'D-aldohexoside:acceptor 3-oxidoreductase',\n",
       "  'comments': 'The enzymes from members of the Rhizobiaceae family (such as Agrobacterium tumefaciens) act on disaccharides that contain a glucose moiety at the non-reducing end, such as sucrose, trehalose, leucrose, palatinose, trehalulose, and maltitol, forming the respective 3′-keto derivatives. cf. EC 1.1.2.11, glucoside 3-dehydrogenase (cytochrome c).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.14': {'ec_num': '1.1.99.14',\n",
       "  'accepted_name': 'glycolate dehydrogenase',\n",
       "  'reaction': 'glycolate + acceptor = glyoxylate + reduced acceptor',\n",
       "  'other_names': 'glycolate oxidoreductase; glycolic acid dehydrogenase; glycolate:(acceptor) 2-oxidoreductase',\n",
       "  'sys_name': 'glycolate:acceptor 2-oxidoreductase',\n",
       "  'comments': 'Also acts on (R)-lactate. 2,6-Dichloroindophenol and phenazine methosulfate can act as acceptors.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.16': {'ec_num': '1.1.99.16',\n",
       "  'accepted_name': 'malate dehydrogenase (acceptor)',\n",
       "  'reaction': '(S)-malate + acceptor = oxaloacetate + reduced acceptor',\n",
       "  'other_names': 'FAD-dependent malate-vitamin K reductase; malate-vitamin K reductase; (S)-malate:(acceptor) oxidoreductase',\n",
       "  'sys_name': '(S)-malate:acceptor oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.18': {'ec_num': '1.1.99.18',\n",
       "  'accepted_name': 'cellobiose dehydrogenase (acceptor)',\n",
       "  'reaction': 'cellobiose + acceptor = cellobiono-1,5-lactone + reduced acceptor',\n",
       "  'other_names': 'cellobiose dehydrogenase; cellobiose oxidoreductase; Phanerochaete chrysosporium cellobiose oxidoreductase; CBOR; cellobiose oxidase; cellobiose:oxygen 1-oxidoreductase; CDH; cellobiose:(acceptor) 1-oxidoreductase',\n",
       "  'sys_name': 'cellobiose:acceptor 1-oxidoreductase',\n",
       "  'comments': 'Also acts, more slowly, on cello-oligosaccharides, lactose and D-glucosyl-1,4-β-D-mannose. The enzyme from the white rot fungus Phanerochaete chrysosporium is unusual in having two redoxin domains, one containing a flavin and the other a protoheme group. It transfers reducing equivalents from cellobiose to two types of redox acceptor: two-electron oxidants, including redox dyes, benzoquinones, and molecular oxygen, and one-electron oxidants, including semiquinone species, iron(II) complexes, and the model acceptor cytochrome c [9]. 2,6-Dichloroindophenol can act as acceptor in vitro.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.19': {'ec_num': '1.1.99.19',\n",
       "  'accepted_name': 'uracil dehydrogenase',\n",
       "  'reaction': 'uracil + acceptor = barbiturate + reduced acceptor',\n",
       "  'other_names': 'uracil oxidase',\n",
       "  'sys_name': 'uracil:(acceptor) oxidoreductase',\n",
       "  'comments': 'Also oxidizes thymine. The enzyme acts on the hydrated derivative of the substrate. Formerly EC 1.2.99.1.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.20': {'ec_num': '1.1.99.20',\n",
       "  'accepted_name': 'alkan-1-ol dehydrogenase (acceptor)',\n",
       "  'reaction': 'primary alcohol + acceptor = aldehyde + reduced acceptor',\n",
       "  'other_names': 'polyethylene glycol dehydrogenase; alkan-1-ol:(acceptor) oxidoreductase',\n",
       "  'sys_name': 'alkan-1-ol:acceptor oxidoreductase',\n",
       "  'comments': 'A quinoprotein. Acts on C3-C16 linear-chain saturated primary alcohols, C4-C7 aldehydes and on non-ionic surfactants containing polyethylene glycol residues, such as Tween 40 and 60, but not on methanol and only very slowly on ethanol. 2,6-Dichloroindophenol can act as acceptor. cf. EC 1.1.99.8 alcohol dehydrogenase (acceptor).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.21': {'ec_num': '1.1.99.21',\n",
       "  'accepted_name': 'D-sorbitol dehydrogenase (acceptor)',\n",
       "  'reaction': 'D-sorbitol + acceptor = L-sorbose + reduced acceptor',\n",
       "  'other_names': 'D-sorbitol:(acceptor) 1-oxidoreductase',\n",
       "  'sys_name': 'D-sorbitol:acceptor 1-oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.22': {'ec_num': '1.1.99.22',\n",
       "  'accepted_name': 'glycerol dehydrogenase (acceptor)',\n",
       "  'reaction': 'glycerol + acceptor = glycerone + reduced acceptor',\n",
       "  'other_names': 'glycerol:(acceptor) 1-oxidoreductase',\n",
       "  'sys_name': 'glycerol:acceptor 1-oxidoreductase',\n",
       "  'comments': 'A quinoprotein. Also acts, more slowly, on a number of other polyols including D-sorbitol, D-arabinitol, meso-erythritol, ribitol and propane-1,2-diol.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.23': {'ec_num': '1.1.99.23',\n",
       "  'accepted_name': 'polyvinyl-alcohol dehydrogenase (acceptor)',\n",
       "  'reaction': 'polyvinyl alcohol + acceptor = oxidized polyvinyl alcohol + reduced acceptor',\n",
       "  'other_names': 'PVA dehydrogenase; polyvinyl-alcohol:(acceptor) oxidoreductase',\n",
       "  'sys_name': 'polyvinyl-alcohol:acceptor oxidoreductase',\n",
       "  'comments': 'A quinoprotein. Phenazine methosulfate and 2,6-dichloroindophenol can act as acceptors. Also acts, more slowly, on 2-hexanol and some other secondary alcohols. cf. EC 1.1.99.8 alcohol dehydrogenase (acceptor) and EC 1.1.99.20 alkan-1-ol dehydrogenase (acceptor).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.24': {'ec_num': '1.1.99.24',\n",
       "  'accepted_name': 'hydroxyacid-oxoacid transhydrogenase',\n",
       "  'reaction': '(S)-3-hydroxybutanoate + 2-oxoglutarate = acetoacetate + (R)-2-hydroxyglutarate',\n",
       "  'other_names': 'transhydrogenase, hydroxy acid-oxo acid',\n",
       "  'sys_name': '(S)-3-hydroxybutanoate:2-oxoglutarate oxidoreductase',\n",
       "  'comments': '4-Hydroxybutanoate and (R)-2-hydroxyglutarate can also act as donors; 4-oxobutanoate can also act as acceptor.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.25': {'ec_num': '1.1.99.25',\n",
       "  'accepted_name': 'quinate dehydrogenase (pyrroloquinoline-quinone)',\n",
       "  'reaction': 'quinate + pyrroloquinoline-quinone = 3-dehydroquinate + reduced pyrroloquinoline-quinone',\n",
       "  'other_names': 'NAD(P)+-independent quinate dehydrogenase; quinate:pyrroloquinoline-quinone 5-oxidoreductase',\n",
       "  'sys_name': 'quinate:pyrroloquinoline-quinone 3-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '<a target=\"new\" href=\"glossary/quinic.html\">quinate</a> = (1<em>R</em>,3<em>R</em>,4<em>R</em>,5<em>R</em>)-1,3,4,5-tetrahydroxycyclohexanecarboxylic acid and is a cyclitol carboxylate <br>The numbering system used for the 3-dehydroquinate is that of the recommendations on cyclitols, <a target = \"new\" href = \"http://www.chem.qmul.ac.uk/iupac/cyclitol/I6t10.html#I8\">sections I-8 and I-9</a>: and is shown in the <a target = \"new\" href=\"reaction/misc/shikim.html\">reaction diagram</a>). The use of the term ’5-dehydroquinate’ for this compound is based on an earlier system of numbering.',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.26': {'ec_num': '1.1.99.26',\n",
       "  'accepted_name': '3-hydroxycyclohexanone dehydrogenase',\n",
       "  'reaction': '3-hydroxycyclohexanone + acceptor = cyclohexane-1,3-dione + reduced acceptor',\n",
       "  'other_names': '',\n",
       "  'sys_name': '3-hydroxycyclohexanone:acceptor 1-oxidoreductase',\n",
       "  'comments': '2,6-Dichloroindophenol and methylene blue can act as acceptors.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.27': {'ec_num': '1.1.99.27',\n",
       "  'accepted_name': '(R)-pantolactone dehydrogenase (flavin)',\n",
       "  'reaction': '(R)-pantolactone + acceptor = 2-dehydropantolactone + reduced acceptor',\n",
       "  'other_names': '2-dehydropantolactone reductase (flavin); 2-dehydropantoyl-lactone reductase (flavin); (R)-pantoyllactone dehydrogenase (flavin)',\n",
       "  'sys_name': '(R)-pantolactone:acceptor oxidoreductase (flavin-containing)',\n",
       "  'comments': 'High specificity for (R)-pantolactone. Phenazine methosulfate (PMS) can act as acceptor. The enzyme has been studied in the bacterium Nocardia asteroides and shown to be membrane-bound and induced by 1,2-propanediol. The FMN cofactor is non-covalently bound.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.28': {'ec_num': '1.1.99.28',\n",
       "  'accepted_name': 'glucose-fructose oxidoreductase',\n",
       "  'reaction': 'D-glucose + D-fructose = D-gluconolactone + D-glucitol',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'D-glucose:D-fructose oxidoreductase',\n",
       "  'comments': 'D-mannose, D-xylose, D-galactose, 2-deoxy-D-glucose and L-arabinose will function as aldose substrates, but with low affinities. The ketose substrate must be in the open-chain form. The apparent affinity for fructose is low, because little of the fructose substrate is in the open-chain form. Xylulose and glycerone (dihydroxyacetone) will replace fructose, but they are poor substrates. The enzyme from Zymomonas mobilis contains tightly bound NADP+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.29': {'ec_num': '1.1.99.29',\n",
       "  'accepted_name': 'pyranose dehydrogenase (acceptor)',\n",
       "  'reaction': '(1) a pyranose + acceptor = a pyranos-2-ulose (or a pyranos-3-ulose or a pyranos-2,3-diulose) + reduced acceptor;;(2) a pyranoside + acceptor = a pyranosid-3-ulose (or a pyranosid-3,4-diulose) + reduced acceptor',\n",
       "  'other_names': 'pyranose dehydrogenase; pyranose-quinone oxidoreductase; quinone-dependent pyranose dehydrogenase; PDH ',\n",
       "  'sys_name': 'pyranose:acceptor oxidoreductase',\n",
       "  'comments': 'Requires FAD. A number of aldoses and ketoses in pyranose form, as well as glycosides, gluco-oligosaccharides, sucrose and lactose can act as a donor. 1,4-Benzoquinone or ferricenium ion (ferrocene oxidized by removal of one electron) can serve as acceptor. Unlike EC 1.1.3.10, pyranose oxidase, this fungal enzyme does not interact with O2 and exhibits extremely broad substrate tolerance with variable regioselectivity (C-3, C-2 or C-3 + C-2 or C-3 + C-4) for (di)oxidation of different sugars. D-Glucose is exclusively or preferentially oxidized at C-3 (depending on the enzyme source), but can also be oxidized at C-2 + C-3. The enzyme also acts on 1→4-α- and 1→4-β-gluco-oligosaccharides, non-reducing gluco-oligosaccharides and L-arabinose, which are not substrates of EC 1.1.3.10. Sugars are oxidized in their pyranose but not in their furanose form.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'ferricenium ion = bis(η5-cyclopentadienyl)iron(1+) ',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.1.99.30': {'ec_num': '1.1.99.30',\n",
       "  'accepted_name': '2-oxo-acid reductase',\n",
       "  'reaction': 'a (2R)-hydroxy-carboxylate + acceptor = a 2-oxocarboxylate + reduced acceptor',\n",
       "  'other_names': '(2R)-hydroxycarboxylate-viologen-oxidoreductase; HVOR; 2-oxoacid reductase',\n",
       "  'sys_name': '(2R)-hydroxy-carboxylate:acceptor oxidoreductase',\n",
       "  'comments': 'Contains [4Fe-4S] and a mononucleotide molybdenum (pyranopterin) cofactor. Has broad substrate specificity, with 2-oxo-monocarboxylates and 2-oxo-dicarboxylates acting as substrates. Branching in a substrate at the C-3 position results in loss of activity. The enzyme from Proteus sp. is inactivated by oxygen.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.2': {'ec_num': '1.2.1.2',\n",
       "  'accepted_name': 'formate dehydrogenase',\n",
       "  'reaction': 'formate + NAD+ = CO2 + NADH',\n",
       "  'other_names': 'formate-NAD+ oxidoreductase; FDH I; FDH II; N-FDH; formic hydrogen-lyase; formate hydrogenlyase; hydrogenlyase; NAD+-linked formate dehydrogenase; NAD+-dependent formate dehydrogenase; formate dehydrogenase (NAD+); NAD+-formate dehydrogenase; formate benzyl-viologen oxidoreductase; formic acid dehydrogenase',\n",
       "  'sys_name': 'formate:NAD+ oxidoreductase',\n",
       "  'comments': 'The enzyme from most aerobic organisms is devoid of redox-active centres but that from the proteobacterium Methylosinus trichosporium contains iron-sulfur centres, flavin and a molybdenum centre [3]. Together with EC 1.12.1.2 hydrogen dehydrogenase, forms a system previously known as formate hydrogenlyase.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.1.2\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.1.2\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.1.2\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.1.2\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.2.1.2%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.3': {'ec_num': '1.2.1.3',\n",
       "  'accepted_name': 'aldehyde dehydrogenase (NAD+)',\n",
       "  'reaction': 'an aldehyde + NAD+ + H2O = a carboxylate + NADH + H+',\n",
       "  'other_names': 'CoA-independent aldehyde dehydrogenase; m-methylbenzaldehyde dehydrogenase; NAD-aldehyde dehydrogenase; NAD-dependent 4-hydroxynonenal dehydrogenase; NAD-dependent aldehyde dehydrogenase; NAD-linked aldehyde dehydrogenase; propionaldehyde dehydrogenase; aldehyde dehydrogenase (NAD)',\n",
       "  'sys_name': 'aldehyde:NAD+ oxidoreductase',\n",
       "  'comments': 'Wide specificity, including oxidation of D-glucuronolactone to D-glucarate.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.1.3\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.1.3\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.1.3\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.1.3\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.2.1.3%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.4': {'ec_num': '1.2.1.4',\n",
       "  'accepted_name': 'aldehyde dehydrogenase (NADP+)',\n",
       "  'reaction': 'an aldehyde + NADP+ + H2O = a carboxylate + NADPH + H+',\n",
       "  'other_names': 'NADP-acetaldehyde dehydrogenase; NADP-dependent aldehyde dehydrogenase; aldehyde dehydrogenase (NADP)',\n",
       "  'sys_name': 'aldehyde:NADP+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.5': {'ec_num': '1.2.1.5',\n",
       "  'accepted_name': 'aldehyde dehydrogenase [NAD(P)+]',\n",
       "  'reaction': 'an aldehyde + NAD(P)+ + H2O = a carboxylate + NAD(P)H + H+',\n",
       "  'other_names': 'ALDH',\n",
       "  'sys_name': 'aldehyde:NAD(P)+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.7': {'ec_num': '1.2.1.7',\n",
       "  'accepted_name': 'benzaldehyde dehydrogenase (NADP+)',\n",
       "  'reaction': 'benzaldehyde + NADP+ + H2O = benzoate + NADPH + 2 H+',\n",
       "  'other_names': 'NADP-linked benzaldehyde dehydrogenase; benzaldehyde dehydrogenase (NADP)',\n",
       "  'sys_name': 'benzaldehyde:NADP+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.1.7\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.1.7\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.1.7\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.1.7\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.2.1.7%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.8': {'ec_num': '1.2.1.8',\n",
       "  'accepted_name': 'betaine-aldehyde dehydrogenase',\n",
       "  'reaction': 'betaine aldehyde + NAD+ + H2O = betaine + NADH + 2 H+',\n",
       "  'other_names': 'betaine aldehyde oxidase; BADH; betaine aldehyde dehydrogenase; BetB',\n",
       "  'sys_name': 'betaine-aldehyde:NAD+ oxidoreductase',\n",
       "  'comments': 'In many bacteria, plants and animals, the osmoprotectant betaine is synthesized in two steps: (1) choline to betaine aldehyde and (2) betaine aldehyde to betaine. This enzyme is involved in the second step and appears to be the same in plants, animals and bacteria. In contrast, different enzymes are involved in the first reaction. In plants, this reaction is catalysed by EC 1.14.15.7 (choline monooxygenase), whereas in animals and many bacteria it is catalysed by either membrane-bound EC 1.1.99.1 (choline dehydrogenase) or soluble EC 1.1.3.17 (choline oxidase) [5]. In some bacteria, betaine is synthesized from glycine through the actions of EC 2.1.1.156 (glycine/sarcosine N-methyltransferase) and EC 2.1.1.157 (sarcosine/dimethylglycine N-methyltransferase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'betaine = glycine betaine = N,N,N-trimethylglycine = N,N,N-trimethylammonioacetate\\nbetaine aldehyde = N,N,N-trimethyl-2-oxoethylammonium',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.9': {'ec_num': '1.2.1.9',\n",
       "  'accepted_name': 'glyceraldehyde-3-phosphate dehydrogenase (NADP+)',\n",
       "  'reaction': 'D-glyceraldehyde 3-phosphate + NADP+ + H2O = 3-phospho-D-glycerate + NADPH + 2 H+',\n",
       "  'other_names': 'triosephosphate dehydrogenase (ambiguous); glyceraldehyde phosphate dehydrogenase (nicotinamide adenine dinucleotide phosphate); glyceraldehyde phosphate dehydrogenase (NADP+); glyceraldehyde 3-phosphate dehydrogenase (NADP+); NADP+-glyceraldehyde phosphate dehydrogenase; NADP+-glyceraldehyde-3-phosphate dehydrogenase; glyceraldehyde-3-phosphate:NADP+ reductase; nonphosphorylating glyceraldehyde-3-phosphate dehydrogenase',\n",
       "  'sys_name': 'D-glyceraldehyde-3-phosphate:NADP+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.10': {'ec_num': '1.2.1.10',\n",
       "  'accepted_name': 'acetaldehyde dehydrogenase (acetylating)',\n",
       "  'reaction': 'acetaldehyde + CoA + NAD+ = acetyl-CoA + NADH + H+',\n",
       "  'other_names': 'aldehyde dehydrogenase (acylating); ADA; acylating acetaldehyde dehyrogenase; DmpF; BphJ',\n",
       "  'sys_name': 'acetaldehyde:NAD+ oxidoreductase (CoA-acetylating)',\n",
       "  'comments': 'Also acts, more slowly, on glycolaldehyde, propanal and butanal. In several bacterial species this enzyme forms a bifunctional complex with EC 4.1.3.39, 4-hydroxy-2-oxovalerate aldolase. The enzymes from the bacteria Burkholderia xenovorans and Thermus thermophilus also perform the reaction of EC 1.2.1.87, propanal dehydrogenase (propanoylating). Involved in the meta-cleavage pathway for the degradation of phenols, methylphenols and catechols. NADP+ can replace NAD+ but the rate of reaction is much slower [3].',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.11': {'ec_num': '1.2.1.11',\n",
       "  'accepted_name': 'aspartate-semialdehyde dehydrogenase',\n",
       "  'reaction': 'L-aspartate 4-semialdehyde + phosphate + NADP+ = L-4-aspartyl phosphate + NADPH + H+',\n",
       "  'other_names': 'aspartate semialdehyde dehydrogenase; aspartic semialdehyde dehydrogenase; L-aspartate-β-semialdehyde:NADP+ oxidoreductase (phosphorylating); aspartic β-semialdehyde dehydrogenase; ASA dehydrogenase',\n",
       "  'sys_name': 'L-aspartate-4-semialdehyde:NADP+ oxidoreductase (phosphorylating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.12': {'ec_num': '1.2.1.12',\n",
       "  'accepted_name': 'glyceraldehyde-3-phosphate dehydrogenase (phosphorylating)',\n",
       "  'reaction': 'D-glyceraldehyde 3-phosphate + phosphate + NAD+ = 3-phospho-D-glyceroyl phosphate + NADH + H+',\n",
       "  'other_names': 'triosephosphate dehydrogenase (ambiguous); glyceraldehyde phosphate dehydrogenase; phosphoglyceraldehyde dehydrogenase; 3-phosphoglyceraldehyde dehydrogenase; NAD+-dependent glyceraldehyde phosphate dehydrogenase; glyceraldehyde phosphate dehydrogenase (NAD+); glyceraldehyde-3-phosphate dehydrogenase (NAD+); NADH-glyceraldehyde phosphate dehydrogenase; glyceraldehyde-3-P-dehydrogenase',\n",
       "  'sys_name': 'D-glyceraldehyde-3-phosphate:NAD+ oxidoreductase (phosphorylating)',\n",
       "  'comments': 'Also acts very slowly on D-glyceraldehyde and some other aldehydes; thiols can replace phosphate.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.13': {'ec_num': '1.2.1.13',\n",
       "  'accepted_name': 'glyceraldehyde-3-phosphate dehydrogenase (NADP+) (phosphorylating)',\n",
       "  'reaction': 'D-glyceraldehyde 3-phosphate + phosphate + NADP+ = 3-phospho-D-glyceroyl phosphate + NADPH + H+',\n",
       "  'other_names': 'triosephosphate dehydrogenase (NADP+); dehydrogenase, glyceraldehyde phosphate (nicotinamide adenine dinucleotide phosphate) (phosphorylating); glyceraldehyde phosphate dehydrogenase (nicotinamide adenine dinucleotide phosphate) (phosphorylating); NADP+-glyceraldehyde-3-phosphate dehydrogenase; NADP+-glyceraldehyde phosphate dehydrogenase; NADP+-dependent glyceraldehyde phosphate dehydrogenase; NADP+-triose phosphate dehydrogenase; glyceraldehyde-3-phosphate dehydrogenase (NADP+) (phosphorylating); GAPDH',\n",
       "  'sys_name': 'D-glyceraldehyde-3-phosphate:NADP+ oxidoreductase (phosphorylating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.15': {'ec_num': '1.2.1.15',\n",
       "  'accepted_name': 'malonate-semialdehyde dehydrogenase',\n",
       "  'reaction': '3-oxopropanoate + NAD(P)+ + H2O = malonate + NAD(P)H + 2 H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '3-oxopropanoate:NAD(P)+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.16': {'ec_num': '1.2.1.16',\n",
       "  'accepted_name': 'succinate-semialdehyde dehydrogenase [NAD(P)+]',\n",
       "  'reaction': 'succinate semialdehyde + NAD(P)+ + H2O = succinate + NAD(P)H + 2 H+',\n",
       "  'other_names': 'succinate semialdehyde dehydrogenase (nicotinamide adenine dinucleotide (phosphate)); succinate-semialdehyde dehydrogenase [NAD(P)]',\n",
       "  'sys_name': 'succinate-semialdehyde:NAD(P)+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.1.16\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.1.16\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.1.16\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.1.16\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.2.1.16%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.17': {'ec_num': '1.2.1.17',\n",
       "  'accepted_name': 'glyoxylate dehydrogenase (acylating)',\n",
       "  'reaction': 'glyoxylate + CoA + NADP+ = oxalyl-CoA + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'glyoxylate:NADP+ oxidoreductase (CoA-oxalylating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.18': {'ec_num': '1.2.1.18',\n",
       "  'accepted_name': 'malonate-semialdehyde dehydrogenase (acetylating)',\n",
       "  'reaction': '3-oxopropanoate + CoA + NAD(P)+ = acetyl-CoA + CO2 + NAD(P)H',\n",
       "  'other_names': 'malonic semialdehyde oxidative decarboxylase',\n",
       "  'sys_name': '3-oxopropanoate:NAD(P)+ oxidoreductase (decarboxylating, CoA-acetylating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.19': {'ec_num': '1.2.1.19',\n",
       "  'accepted_name': 'aminobutyraldehyde dehydrogenase',\n",
       "  'reaction': '4-aminobutanal + NAD+ + H2O = 4-aminobutanoate + NADH + 2 H+',\n",
       "  'other_names': 'γ-guanidinobutyraldehyde dehydrogenase (ambiguous); ABAL dehydrogenase; 4-aminobutyraldehyde dehydrogenase; 4-aminobutanal dehydrogenase; γ-aminobutyraldehyde dehydroganase; 1-pyrroline dehydrogenase; ABALDH; YdcW',\n",
       "  'sys_name': '4-aminobutanal:NAD+ 1-oxidoreductase',\n",
       "  'comments': 'The enzyme from some species exhibits broad substrate specificity and has a marked preference for straight-chain aldehydes (up to 7 carbon atoms) as substrates [9]. The plant enzyme also acts on 4-guanidinobutanal (cf. EC 1.2.1.54 γ-guanidinobutyraldehyde dehydrogenase). As 1-pyrroline and 4-aminobutanal are in equilibrium and can be interconverted spontaneously, 1-pyrroline may act as the starting substrate. The enzyme forms part of the arginine-catabolism pathway [8] and belongs in the aldehyde dehydrogenase superfamily [9].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '4-aminobutanoate = γ-aminobutyrate = GABA',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.20': {'ec_num': '1.2.1.20',\n",
       "  'accepted_name': 'glutarate-semialdehyde dehydrogenase',\n",
       "  'reaction': '5-oxopentanoate + NADP+ + H2O = glutarate + NADPH + H+',\n",
       "  'other_names': 'glutarate semialdehyde dehydrogenase; davD (gene name)',\n",
       "  'sys_name': 'glutarate-semialdehyde:NADP+ oxidoreductase',\n",
       "  'comments': 'The enzyme, characterized from multiple Pseudomonas strains, participates in L-lysine degradation. Unlike earlier claims, it prefers NADP+ to NAD+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '5-oxopentanoate = glutarate semialdehyde',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.21': {'ec_num': '1.2.1.21',\n",
       "  'accepted_name': 'glycolaldehyde dehydrogenase',\n",
       "  'reaction': 'glycolaldehyde + NAD+ + H2O = glycolate + NADH + 2 H+',\n",
       "  'other_names': 'glycol aldehyde dehydrogenase',\n",
       "  'sys_name': 'glycolaldehyde:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.22': {'ec_num': '1.2.1.22',\n",
       "  'accepted_name': 'lactaldehyde dehydrogenase',\n",
       "  'reaction': '(S)-lactaldehyde + NAD+ + H2O = (S)-lactate + NADH + 2 H+',\n",
       "  'other_names': 'L-lactaldehyde:NAD oxidoreductase; nicotinamide adenine dinucleotide (NAD)-linked dehydrogenase',\n",
       "  'sys_name': '(S)-lactaldehyde:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.23': {'ec_num': '1.2.1.23',\n",
       "  'accepted_name': '2-oxoaldehyde dehydrogenase (NAD+)',\n",
       "  'reaction': 'a 2-oxoaldehyde + NAD+ + H2O = a 2-oxo carboxylate + NADH + H+',\n",
       "  'other_names': 'α-ketoaldehyde dehydrogenase; methylglyoxal dehydrogenase; NAD+-linked α-ketoaldehyde dehydrogenase; 2-ketoaldehyde dehydrogenase; NAD+-dependent α-ketoaldehyde dehydrogenase',\n",
       "  'sys_name': '2-oxoaldehyde:NAD+ 2-oxidoreductase',\n",
       "  'comments': 'Not identical with EC 1.2.1.49 2-oxoaldehyde dehydrogenase (NADP+).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.24': {'ec_num': '1.2.1.24',\n",
       "  'accepted_name': 'succinate-semialdehyde dehydrogenase (NAD+)',\n",
       "  'reaction': 'succinate semialdehyde + NAD+ + H2O = succinate + NADH + 2 H+',\n",
       "  'other_names': 'succinate semialdehyde dehydrogenase (NAD+); succinic semialdehyde dehydrogenase (NAD+); succinyl semialdehyde dehydrogenase (NAD+); succinate semialdehyde:NAD+ oxidoreductase',\n",
       "  'sys_name': 'succinate-semialdehyde:NAD+ oxidoreductase',\n",
       "  'comments': 'This enzyme participates in the degradation of glutamate and 4-aminobutyrate. It is similar to EC 1.2.1.79 [succinate-semialdehyde dehydrogenase (NADP+)], and EC 1.2.1.16 [succinate-semialdehyde dehydrogenase (NAD(P)+)], but is specific for NAD+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.25': {'ec_num': '1.2.1.25',\n",
       "  'accepted_name': 'branched-chain α-keto acid dehydrogenase system',\n",
       "  'reaction': '3-methyl-2-oxobutanoate + CoA + NAD+ = 2-methylpropanoyl-CoA + CO2 + NADH',\n",
       "  'other_names': 'branched-chain α-keto acid dehydrogenase complex; 2-oxoisovalerate dehydrogenase; α-ketoisovalerate dehydrogenase; 2-oxoisovalerate dehydrogenase (acylating)',\n",
       "  'sys_name': '3-methyl-2-oxobutanoate:NAD+ 2-oxidoreductase (CoA-methylpropanoylating)',\n",
       "  'comments': 'This enzyme system catalyses the oxidative decarboxylation of branched-chain α-keto acids derived from L-leucine, L-isoleucine, and L-valine to branched-chain acyl-CoAs. It belongs to the 2-oxoacid dehydrogenase system family, which also includes EC 1.2.1.104, pyruvate dehydrogenase system, EC 1.2.1.105, 2-oxoglutarate dehydrogenase system, EC 1.4.1.27, glycine cleavage system, and EC 2.3.1.190, acetoin dehydrogenase system. With the exception of the glycine cleavage system, which contains 4 components, the 2-oxoacid dehydrogenase systems share a common structure, consisting of three main components, namely a 2-oxoacid dehydrogenase (E1), a dihydrolipoamide acyltransferase (E2), and dihydrolipoamide dehydrogenase (E3). The reaction catalysed by this system is the sum of three activities: EC 1.2.4.4, 3-methyl-2-oxobutanoate dehydrogenase (2-methylpropanoyl-transferring), EC 2.3.1.168, dihydrolipoyllysine-residue (2-methylpropanoyl)transferase, and EC 1.8.1.4, dihydrolipoyl dehydrogenase. The system also acts on (S)-3-methyl-2-oxopentanoate and 4-methyl-2-oxopentanoate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.26': {'ec_num': '1.2.1.26',\n",
       "  'accepted_name': '2,5-dioxovalerate dehydrogenase',\n",
       "  'reaction': '2,5-dioxopentanoate + NADP+ + H2O = 2-oxoglutarate + NADPH + 2 H+',\n",
       "  'other_names': '2-oxoglutarate semialdehyde dehydrogenase; α-ketoglutaric semialdehyde dehydrogenase',\n",
       "  'sys_name': '2,5-dioxopentanoate:NADP+ 5-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.27': {'ec_num': '1.2.1.27',\n",
       "  'accepted_name': 'methylmalonate-semialdehyde dehydrogenase (CoA-acylating)',\n",
       "  'reaction': '2-methyl-3-oxopropanoate + CoA + H2O + NAD+ = propanoyl-CoA + HCO3- + NADH',\n",
       "  'other_names': 'MSDH; MMSA dehydrogenase; iolA (gene name); methylmalonate-semialdehyde dehydrogenase (acylating)',\n",
       "  'sys_name': '2-methyl-3-oxopropanoate:NAD+ 3-oxidoreductase (CoA-propanoylating)',\n",
       "  'comments': 'Also converts 3-oxopropanoate into acetyl-CoA [3]. The reaction occurs in two steps with the decarboxylation process preceding CoA-binding [3]. Bicarbonate rather than CO2 is released as a final product [3].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'methylmalonate semialdehyde = 2-methyl-3-oxopropanoate',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.28': {'ec_num': '1.2.1.28',\n",
       "  'accepted_name': 'benzaldehyde dehydrogenase (NAD+)',\n",
       "  'reaction': 'benzaldehyde + NAD+ + H2O = benzoate + NADH + 2 H+',\n",
       "  'other_names': 'benzaldehyde (NAD) dehydrogenase; benzaldehyde dehydrogenase (NAD)',\n",
       "  'sys_name': 'benzaldehyde:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.1.28\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.1.28\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.1.28\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.1.28\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.2.1.28%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.29': {'ec_num': '1.2.1.29',\n",
       "  'accepted_name': 'aryl-aldehyde dehydrogenase',\n",
       "  'reaction': 'an aromatic aldehyde + NAD+ + H2O = an aromatic acid + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'aryl-aldehyde:NAD+ oxidoreductase',\n",
       "  'comments': 'Oxidizes a number of aromatic aldehydes, but not aliphatic aldehydes.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.1.29\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.1.29\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.1.29\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.1.29\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.30': {'ec_num': '1.2.1.30',\n",
       "  'accepted_name': 'carboxylate reductase (NADP+)',\n",
       "  'reaction': 'an aromatic aldehyde + NADP+ + AMP + diphosphate = an aromatic acid + NADPH + H+ + ATP',\n",
       "  'other_names': 'aromatic acid reductase; aryl-aldehyde dehydrogenase (NADP+)',\n",
       "  'sys_name': 'aryl-aldehyde:NADP+ oxidoreductase (ATP-forming)',\n",
       "  'comments': 'The enzyme contains an adenylation domain, a phosphopantetheinyl binding domain, and a reductase domain, and requires activation by attachment of a phosphopantetheinyl group. The enzyme activates its substrate to an adenylate form, followed by a transfer to the phosphopantetheinyl binding domain. The resulting thioester is subsequently transferred to the reductase domain, where it is reduced to an aldehyde and released.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.31': {'ec_num': '1.2.1.31',\n",
       "  'accepted_name': 'L-aminoadipate-semialdehyde dehydrogenase',\n",
       "  'reaction': '(S)-2-amino-6-oxohexanoate + NAD(P)+ + H2O = L-2-aminoadipate + NAD(P)H + H+ (overall reaction);;(1a) (S)-2-amino-6-oxohexanoate = (S)-2,3,4,5-tetrahydropyridine-2-carboxylate + H2O (spontaneous);;(1b) (S)-2,3,4,5-tetrahydropyridine-2-carboxylate + NAD(P)+ + 2 H2O = L-2-aminoadipate + NAD(P)H + H+',\n",
       "  'other_names': 'aminoadipate semialdehyde dehydrogenase; 2-aminoadipate semialdehyde dehydrogenase; α-aminoadipate-semialdehyde dehydrogenase; α-aminoadipate reductase; 2-aminoadipic semialdehyde dehydrogenase; L-α-aminoadipate δ-semialdehyde oxidoreductase; L-α-aminoadipate δ-semialdehyde:NAD+ oxidoreductase; L-α-aminoadipate δ-semialdehyde:nicotinamide adenine dinucleotide oxidoreductase; L-2-aminoadipate 6-semialdehyde:NAD(P)+ 6-oxidoreductase',\n",
       "  'sys_name': '(S)-2-amino-6-oxohexanoate:NAD(P)+ 6-oxidoreductase',\n",
       "  'comments': '(S)-2-amino-6-oxohexanoate undergoes a spontaneous dehydration forming the cyclic (S)-2,3,4,5-tetrahydropyridine-2-carboxylate, which serves as a substrate for the hydrogenation reaction.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '(S)-2-amino-6-oxohexanoate = L-2-aminoadipate 6-semialdehyde = L-allysine\\nL-1-piperideine 6-carboxylate = (S)-2,3,4,5-tetrahydropyridine-2-carboxylate = (S)-1,6-didehydropiperidine-2-carboxylate',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.32': {'ec_num': '1.2.1.32',\n",
       "  'accepted_name': 'aminomuconate-semialdehyde dehydrogenase',\n",
       "  'reaction': '2-aminomuconate 6-semialdehyde + NAD+ + H2O = 2-aminomuconate + NADH + 2 H+',\n",
       "  'other_names': '2-aminomuconate semialdehyde dehydrogenase; 2-hydroxymuconic acid semialdehyde dehydrogenase; 2-hydroxymuconate semialdehyde dehydrogenase; α-aminomuconic ε-semialdehyde dehydrogenase; α-hydroxymuconic ε-semialdehyde dehydrogenase; 2-hydroxymuconic semialdehyde dehydrogenase',\n",
       "  'sys_name': '2-aminomuconate-6-semialdehyde:NAD+ 6-oxidoreductase',\n",
       "  'comments': 'Also acts on 2-hydroxymuconate semialdehyde.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.1.32\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.1.32\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.1.32\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.1.32\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.2.1.32%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.33': {'ec_num': '1.2.1.33',\n",
       "  'accepted_name': '(R)-dehydropantoate dehydrogenase',\n",
       "  'reaction': '(R)-4-dehydropantoate + NAD+ + H2O = (R)-3,3-dimethylmalate + NADH + 2 H+',\n",
       "  'other_names': 'D-aldopantoate dehydrogenase; D-2-hydroxy-3,3-dimethyl-3-formylpropionate:diphosphopyridine nucleotide (DPN+) oxidoreductase',\n",
       "  'sys_name': '(R)-4-dehydropantoate:NAD+ 4-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.36': {'ec_num': '1.2.1.36',\n",
       "  'accepted_name': 'retinal dehydrogenase',\n",
       "  'reaction': 'retinal + NAD+ + H2O = retinoate + NADH + 2 H+',\n",
       "  'other_names': 'cytosolic retinal dehydrogenase',\n",
       "  'sys_name': 'retinal:NAD+ oxidoreductase',\n",
       "  'comments': 'A metalloflavoprotein (FAD). Acts on both the 11-trans- and 13-cis-forms of retinal.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.38': {'ec_num': '1.2.1.38',\n",
       "  'accepted_name': 'N-acetyl-γ-glutamyl-phosphate reductase',\n",
       "  'reaction': 'N-acetyl-L-glutamate 5-semialdehyde + NADP+ + phosphate = N-acetyl-L-glutamyl 5-phosphate + NADPH + H+',\n",
       "  'other_names': 'reductase, acetyl-γ-glutamyl phosphate; N-acetylglutamate 5-semialdehyde dehydrogenase; N-acetylglutamic γ-semialdehyde dehydrogenase; N-acetyl-L-glutamate γ-semialdehyde:NADP+ oxidoreductase (phosphorylating)',\n",
       "  'sys_name': 'N-acetyl-L-glutamate-5-semialdehyde:NADP+ 5-oxidoreductase (phosphorylating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.39': {'ec_num': '1.2.1.39',\n",
       "  'accepted_name': 'phenylacetaldehyde dehydrogenase',\n",
       "  'reaction': 'phenylacetaldehyde + NAD+ + H2O = phenylacetate + NADH + 2 H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'phenylacetaldehyde:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.1.39\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.1.39\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.1.39\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.1.39\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.2.1.39%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.40': {'ec_num': '1.2.1.40',\n",
       "  'accepted_name': '3α,7α,12α-trihydroxycholestan-26-al 26-oxidoreductase',\n",
       "  'reaction': '(25R)-3α,7α,12α-trihydroxy-5β-cholestan-26-al + NAD+ + H2O = (25R)-3α,7α,12α-trihydroxy-5β-cholestan-26-oate + NADH + 2 H+',\n",
       "  'other_names': 'cholestanetriol-26-al 26-dehydrogenase; 3α,7α,12α-trihydroxy-5β-cholestan-26-al dehydrogenase; trihydroxydeoxycoprostanal dehydrogenase; THAL-NAD oxidoreductase; 3α,7α,12α-trihydroxy-5β-cholestan-26-al:NAD+ 26-oxidoreductase',\n",
       "  'sys_name': '(25R)-3α,7α,12α-trihydroxy-5β-cholestan-26-al:NAD+ 26-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.41': {'ec_num': '1.2.1.41',\n",
       "  'accepted_name': 'glutamate-5-semialdehyde dehydrogenase',\n",
       "  'reaction': 'L-glutamate 5-semialdehyde + phosphate + NADP+ = L-glutamyl 5-phosphate + NADPH + H+',\n",
       "  'other_names': 'β-glutamylphosphate reductase; γ-glutamyl phosphate reductase; β-glutamylphosphate reductase; glutamate semialdehyde dehydrogenase; glutamate-γ-semialdehyde dehydrogenase',\n",
       "  'sys_name': 'L-glutamate-5-semialdehyde:NADP+ 5-oxidoreductase (phosphorylating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.42': {'ec_num': '1.2.1.42',\n",
       "  'accepted_name': 'hexadecanal dehydrogenase (acylating)',\n",
       "  'reaction': 'hexadecanal + CoA + NAD+ = hexadecanoyl-CoA + NADH + H+',\n",
       "  'other_names': 'fatty acyl-CoA reductase',\n",
       "  'sys_name': 'hexadecanal:NAD+ oxidoreductase (CoA-acylating)',\n",
       "  'comments': 'Also acts, more slowly, on octadecanoyl-CoA.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.1.42\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.1.42\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.1.42\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.1.42\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.43': {'ec_num': '1.2.1.43',\n",
       "  'accepted_name': 'formate dehydrogenase (NADP+)',\n",
       "  'reaction': 'formate + NADP+ = CO2 + NADPH',\n",
       "  'other_names': 'NADP+-dependent formate dehydrogenase',\n",
       "  'sys_name': 'formate:NADP+ oxidoreductase',\n",
       "  'comments': 'A tungsten-selenium-iron protein.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.44': {'ec_num': '1.2.1.44',\n",
       "  'accepted_name': 'cinnamoyl-CoA reductase',\n",
       "  'reaction': 'cinnamaldehyde + CoA + NADP+ = cinnamoyl-CoA + NADPH + H+',\n",
       "  'other_names': 'feruloyl-CoA reductase; cinnamoyl-coenzyme A reductase; ferulyl-CoA reductase; feruloyl coenzyme A reductase; p-hydroxycinnamoyl coenzyme A reductase; cinnamoyl-CoA:NADPH reductase',\n",
       "  'sys_name': 'cinnamaldehyde:NADP+ oxidoreductase (CoA-cinnamoylating)',\n",
       "  'comments': 'Acts also on a number of substituted cinnamoyl esters of coenzyme A.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.45': {'ec_num': '1.2.1.45',\n",
       "  'accepted_name': '4-carboxy-2-hydroxymuconate-6-semialdehyde dehydrogenase',\n",
       "  'reaction': '4-carboxy-2-hydroxy-cis,cis-muconate 6-semialdehyde + NADP+ + H2O = 4-carboxy-2-hydroxy-cis,cis-muconate + NADPH + 2 H+',\n",
       "  'other_names': '2-hydroxy-4-carboxymuconate 6-semialdehyde dehydrogenase; 4-carboxy-2-hydroxy-cis,cis-muconate-6-semialdehyde:NADP+ oxidoreductase; α-hydroxy-γ-carboxymuconic ε-semialdehyde dehydrogenase',\n",
       "  'sys_name': '4-carboxy-2-hydroxy-cis,cis-muconate-6-semialdehyde:NADP+ 6-oxidoreductase',\n",
       "  'comments': 'The enzyme does not act on unsubstituted aliphatic or aromatic aldehydes or glucose; NAD+ can replace NADP+, but with lower affinity.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.81': {'ec_num': '1.2.1.81',\n",
       "  'accepted_name': 'sulfoacetaldehyde dehydrogenase (acylating)',\n",
       "  'reaction': '2-sulfoacetaldehyde + CoA + NADP+ = sulfoacetyl-CoA + NADPH + H+ ',\n",
       "  'other_names': 'SauS',\n",
       "  'sys_name': '2-sulfoacetaldehyde:NADP+ oxidoreductase (CoA-acetylating)',\n",
       "  'comments': 'The enzyme is involved in degradation of sulfoacetate. In this pathway the reaction is catalysed in the reverse direction. The enzyme is specific for sulfoacetaldehyde and NADP+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '2-sulfoacetaldehyde = 2-oxoethanesulfonate',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.46': {'ec_num': '1.2.1.46',\n",
       "  'accepted_name': 'formaldehyde dehydrogenase',\n",
       "  'reaction': 'formaldehyde + NAD+ + H2O = formate + NADH + 2 H+',\n",
       "  'other_names': 'NAD-linked formaldehyde dehydrogenase; NAD-dependent formaldehyde dehydrogenase',\n",
       "  'sys_name': 'formaldehyde:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.1.46\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.1.46\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.1.46\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.1.46\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.2.1.46%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.47': {'ec_num': '1.2.1.47',\n",
       "  'accepted_name': '4-trimethylammoniobutyraldehyde dehydrogenase',\n",
       "  'reaction': '4-trimethylammoniobutanal + NAD+ + H2O = 4-trimethylammoniobutanoate + NADH + 2 H+',\n",
       "  'other_names': '4-trimethylaminobutyraldehyde dehydrogenase; 4-N-trimethylaminobutyraldehyde dehydrogenase',\n",
       "  'sys_name': '4-trimethylammoniobutanal:NAD+ 1-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.48': {'ec_num': '1.2.1.48',\n",
       "  'accepted_name': 'long-chain-aldehyde dehydrogenase',\n",
       "  'reaction': 'a long-chain aldehyde + NAD+ + H2O = a long-chain carboxylate + NADH + 2 H+',\n",
       "  'other_names': 'long-chain aliphatic aldehyde dehydrogenase; long-chain fatty aldehyde dehydrogenase; fatty aldehyde:NAD+ oxidoreductase',\n",
       "  'sys_name': 'long-chain-aldehyde:NAD+ oxidoreductase',\n",
       "  'comments': 'The best substrate is dodecylaldehyde.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.1.48\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.1.48\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.1.48\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.1.48\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.49': {'ec_num': '1.2.1.49',\n",
       "  'accepted_name': '2-oxoaldehyde dehydrogenase (NADP+)',\n",
       "  'reaction': 'a 2-oxoaldehyde + NADP+ + H2O = a 2-oxo carboxylate + NADPH + H+',\n",
       "  'other_names': 'α-ketoaldehyde dehydrogenase; methylglyoxal dehydrogenase; NADP+-linked α-ketoaldehyde dehydrogenase; 2-ketoaldehyde dehydrogenase; NADP+-dependent α-ketoaldehyde dehydrogenase',\n",
       "  'sys_name': '2-oxoaldehyde:NADP+ 2-oxidoreductase',\n",
       "  'comments': 'Not identical with EC 1.2.1.23 2-oxoaldehyde dehydrogenase (NAD+).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.50': {'ec_num': '1.2.1.50',\n",
       "  'accepted_name': 'long-chain acyl-protein thioester reductase',\n",
       "  'reaction': 'a long-chain aldehyde + [protein]-L-cysteine + NADP+ = a [protein]-S-(long-chain fatty acyl)-L-cysteine + NADPH + H+;;',\n",
       "  'other_names': 'luxC (gene name); acyl-CoA reductase; acyl coenzyme A reductase; long-chain-aldehyde:NADP+ oxidoreductase (acyl-CoA-forming); long-chain-fatty-acyl-CoA reductase',\n",
       "  'sys_name': 'long-chain-aldehyde:NADP+ oxidoreductase (protein thioester-forming)',\n",
       "  'comments': 'Together with a hydrolase component (EC 3.1.2.2 and EC 3.1.2.14) and a synthetase component (EC 6.2.1.19), this enzyme forms a multienzyme fatty acid reductase complex that produces the long-chain aldehyde substrate of the bacterial luciferase enzyme (EC 1.14.14.3). The enzyme is acylated by receiving an acyl group from EC 6.2.1.19, and catalyses the reduction of the acyl group, releasing the aldehyde product. The enzyme is also able to accept the acyl group from a long-chain acyl-CoA.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.51': {'ec_num': '1.2.1.51',\n",
       "  'accepted_name': 'pyruvate dehydrogenase (NADP+)',\n",
       "  'reaction': 'pyruvate + CoA + NADP+ = acetyl-CoA + CO2 + NADPH',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'pyruvate:NADP+ 2-oxidoreductase (CoA-acetylating)',\n",
       "  'comments': 'The Euglena enzyme can also use FAD or methyl viologen as acceptor, more slowly. The enzyme is inhibited by oxygen.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'methyl viologen = 1,1′-dimethyl-4,4′-bipyridine-1,1′-diium',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.52': {'ec_num': '1.2.1.52',\n",
       "  'accepted_name': 'oxoglutarate dehydrogenase (NADP+)',\n",
       "  'reaction': '2-oxoglutarate + CoA + NADP+ = succinyl-CoA + CO2 + NADPH',\n",
       "  'other_names': 'oxoglutarate dehydrogenase (NADP)',\n",
       "  'sys_name': '2-oxoglutarate:NADP+ 2-oxidoreductase (CoA-succinylating)',\n",
       "  'comments': 'The Euglena enzyme can also use NAD+ as acceptor, but more slowly.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.53': {'ec_num': '1.2.1.53',\n",
       "  'accepted_name': '4-hydroxyphenylacetaldehyde dehydrogenase',\n",
       "  'reaction': '4-hydroxyphenylacetaldehyde + NAD+ + H2O = 4-hydroxyphenylacetate + NADH + 2 H+',\n",
       "  'other_names': '4-HPAL dehydrogenase',\n",
       "  'sys_name': '4-hydroxyphenylacetaldehyde:NAD+ oxidoreductase',\n",
       "  'comments': 'With EC 4.2.1.87 octopamine dehydratase, brings about the metabolism of octopamine in Pseudomonas.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.54': {'ec_num': '1.2.1.54',\n",
       "  'accepted_name': 'γ-guanidinobutyraldehyde dehydrogenase',\n",
       "  'reaction': '4-guanidinobutanal + NAD+ + H2O = 4-guanidinobutanoate + NADH + 2 H+',\n",
       "  'other_names': 'α-guanidinobutyraldehyde dehydrogenase; 4-guanidinobutyraldehyde dehydrogenase; GBAL dehydrogenase',\n",
       "  'sys_name': '4-guanidinobutanal:NAD+ 1-oxidoreductase',\n",
       "  'comments': 'Involved in the degradation of arginine in Pseudomonas putida (cf. EC 1.2.1.19 aminobutyraldehyde dehydrogenase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.57': {'ec_num': '1.2.1.57',\n",
       "  'accepted_name': 'butanal dehydrogenase',\n",
       "  'reaction': 'butanal + CoA + NAD(P)+ = butanoyl-CoA + NAD(P)H + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'butanal:NAD(P)+ oxidoreductase (CoA-acylating)',\n",
       "  'comments': 'Also acts on acetaldehyde, but more slowly.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.58': {'ec_num': '1.2.1.58',\n",
       "  'accepted_name': 'phenylglyoxylate dehydrogenase (acylating)',\n",
       "  'reaction': 'phenylglyoxylate + NAD+ + CoA = benzoyl-S-CoA + CO2 + NADH',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'phenylglyoxylate:NAD+ oxidoreductase',\n",
       "  'comments': 'Requires thiamine diphosphate as cofactor. The enzyme from the denitrifying bacterium Azoarcus evansii is specific for phenylglyoxylate. 2-Oxoisovalerate is oxidized at 15% of the rate for phenylglyoxylate. Also reduces viologen dyes. Contains iron-sulfur centres and FAD.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.59': {'ec_num': '1.2.1.59',\n",
       "  'accepted_name': 'glyceraldehyde-3-phosphate dehydrogenase (NAD(P)+) (phosphorylating)',\n",
       "  'reaction': 'D-glyceraldehyde 3-phosphate + phosphate + NAD(P)+ = 3-phospho-D-glyceroyl phosphate + NAD(P)H + H+',\n",
       "  'other_names': 'triosephosphate dehydrogenase (NAD(P)); glyceraldehyde-3-phosphate dehydrogenase (NAD(P)) (phosphorylating)',\n",
       "  'sys_name': 'D-glyceraldehyde 3-phosphate:NAD(P)+ oxidoreductase (phosphorylating)',\n",
       "  'comments': 'NAD+ and NADP+ can be used as cofactors with similar efficiency, unlike EC 1.2.1.12 glyceraldehyde-3-phosphate dehydrogenase (phosphorylating) and EC 1.2.1.13 glyceraldehyde-3-phosphate dehydrogenase (NADP+) (phosphorylating), which are NAD+- and NADP+-dependent, respectively.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.60': {'ec_num': '1.2.1.60',\n",
       "  'accepted_name': '5-carboxymethyl-2-hydroxymuconic-semialdehyde dehydrogenase',\n",
       "  'reaction': '5-carboxymethyl-2-hydroxymuconate semialdehyde + H2O + NAD+ = 5-carboxymethyl-2-hydroxymuconate + NADH + 2 H+',\n",
       "  'other_names': 'carboxymethylhydroxymuconic semialdehyde dehydrogenase',\n",
       "  'sys_name': '5-carboxymethyl-2-hydroxymuconic-semialdehyde:NAD+ oxidoreductase',\n",
       "  'comments': 'Involved in the tyrosine degradation pathway in Arthrobacter sp.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.1.60\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.1.60\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.1.60\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.1.60\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.2.1.60%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.61': {'ec_num': '1.2.1.61',\n",
       "  'accepted_name': '4-hydroxymuconic-semialdehyde dehydrogenase',\n",
       "  'reaction': '4-hydroxymuconic semialdehyde + NAD+ + H2O = maleylacetate + NADH + 2 H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '4-hydroxymuconic-semialdehyde:NAD+ oxidoreductase',\n",
       "  'comments': 'Involved in the 4-nitrophenol degradation pathway.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.1.61\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.1.61\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.1.61\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.1.61\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.62': {'ec_num': '1.2.1.62',\n",
       "  'accepted_name': '4-formylbenzenesulfonate dehydrogenase',\n",
       "  'reaction': '4-formylbenzenesulfonate + NAD+ + H2O = 4-sulfobenzoate + NADH + 2 H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '4-formylbenzenesulfonate:NAD+ oxidoreductase',\n",
       "  'comments': 'Involved in the toluene-4-sulfonate degradation pathway.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.1.62\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.1.62\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.1.62\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.1.62\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.63': {'ec_num': '1.2.1.63',\n",
       "  'accepted_name': '6-oxohexanoate dehydrogenase',\n",
       "  'reaction': '6-oxohexanoate + NADP+ + H2O = adipate + NADPH + 2 H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '6-oxohexanoate:NADP+ oxidoreductase',\n",
       "  'comments': 'Last step in the cyclohexanol degradation pathway in Acinetobacter sp.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.1.63\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.1.63\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.1.63\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.1.63\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.64': {'ec_num': '1.2.1.64',\n",
       "  'accepted_name': '4-hydroxybenzaldehyde dehydrogenase (NAD+)',\n",
       "  'reaction': '4-hydroxybenzaldehyde + NAD+ + H2O = 4-hydroxybenzoate + NADH + 2 H+',\n",
       "  'other_names': 'p-hydroxybenzaldehyde dehydrogenase (ambiguous); 4-hydroxybenzaldehyde dehydrogenase (ambiguous)',\n",
       "  'sys_name': '4-hydroxybenzaldehyde:NAD+ oxidoreductase',\n",
       "  'comments': 'The bacterial enzyme (characterized from an unidentified denitrifying bacterium) is involved in an anaerobic toluene degradation pathway. The plant enzyme is involved in formation of 4-hydroxybenzoate, a cell wall-bound phenolic acid that plays a major role in plant defense against pathogens. cf. EC 1.2.1.96, 4-hydroxybenzaldehyde dehydrogenase (NADP+).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.1.64\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.1.64\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.1.64\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.1.64\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.65': {'ec_num': '1.2.1.65',\n",
       "  'accepted_name': 'salicylaldehyde dehydrogenase',\n",
       "  'reaction': 'salicylaldehyde + NAD+ + H2O = salicylate + NADH + 2 H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'salicylaldehyde:NAD+ oxidoreductase',\n",
       "  'comments': 'Involved in the naphthalene degradation pathway in some bacteria.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.1.65\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.1.65\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.1.65\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.1.65\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.2.1.65%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': 'salicylaldehyde = 2-hydroxybenzaldehyde',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.66': {'ec_num': '1.2.1.66',\n",
       "  'accepted_name': 'mycothiol-dependent formaldehyde dehydrogenase',\n",
       "  'reaction': 'formaldehyde + mycothiol + NAD+ = S-formylmycothiol + NADH + 2 H+',\n",
       "  'other_names': 'NAD/factor-dependent formaldehyde dehydrogenase',\n",
       "  'sys_name': 'formaldehyde:NAD+ oxidoreductase (mycothiol-formylating)',\n",
       "  'comments': 'The S-formylmycothiol formed hydrolyses to mycothiol and formate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'mycothiol = 1-O-[2-(N-acetyl-L-cysteinamido)-2-deoxy-α-D-glucopyranosyl]-D-myo-inositol',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.67': {'ec_num': '1.2.1.67',\n",
       "  'accepted_name': 'vanillin dehydrogenase',\n",
       "  'reaction': 'vanillin + NAD+ + H2O = vanillate + NADH + 2 H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'vanillin:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.1.67\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.1.67\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.1.67\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.1.67\" target=\"new\">KEGG</a>',\n",
       "  'glossary': 'vanillate = 4-hydroxy-3-methoxybenzoate<br>\\nvanillin = 4-hydroxy-3-methoxybenzaldehyde',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.68': {'ec_num': '1.2.1.68',\n",
       "  'accepted_name': 'coniferyl-aldehyde dehydrogenase',\n",
       "  'reaction': 'coniferyl aldehyde + H2O + NAD(P)+ = ferulate + NAD(P)H + 2 H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'coniferyl aldehyde:NAD(P)+ oxidoreductase',\n",
       "  'comments': 'Also oxidizes other aromatic aldehydes, but not aliphatic aldehydes.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.69': {'ec_num': '1.2.1.69',\n",
       "  'accepted_name': 'fluoroacetaldehyde dehydrogenase',\n",
       "  'reaction': 'fluoroacetaldehyde + NAD+ + H2O = fluoroacetate + NADH + 2 H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'fluoroacetaldehyde:NAD+ oxidoreductase',\n",
       "  'comments': 'The enzyme from Streptomyces cattleya has a high affinity for fluoroacetate and glycolaldehyde but not for acetaldehyde.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.1.70': {'ec_num': '1.2.1.70',\n",
       "  'accepted_name': 'glutamyl-tRNA reductase',\n",
       "  'reaction': 'L-glutamate 1-semialdehyde + NADP+ + tRNAGlu = L-glutamyl-tRNAGlu + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'L-glutamate-semialdehyde:NADP+ oxidoreductase (L-glutamyl-tRNAGlu-forming)',\n",
       "  'comments': 'This enzyme forms part of the pathway for the biosynthesis of 5-aminolevulinate from glutamate, known as the  C5 pathway.  The route shown in the diagram is used in most eubacteria, and in all archaebacteria, algae and plants. However, in the α-proteobacteria, EC 2.3.1.37, 5-aminolevulinate synthase, is used in an alternative route to produce the product 5-aminolevulinate from succinyl-CoA and glycine. This route is found in the mitochondria of fungi and animals, organelles that are considered to be derived from an endosymbiotic α-proteobacterium. Although higher plants do not possess EC 2.3.1.37, the protistan Euglena gracilis possesses both the C5 pathway and EC 2.3.1.37.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.2.1': {'ec_num': '1.2.2.1',\n",
       "  'accepted_name': 'formate dehydrogenase (cytochrome)',\n",
       "  'reaction': 'formate + 2 ferricytochrome b1 = CO2 + 2 ferrocytochrome b1 + 2 H+',\n",
       "  'other_names': 'formate dehydrogenase; formate:cytochrome b1 oxidoreductase',\n",
       "  'sys_name': 'formate:ferricytochrome-b1 oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.2.2': {'ec_num': '1.2.2.2',\n",
       "  'accepted_name': 'pyruvate dehydrogenase (cytochrome)',\n",
       "  'reaction': 'pyruvate + ferricytochrome b1 + H2O = acetate + CO2 + ferrocytochrome b1',\n",
       "  'other_names': 'pyruvate dehydrogenase; pyruvic dehydrogenase; pyruvic (cytochrome b1) dehydrogenase; pyruvate:ubiquinone-8-oxidoreductase; pyruvate oxidase (ambiguous)',\n",
       "  'sys_name': 'pyruvate:ferricytochrome-b1 oxidoreductase',\n",
       "  'comments': 'A flavoprotein requiring thiamine diphosphate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'thiamine diphosphate = 3-[(4-amino-2-methylpyrimidin-5-yl)methyl]-5-(2-diphosphoethyl)-4-methyl-1,3-thiazolium',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.2.3': {'ec_num': '1.2.2.3',\n",
       "  'accepted_name': 'formate dehydrogenase (cytochrome-c-553)',\n",
       "  'reaction': 'formate + 2 ferricytochrome c-553 = CO2 + 2 ferrocytochrome c-553 + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'formate:ferricytochrome-c-553 oxidoreductase',\n",
       "  'comments': 'Yeast cytochrome c, ferricyanide and phenazine methosulfate can act as acceptor.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.2.4': {'ec_num': '1.2.2.4',\n",
       "  'accepted_name': 'carbon-monoxide dehydrogenase (cytochrome b-561)',\n",
       "  'reaction': 'CO + H2O + 2 ferricytochrome b-561 = CO2 + 2 H+ + 2 ferrocytochrome b-561',\n",
       "  'other_names': 'carbon monoxide oxidase; carbon monoxide oxygenase (cytochrome b-561); carbon monoxide:methylene blue oxidoreductase; CO dehydrogenase; carbon-monoxide dehydrogenase',\n",
       "  'sys_name': 'carbon monoxide,water:cytochrome b-561 oxidoreductase',\n",
       "  'comments': 'Contains molybdopterin cytosine dinucleotide, FAD and [2Fe-2S]-clusters. Oxygen, methylene blue and iodonitrotetrazolium chloride can act as nonphysiological electron acceptors.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.2.4\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.2.4\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.2.4\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.2.4\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.3.1': {'ec_num': '1.2.3.1',\n",
       "  'accepted_name': 'aldehyde oxidase',\n",
       "  'reaction': 'an aldehyde + H2O + O2 = a carboxylate + H2O2',\n",
       "  'other_names': 'quinoline oxidase; retinal oxidase',\n",
       "  'sys_name': 'aldehyde:oxygen oxidoreductase ',\n",
       "  'comments': 'Contains molybdenum, [2Fe-2S] centres and FAD. The enzyme from liver exhibits a broad substrate specificity, and is involved in the metabolism of xenobiotics, including the oxidation of N-heterocycles and aldehydes and the reduction of N-oxides, nitrosamines, hydroxamic acids, azo dyes, nitropolycyclic aromatic hydrocarbons, and sulfoxides [4,6].\\nThe enzyme is also responsible for the oxidation of retinal, an activity that was initially attributed to a distinct enzyme (EC 1.2.3.11, retinal oxidase) [5,7].',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.3.1\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.3.1\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.3.1\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.3.1\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.2.3.1%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.3.3': {'ec_num': '1.2.3.3',\n",
       "  'accepted_name': 'pyruvate oxidase',\n",
       "  'reaction': 'pyruvate + phosphate + O2 = acetyl phosphate + CO2 + H2O2',\n",
       "  'other_names': 'pyruvic oxidase; phosphate-dependent pyruvate oxidase',\n",
       "  'sys_name': 'pyruvate:oxygen 2-oxidoreductase (phosphorylating)',\n",
       "  'comments': 'A flavoprotein (FAD) requiring thiamine diphosphate. Two reducing equivalents are transferred from the resonant carbanion/enamine forms of 2-hydroxyethyl-thiamine-diphosphate to the adjacent flavin cofactor, yielding 2-acetyl-thiamine diphosphate (AcThDP) and reduced flavin. FADH2 is reoxidized by O2 to yield H2O2 and FAD and AcThDP is cleaved phosphorolytically to acetyl phosphate and thiamine diphosphate [2].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'thiamine diphosphate = 3-[(4-amino-2-methylpyrimidin-5-yl)methyl]-5-(2-diphosphoethyl)-4-methyl-1,3-thiazolium',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.3.4': {'ec_num': '1.2.3.4',\n",
       "  'accepted_name': 'oxalate oxidase',\n",
       "  'reaction': 'oxalate + O2 + 2 H+ = 2 CO2 + H2O2',\n",
       "  'other_names': 'aero-oxalo dehydrogenase; oxalic acid oxidase',\n",
       "  'sys_name': 'oxalate:oxygen oxidoreductase',\n",
       "  'comments': 'Contains Mn2+ as a cofactor. The enzyme is not a flavoprotein as had been thought [3].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.3.5': {'ec_num': '1.2.3.5',\n",
       "  'accepted_name': 'glyoxylate oxidase',\n",
       "  'reaction': 'glyoxylate + H2O + O2 = oxalate + H2O2',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'glyoxylate:oxygen oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.3.6': {'ec_num': '1.2.3.6',\n",
       "  'accepted_name': 'pyruvate oxidase (CoA-acetylating)',\n",
       "  'reaction': 'pyruvate + CoA + O2 = acetyl-CoA + CO2 + H2O2',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'pyruvate:oxygen 2-oxidoreductase (CoA-acetylating)',\n",
       "  'comments': 'A flavoprotein (FAD). May be identical with EC 1.2.7.1 pyruvate synthase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.3.7': {'ec_num': '1.2.3.7',\n",
       "  'accepted_name': 'indole-3-acetaldehyde oxidase',\n",
       "  'reaction': '(indol-3-yl)acetaldehyde + H2O + O2 = (indol-3-yl)acetate + H2O2',\n",
       "  'other_names': 'indoleacetaldehyde oxidase; IAAld oxidase; AO1; indole-3-acetaldehyde:oxygen oxidoreductase',\n",
       "  'sys_name': '(indol-3-yl)acetaldehyde:oxygen oxidoreductase',\n",
       "  'comments': 'A hemoprotein. This enzyme is an isoform of aldehyde oxidase (EC 1.2.3.1). It has a preference for aldehydes having an indole-ring structure as substrate [6,7]. It may play a role in plant hormone biosynthesis as its activity is higher in the auxin-overproducing mutant, super-root1, than in wild-type Arabidopsis thaliana [7]. While (indol-3-yl)acetaldehyde is the preferred substrate, it also oxidizes indole-3-carbaldehyde and acetaldehyde, but more slowly. The enzyme from maize contains FAD, iron and molybdenum [4].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.3.8': {'ec_num': '1.2.3.8',\n",
       "  'accepted_name': 'pyridoxal oxidase',\n",
       "  'reaction': 'pyridoxal + H2O + O2 = 4-pyridoxate + (?)',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'pyridoxal:oxygen 4-oxidoreductase',\n",
       "  'comments': 'A molybdenum protein.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.3.8\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.3.8\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.3.8\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.3.8\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.2.3.8%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.3.9': {'ec_num': '1.2.3.9',\n",
       "  'accepted_name': 'aryl-aldehyde oxidase',\n",
       "  'reaction': 'an aromatic aldehyde + O2 + H2O = an aromatic carboxylate + H2O2',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'aryl-aldehyde:oxygen oxidoreductase',\n",
       "  'comments': 'Acts on benzaldehyde, vanillin and a number of other aromatic aldehydes, but not on aliphatic aldehydes or sugars.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.3.11': {'ec_num': '1.2.3.11',\n",
       "  'accepted_name': 'retinal oxidase',\n",
       "  'reaction': 'retinal + O2 + H2O = retinoate + H2O2',\n",
       "  'other_names': 'retinene oxidase',\n",
       "  'sys_name': 'retinal:oxygen oxidoreductase',\n",
       "  'comments': 'May be the same as EC 1.2.3.1, aldehyde oxidase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.3.13': {'ec_num': '1.2.3.13',\n",
       "  'accepted_name': '4-hydroxyphenylpyruvate oxidase',\n",
       "  'reaction': '2 4-hydroxyphenylpyruvate + O2 = 2 4-hydroxyphenylacetate + 2 CO2',\n",
       "  'other_names': '',\n",
       "  'sys_name': '4-hydroxyphenylpyruvate:oxygen oxidoreductase (decarboxylating)',\n",
       "  'comments': 'Involved in tyrosine degradation pathway in Arthrobacter sp.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.3.13\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.3.13\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.3.13\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.3.13\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.3.14': {'ec_num': '1.2.3.14',\n",
       "  'accepted_name': 'abscisic-aldehyde oxidase',\n",
       "  'reaction': 'abscisic aldehyde + H2O + O2 = abscisate + H2O2',\n",
       "  'other_names': 'abscisic aldehyde oxidase; AAO3; AOd; AOδ',\n",
       "  'sys_name': 'abscisic-aldehyde:oxygen oxidoreductase',\n",
       "  'comments': 'Acts on both (+)- and (-)-abscisic aldehyde. Involved in the abscisic-acid biosynthesis pathway in plants, along with EC 1.1.1.288, (xanthoxin dehydrogenase), EC 1.13.11.51 (9-cis-epoxycarotenoid dioxygenase) and EC 1.14.13.93 [(+)-abscisic acid 8′-hydroxylase]. While abscisic aldehyde is the best substrate, the enzyme also acts with indole-3-aldehyde, 1-naphthaldehyde and benzaldehyde as substrates, but more slowly [3].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.4.1': {'ec_num': '1.2.4.1',\n",
       "  'accepted_name': 'pyruvate dehydrogenase (acetyl-transferring)',\n",
       "  'reaction': 'pyruvate + [dihydrolipoyllysine-residue acetyltransferase] lipoyllysine = [dihydrolipoyllysine-residue acetyltransferase] S-acetyldihydrolipoyllysine + CO2',\n",
       "  'other_names': 'pyruvate decarboxylase (ambiguous); pyruvate dehydrogenase (ambiguous); pyruvate dehydrogenase (lipoamide); pyruvate:lipoamide 2-oxidoreductase (decarboxylating and acceptor-acetylating); pyruvic acid dehydrogenase; pyruvic dehydrogenase (ambiguous)',\n",
       "  'sys_name': 'pyruvate:[dihydrolipoyllysine-residue acetyltransferase]-lipoyllysine 2-oxidoreductase (decarboxylating, acceptor-acetylating)',\n",
       "  'comments': 'Contains thiamine diphosphate. It is a component (in multiple copies) of the multienzyme pyruvate dehydrogenase complex, EC 1.2.1.104, in which it is bound to a core of molecules of EC 2.3.1.12, dihydrolipoyllysine-residue acetyltransferase, which also binds multiple copies of EC 1.8.1.4, dihydrolipoyl dehydrogenase. It does not act on free lipoamide or lipoyllysine, but only on the lipoyllysine residue in EC 2.3.1.12.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'dihydrolipoyl group\\nthiamine diphosphate = 3-[(4-amino-2-methylpyrimidin-5-yl)methyl]-5-(2-diphosphoethyl)-4-methyl-1,3-thiazolium',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.4.2': {'ec_num': '1.2.4.2',\n",
       "  'accepted_name': 'oxoglutarate dehydrogenase (succinyl-transferring)',\n",
       "  'reaction': '2-oxoglutarate + [dihydrolipoyllysine-residue succinyltransferase] lipoyllysine = [dihydrolipoyllysine-residue succinyltransferase] S-succinyldihydrolipoyllysine + CO2',\n",
       "  'other_names': '2-ketoglutarate dehydrogenase; 2-oxoglutarate dehydrogenase; 2-oxoglutarate: lipoate oxidoreductase; 2-oxoglutarate:lipoamide 2-oxidoreductase (decarboxylating and acceptor-succinylating); α-ketoglutarate dehydrogenase; alphaketoglutaric acid dehydrogenase; α-ketoglutaric dehydrogenase; α-oxoglutarate dehydrogenase; AKGDH; OGDC; ketoglutaric dehydrogenase; oxoglutarate decarboxylase (misleading); oxoglutarate dehydrogenase; oxoglutarate dehydrogenase (lipoamide)',\n",
       "  'sys_name': '2-oxoglutarate:[dihydrolipoyllysine-residue succinyltransferase]-lipoyllysine 2-oxidoreductase (decarboxylating, acceptor-succinylating)',\n",
       "  'comments': 'Contains thiamine diphosphate. It is a component of the multienzyme 2-oxoglutarate dehydrogenase complex, EC 1.2.1.105, in which multiple copies of it are bound to a core of molecules of EC 2.3.1.61, dihydrolipoyllysine-residue succinyltransferase, which also binds multiple copies of EC 1.8.1.4, dihydrolipoyl dehydrogenase. It does not act on free lipoamide or lipoyllysine, but only on the lipoyllysine residue in EC 2.3.1.61.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'dihydrolipoyl group\\nthiamine diphosphate = 3-[(4-amino-2-methylpyrimidin-5-yl)methyl]-5-(2-diphosphoethyl)-4-methyl-1,3-thiazolium',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.4.4': {'ec_num': '1.2.4.4',\n",
       "  'accepted_name': '3-methyl-2-oxobutanoate dehydrogenase (2-methylpropanoyl-transferring)',\n",
       "  'reaction': '3-methyl-2-oxobutanoate + [dihydrolipoyllysine-residue (2-methylpropanoyl)transferase] lipoyllysine = [dihydrolipoyllysine-residue (2-methylpropanoyl)transferase] S-(2-methylpropanoyl)dihydrolipoyllysine + CO2',\n",
       "  'other_names': '2-oxoisocaproate dehydrogenase; 2-oxoisovalerate (lipoate) dehydrogenase; 3-methyl-2-oxobutanoate dehydrogenase (lipoamide); 3-methyl-2-oxobutanoate:lipoamide oxidoreductase (decarboxylating and acceptor-2-methylpropanoylating); α-keto-α-methylvalerate dehydrogenase; α-ketoisocaproate dehydrogenase; α-ketoisocaproic dehydrogenase; α-ketoisocaproic-α-keto-α-methylvaleric dehydrogenase; α-ketoisovalerate dehydrogenase; α-oxoisocaproate dehydrogenase; BCKDH (ambiguous); BCOAD; branched chain keto acid dehydrogenase; branched-chain (-2-oxoacid) dehydrogenase (BCD); branched-chain 2-keto acid dehydrogenase; branched-chain 2-oxo acid dehydrogenase; branched-chain α-keto acid dehydrogenase; branched-chain α-oxo acid dehydrogenase; branched-chain keto acid dehydrogenase; branched-chain ketoacid dehydrogenase; dehydrogenase, 2-oxoisovalerate (lipoate); dehydrogenase, branched chain α-keto acid',\n",
       "  'sys_name': '3-methyl-2-oxobutanoate:[dihydrolipoyllysine-residue (2-methylpropanoyl)transferase]-lipoyllysine 2-oxidoreductase (decarboxylating, acceptor-2-methylpropanoylating)',\n",
       "  'comments': 'Contains thiamine diphosphate. It acts not only on 3-methyl-2-oxobutanaoate, but also on 4-methyl-2-oxopentanoate and (S)-3-methyl-2-oxopentanoate, so that it acts on the 2-oxo acids that derive from the action of transaminases on valine, leucine and isoleucine. It is a component of the multienzyme 3-methyl-2-oxobutanoate dehydrogenase complex in which multiple copies of it are bound to a core of molecules of EC 2.3.1.168, dihydrolipoyllysine-residue (2-methylpropanoyl)transferase, which also binds multiple copies of EC 1.8.1.4, dihydrolipoyl dehydrogenase. It does not act on free lipoamide or lipoyllysine, but only on the lipoyllysine residue in EC 2.3.1.168.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'dihydrolipoyl group\\nthiamine diphosphate = 3-[(4-amino-2-methylpyrimidin-5-yl)methyl]-5-(2-diphosphoethyl)-4-methyl-1,3-thiazolium',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.7.1': {'ec_num': '1.2.7.1',\n",
       "  'accepted_name': 'pyruvate synthase',\n",
       "  'reaction': 'pyruvate + CoA + 2 oxidized ferredoxin = acetyl-CoA + CO2 + 2 reduced ferredoxin + 2 H+',\n",
       "  'other_names': 'pyruvate oxidoreductase; pyruvate synthetase; pyruvate:ferredoxin oxidoreductase; pyruvic-ferredoxin oxidoreductase; 2-oxobutyrate synthase; α-ketobutyrate-ferredoxin oxidoreductase; 2-ketobutyrate synthase; α-ketobutyrate synthase; 2-oxobutyrate-ferredoxin oxidoreductase; 2-oxobutanoate:ferredoxin 2-oxidoreductase (CoA-propionylating); 2-oxobutanoate:ferredoxin 2-oxidoreductase (CoA-propanoylating)',\n",
       "  'sys_name': 'pyruvate:ferredoxin 2-oxidoreductase (CoA-acetylating)',\n",
       "  'comments': 'Contains thiamine diphosphate and [4Fe-4S] clusters. The enzyme also decarboxylates 2-oxobutyrate with lower efficiency, but shows no activity with 2-oxoglutarate. This enzyme is a member of the 2-oxoacid oxidoreductases, a family of enzymes that oxidatively decarboxylate different 2-oxoacids to form their CoA derivatives, and are differentiated based on their substrate specificity. For examples of other members of this family, see EC 1.2.7.3, 2-oxoglutarate synthase and EC 1.2.7.7, 3-methyl-2-oxobutanoate dehydrogenase (ferredoxin).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.7.2': {'ec_num': '1.2.7.2',\n",
       "  'accepted_name': '2-oxobutyrate synthase',\n",
       "  'reaction': '2-oxobutanoate + CoA + 2 oxidized ferredoxin = propanoyl-CoA + CO2 + 2 reduced ferredoxin + H+',\n",
       "  'other_names': 'α-ketobutyrate-ferredoxin oxidoreductase; 2-ketobutyrate synthase; α-ketobutyrate synthase; 2-oxobutyrate-ferredoxin oxidoreductase; 2-oxobutanoate:ferredoxin 2-oxidoreductase (CoA-propionylating)',\n",
       "  'sys_name': '2-oxobutanoate:ferredoxin 2-oxidoreductase (CoA-propanoylating)',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.7.2\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.7.2\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.7.2\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.7.2\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.7.3': {'ec_num': '1.2.7.3',\n",
       "  'accepted_name': '2-oxoglutarate synthase',\n",
       "  'reaction': '2-oxoglutarate + CoA + 2 oxidized ferredoxin = succinyl-CoA + CO2 + 2 reduced ferredoxin + 2 H+',\n",
       "  'other_names': '2-ketoglutarate ferredoxin oxidoreductase; 2-oxoglutarate:ferredoxin oxidoreductase; KGOR; 2-oxoglutarate ferredoxin oxidoreductase; 2-oxoglutarate:ferredoxin 2-oxidoreductase (CoA-succinylating)',\n",
       "  'sys_name': '2-oxoglutarate:ferredoxin oxidoreductase (decarboxylating)',\n",
       "  'comments': 'The enzyme contains thiamine diphosphate and two [4Fe-4S] clusters. Highly specific for 2-oxoglutarate. This enzyme is a member of the 2-oxoacid oxidoreductases, a family of enzymes that oxidatively decarboxylate different 2-oxoacids to form their CoA derivatives, and are differentiated based on their substrate specificity. For examples of other members of this family, see EC 1.2.7.1, pyruvate synthase and EC 1.2.7.7, 3-methyl-2-oxobutanoate dehydrogenase (ferredoxin).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.7.4': {'ec_num': '1.2.7.4',\n",
       "  'accepted_name': 'anaerobic carbon monoxide dehydrogenase',\n",
       "  'reaction': 'CO + H2O + 2 oxidized ferredoxin = CO2 + 2 reduced ferredoxin + 2 H+',\n",
       "  'other_names': 'Ni-CODH; carbon-monoxide dehydrogenase (ferredoxin)',\n",
       "  'sys_name': 'carbon-monoxide,water:ferredoxin oxidoreductase',\n",
       "  'comments': 'This prokaryotic enzyme catalyses the reversible reduction of CO2 to CO. The electrons are transferred to redox proteins such as ferredoxin. In purple sulfur bacteria and methanogenic archaea it catalyses the oxidation of CO to CO2, which is incorporated by the Calvin-Benson-Basham cycle or released, respectively. In acetogenic and sulfate-reducing microbes it catalyses the reduction of CO2 to CO, which is incorporated into acetyl CoA by EC 2.3.1.169, CO-methylating acetyl-CoA synthase, with which the enzyme forms a tight complex in those organisms. The enzyme contains five metal clusters per homodimeric enzyme: two nickel-iron-sulfur clusters called the C-Clusters, one [4Fe-4S] D-cluster; and two [4Fe-4S] B-clusters. In methanogenic archaea additional [4Fe-4S] clusters exist, presumably as part of the electron transfer chain. In purple sulfur bacteria the enzyme forms complexes with the Ni-Fe-S protein EC 1.12.7.2, ferredoxin hydrogenase, which catalyse the overall reaction: CO + H2O = CO2 + H2. cf. EC 1.2.5.3, aerobic carbon monoxide dehydrogenase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.7.5': {'ec_num': '1.2.7.5',\n",
       "  'accepted_name': 'aldehyde ferredoxin oxidoreductase',\n",
       "  'reaction': 'an aldehyde + H2O + 2 oxidized ferredoxin = a carboxylate + 2 H+ + 2 reduced ferredoxin',\n",
       "  'other_names': 'AOR',\n",
       "  'sys_name': 'aldehyde:ferredoxin oxidoreductase',\n",
       "  'comments': 'This is an oxygen-sensitive enzyme that contains tungsten-molybdopterin and iron-sulfur clusters. Catalyses the oxidation of aldehydes (including crotonaldehyde, acetaldehyde, formaldehyde and glyceraldehyde) to their corresponding acids. However, it does not oxidize glyceraldehyde 3-phosphate [see EC 1.2.7.6, glyceraldehyde-3-phosphate dehydrogenase (ferredoxin)]. Can use ferredoxin or methyl viologen but not NAD(P)+ as electron acceptor.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.7.6': {'ec_num': '1.2.7.6',\n",
       "  'accepted_name': 'glyceraldehyde-3-phosphate dehydrogenase (ferredoxin)',\n",
       "  'reaction': 'D-glyceraldehyde-3-phosphate + H2O + 2 oxidized ferredoxin = 3-phospho-D-glycerate + 2  H+ + 2 reduced ferredoxin',\n",
       "  'other_names': 'GAPOR; glyceraldehyde-3-phosphate Fd oxidoreductase; glyceraldehyde-3-phosphate ferredoxin reductase',\n",
       "  'sys_name': 'D-glyceraldehyde-3-phosphate:ferredoxin oxidoreductase',\n",
       "  'comments': 'Contains tungsten-molybdopterin and iron-sulfur clusters. This enzyme is thought to function in place of glyceralde-3-phosphate dehydrogenase and possibly phosphoglycerate kinase in the novel Embden-Meyerhof-type glycolytic pathway found in Pyrococcus furiosus [1]. It is specific for glyceraldehyde-3-phosphate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.7.7': {'ec_num': '1.2.7.7',\n",
       "  'accepted_name': '3-methyl-2-oxobutanoate dehydrogenase (ferredoxin)',\n",
       "  'reaction': '3-methyl-2-oxobutanoate + CoA + 2 oxidized ferredoxin = S-(2-methylpropanoyl)-CoA + CO2 + 2 reduced ferredoxin + H+',\n",
       "  'other_names': '2-ketoisovalerate ferredoxin reductase; 3-methyl-2-oxobutanoate synthase (ferredoxin); VOR; branched-chain ketoacid ferredoxin reductase; branched-chain oxo acid ferredoxin reductase; keto-valine-ferredoxin oxidoreductase; ketoisovalerate ferredoxin reductase; 2-oxoisovalerate ferredoxin reductase',\n",
       "  'sys_name': '3-methyl-2-oxobutanoate:ferredoxin oxidoreductase (decarboxylating; CoA-2-methylpropanoylating)',\n",
       "  'comments': 'The enzyme is CoA-dependent and contains thiamine diphosphate and iron-sulfur clusters. Preferentially utilizes 2-oxo-acid derivatives of branched chain amino acids, e.g. 3-methyl-2-oxopentanoate, 4-methyl-2-oxo-pentanoate, and 2-oxobutanoate. This enzyme is a member of the 2-oxoacid oxidoreductases, a family of enzymes that reversibly catalyse the oxidative decarboxylation of different 2-oxoacids to form their CoA derivatives, and are differentiated based on their substrate specificity. For examples of other members of this family, see EC 1.2.7.1, pyruvate synthase, and EC 1.2.7.3, 2-oxoglutarate synthase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.7.8': {'ec_num': '1.2.7.8',\n",
       "  'accepted_name': 'indolepyruvate ferredoxin oxidoreductase',\n",
       "  'reaction': '(indol-3-yl)pyruvate + CoA + 2 oxidized ferredoxin = S-2-(indol-3-yl)acetyl-CoA + CO2 + 2 reduced ferredoxin + H+',\n",
       "  'other_names': '3-(indol-3-yl)pyruvate synthase (ferredoxin); IOR ',\n",
       "  'sys_name': '3-(indol-3-yl)pyruvate:ferredoxin oxidoreductase (decarboxylating, CoA-indole-acetylating)',\n",
       "  'comments': 'Contains thiamine diphosphate and [4Fe-4S] clusters. Preferentially utilizes the transaminated forms of aromatic amino acids and can use phenylpyruvate and p-hydroxyphenylpyruvate as substrates. This enzyme, which is found in archaea, is a member of the 2-oxoacid oxidoreductases, a family of enzymes that oxidatively decarboxylate different 2-oxoacids to form their CoA derivatives, and are differentiated based on their substrate specificity. For examples of other members of this family, see EC 1.2.7.3, 2-oxoglutarate synthase and EC 1.2.7.7, 3-methyl-2-oxobutanoate dehydrogenase (ferredoxin).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.7.9': {'ec_num': '1.2.7.9',\n",
       "  'accepted_name': '',\n",
       "  'reaction': '',\n",
       "  'other_names': '',\n",
       "  'sys_name': '',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.99.2': {'ec_num': '1.2.99.2',\n",
       "  'accepted_name': 'carbon-monoxide dehydrogenase (acceptor)',\n",
       "  'reaction': 'CO + H2O + A = CO2 + AH2',\n",
       "  'other_names': 'anaerobic carbon monoxide dehydrogenase; carbon monoxide oxygenase; carbon-monoxide dehydrogenase; carbon-monoxide:(acceptor) oxidoreductase',\n",
       "  'sys_name': 'carbon-monoxide:acceptor oxidoreductase',\n",
       "  'comments': 'Contains a [Ni3Fe-4S] cluster and [4Fe-4S] clusters. It uses many electron acceptors, including ferredoxin, methyl viologen and benzyl viologen and flavins, but not pyridine nucleotides. Forms part of a membrane-bound multienzyme complex with EC 1.12.99.6, hydrogenase (acceptor), which catalyses the overall reaction: CO + H2O = CO2 + H2.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.99.2\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.99.2\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.99.2\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.99.2\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.2.99.2%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.99.3': {'ec_num': '1.2.99.3',\n",
       "  'accepted_name': 'aldehyde dehydrogenase (pyrroloquinoline-quinone)',\n",
       "  'reaction': 'an aldehyde + acceptor + H2O = a carboxylate + reduced acceptor',\n",
       "  'other_names': 'aldehyde dehydrogenase (acceptor)',\n",
       "  'sys_name': 'aldehyde:(pyrroloquinoline-quinone) oxidoreductase',\n",
       "  'comments': 'A quinoprotein. Wide specificity; acts on straight-chain aldehydes up to C10, aromatic aldehydes, glyoxylate and glyceraldehyde.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.99.4': {'ec_num': '1.2.99.4',\n",
       "  'accepted_name': 'formaldehyde dismutase',\n",
       "  'reaction': '2 formaldehyde + H2O = formate + methanol',\n",
       "  'other_names': 'aldehyde dismutase; cannizzanase; nicotinoprotein aldehyde dismutase',\n",
       "  'sys_name': 'formaldehyde:formaldehyde oxidoreductase',\n",
       "  'comments': 'The enzyme contains a tightly but noncovalently bound NADP(H) cofactor, as well as Zn2+ and Mg2+. The enzyme from Mycobacterium sp. DSM 3803 also catalyses the reactions of EC 1.1.99.36, NDMA-dependent alcohol dehydrogenase and EC 1.1.99.37, NDMA-dependent methanol dehydrogenase [3]. Formaldehyde and acetaldehyde can act as donors; formaldehyde, acetaldehyde and propanal can act as acceptors [1,2].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.99.5': {'ec_num': '1.2.99.5',\n",
       "  'accepted_name': 'formylmethanofuran dehydrogenase',\n",
       "  'reaction': 'formylmethanofuran + H2O + acceptor = CO2 + methanofuran + reduced acceptor',\n",
       "  'other_names': 'formylmethanofuran:(acceptor) oxidoreductase',\n",
       "  'sys_name': 'formylmethanofuran:acceptor oxidoreductase',\n",
       "  'comments': 'A molybdoprotein containing a pterin cofactor. Tungstate can substitute for molybdate. The enzyme catalyses a reversible reaction in methanogenic bacteria, and is involved in methanogenesis from CO2 as well as the oxidation of methyl-coenzyme M to CO2. Methylviologen can act as acceptor. Also oxidizes N-furfurylformamide.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.2.99.5\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.2.99.5\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.2.99.5\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.2.99.5\" target=\"new\">KEGG</a>',\n",
       "  'glossary': 'methanofuran = 4-[4-(2-{[(4R*,5S*)-4,5,7-tricarboxyheptanoyl]-γ-L-glutamyl-γ-L-glutamylamino}ethyl)phenoxymethyl]furfurylamine',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.99.6': {'ec_num': '1.2.99.6',\n",
       "  'accepted_name': 'carboxylate reductase',\n",
       "  'reaction': 'an aldehyde + acceptor + H2O = a carboxylate + reduced acceptor',\n",
       "  'other_names': 'aldehyde:(acceptor) oxidoreductase',\n",
       "  'sys_name': 'aldehyde:acceptor oxidoreductase',\n",
       "  'comments': 'A tungsten protein. Methyl viologen can act as acceptor. In the reverse direction, non-activated acids are reduced by reduced viologens to aldehydes, but not to the corresponding alcohols.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.2.99.7': {'ec_num': '1.2.99.7',\n",
       "  'accepted_name': 'aldehyde dehydrogenase (FAD-independent)',\n",
       "  'reaction': 'an aldehyde + H2O + acceptor = a carboxylate + reduced acceptor',\n",
       "  'other_names': 'aldehyde oxidase; aldehyde oxidoreductase; Mop; AORDd',\n",
       "  'sys_name': 'aldehyde:acceptor oxidoreductase (FAD-independent)',\n",
       "  'comments': 'Belongs to the xanthine oxidase family of enzymes. The enzyme from Desulfovibrio sp. contains a molybdenum-molybdopterin-cytosine dinucleotide (MCD) complex and two types of [2Fe-2S] cluster per monomer, but does not contain FAD.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.1': {'ec_num': '1.3.1.1',\n",
       "  'accepted_name': 'dihydropyrimidine dehydrogenase (NAD+)',\n",
       "  'reaction': '(1) 5,6-dihydrouracil + NAD+ = uracil + NADH + H+;;(2) 5,6-dihydrothymine + NAD+ = thymine + NADH + H+',\n",
       "  'other_names': 'dihydropyrimidine dehydrogenase; dihydrothymine dehydrogenase; pyrimidine reductase; thymine reductase; uracil reductase; dihydrouracil dehydrogenase (NAD+)',\n",
       "  'sys_name': '5,6-dihydropyrimidine:NAD+ oxidoreductase',\n",
       "  'comments': 'An iron-sulfur flavoenzyme. The enzyme was originally discovered in the uracil-fermenting bacterium, Clostridium uracilicum, which utilizes uracil and thymine as nitrogen and carbon sources for growth [1]. Since then the enzyme was found in additional organisms including Alcaligenes eutrophus [2], Pseudomonas strains [3,4] and Escherichia coli [5,6].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.2': {'ec_num': '1.3.1.2',\n",
       "  'accepted_name': 'dihydropyrimidine dehydrogenase (NADP+)',\n",
       "  'reaction': '5,6-dihydrouracil + NADP+ = uracil + NADPH + H+',\n",
       "  'other_names': 'dihydrothymine dehydrogenase; dihydrouracil dehydrogenase (NADP+); 4,5-dihydrothymine: oxidoreductase; DPD; DHPDH; dehydrogenase, dihydrouracil (nicotinamide adenine dinucleotide phosphate); DHU dehydrogenase; hydropyrimidine dehydrogenase; dihydropyrimidine dehydrogenase (NADP)',\n",
       "  'sys_name': '5,6-dihydrouracil:NADP+ 5-oxidoreductase',\n",
       "  'comments': 'Also acts on dihydrothymine.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.3': {'ec_num': '1.3.1.3',\n",
       "  'accepted_name': 'Δ4-3-oxosteroid 5β-reductase',\n",
       "  'reaction': 'a 3-oxo-5β-steroid + NADP+ = a 3-oxo-Δ4-steroid + NADPH + H+',\n",
       "  'other_names': '3-oxo-Δ4-steroid 5β-reductase; 5β-reductase; androstenedione 5β-reductase; cholestenone 5β-reductase; cortisone 5β-reductase; cortisone β-reductase; cortisone Δ4-5β-reductase; steroid 5β-reductase; testosterone 5β-reductase; Δ4-3-ketosteroid 5β-reductase; Δ4-5β-reductase; Δ4-hydrogenase; 4,5β-dihydrocortisone:NADP+ Δ4-oxidoreductase; 3-oxo-5β-steroid:NADP+ Δ4-oxidoreductase; 5β-cholestan-3-one:NADP+ 4,5-oxidoreductase',\n",
       "  'sys_name': '3-oxo-5β-steroid:NADP+ 4,5-oxidoreductase',\n",
       "  'comments': 'The enzyme from human efficiently catalyses the reduction of progesterone, androstenedione, 17α-hydroxyprogesterone and testosterone to 5β-reduced metabolites; it can also act on aldosterone, corticosterone and cortisol, but to a lesser extent [8]. The bile acid intermediates 7α,12α-dihydroxy-4-cholesten-3-one and 7α-hydroxy-4-cholesten-3-one can also act as substrates [9].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-05-29 09:17:40'},\n",
       " '1.3.1.4': {'ec_num': '1.3.1.4',\n",
       "  'accepted_name': 'cortisone α-reductase',\n",
       "  'reaction': '4,5α-dihydrocortisone + NADP+ = cortisone + NADPH + H+',\n",
       "  'other_names': 'cortisone Δ4-5α-reductase; microsomal steroid reductase (5α); Δ4-3-ketosteroid reductase (5α); Δ4-3-oxosteroid-5α-reductase; NADPH:Δ4-3-oxosteroid-5α-oxidoreductase; Δ4-5α-reductase',\n",
       "  'sys_name': '4,5α-dihydrocortisone:NADP+ Δ4-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.5': {'ec_num': '1.3.1.5',\n",
       "  'accepted_name': 'cucurbitacin Δ23-reductase',\n",
       "  'reaction': '23,24-dihydrocucurbitacin B + NAD(P)+ = cucurbitacin B + NAD(P)H + H+',\n",
       "  'other_names': 'NAD(P)H: cucurbitacin B Δ23-oxidoreductase',\n",
       "  'sys_name': '23,24-dihydrocucurbitacin:NAD(P)+ Δ23-oxidoreductase',\n",
       "  'comments': 'Requires Mn2+. Fe2+ or Zn2+ can replace Mn2+ to some extent.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.6': {'ec_num': '1.3.1.6',\n",
       "  'accepted_name': 'fumarate reductase (NADH)',\n",
       "  'reaction': 'succinate + NAD+ = fumarate + NADH + H+',\n",
       "  'other_names': 'NADH-fumarate reductase; NADH-dependent fumarate reductase; fumarate reductase (NADH2)',\n",
       "  'sys_name': 'succinate:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.7': {'ec_num': '1.3.1.7',\n",
       "  'accepted_name': 'meso-tartrate dehydrogenase',\n",
       "  'reaction': 'meso-tartrate + NAD+ = dihydroxyfumarate + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'meso-tartrate:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.8': {'ec_num': '1.3.1.8',\n",
       "  'accepted_name': 'acyl-CoA dehydrogenase (NADP+)',\n",
       "  'reaction': 'acyl-CoA + NADP+ = 2,3-dehydroacyl-CoA + NADPH + H+',\n",
       "  'other_names': '2-enoyl-CoA reductase; dehydrogenase, acyl coenzyme A (nicotinamide adenine dinucleotide phosphate); enoyl coenzyme A reductase; crotonyl coenzyme A reductase; crotonyl-CoA reductase; acyl-CoA dehydrogenase (NADP+)',\n",
       "  'sys_name': 'acyl-CoA:NADP+ 2-oxidoreductase',\n",
       "  'comments': 'The liver enzyme acts on enoyl-CoA derivatives of carbon chain length 4 to 16, with optimum activity on 2-hexenoyl-CoA. In Escherichia coli, cis-specific and trans-specific enzymes exist [EC 1.3.1.37 cis-2-enoyl-CoA reductase (NADPH) and EC 1.3.1.38 trans-2-enoyl-CoA reductase (NADPH)].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.9': {'ec_num': '1.3.1.9',\n",
       "  'accepted_name': 'enoyl-[acyl-carrier-protein] reductase (NADH)',\n",
       "  'reaction': 'an acyl-[acyl-carrier protein] + NAD+ = a trans-2,3-dehydroacyl-[acyl-carrier protein] + NADH + H+',\n",
       "  'other_names': 'enoyl-[acyl carrier protein] reductase; enoyl-ACP reductase; NADH-enoyl acyl carrier protein reductase; NADH-specific enoyl-ACP reductase; acyl-[acyl-carrier-protein]:NAD+ oxidoreductase; fabI (gene name)',\n",
       "  'sys_name': 'acyl-[acyl-carrier protein]:NAD+ oxidoreductase',\n",
       "  'comments': 'The enzyme catalyses an essential step in fatty acid biosynthesis, the reduction of the 2,3-double bond in enoyl-acyl-[acyl-carrier-protein] derivatives of the elongating fatty acid moiety. The enzyme from the bacterium Escherichia coli accepts substrates with carbon chain length from 4 to 18 [3]. The FAS-I enzyme from the bacterium Mycobacterium tuberculosis prefers substrates with carbon chain length from 12 to 24 carbons.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.10': {'ec_num': '1.3.1.10',\n",
       "  'accepted_name': 'enoyl-[acyl-carrier-protein] reductase (NADPH, Si-specific)',\n",
       "  'reaction': 'an acyl-[acyl-carrier protein] + NADP+ = a trans-2,3-dehydroacyl-[acyl-carrier protein] + NADPH + H+',\n",
       "  'other_names': 'acyl-ACP dehydrogenase (ambiguous); enoyl-[acyl carrier protein] (reduced nicotinamide adenine dinucleotide phosphate) reductase; NADPH 2-enoyl Co A reductase; enoyl acyl-carrier-protein reductase (ambiguous); enoyl-ACP reductase (ambiguous); acyl-[acyl-carrier-protein]:NADP+ oxidoreductase (B-specific); acyl-[acyl-carrier protein]:NADP+ oxidoreductase (B-specific); enoyl-[acyl-carrier-protein] reductase (NADPH, B-specific)',\n",
       "  'sys_name': 'acyl-[acyl-carrier protein]:NADP+ oxidoreductase (Si-specific)',\n",
       "  'comments': 'One of the activities of EC 2.3.1.86, fatty-acyl-CoA synthase system, an enzyme found in yeasts (Ascomycota and Basidiomycota). Catalyses the reduction of enoyl-acyl-[acyl-carrier protein] derivatives of carbon chain length from 4 to 16. The yeast enzyme is Si-specific with respect to NADP+. cf. EC 1.3.1.39, enoyl-[acyl-carrier-protein] reductase (NADPH, Re-specific) and EC 1.3.1.104, enoyl-[acyl-carrier-protein] reductase (NADPH), which describes enzymes whose stereo-specificity towards NADPH is not known. See also EC 1.3.1.9, enoyl-[acyl-carrier-protein] reductase (NADH).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.11': {'ec_num': '1.3.1.11',\n",
       "  'accepted_name': '2-coumarate reductase',\n",
       "  'reaction': '3-(2-hydroxyphenyl)propanoate + NAD+ = 2-coumarate + NADH + H+',\n",
       "  'other_names': 'melilotate dehydrogenase',\n",
       "  'sys_name': '3-(2-hydroxyphenyl)propanoate:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.12': {'ec_num': '1.3.1.12',\n",
       "  'accepted_name': 'prephenate dehydrogenase',\n",
       "  'reaction': 'prephenate + NAD+ = 4-hydroxyphenylpyruvate + CO2 + NADH',\n",
       "  'other_names': 'hydroxyphenylpyruvate synthase; chorismate mutase—prephenate dehydrogenase',\n",
       "  'sys_name': 'prephenate:NAD+ oxidoreductase (decarboxylating)',\n",
       "  'comments': 'This enzyme in the enteric bacteria also possesses chorismate mutase activity (EC 5.4.99.5 chorismate mutase) and converts chorismate into prephenate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.13': {'ec_num': '1.3.1.13',\n",
       "  'accepted_name': 'prephenate dehydrogenase (NADP+)',\n",
       "  'reaction': 'prephenate + NADP+ = 4-hydroxyphenylpyruvate + CO2 + NADPH',\n",
       "  'other_names': 'prephenate dehydrogenase (ambiguous); prephenate (nicotinamide adenine dinucleotide phosphate) dehydrogenase; prephenate dehydrogenase (NADP)',\n",
       "  'sys_name': 'prephenate:NADP+ oxidoreductase (decarboxylating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.14': {'ec_num': '1.3.1.14',\n",
       "  'accepted_name': 'dihydroorotate dehydrogenase (NAD+)',\n",
       "  'reaction': '(S)-dihydroorotate + NAD+ = orotate + NADH + H+',\n",
       "  'other_names': 'orotate reductase (NADH); orotate reductase (NADH2); DHOdehase (ambiguous); DHOD (ambiguous); DHODase (ambiguous); dihydroorotate oxidase, pyrD (gene name)',\n",
       "  'sys_name': '(S)-dihydroorotate:NAD+ oxidoreductase',\n",
       "  'comments': 'Binds FMN, FAD and a [2Fe-2S] cluster. The enzyme consists of two subunits, an FMN binding catalytic subunit and a FAD and iron-sulfur binding electron transfer subunit [4]. The reaction, which takes place in the cytosol, is the only redox reaction in the de-novo biosynthesis of pyrimidine nucleotides. Other class 1 dihydroorotate dehydrogenases use either fumarate (EC 1.3.98.1) or NADP+ (EC 1.3.1.15) as electron acceptor. The membrane bound class 2 dihydroorotate dehydrogenase (EC 1.3.5.2) uses quinone as electron acceptor.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.15': {'ec_num': '1.3.1.15',\n",
       "  'accepted_name': 'dihydroorotate dehydrogenase (NADP+)',\n",
       "  'reaction': '(S)-dihydroorotate + NADP+ = orotate + NADPH + H+',\n",
       "  'other_names': 'orotate reductase; dihydro-orotic dehydrogenase; L-5,6-dihydro-orotate:NAD+ oxidoreductase; orotate reductase (NADPH)',\n",
       "  'sys_name': '(S)-dihydroorotate:NADP+ oxidoreductase',\n",
       "  'comments': 'Binds FMN and FAD [2]. Other class 1 dihydroorotate dehydrogenases use either fumarate (EC 1.3.98.1) or NAD+ (EC 1.3.1.14) as electron acceptor. The membrane bound class 2 dihydroorotate dehydrogenase (EC 1.3.5.2) uses quinone as electron acceptor .',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.16': {'ec_num': '1.3.1.16',\n",
       "  'accepted_name': 'β-nitroacrylate reductase',\n",
       "  'reaction': '3-nitropropanoate + NADP+ = 3-nitroacrylate + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '3-nitropropanoate:NADP+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.17': {'ec_num': '1.3.1.17',\n",
       "  'accepted_name': '3-methyleneoxindole reductase',\n",
       "  'reaction': '3-methyl-1,3-dihydroindol-2-one + NADP+ = 3-methylene-1,3-dihydro-2H-indol-2-one + NADPH + H+',\n",
       "  'other_names': '3-methyloxindole:NADP+ oxidoreductase',\n",
       "  'sys_name': '3-methyl-1,3-dihydroindol-2-one:NADP+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '3-methyloxindole = 3-methylindolin-2-one = 3-methyl-1,3-dihydroindol-2-one',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.18': {'ec_num': '1.3.1.18',\n",
       "  'accepted_name': 'kynurenate-7,8-dihydrodiol dehydrogenase',\n",
       "  'reaction': '7,8-dihydro-7,8-dihydroxykynurenate + NAD+ = 7,8-dihydroxykynurenate + NADH + H+',\n",
       "  'other_names': '7,8-dihydro-7,8-dihydroxykynurenate dehydrogenase; 7,8-dihydroxykynurenic acid 7,8-diol dehydrogenase',\n",
       "  'sys_name': '7,8-dihydro-7,8-dihydroxykynurenate:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.19': {'ec_num': '1.3.1.19',\n",
       "  'accepted_name': 'cis-1,2-dihydrobenzene-1,2-diol dehydrogenase',\n",
       "  'reaction': 'cis-1,2-dihydrobenzene-1,2-diol + NAD+ = catechol + NADH + H+',\n",
       "  'other_names': 'cis-benzene glycol dehydrogenase; cis-1,2-dihydrocyclohexa-3,5-diene (nicotinamide adenine dinucleotide) oxidoreductase;',\n",
       "  'sys_name': 'cis-1,2-dihydrobenzene-1,2-diol:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.1.19\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.1.19\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.1.19\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.1.19\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.20': {'ec_num': '1.3.1.20',\n",
       "  'accepted_name': 'trans-1,2-dihydrobenzene-1,2-diol dehydrogenase',\n",
       "  'reaction': 'trans-1,2-dihydrobenzene-1,2-diol + NADP+ = catechol + NADPH + H+',\n",
       "  'other_names': 'dihydrodiol dehydrogenase',\n",
       "  'sys_name': 'trans-1,2-dihydrobenzene-1,2-diol:NADP+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.21': {'ec_num': '1.3.1.21',\n",
       "  'accepted_name': '7-dehydrocholesterol reductase',\n",
       "  'reaction': 'cholesterol + NADP+ = cholesta-5,7-dien-3β-ol + NADPH + H+',\n",
       "  'other_names': 'DHCR7 (gene name); 7-DHC reductase; 7-dehydrocholesterol dehydrogenase/cholesterol oxidase; Δ7-sterol reductase',\n",
       "  'sys_name': 'cholesterol:NADP+ Δ7-oxidoreductase',\n",
       "  'comments': 'The enzyme is part of the cholesterol biosynthesis pathway.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.22': {'ec_num': '1.3.1.22',\n",
       "  'accepted_name': '3-oxo-5α-steroid 4-dehydrogenase (NADP+)',\n",
       "  'reaction': 'a 3-oxo-5α-steroid + NADP+ = a 3-oxo-Δ4-steroid + NADPH + H+',\n",
       "  'other_names': 'cholestenone 5α-reductase; testosterone Δ4-5α-reductase; steroid 5α-reductase; 3-oxosteroid Δ4-dehydrogenase; 5α-reductase; steroid 5α-hydrogenase; 3-oxosteroid 5α-reductase; testosterone Δ4-hydrogenase; 4-ene-3-oxosteroid 5α-reductase; reduced nicotinamide adenine dinucleotide phosphate:Δ4-3-ketosteroid 5α-oxidoreductase; 4-ene-5α-reductase; Δ4-3-ketosteroid 5α-oxidoreductase; cholest-4-en-3-one 5α-reductase; testosterone 5α-reductase; 3-oxo-5α-steroid 4-dehydrogenase',\n",
       "  'sys_name': '3-oxo-5α-steroid:NADP+ Δ4-oxidoreductase',\n",
       "  'comments': 'The enzyme catalyses the conversion of assorted 3-oxo-Δ4 steroids into their corresponding 5α form. Substrates for the mammalian enzyme include testosterone, progesterone, and corticosterone. Substrates for the plant enzyme are brassinosteroids such as campest-4-en-3-one and (22α)-hydroxy-campest-4-en-3-one. cf. EC 1.3.99.5, 3-oxo-5α-steroid 4-dehydrogenase (acceptor).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.24': {'ec_num': '1.3.1.24',\n",
       "  'accepted_name': 'biliverdin reductase',\n",
       "  'reaction': 'bilirubin + NAD(P)+ = biliverdin + NAD(P)H + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'bilirubin:NAD(P)+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.25': {'ec_num': '1.3.1.25',\n",
       "  'accepted_name': '1,6-dihydroxycyclohexa-2,4-diene-1-carboxylate dehydrogenase',\n",
       "  'reaction': '(1R,6S)-1,6-dihydroxycyclohexa-2,4-diene-1-carboxylate + NAD+ = catechol + CO2 + NADH + H+',\n",
       "  'other_names': '3,5-cyclohexadiene-1,2-diol-1-carboxylate dehydrogenase; 3,5-cyclohexadiene-1,2-diol-1-carboxylic acid dehydrogenase; dihydrodihydroxybenzoate dehydrogenase; DHBDH; cis-1,2-dihydroxycyclohexa-3,5-diene-1-carboxylate dehydrogenase; 2-hydro-1,2-dihydroxybenzoate dehydrogenase; cis-1,2-dihydroxycyclohexa-3,5-diene-1-carboxylate:NAD+ oxidoreductase; dihydrodihydroxybenzoate dehydrogenase; (1R,6R)-1,6-dihydroxycyclohexa-2,4-diene-1-carboxylate:NAD+ oxidoreductase (decarboxylating)',\n",
       "  'sys_name': '(1R,6S)-1,6-dihydroxycyclohexa-2,4-diene-1-carboxylate:NAD+ oxidoreductase (decarboxylating)',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.1.25\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.1.25\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.1.25\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.1.25\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.26': {'ec_num': '1.3.1.26',\n",
       "  'accepted_name': 'dihydrodipicolinate reductase',\n",
       "  'reaction': '(S)-2,3,4,5-tetrahydropyridine-2,6-dicarboxylate + NAD(P)+ = (S)-2,3-dihydrodipicolinate + NAD(P)H + H+',\n",
       "  'other_names': 'dihydrodipicolinic acid reductase; 2,3,4,5-tetrahydrodipicolinate:NAD(P)+ oxidoreductase',\n",
       "  'sys_name': '(S)-2,3,4,5-tetrahydropyridine-2,6-dicarboxylate:NAD(P)+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.27': {'ec_num': '1.3.1.27',\n",
       "  'accepted_name': '2-hexadecenal reductase',\n",
       "  'reaction': 'hexadecanal + NADP+ = 2-trans-hexadecenal + NADPH + H+',\n",
       "  'other_names': '2-alkenal reductase; hexadecanal: NADP+ oxidoreductase',\n",
       "  'sys_name': 'hexadecanal:NADP+ Δ2-oxidoreductase',\n",
       "  'comments': 'Specific for long chain 2-trans- and 2-cis-alkenals, with chain length optimum around 14 to 16 carbon atoms.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.28': {'ec_num': '1.3.1.28',\n",
       "  'accepted_name': '2,3-dihydro-2,3-dihydroxybenzoate dehydrogenase',\n",
       "  'reaction': '(2S,3S)-2,3-dihydro-2,3-dihydroxybenzoate + NAD+ = 2,3-dihydroxybenzoate + NADH + H+',\n",
       "  'other_names': '2,3-DHB dehydrogenase; 2,3-dihydro-2,3-dihydroxybenzoate:NAD+ oxidoreductase',\n",
       "  'sys_name': '(2S,3S)-2,3-dihydro-2,3-dihydroxybenzoate:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.29': {'ec_num': '1.3.1.29',\n",
       "  'accepted_name': 'cis-1,2-dihydro-1,2-dihydroxynaphthalene dehydrogenase',\n",
       "  'reaction': '(1R,2S)-1,2-dihydronaphthalene-1,2-diol + NAD+ = naphthalene-1,2-diol + NADH + H+',\n",
       "  'other_names': '(+)-cis-naphthalene dihydrodiol dehydrogenase; naphthalene dihydrodiol dehydrogenase; cis-dihydrodiol naphthalene dehydrogenase; cis-1,2-dihydronaphthalene-1,2-diol:NAD+ 1,2-oxidoreductase',\n",
       "  'sys_name': '(1R,2S)-1,2-dihydronaphthalene-1,2-diol:NAD+ 1,2-oxidoreductase',\n",
       "  'comments': 'Also acts, at half the rate, on cis-anthracene dihydrodiol and cis-phenanthrene dihydrodiol.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.1.29\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.1.29\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.1.29\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.1.29\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.30': {'ec_num': '1.3.1.30',\n",
       "  'accepted_name': 'progesterone 5α-reductase',\n",
       "  'reaction': '5α-pregnan-3,20-dione + NADP+ = progesterone + NADPH + H+',\n",
       "  'other_names': 'steroid 5-α-reductase; Δ4-steroid 5α-reductase (progesterone)',\n",
       "  'sys_name': '5α-pregnan-3,20-dione:NADP+ 5-oxidoreductase',\n",
       "  'comments': 'Testosterone and 20α-hydroxy-4-pregn-3-one can act in place of progesterone.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.31': {'ec_num': '1.3.1.31',\n",
       "  'accepted_name': '2-enoate reductase',\n",
       "  'reaction': 'butanoate + NAD+ = but-2-enoate + NADH + H+',\n",
       "  'other_names': 'enoate reductase',\n",
       "  'sys_name': 'butanoate:NAD+ Δ2-oxidoreductase',\n",
       "  'comments': 'An iron-sulfur-flavoprotein (FAD). Acts (in the reverse direction) on a wide range of alkyl and aryl αβ-unsaturated carboxylate ions; but-2-enoate was the best substrate tested.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.32': {'ec_num': '1.3.1.32',\n",
       "  'accepted_name': 'maleylacetate reductase',\n",
       "  'reaction': '3-oxoadipate + NAD(P)+ = 2-maleylacetate + NAD(P)H + H+',\n",
       "  'other_names': 'maleolylacetate reductase',\n",
       "  'sys_name': '3-oxoadipate:NAD(P)+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.1.32\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.1.32\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.1.32\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.1.32\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.3.1.32%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.33': {'ec_num': '1.3.1.33',\n",
       "  'accepted_name': 'protochlorophyllide reductase',\n",
       "  'reaction': 'chlorophyllide a + NADP+ = protochlorophyllide + NADPH + H+',\n",
       "  'other_names': 'NADPH2-protochlorophyllide oxidoreductase; NADPH-protochlorophyllide oxidoreductase; NADPH-protochlorophyllide reductase; protochlorophyllide oxidoreductase (ambiguous); protochlorophyllide photooxidoreductase; light-dependent protochlorophyllide reductase',\n",
       "  'sys_name': 'chlorophyllide-a:NADP+ 7,8-oxidoreductase',\n",
       "  'comments': 'The enzyme catalyses a light-dependent trans-reduction of the D-ring of protochlorophyllide; the product has the (7S,8S)-configuration.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.34': {'ec_num': '1.3.1.34',\n",
       "  'accepted_name': '2,4-dienoyl-CoA reductase [(2E)-enoyl-CoA-producing]',\n",
       "  'reaction': '(1) a (2E)-2-enoyl-CoA + NADP+ = a (2E,4E)-2,4-dienoyl-CoA + NADPH + H+;;(2) a (2E)-2-enoyl-CoA + NADP+ = a (2E,4Z)-2,4-dienoyl-CoA + NADPH + H+',\n",
       "  'other_names': 'fadH (gene name); 4-enoyl-CoA reductase (NADPH) (ambiguous); 4-enoyl coenzyme A (reduced nicotinamide adenine dinucleotide phosphate) reductase (ambiguous); 4-enoyl-CoA reductase (ambiguous); 2,4-dienoyl-CoA reductase (NADPH) (ambiguous); trans-2,3-didehydroacyl-CoA:NADP+ 4-oxidoreductase',\n",
       "  'sys_name': '(2E)-2-enoyl-CoA:NADP+ 4-oxidoreductase',\n",
       "  'comments': 'This bacterial enzyme catalyses the reduction of either (2E,4E)-2,4-dienoyl-CoA or (2E,4Z)-2,4-dienoyl-CoA to (2E)-2-enoyl-CoA. The enzyme from Escherichia coli contains FAD, FMN, and an [4Fe-4S] iron sulfur cluster. cf. EC 1.3.1.124, 2,4-dienoyl-CoA reductase [(3E)-enoyl-CoA-producing].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.35': {'ec_num': '1.3.1.35',\n",
       "  'accepted_name': 'phosphatidylcholine desaturase',\n",
       "  'reaction': '1-acyl-2-oleoyl-sn-glycero-3-phosphocholine + NAD+ = 1-acyl-2-linoleoyl-sn-glycero-3-phosphocholine + NADH + H+',\n",
       "  'other_names': 'oleate desaturase; linoleate synthase; oleoyl-CoA desaturase; oleoylphosphatidylcholine desaturase',\n",
       "  'sys_name': '1-acyl-2-oleoyl-sn-glycero-3-phosphocholine:NAD+ Δ12-oxidoreductase',\n",
       "  'comments': 'Also desaturates phosphatidylcholine containing the oleoyl group on O-2 of the glycerol residue.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.36': {'ec_num': '1.3.1.36',\n",
       "  'accepted_name': 'geissoschizine dehydrogenase',\n",
       "  'reaction': 'geissoschizine + NADP+ = 4,21-didehydrogeissoschizine + NADPH',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'geissoschizine:NADP+ 4,21-oxidoreductase',\n",
       "  'comments': 'Involved in the interconversion of heteroyohimbine alkaloids in Catharanthus roseus.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.37': {'ec_num': '1.3.1.37',\n",
       "  'accepted_name': 'cis-2-enoyl-CoA reductase (NADPH)',\n",
       "  'reaction': 'acyl-CoA + NADP+ = cis-2,3-dehydroacyl-CoA + NADPH + H+',\n",
       "  'other_names': 'NADPH-dependent cis-enoyl-CoA reductase; reductase, cis-2-enoyl coenzyme A; cis-2-enoyl-coenzyme A reductase; cis-2-enoyl-CoA reductase (NADPH)',\n",
       "  'sys_name': 'acyl-CoA:NADP+ cis-2-oxidoreductase',\n",
       "  'comments': 'Not identical with EC 1.3.1.38 trans-2-enoyl-CoA reductase (NADPH) [cf. EC 1.3.1.8 acyl-CoA dehydrogenase (NADP+)].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.38': {'ec_num': '1.3.1.38',\n",
       "  'accepted_name': 'trans-2-enoyl-CoA reductase (NADPH)',\n",
       "  'reaction': 'acyl-CoA + NADP+ = trans-2,3-dehydroacyl-CoA + NADPH + H+',\n",
       "  'other_names': 'NADPH-dependent trans-2-enoyl-CoA reductase; reductase, trans-enoyl coenzyme A; trans-2-enoyl-CoA reductase (NADPH2)',\n",
       "  'sys_name': 'acyl-CoA:NADP+ trans-2-oxidoreductase',\n",
       "  'comments': 'Not identical with EC 1.3.1.37 cis-2-enoyl-CoA reductase (NADPH) [cf. EC 1.3.1.8 acyl-CoA dehydrogenase (NADP+)].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.39': {'ec_num': '1.3.1.39',\n",
       "  'accepted_name': 'enoyl-[acyl-carrier-protein] reductase (NADPH, Re-specific)',\n",
       "  'reaction': 'an acyl-[acyl-carrier protein] + NADP+ = a trans-2,3-dehydroacyl-[acyl-carrier protein] + NADPH + H+',\n",
       "  'other_names': 'acyl-ACP dehydrogenase; enoyl-[acyl carrier protein] (reduced nicotinamide adenine dinucleotide phosphate) reductase; NADPH 2-enoyl Co A reductase; enoyl-ACp reductase; enoyl-[acyl-carrier-protein] reductase (NADPH2, A-specific); acyl-[acyl-carrier-protein]:NADP+ oxidoreductase (A-specific); enoyl-[acyl-carrier-protein] reductase (NADPH, A-specific); acyl-[acyl-carrier protein]:NADP+ oxidoreductase (A-specific)',\n",
       "  'sys_name': 'acyl-[acyl-carrier protein]:NADP+ oxidoreductase (Re-specific)',\n",
       "  'comments': 'This enzyme completes each cycle of fatty acid elongation by catalysing the stereospecific reduction of the double bond at position 2 of a growing fatty acid chain, while linked to an acyl-carrier protein. It is one of the activities of EC 2.3.1.85, fatty-acid synthase system. The mammalian enzyme is Re-specific with respect to NADP+. cf. EC 1.3.1.10, enoyl-[acyl-carrier-protein] reductase (NADPH, Si-specific) and EC 1.3.1.104, enoyl-[acyl-carrier-protein] reductase (NADPH).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.40': {'ec_num': '1.3.1.40',\n",
       "  'accepted_name': '2-hydroxy-6-oxo-6-phenylhexa-2,4-dienoate reductase',\n",
       "  'reaction': '2,6-dioxo-6-phenylhexanoate + NADP+ = 2-hydroxy-6-oxo-6-phenylhexa-2,4-dienoate + NADPH + H+',\n",
       "  'other_names': '2-hydroxy-6-oxo-phenylhexa-2,4-dienoate (reduced nicotinamide adenine dinucleotide phosphate) reductase',\n",
       "  'sys_name': '2,6-dioxo-6-phenylhexanoate:NADP+ Δ2-oxidoreductase',\n",
       "  'comments': 'Broad specificity; reduces a number of compounds produced by Pseudomonas from aromatic hydrocarbons by ring fission.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.41': {'ec_num': '1.3.1.41',\n",
       "  'accepted_name': 'xanthommatin reductase',\n",
       "  'reaction': '5,12-dihydroxanthommatin + NAD+ = xanthommatin + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '5,12-dihydroxanthommatin:NAD+ oxidoreductase',\n",
       "  'comments': 'From Drosophila melanogaster.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.42': {'ec_num': '1.3.1.42',\n",
       "  'accepted_name': '12-oxophytodienoate reductase',\n",
       "  'reaction': '(9S,13S,15Z)-12-oxo-10,11-dihydrophyto-15-enoate + NADP+ = (9S,13S,15Z)-12-oxophyto-10,15-dienoate + NADPH + H+',\n",
       "  'other_names': '12-oxo-phytodienoic acid reductase; 8-[(1R,2R)-3-oxo-2-{(Z)-pent-2-enyl}cyclopentyl]octanoate:NADP+ 4-oxidoreductase; (9S,13S)-10,11-dihydro-12-oxo-15-phytoenoate:NADP+ 4-oxidoreductase; (9S,13S)-12-oxophyto-15-enoate:NADP+ 10-oxidoreductase',\n",
       "  'sys_name': '(9S,13S,15Z)-12-oxo-10,11-dihydrophyto-15-enoate:NADP+ 10-oxidoreductase',\n",
       "  'comments': 'The enzyme catalyses the reduction of (9S,13S,15Z)-12-oxophyto-10,15-dienoate during the biosynthesis of jasmonate from α-linolenate in Zea mays.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '(9S,13S,15Z)-12-oxo-10,11-dihydrophyto-15-enoate = 8-[(1S,2S)-3-oxo-2-{(Z)-pent-2-en-1-yl}cyclopentyl]octanoate',\n",
       "  'last_change': '2023-05-29 10:00:51'},\n",
       " '1.3.1.43': {'ec_num': '1.3.1.43',\n",
       "  'accepted_name': 'arogenate dehydrogenase',\n",
       "  'reaction': 'L-arogenate + NAD+ = L-tyrosine + NADH + CO2',\n",
       "  'other_names': 'arogenic dehydrogenase (ambiguous); cyclohexadienyl dehydrogenase (ambiguous); pretyrosine dehydrogenase (ambiguous); L-arogenate:NAD+ oxidoreductase; arogenate dehydrogenase (NAD+)',\n",
       "  'sys_name': 'L-arogenate:NAD+ oxidoreductase (decarboxylating)',\n",
       "  'comments': 'Arogenate dehydrogenases may utilize NAD+ (EC 1.3.1.43), NADP+ (EC 1.3.1.78), or both (EC 1.3.1.79). NAD+-specific enzymes have been reported from some bacteria [2] and plants [3]. Some enzymes also possess the activity of EC 1.3.1.12, prephenate dehydrogenase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'L-arogenate = 1-[(2S)-2-amino-2-carboxyethyl]-4-hydroxycyclohexa-2,5-diene-1-carboxylate',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.44': {'ec_num': '1.3.1.44',\n",
       "  'accepted_name': 'trans-2-enoyl-CoA reductase (NAD+)',\n",
       "  'reaction': 'acyl-CoA + NAD+ = trans-didehydroacyl-CoA + NADH + H+',\n",
       "  'other_names': 'trans-2-enoyl-CoA reductase (NAD)',\n",
       "  'sys_name': 'acyl-CoA:NAD+ trans-2-oxidoreductase',\n",
       "  'comments': 'The enzyme from Euglena gracilis acts on crotonoyl-CoA and, more slowly, on trans-hex-2-enoyl-CoA and trans-oct-2-enoyl-CoA.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.45': {'ec_num': '1.3.1.45',\n",
       "  'accepted_name': '2′-hydroxyisoflavone reductase',\n",
       "  'reaction': 'vestitone + NADP+ = 2′-hydroxyformononetin + NADPH + H+',\n",
       "  'other_names': 'NADPH:2′-hydroxyisoflavone oxidoreductase; isoflavone reductase; 2′,7-dihydroxy-4′,5′-methylenedioxyisoflavone reductase',\n",
       "  'sys_name': 'vestitone:NADP+ oxidoreductase',\n",
       "  'comments': 'In the reverse reaction, a 2′-hydroxyisoflavone is reduced to an isoflavanone; 2′-hydroxypseudobaptigenin also acts. Involved in the biosynthesis of the pterocarpin phytoalexins medicarpin and maackiain.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.46': {'ec_num': '1.3.1.46',\n",
       "  'accepted_name': 'biochanin-A reductase',\n",
       "  'reaction': 'dihydrobiochanin A + NADP+ = biochanin A + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'dihydrobiochanin-A:NADP+ Δ2-oxidoreductase',\n",
       "  'comments': 'Some other isoflavones are reduced to the corresponding isoflavanones.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.47': {'ec_num': '1.3.1.47',\n",
       "  'accepted_name': 'α-santonin 1,2-reductase',\n",
       "  'reaction': '1,2-dihydrosantonin + NAD(P)+ = α-santonin + NAD(P)H + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '1,2-dihydrosantonin:NAD(P)+ 1,2-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.48': {'ec_num': '1.3.1.48',\n",
       "  'accepted_name': '13,14-dehydro-15-oxoprostaglandin 13-reductase',\n",
       "  'reaction': '11α-hydroxy-9,15-dioxoprostanoate + NAD(P)+ = (13E)-11α-hydroxy-9,15-dioxoprost-13-enoate + NAD(P)H + H+',\n",
       "  'other_names': '15-oxo-Δ13-prostaglandin reductase; Δ13-15-ketoprostaglandin reductase; 15-ketoprostaglandin Δ13-reductase; prostaglandin Δ13-reductase; prostaglandin 13-reductase; (5Z)-(15S)-11α-hydroxy-9,15-dioxoprostanoate:NAD(P)+ Δ13-oxidoreductase; (5Z)-11α-hydroxy-9,15-dioxoprost-5-enoate:NAD(P)+ Δ13-oxidoreductase',\n",
       "  'sys_name': '11α-hydroxy-9,15-dioxoprostanoate:NAD(P)+ Δ13-oxidoreductase',\n",
       "  'comments': 'Reduces 13,14-dehydro-15-oxoprostaglandins to 13,14-dihydro derivatives. The enzyme from placenta is specific for NAD+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.49': {'ec_num': '1.3.1.49',\n",
       "  'accepted_name': 'cis-3,4-dihydrophenanthrene-3,4-diol dehydrogenase',\n",
       "  'reaction': '(+)-cis-3,4-dihydrophenanthrene-3,4-diol + NAD+ = phenanthrene-3,4-diol + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '(+)-cis-3,4-dihydrophenanthrene-3,4-diol:NAD+ 3,4-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.1.49\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.1.49\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.1.49\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.1.49\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.51': {'ec_num': '1.3.1.51',\n",
       "  'accepted_name': '2′-hydroxydaidzein reductase',\n",
       "  'reaction': '2′-hydroxy-2,3-dihydrodaidzein + NADP+ = 2′-hydroxydaidzein + NADPH + H+',\n",
       "  'other_names': 'NADPH:2′-hydroxydaidzein oxidoreductase; HDR; 2′-hydroxydihydrodaidzein:NADP+ 2′-oxidoreductase',\n",
       "  'sys_name': '2′-hydroxy-2,3-dihydrodaidzein:NADP+ 2′-oxidoreductase',\n",
       "  'comments': 'In the reverse reaction, the 2′-hydroxyisoflavone (2′-hydroxydaidzein) is reduced to an isoflavanone. Also acts on 2′-hydroxyformononetin and to a small extent on 2′-hydroxygenistein. Involved in the biosynthesis of the phytoalexin glyceollin. The isoflavones biochanin A, daidzein and genestein as well as the flavonoids apigenin, kaempferol and quercetin do not act as substrates.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.52': {'ec_num': '1.3.1.52',\n",
       "  'accepted_name': '2-methyl-branched-chain-enoyl-CoA reductase',\n",
       "  'reaction': '2-methylbutanoyl-CoA + NAD+ = 2-methylcrotonoyl-CoA + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '2-methyl-branched-chain-acyl-CoA:NAD+ 2-oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD) from Ascaris suum. The reaction proceeds only in the presence of another flavoprotein (’electron-transferring flavoprotein’).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.53': {'ec_num': '1.3.1.53',\n",
       "  'accepted_name': '(3S,4R)-3,4-dihydroxycyclohexa-1,5-diene-1,4-dicarboxylate dehydrogenase',\n",
       "  'reaction': '(3S,4R)-3,4-dihydroxycyclohexa-1,5-diene-1,4-dicarboxylate + NAD+ = 3,4-dihydroxybenzoate + CO2 + NADH',\n",
       "  'other_names': '(1R,2S)-dihydroxy-3,5-cyclohexadiene-1,4-dicarboxylate dehydrogenase; terephthalate 1,2-cis-dihydrodiol dehydrogenase; cis-4,5-dihydroxycyclohexa-1(6),2-diene-1,4-dicarboxylate:NAD+ oxidoreductase (decarboxylating)',\n",
       "  'sys_name': '(3S,4R)-3,4-dihydroxycyclohexa-1,5-diene-1,4-dicarboxylate:NAD+ oxidoreductase',\n",
       "  'comments': 'Requires FeII. Involved in the terephthalate degradation pathway in bacteria [2].',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.1.53\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.1.53\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.1.53\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.1.53\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '(3S,4R)-3,4-dihydroxycyclohexa-1,5-diene-1,4-dicarboxylate = cis-4,5-dihydroxycyclohexa-1(6),2-diene-1,4-dicarboxylate',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.54': {'ec_num': '1.3.1.54',\n",
       "  'accepted_name': 'precorrin-6A reductase',\n",
       "  'reaction': 'precorrin-6B + NADP+ = precorrin-6A + NADPH + H+',\n",
       "  'other_names': 'precorrin-6X reductase; precorrin-6Y:NADP+ oxidoreductase; CobK',\n",
       "  'sys_name': 'precorrin-6B:NADP+ oxidoreductase',\n",
       "  'comments': 'The enzyme, which participates in the aerobic (late cobalt insertion) pathway of adenosylcobalamin biosynthesis, catalyses the reduction of the double bond between C-18 and C-19 of precorrin-6A. See EC 1.3.1.106, cobalt-precorrin-6A reductase, for the corresponding enzyme that participates in the anaerobic cobalamin biosynthesis pathway.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.56': {'ec_num': '1.3.1.56',\n",
       "  'accepted_name': 'cis-2,3-dihydrobiphenyl-2,3-diol dehydrogenase',\n",
       "  'reaction': 'cis-3-phenylcyclohexa-3,5-diene-1,2-diol + NAD+ = biphenyl-2,3-diol + NADH + H+',\n",
       "  'other_names': '2,3-dihydro-2,3-dihydroxybiphenyl dehydrogenase',\n",
       "  'sys_name': 'cis-3-phenylcyclohexa-3,5-diene-1,2-diol:NAD+ oxidoreductase',\n",
       "  'comments': 'Catalyses the second step in the biphenyl degradation pathway in bacteria.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.1.56\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.1.56\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.1.56\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.1.56\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.3.1.56%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.57': {'ec_num': '1.3.1.57',\n",
       "  'accepted_name': 'phloroglucinol reductase',\n",
       "  'reaction': 'dihydrophloroglucinol + NADP+ = phloroglucinol + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'dihydrophloroglucinol:NADP+ oxidoreductase',\n",
       "  'comments': 'Involved in the gallate anaerobic degradation pathway in bacteria.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.1.57\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.1.57\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.1.57\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.1.57\" target=\"new\">KEGG</a>',\n",
       "  'glossary': 'phloroglucinol = 1,3,5-trihydroxybenzene',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.58': {'ec_num': '1.3.1.58',\n",
       "  'accepted_name': '2,3-dihydroxy-2,3-dihydro-p-cumate dehydrogenase',\n",
       "  'reaction': 'cis-5,6-dihydroxy-4-isopropylcyclohexa-1,3-dienecarboxylate + NAD+ = 2,3-dihydroxy-p-cumate + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'cis-2,3-dihydroxy-2,3-dihydro-p-cumate:NAD+ oxidoreductase',\n",
       "  'comments': 'Involved in the p-cymene degradation pathway in Pseudomonas putida. ',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.1.58\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.1.58\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.1.58\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.1.58\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '<em>p</em>-cumic acid = 4-isopropylbenzoic acid<br>\\n<em>p</em>-cymene = 1-methyl-4-isopropylbenzene',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.59': {'ec_num': '1.3.1.59',\n",
       "  'accepted_name': '1,2-dihydroxy-3-methyl-1,2-dihydrobenzoate dehydrogenase',\n",
       "  'reaction': '1,6-dihydroxy-5-methylcyclohexa-2,4-diene-1-carboxylate + NAD+ = 3-methylcatechol + CO2 + NADH',\n",
       "  'other_names': '1,6-dihydroxy-5-methylcyclohexa-2,4-dienecarboxylate dehydrogenase; 1,6-dihydroxy-5-methylcyclohexa-2,4-dienecarboxylate:NAD+ oxidoreductase (decarboxylating)',\n",
       "  'sys_name': '1,6-dihydroxy-5-methylcyclohexa-2,4-diene-1-carboxylate:NAD+ oxidoreductase (decarboxylating)',\n",
       "  'comments': 'Involved in the m-xylene degradation pathway in bacteria.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.60': {'ec_num': '1.3.1.60',\n",
       "  'accepted_name': 'dibenzothiophene dihydrodiol dehydrogenase',\n",
       "  'reaction': 'cis-1,2-dihydroxy-1,2-dihydrodibenzothiophene + NAD+ = 1,2-dihydroxydibenzothiophene + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'cis-1,2-dihydroxy-1,2-dihydrodibenzothiophene:NAD+ oxidoreductase',\n",
       "  'comments': 'Involved in the dibenzothiophene degradation pathway in bacteria.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.61': {'ec_num': '1.3.1.61',\n",
       "  'accepted_name': 'terephthalate 1,2-cis-dihydrodiol dehydrogenase',\n",
       "  'reaction': 'cis-4,5-dihydroxycyclohexa-1(6),2-diene-1,4-dicarboxylate + NAD+ = 3,4-dihydroxybenzoate + CO2 + NADH',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'cis-4,5-dihydroxycyclohexa-1(6),2-diene-1,4-dicarboxylate:NAD+ oxidoreductase (decarboxylating)',\n",
       "  'comments': 'Involved in the terephthalate degradation pathway in bacteria.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.1.61\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.1.61\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.1.61\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.1.61\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.62': {'ec_num': '1.3.1.62',\n",
       "  'accepted_name': 'pimeloyl-CoA dehydrogenase',\n",
       "  'reaction': 'pimeloyl-CoA + NAD+ = 6-carboxyhex-2-enoyl-CoA + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'pimeloyl-CoA:NAD+ oxidoreductase',\n",
       "  'comments': 'Involved in the benzoate degradation (anaerobic) pathway in bacteria.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.1.62\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.1.62\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.1.62\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.1.62\" target=\"new\">KEGG</a>',\n",
       "  'glossary': 'pimelic acid = heptanedioic acid',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.63': {'ec_num': '1.3.1.63',\n",
       "  'accepted_name': '2,4-dichlorobenzoyl-CoA reductase',\n",
       "  'reaction': '4-chlorobenzoyl-CoA + NADP+ + chloride = 2,4-dichlorobenzoyl-CoA + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '4-chlorobenzoyl-CoA:NADP+ oxidoreductase (halogenating)',\n",
       "  'comments': 'Acts in the reverse direction to form part of the 2,4-dichlorobenzoate degradation pathway in bacteria.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.1.63\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.1.63\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.1.63\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.1.63\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.64': {'ec_num': '1.3.1.64',\n",
       "  'accepted_name': 'phthalate 4,5-cis-dihydrodiol dehydrogenase',\n",
       "  'reaction': 'cis-4,5-dihydroxycyclohexa-1(6),2-diene-1,2-dicarboxylate + NAD+ = 4,5-dihydroxyphthalate + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'cis-4,5-dihydroxycyclohexa-1(6),2-diene-1,2-dicarboxylate:NAD+ oxidoreductase',\n",
       "  'comments': 'Involved in the phthalate degradation pathway in bacteria.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.1.64\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.1.64\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.1.64\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.1.64\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.65': {'ec_num': '1.3.1.65',\n",
       "  'accepted_name': '5,6-dihydroxy-3-methyl-2-oxo-1,2,5,6-tetrahydroquinoline dehydrogenase',\n",
       "  'reaction': '5,6-dihydroxy-3-methyl-2-oxo-1,2,5,6-tetrahydroquinoline + NAD+ = 5,6-dihydroxy-3-methyl-2-oxo-1,2-dihydroquinoline + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '5,6-dihydroxy-3-methyl-2-oxo-1,2,5,6-tetrahydroquinoline:NAD+ oxidoreductase',\n",
       "  'comments': 'Acts in the reverse direction to form part of the 3-methylquinoline degradation pathway in bacteria.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.1.65\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.1.65\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.1.65\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.1.65\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.66': {'ec_num': '1.3.1.66',\n",
       "  'accepted_name': 'cis-dihydroethylcatechol dehydrogenase',\n",
       "  'reaction': 'cis-1,2-dihydro-3-ethylcatechol + NAD+ = 3-ethylcatechol + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'cis-1,2-dihydro-3-ethylcatechol:NAD+ oxidoreductase',\n",
       "  'comments': 'Involved in the ethylbenzene degradation pathway in bacteria.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.1.66\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.1.66\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.1.66\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.1.66\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.67': {'ec_num': '1.3.1.67',\n",
       "  'accepted_name': 'cis-1,2-dihydroxy-4-methylcyclohexa-3,5-diene-1-carboxylate dehydrogenase',\n",
       "  'reaction': 'cis-1,2-dihydroxy-4-methylcyclohexa-3,5-diene-1-carboxylate + NAD(P)+ = 4-methylcatechol + NAD(P)H + CO2',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'cis-1,2-dihydroxy-4-methylcyclohexa-3,5-diene-1-carboxylate:NAD(P)+ oxidoreductase (decarboxylating)',\n",
       "  'comments': 'Involved in the p-xylene degradation pathway in bacteria.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.1.67\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.1.67\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.1.67\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.1.67\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.68': {'ec_num': '1.3.1.68',\n",
       "  'accepted_name': '1,2-dihydroxy-6-methylcyclohexa-3,5-dienecarboxylate dehydrogenase',\n",
       "  'reaction': '1,2-dihydroxy-6-methylcyclohexa-3,5-dienecarboxylate + NAD+ = 3-methylcatechol + NADH + CO2',\n",
       "  'other_names': '',\n",
       "  'sys_name': '1,2-dihydroxy-6-methylcyclohexa-3,5-dienecarboxylate:NAD+ oxidoreductase (decarboxylating)',\n",
       "  'comments': 'Involved in the o-xylene degradation pathway in bacteria.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.1.68\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.1.68\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.1.68\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.1.68\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.69': {'ec_num': '1.3.1.69',\n",
       "  'accepted_name': 'zeatin reductase',\n",
       "  'reaction': 'dihydrozeatin + NADP+ = zeatin + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'dihydrozeatin:NADP+ oxidoreductase',\n",
       "  'comments': 'Previously classified erroneously as EC 1.1.1.242.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'zeatin = (E)-2-methyl-4-(9H-purin-6-ylamino)but-2-en-1-ol = (E)-N6-(4-hydroxy-3-methylbut-2-enyl)adenine',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.70': {'ec_num': '1.3.1.70',\n",
       "  'accepted_name': 'Δ14-sterol reductase',\n",
       "  'reaction': '4,4-dimethyl-5α-cholesta-8,24-dien-3β-ol + NADP+ = 4,4-dimethyl-5α-cholesta-8,14,24-trien-3β-ol + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '4,4-dimethyl-5α-cholesta-8,24-dien-3β-ol:NADP+ Δ14-oxidoreductase',\n",
       "  'comments': 'This enzyme acts on a range of steroids with a 14(15)-double bond.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.71': {'ec_num': '1.3.1.71',\n",
       "  'accepted_name': 'Δ24(241)-sterol reductase',\n",
       "  'reaction': 'ergosterol + NADP+ = ergosta-5,7,22,24(241)-tetraen-3β-ol + NADPH + H+',\n",
       "  'other_names': 'sterol Δ24(28)-methylene reductase; sterol Δ24(28)-reductase',\n",
       "  'sys_name': 'ergosterol:NADP+ Δ24(241)-oxidoreductase',\n",
       "  'comments': 'Acts on a range of steroids with a 24(241)-double bond.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.72': {'ec_num': '1.3.1.72',\n",
       "  'accepted_name': 'Δ24-sterol reductase',\n",
       "  'reaction': '5α-cholest-7-en-3β-ol + NADP+ = 5α-cholesta-7,24-dien-3β-ol + NADPH + H+',\n",
       "  'other_names': 'lanosterol Δ24-reductase',\n",
       "  'sys_name': 'sterol:NADP+ Δ24-oxidoreductase',\n",
       "  'comments': 'Acts on a range of steroids with a 24(25)-double bond, including lanosterol, desmosterol and zymosterol.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'desmosterol = cholesta-5,24-dien-3β-ol<br>\\n<a href=\"http://www.chem.qmul.ac.uk/iupac/sectionF/terp2.html#K\" target=\"new\">lanosterol</a> = 4,4,14-trimethyl-5α-cholesta-8,24-dien-3β-ol<br>\\nzymostrol = 5α-cholesta-8,24-dien-3β-ol',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.73': {'ec_num': '1.3.1.73',\n",
       "  'accepted_name': '1,2-dihydrovomilenine reductase',\n",
       "  'reaction': '17-O-acetylnorajmaline + NADP+ = 1,2-dihydrovomilenine + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '17-O-acetylnorajmaline:NADP+ oxidoreductase',\n",
       "  'comments': 'Forms part of the ajmaline biosynthesis pathway.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.74': {'ec_num': '1.3.1.74',\n",
       "  'accepted_name': '2-alkenal reductase [NAD(P)+]',\n",
       "  'reaction': 'a n-alkanal + NAD(P)+ = an alk-2-enal + NAD(P)H + H+',\n",
       "  'other_names': 'NAD(P)H-dependent alkenal/one oxidoreductase; NADPH:2-alkenal α,β-hydrogenase; 2-alkenal reductase',\n",
       "  'sys_name': 'n-alkanal:NAD(P)+ 2-oxidoreductase',\n",
       "  'comments': 'Highly specific for 4-hydroxynon-2-enal and non-2-enal. Alk-2-enals of shorter chain have lower affinities. Exhibits high activities also for alk-2-enones such as but-3-en-2-one and pent-3-en-2-one. Inactive with cyclohex-2-en-1-one and 12-oxophytodienoic acid. Involved in the detoxication of α,β-unsaturated aldehydes and ketones [cf. EC 1.3.1.102, 2-alkenal reductase (NADP+)].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.75': {'ec_num': '1.3.1.75',\n",
       "  'accepted_name': '3,8-divinyl protochlorophyllide a 8-vinyl-reductase (NADPH)',\n",
       "  'reaction': ' protochlorophyllide a + NADP+ = 3,8-divinyl protochlorophyllide a + NADPH + H+\\n ',\n",
       "  'other_names': 'DVR (gene name); bciA (gene name); [4-vinyl]chlorophyllide a reductase; 4VCR; chlorophyllide-a:NADP+ oxidoreductase; divinyl chlorophyllide a 8-vinyl-reductase; plant-type divinyl chlorophyllide a 8-vinyl-reductase',\n",
       "  'sys_name': 'protochlorophyllide-a:NADP+ C-81-oxidoreductase',\n",
       "  'comments': 'The enzyme, found in higher plants, green algae, and some phototrophic bacteria, is involved in the production of monovinyl versions of (bacterio)chlorophyll pigments from their divinyl precursors. It can also act on 3,8-divinyl chlorophyllide a. cf. EC 1.3.7.13, 3,8-divinyl protochlorophyllide a 8-vinyl-reductase (ferredoxin).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.76': {'ec_num': '1.3.1.76',\n",
       "  'accepted_name': 'precorrin-2 dehydrogenase',\n",
       "  'reaction': 'precorrin-2 + NAD+ = sirohydrochlorin + NADH + H+',\n",
       "  'other_names': 'Met8p; SirC; CysG',\n",
       "  'sys_name': 'precorrin-2:NAD+ oxidoreductase',\n",
       "  'comments': 'This enzyme catalyses the second of three steps leading to the formation of siroheme from uroporphyrinogen III. The first step involves the donation of two S-adenosyl-L-methionine-derived methyl groups to carbons 2 and 7 of uroporphyrinogen III to form precorrin-2 (EC 2.1.1.107, uroporphyrin-III C-methyltransferase) and the third step involves the chelation of ferrous iron to sirohydrochlorin to form siroheme (EC 4.99.1.4, sirohydrochlorin ferrochelatase). In Saccharomyces cerevisiae, the last two steps are carried out by a single bifunctional enzyme, Met8p. In some bacteria, steps 1-3 are catalysed by a single multifunctional protein called CysG, whereas in Bacillus megaterium, three separate enzymes carry out each of the steps, with SirC being responsible for the above reaction.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.77': {'ec_num': '1.3.1.77',\n",
       "  'accepted_name': 'anthocyanidin reductase [(2R,3R)-flavan-3-ol-forming]',\n",
       "  'reaction': 'a (2R,3R)-flavan-3-ol + 2 NAD(P)+ = an anthocyanidin with a 3-hydroxy group + 2 NAD(P)H + H+',\n",
       "  'other_names': 'ANR (gene name) (ambiguous); flavan-3-ol:NAD(P)+ oxidoreductase; anthocyanidin reductase (ambiguous)',\n",
       "  'sys_name': '(2R,3R)-flavan-3-ol:NAD(P)+ 3,4-oxidoreductase',\n",
       "  'comments': 'The enzyme participates in the flavonoid biosynthesis pathway found in plants. It catalyses the double reduction of anthocyanidins, producing (2R,3R)-flavan-3-ol monomers required for the formation of proanthocyanidins. While the enzyme from the legume Medicago truncatula (MtANR) can use both NADPH and NADH as reductant, that from the crucifer Arabidopsis thaliana (AtANR) uses only NADPH. Also, while the substrate preference of MtANR is cyanidin>pelargonidin>delphinidin, the reverse preference is found with AtANR. cf. EC 1.3.1.112, anthocyanidin reductase [(2S)-flavan-3-ol-forming].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.78': {'ec_num': '1.3.1.78',\n",
       "  'accepted_name': 'arogenate dehydrogenase (NADP+)',\n",
       "  'reaction': 'L-arogenate + NADP+ = L-tyrosine + NADPH + CO2',\n",
       "  'other_names': 'arogenic dehydrogenase (ambiguous); pretyrosine dehydrogenase (ambiguous); TyrAAT1; TyrAAT2; TyrAa',\n",
       "  'sys_name': 'L-arogenate:NADP+ oxidoreductase (decarboxylating)',\n",
       "  'comments': 'Arogenate dehydrogenases may utilize NAD+ (EC 1.3.1.43), NADP+ (EC 1.3.1.78), or both (EC 1.3.1.79). NADP+-dependent enzymes usually predominate in higher plants.The enzyme from the cyanobacterium Synechocystis sp. PCC 6803 and the TyrAAT1 isoform of the plant Arabidopsis thaliana cannot use prephenate as a substrate, while the Arabidopsis isoform TyrAAT2 can use it very poorly [2,3].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'L-arogenate = 1-[(2S)-2-amino-2-carboxyethyl]-4-hydroxycyclohexa-2,5-diene-1-carboxylate',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.1.79': {'ec_num': '1.3.1.79',\n",
       "  'accepted_name': 'arogenate dehydrogenase [NAD(P)+]',\n",
       "  'reaction': 'L-arogenate + NAD(P)+ = L-tyrosine + NAD(P)H + CO2',\n",
       "  'other_names': 'arogenic dehydrogenase (ambiguous); cyclohexadienyl dehydrogenase; pretyrosine dehydrogenase (ambiguous)',\n",
       "  'sys_name': 'L-arogenate:NAD(P)+ oxidoreductase (decarboxylating)',\n",
       "  'comments': 'Arogenate dehydrogenases may utilize NAD+ (EC 1.3.1.43), NADP+ (EC 1.3.1.78), or both (EC 1.3.1.79). Enzymes that can utilize both cofactors have been reported from some Proteobacteria, including Burkholderia caryophylli, Burkholderia cepacia, Pseudomonas marginata and Delftia acidovorans.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'L-arogenate = 1-[(2S)-2-amino-2-carboxyethyl]-4-hydroxycyclohexa-2,5-diene-1-carboxylate',\n",
       "  'last_change': '2023-02-11 18:31:18'},\n",
       " '1.3.2.3': {'ec_num': '1.3.2.3',\n",
       "  'accepted_name': 'L-galactonolactone dehydrogenase',\n",
       "  'reaction': 'L-galactono-1,4-lactone + 4 ferricytochrome c = L-dehydroascorbate + 4 ferrocytochrome c + 4 H+ (overall reaction);;(1a) L-galactono-1,4-lactone + 2 ferricytochrome c = L-ascorbate + 2 ferrocytochrome c + 2 H+;;(1b) L-ascorbate + 2 ferricytochrome c = L-dehydroascorbate + 2 ferrocytochrome c + 2 H+ (spontaneous)',\n",
       "  'other_names': 'galactonolactone dehydrogenase; L-galactono-γ-lactone dehydrogenase; L-galactono-γ-lactone:ferricytochrome-c oxidoreductase; GLDHase; GLDase',\n",
       "  'sys_name': 'L-galactono-1,4-lactone:ferricytochrome-c oxidoreductase',\n",
       "  'comments': 'This enzyme catalyses the final step in the biosynthesis of L-ascorbic acid in higher plants and in nearly all higher animals with the exception of primates and some birds [5]. The enzyme is very specific for its substrate L-galactono-1,4-lactone as D-galactono-γ-lactone, D-gulono-γ-lactone, L-gulono-γ-lactone, D-erythronic-γ-lactone, D-xylonic-γ-lactone, L-mannono-γ-lactone, D-galactonate, D-glucuronate and D-gluconate are not substrates [5]. FAD, NAD+, NADP+ and O2 (cf. EC 1.3.3.12, L-galactonolactone oxidase) cannot act as electron acceptor [5].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:19'},\n",
       " '1.3.3.1': {'ec_num': '1.3.3.1',\n",
       "  'accepted_name': 'dihydroorotate oxidase',\n",
       "  'reaction': '(S)-dihydroorotate + O2 = orotate + H2O2',\n",
       "  'other_names': 'DHOdehase; dihydroorotate dehydrogenase (ambiguous); dihydoorotic acid dehydrogenase; (DHO) dehydrogenase; 4,5-L-dihydroorotate:oxygen oxidoreductase',\n",
       "  'sys_name': '(S)-dihydroorotate:oxygen oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD, FMN). Ferricyanide can act as acceptor.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.3.3': {'ec_num': '1.3.3.3',\n",
       "  'accepted_name': 'coproporphyrinogen oxidase',\n",
       "  'reaction': 'coproporphyrinogen III + O2 + 2 H+ = protoporphyrinogen-IX + 2 CO2 + 2 H2O',\n",
       "  'other_names': 'coproporphyrinogen III oxidase; coproporphyrinogenase',\n",
       "  'sys_name': 'coproporphyrinogen:oxygen oxidoreductase (decarboxylating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.3.4': {'ec_num': '1.3.3.4',\n",
       "  'accepted_name': 'protoporphyrinogen oxidase',\n",
       "  'reaction': 'protoporphyrinogen IX + 3 O2 = protoporphyrin IX + 3 H2O2',\n",
       "  'other_names': 'protoporphyrinogen IX oxidase; protoporphyrinogenase; PPO; Protox; HemG; HemY',\n",
       "  'sys_name': 'protoporphyrinogen-IX:oxygen oxidoreductase',\n",
       "  'comments': 'This is the last common enzyme in the biosynthesis of chlorophylls and heme [8]. Two isoenzymes exist in plants: one in plastids and the other in mitochondria. This is the target enzyme of phthalimide-type and diphenylether-type herbicides [8]. The enzyme from oxygen-dependent species contains FAD [9]. Also slowly oxidizes mesoporphyrinogen IX.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.3.5': {'ec_num': '1.3.3.5',\n",
       "  'accepted_name': 'bilirubin oxidase',\n",
       "  'reaction': '2 bilirubin + O2 = 2 biliverdin + 2 H2O',\n",
       "  'other_names': 'bilirubin oxidase M-1',\n",
       "  'sys_name': 'bilirubin:oxygen oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.3.6': {'ec_num': '1.3.3.6',\n",
       "  'accepted_name': 'acyl-CoA oxidase',\n",
       "  'reaction': 'acyl-CoA + O2 = trans-2,3-dehydroacyl-CoA + H2O2',\n",
       "  'other_names': 'fatty acyl-CoA oxidase; acyl coenzyme A oxidase; fatty acyl-coenzyme A oxidase',\n",
       "  'sys_name': 'acyl-CoA:oxygen 2-oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD). Acts on CoA derivatives of fatty acids with chain lengths from 8 to 18.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.3.7': {'ec_num': '1.3.3.7',\n",
       "  'accepted_name': 'dihydrouracil oxidase',\n",
       "  'reaction': '5,6-dihydrouracil + O2 = uracil + H2O2',\n",
       "  'other_names': '',\n",
       "  'sys_name': '5,6-dihydrouracil:oxygen oxidoreductase',\n",
       "  'comments': 'Also oxidizes dihydrothymine to thymine. A flavoprotein (FMN).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.3.8': {'ec_num': '1.3.3.8',\n",
       "  'accepted_name': 'tetrahydroberberine oxidase',\n",
       "  'reaction': '(S)-tetrahydroberberine + 2 O2 = berberine + 2 H2O2',\n",
       "  'other_names': '(S)-THB oxidase',\n",
       "  'sys_name': '(S)-tetrahydroberberine:oxygen oxidoreductase',\n",
       "  'comments': 'The enzyme from Berberis sp. is a flavoprotein; that from Coptis japonica is not. (R)-Tetrahydroberberines are not oxidized.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.3.9': {'ec_num': '1.3.3.9',\n",
       "  'accepted_name': 'secologanin synthase',\n",
       "  'reaction': 'loganin + NADPH + H+ + O2 = secologanin + NADP+ + 2 H2O',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'loganin:oxygen oxidoreductase (ring-cleaving)',\n",
       "  'comments': 'A heme-thiolate protein (P-450). Secologanin is the precursor of the monoterpenoid indole alkaloids and ipecac alkaloids.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.3.10': {'ec_num': '1.3.3.10',\n",
       "  'accepted_name': 'tryptophan α,β-oxidase',\n",
       "  'reaction': 'L-tryptophan + O2 = α,β-didehydrotryptophan + H2O2',\n",
       "  'other_names': 'L-tryptophan 2′,3′-oxidase; L-tryptophan α,β-dehydrogenase',\n",
       "  'sys_name': 'L-tryptophan:oxygen α,β-oxidoreductase',\n",
       "  'comments': 'Requires heme. The enzyme from Chromobacterium violaceum is specific for tryptophan derivatives possessing its carboxyl group free or as an amide or ester, and an unsubstituted indole ring. Also catalyses the α,β dehydrogenation of L-tryptophan side chains in peptides. The product of the reaction can hydrolyse spontaneously to form (indol-3-yl)pyruvate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.3.11': {'ec_num': '1.3.3.11',\n",
       "  'accepted_name': 'pyrroloquinoline-quinone synthase',\n",
       "  'reaction': '6-(2-amino-2-carboxyethyl)-7,8-dioxo-1,2,3,4,7,8-hexahydroquinoline-2,4-dicarboxylate + 3 O2 = 4,5-dioxo-4,5-dihydro-1H-pyrrolo[2,3-f]quinoline-2,7,9-tricarboxylate + 2 H2O2 + 2 H2O',\n",
       "  'other_names': 'PqqC; 6-(2-amino-2-carboxyethyl)-7,8-dioxo-1,2,3,4,5,6,7,8-octahydroquinoline-2,4-dicarboxylate:oxygen oxidoreductase (cyclizing) [incorrect]',\n",
       "  'sys_name': '6-(2-amino-2-carboxyethyl)-7,8-dioxo-1,2,3,4,7,8-hexahydroquinoline-2,4-dicarboxylate:oxygen oxidoreductase (cyclizing)',\n",
       "  'comments': 'So far only a single turnover of the enzyme has been observed, and the pyrroloquinoline quinone remains bound to it. It is not yet known what releases the product in the bacterium.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.5.1': {'ec_num': '1.3.5.1',\n",
       "  'accepted_name': 'succinate dehydrogenase',\n",
       "  'reaction': 'succinate + a quinone = fumarate + a quinol',\n",
       "  'other_names': 'succinate dehydrogenase (quinone); succinate dehydrogenase (ubiquinone); succinic dehydrogenase; complex II (ambiguous); succinate dehydrogenase complex; SDH (ambiguous); succinate:ubiquinone oxidoreductase; fumarate reductase (quinol); FRD; menaquinol-fumarate oxidoreductase; succinate dehydrogenase (menaquinone); succinate:menaquinone oxidoreductase; fumarate reductase (menaquinone)',\n",
       "  'sys_name': 'succinate:quinone oxidoreductase',\n",
       "  'comments': 'A complex generally comprising an FAD-containing component that also binds the carboxylate substrate (A subunit), a component that contains three different iron-sulfur centers [2Fe-2S], [4Fe-4S], and [3Fe-4S] (B subunit), and a hydrophobic membrane-anchor component (C, or C and D subunits) that is also the site of the interaction with quinones. The enzyme is found in the inner mitochondrial membrane in eukaryotes and the plasma membrane of bacteria and archaea, with the hydrophilic domain extending into the mitochondrial matrix and the cytoplasm, respectively. Under aerobic conditions the enzyme catalyses succinate oxidation, a key step in the citric acid (TCA) cycle, transferring the electrons to quinones in the membrane, thus linking the TCA cycle with the aerobic respiratory chain (where it is known as complex II). Under anaerobic conditions the enzyme functions as a fumarate reductase, transferring electrons from the quinol pool to fumarate, and participating in anaerobic respiration with fumarate as the terminal electron acceptor. The enzyme interacts with the quinone produced by the organism, such as ubiquinone, menaquinone, caldariellaquinone, thermoplasmaquinone, rhodoquinone etc. Some of the enzymes contain two heme subunits in their membrane anchor subunit. These enzymes catalyse an electrogenic reaction and are thus classified as EC 7.1.1.12, succinate dehydrogenase (electrogenic, proton-motive force generating).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.7.1': {'ec_num': '1.3.7.1',\n",
       "  'accepted_name': '6-hydroxynicotinate reductase',\n",
       "  'reaction': '6-oxo-1,4,5,6-tetrahydronicotinate + oxidized ferredoxin = 6-hydroxynicotinate + reduced ferredoxin',\n",
       "  'other_names': '6-oxotetrahydronicotinate dehydrogenase; 6-hydroxynicotinic reductase; HNA reductase; 1,4,5,6-tetrahydro-6-oxonicotinate:ferredoxin oxidoreductase',\n",
       "  'sys_name': '6-oxo-1,4,5,6-tetrahydronicotinate:ferredoxin oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.7.2': {'ec_num': '1.3.7.2',\n",
       "  'accepted_name': '15,16-dihydrobiliverdin:ferredoxin oxidoreductase',\n",
       "  'reaction': '15,16-dihydrobiliverdin + oxidized ferredoxin = biliverdin IXα + reduced ferredoxin',\n",
       "  'other_names': 'PebA ',\n",
       "  'sys_name': '15,16-dihydrobiliverdin:ferredoxin oxidoreductase',\n",
       "  'comments': 'Catalyses the two-electron reduction of biliverdin IXα at the C15 methine bridge. It has been proposed that this enzyme and EC 1.3.7.3, phycoerythrobilin:ferredoxin oxidoreductase, function as a dual enzyme complex in the conversion of biliverdin IXα into phycoerythrobilin.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.7.3': {'ec_num': '1.3.7.3',\n",
       "  'accepted_name': 'phycoerythrobilin:ferredoxin oxidoreductase',\n",
       "  'reaction': '(3Z)-phycoerythrobilin + oxidized ferredoxin = 15,16-dihydrobiliverdin + reduced ferredoxin',\n",
       "  'other_names': 'PebB ',\n",
       "  'sys_name': '(3Z)-phycoerythrobilin:ferredoxin oxidoreductase',\n",
       "  'comments': 'Catalyses the two-electron reduction of the C2 and C31 diene system of 15,16-dihydrobiliverdin. Specific for 15,16-dihydrobiliverdin. It has been proposed that this enzyme and EC 1.3.7.2, 15,16-dihydrobiliverdin:ferredoxin oxidoreductase, function as a dual enzyme complex in the conversion of biliverdin IXα to phycoerythrobilin.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.7.4': {'ec_num': '1.3.7.4',\n",
       "  'accepted_name': 'phytochromobilin:ferredoxin oxidoreductase',\n",
       "  'reaction': '(3Z)-phytochromobilin + 2 oxidized ferredoxin = biliverdin IXα + 2 reduced ferredoxin',\n",
       "  'other_names': 'HY2; PPhi B synthase; phytochromobilin synthase',\n",
       "  'sys_name': '(3Z)-phytochromobilin:ferredoxin oxidoreductase',\n",
       "  'comments': 'Catalyses the two-electron reduction of biliverdin IXα. Can use [2Fe-2S] ferredoxins from a number of sources as acceptor but not the [4Fe-4S] ferredoxin from Clostridium pasteurianum. The isomerization of (3Z)-phytochromobilin to (3E)-phytochromobilin is thought to occur prior to covalent attachment to apophytochrome in the plant cell cytoplasm. Flavodoxins can be used instead of ferredoxin.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.7.5': {'ec_num': '1.3.7.5',\n",
       "  'accepted_name': 'phycocyanobilin:ferredoxin oxidoreductase',\n",
       "  'reaction': '(3Z)-phycocyanobilin + 4 oxidized ferredoxin = biliverdin IXα + 4 reduced ferredoxin',\n",
       "  'other_names': '',\n",
       "  'sys_name': '(3Z)-phycocyanobilin:ferredoxin oxidoreductase',\n",
       "  'comments': 'Catalyses the four-electron reduction of biliverdin IXα (2-electron reduction at both the A and D rings). Reaction proceeds via an isolatable 2-electron intermediate, 181,182-dihydrobiliverdin. Flavodoxins can be used instead of ferredoxin. The direct conversion of biliverdin IXα (BV) to (3Z)-phycocyanolbilin (PCB) in the cyanobacteria Synechocystis sp. PCC 6803, Anabaena sp. PCC7120 and Nostoc punctiforme is in contrast to the proposed pathways of PCB biosynthesis in the red alga Cyanidium caldarium, which involves (3Z)-phycoerythrobilin (PEB) as an intermediate [2] and in the green alga Mesotaenium caldariorum, in which PCB is an isolable intermediate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.1': {'ec_num': '1.3.99.1',\n",
       "  'accepted_name': 'succinate dehydrogenase',\n",
       "  'reaction': 'succinate + acceptor = fumarate + reduced acceptor',\n",
       "  'other_names': 'succinic dehydrogenase; fumarate reductase; fumaric hydrogenase; succinodehydrogenase; succinic acid dehydrogenase; succinate oxidoreductase; succinyl dehydrogenase; succinate:(acceptor) oxidoreductase',\n",
       "  'sys_name': 'succinate:acceptor oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD) containing iron-sulfur centres. A component of EC 1.3.5.1 succinate dehydrogenase (ubiquinone).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.2': {'ec_num': '1.3.99.2',\n",
       "  'accepted_name': 'butyryl-CoA dehydrogenase',\n",
       "  'reaction': 'butanoyl-CoA + acceptor = 2-butenoyl-CoA + reduced acceptor',\n",
       "  'other_names': 'butyryl dehydrogenase; unsaturated acyl-CoA reductase; ethylene reductase; enoyl-coenzyme A reductase; unsaturated acyl coenzyme A reductase; butyryl coenzyme A dehydrogenase; short-chain acyl CoA dehydrogenase; short-chain acyl-coenzyme A dehydrogenase; 3-hydroxyacyl CoA reductase; butanoyl-CoA:(acceptor) 2,3-oxidoreductase',\n",
       "  'sys_name': 'butanoyl-CoA:acceptor 2,3-oxidoreductase',\n",
       "  'comments': 'A flavoprotein; forms, with another flavoprotein (’electron-transferring flavoproteins’) and EC 1.5.5.1 electron-transferring-flavoprotein dehydrogenase, a system reducing ubiquinone and other acceptors.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.99.2\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.99.2\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.99.2\" target=\"new\">EXPASY</a>, <a href=\"https://randr.nist.gov/enzyme/Default.aspx\" target=\"new\">GTD</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.99.2\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.3.99.2%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.3': {'ec_num': '1.3.99.3',\n",
       "  'accepted_name': 'acyl-CoA dehydrogenase',\n",
       "  'reaction': 'acyl-CoA + acceptor = 2,3-dehydroacyl-CoA + reduced acceptor',\n",
       "  'other_names': 'acyl dehydrogenase; fatty acyl coenzyme A dehydrogenase; acyl coenzyme A dehydrogenase; fatty-acyl-CoA dehydrogenase; acyl CoA dehydrogenase; general acyl CoA dehydrogenase; medium-chain acyl-coenzyme A dehydrogenase; medium-chain acyl-CoA dehydrogenase; long-chain acyl coenzyme A dehydrogenase; long-chain acyl-CoA dehydrogenase; acyl-CoA:(acceptor) 2,3-oxidoreductase',\n",
       "  'sys_name': 'acyl-CoA:acceptor 2,3-oxidoreductase',\n",
       "  'comments': 'A flavoprotein; forms, with another flavoprotein (’electron-transferring flavoprotein’) and EC 1.5.5.1 electron-transferring-flavoprotein dehydrogenase, a system reducing ubiquinone and other acceptors. ',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.4': {'ec_num': '1.3.99.4',\n",
       "  'accepted_name': '3-oxosteroid 1-dehydrogenase',\n",
       "  'reaction': 'a 3-oxosteroid + acceptor = a 3-oxo-Δ1-steroid + reduced acceptor',\n",
       "  'other_names': '3-oxosteroid Δ1-dehydrogenase; Δ1-dehydrogenase; 3-ketosteroid-1-en-dehydrogenase; 3-ketosteroid-Δ1-dehydrogenase; 1-ene-dehydrogenase; 3-oxosteroid:(2,6-dichlorphenolindophenol) Δ1-oxidoreductase; 4-en-3-oxosteroid:(acceptor)-1-en-oxido-reductase; Δ1-steroid reductase; 3-oxosteroid:(acceptor) Δ1-oxidoreductase',\n",
       "  'sys_name': '3-oxosteroid:acceptor Δ1-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.99.4\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.99.4\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.99.4\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.99.4\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.3.99.4%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.5': {'ec_num': '1.3.99.5',\n",
       "  'accepted_name': '3-oxo-5α-steroid 4-dehydrogenase (acceptor)',\n",
       "  'reaction': 'a 3-oxo-5α-steroid + acceptor = a 3-oxo-Δ4-steroid + reduced acceptor',\n",
       "  'other_names': 'steroid 5α-reductase; 3-oxosteroid Δ4-dehydrogenase; 3-oxo-5α-steroid Δ4-dehydrogenase; steroid Δ4-5α-reductase; Δ4-3-keto steroid 5α-reductase; Δ4-3-oxo steroid reductase; Δ4-3-ketosteroid5α-oxidoreductase; Δ4-3-oxosteroid-5α-reductase; 3-keto-Δ4-steroid-5α-reductase; 5α-reductase; testosterone 5α-reductase; 4-ene-3-ketosteroid-5α-oxidoreductase; Δ4-5α-dehydrogenase; 3-oxo-5α-steroid:(acceptor) Δ4-oxidoreductase; tesI (gene name)',\n",
       "  'sys_name': '3-oxo-5α-steroid:acceptor Δ4-oxidoreductase',\n",
       "  'comments': 'A flavoprotein. This bacterial enzyme, characterized from Comamonas testosteroni, is involved in androsterone degradation. cf. EC 1.3.1.22, 3-oxo-5α-steroid 4-dehydrogenase (NADP+).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.6': {'ec_num': '1.3.99.6',\n",
       "  'accepted_name': '3-oxo-5β-steroid 4-dehydrogenase',\n",
       "  'reaction': 'a 3-oxo-5β-steroid + acceptor = a 3-oxo-Δ4-steroid + reduced acceptor',\n",
       "  'other_names': '3-oxo-5β-steroid:(acceptor) Δ4-oxidoreductase',\n",
       "  'sys_name': '3-oxo-5β-steroid:acceptor Δ4-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.7': {'ec_num': '1.3.99.7',\n",
       "  'accepted_name': 'glutaryl-CoA dehydrogenase',\n",
       "  'reaction': 'glutaryl-CoA + acceptor = crotonoyl-CoA + CO2 + reduced acceptor',\n",
       "  'other_names': 'glutaryl coenzyme A dehydrogenase; glutaryl-CoA:(acceptor) 2,3-oxidoreductase (decarboxylating)',\n",
       "  'sys_name': 'glutaryl-CoA:acceptor 2,3-oxidoreductase (decarboxylating)',\n",
       "  'comments': 'A flavoprotein.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.99.7\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.99.7\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.99.7\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.99.7\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.3.99.7%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.8': {'ec_num': '1.3.99.8',\n",
       "  'accepted_name': '2-furoyl-CoA dehydrogenase',\n",
       "  'reaction': '2-furoyl-CoA + H2O + acceptor = S-(5-hydroxy-2-furoyl)-CoA + reduced acceptor',\n",
       "  'other_names': 'furoyl-CoA hydroxylase; 2-furoyl coenzyme A hydroxylase; 2-furoyl coenzyme A dehydrogenase; 2-furoyl-CoA:(acceptor) 5-oxidoreductase (hydroxylating)',\n",
       "  'sys_name': '2-furoyl-CoA:acceptor 5-oxidoreductase (hydroxylating)',\n",
       "  'comments': 'A copper protein. The oxygen atom of the -OH produced is derived from water, not O2; the actual oxidative step is probably dehydrogenation of a hydrated form -CHOH-CH2- to -C(OH)=CH-, which tautomerizes non-enzymically to -CO-CH2-, giving (5-oxo-4,5-dihydro-2-furoyl)-CoA. Methylene blue, nitro blue, tetrazolium and a membrane fraction from Pseudomonas putida can act as acceptors.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.99.8\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.99.8\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.99.8\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.99.8\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.10': {'ec_num': '1.3.99.10',\n",
       "  'accepted_name': 'isovaleryl-CoA dehydrogenase',\n",
       "  'reaction': '3-methylbutanoyl-CoA + acceptor = 3-methylbut-2-enoyl-CoA + reduced acceptor',\n",
       "  'other_names': 'isovaleryl-coenzyme A dehydrogenase; isovaleroyl-coenzyme A dehydrogenase; 3-methylbutanoyl-CoA:(acceptor) oxidoreductase',\n",
       "  'sys_name': '3-methylbutanoyl-CoA:acceptor oxidoreductase',\n",
       "  'comments': 'A flavoprotein; forms with electron-transferring flavoprotein and EC 1.5.5.1 (electron-transferring-flavoprotein dehydrogenase) a system reducing ubiquinone. n-Pentanoate can act as donor. Not identical with EC 1.3.8.1 (butyryl-CoA dehydrogenase), EC 1.3.99.3 (acyl-CoA dehydrogenase) or EC 1.3.99.12 (2-methylacyl-CoA dehydrogenase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.11': {'ec_num': '1.3.99.11',\n",
       "  'accepted_name': 'dihydroorotate dehydrogenase',\n",
       "  'reaction': '(S)-dihydroorotate + acceptor = orotate + reduced acceptor',\n",
       "  'other_names': 'dihydroorotate:ubiquinone oxidoreductase; (S)-dihydroorotate:(acceptor) oxidoreductase',\n",
       "  'sys_name': '(S)-dihydroorotate:acceptor oxidoreductase',\n",
       "  'comments': 'An iron-zinc protein. Oxygen can act as acceptor, but much more slowly than 2,6-dichloroindophenol or 1,10-phenanthroline.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.12': {'ec_num': '1.3.99.12',\n",
       "  'accepted_name': '2-methylacyl-CoA dehydrogenase',\n",
       "  'reaction': '2-methylbutanoyl-CoA + acceptor = 2-methylbut-2-enoyl-CoA + reduced acceptor',\n",
       "  'other_names': 'branched-chain acyl-CoA dehydrogenase; 2-methyl branched chain acyl-CoA dehydrogenase; 2-methylbutanoyl-CoA:(acceptor) oxidoreductase',\n",
       "  'sys_name': '2-methylbutanoyl-CoA:acceptor oxidoreductase',\n",
       "  'comments': 'Also oxidizes 2-methylpropanoyl-CoA. Not identical with EC 1.3.8.1 (butyryl-CoA dehydrogenase), EC 1.3.8.7 (medium-chain acyl-CoA dehydrogenase), EC 1.3.8.8 (long-chain acyl-CoA dehydrogenase), EC 1.3.8.9 (very-long-chain acyl-CoA dehydrogenase) or EC 1.3.99.10 (isovaleryl-CoA dehydrogenase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.13': {'ec_num': '1.3.99.13',\n",
       "  'accepted_name': 'long-chain-acyl-CoA dehydrogenase',\n",
       "  'reaction': 'acyl-CoA + acceptor = 2,3-dehydroacyl-CoA + reduced acceptor',\n",
       "  'other_names': 'palmitoyl-CoA dehydrogenase; palmitoyl-coenzyme A dehydrogenase; long-chain acyl-coenzyme A dehydrogenase; long-chain-acyl-CoA:(acceptor) 2,3-oxidoreductase',\n",
       "  'sys_name': 'long-chain-acyl-CoA:acceptor 2,3-oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD); forms with another flavoprotein (’electron-transferring flavoprotein’) and EC 1.5.5.1 a system reducing ubiquinone and other acceptors. Not identical with EC 1.3.8.1 (butyryl-CoA dehydrogenase), EC 1.3.99.3 (acyl-CoA dehydrogenase), EC 1.3.99.10 (isovaleryl-CoA dehydrogenase) or EC 1.3.99.12 (2-methylacyl-CoA dehydrogenase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.14': {'ec_num': '1.3.99.14',\n",
       "  'accepted_name': 'cyclohexanone dehydrogenase',\n",
       "  'reaction': 'cyclohexanone + acceptor = cyclohex-2-enone + reduced acceptor',\n",
       "  'other_names': 'cyclohexanone:(acceptor) 2-oxidoreductase',\n",
       "  'sys_name': 'cyclohexanone:acceptor 2-oxidoreductase',\n",
       "  'comments': '2,6-Dichloroindophenol can act as acceptor. The corresponding ketones of cyclopentane and cycloheptane cannot act as donors.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.15': {'ec_num': '1.3.99.15',\n",
       "  'accepted_name': 'benzoyl-CoA reductase',\n",
       "  'reaction': 'benzoyl-CoA + reduced acceptor + 2 ATP + 2 H2O = cyclohexa-1,5-diene-1-carbonyl-CoA + acceptor + 2 ADP + 2 phosphate',\n",
       "  'other_names': 'benzoyl-CoA reductase (dearomatizing)',\n",
       "  'sys_name': 'cyclohexa-1,5-diene-1-carbonyl-CoA:acceptor oxidoreductase (aromatizing, ATP-forming)',\n",
       "  'comments': 'An iron-sulfur protein. Requires Mg2+ or Mn2+. Reduced methyl viologen can act as electron donor. Inactive towards aromatic acids that are not CoA esters but will also catalyse the reaction: ammonia + acceptor + 2 ADP + 2 phosphate + H2O = hydroxylamine + reduced acceptor + 2 ATP. In the presence of reduced acceptor, but in the absence of oxidizable substrate, the enzyme catalyses the hydrolysis of ATP to ADP plus phosphate.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.99.15\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.99.15\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.99.15\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.99.15\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.16': {'ec_num': '1.3.99.16',\n",
       "  'accepted_name': 'isoquinoline 1-oxidoreductase',\n",
       "  'reaction': 'isoquinoline + acceptor + H2O = isoquinolin-1(2H)-one + reduced acceptor',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'isoquinoline:acceptor 1-oxidoreductase (hydroxylating)',\n",
       "  'comments': 'The enzyme from Pseudomonas diminuta is specific towards N-containing N-heterocyclic substrates, including isoquinoline, isoquinolin-5-ol, phthalazine and quinazoline. Electron acceptors include 1,2-benzoquinone, cytochrome c, ferricyanide, iodonitrotetrazolium chloride, nitroblue tetrazolium, Meldola blue and phenazine methosulfate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.17': {'ec_num': '1.3.99.17',\n",
       "  'accepted_name': 'quinoline 2-oxidoreductase',\n",
       "  'reaction': 'quinoline + acceptor + H2O = quinolin-2(1H)-one + reduced acceptor',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'quinoline:acceptor 2-oxidoreductase (hydroxylating)',\n",
       "  'comments': 'Quinolin-2-ol, quinolin-7-ol, quinolin-8-ol, 3-, 4- and 8-methylquinolines and 8-chloroquinoline are substrates. Iodonitrotetrazolium chloride can act as an electron acceptor.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.99.17\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.99.17\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.99.17\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.99.17\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.3.99.17%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.18': {'ec_num': '1.3.99.18',\n",
       "  'accepted_name': 'quinaldate 4-oxidoreductase',\n",
       "  'reaction': 'quinaldate + acceptor + H2O = kynurenate + reduced acceptor',\n",
       "  'other_names': 'quinaldic acid 4-oxidoreductase',\n",
       "  'sys_name': 'quinoline-2-carboxylate:acceptor 4-oxidoreductase (hydroxylating)',\n",
       "  'comments': 'The enzyme from Pseudomonas sp. AK2 also acts on quinoline-8-carboxylate, whereas that from Serratia marcescens 2CC-1 will oxidize nicotinate; quinaldate is a substrate for both of these enzymes. 2,4,6-Trinitrobenzene sulfonate, 1,4-benzoquinone, 1,2-naphthoquinone, nitroblue tetrazolium, thionine and menadione will serve as an electron acceptor for the former enzyme and ferricyanide for the latter; Meldola blue, iodonitrotetrazolium chloride, phenazine methosulfate, 2,6-dichlorophenolindophenol and cytochrome c will act as electron acceptors for both.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.19': {'ec_num': '1.3.99.19',\n",
       "  'accepted_name': 'quinoline-4-carboxylate 2-oxidoreductase',\n",
       "  'reaction': 'quinoline-4-carboxylate + acceptor + H2O = 2-oxo-1,2-dihydroquinoline-4-carboxylate + reduced acceptor',\n",
       "  'other_names': 'quinaldic acid 4-oxidoreductase; quinoline-4-carboxylate:acceptor 2-oxidoreductase (hydroxylating)',\n",
       "  'sys_name': 'quinoline-4-carboxylate:acceptor 2-oxidoreductase (hydroxylating)',\n",
       "  'comments': 'A molybdenum—iron—sulfur flavoprotein with molybdopterin cytosine dinucleotide as the molybdenum cofactor. Quinoline, 4-methylquinoline and 4-chloroquinoline can also serve as substrates for the enzyme from Agrobacterium sp. 1B. Iodonitrotetrazolium chloride, thionine, menadione and 2,6-dichlorophenolindophenol can act as electron acceptors.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.20': {'ec_num': '1.3.99.20',\n",
       "  'accepted_name': '4-hydroxybenzoyl-CoA reductase',\n",
       "  'reaction': 'benzoyl-CoA + acceptor + H2O = 4-hydroxybenzoyl-CoA + reduced acceptor',\n",
       "  'other_names': '4-hydroxybenzoyl-CoA reductase (dehydroxylating); 4-hydroxybenzoyl-CoA:(acceptor) oxidoreductase',\n",
       "  'sys_name': 'benzoyl-CoA:acceptor oxidoreductase',\n",
       "  'comments': 'A molybdenum-flavin-iron-sulfur protein that is involved in the anaerobic pathway of phenol metabolism in bacteria. A ferredoxin with two [4Fe-4S] clusters functions as the natural electron donor [3]',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.99.20\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.99.20\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.99.20\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.99.20\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.3.99.20%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.21': {'ec_num': '1.3.99.21',\n",
       "  'accepted_name': '(R)-benzylsuccinyl-CoA dehydrogenase',\n",
       "  'reaction': '(R)-2-benzylsuccinyl-CoA + 2 electron-transferring flavoprotein = (E)-2-benzylidenesuccinyl-CoA + 2 reduced electron-transferring flavoprotein',\n",
       "  'other_names': 'BbsG; (R)-benzylsuccinyl-CoA:(acceptor) oxidoreductase',\n",
       "  'sys_name': '(R)-benzylsuccinyl-CoA:acceptor oxidoreductase',\n",
       "  'comments': 'Requires FAD as cofactor. The enzyme is highly specific for (R)-benzylsuccinyl-CoA and is inhibited by (S)-benzylsuccinyl-CoA. Forms the third step in the anaerobic toluene metabolic pathway in Thauera aromatica. Uses the ferricenium ion as the preferred artificial electron acceptor. Unlike other acyl-CoA dehydrogenases, this enzyme exhibits high substrate- and enantiomer specificity.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.3.99.21\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.3.99.21\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.3.99.21\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.3.99.21\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.22': {'ec_num': '1.3.99.22',\n",
       "  'accepted_name': 'coproporphyrinogen dehydrogenase',\n",
       "  'reaction': 'coproporphyrinogen III + 2 S-adenosyl-L-methionine = protoporphyrinogen IX + 2 CO2 + 2 L-methionine + 2 5′-deoxyadenosine',\n",
       "  'other_names': 'oxygen-independent coproporphyrinogen-III oxidase; HemN; coproporphyrinogen III oxidase',\n",
       "  'sys_name': 'coproporphyrinogen-III:S-adenosyl-L-methionine oxidoreductase (decarboxylating)',\n",
       "  'comments': 'This enzyme differs from EC 1.3.3.3, coproporphyrinogen oxidase, by using S-adenosyl-L-methionine (AdoMet) instead of oxygen as oxidant. It occurs mainly in bacteria, whereas eukaryotes use the oxygen-dependent oxidase. The reaction starts by using an electron from the reduced form of the enzyme’s [4Fe-4S] cluster to split AdoMet into methionine and the radical 5′-deoxyadenosin-5′-yl. This radical initiates attack on the 2-carboxyethyl groups, leading to their conversion into vinyl groups. This conversion, —·CH-CH2-COO- → —CH=CH2 + CO2 + e- replaces the electron initially used.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.3.99.23': {'ec_num': '1.3.99.23',\n",
       "  'accepted_name': 'all-trans-retinol 13,14-reductase',\n",
       "  'reaction': 'all-trans-13,14-dihydroretinol + acceptor = all-trans-retinol + reduced acceptor',\n",
       "  'other_names': 'retinol saturase; RetSat; (13,14)-all-trans-retinol saturase; all-trans-retinol:all-trans-13,14-dihydroretinol saturase',\n",
       "  'sys_name': 'all-trans-13,14-dihydroretinol:acceptor 13,14-oxidoreductase',\n",
       "  'comments': 'The reaction is only known to occur in the opposite direction to that given above, with the enzyme being specific for all-trans-retinol as substrate. Neither all-trans-retinoic acid nor 9-cis, 11-cis or 13-cis-retinol isomers are substrates. May play a role in the metabolism of vitamin A.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.1': {'ec_num': '1.4.1.1',\n",
       "  'accepted_name': 'alanine dehydrogenase',\n",
       "  'reaction': 'L-alanine + H2O + NAD+ = pyruvate + NH3 + NADH + H+',\n",
       "  'other_names': 'AlaDH; L-alanine dehydrogenase; NAD-linked alanine dehydrogenase; α-alanine dehydrogenase; NAD-dependent alanine dehydrogenase; alanine oxidoreductase; NADH-dependent alanine dehydrogenase',\n",
       "  'sys_name': 'L-alanine:NAD+ oxidoreductase (deaminating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.2': {'ec_num': '1.4.1.2',\n",
       "  'accepted_name': 'glutamate dehydrogenase',\n",
       "  'reaction': 'L-glutamate + H2O + NAD+ = 2-oxoglutarate + NH3 + NADH + H+',\n",
       "  'other_names': 'glutamic dehydrogenase; glutamate dehydrogenase (NAD); glutamate oxidoreductase; glutamic acid dehydrogenase; L-glutamate dehydrogenase; NAD-dependent glutamate dehydrogenase; NAD-dependent glutamic dehydrogenase; NAD-glutamate dehydrogenase; NAD-linked glutamate dehydrogenase; NAD-linked glutamic dehydrogenase; NAD-specific glutamic dehydrogenase; NAD-specific glutamate dehydrogenase; NAD:glutamate oxidoreductase; NADH-linked glutamate dehydrogenase',\n",
       "  'sys_name': 'L-glutamate:NAD+ oxidoreductase (deaminating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.3': {'ec_num': '1.4.1.3',\n",
       "  'accepted_name': 'glutamate dehydrogenase [NAD(P)+]',\n",
       "  'reaction': 'L-glutamate + H2O + NAD(P)+ = 2-oxoglutarate + NH3 + NAD(P)H + H+',\n",
       "  'other_names': 'glutamic dehydrogenase; glutamate dehydrogenase [NAD(P)]',\n",
       "  'sys_name': 'L-glutamate:NAD(P)+ oxidoreductase (deaminating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.4': {'ec_num': '1.4.1.4',\n",
       "  'accepted_name': 'glutamate dehydrogenase (NADP+)',\n",
       "  'reaction': 'L-glutamate + H2O + NADP+ = 2-oxoglutarate + NH3 + NADPH + H+',\n",
       "  'other_names': 'glutamic dehydrogenase; dehydrogenase, glutamate (nicotinamide adenine dinucleotide (phosphate)); glutamic acid dehydrogenase; L-glutamate dehydrogenase; L-glutamic acid dehydrogenase; NAD(P)-glutamate dehydrogenase; NAD(P)H-dependent glutamate dehydrogenase; glutamate dehydrogenase (NADP)',\n",
       "  'sys_name': 'L-glutamate:NADP+ oxidoreductase (deaminating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.5': {'ec_num': '1.4.1.5',\n",
       "  'accepted_name': 'L-amino-acid dehydrogenase',\n",
       "  'reaction': 'an L-amino acid + H2O + NAD+ = a 2-oxo carboxylate + NH3 + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'L-amino-acid:NAD+ oxidoreductase (deaminating)',\n",
       "  'comments': 'Acts on aliphatic amino acids.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.7': {'ec_num': '1.4.1.7',\n",
       "  'accepted_name': 'serine 2-dehydrogenase',\n",
       "  'reaction': 'L-serine + H2O + NAD+ = 3-hydroxypyruvate + NH3 + NADH + H+',\n",
       "  'other_names': 'L-serine:NAD oxidoreductase (deaminating); serine dehydrogenase',\n",
       "  'sys_name': 'L-serine:NAD+ 2-oxidoreductase (deaminating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.8': {'ec_num': '1.4.1.8',\n",
       "  'accepted_name': 'valine dehydrogenase (NADP+)',\n",
       "  'reaction': 'L-valine + H2O + NADP+ = 3-methyl-2-oxobutanoate + NH3 + NADPH + H+',\n",
       "  'other_names': 'valine dehydrogenase (nicotinanide adenine dinucleotide phosphate); valine dehydrogenase (NADP)',\n",
       "  'sys_name': 'L-valine:NADP+ oxidoreductase (deaminating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.9': {'ec_num': '1.4.1.9',\n",
       "  'accepted_name': 'leucine dehydrogenase',\n",
       "  'reaction': 'L-leucine + H2O + NAD+ = 4-methyl-2-oxopentanoate + NH3 + NADH + H+',\n",
       "  'other_names': 'L-leucine dehydrogenase; L-leucine:NAD+ oxidoreductase, deaminating; LeuDH',\n",
       "  'sys_name': 'L-leucine:NAD+ oxidoreductase (deaminating)',\n",
       "  'comments': 'Also acts on isoleucine, valine, norvaline and norleucine.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.10': {'ec_num': '1.4.1.10',\n",
       "  'accepted_name': 'glycine dehydrogenase',\n",
       "  'reaction': 'glycine + H2O + NAD+ = glyoxylate + NH3 + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'glycine:NAD+ oxidoreductase (deaminating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.11': {'ec_num': '1.4.1.11',\n",
       "  'accepted_name': 'L-erythro-3,5-diaminohexanoate dehydrogenase',\n",
       "  'reaction': 'L-erythro-3,5-diaminohexanoate + H2O + NAD+ = (S)-5-amino-3-oxohexanoate + NH3 + NADH + H+',\n",
       "  'other_names': 'L-3,5-diaminohexanoate dehydrogenase',\n",
       "  'sys_name': 'L-erythro-3,5-diaminohexanoate:NAD+ oxidoreductase (deaminating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.12': {'ec_num': '1.4.1.12',\n",
       "  'accepted_name': '2,4-diaminopentanoate dehydrogenase',\n",
       "  'reaction': '(2R,4S)-2,4-diaminopentanoate + H2O + NAD(P)+ = (2R)-2-amino-4-oxopentanoate + NH3 + NAD(P)H + H+',\n",
       "  'other_names': '2,4-diaminopentanoic acid C4 dehydrogenase',\n",
       "  'sys_name': '(2R,4S)-2,4-diaminopentanoate:NAD(P)+ oxidoreductase (deaminating)',\n",
       "  'comments': 'Also acts, more slowly, on 2,5-diaminohexanoate forming 2-amino-5-oxohexanoate, which then cyclizes non-enzymically to 1-pyrroline-2-methyl-5-carboxylate. It has equal activity with NAD+ and NADP+ [cf. EC 1.4.1.26, 2,4-diaminopentanoate dehydrogenase (NAD+)].',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.13': {'ec_num': '1.4.1.13',\n",
       "  'accepted_name': 'glutamate synthase (NADPH)',\n",
       "  'reaction': ' 2 L-glutamate + NADP+ = L-glutamine + 2-oxoglutarate + NADPH + H+ (overall reaction);;(1a) L-glutamate + NH3 = L-glutamine + H2O;;(1b) L-glutamate + NADP+ + H2O = NH3 + 2-oxoglutarate + NADPH + H+\\n ',\n",
       "  'other_names': 'glutamate (reduced nicotinamide adenine dinucleotide phosphate) synthase; L-glutamate synthase; L-glutamate synthetase; glutamate synthetase (NADP); NADPH-dependent glutamate synthase; glutamine-ketoglutaric aminotransferase; NADPH-glutamate synthase; NADPH-linked glutamate synthase; glutamine amide-2-oxoglutarate aminotransferase (oxidoreductase, NADP); L-glutamine:2-oxoglutarate aminotransferase, NADPH oxidizing; GOGAT',\n",
       "  'sys_name': 'L-glutamate:NADP+ oxidoreductase (transaminating)',\n",
       "  'comments': 'Binds FMN, FAD, 2 [4Fe-4S] clusters and 1 [3Fe-4S] cluster. The reaction takes place in the direction of L-glutamate production. The protein is composed of two subunits, α and β. The α subunit is composed of two domains, one hydrolysing L-glutamine to NH3 and L-glutamate (cf. EC 3.5.1.2, glutaminase), the other combining the produced NH3 with 2-oxoglutarate to produce a second molecule of L-glutamate (cf. EC 1.4.1.4, glutamate dehydrogenase [NADP+]). The β subunit transfers electrons from the cosubstrate. The NH3 is channeled within the α subunit through a 31 Å channel. The chanelling is very efficient and in the intact α-β complex ammonia is produced only within the complex. In the absence of the β subunit, coupling between the two domains of the α subunit is compromised and some ammonium can leak. ',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.14': {'ec_num': '1.4.1.14',\n",
       "  'accepted_name': 'glutamate synthase (NADH)',\n",
       "  'reaction': '2 L-glutamate + NAD+ = L-glutamine + 2-oxoglutarate + NADH + H+;;(1a) L-glutamate + NH3 = L-glutamine + H2O;;(1b) L-glutamate + NAD+ + H2O = NH3 + 2-oxoglutarate + NADH + H+',\n",
       "  'other_names': 'glutamate (reduced nicotinamide adenine dinucleotide) synthase; NADH: GOGAT; L-glutamate synthase (NADH); L-glutamate synthetase; NADH-glutamate synthase; NADH-dependent glutamate synthase',\n",
       "  'sys_name': 'L-glutamate:NAD+ oxidoreductase (transaminating)',\n",
       "  'comments': 'A flavoprotein (FMN). The reaction takes place in the direction of L-glutamate production. The protein is composed of two domains, one hydrolysing L-glutamine to NH3 and L-glutamate (cf. EC 3.5.1.2, glutaminase), the other combining the produced NH3 with 2-oxoglutarate to produce a second molecule of L-glutamate (cf. EC 1.4.1.2, glutamate dehydrogenase). ',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.15': {'ec_num': '1.4.1.15',\n",
       "  'accepted_name': 'lysine dehydrogenase',\n",
       "  'reaction': 'L-lysine + NAD+ = 1,2-didehydropiperidine-2-carboxylate + NH3 + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'L-lysine:NAD+ oxidoreductase (deaminating, cyclizing)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.16': {'ec_num': '1.4.1.16',\n",
       "  'accepted_name': 'diaminopimelate dehydrogenase',\n",
       "  'reaction': 'meso-2,6-diaminoheptanedioate + H2O + NADP+ = L-2-amino-6-oxoheptanedioate + NH3 + NADPH + H+',\n",
       "  'other_names': 'meso-α,ε-diaminopimelate dehydrogenase; meso-diaminopimelate dehydrogenase',\n",
       "  'sys_name': 'meso-2,6-diaminoheptanedioate:NADP+ oxidoreductase (deaminating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.17': {'ec_num': '1.4.1.17',\n",
       "  'accepted_name': 'N-methylalanine dehydrogenase',\n",
       "  'reaction': 'N-methyl-L-alanine + H2O + NADP+ = pyruvate + methylamine + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'N-methyl-L-alanine:NADP+ oxidoreductase (demethylating, deaminating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.18': {'ec_num': '1.4.1.18',\n",
       "  'accepted_name': 'lysine 6-dehydrogenase',\n",
       "  'reaction': 'L-lysine + NAD+ = (S)-2,3,4,5-tetrahydropyridine-2-carboxylate + NADH + H+ + NH3 (overall reaction);;(1a) L-lysine + NAD+ + H2O = (S)-2-amino-6-oxohexanoate + NADH + H+ + NH3;;(1b) (S)-2-amino-6-oxohexanoate = (S)-2,3,4,5-tetrahydropyridine-2-carboxylate + H2O (spontaneous)',\n",
       "  'other_names': 'L-lysine ε-dehydrogenase; L-lysine 6-dehydrogenase; LysDH',\n",
       "  'sys_name': 'L-lysine:NAD+ 6-oxidoreductase (deaminating)',\n",
       "  'comments': 'The enzyme is highly specific for L-lysine as substrate, although S-(2-aminoethyl)-L-cysteine can act as a substrate, but more slowly. While the enzyme from Agrobacterium tumefaciens can use only NAD+, that from the thermophilic bacterium Geobacillus stearothermophilus can also use NADP+, but more slowly [1,4].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '(S)-2-amino-6-oxohexanoate = L-2-aminoadipate 6-semialdehyde = L-allysine\\nL-1-piperideine 6-carboxylate = (S)-2,3,4,5-tetrahydropyridine-2-carboxylate = (S)-1,6-didehydropiperidine-2-carboxylate',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.19': {'ec_num': '1.4.1.19',\n",
       "  'accepted_name': 'tryptophan dehydrogenase',\n",
       "  'reaction': 'L-tryptophan + NAD(P)+ + H2O = (indol-3-yl)pyruvate + NH3 + NAD(P)H + H+',\n",
       "  'other_names': 'NAD(P)+-L-tryptophan dehydrogenase; L-tryptophan dehydrogenase; L-Trp-dehydrogenase; TDH',\n",
       "  'sys_name': 'L-tryptophan:NAD(P)+ oxidoreductase (deaminating)',\n",
       "  'comments': 'Activated by Ca2+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.20': {'ec_num': '1.4.1.20',\n",
       "  'accepted_name': 'phenylalanine dehydrogenase',\n",
       "  'reaction': 'L-phenylalanine + H2O + NAD+ = phenylpyruvate + NH3 + NADH + H+',\n",
       "  'other_names': 'L-phenylalanine dehydrogenase; PHD',\n",
       "  'sys_name': 'L-phenylalanine:NAD+ oxidoreductase (deaminating)',\n",
       "  'comments': 'The enzymes from Bacillus badius and Sporosarcina ureae are highly specific for L-phenylalanine; that from Bacillus sphaericus also acts on L-tyrosine.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.1.21': {'ec_num': '1.4.1.21',\n",
       "  'accepted_name': 'aspartate dehydrogenase',\n",
       "  'reaction': 'L-aspartate + H2O + NAD(P)+ = oxaloacetate + NH3 + NAD(P)H + H+ (overall reaction);;(1a) L-aspartate + NAD(P)+ = 2-iminosuccinate + NAD(P)H + H+;;(1b) 2-iminosuccinate + H2O = oxaloacetate + NH3 (spontaneous)',\n",
       "  'other_names': 'AspDH; NAD-dependent aspartate dehydrogenase; NADH2-dependent aspartate dehydrogenase; NADP+-dependent aspartate dehydrogenase; nadX (gene name); L-aspartate:NAD(P)+ oxidoreductase (deaminating)',\n",
       "  'sys_name': 'L-aspartate:NAD(P)+ oxidoreductase (2-iminosuccinate-forming)',\n",
       "  'comments': 'The enzyme is strictly specific for L-aspartate as substrate. It produces the unstable compound 2-iminosuccinate, which, in the presence of water, hydrolyses spontaneously to form oxaloacetate. The enzyme from some archaea and thermophilic bacteria is likely to transfer 2-iminosuccinate directly to EC 2.5.1.72, quinolinate synthase, preventing its hydrolysis and enabling the de novo biosynthesis of NAD+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.2.1': {'ec_num': '1.4.2.1',\n",
       "  'accepted_name': 'glycine dehydrogenase (cytochrome)',\n",
       "  'reaction': 'glycine + H2O + 2 ferricytochrome c = glyoxylate + NH3 + 2 ferrocytochrome c + 2 H+',\n",
       "  'other_names': 'glycine—cytochrome c reductase',\n",
       "  'sys_name': 'glycine:ferricytochrome-c oxidoreductase (deaminating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.3.1': {'ec_num': '1.4.3.1',\n",
       "  'accepted_name': 'D-aspartate oxidase',\n",
       "  'reaction': 'D-aspartate + H2O + O2 = oxaloacetate + NH3 + H2O2',\n",
       "  'other_names': 'aspartic oxidase; D-aspartic oxidase',\n",
       "  'sys_name': 'D-aspartate:oxygen oxidoreductase (deaminating)',\n",
       "  'comments': 'A flavoprotein (FAD).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.3.2': {'ec_num': '1.4.3.2',\n",
       "  'accepted_name': 'L-amino-acid oxidase',\n",
       "  'reaction': 'an L-amino acid + H2O + O2 = a 2-oxo carboxylate + NH3 + H2O2',\n",
       "  'other_names': 'ophio-amino-acid oxidase (ambiguous)',\n",
       "  'sys_name': 'L-amino-acid:oxygen oxidoreductase (deaminating)',\n",
       "  'comments': 'A flavoprotein (FAD).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.3.3': {'ec_num': '1.4.3.3',\n",
       "  'accepted_name': 'D-amino-acid oxidase',\n",
       "  'reaction': 'a D-amino acid + H2O + O2 = a 2-oxo carboxylate + NH3 + H2O2',\n",
       "  'other_names': 'ophio-amino-acid oxidase (ambiguous); L-amino acid:O2 oxidoreductase; new yellow enzyme',\n",
       "  'sys_name': 'D-amino-acid:oxygen oxidoreductase (deaminating)',\n",
       "  'comments': 'A flavoprotein (FAD). Wide specificity for D-amino acids. Also acts on glycine.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.3.4': {'ec_num': '1.4.3.4',\n",
       "  'accepted_name': 'monoamine oxidase',\n",
       "  'reaction': 'RCH2NHR′ + H2O + O2 = RCHO + R′NH2 + H2O2',\n",
       "  'other_names': 'adrenalin oxidase; adrenaline oxidase; amine oxidase (ambiguous); amine oxidase (flavin-containing); amine:oxygen oxidoreductase (deaminating) (flavin-containing); epinephrine oxidase; MAO; MAO A; MAO B; MAO-A; MAO-B; monoamine oxidase A; monoamine oxidase B; monoamine:O2 oxidoreductase (deaminating); polyamine oxidase (ambiguous); serotonin deaminase; spermidine oxidase (ambiguous); spermine oxidase (ambiguous); tyraminase; tyramine oxidase',\n",
       "  'sys_name': 'amine:oxygen oxidoreductase (deaminating)',\n",
       "  'comments': 'A mitochondrial outer-membrane flavoprotein (FAD) that catalyses the oxidative deamination of neurotransmitters and biogenic amines [3]. Acts on primary amines, and also on some secondary and tertiary amines. It differs from EC 1.4.3.21, primary-amine oxidase as it can oxidize secondary and tertiary amines but not methylamine. This enzyme is inhibited by  acetylenic compounds such as chlorgyline, 1-deprenyl and pargyline but, unlike EC 1.4.3.21 and EC 1.4.3.22 (diamine oxidase), it is not inhibited by semicarbazide.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.4.3.4\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.4.3.4\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.4.3.4\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.4.3.4\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.4.3.4%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.3.5': {'ec_num': '1.4.3.5',\n",
       "  'accepted_name': 'pyridoxal 5′-phosphate synthase',\n",
       "  'reaction': '(1) pyridoxamine 5′-phosphate + H2O + O2 = pyridoxal 5′-phosphate + NH3 + H2O2;;(2) pyridoxine 5′-phosphate + O2 = pyridoxal 5′-phosphate + H2O2',\n",
       "  'other_names': 'pyridoxamine 5′-phosphate oxidase; pyridoxamine phosphate oxidase; pyridoxine (pyridoxamine)phosphate oxidase; pyridoxine (pyridoxamine) 5′-phosphate oxidase; pyridoxaminephosphate oxidase (EC 1.4.3.5: deaminating); PMP oxidase; pyridoxol-5′-phosphate:oxygen oxidoreductase (deaminating) (incorrect); pyridoxamine-phosphate oxidase; PdxH',\n",
       "  'sys_name': 'pyridoxamine-5′-phosphate:oxygen oxidoreductase (deaminating)',\n",
       "  'comments': 'A flavoprotein (FMN). In Escherichia coli, the cofactor pyridoxal 5′-phosphate is synthesized de novo by a pathway that involves EC 1.2.1.72 (erythrose-4-phosphate dehydrogenase), EC 1.1.1.290 (4-phosphoerythronate dehydrogenase), EC 2.6.1.52 (phosphoserine transaminase), EC 1.1.1.262 (4-hydroxythreonine-4-phosphate dehydrogenase), EC 2.6.99.2 (pyridoxine 5′-phosphate synthase) and EC 1.4.3.5 (with pyridoxine 5′-phosphate as substrate). N4′-Substituted pyridoxamine derivatives are also oxidized in reaction (1) to form pyridoxal 5-phosphate and the corresponding primary amine.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'pyridoxamine = 4-aminomethyl-3-hydroxy-5-hydroxymethyl-2-methylpyridine',\n",
       "  'last_change': '2023-07-27 09:35:41'},\n",
       " '1.4.3.6': {'ec_num': '1.4.3.6',\n",
       "  'accepted_name': 'amine oxidase (copper-containing)',\n",
       "  'reaction': 'RCH2NH2 + H2O + O2 = RCHO + NH3 + H2O2',\n",
       "  'other_names': 'diamine oxidase; diamino oxhydrase; histaminase; amine oxidase; monoamine oxidase; amine oxidase (pyridoxal containing); benzylamine oxidase; histamine deaminase; histamine oxidase; Cu-amine oxidase; amine oxygen oxidoreductase; diamine:O2 oxidoreductase (deaminating); semicarbazide-sensitive amine oxidase; SSAO',\n",
       "  'sys_name': 'amine:oxygen oxidoreductase (deaminating) (copper-containing)',\n",
       "  'comments': 'A group of enzymes including those oxidizing primary monoamines, diamines and histamine. They are copper quinoproteins. One form of EC 1.3.1.15 orotate reductase (NADPH) from rat kidney also catalyses this reaction.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.3.7': {'ec_num': '1.4.3.7',\n",
       "  'accepted_name': 'D-glutamate oxidase',\n",
       "  'reaction': 'D-glutamate + H2O + O2 = 2-oxoglutarate + NH3 + H2O2',\n",
       "  'other_names': 'D-glutamic oxidase; D-glutamic acid oxidase',\n",
       "  'sys_name': 'D-glutamate:oxygen oxidoreductase (deaminating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.3.8': {'ec_num': '1.4.3.8',\n",
       "  'accepted_name': 'ethanolamine oxidase',\n",
       "  'reaction': 'ethanolamine + H2O + O2 = glycolaldehyde + NH3 + H2O2',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'ethanolamine:oxygen oxidoreductase (deaminating)',\n",
       "  'comments': 'A cobamide-protein.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.3.10': {'ec_num': '1.4.3.10',\n",
       "  'accepted_name': 'putrescine oxidase',\n",
       "  'reaction': 'putrescine + O2 + H2O = 4-aminobutanal + NH3 + H2O2',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'putrescine:oxygen oxidoreductase (deaminating)',\n",
       "  'comments': 'A flavoprotein (FAD). 4-Aminobutanal condenses non-enzymically to 1-pyrroline.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'putrescine = butane-1,4-diamine',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.3.11': {'ec_num': '1.4.3.11',\n",
       "  'accepted_name': 'L-glutamate oxidase',\n",
       "  'reaction': 'L-glutamate + O2 + H2O = 2-oxoglutarate + NH3 + H2O2',\n",
       "  'other_names': 'glutamate (acceptor) dehydrogenase; glutamate oxidase; glutamic acid oxidase; glutamic dehydrogenase (acceptor); L-glutamic acid oxidase',\n",
       "  'sys_name': 'L-glutamate:oxygen oxidoreductase (deaminating)',\n",
       "  'comments': 'A flavoprotein (FAD).The enzyme from Azotobacter previously listed under this number, which did not produce H2O2, was a crude cell-free extract that probably contained catalase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.3.12': {'ec_num': '1.4.3.12',\n",
       "  'accepted_name': 'cyclohexylamine oxidase',\n",
       "  'reaction': 'cyclohexylamine + O2 + H2O = cyclohexanone + NH3 + H2O2',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'cyclohexylamine:oxygen oxidoreductase (deaminating)',\n",
       "  'comments': 'A flavoprotein (FAD). Some other cyclic amines can act instead of cyclohexylamine, but not simple aliphatic and aromatic amides.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.4.3.12\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.4.3.12\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.4.3.12\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.4.3.12\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.3.13': {'ec_num': '1.4.3.13',\n",
       "  'accepted_name': 'protein-lysine 6-oxidase',\n",
       "  'reaction': '[protein]-L-lysine + O2 + H2O = [protein]-(S)-2-amino-6-oxohexanoate + NH3 + H2O2',\n",
       "  'other_names': 'lysyl oxidase',\n",
       "  'sys_name': 'protein-L-lysine:oxygen 6-oxidoreductase (deaminating)',\n",
       "  'comments': 'Also acts on protein 5-hydroxylysine. This enzyme catalyses the final known enzymic step required for collagen and elastin cross-linking in the biosynthesis of normal mature extracellular matrices [4]. These reactions play an important role for the development, elasticity and extensibility of connective tissue. The enzyme is also active on free amines, such as cadaverine or benzylamine [4,5]. Some isoforms can also use [protein]-N(6)-acetyl-L-lysine as substrate deacetamidating the substrate [6]. ',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '(S)-2-amino-6-oxohexanoate = L-allysine',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.3.14': {'ec_num': '1.4.3.14',\n",
       "  'accepted_name': 'L-lysine oxidase',\n",
       "  'reaction': 'L-lysine + O2 + H2O = 6-amino-2-oxohexanoate + NH3 + H2O2',\n",
       "  'other_names': 'L-lysine α-oxidase; L-lysyl-α-oxidase',\n",
       "  'sys_name': 'L-lysine:oxygen 2-oxidoreductase (deaminating)',\n",
       "  'comments': 'Also acts, more slowly, on L-ornithine, L-phenylalanine, L-arginine and L-histidine.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.3.15': {'ec_num': '1.4.3.15',\n",
       "  'accepted_name': 'D-glutamate(D-aspartate) oxidase',\n",
       "  'reaction': '(1) D-glutamate + H2O + O2 = 2-oxoglutarate + NH3 + H2O2;;(2) D-aspartate + H2O + O2 = oxaloacetate + NH3 + H2O2',\n",
       "  'other_names': 'D-glutamic-aspartic oxidase; D-monoaminodicarboxylic acid oxidase',\n",
       "  'sys_name': 'D-glutamate(D-aspartate):oxygen oxidoreductase (deaminating)',\n",
       "  'comments': 'A flavoprotein (FAD). D-Glutamate and D-aspartate are oxidized at the same rate. Other D-monoaminodicarboxylates, and other D- and L-amino acids, are not oxidized. cf. EC 1.4.3.7, D-glutamate oxidase and EC 1.4.3.1, D-aspartate oxidase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.3.16': {'ec_num': '1.4.3.16',\n",
       "  'accepted_name': 'L-aspartate oxidase',\n",
       "  'reaction': 'L-aspartate + O2 = iminosuccinate + H2O2',\n",
       "  'other_names': 'NadB; Laspo; AO',\n",
       "  'sys_name': 'L-aspartate:oxygen oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD). L-Aspartate oxidase catalyses the first step in the de novo biosynthesis of NAD+ in some bacteria. O2 can be replaced by fumarate as electron acceptor, yielding succinate [5]. The ability of the enzyme to use both O2 and fumarate in cofactor reoxidation enables it to function under both aerobic and anaerobic conditions [5]. Iminosuccinate can either be hydrolysed to form oxaloacetate and NH3 or can be used by EC 2.5.1.72, quinolinate synthase, in the production of quinolinate. The enzyme is a member of the succinate dehydrogenase/fumarate-reductase family of enzymes [5]. ',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.3.19': {'ec_num': '1.4.3.19',\n",
       "  'accepted_name': 'glycine oxidase',\n",
       "  'reaction': 'glycine + H2O + O2 = glyoxylate + NH3 + H2O2 (overall reaction);;(1a) glycine + O2 = 2-iminoacetate + H2O2;;(1b) 2-iminoacetate + H2O = glyoxylate + NH3',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'glycine:oxygen oxidoreductase (deaminating)',\n",
       "  'comments': 'A flavoenzyme containing non-covalently bound FAD. The enzyme from Bacillus subtilis is active with glycine, sarcosine, N-ethylglycine, D-alanine, D-α-aminobutyrate, D-proline, D-pipecolate and N-methyl-D-alanine. It differs from EC 1.4.3.3, D-amino-acid oxidase, due to its activity on sarcosine and D-pipecolate. The intermediate 2-iminoacetate is used directly by EC 2.8.1.10, thiazole synthase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:21'},\n",
       " '1.4.4.2': {'ec_num': '1.4.4.2',\n",
       "  'accepted_name': 'glycine dehydrogenase (aminomethyl-transferring)',\n",
       "  'reaction': 'glycine + [glycine-cleavage complex H protein]-N6-lipoyl-L-lysine = [glycine-cleavage complex H protein]-S-aminomethyl-N6-dihydrolipoyl-L-lysine + CO2',\n",
       "  'other_names': 'P-protein; glycine decarboxylase; glycine-cleavage complex; glycine:lipoylprotein oxidoreductase (decarboxylating and acceptor-aminomethylating); protein P1; glycine dehydrogenase (decarboxylating); glycine cleavage system P-protein; glycine-cleavage complex P-protein',\n",
       "  'sys_name': 'glycine:H-protein-lipoyllysine oxidoreductase (decarboxylating, acceptor-amino-methylating)',\n",
       "  'comments': 'A pyridoxal-phosphate protein. A component of the glycine cleavage system, which is composed of four components that only loosely associate: the P protein (EC 1.4.4.2), the T protein (EC 2.1.2.10, aminomethyltransferase), the L protein (EC 1.8.1.4, dihydrolipoyl dehydrogenase) and the lipoyl-bearing H protein [3]. Previously known as glycine synthase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'dihydrolipoyl group',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.4.7.1': {'ec_num': '1.4.7.1',\n",
       "  'accepted_name': 'glutamate synthase (ferredoxin)',\n",
       "  'reaction': '2 L-glutamate + 2 oxidized ferredoxin = L-glutamine + 2-oxoglutarate + 2 reduced ferredoxin + 2 H+ (overall reaction);;(1a) L-glutamate + NH3 = L-glutamine + H2O;;(1b) L-glutamate + 2 oxidized ferredoxin + H2O = NH3 + 2-oxoglutarate + 2 reduced ferredoxin + 2 H+',\n",
       "  'other_names': 'ferredoxin-dependent glutamate synthase; ferredoxin-glutamate synthase; glutamate synthase (ferredoxin-dependent)',\n",
       "  'sys_name': 'L-glutamate:ferredoxin oxidoreductase (transaminating)',\n",
       "  'comments': 'Binds a [3Fe-4S] cluster as well as FAD and FMN. The protein is composed of two domains, one hydrolysing L-glutamine to NH3 and L-glutamate (cf. EC 3.5.1.2, glutaminase), the other combining the produced NH3 with 2-oxoglutarate to produce a second molecule of L-glutamate. The NH3 is channeled through a 24 Å channel in the active protein. No hydrolysis of glutamine takes place without ferredoxin and 2-oxoglutarate being bound to the protein [5,6].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.4.99.1': {'ec_num': '1.4.99.1',\n",
       "  'accepted_name': 'D-amino-acid dehydrogenase',\n",
       "  'reaction': 'a D-amino acid + H2O + acceptor = a 2-oxo carboxylate + NH3 + reduced acceptor',\n",
       "  'other_names': 'D-amino-acid:(acceptor) oxidoreductase (deaminating)',\n",
       "  'sys_name': 'D-amino-acid:acceptor oxidoreductase (deaminating)',\n",
       "  'comments': 'A flavoprotein (FAD). Acts to some extent on all D-amino acids, except D-aspartate and D-glutamate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.4.99.2': {'ec_num': '1.4.99.2',\n",
       "  'accepted_name': 'taurine dehydrogenase',\n",
       "  'reaction': 'taurine + H2O + acceptor = 2-sulfoacetaldehyde + NH3 + reduced acceptor',\n",
       "  'other_names': 'taurine:(acceptor) oxidoreductase (deaminating)',\n",
       "  'sys_name': 'taurine:acceptor oxidoreductase (deaminating)',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.4.99.2\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.4.99.2\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.4.99.2\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.4.99.2\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '2-sulfoacetaldehyde = 2-oxoethanesulfonate<br>\\ntaurine = 2-aminoethanesulfonate',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.4.99.3': {'ec_num': '1.4.99.3',\n",
       "  'accepted_name': 'amine dehydrogenase',\n",
       "  'reaction': 'RCH2NH2 + H2O + acceptor = RCHO + NH3 + reduced acceptor',\n",
       "  'other_names': 'methylamine dehydrogenase; primary-amine dehydrogenase; amine: (acceptor) oxidoreductase (deaminating); primary-amine:(acceptor) oxidoreductase (deaminating)',\n",
       "  'sys_name': 'primary-amine:acceptor oxidoreductase (deaminating)',\n",
       "  'comments': 'A quinoprotein.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.4.99.3\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.4.99.3\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.4.99.3\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.4.99.3\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.4.99.3%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.4.99.4': {'ec_num': '1.4.99.4',\n",
       "  'accepted_name': 'aralkylamine dehydrogenase',\n",
       "  'reaction': 'RCH2NH2 + H2O + acceptor = RCHO + NH3 + reduced acceptor',\n",
       "  'other_names': 'aromatic amine dehydrogenase; dehydrogenase, arylamine; tyramine dehydrogenase; aralkylamine:(acceptor) oxidoreductase (deaminating)',\n",
       "  'sys_name': 'aralkylamine:acceptor oxidoreductase (deaminating)',\n",
       "  'comments': 'Phenazine methosulfate can act as acceptor. Acts on aromatic amines and, more slowly, on some long-chain aliphatic amines, but not on methylamine or ethylamine (cf. EC 1.4.99.3 amine dehydrogenase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.4.99.5': {'ec_num': '1.4.99.5',\n",
       "  'accepted_name': 'glycine dehydrogenase (cyanide-forming)',\n",
       "  'reaction': 'glycine + 2 acceptor = hydrogen cyanide + CO2 + 2 reduced acceptor',\n",
       "  'other_names': 'hydrogen cyanide synthase; HCN synthase',\n",
       "  'sys_name': 'glycine:acceptor oxidoreductase (hydrogen-cyanide-forming)',\n",
       "  'comments': 'The enzyme from Pseudomonas sp. contains FAD. The enzyme is membrane-bound, and the 2-electron acceptor is a component of the respiratory chain. The enzyme can act with various artificial electron acceptors, including phenazine methosulfate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.1': {'ec_num': '1.5.1.1',\n",
       "  'accepted_name': '1-piperideine-2-carboxylate/1-pyrroline-2-carboxylate reductase [NAD(P)H]',\n",
       "  'reaction': ' (1) L-pipecolate + NAD(P)+ = 1-piperideine-2-carboxylate + NAD(P)H + H+;; (2) L-proline + NAD(P)+ = 1-pyrroline-2-carboxylate + NAD(P)H + H+ \\n ',\n",
       "  'other_names': 'Δ1-pyrroline-2-carboxylate reductase; DELTA1-pyrroline-2-carboxylate reductase; DELTA1-piperideine-2-carboxylate/1-pyrroline-2-carboxylate reductase (ambiguous); AbLhpI; pyrroline-2-carboxylate reductase; L-proline:NAD(P)+ 2-oxidoreductase',\n",
       "  'sys_name': 'L-pipecolate/L-proline:NAD(P)+ 2-oxidoreductase',\n",
       "  'comments': 'The enzymes, characterized from the bacterium Azospirillum brasilense, is involved in trans-3-hydroxy-L-proline metabolism. In contrast to EC 1.5.1.21, 1-piperideine-2-carboxylate/1-pyrroline-2-carboxylate reductase (NADPH), which is specific for NADPH, this enzyme shows similar activity with NADPH and NADH.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.2': {'ec_num': '1.5.1.2',\n",
       "  'accepted_name': 'pyrroline-5-carboxylate reductase',\n",
       "  'reaction': 'L-proline + NAD(P)+ = 1-pyrroline-5-carboxylate + NAD(P)H + H+',\n",
       "  'other_names': 'proline oxidase; L-proline oxidase; 1-pyrroline-5-carboxylate reductase; NADPH-L-Δ1-pyrroline carboxylic acid reductase; L-proline-NAD(P)+ 5-oxidoreductase',\n",
       "  'sys_name': 'L-proline:NAD(P)+ 5-oxidoreductase',\n",
       "  'comments': 'Also reduces 1-pyrroline-3-hydroxy-5-carboxylate to L-hydroxyproline.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.3': {'ec_num': '1.5.1.3',\n",
       "  'accepted_name': 'dihydrofolate reductase',\n",
       "  'reaction': '5,6,7,8-tetrahydrofolate + NADP+ = 7,8-dihydrofolate + NADPH + H+',\n",
       "  'other_names': 'tetrahydrofolate dehydrogenase; DHFR; pteridine reductase:dihydrofolate reductase; dihydrofolate reductase:thymidylate synthase; thymidylate synthetase-dihydrofolate reductase; folic acid reductase; folic reductase; dihydrofolic acid reductase; dihydrofolic reductase; 7,8-dihydrofolate reductase; NADPH-dihydrofolate reductase',\n",
       "  'sys_name': '5,6,7,8-tetrahydrofolate:NADP+ oxidoreductase',\n",
       "  'comments': 'The enzyme from animals and some micro-organisms also slowly reduces folate to 5,6,7,8-tetrahydrofolate.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-08-04 18:56:11'},\n",
       " '1.5.1.5': {'ec_num': '1.5.1.5',\n",
       "  'accepted_name': 'methylenetetrahydrofolate dehydrogenase (NADP+)',\n",
       "  'reaction': '5,10-methylenetetrahydrofolate + NADP+ = 5,10-methenyltetrahydrofolate + NADPH + H+',\n",
       "  'other_names': 'N5,N10-methylenetetrahydrofolate dehydrogenase; 5,10-methylenetetrahydrofolate:NADP oxidoreductase; 5,10-methylenetetrahydrofolate dehydrogenase; methylenetetrahydrofolate dehydrogenase; methylenetetrahydrofolate dehydrogenase (NADP)',\n",
       "  'sys_name': '5,10-methylenetetrahydrofolate:NADP+ oxidoreductase',\n",
       "  'comments': 'In eukaryotes, occurs as a trifunctional enzyme also having methenyltetrahydrofolate cyclohydrolase (EC 3.5.4.9) and formate—tetrahydrofolate ligase (EC 6.3.4.3) activity. In some prokaryotes occurs as a bifunctional enzyme also having methenyltetrahydrofolate cyclohydrolase activity (EC 3.5.4.9).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-08-04 18:56:14'},\n",
       " '1.5.1.6': {'ec_num': '1.5.1.6',\n",
       "  'accepted_name': 'formyltetrahydrofolate dehydrogenase',\n",
       "  'reaction': '10-formyltetrahydrofolate + NADP+ + H2O = tetrahydrofolate + CO2 + NADPH + H+',\n",
       "  'other_names': '10-formyl tetrahydrofolate:NADP oxidoreductase; 10-formyl-H2PtGlu:NADP oxidoreductase; 10-formyl-H4folate dehydrogenase; N10-formyltetrahydrofolate dehydrogenase; 10-formyltetrahydrofolate dehydrogenase',\n",
       "  'sys_name': '10-formyltetrahydrofolate:NADP+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-07-27 09:35:46'},\n",
       " '1.5.1.7': {'ec_num': '1.5.1.7',\n",
       "  'accepted_name': 'saccharopine dehydrogenase (NAD+, L-lysine-forming)',\n",
       "  'reaction': 'N6-(L-1,3-dicarboxypropyl)-L-lysine + NAD+ + H2O = L-lysine + 2-oxoglutarate + NADH + H+',\n",
       "  'other_names': 'lysine-2-oxoglutarate reductase; dehydrogenase, saccharopine (nicotinamide adenine dinucleotide, lysine forming); ε-N-(L-glutaryl-2)-L-lysine:NAD oxidoreductase (L-lysine forming); N6-(glutar-2-yl)-L-lysine:NAD oxidoreductase (L-lysine-forming); 6-N-(L-1,3-dicarboxypropyl)-L-lysine:NAD+ oxidoreductase (L-lysine-forming)',\n",
       "  'sys_name': 'N6-(L-1,3-dicarboxypropyl)-L-lysine:NAD+ oxidoreductase (L-lysine-forming)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'L-saccharopine = N6-(L-1,3-dicarboxypropyl)-L-lysine',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.8': {'ec_num': '1.5.1.8',\n",
       "  'accepted_name': 'saccharopine dehydrogenase (NADP+, L-lysine-forming)',\n",
       "  'reaction': 'N6-(L-1,3-dicarboxypropyl)-L-lysine + NADP+ + H2O = L-lysine + 2-oxoglutarate + NADPH + H+',\n",
       "  'other_names': 'lysine-2-oxoglutarate reductase; lysine-ketoglutarate reductase; L-lysine-α-ketoglutarate reductase; lysine:α-ketoglutarate:TPNH oxidoreductase (ε-N-[gultaryl-2]-L-lysine forming); saccharopine (nicotinamide adenine dinucleotide phosphate, lysine-forming) dehydrogenase; 6-N-(L-1,3-dicarboxypropyl)-L-lysine:NADP+ oxidoreductase (L-lysine-forming)',\n",
       "  'sys_name': 'N6-(L-1,3-dicarboxypropyl)-L-lysine:NADP+ oxidoreductase (L-lysine-forming)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'L-saccharopine = N6-(L-1,3-dicarboxypropyl)-L-lysine',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.9': {'ec_num': '1.5.1.9',\n",
       "  'accepted_name': 'saccharopine dehydrogenase (NAD+, L-glutamate-forming)',\n",
       "  'reaction': 'N6-(L-1,3-dicarboxypropyl)-L-lysine + NAD+ + H2O = L-glutamate + (S)-2-amino-6-oxohexanoate + NADH + H+',\n",
       "  'other_names': 'dehydrogenase, saccharopine (nicotinamide adenine dinucleotide, glutamate-forming); saccharopin dehydrogenase; NAD+ oxidoreductase (L-2-aminoadipic-δ-semialdehyde and glutamate forming); aminoadipic semialdehyde synthase; 6-N-(L-1,3-dicarboxypropyl)-L-lysine:NAD+ oxidoreductase (L-glutamate-forming)',\n",
       "  'sys_name': 'N6-(L-1,3-dicarboxypropyl)-L-lysine:NAD+ oxidoreductase (L-glutamate-forming)',\n",
       "  'comments': 'The activities of this enzyme along with EC 1.5.1.8, saccharopine dehydrogenase (NADP+, L-lysine-forming), occur on a single protein.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'saccharopine = N6-(L-1,3-dicarboxypropyl)-L-lysine\\n(S)-2-amino-6-oxohexanoate = L-2-aminoadipate 6-semialdehyde = L-allysine',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.10': {'ec_num': '1.5.1.10',\n",
       "  'accepted_name': 'saccharopine dehydrogenase (NADP+, L-glutamate-forming)',\n",
       "  'reaction': 'N6-(L-1,3-dicarboxypropyl)-L-lysine + NADP+ + H2O = L-glutamate + (S)-2-amino-6-oxohexanoate + NADPH + H+',\n",
       "  'other_names': 'saccharopine (nicotinamide adenine dinucleotide phosphate, glutamate-forming) dehydrogenase; aminoadipic semialdehyde-glutamic reductase; aminoadipate semialdehyde-glutamate reductase; aminoadipic semialdehyde-glutamate reductase; ε-N-(L-glutaryl-2)-L-lysine:NAD+(P) oxidoreductase (L-2-aminoadipate-semialdehyde forming); saccharopine reductase; 6-N-(L-1,3-dicarboxypropyl)-L-lysine:NADP+ oxidoreductase (L-glutamate-forming)',\n",
       "  'sys_name': 'N6-(L-1,3-dicarboxypropyl)-L-lysine:NADP+ oxidoreductase (L-glutamate-forming)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'L-saccharopine = N6-(L-1,3-dicarboxypropyl)-L-lysine\\n(S)-2-amino-6-oxohexanoate = L-2-aminoadipate 6-semialdehyde = L-allysine',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.11': {'ec_num': '1.5.1.11',\n",
       "  'accepted_name': 'D-octopine dehydrogenase',\n",
       "  'reaction': 'N2-(D-1-carboxyethyl)-L-arginine + NAD+ + H2O = L-arginine + pyruvate + NADH + H+',\n",
       "  'other_names': 'D-octopine synthase; octopine dehydrogenase; octopine:NAD+ oxidoreductase; ODH; 2-N-(D-1-carboxyethyl)-L-arginine:NAD+ oxidoreductase (L-arginine-forming)',\n",
       "  'sys_name': 'N2-(D-1-carboxyethyl)-L-arginine:NAD+ oxidoreductase (L-arginine-forming)',\n",
       "  'comments': 'In the reverse direction, acts also on L-ornithine, L-lysine and L-histidine.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.12': {'ec_num': '1.5.1.12',\n",
       "  'accepted_name': '1-pyrroline-5-carboxylate dehydrogenase',\n",
       "  'reaction': 'L-glutamate γ-semialdehyde + NAD+ + H2O = L-glutamate + NADH + H+',\n",
       "  'other_names': 'Δ1-pyrroline-5-carboxylate dehydrogenase; 1-pyrroline dehydrogenase; pyrroline-5-carboxylate dehydrogenase; pyrroline-5-carboxylic acid dehydrogenase; L-pyrroline-5-carboxylate-NAD+ oxidoreductase; 1-pyrroline-5-carboxylate:NAD+ oxidoreductase; Δ1-pyrroline-5-carboxylic acid dehydrogenase',\n",
       "  'sys_name': 'L-glutamate γ-semialdehyde:NAD+ oxidoreductase',\n",
       "  'comments': 'This enzyme catalyses the irreversible oxidation of glutamate-γ-semialdehyde to glutamate as part of the proline degradation pathway. (S)-1-pyrroline-5-carboxylate, the product of the first enzyme of the pathway (EC 1.5.99.8, proline dehydrogenase) is in spontaneous equilibrium with its tautomer L-glutamate γ-semialdehyde. In many bacterial species, both activities are carried out by a single bifunctional enzyme [3,4].The enzyme can also oxidize other 1-pyrrolines, e.g. 3-hydroxy-1-pyrroline-5-carboxylate is converted into 4-hydroxyglutamate and (R)-1-pyrroline-5-carboxylate is converted into D-glutamate. NADP+ can also act as acceptor, but with lower activity [5].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.15': {'ec_num': '1.5.1.15',\n",
       "  'accepted_name': 'methylenetetrahydrofolate dehydrogenase (NAD+)',\n",
       "  'reaction': '5,10-methylenetetrahydrofolate + NAD+ = 5,10-methenyltetrahydrofolate + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '5,10-methylenetetrahydrofolate:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-08-04 18:57:39'},\n",
       " '1.5.1.16': {'ec_num': '1.5.1.16',\n",
       "  'accepted_name': 'D-lysopine dehydrogenase',\n",
       "  'reaction': 'N2-(D-1-carboxyethyl)-L-lysine + NADP+ + H2O = L-lysine + pyruvate + NADPH + H+',\n",
       "  'other_names': 'D-lysopine synthase; lysopine dehydrogenase; D(+)-lysopine dehydrogenase; 2-N-(D-1-carboxyethyl)-L-lysine:NADP+ oxidoreductase (L-lysine-forming)',\n",
       "  'sys_name': 'N2-(D-1-carboxyethyl)-L-lysine:NADP+ oxidoreductase (L-lysine-forming)',\n",
       "  'comments': 'In the reverse reaction, a number of L-amino acids can act instead of L-lysine, and 2-oxobutanoate and, to a lesser extent, glyoxylate can act instead of pyruvate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.17': {'ec_num': '1.5.1.17',\n",
       "  'accepted_name': 'alanopine dehydrogenase',\n",
       "  'reaction': '2,2′-iminodipropanoate + NAD+ + H2O = L-alanine + pyruvate + NADH + H+',\n",
       "  'other_names': 'ALPDH; alanopine[meso-N-(1-carboxyethyl)-alanine]dehydrogenase; meso-N-(1-carboxyethyl)-alanine:NAD+ oxidoreductase; alanopine: NAD+ oxidoreductase; ADH (ambiguous); alanopine:NAD+ oxidoreductase',\n",
       "  'sys_name': '2,2′-iminodipropanoate:NAD+ oxidoreductase (L-alanine-forming)',\n",
       "  'comments': 'In the reverse reaction, L-alanine can be replaced by L-cysteine, L-serine or L-threonine; glycine acts very slowly (cf. EC 1.5.1.22 strombine dehydrogenase).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.18': {'ec_num': '1.5.1.18',\n",
       "  'accepted_name': 'ephedrine dehydrogenase',\n",
       "  'reaction': '(-)-ephedrine + NAD+ = (R)-2-methylimino-1-phenylpropan-1-ol + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '(-)-ephedrine:NAD+ 2-oxidoreductase',\n",
       "  'comments': 'The product immediately hydrolyses to methylamine and 1-hydroxy-1-phenylpropan-2-one. Acts on a number of related compounds including (-)-sympatol, (+)-pseudoephedrine and (+)-norephedrine.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.19': {'ec_num': '1.5.1.19',\n",
       "  'accepted_name': 'D-nopaline dehydrogenase',\n",
       "  'reaction': 'N2-(D-1,3-dicarboxypropyl)-L-arginine + NADP+ + H2O = L-arginine + 2-oxoglutarate + NADPH + H+',\n",
       "  'other_names': 'D-nopaline synthase; nopaline dehydrogenase; nopaline synthase; NOS; 2-N-(D-1,3-dicarboxypropyl)-L-arginine:NADP+ oxidoreductase (L-arginine-forming)',\n",
       "  'sys_name': 'N2-(D-1,3-dicarboxypropyl)-L-arginine:NADP+ oxidoreductase (L-arginine-forming)',\n",
       "  'comments': 'In the reverse direction, forms D-nopaline from L-arginine and D-ornaline from L-ornithine.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.20': {'ec_num': '1.5.1.20',\n",
       "  'accepted_name': 'methylenetetrahydrofolate reductase [NAD(P)H]',\n",
       "  'reaction': '5-methyltetrahydrofolate + NAD(P)+ = 5,10-methylenetetrahydrofolate + NAD(P)H + H+',\n",
       "  'other_names': 'MTHFR (gene name)',\n",
       "  'sys_name': '5-methyltetrahydrofolate:NAD(P)+ oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD). The enzyme catalyses the reversible conversion of 5,10-methylenetetrahydrofolate to 5-methyltetrahydrofolate, playing an important role in folate metabolism by regulating the distribution of one-carbon moieties between cellular methylation reactions and nucleic acid synthesis. This enzyme, characterized from Protozoan parasites of the genus Leishmania, is unique among similar characterized eukaryotic enzymes in that it lacks the C-terminal allosteric regulatory domain (allowing it to catalyse a reversible reaction) and uses NADH and NADPH with equal efficiency under physiological conditions. cf. EC 1.5.1.53, methylenetetrahydrofolate reductase (NADPH); EC 1.5.1.54, methylenetetrahydrofolate reductase (NADH); and EC 1.5.7.1, methylenetetrahydrofolate reductase (ferredoxin).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-07-10 14:09:22'},\n",
       " '1.5.1.53': {'ec_num': '1.5.1.53',\n",
       "  'accepted_name': 'methylenetetrahydrofolate reductase (NADPH)',\n",
       "  'reaction': '5-methyltetrahydrofolate + NADP+ = 5,10-methylenetetrahydrofolate + NADPH + H+',\n",
       "  'other_names': 'MTHFR (gene name); methylenetetrahydrofolate (reduced nicotinamide adenine dinucleotide phosphate) reductase; 5,10-methylenetetrahydrofolate reductase (NADPH); 5,10-methylenetetrahydrofolic acid reductase (ambiguous); 5,10-CH2-H4folate reductase (ambiguous); methylenetetrahydrofolate reductase (NADPH2); 5,10-methylenetetrahydrofolate reductase (ambiguous); methylenetetrahydrofolate reductase (ambiguous); N5,10-methylenetetrahydrofolate reductase (ambiguous); 5,10-methylenetetrahydropteroylglutamate reductase (ambiguous); N5,N10-methylenetetrahydrofolate reductase (ambiguous); methylenetetrahydrofolic acid reductase (ambiguous); 5-methyltetrahydrofolate:(acceptor) oxidoreductase (incorrect); 5,10-methylenetetrahydrofolate reductase (FADH2) (ambiguous)',\n",
       "  'sys_name': '5-methyltetrahydrofolate:NADP+ oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD). The enzyme from yeast and mammals catalyses a physiologically irreversible reduction of 5,10-methylenetetrahydrofolate to 5-methyltetrahydrofolate using NADPH as the electron donor. It plays an important role in folate metabolism by regulating the distribution of one-carbon moieties between cellular methylation reactions and nucleic acid synthesis. The enzyme contains an N-terminal catalytic domain and a C-terminal allosteric regulatory domain that binds S-adenosyl-L-methionine, which acts as an inhibitor. cf. EC 1.5.1.54, methylenetetrahydrofolate reductase (NADH); EC 1.5.1.20, methylenetetrahydrofolate reductase [NAD(P)H]; and EC 1.5.7.1, methylenetetrahydrofolate reductase (ferredoxin).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.21': {'ec_num': '1.5.1.21',\n",
       "  'accepted_name': '1-piperideine-2-carboxylate/1-pyrroline-2-carboxylate reductase (NADPH)',\n",
       "  'reaction': '(1) L-pipecolate + NADP+ = 1-piperideine-2-carboxylate + NADPH + H+;; (2) L-proline + NADP+ = 1-pyrroline-2-carboxylate + NADPH + H+',\n",
       "  'other_names': 'Pyr2C reductase; 1,2-didehydropipecolate reductase; P2C reductase; 1,2-didehydropipecolic reductase; DELTA1-piperideine-2-carboxylate/1-pyrroline-2-carboxylate reductase (ambiguous); L-pipecolate:NADP+ 2-oxidoreductase; DELTA1-piperideine-2-carboxylate reductase; Δ1-piperideine-2-carboxylate reductase',\n",
       "  'sys_name': 'L-pipecolate/L-proline:NADP+ 2-oxidoreductase',\n",
       "  'comments': 'The enzyme is involved in the catabolism of D-lysine and D-proline in bacteria that belong to the Pseudomonas genus. In contrast to EC 1.5.1.1, 1-piperideine-2-carboxylate/1-pyrroline-2-carboxylate reductase [NAD(P)H], which shows similar activity with NADPH and NADH, this enzyme is specific for NADPH.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '1-piperideine-2-carboxylate = 3,4,5,6-tetrahydropyridine-2-carboxylate',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.22': {'ec_num': '1.5.1.22',\n",
       "  'accepted_name': 'strombine dehydrogenase',\n",
       "  'reaction': 'N-(carboxymethyl)-D-alanine + NAD+ + H2O = glycine + pyruvate + NADH + H+',\n",
       "  'other_names': 'strombine[N-(carboxymethyl)-D-alanine]dehydrogenase; N-(carboxymethyl)-D-alanine: NAD+ oxidoreductase',\n",
       "  'sys_name': 'N-(carboxymethyl)-D-alanine:NAD+ oxidoreductase (glycine-forming)',\n",
       "  'comments': 'Also catalyses the reaction of EC 1.5.1.17 alanopine dehydrogenase, but more slowly. Does not act on L-strombine.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.23': {'ec_num': '1.5.1.23',\n",
       "  'accepted_name': 'tauropine dehydrogenase',\n",
       "  'reaction': 'tauropine + NAD+ + H2O = taurine + pyruvate + NADH + H+',\n",
       "  'other_names': '2-N-(D-1-carboxyethyl)taurine:NAD+ oxidoreductase (taurine-forming)',\n",
       "  'sys_name': 'N2-(D-1-carboxyethyl)taurine:NAD+ oxidoreductase (taurine-forming)',\n",
       "  'comments': 'In the reverse reaction, alanine can act instead of taurine, but more slowly, and 2-oxobutanoate and 2-oxopentanoate can act instead of pyruvate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'tauropine = N2-(D-1-carboxyethyl)-2-aminoethanesulfonate',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.24': {'ec_num': '1.5.1.24',\n",
       "  'accepted_name': 'N5-(carboxyethyl)ornithine synthase',\n",
       "  'reaction': 'N5-(L-1-carboxyethyl)-L-ornithine + NADP+ + H2O = L-ornithine + pyruvate + NADPH + H+',\n",
       "  'other_names': '5-N-(L-1-carboxyethyl)-L-ornithine:NADP+ oxidoreductase (L-ornithine-forming)',\n",
       "  'sys_name': 'N5-(L-1-carboxyethyl)-L-ornithine:NADP+ oxidoreductase (L-ornithine-forming)',\n",
       "  'comments': 'In the reverse direction, L-lysine can act instead of L-ornithine, but more slowly. Acts on the amino group. cf. EC 1.5.1.16, D-lysopine dehydrogenase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.25': {'ec_num': '1.5.1.25',\n",
       "  'accepted_name': 'thiomorpholine-carboxylate dehydrogenase',\n",
       "  'reaction': 'thiomorpholine 3-carboxylate + NAD(P)+ = 3,4-dehydro-thiomorpholine-3-carboxylate + NAD(P)H + H+',\n",
       "  'other_names': 'ketimine reductase; ketimine-reducing enzyme',\n",
       "  'sys_name': 'thiomorpholine-3-carboxylate:NAD(P)+ 5,6-oxidoreductase',\n",
       "  'comments': 'The product is the cyclic imine of the 2-oxoacid corresponding to S-(2-aminoethyl)cysteine. In the reverse direction, a number of other cyclic unsaturated compounds can act as substrates, but more slowly.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.26': {'ec_num': '1.5.1.26',\n",
       "  'accepted_name': 'β-alanopine dehydrogenase',\n",
       "  'reaction': 'β-alanopine + NAD+ + H2O = β-alanine + pyruvate + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'N-(D-1-carboxyethyl)-β-alanine:NAD+ oxidoreductase (β-alanine-forming)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.27': {'ec_num': '1.5.1.27',\n",
       "  'accepted_name': '1,2-dehydroreticulinium reductase (NADPH)',\n",
       "  'reaction': '(R)-reticuline + NADP+ = 1,2-dehydroreticulinium + NADPH + H+',\n",
       "  'other_names': '1,2-dehydroreticulinium ion reductase',\n",
       "  'sys_name': '(R)-reticuline:NADP+ oxidoreductase',\n",
       "  'comments': 'Reduces the 1,2-dehydroreticulinium ion to (R)-reticuline, which is a direct precursor of morphinan alkaloids in the poppy plant. The enzyme does not catalyse the reverse reaction to any significant extent under physiological conditions.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.28': {'ec_num': '1.5.1.28',\n",
       "  'accepted_name': 'opine dehydrogenase',\n",
       "  'reaction': '(2S)-2-{[1-(R)-carboxyethyl]amino}pentanoate + NAD+ + H2O = L-2-aminopentanoic acid + pyruvate + NADH + H+',\n",
       "  'other_names': '(2S)-2-{[1-(R)-carboxyethyl]amino}pentanoate dehydrogenase (NAD+, L-aminopentanoate-forming)',\n",
       "  'sys_name': '(2S)-2-{[1-(R)-carboxyethyl]amino}pentanoate:NAD+ oxidoreductase (L-aminopentanoate-forming)',\n",
       "  'comments': 'In the forward direction, the enzyme from Arthrobacter sp. acts also on secondary amine dicarboxylates such as N-(1-carboxyethyl)methionine and N-(1-carboxyethyl)phenylalanine. Dehydrogenation forms an imine, which dissociates to the amino acid and pyruvate. In the reverse direction, the enzyme acts also on neutral amino acids as an amino donor. They include L-amino acids such as 2-aminopentanoic acid, 2-aminobutyric acid, 2-aminohexanoic acid, 3-chloroalanine, O-acetylserine, methionine, isoleucine, valine, phenylalanine, leucine and alanine. The amino acceptors include 2-oxoacids such as pyruvate, oxaloacetate, glyoxylate and 2-oxobutyrate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.29': {'ec_num': '1.5.1.29',\n",
       "  'accepted_name': 'FMN reductase [NAD(P)H]',\n",
       "  'reaction': 'reduced riboflavin + NAD(P)+ = riboflavin + NAD(P)H + H+',\n",
       "  'other_names': 'NAD(P)H-FMN reductase (ambiguous); NAD(P)H-dependent FMN reductase (ambiguous); NAD(P)H:FMN oxidoreductase (ambiguous); NAD(P)H:flavin oxidoreductase (ambiguous); NAD(P)H2 dehydrogenase (FMN) (ambiguous); NAD(P)H2:FMN oxidoreductase (ambiguous); riboflavin mononucleotide reductase (ambiguous); flavine mononucleotide reductase (ambiguous); riboflavin mononucleotide (reduced nicotinamide adenine dinucleotide (phosphate)) reductase; flavin mononucleotide reductase (ambiguous); riboflavine mononucleotide reductase (ambiguous); Fre (ambiguous)',\n",
       "  'sys_name': 'riboflavin:NAD(P)+ oxidoreductase',\n",
       "  'comments': 'Catalyses the reduction of soluble flavins by reduced pyridine nucleotides. Highest activity with riboflavin. When NADH is used as acceptor, the enzyme can also utilize FMN and FAD as substrates, with lower activity than riboflavin. When NADPH is used as acceptor, the enzyme has a very low activity with FMN and no activity with FAD [1].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.30': {'ec_num': '1.5.1.30',\n",
       "  'accepted_name': 'flavin reductase (NADPH)',\n",
       "  'reaction': 'reduced riboflavin + NADP+ = riboflavin + NADPH + H+',\n",
       "  'other_names': 'NADPH:flavin oxidoreductase; riboflavin mononucleotide (reduced nicotinamide adenine dinucleotide phosphate) reductase; flavin mononucleotide reductase; flavine mononucleotide reductase; FMN reductase (NADPH); NADPH-dependent FMN reductase; NADPH-flavin reductase; NADPH-FMN reductase; NADPH-specific FMN reductase; riboflavin mononucleotide reductase; riboflavine mononucleotide reductase; NADPH2 dehydrogenase (flavin); NADPH2:riboflavin oxidoreductase',\n",
       "  'sys_name': 'reduced-riboflavin:NADP+ oxidoreductase',\n",
       "  'comments': 'The enzyme reduces riboflavin, and, less efficiently, FMN and FAD. NADH is oxidized less efficiently than NADPH.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.31': {'ec_num': '1.5.1.31',\n",
       "  'accepted_name': 'berberine reductase',\n",
       "  'reaction': '(R)-canadine + 2 NADP+ = berberine + 2 NADPH + H+',\n",
       "  'other_names': '(R)-canadine synthase',\n",
       "  'sys_name': '(R)-tetrahydroberberine:NADP+ oxidoreductase',\n",
       "  'comments': 'Involved in alkaloid biosynthesis in Corydalis cava to give (R)-canadine with the opposite configuration to the precursor of berberine (see EC 1.3.3.8 tetrahydroberberine oxidase). Also acts on 7,8-dihydroberberine.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.32': {'ec_num': '1.5.1.32',\n",
       "  'accepted_name': 'vomilenine reductase',\n",
       "  'reaction': '1,2-dihydrovomilenine + NADP+ = vomilenine + NADPH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '1,2-dihydrovomilenine:NADP+ oxidoreductase',\n",
       "  'comments': 'Forms part of the ajmaline biosynthesis pathway.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.33': {'ec_num': '1.5.1.33',\n",
       "  'accepted_name': 'pteridine reductase',\n",
       "  'reaction': '5,6,7,8-tetrahydrobiopterin + 2 NADP+ = biopterin + 2 NADPH + 2 H+',\n",
       "  'other_names': 'PTR1; pteridine reductase 1',\n",
       "  'sys_name': '5,6,7,8-tetrahydrobiopterin:NADP+ oxidoreductase',\n",
       "  'comments': 'The enzyme from Leishmania (both amastigote and promastigote forms) catalyses the reduction by NADPH of folate and a wide variety of unconjugated pterins, including biopterin, to their tetrahydro forms. It also catalyses the reduction of 7,8-dihydropterins and 7,8-dihydrofolate to their tetrahydro forms. In contrast to EC 1.5.1.3 (dihydrofolate reductase) and EC 1.5.1.34 (6,7-dihydropteridine reductase), pteridine reductase will not catalyse the reduction of the quinonoid form of dihydrobiopterin. The enzyme is specific for NADPH; no activity has been detected with NADH. It also differs from EC 1.5.1.3 (dihydrofolate reductase) in being specific for the Si-face of NADPH.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.1.34': {'ec_num': '1.5.1.34',\n",
       "  'accepted_name': '6,7-dihydropteridine reductase',\n",
       "  'reaction': 'a 5,6,7,8-tetrahydropteridine + NAD(P)+ = a 6,7-dihydropteridine + NAD(P)H + H+',\n",
       "  'other_names': '6,7-dihydropteridine:NAD(P)H oxidoreductase; DHPR; NAD(P)H:6,7-dihydropteridine oxidoreductase; NADH-dihydropteridine reductase; NADPH-dihydropteridine reductase; NADPH-specific dihydropteridine reductase; dihydropteridine (reduced nicotinamide adenine dinucleotide) reductase; dihydropteridine reductase; dihydropteridine reductase (NADH); 5,6,7,8-tetrahydropteridine:NAD(P)H+ oxidoreductase',\n",
       "  'sys_name': '5,6,7,8-tetrahydropteridine:NAD(P)+ oxidoreductase',\n",
       "  'comments': 'The substrate is the quinonoid form of dihydropteridine. Not identical with EC 1.5.1.3 dihydrofolate reductase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.3.1': {'ec_num': '1.5.3.1',\n",
       "  'accepted_name': 'sarcosine oxidase (formaldehyde-forming)',\n",
       "  'reaction': 'sarcosine + H2O + O2 = glycine + formaldehyde + H2O2',\n",
       "  'other_names': 'MSOX; monomeric sarcosine oxidase; sarcosine oxidase (ambiguous)',\n",
       "  'sys_name': 'sarcosine:oxygen oxidoreductase (demethylating)',\n",
       "  'comments': 'The enzyme, reported from bacteria and fungi, catalyses the oxidative demethylation of sarcosine. It contains a FAD cofactor bound to an L-cysteine residue. cf. EC 1.5.3.24, sarcosine oxidase (5,10-methylenetetrahydrofolate-forming).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.3.2': {'ec_num': '1.5.3.2',\n",
       "  'accepted_name': 'N-methyl-L-amino-acid oxidase',\n",
       "  'reaction': 'an N-methyl-L-amino acid + H2O + O2 = an L-amino acid + formaldehyde + H2O2',\n",
       "  'other_names': 'N-methylamino acid oxidase; demethylase',\n",
       "  'sys_name': 'N-methyl-L-amino-acid:oxygen oxidoreductase (demethylating)',\n",
       "  'comments': 'A flavoprotein.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.3.4': {'ec_num': '1.5.3.4',\n",
       "  'accepted_name': 'N6-methyl-lysine oxidase',\n",
       "  'reaction': 'N6-methyl-L-lysine + H2O + O2 = L-lysine + formaldehyde + H2O2',\n",
       "  'other_names': 'ε-alkyl-L-lysine:oxygen oxidoreductase ; N6-methyllysine oxidase; ε-N-methyllysine demethylase; ε-alkyllysinase; 6-N-methyl-L-lysine:oxygen oxidoreductase (demethylating)',\n",
       "  'sys_name': 'N6-methyl-L-lysine:oxygen oxidoreductase (demethylating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.3.5': {'ec_num': '1.5.3.5',\n",
       "  'accepted_name': '(S)-6-hydroxynicotine oxidase',\n",
       "  'reaction': '(S)-6-hydroxynicotine + H2O + O2 = 1-(6-hydroxypyridin-3-yl)-4-(methylamino)butan-1-one + H2O2 (overall reaction);;(1a) (S)-6-hydroxynicotine + O2 = 5-(N-methyl-4,5-dihydro-1H-pyrrol-2-yl)pyridin-2-ol + H2O2;;(1b) 5-(N-methyl-4,5-dihydro-1H-pyrrol-2-yl)pyridin-2-ol + H2O = 1-(6-hydroxypyridin-3-yl)-4-(methylamino)butan-1-one (spontaneous)',\n",
       "  'other_names': 'L-6-hydroxynicotine oxidase; 6-hydroxy-L-nicotine oxidase; 6-hydroxy-L-nicotine:oxygen oxidoreductase; nctB (gene name)',\n",
       "  'sys_name': '(S)-6-hydroxynicotine:oxygen oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD). The enzyme, which participates in nicotine degradation, is specific for the (S) isomer of 6-hydroxynicotine. The bacterium Arthrobacter nicotinovorans, in which this enzyme was originally discovered, has a different enzyme that catalyses a similar reaction with the less common (R)-isomer (cf. EC 1.5.3.6, (R)-6-hydroxynicotine oxidase).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.5.3.5\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.5.3.5\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.5.3.5\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.5.3.5\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.5.3.5%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '(S)-6-hydroxynicotine = 5-[(2S)-1-methylpyrrolidin-2-yl]pyridin-2-ol\\n1-(6-hydroxypyridin-3-yl)-4-(methylamino)butan-1-one = 6-hydroxypseudooxynicotine\\n5-(N-methyl-4,5-dihydro-1H-pyrrol-2-yl)pyridin-2-ol = 6-hydroxy-N-methylmyosmine\\n',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.3.6': {'ec_num': '1.5.3.6',\n",
       "  'accepted_name': '(R)-6-hydroxynicotine oxidase',\n",
       "  'reaction': '(R)-6-hydroxynicotine + H2O + O2 = 1-(6-hydroxypyridin-3-yl)-4-(methylamino)butan-1-one + H2O2 (overall reaction);;(1a) (R)-6-hydroxynicotine + O2 = 5-(N-methyl-4,5-dihydro-1H-pyrrol-2-yl)pyridin-2-ol + H2O2 ;;(1b) 5-(N-methyl-4,5-dihydro-1H-pyrrol-2-yl)pyridin-2-ol + H2O = 1-(6-hydroxypyridin-3-yl)-4-(methylamino)butan-1-one (spontaneous)',\n",
       "  'other_names': 'D-6-hydroxynicotine oxidase; 6-hydroxy-D-nicotine oxidase',\n",
       "  'sys_name': '(R)-6-hydroxynicotine:oxygen oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD). The enzyme, which participates in nicotine degradation, is specific for (R) isomer of 6-hydroxynicotine, derived from the uncommon (R)-nicotine. The bacterium Arthrobacter nicotinovorans, in which this enzyme was originally discovered, has a different enzyme that catalyses a similar reaction with the (S)-isomer (cf. EC 1.5.3.5, (S)-6-hydroxynicotine oxidase).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.5.3.6\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.5.3.6\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.5.3.6\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.5.3.6\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.5.3.6%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '(R)-6-hydroxynicotine = 5-[(2R)-1-methylpyrrolidin-2-yl]pyridin-2-ol\\n5-(N-methyl-4,5-dihydro-1H-pyrrol-2-yl)pyridin-2-ol = 6-hydroxy-N-methylmyosmine\\n1-(6-hydroxypyridin-3-yl)-4-(methylamino)butan-1-one = 6-hydroxypseudooxynicotine',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.3.7': {'ec_num': '1.5.3.7',\n",
       "  'accepted_name': 'L-pipecolate oxidase',\n",
       "  'reaction': 'L-pipecolate + O2 = (S)-2,3,4,5-tetrahydropyridine-2-carboxylate + H2O2',\n",
       "  'other_names': 'pipecolate oxidase; L-pipecolic acid oxidase',\n",
       "  'sys_name': 'L-pipecolate:oxygen 1,6-oxidoreductase',\n",
       "  'comments': 'The product reacts with water to form (S)-2-amino-6-oxohexanoate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '<small>L</small>-1-piperideine 6-carboxylate = (<em>S</em>)-2,3,4,5-tetrahydropyridine-2-carboxylate = (<em>S</em>)-1,6-didehydropiperidine-2-carboxylate<br> (<em>S</em>)-2-amino-6-oxohexanoate = <small>L</small>-2-aminoadipate 6-semialdehyde = <small>L</small>-allysine',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.3.10': {'ec_num': '1.5.3.10',\n",
       "  'accepted_name': 'dimethylglycine oxidase',\n",
       "  'reaction': 'N,N-dimethylglycine + 5,6,7,8-tetrahydrofolate + O2 = sarcosine + 5,10-methylenetetrahydrofolate + H2O2',\n",
       "  'other_names': 'dmg (gene name); N,N-dimethylglycine:oxygen oxidoreductase (demethylating)',\n",
       "  'sys_name': 'N,N-dimethylglycine,5,6,7,8-tetrahydrofolate:oxygen oxidoreductase (demethylating,5,10-methylenetetrahydrofolate-forming)',\n",
       "  'comments': 'A flavoprotein (FAD). The enzyme, characterized from the bacterium Arthrobacter globiformis, contains two active sites connected by a large \"reaction chamber\". An imine intermediate is transferred between the sites, eliminating the production of toxic formaldehyde. In the absence of folate the enzyme does form formaldehyde. Does not oxidize sarcosine. cf. EC 1.5.8.4, dimethylglycine dehydrogenase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.3.11': {'ec_num': '1.5.3.11',\n",
       "  'accepted_name': 'polyamine oxidase',\n",
       "  'reaction': 'N1-acetylspermine + O2 + H2O = N1-acetylspermidine + 3-aminopropanal + H2O2',\n",
       "  'other_names': '1-N-acetylspermidine:oxygen oxidoreductase (deaminating)',\n",
       "  'sys_name': 'N1-acetylspermidine:oxygen oxidoreductase (deaminating)',\n",
       "  'comments': 'A flavoprotein requiring Fe2+. Also acts on N1-acetylspermidine and N1,N12-diacetylspermine.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.3.12': {'ec_num': '1.5.3.12',\n",
       "  'accepted_name': 'dihydrobenzophenanthridine oxidase',\n",
       "  'reaction': '(1) dihydrosanguinarine + O2 = sanguinarine + H2O2;;(2) dihydrochelirubine + O2 = chelirubine + H2O2;;(3) dihydromacarpine + O2 = macarpine + H2O2',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'dihydrobenzophenanthridine:oxygen oxidoreductase',\n",
       "  'comments': 'A CuII enzyme found in higher plants that produces oxidized forms of the benzophenanthridine alkaloids',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.4.1': {'ec_num': '1.5.4.1',\n",
       "  'accepted_name': 'pyrimidodiazepine synthase',\n",
       "  'reaction': '2-amino-6-acetyl-3,7,8,9-tetrahydro-3H-pyrimido[4,5-b][1,4]diazepin-4-one + glutathione disulfide + H2O = 6-pyruvoyltetrahydropterin + 2 glutathione',\n",
       "  'other_names': 'PDA synthase; pyrimidodiazepine:oxidized-glutathione oxidoreductase (ring-opening, cyclizing); pyrimidodiazepine:glutathione-disulfide oxidoreductase (ring-opening, cyclizing)',\n",
       "  'sys_name': '2-amino-6-acetyl-3,7,8,9-tetrahydro-3H-pyrimido[4,5-b][1,4]diazepin-4-one:glutathione-disulfide oxidoreductase (ring-opening, cyclizing)',\n",
       "  'comments': 'In the reverse direction, the reduction of 6-pyruvoyl-tetrahydropterin is accompanied by the opening of the 6-membered pyrazine ring and the formation of the 7-membered diazepine ring. The pyrimidodiazepine formed is an acetyldihydro derivative. Involved in the formation of the eye pigment drosopterin in Drosophila melanogaster.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '2-amino-6-acetyl-3,7,8,9-tetrahydro-3H-pyrimido[4,5-b][1,4]diazepin-4-one = pyrimidodiazepine',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.5.1': {'ec_num': '1.5.5.1',\n",
       "  'accepted_name': 'electron-transferring-flavoprotein dehydrogenase',\n",
       "  'reaction': 'reduced electron-transferring flavoprotein + ubiquinone = electron-transferring flavoprotein + ubiquinol',\n",
       "  'other_names': 'ETF-QO; ETF:ubiquinone oxidoreductase; electron transfer flavoprotein dehydrogenase; electron transfer flavoprotein Q oxidoreductase; electron transfer flavoprotein-ubiquinone oxidoreductase; electron transfer flavoprotein reductase',\n",
       "  'sys_name': 'electron-transferring-flavoprotein:ubiquinone oxidoreductase',\n",
       "  'comments': 'An iron-sulfur flavoprotein, forming part of the mitochondrial electron-transfer system.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.7.1': {'ec_num': '1.5.7.1',\n",
       "  'accepted_name': 'methylenetetrahydrofolate reductase (ferredoxin)',\n",
       "  'reaction': '5-methyltetrahydrofolate + 2 oxidized ferredoxin = 5,10-methylenetetrahydrofolate + 2 reduced ferredoxin + 2 H+',\n",
       "  'other_names': '5,10-methylenetetrahydrofolate reductase',\n",
       "  'sys_name': '5-methyltetrahydrofolate:ferredoxin oxidoreductase',\n",
       "  'comments': 'An iron-sulfur flavoprotein that also contains zinc. The enzyme from Clostridium formicoaceticum catalyses the reduction of methylene blue, menadione, benzyl viologen, rubredoxin or FAD with 5-methyltetrahydrofolate and the oxidation of reduced ferredoxin or FADH2 with 5,10-methylenetetrahydrofolate. However, unlike EC 1.5.1.53, methylenetetrahydrofolate reductase (NADPH); EC 1.5.1.54, methylenetetrahydrofolate reductase (NADH); or EC 1.5.1.20, methylenetetrahydrofolate reductase [NAD(P)H], there is no activity with either NADH or NADP+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-07-27 09:35:48'},\n",
       " '1.5.8.1': {'ec_num': '1.5.8.1',\n",
       "  'accepted_name': 'dimethylamine dehydrogenase',\n",
       "  'reaction': 'dimethylamine + H2O + electron-transfer flavoprotein = methylamine + formaldehyde + reduced electron-transfer flavoprotein',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'dimethylamine:electron-transfer flavoprotein oxidoreductase',\n",
       "  'comments': 'Contains FAD and a [4Fe-4S] cluster.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.5.8.1\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.5.8.1\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.5.8.1\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.5.8.1\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.8.2': {'ec_num': '1.5.8.2',\n",
       "  'accepted_name': 'trimethylamine dehydrogenase',\n",
       "  'reaction': 'trimethylamine + H2O + electron-transfer flavoprotein = dimethylamine + formaldehyde + reduced electron-transfer flavoprotein',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'trimethylamine:electron-transfer flavoprotein oxidoreductase (demethylating)',\n",
       "  'comments': 'A number of alkyl-substituted derivatives of trimethylamine can also act as electron donors; phenazine methosulfate and 2,6-dichloroindophenol can act as electron acceptors. Contains FAD and a [4Fe-4S] cluster.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.5.8.2\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.5.8.2\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.5.8.2\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.5.8.2\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.5.8.2%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.99.1': {'ec_num': '1.5.99.1',\n",
       "  'accepted_name': 'sarcosine dehydrogenase',\n",
       "  'reaction': 'sarcosine + acceptor + H2O = glycine + formaldehyde + reduced acceptor',\n",
       "  'other_names': 'sarcosine N-demethylase; monomethylglycine dehydrogenase; sarcosine:(acceptor) oxidoreductase (demethylating)',\n",
       "  'sys_name': 'sarcosine:acceptor oxidoreductase (demethylating)',\n",
       "  'comments': 'A flavoprotein (FMN).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.99.2': {'ec_num': '1.5.99.2',\n",
       "  'accepted_name': 'dimethylglycine dehydrogenase',\n",
       "  'reaction': 'N,N-dimethylglycine + acceptor + H2O = sarcosine + formaldehyde + reduced acceptor',\n",
       "  'other_names': 'N,N-dimethylglycine oxidase; N,N-dimethylglycine:(acceptor) oxidoreductase (demethylating)',\n",
       "  'sys_name': 'N,N-dimethylglycine:acceptor oxidoreductase (demethylating)',\n",
       "  'comments': 'A flavoprotein.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.99.3': {'ec_num': '1.5.99.3',\n",
       "  'accepted_name': 'L-pipecolate dehydrogenase',\n",
       "  'reaction': 'L-pipecolate + acceptor = (S)-2,3,4,5-tetrahydropyridine-2-carboxylate + reduced acceptor',\n",
       "  'other_names': 'L-pipecolate:(acceptor) 1,6-oxidoreductase',\n",
       "  'sys_name': 'L-pipecolate:acceptor 1,6-oxidoreductase',\n",
       "  'comments': 'The product reacts with water to form (S)-2-amino-6-oxohexanoate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '(<em>S</em>)-2-amino-6-oxohexanoate = <small>L</small>-2-aminoadipate 6-semialdehyde = <small>L</small>-allysine<br> <small>L</small>-1-piperideine 6-carboxylate = (<em>S</em>)-2,3,4,5-tetrahydropyridine-2-carboxylate = (<em>S</em>)-1,6-didehydropiperidine-2-carboxylate',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.99.4': {'ec_num': '1.5.99.4',\n",
       "  'accepted_name': 'nicotine 6-hydroxylase',\n",
       "  'reaction': '(S)-nicotine + acceptor + H2O = (S)-6-hydroxynicotine + reduced acceptor',\n",
       "  'other_names': 'nicotine oxidase; D-nicotine oxidase; nicotine:(acceptor) 6-oxidoreductase (hydroxylating); L-nicotine oxidase; nicotine dehydrogenase (incorrect)',\n",
       "  'sys_name': 'nicotine:acceptor 6-oxidoreductase (hydroxylating)',\n",
       "  'comments': 'A metalloprotein (FMN). The enzyme can act on both the naturally found (S)-enantiomer and the synthetic (R)-enantiomer of nicotine, with retention of configuration in both cases [4].',\n",
       "  'links': '<a href=\"http://www.brenda-enzymes.org/enzyme.php?ecno=1.5.99.4\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.5.99.4\" target=\"new\">EAWAG-BBD</a>, <a href=\"http://enzyme.expasy.org/EC/1.5.99.4\" target=\"new\">EXPASY</a>, <a href=\"http://www.genome.ad.jp/dbget-bin/www_bget?ec:1.5.99.4\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-07-10 14:09:24'},\n",
       " '1.5.99.5': {'ec_num': '1.5.99.5',\n",
       "  'accepted_name': 'methylglutamate dehydrogenase',\n",
       "  'reaction': 'N-methyl-L-glutamate + acceptor + H2O = L-glutamate + formaldehyde + reduced acceptor',\n",
       "  'other_names': 'N-methylglutamate dehydrogenase; N-methyl-L-glutamate:(acceptor) oxidoreductase (demethylating)',\n",
       "  'sys_name': 'N-methyl-L-glutamate:acceptor oxidoreductase (demethylating)',\n",
       "  'comments': 'A number of N-methyl-substituted amino acids can act as donor; 2,6-dichloroindophenol is the best acceptor.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.99.6': {'ec_num': '1.5.99.6',\n",
       "  'accepted_name': 'spermidine dehydrogenase',\n",
       "  'reaction': 'spermidine + acceptor + H2O = propane-1,3-diamine + 4-aminobutanal + reduced acceptor',\n",
       "  'other_names': 'spermidine:(acceptor) oxidoreductase',\n",
       "  'sys_name': 'spermidine:acceptor oxidoreductase',\n",
       "  'comments': 'A flavohemoprotein (FAD). Ferricyanide, 2,6-dichloroindophenol and cytochrome c can act as acceptor. 4-Aminobutanal condenses non-enzymically to 1-pyrroline.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'spermidine = N-(3-aminopropyl)butane-1,4-diamine',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.99.8': {'ec_num': '1.5.99.8',\n",
       "  'accepted_name': 'proline dehydrogenase',\n",
       "  'reaction': 'L-proline + acceptor = (S)-1-pyrroline-5-carboxylate + reduced acceptor',\n",
       "  'other_names': 'L-proline dehydrogenase; L-proline:(acceptor) oxidoreductase',\n",
       "  'sys_name': 'L-proline:acceptor oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD). In many organisms, ranging from bacteria to mammals, proline is oxidized to glutamate in a two-step process involving this enzyme and EC 1.2.1.88, L-glutamate γ-semialdehyde dehydrogenase [2]. Both activities are carried out by the same enzyme in enterobacteria [2,3].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.99.9': {'ec_num': '1.5.99.9',\n",
       "  'accepted_name': 'methylenetetrahydromethanopterin dehydrogenase',\n",
       "  'reaction': '5,10-methylenetetrahydromethanopterin + coenzyme F420 = 5,10-methenyltetrahydromethanopterin + reduced coenzyme F420',\n",
       "  'other_names': 'N5,N10-methylenetetrahydromethanopterin dehydrogenase; 5,10-methylenetetrahydromethanopterin dehydrogenase',\n",
       "  'sys_name': '5,10-methylenetetrahydromethanopterin:coenzyme-F420 oxidoreductase',\n",
       "  'comments': 'Coenzyme F420 is a 7,8-didemethyl-8-hydroxy-5-deazariboflavin derivative; methanopterin is a pterin analogue. The enzyme is involved in the formation of methane from CO2 in Methanobacterium thermoautotrophicum.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.5.99.9\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.5.99.9\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.5.99.9\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.5.99.9\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.5.99.9%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.99.11': {'ec_num': '1.5.99.11',\n",
       "  'accepted_name': '5,10-methylenetetrahydromethanopterin reductase',\n",
       "  'reaction': '5-methyltetrahydromethanopterin + coenzyme F420 = 5,10-methylenetetrahydromethanopterin + reduced coenzyme F420',\n",
       "  'other_names': '5,10-methylenetetrahydromethanopterin cyclohydrolase; N5,N10-methylenetetrahydromethanopterin reductase; methylene-H4MPT reductase; coenzyme F420-dependent N5,N10-methenyltetrahydromethanopterin reductase; N5,N10-methylenetetrahydromethanopterin:coenzyme-F420 oxidoreductase',\n",
       "  'sys_name': '5-methyltetrahydromethanopterin:coenzyme-F420 oxidoreductase',\n",
       "  'comments': 'Catalyses an intermediate step in methanogenesis from CO2 and H2 in bacteria.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.5.99.11\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.5.99.11\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.5.99.11\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.5.99.11\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.5.99.11%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.5.99.12': {'ec_num': '1.5.99.12',\n",
       "  'accepted_name': 'cytokinin dehydrogenase',\n",
       "  'reaction': 'N6-prenyladenine + acceptor + H2O = adenine + 3-methylbut-2-enal + reduced acceptor',\n",
       "  'other_names': 'N6-dimethylallyladenine:(acceptor) oxidoreductase; 6-N-dimethylallyladenine:acceptor oxidoreductase; OsCKX2; CKX; cytokinin oxidase/dehydrogenase; N6-dimethylallyladenine:acceptor oxidoreductase',\n",
       "  'sys_name': 'N6-prenyladenine:acceptor oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD). Catalyses the oxidation of cytokinins, a family of N6-substituted adenine derivatives that are plant hormones, where the substituent is a prenyl group. Although this activity was previously thought to be catalysed by a hydrogen-peroxide-forming oxidase, this enzyme does not require oxygen for activity and does not form hydrogen peroxide. 2,6-Dichloroindophenol, methylene blue, nitroblue tetrazolium, phenazine methosulfate and copper(II) in the presence of imidazole can act as acceptors. This enzyme plays a part in regulating rice-grain production, with lower levels of the enzyme resulting in enhanced grain production [2].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'zeatin = (E)-2-methyl-4-(9H-purin-6-ylamino)but-2-en-1-ol = (E)-N6-(4-hydroxy-3-methylbut-2-enyl)adenine\\nN6-prenyladenine = N6-(3-methylbut-2-en-1-yl)purin-6-amine',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.6.1.1': {'ec_num': '1.6.1.1',\n",
       "  'accepted_name': 'NAD(P)+ transhydrogenase (Si-specific)',\n",
       "  'reaction': 'NADPH + NAD+ = NADP+ + NADH',\n",
       "  'other_names': 'pyridine nucleotide transhydrogenase; transhydrogenase; NAD(P)+ transhydrogenase; nicotinamide adenine dinucleotide (phosphate) transhydrogenase; NAD+ transhydrogenase; NADH transhydrogenase; nicotinamide nucleotide transhydrogenase; NADPH-NAD+ transhydrogenase; pyridine nucleotide transferase; NADPH-NAD+ oxidoreductase; NADH-NADP+-transhydrogenase; NADPH:NAD+ transhydrogenase; H+-Thase; non-energy-linked transhydrogenase; NADPH:NAD+ oxidoreductase (B-specific); NAD(P)+ transhydrogenase (B-specific)',\n",
       "  'sys_name': 'NADPH:NAD+ oxidoreductase (Si-specific)',\n",
       "  'comments': 'The enzyme from Azotobacter vinelandii is a flavoprotein (FAD). It is Si-specific with respect to both NAD+ and NADP+. See EC 1.6.1.3, NAD(P)+ transhydrogenase, for enzymes whose stereo specificity is not known.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-08-04 18:57:47'},\n",
       " '1.6.1.2': {'ec_num': '1.6.1.2',\n",
       "  'accepted_name': 'NAD(P)+ transhydrogenase (Re/Si-specific)',\n",
       "  'reaction': 'NADPH + NAD+ = NADP+ + NADH',\n",
       "  'other_names': 'pyridine nucleotide transhydrogenase; transhydrogenase; NAD(P)+ transhydrogenase; nicotinamide adenine dinucleotide (phosphate) transhydrogenase; NAD+ transhydrogenase; NADH transhydrogenase; nicotinamide nucleotide transhydrogenase; NADPH-NAD+ transhydrogenase; pyridine nucleotide transferase; NADPH-NAD+ oxidoreductase; NADH-NADP+-transhydrogenase; NADPH:NAD+ transhydrogenase; H+-Thase; energy-linked transhydrogenase; NAD(P) transhydrogenase (AB-specific); NAD(P)+ transhydrogenase (AB-specific); NADPH:NAD+ oxidoreductase (AB-specific)',\n",
       "  'sys_name': 'NADPH:NAD+ oxidoreductase (Re/Si-specific)',\n",
       "  'comments': 'The enzyme from heart mitochondria is Re-specific with respect to NAD+ and Si-specific with respect to NADP+ [cf. EC 1.6.1.1 NAD(P)+ transhydrogenase (Si-specific)].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-07-27 09:35:52'},\n",
       " '1.6.2.2': {'ec_num': '1.6.2.2',\n",
       "  'accepted_name': 'cytochrome-b5 reductase',\n",
       "  'reaction': 'NADH + 2 ferricytochrome b5 = NAD+ + H+ + 2 ferrocytochrome b5',\n",
       "  'other_names': 'cytochrome b5 reductase; dihydronicotinamide adenine dinucleotide-cytochrome b5 reductase; reduced nicotinamide adeninedinucleotide-cytochrome b5 reductase; NADH-ferricytochrome b5 oxidoreductase; NADH-cytochrome b5 reductase; NADH 5α-reductase ; NADH-cytochrome-b5 reductase',\n",
       "  'sys_name': 'NADH:ferricytochrome-b5 oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.6.2.4': {'ec_num': '1.6.2.4',\n",
       "  'accepted_name': 'NADPH—hemoprotein reductase',\n",
       "  'reaction': 'NADPH + H+ + n oxidized hemoprotein = NADP+ + n reduced hemoprotein',\n",
       "  'other_names': 'CPR; FAD-cytochrome c reductase; NADP-cytochrome c reductase; NADP-cytochrome reductase; NADPH-dependent cytochrome c reductase; NADPH:P-450 reductase; NADPH:ferrihemoprotein oxidoreductase; NADPH—cytochrome P-450 oxidoreductase; NADPH-cytochrome c oxidoreductase; NADPH-cytochrome c reductase; NADPH—cytochrome p-450 reductase; NADPH-ferricytochrome c oxidoreductase; NADPH-ferrihemoprotein reductase; TPNH2 cytochrome c reductase; TPNH-cytochrome c reductase; aldehyde reductase (NADPH-dependent); cytochrome P-450 reductase; cytochrome c reductase (reduced nicotinamide adenine dinucleotide phosphate, NADPH, NADPH-dependent); dihydroxynicotinamide adenine dinucleotide phosphate-cytochrome c reductase; ferrihemoprotein P-450 reductase; reduced nicotinamide adenine dinucleotide phosphate-cytochrome c reductase; reductase, cytochrome c (reduced nicotinamide adenine dinucleotide phosphate)',\n",
       "  'sys_name': 'NADPH:hemoprotein oxidoreductase',\n",
       "  'comments': 'A flavoprotein containing both FMN and FAD. This enzyme catalyses the transfer of electrons from NADPH, an obligatory two-electron donor, to microsomal P-450 monooxygenases (e.g. EC 1.14.14.1, unspecific monooxygenase) by stabilizing the one-electron reduced form of the flavin cofactors FAD and FMN. It also reduces cytochrome b5 and cytochrome c. The number n in the equation is 1 if the hemoprotein undergoes a 2-electron reduction, and is 2 if it undergoes a 1-electron reduction.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.6.2.5': {'ec_num': '1.6.2.5',\n",
       "  'accepted_name': 'NADPH—cytochrome-c2 reductase',\n",
       "  'reaction': 'NADPH + 2 ferricytochrome c2 = NADP+ + H+ + 2 ferrocytochrome c2',\n",
       "  'other_names': 'cytochrome c2 reductase (reduced nicotinamide adenine dinucleotide phosphate); cytochrome c2 reductase (reduced nicotinamide adinine dinucleotide phosphate, NADPH)',\n",
       "  'sys_name': 'NADPH:ferricytochrome-c2 oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.6.2.6': {'ec_num': '1.6.2.6',\n",
       "  'accepted_name': 'leghemoglobin reductase',\n",
       "  'reaction': 'NAD(P)H + H+ + 2 ferrileghemoglobin = NAD(P)+ + 2 ferroleghemoglobin',\n",
       "  'other_names': 'ferric leghemoglobin reductase',\n",
       "  'sys_name': 'NAD(P)H:ferrileghemoglobin oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.6.3.1': {'ec_num': '1.6.3.1',\n",
       "  'accepted_name': 'NAD(P)H oxidase (H2O2-forming)',\n",
       "  'reaction': 'NAD(P)H + H+ + O2 = NAD(P)+ + H2O2',\n",
       "  'other_names': 'THOX2; ThOX; dual oxidase; p138tox; thyroid NADPH oxidase; thyroid oxidase; thyroid oxidase 2; NADPH oxidase; NAD(P)H:oxygen oxidoreductase; NAD(P)H oxidase',\n",
       "  'sys_name': 'NAD(P)H:oxygen oxidoreductase (H2O2-forming)',\n",
       "  'comments': 'Requires FAD, heme and calcium. When calcium is present, this transmembrane glycoprotein generates H2O2 by transfering electrons from intracellular NAD(P)H to extracellular molecular oxygen. The electron bridge within the enzyme contains one molecule of FAD and probably two heme groups. This flavoprotein is expressed at the apical membrane of thyrocytes, and provides H2O2 for the thyroid peroxidase-catalysed biosynthesis of thyroid hormones.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.6.5.2': {'ec_num': '1.6.5.2',\n",
       "  'accepted_name': 'NAD(P)H dehydrogenase (quinone)',\n",
       "  'reaction': 'NAD(P)H + H+ + a quinone = NAD(P)+ + a hydroquinone',\n",
       "  'other_names': 'menadione reductase; phylloquinone reductase; quinone reductase; dehydrogenase, reduced nicotinamide adenine dinucleotide (phosphate, quinone); DT-diaphorase; flavoprotein NAD(P)H-quinone reductase; menadione oxidoreductase; NAD(P)H dehydrogenase; NAD(P)H menadione reductase; NAD(P)H-quinone dehydrogenase; NAD(P)H-quinone oxidoreductase; NAD(P)H: (quinone-acceptor)oxidoreductase; NAD(P)H: menadione oxidoreductase; NADH-menadione reductase; naphthoquinone reductase; p-benzoquinone reductase; reduced NAD(P)H dehydrogenase; viologen accepting pyridine nucleotide oxidoreductase; vitamin K reductase; diaphorase; reduced nicotinamide-adenine dinucleotide (phosphate) dehydrogenase; vitamin-K reductase; NAD(P)H2 dehydrogenase (quinone); NQO1; QR1; NAD(P)H:(quinone-acceptor) oxidoreductase',\n",
       "  'sys_name': 'NAD(P)H:quinone oxidoreductase',\n",
       "  'comments': 'A flavoprotein. The enzyme catalyses a two-electron reduction and has a preference for short-chain acceptor quinones, such as ubiquinone, benzoquinone, juglone and duroquinone [6]. The animal, but not the plant, form of the enzyme is inhibited by dicoumarol.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.6.5.3': {'ec_num': '1.6.5.3',\n",
       "  'accepted_name': 'NADH:ubiquinone reductase (H+-translocating)',\n",
       "  'reaction': 'NADH + ubiquinone + 6 H+[side 1] = NAD+ + ubiquinol + 7 H+[side 2]',\n",
       "  'other_names': 'ubiquinone reductase (ambiguous); type 1 dehydrogenase; complex 1 dehydrogenase; coenzyme Q reductase (ambiguous); complex I (electron transport chain); complex I (mitochondrial electron transport); complex I (NADH:Q1 oxidoreductase); dihydronicotinamide adenine dinucleotide-coenzyme Q reductase (ambiguous); DPNH-coenzyme Q reductase (ambiguous); DPNH-ubiquinone reductase (ambiguous); mitochondrial electron transport complex 1; mitochondrial electron transport complex I; NADH coenzyme Q1 reductase; NADH-coenzyme Q oxidoreductase (ambiguous); NADH-coenzyme Q reductase (ambiguous); NADH-CoQ oxidoreductase (ambiguous); NADH-CoQ reductase (ambiguous); NADH-ubiquinone reductase (ambiguous); NADH-ubiquinone oxidoreductase (ambiguous); NADH-ubiquinone-1 reductase; reduced nicotinamide adenine dinucleotide-coenzyme Q reductase (ambiguous); NADH:ubiquinone oxidoreductase complex; NADH-Q6 oxidoreductase (ambiguous); electron transfer complex I; NADH2 dehydrogenase (ubiquinone)',\n",
       "  'sys_name': 'NADH:ubiquinone oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FMN) containing iron-sulfur clusters. The complex is present in mitochondria and aerobic bacteria. Breakdown of the complex can release EC 1.6.99.3, NADH dehydrogenase. In photosynthetic bacteria, reversed electron transport through this enzyme can reduce NAD+ to NADH.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.6.5.4': {'ec_num': '1.6.5.4',\n",
       "  'accepted_name': 'monodehydroascorbate reductase (NADH)',\n",
       "  'reaction': 'NADH + H+ + 2 monodehydroascorbate = NAD+ + 2 ascorbate',\n",
       "  'other_names': 'NADH:semidehydroascorbic acid oxidoreductase; MDHA; semidehydroascorbate reductase; AFR (ambiguous); AFR-reductase; ascorbic free radical reductase; ascorbate free radical reductase; SOR (ambiguous); MDAsA reductase (NADPH); SDA reductase; NADH:ascorbate radical oxidoreductase; NADH-semidehydroascorbate oxidoreductase; ascorbate free-radical reductase; NADH:AFR oxidoreductase; monodehydroascorbate reductase (NADH2)',\n",
       "  'sys_name': 'NADH:monodehydroascorbate oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.6.5.5': {'ec_num': '1.6.5.5',\n",
       "  'accepted_name': 'NADPH:quinone reductase',\n",
       "  'reaction': 'NADPH + H+ + 2 quinone = NADP+ + 2 semiquinone',\n",
       "  'other_names': 'NADPH2:quinone reductase',\n",
       "  'sys_name': 'NADPH:quinone oxidoreductase',\n",
       "  'comments': 'A zinc enzyme, specific for NADPH. Catalyses the one-electron reduction of certain quinones, with the orthoquinones 1,2-naphthoquinone and 9,10-phenanthrenequinone being the best substrates [1]. Dicoumarol [cf. EC 1.6.5.2 NAD(P)H dehydrogenase (quinone)] and nitrofurantoin are competitive inhibitors with respect to the quinone substrate. The semiquinone  free-radical product may be non-enzymically reduced to the hydroquinone or oxidized back to quinone in the presence of O2 [1]. In some mammals, the enzyme is abundant in the lens of the eye, where it is identified with the protein ζ-crystallin.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.6.5.6': {'ec_num': '1.6.5.6',\n",
       "  'accepted_name': 'p-benzoquinone reductase (NADPH)',\n",
       "  'reaction': 'NADPH + H+ + p-benzoquinone = NADP+ + hydroquinone',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'NADPH:p-benzoquinone oxidoreductase',\n",
       "  'comments': 'Involved in the 4-nitrophenol degradation pathway in bacteria.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.6.5.6\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.6.5.6\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.6.5.6\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.6.5.6\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.6.5.6%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.6.5.7': {'ec_num': '1.6.5.7',\n",
       "  'accepted_name': '2-hydroxy-1,4-benzoquinone reductase',\n",
       "  'reaction': '2-hydroxy-1,4-benzoquinone + NADH + H+ = hydroxyquinol + NAD+',\n",
       "  'other_names': 'hydroxybenzoquinone reductase; 1,2,4-trihydroxybenzene:NAD oxidoreductase',\n",
       "  'sys_name': 'NADH:2-hydroxy-1,4-benzoquinone oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FMN) that differs in substrate specificity from other quinone reductases. The enzyme in Burkholderia cepacia is inducible by 2,4,5-trichlorophenoxyacetate.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.6.5.7\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.6.5.7\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.6.5.7\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.6.5.7\" target=\"new\">KEGG</a>',\n",
       "  'glossary': 'hydroxyquinol = 1,2,4-trihydroxybenzene',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.6.6.9': {'ec_num': '1.6.6.9',\n",
       "  'accepted_name': 'trimethylamine-N-oxide reductase',\n",
       "  'reaction': 'NADH + H+ + trimethylamine N-oxide = NAD+ + trimethylamine + H2O',\n",
       "  'other_names': 'trimethylamine N-oxide reductase; trimethylamine oxide reductase; TMAO reductase; TOR',\n",
       "  'sys_name': 'NADH:trimethylamine-N-oxide oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.6.6.9\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.6.6.9\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.6.6.9\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.6.6.9\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.6.99.1': {'ec_num': '1.6.99.1',\n",
       "  'accepted_name': 'NADPH dehydrogenase',\n",
       "  'reaction': 'NADPH + H+ + acceptor = NADP+ + reduced acceptor',\n",
       "  'other_names': 'NADPH2 diaphorase; NADPH diaphorase; OYE; diaphorase; dihydronicotinamide adenine dinucleotide phosphate dehydrogenase; NADPH-dehydrogenase; NADPH-diaphorase; NADPH2-dehydrogenase; old yellow enzyme; reduced nicotinamide adenine dinucleotide phosphate dehydrogenase; TPNH dehydrogenase; TPNH-diaphorase; triphosphopyridine diaphorase; triphosphopyridine nucleotide diaphorase; NADPH2 dehydrogenase; NADPH:(acceptor) oxidoreductase',\n",
       "  'sys_name': 'NADPH:acceptor oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FMN in yeast, FAD in plants).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.6.99.1\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.6.99.1\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.6.99.1\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.6.99.1\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.6.99.1%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.6.99.3': {'ec_num': '1.6.99.3',\n",
       "  'accepted_name': 'NADH dehydrogenase',\n",
       "  'reaction': 'NADH + H+ + acceptor = NAD+ + reduced acceptor',\n",
       "  'other_names': 'cytochrome c reductase; type 1 dehydrogenase; β-NADH dehydrogenase dinucleotide; diaphorase; dihydrocodehydrogenase I dehydrogenase; dihydronicotinamide adenine dinucleotide dehydrogenase; diphosphopyridine diaphorase; DPNH diaphorase; NADH diaphorase; NADH hydrogenase; NADH oxidoreductase; NADH-menadione oxidoreductase; reduced diphosphopyridine nucleotide diaphorase; NADH:cytochrome c oxidoreductase; NADH2 dehydrogenase; NADH:(acceptor) oxidoreductase',\n",
       "  'sys_name': 'NADH:acceptor oxidoreductase',\n",
       "  'comments': 'A flavoprotein containing iron-sulfur centres. After preparations have been subjected to certain treatments, cytochrome c may act as an acceptor. Under normal conditions, two protons are extruded from the cytoplasm or the intramitochondrial or stromal compartment. Present in a mitochondrial complex as EC 7.1.1.2, NADH:ubiquinone reductase (H+-translocating).',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.6.99.5': {'ec_num': '1.6.99.5',\n",
       "  'accepted_name': 'NADH dehydrogenase (quinone)',\n",
       "  'reaction': 'NADH + H+ + acceptor = NAD+ + reduced acceptor',\n",
       "  'other_names': 'reduced nicotinamide adenine dinucleotide (quinone) dehydrogenase; NADH-quinone oxidoreductase; DPNH-menadione reductase; D-diaphorase; NADH2 dehydrogenase (quinone)',\n",
       "  'sys_name': 'NADH:(quinone-acceptor) oxidoreductase',\n",
       "  'comments': 'Menaquinone can act as acceptor. Inhibited by AMP and 2,4-dinitrophenol but not by dicoumarol or folic acid derivatives.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.6.99.6': {'ec_num': '1.6.99.6',\n",
       "  'accepted_name': 'NADPH dehydrogenase (quinone)',\n",
       "  'reaction': 'NADPH + H+ + acceptor = NADP+ + reduced acceptor',\n",
       "  'other_names': 'reduced nicotinamide adenine dinucleotide phosphate (quinone) dehydrogenase; NADPH oxidase; NADPH2 dehydrogenase (quinone)',\n",
       "  'sys_name': 'NADPH:(quinone-acceptor) oxidoreductase',\n",
       "  'comments': 'A flavoprotein. Menaquinone can act as acceptor. Inhibited by dicoumarol and folic acid derivatives but not by 2,4-dinitrophenol.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.1.1': {'ec_num': '1.7.1.1',\n",
       "  'accepted_name': 'nitrate reductase (NADH)',\n",
       "  'reaction': 'nitrite + NAD+ + H2O = nitrate + NADH + H+',\n",
       "  'other_names': 'assimilatory nitrate reductase (ambiguous); NADH-nitrate reductase; NADH-dependent nitrate reductase; assimilatory NADH: nitrate reductase; nitrate reductase (NADH2); NADH2:nitrate oxidoreductase',\n",
       "  'sys_name': 'nitrite:NAD+ oxidoreductase',\n",
       "  'comments': 'An iron-sulfur molybdenum flavoprotein.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.7.1.1\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.7.1.1\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.7.1.1\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.7.1.1\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.7.1.1%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.1.2': {'ec_num': '1.7.1.2',\n",
       "  'accepted_name': 'nitrate reductase [NAD(P)H]',\n",
       "  'reaction': 'nitrite + NAD(P)+ + H2O = nitrate + NAD(P)H + H+',\n",
       "  'other_names': 'assimilatory nitrate reductase ( ambiguous); assimilatory NAD(P)H-nitrate reductase; NAD(P)H bispecific nitrate reductase; nitrate reductase (reduced nicotinamide adenine dinucleotide (phosphate)); nitrate reductase NAD(P)H; NAD(P)H-nitrate reductase; nitrate reductase [NAD(P)H2]; NAD(P)H2:nitrate oxidoreductase',\n",
       "  'sys_name': 'nitrite:NAD(P)+ oxidoreductase',\n",
       "  'comments': 'An iron-sulfur molybdenum flavoprotein.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.1.3': {'ec_num': '1.7.1.3',\n",
       "  'accepted_name': 'nitrate reductase (NADPH)',\n",
       "  'reaction': 'nitrite + NADP+ + H2O = nitrate + NADPH + H+',\n",
       "  'other_names': 'assimilatory nitrate reductase (ambiguous); assimilatory reduced nicotinamide adenine dinucleotide phosphate-nitrate reductase; NADPH-nitrate reductase; assimilatory NADPH-nitrate reductase; triphosphopyridine nucleotide-nitrate reductase; NADPH:nitrate reductase; nitrate reductase (NADPH2); NADPH2:nitrate oxidoreductase',\n",
       "  'sys_name': 'nitrite:NADP+ oxidoreductase',\n",
       "  'comments': 'An iron-sulfur molybdenum flavoprotein.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.1.4': {'ec_num': '1.7.1.4',\n",
       "  'accepted_name': 'nitrite reductase [NAD(P)H]',\n",
       "  'reaction': 'NH3 + 3 NAD(P)+ + 2 H2O = nitrite + 3 NAD(P)H + 5 H+',\n",
       "  'other_names': 'nitrite reductase (reduced nicotinamide adenine dinucleotide (phosphate)); assimilatory nitrite reductase (ambiguous); nitrite reductase [NAD(P)H2]; NAD(P)H2:nitrite oxidoreductase; nit-6 (gene name)',\n",
       "  'sys_name': 'ammonia:NAD(P)+ oxidoreductase',\n",
       "  'comments': 'An iron-sulfur flavoprotein (FAD) containing siroheme. The enzymes from the fungi Neurospora crassa [1], Emericella nidulans [2] and Candida nitratophila [4] can use either NADPH or NADH as electron donor. cf. EC 1.7.1.15, nitrite reductase (NADH).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.1.5': {'ec_num': '1.7.1.5',\n",
       "  'accepted_name': 'hyponitrite reductase',\n",
       "  'reaction': '2 hydroxylamine + 2 NAD+ = hyponitrous acid + 2 NADH + 2 H+',\n",
       "  'other_names': 'NADH2:hyponitrite oxidoreductase',\n",
       "  'sys_name': 'hydroxylamine:NAD+ oxidoreductase',\n",
       "  'comments': 'A metalloprotein.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'hyponitrous acid = HO-N=N-OH',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.1.6': {'ec_num': '1.7.1.6',\n",
       "  'accepted_name': 'azobenzene reductase',\n",
       "  'reaction': 'N,N-dimethyl-1,4-phenylenediamine + aniline + 2  NADP+ = 4-(dimethylamino)azobenzene + 2 NADPH + 2 H+',\n",
       "  'other_names': 'new coccine (NC)-reductase; NC-reductase; azo-dye reductase; orange II azoreductase; NAD(P)H:1-(4′-sulfophenylazo)-2-naphthol oxidoreductase; orange I azoreductase; azo reductase; azoreductase; nicotinamide adenine dinucleotide (phosphate) azoreductase; NADPH2-dependent azoreductase; dimethylaminobenzene reductase; p-dimethylaminoazobenzene azoreductase; dibromopropylaminophenylazobenzoic azoreductase; N,N-dimethyl-4-phenylazoaniline azoreductase; p-aminoazobenzene reductase; methyl red azoreductase; NADPH2:4-(dimethylamino)azobenzene oxidoreductase',\n",
       "  'sys_name': 'N,N-dimethyl-1,4-phenylenediamine, aniline:NADP+ oxidoreductase',\n",
       "  'comments': 'The reaction occurs in the reverse direction to that shown above. Other azo dyes, such as Methyl Red, Rocceline, Solar Orange and Sumifix Black B can also be reduced [2].',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.7.1.6\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.7.1.6\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.7.1.6\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.7.1.6\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.7.1.6%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '4-(dimethylamino)azobenzene = Methyl Yellow',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.1.7': {'ec_num': '1.7.1.7',\n",
       "  'accepted_name': 'GMP reductase',\n",
       "  'reaction': 'IMP + NH3 + NADP+ = GMP + NADPH + H+',\n",
       "  'other_names': 'guanosine 5′-monophosphate reductase; NADPH:GMP oxidoreductase (deaminating); guanosine monophosphate reductase; guanylate reductase; NADPH2:guanosine-5′-phosphate oxidoreductase (deaminating); guanosine 5′-phosphate reductase',\n",
       "  'sys_name': 'inosine-5′-phosphate:NADP+ oxidoreductase (aminating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'IMP = inosine 5′-phosphate\\nGMP = guanosine 5′-phosphate',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.1.9': {'ec_num': '1.7.1.9',\n",
       "  'accepted_name': 'nitroquinoline-N-oxide reductase',\n",
       "  'reaction': '4-(hydroxyamino)quinoline N-oxide + 2 NAD(P)+ + H2O = 4-nitroquinoline N-oxide + 2 NAD(P)H + 2 H+',\n",
       "  'other_names': '4-nitroquinoline 1-oxide reductase; 4NQO reductase; NAD(P)H2:4-nitroquinoline-N-oxide oxidoreductase',\n",
       "  'sys_name': '4-(hydroxyamino)quinoline N-oxide:NADP+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.1.10': {'ec_num': '1.7.1.10',\n",
       "  'accepted_name': 'hydroxylamine reductase (NADH)',\n",
       "  'reaction': 'NH3 + NAD+ + H2O = hydroxylamine + NADH + H+',\n",
       "  'other_names': 'hydroxylamine reductase; ammonium dehydrogenase; NADH-hydroxylamine reductase; N-hydroxy amine reductase; hydroxylamine reductase (NADH2); NADH2:hydroxylamine oxidoreductase',\n",
       "  'sys_name': 'ammonium:NAD+ oxidoreductase',\n",
       "  'comments': 'Also acts on some hydroxamates.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.1.11': {'ec_num': '1.7.1.11',\n",
       "  'accepted_name': '4-(dimethylamino)phenylazoxybenzene reductase',\n",
       "  'reaction': '4-(dimethylamino)phenylazobenzene + NADP+ + H2O = 4-(dimethylamino)phenylazoxybenzene + NADPH + H+',\n",
       "  'other_names': 'N,N-dimethyl-p-aminoazobenzene oxide reductase; dimethylaminoazobenzene N-oxide reductase; NADPH-dependent DMAB N-oxide reductase; NADPH:4-(dimethylamino)phenylazoxybenzene oxidoreductase',\n",
       "  'sys_name': '4-(dimethylamino)phenylazobenzene:NADP+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.1.12': {'ec_num': '1.7.1.12',\n",
       "  'accepted_name': 'N-hydroxy-2-acetamidofluorene reductase',\n",
       "  'reaction': '2-acetamidofluorene + NAD(P)+ + H2O = N-hydroxy-2-acetamidofluorene + NAD(P)H + H+',\n",
       "  'other_names': 'N-hydroxy-2-acetylaminofluorene reductase; NAD(P)H2:N-hydroxy-2-acetamidofluorene N-oxidoreductase',\n",
       "  'sys_name': '2-acetamidofluorene:NAD(P)+ oxidoreductase',\n",
       "  'comments': 'Also acts, more slowly, on N-hydroxy-4-acetamidobiphenyl.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.2.1': {'ec_num': '1.7.2.1',\n",
       "  'accepted_name': 'nitrite reductase (NO-forming)',\n",
       "  'reaction': 'nitric oxide + H2O + ferricytochrome c = nitrite + ferrocytochrome c + 2 H+',\n",
       "  'other_names': 'cd-cytochrome nitrite reductase; [nitrite reductase (cytochrome)] [misleading, see comments.]; cytochrome c-551:O2, NO2+ oxidoreductase; cytochrome cd; cytochrome cd1; hydroxylamine (acceptor) reductase; methyl viologen-nitrite reductase; nitrite reductase (cytochrome; NO-forming)',\n",
       "  'sys_name': 'nitric-oxide:ferricytochrome-c oxidoreductase',\n",
       "  'comments': 'The reaction is catalysed by two types of enzymes, found in the perimplasm of denitrifying bacteria. One type comprises proteins containing multiple copper centres, the other a heme protein, cytochrome cd1. Acceptors include c-type cytochromes such as cytochrome c-550 or cytochrome c-551 from Paracoccus denitrificans or Pseudomonas aeruginosa, and small blue copper proteins such as azurin and pseudoazurin. Cytochrome cd1 also has oxidase and hydroxylamine reductase activities. May also catalyse the reaction of hydroxylamine reductase (EC 1.7.99.1) since this is a well-known activity of cytochrome cd1.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.7.2.1\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.7.2.1\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.7.2.1\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.7.2.1\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.7.2.1%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': 'nitric oxide = NO = nitrogen(II) oxide',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.2.2': {'ec_num': '1.7.2.2',\n",
       "  'accepted_name': 'nitrite reductase (cytochrome; ammonia-forming)',\n",
       "  'reaction': 'NH3 + 2 H2O + 6 ferricytochrome c = nitrite + 6 ferrocytochrome c + 7 H+',\n",
       "  'other_names': 'cytochrome c nitrite reductase; multiheme nitrite reductase',\n",
       "  'sys_name': 'ammonia:ferricytochrome-c oxidoreductase',\n",
       "  'comments': 'Found as a multiheme cytochrome in many bacteria. The enzyme from Escherichia coli contains five hemes c and requires Ca2+. It also reduces nitric oxide and hydroxylamine to ammonia, and sulfite to sulfide.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.7.2.2\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.7.2.2\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.7.2.2\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.7.2.2\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.7.2.2%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.2.3': {'ec_num': '1.7.2.3',\n",
       "  'accepted_name': 'trimethylamine-N-oxide reductase',\n",
       "  'reaction': 'trimethylamine + 2 (ferricytochrome c)-subunit + H2O = trimethylamine N-oxide + 2 (ferrocytochrome c)-subunit + 2 H+',\n",
       "  'other_names': 'TMAO reductase; TOR; torA (gene name); torZ (gene name); bisZ (gene name); trimethylamine-N-oxide reductase (cytochrome c)',\n",
       "  'sys_name': 'trimethylamine:cytochrome c oxidoreductase',\n",
       "  'comments': 'Contains bis(molybdopterin guanine dinucleotide)molybdenum cofactor. The reductant is a membrane-bound multiheme cytochrome c. Also reduces dimethyl sulfoxide to dimethyl sulfide.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.3.1': {'ec_num': '1.7.3.1',\n",
       "  'accepted_name': 'nitroalkane oxidase',\n",
       "  'reaction': 'a nitroalkane + H2O + O2 = an aldehyde or ketone + nitrite + H2O2',\n",
       "  'other_names': 'nitroethane oxidase; NAO; nitroethane:oxygen oxidoreductase',\n",
       "  'sys_name': 'nitroalkane:oxygen oxidoreductase',\n",
       "  'comments': 'Has an absolute requirement for FAD [4]. While nitroethane may be the physiological substrate [2], the enzyme also acts on several other nitroalkanes, including 1-nitropropane, 2-nitropropane, 1-nitrobutane, 1-nitropentane, 1-nitrohexane, nitrocyclohexane and some nitroalkanols [4]. Differs from EC 1.13.12.16, nitronate monooxygenase, in that the preferred substrates are neutral nitroalkanes rather than anionic nitronates [4].',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.7.3.1\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.7.3.1\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.7.3.1\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.7.3.1\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.7.3.1%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.3.2': {'ec_num': '1.7.3.2',\n",
       "  'accepted_name': 'acetylindoxyl oxidase',\n",
       "  'reaction': 'N-acetylindoxyl + O2 = N-acetylisatin + (?)',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'N-acetylindoxyl:oxygen oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.3.3': {'ec_num': '1.7.3.3',\n",
       "  'accepted_name': 'factor-independent urate hydroxylase',\n",
       "  'reaction': 'urate + O2 + H2O = 5-hydroxyisourate + H2O2',\n",
       "  'other_names': 'uric acid oxidase; uricase; uricase II; urate oxidase',\n",
       "  'sys_name': 'urate:oxygen oxidoreductase',\n",
       "  'comments': 'This enzyme was previously thought to be a copper protein, but it is now known that the enzymes from soy bean (Glycine max), the mould Aspergillus flavus and Bacillus subtilis contains no copper nor any other transition-metal ion. The 5-hydroxyisourate formed decomposes spontaneously to form allantoin and CO2, although there is an enzyme-catalysed pathway in which EC 3.5.2.17, hydroxyisourate hydrolase, catalyses the first step. The enzyme is different from EC 1.14.13.113 (FAD-dependent urate hydroxylase).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.7.3.3\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.7.3.3\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.7.3.3\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.7.3.3\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.7.3.3%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.3.4': {'ec_num': '1.7.3.4',\n",
       "  'accepted_name': 'hydroxylamine oxidase',\n",
       "  'reaction': 'hydroxylamine + O2 + ferricytochrome c = nitrite + H2O + ferrocytochrome c',\n",
       "  'other_names': 'HAO (ambiguous); hydroxylamine oxidoreductase (ambiguous)',\n",
       "  'sys_name': 'hydroxylamine:oxygen oxidoreductase',\n",
       "  'comments': 'The enzyme from the heterotrophic nitrifying bacterium Paracoccus denitrificans contains three to five non-heme, non-iron-sulfur iron atoms and interacts with cytochrome c556 and pseudoazurin [2,3].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.3.5': {'ec_num': '1.7.3.5',\n",
       "  'accepted_name': '3-aci-nitropropanoate oxidase',\n",
       "  'reaction': '3-aci-nitropropanoate + O2 + H2O = 3-oxopropanoate + nitrite + H2O2',\n",
       "  'other_names': 'propionate-3-nitronate oxidase',\n",
       "  'sys_name': '3-aci-nitropropanoate:oxygen oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FMN). The primary products of the enzymic reaction are probably the nitropropanoate free radical and superoxide. Also acts, more slowly, on 4-aci-nitrobutanoate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.7.1': {'ec_num': '1.7.7.1',\n",
       "  'accepted_name': 'ferredoxin—nitrite reductase',\n",
       "  'reaction': 'NH3 + 2 H2O + 6 oxidized ferredoxin = nitrite + 6 reduced ferredoxin + 7 H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'ammonia:ferredoxin oxidoreductase',\n",
       "  'comments': 'An iron protein. Contains siroheme and [4Fe-4S] clusters.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.7.2': {'ec_num': '1.7.7.2',\n",
       "  'accepted_name': 'ferredoxin—nitrate reductase',\n",
       "  'reaction': 'nitrite + H2O + 2 oxidized ferredoxin = nitrate + 2 reduced ferredoxin + 2 H+',\n",
       "  'other_names': 'assimilatory nitrate reductase (ambiguous); nitrate (ferredoxin) reductase; assimilatory ferredoxin-nitrate reductase',\n",
       "  'sys_name': 'nitrite:ferredoxin oxidoreductase',\n",
       "  'comments': 'A molybdenum-iron-sulfur protein.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.7.7.2\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.7.7.2\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.7.7.2\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.7.7.2\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.7.7.2%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.99.1': {'ec_num': '1.7.99.1',\n",
       "  'accepted_name': 'hydroxylamine reductase',\n",
       "  'reaction': 'NH3 + H2O + acceptor = hydroxylamine + reduced acceptor',\n",
       "  'other_names': 'hydroxylamine (acceptor) reductase; ammonia:(acceptor) oxidoreductase',\n",
       "  'sys_name': 'ammonia:acceptor oxidoreductase',\n",
       "  'comments': 'A flavoprotein. Reduced pyocyanine, methylene blue and flavins act as donors for the reduction of hydroxylamine. May be identical to EC 1.7.2.1, nitrite reductase (NO-forming).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.99.4': {'ec_num': '1.7.99.4',\n",
       "  'accepted_name': 'nitrate reductase',\n",
       "  'reaction': 'nitrite + acceptor = nitrate + reduced acceptor',\n",
       "  'other_names': 'respiratory nitrate reductase; nitrate reductase (acceptor); nitrite:(acceptor) oxidoreductase',\n",
       "  'sys_name': 'nitrite:acceptor oxidoreductase',\n",
       "  'comments': 'The Pseudomonas enzyme is a cytochrome, but the enzyme from Micrococcus halodenitrificans is an iron protein containing molybdenum. Reduced benzyl viologen and other dyes bring about the reduction of nitrate.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.7.99.4\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.7.99.4\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.7.99.4\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.7.99.4\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.7.99.4%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.99.6': {'ec_num': '1.7.99.6',\n",
       "  'accepted_name': 'nitrous-oxide reductase',\n",
       "  'reaction': 'nitrogen + H2O + acceptor = nitrous oxide + reduced acceptor',\n",
       "  'other_names': 'nitrous oxide reductase; N2O reductase; nitrogen:(acceptor) oxidoreductase (N2O-forming)',\n",
       "  'sys_name': 'nitrogen:acceptor oxidoreductase (N2O-forming)',\n",
       "  'comments': 'A copper protein. Reduced viologens or methylene blue act as donors for the reduction of nitrous oxide.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.99.7': {'ec_num': '1.7.99.7',\n",
       "  'accepted_name': 'nitric-oxide reductase',\n",
       "  'reaction': 'nitrous oxide + acceptor + H2O = 2 nitric oxide + reduced acceptor',\n",
       "  'other_names': 'nitrogen oxide reductase; nitrous-oxide:(acceptor) oxidoreductase (NO-forming)',\n",
       "  'sys_name': 'nitrous-oxide:acceptor oxidoreductase (NO-forming)',\n",
       "  'comments': 'A heterodimer of cytochromes b and c. Phenazine methosulfate can act as acceptor.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.7.99.8': {'ec_num': '1.7.99.8',\n",
       "  'accepted_name': 'hydrazine oxidoreductase',\n",
       "  'reaction': 'hydrazine + acceptor = N2 + reduced acceptor',\n",
       "  'other_names': 'HAO (ambiguous; cf. EC 1.7.3.4 hydroxylamine oxidase)',\n",
       "  'sys_name': 'hydrazine:acceptor oxidoreductase',\n",
       "  'comments': 'The enzyme is involved in the pathway of anaerobic ammonium oxidation in anammox bacteria. The conversion of hydroxylamine by the enzyme from Brocadia anammoxidans results in the formation of NO and N2O [1]. Hydroxylamine is not a substrate for the enzyme from planctomycete KSU-1 [2]. ',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.56': {'ec_num': '1.13.11.56',\n",
       "  'accepted_name': '1,2-dihydroxynaphthalene dioxygenase',\n",
       "  'reaction': 'naphthalene-1,2-diol + O2 = 2-hydroxy-2H-chromene-2-carboxylate',\n",
       "  'other_names': '1,2-DHN dioxygenase; DHNDO; 1,2-dihydroxynaphthalene oxygenase; 1,2-dihydroxynaphthalene:oxygen oxidoreductase',\n",
       "  'sys_name': 'naphthalene-1,2-diol:oxygen oxidoreductase',\n",
       "  'comments': 'This enzyme is involved in naphthalene degradation. Requires Fe2+.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.56\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.56\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.56\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.56\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.13.11.56%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.1.2': {'ec_num': '1.8.1.2',\n",
       "  'accepted_name': 'assimilatory sulfite reductase (NADPH)',\n",
       "  'reaction': 'hydrogen sulfide + 3 NADP+ + 3 H2O = sulfite + 3 NADPH + 3 H+',\n",
       "  'other_names': 'sulfite reductase (NADPH); sulfite (reduced nicotinamide adenine dinucleotide phosphate) reductase; NADPH-sulfite reductase; NADPH-dependent sulfite reductase; H2S-NADP oxidoreductase; sulfite reductase (NADPH2); MET5 (gene name); MET10 (gene name); cysI (gene name); cysJ (gene name)',\n",
       "  'sys_name': 'hydrogen-sulfide:NADP+ oxidoreductase',\n",
       "  'comments': 'Contains siroheme, [4Fe-4S] cluster, FAD and FMN. The enzyme, which catalyses the six-electron reduction of sulfite to sulfide, is involved in sulfate assimilation in bacteria and yeast. Different from EC 1.8.99.5, dissimilatory sulfite reductase, which is involved in prokaryotic sulfur-based energy metabolism. cf. EC 1.8.7.1, assimilatory sulfite reductase (ferredoxin).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.1.3': {'ec_num': '1.8.1.3',\n",
       "  'accepted_name': 'hypotaurine dehydrogenase',\n",
       "  'reaction': 'hypotaurine + H2O + NAD+ = taurine + NADH + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'hypotaurine:NAD+ oxidoreductase',\n",
       "  'comments': 'A molybdohemoprotein.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.1.4': {'ec_num': '1.8.1.4',\n",
       "  'accepted_name': 'dihydrolipoyl dehydrogenase',\n",
       "  'reaction': 'protein N6-(dihydrolipoyl)lysine + NAD+ = protein N6-(lipoyl)lysine + NADH + H+',\n",
       "  'other_names': 'LDP-Glc; LDP-Val; dehydrolipoate dehydrogenase; diaphorase; dihydrolipoamide dehydrogenase; dihydrolipoamide:NAD+ oxidoreductase; dihydrolipoic dehydrogenase; dihydrothioctic dehydrogenase; lipoamide dehydrogenase (NADH); lipoamide oxidoreductase (NADH); lipoamide reductase; lipoamide reductase (NADH); lipoate dehydrogenase; lipoic acid dehydrogenase; lipoyl dehydrogenase; protein-6-N-(dihydrolipoyl)lysine:NAD+ oxidoreductase',\n",
       "  'sys_name': 'protein-N6-(dihydrolipoyl)lysine:NAD+ oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD). A component of the multienzyme 2-oxo-acid dehydrogenase complexes. In the pyruvate dehydrogenase complex, it binds to the core of EC 2.3.1.12, dihydrolipoyllysine-residue acetyltransferase, and catalyses oxidation of its dihydrolipoyl groups. It plays a similar role in the oxoglutarate and 3-methyl-2-oxobutanoate dehydrogenase complexes. Another substrate is the dihydrolipoyl group in the H-protein of the glycine-cleavage system (click here for diagram), in which it acts, together with EC 1.4.4.2, glycine dehydrogenase (decarboxylating), and EC 2.1.2.10, aminomethyltransferase, to break down glycine. It can also use free dihydrolipoate, dihydrolipoamide or dihydrolipoyllysine as substrate. This enzyme was first shown to catalyse the oxidation of NADH by methylene blue; this activity was called diaphorase. The glycine cleavage system is composed of four components that only loosely associate: the P protein (EC 1.4.4.2), the T protein (EC 2.1.2.10), the L protein (EC 1.8.1.4) and the lipoyl-bearing H protein [6].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'dihydrolipoyl = (6R)-6,8-disulfanyloctanoyl\\nFor structure of dihydrolipoyl, click here',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.1.5': {'ec_num': '1.8.1.5',\n",
       "  'accepted_name': '2-oxopropyl-CoM reductase (carboxylating)',\n",
       "  'reaction': 'CoM + acetoacetate + NADP+ = 2-oxopropyl-CoM + CO2 + NADPH',\n",
       "  'other_names': 'NADPH:2-(2-ketopropylthio)ethanesulfonate oxidoreductase/carboxylase; NADPH:2-ketopropyl-coenzyme M oxidoreductase/carboxylase; 2-mercaptoethanesulfonate,acetoacetate:NADP+ oxidoreductase (decarboxylating)',\n",
       "  'sys_name': '2-sulfanylethane-1-sulfonate,acetoacetate:NADP+ oxidoreductase (decarboxylating)',\n",
       "  'comments': 'Also acts on thioethers longer in chain length on the oxo side, e.g. 2-oxobutyl-CoM, but this portion must be attached to CoM (2-sulfanylethane-1-sulfonate); no CoM analogs will substitute. This enzyme forms component II of a four-component enzyme system <a target=\"new\" href=\"reaction/comprising <a href=\"query.php?ec=4.4.1.23\" target=\"new\">EC 4.4.1.23</a> (2-hydroxypropyl-CoM lyase; component I), EC 1.8.1.5 [2-oxopropyl-CoM reductase (carboxylating); component II], <a href=\"query.php?ec=1.1.1.268\" target=\"new\">EC 1.1.1.268</a> [2-(<em>R</em>)-hydroxypropyl-CoM dehydrogenase; component III] and <a href=\"query.php?ec=1.1.1.269\" target=\"new\">EC 1.1.1.269</a> [2-(<em>S</em>)-hydroxypropyl-CoM dehydrogenase; component IV].html\">click here</a> that is involved in epoxyalkane carboxylation in <em>Xanthobacter</em> sp. strain Py2.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.8.1.5\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.8.1.5\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.8.1.5\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.8.1.5\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.8.1.5%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': 'coenzyme M (CoM) = 2-sulfanylethane-1-sulfonate = 2-mercaptoethanesulfonate (deprecated)',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.1.6': {'ec_num': '1.8.1.6',\n",
       "  'accepted_name': 'cystine reductase',\n",
       "  'reaction': '2 L-cysteine + NAD+ = L-cystine + NADH + H+',\n",
       "  'other_names': 'cystine reductase (NADH); NADH-dependent cystine reductase; cystine reductase (NADH2); NADH2:L-cystine oxidoreductase',\n",
       "  'sys_name': 'L-cysteine:NAD+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.1.7': {'ec_num': '1.8.1.7',\n",
       "  'accepted_name': 'glutathione-disulfide reductase',\n",
       "  'reaction': '2 glutathione + NADP+ = glutathione disulfide + NADPH + H+',\n",
       "  'other_names': 'glutathione reductase; glutathione reductase (NADPH); NADPH-glutathione reductase; GSH reductase; GSSG reductase; NADPH-GSSG reductase; glutathione S-reductase; NADPH:oxidized-glutathione oxidoreductase',\n",
       "  'sys_name': 'glutathione:NADP+ oxidoreductase',\n",
       "  'comments': 'A dimeric flavoprotein (FAD); activity is dependent on a redox-active disulfide in each of the active centres.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': 'The term ’oxidized glutathione’ has been replaced by the term ’glutathione disulfide’ as the former is ambiguous. S,S′-Biglutathione may also be used to refer to this compound.',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.1.8': {'ec_num': '1.8.1.8',\n",
       "  'accepted_name': 'protein-disulfide reductase',\n",
       "  'reaction': 'protein-dithiol + NAD(P)+ = protein-disulfide + NAD(P)H + H+',\n",
       "  'other_names': 'protein disulphide reductase; insulin-glutathione transhydrogenase; disulfide reductase; NAD(P)H2:protein-disulfide oxidoreductase',\n",
       "  'sys_name': 'protein-dithiol:NAD(P)+ oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.1.9': {'ec_num': '1.8.1.9',\n",
       "  'accepted_name': 'thioredoxin-disulfide reductase',\n",
       "  'reaction': 'thioredoxin + NADP+ = thioredoxin disulfide + NADPH + H+',\n",
       "  'other_names': 'NADP-thioredoxin reductase; NADPH-thioredoxin reductase; thioredoxin reductase (NADPH); NADPH2:oxidized thioredoxin oxidoreductase',\n",
       "  'sys_name': 'thioredoxin:NADP+ oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD). ',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': 'The term ’oxidized thioredoxin’ has been replaced by ’thioredoxin disulfide’ as the former is ambiguous',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.1.10': {'ec_num': '1.8.1.10',\n",
       "  'accepted_name': 'CoA-glutathione reductase',\n",
       "  'reaction': 'CoA + glutathione + NADP+ = CoA-glutathione + NADPH + H+',\n",
       "  'other_names': 'coenzyme A glutathione disulfide reductase; NADPH-dependent coenzyme A-SS-glutathione reductase; coenzyme A disulfide-glutathione reductase; NADPH2:CoA-glutathione oxidoreductase',\n",
       "  'sys_name': 'glutathione:NADP+ oxidoreductase (CoA-acylating)',\n",
       "  'comments': 'A flavoprotein. The substrate is a mixed disulfide. May be identical to EC 1.8.1.9, thioredoxin-disulfide reductase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.1.11': {'ec_num': '1.8.1.11',\n",
       "  'accepted_name': 'asparagusate reductase',\n",
       "  'reaction': '3-sulfanyl-2-(sulfanylmethyl)propanoate + NAD+ = asparagusate + NADH + H+',\n",
       "  'other_names': 'asparagusate dehydrogenase; asparagusic dehydrogenase; asparagusate reductase (NADH2); NADH2:asparagusate oxidoreductase; 3-mercapto-2-mercaptomethylpropanoate:NAD+ oxidoreductase',\n",
       "  'sys_name': '3-sulfanyl-2-(sulfanylmethyl)propanoate:NAD+ oxidoreductase',\n",
       "  'comments': 'Also acts on lipoate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '3-sulfanyl-2-(sulfanylmethyl)propanoate = 3-mercapto-2-mercaptomethylpropanoate (deprecated)\\nasparagusate = 1,2-dithiolane-4-carboxylate',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.1.12': {'ec_num': '1.8.1.12',\n",
       "  'accepted_name': 'trypanothione-disulfide reductase',\n",
       "  'reaction': 'trypanothione + NADP+ = trypanothione disulfide + NADPH + H+',\n",
       "  'other_names': 'trypanothione reductase; NADPH2:trypanothione oxidoreductase',\n",
       "  'sys_name': 'trypanothione:NADP+ oxidoreductase',\n",
       "  'comments': 'Trypanothione disulfide is the oxidized form of N1,N8-bis(glutathionyl)-spermidine from the insect-parasitic trypanosomatid Crithidia fasciculata. The enzyme from Crithidia fasciculata is a flavoprotein (FAD), whose activity is dependent on a redox-active cystine at the active centre. (cf. EC 1.8.1.7, glutathione-disulfide reductase)',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '<a target=\"new\" href=\"glossary/spermidine.html\">spermidine</a> = <em>N</em>-(3-aminopropyl)butane-1,4-diamine<br>\\ntrypanothione = <em>N</em><small><sup>1</sup></small>,<em>N</em><small><sup>8</sup></small>-bis(glutathionyl)spermidine',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.1.13': {'ec_num': '1.8.1.13',\n",
       "  'accepted_name': 'bis-γ-glutamylcystine reductase',\n",
       "  'reaction': '2 γ-glutamylcysteine + NADP+ = bis-γ-glutamylcystine + NADPH + H+',\n",
       "  'other_names': 'NADPH2:bis-γ-glutamylcysteine oxidoreductase; GSR',\n",
       "  'sys_name': 'γ-glutamylcysteine:NADP+ oxidoreductase',\n",
       "  'comments': 'Contains FAD. The enzyme, which is found only in halobacteria, maintains the concentration of γ-glutamylcysteine, the major low molecular weight thiol in halobacteria. Not identical with EC 1.8.1.7 (glutathione-disulfide reductase) or EC 1.8.1.14 (CoA-disulfide reductase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.1.14': {'ec_num': '1.8.1.14',\n",
       "  'accepted_name': 'CoA-disulfide reductase',\n",
       "  'reaction': '2 CoA + NADP+ = CoA-disulfide + NADPH + H+',\n",
       "  'other_names': 'CoA-disulfide reductase (NADH2); NADH2:CoA-disulfide oxidoreductase; CoA:NAD+ oxidoreductase (misleading); CoADR; coenzyme A disulfide reductase',\n",
       "  'sys_name': 'CoA:NADP+ oxidoreductase',\n",
       "  'comments': 'A flavoprotein. Not identical with EC 1.8.1.6 (cystine reductase), EC 1.8.1.7 (glutathione-disulfide reductase) or EC 1.8.1.13 (bis-γ-glutamylcystine reductase). The enzyme from the bacterium Staphylococcus aureus has a strong preference for NADPH [3], while the bacterium Bacillus megaterium contains both NADH and NADPH-dependent enzymes [1].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.1.15': {'ec_num': '1.8.1.15',\n",
       "  'accepted_name': 'mycothione reductase',\n",
       "  'reaction': '2 mycothiol + NAD(P)+ = mycothione + NAD(P)H + H+',\n",
       "  'other_names': 'mycothiol-disulfide reductase',\n",
       "  'sys_name': 'mycothiol:NAD(P)+ oxidoreductase',\n",
       "  'comments': 'Contains FAD. No activity with glutathione, trypanothione or coenzyme A as substrate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'mycothiol = 1-O-[2-(N2-acetyl-L-cysteinamido)-2-deoxy--D-glucopyranosyl]-1D-myo-inositol\\nmycothione = oxidized (disulfide) form of mycothiol',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.2.1': {'ec_num': '1.8.2.1',\n",
       "  'accepted_name': 'sulfite dehydrogenase (cytochrome)',\n",
       "  'reaction': 'sulfite + 2 ferricytochrome c + H2O = sulfate + 2 ferrocytochrome c + 2 H+',\n",
       "  'other_names': 'sulfite cytochrome c reductase; sulfite-cytochrome c oxidoreductase; sulfite oxidase (ambiguous); sulfite dehydrogenase (ambiguous); sorAB (gene names)',\n",
       "  'sys_name': 'sulfite:ferricytochrome-c oxidoreductase',\n",
       "  'comments': 'Associated with cytochrome c-551. The enzyme from the bacterium Starkeya novella contains a molybdopyranopterin cofactor and a smaller monoheme cytochrome c subunit. cf. EC 1.8.5.6, sulfite dehydrogenase (quinone).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.2.2': {'ec_num': '1.8.2.2',\n",
       "  'accepted_name': 'thiosulfate dehydrogenase',\n",
       "  'reaction': '2 thiosulfate + 2 ferricytochrome c = tetrathionate + 2 ferrocytochrome c',\n",
       "  'other_names': 'tsdA (gene name); tetrathionate synthase; thiosulfate oxidase; thiosulfate-oxidizing enzyme; thiosulfate-acceptor oxidoreductase',\n",
       "  'sys_name': 'thiosulfate:ferricytochrome-c oxidoreductase',\n",
       "  'comments': 'The enzyme catalyses the reversible formation of a sulfur-sulfur bond between the sulfane atoms of two thiosulfate molecules, yielding tetrathionate and releasing two electrons. In many bacterial species the enzyme is a diheme c-type cytochrome. In a number of organisms, including Thiomonas intermedia and Sideroxydans lithotrophicus, a second diheme cytochrome (TsdB) acts as the electron acceptor. However, some organisms, such as Allochromatium vinosum, lack TsdB. The electron acceptor in these organisms may be the high-potential iron-sulfur protein (HiPIP).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.3.1': {'ec_num': '1.8.3.1',\n",
       "  'accepted_name': 'sulfite oxidase',\n",
       "  'reaction': 'sulfite + O2 + H2O = sulfate + H2O2',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'sulfite:oxygen oxidoreductase',\n",
       "  'comments': 'A molybdohemoprotein.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.3.2': {'ec_num': '1.8.3.2',\n",
       "  'accepted_name': 'thiol oxidase',\n",
       "  'reaction': '2 R′C(R)SH + O2 = R′C(R)S-S(R)CR′ + H2O2',\n",
       "  'other_names': 'sulfhydryl oxidase',\n",
       "  'sys_name': 'thiol:oxygen oxidoreductase',\n",
       "  'comments': 'R may be =S or =O, or a variety of other groups. The enzyme is not specific for R′.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.8.3.2\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.8.3.2\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.8.3.2\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.8.3.2\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.8.3.2%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.3.3': {'ec_num': '1.8.3.3',\n",
       "  'accepted_name': 'glutathione oxidase',\n",
       "  'reaction': '2 glutathione + O2 = glutathione disulfide + H2O2',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'glutathione:oxygen oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD). Also acts, more slowly, on L-cysteine and several other thiols.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.3.4': {'ec_num': '1.8.3.4',\n",
       "  'accepted_name': 'methanethiol oxidase',\n",
       "  'reaction': 'methanethiol + O2 + H2O = formaldehyde + hydrogen sulfide + H2O2',\n",
       "  'other_names': 'methylmercaptan oxidase; methyl mercaptan oxidase; (MM)-oxidase; MT-oxidase',\n",
       "  'sys_name': 'methanethiol:oxygen oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.8.3.4\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.8.3.4\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.8.3.4\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.8.3.4\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.3.5': {'ec_num': '1.8.3.5',\n",
       "  'accepted_name': 'prenylcysteine oxidase',\n",
       "  'reaction': 'an S-prenyl-L-cysteine + O2 + H2O = a prenal + L-cysteine + H2O2',\n",
       "  'other_names': 'prenylcysteine lyase',\n",
       "  'sys_name': 'S-prenyl-L-cysteine:oxygen oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD). Cleaves the thioether bond of S-prenyl-L-cysteines, such as S-farnesylcysteine and S-geranylgeranylcysteine. N-Acetyl-prenylcysteine and prenylcysteinyl peptides are not substrates. May represent the final step in the degradation of prenylated proteins in mammalian tissues. Originally thought to be a simple lyase so it had been classified as EC 4.4.1.18.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.4.1': {'ec_num': '1.8.4.1',\n",
       "  'accepted_name': 'glutathione—homocystine transhydrogenase',\n",
       "  'reaction': '2 glutathione + homocystine = glutathione disulfide + 2 homocysteine',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'glutathione:homocystine oxidoreductase',\n",
       "  'comments': 'The reactions catalysed by this enzyme and by others in this subclass may be similar to those catalysed by EC 2.5.1.18 glutathione transferase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.4.2': {'ec_num': '1.8.4.2',\n",
       "  'accepted_name': 'protein-disulfide reductase (glutathione)',\n",
       "  'reaction': '2 glutathione + protein-disulfide = glutathione-disulfide + protein-dithiol',\n",
       "  'other_names': 'glutathione-insulin transhydrogenase; insulin reductase; reductase, protein disulfide (glutathione); protein disulfide transhydrogenase; glutathione-protein disulfide oxidoreductase; protein disulfide reductase (glutathione); GSH-insulin transhydrogenase; protein-disulfide interchange enzyme; protein-disulfide isomerase/oxidoreductase; thiol:protein-disulfide oxidoreductase; thiol-protein disulphide oxidoreductase',\n",
       "  'sys_name': 'glutathione:protein-disulfide oxidoreductase',\n",
       "  'comments': 'Reduces insulin and some other proteins.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.4.3': {'ec_num': '1.8.4.3',\n",
       "  'accepted_name': 'glutathione—CoA-glutathione transhydrogenase',\n",
       "  'reaction': 'CoA + glutathione disulfide = CoA-glutathione + glutathione',\n",
       "  'other_names': 'glutathione-coenzyme A glutathione disulfide transhydrogenase; glutathione-coenzyme A glutathione disulfide transhydrogenase; glutathione coenzyme A-glutathione transhydrogenase; glutathione:coenzyme A-glutathione transhydrogenase; coenzyme A:oxidized-glutathione oxidoreductase; coenzyme A:glutathione-disulfide oxidoreductase ',\n",
       "  'sys_name': 'CoA:glutathione-disulfide oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.4.4': {'ec_num': '1.8.4.4',\n",
       "  'accepted_name': 'glutathione—cystine transhydrogenase',\n",
       "  'reaction': '2 glutathione + cystine = glutathione disulfide + 2 cysteine',\n",
       "  'other_names': 'GSH-cystine transhydrogenase; NADPH-dependent GSH-cystine transhydrogenase',\n",
       "  'sys_name': 'glutathione:cystine oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.4.5': {'ec_num': '1.8.4.5',\n",
       "  'accepted_name': 'methionine-S-oxide reductase',\n",
       "  'reaction': ' L-methionine + thioredoxin disulfide + H2O = L-methionine S-oxide + thioredoxin',\n",
       "  'other_names': 'methyl sulfoxide reductase I and II; acetylmethionine sulfoxide reductase; methionine sulfoxide reductase; L-methionine:oxidized-thioredoxin S-oxidoreductase',\n",
       "  'sys_name': 'L-methionine:thioredoxin-disulfide S-oxidoreductase',\n",
       "  'comments': ' In the reverse reaction, dithiothreitol can replace reduced thioredoxin, and other methyl sulfoxides can replace methionine sulfoxide.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.4.6': {'ec_num': '1.8.4.6',\n",
       "  'accepted_name': 'protein-methionine-S-oxide reductase',\n",
       "  'reaction': 'protein L-methionine + thioredoxin disulfide = protein L-methionine S-oxide + thioredoxin',\n",
       "  'other_names': 'methionine sulfoxide (protein) reductase; methionine sulfoxide peptide reductase; protein (methionine sulfoxide) reductase; Met(O)-peptide reductase; peptide methionine sulfoxide reductase; protein-L-methionine:oxidized-thioredoxin S-oxidoreductase',\n",
       "  'sys_name': 'protein-L-methionine:thioredoxin-disulfide S-oxidoreductase',\n",
       "  'comments': 'Dithiothreitol, but not 2-mercaptoethanol, can replace a reduced thioredoxin in the reverse reaction. Does not act on free methionine (cf. EC 1.8.4.5 methionine-S-oxide reductase)',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.4.7': {'ec_num': '1.8.4.7',\n",
       "  'accepted_name': 'enzyme-thiol transhydrogenase (glutathione-disulfide)',\n",
       "  'reaction': '[xanthine dehydrogenase] + glutathione disulfide = [xanthine oxidase] + 2 glutathione',\n",
       "  'other_names': '[xanthine-dehydrogenase]:oxidized-glutathione S-oxidoreductase; enzyme-thiol transhydrogenase (oxidized-glutathione); glutathione-dependent thiol:disulfide oxidoreductase; thiol:disulphide oxidoreductase',\n",
       "  'sys_name': '[xanthine-dehydrogenase]:glutathione-disulfide S-oxidoreductase',\n",
       "  'comments': 'Converts EC 1.17.1.4 xanthine dehydrogenase into EC 1.17.3.2 xanthine oxidase in the presence of glutathione disulfide; also reduces the disulfide bond of ricin. Not inhibited by Cu2+ or thiol reagents.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.4.8': {'ec_num': '1.8.4.8',\n",
       "  'accepted_name': 'phosphoadenylyl-sulfate reductase (thioredoxin)',\n",
       "  'reaction': 'adenosine 3′,5′-bisphosphate + sulfite + thioredoxin disulfide = 3′-phosphoadenylyl sulfate + thioredoxin',\n",
       "  'other_names': 'PAPS reductase, thioredoxin-dependent; PAPS reductase; thioredoxin:adenosine 3′-phosphate 5′-phosphosulfate reductase; 3′-phosphoadenylylsulfate reductase; thioredoxin:3′-phospho-adenylylsulfate reductase; phosphoadenosine-phosphosulfate reductase; adenosine 3′,5′-bisphosphate,sulfite:oxidized-thioredoxin oxidoreductase (3′-phosphoadenosine-5′-phosphosulfate-forming)',\n",
       "  'sys_name': 'adenosine 3′,5′-bisphosphate,sulfite:thioredoxin-disulfide oxidoreductase (3′-phosphoadenosine-5′-phosphosulfate-forming)',\n",
       "  'comments': 'Specific for PAPS. The enzyme from Escherichia coli will use thioredoxins from other species.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '3′-phosphoadenylyl sulfate = PAPS',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.4.9': {'ec_num': '1.8.4.9',\n",
       "  'accepted_name': 'adenylyl-sulfate reductase (glutathione)',\n",
       "  'reaction': 'AMP + sulfite + glutathione disulfide = adenylyl sulfate + 2 glutathione',\n",
       "  'other_names': '5′-adenylylsulfate reductase (also used for EC 1.8.99.2); AMP,sulfite:oxidized-glutathione oxidoreductase (adenosine-5′-phosphosulfate-forming); plant-type 5′-adenylylsulfate reductase',\n",
       "  'sys_name': 'AMP,sulfite:glutathione-disulfide oxidoreductase (adenosine-5′-phosphosulfate-forming)',\n",
       "  'comments': 'This enzyme differs from EC 1.8.99.2, adenylyl-sulfate reductase, in using glutathione as the reductant. Glutathione can be replaced by γ-glutamylcysteine or dithiothreitol, but not by thioredoxin, glutaredoxin or 2-sulfanylethan-1-ol (2-mercaptoethanol). The enzyme from the mouseear cress, Arabidopsis thaliana, contains a glutaredoxin-like domain. The enzyme is also found in other photosynthetic eukaryotes, e.g., the Madagascar periwinkle, Catharanthus roseus and the hollow green seaweed, Ulva intestinalis.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.4.10': {'ec_num': '1.8.4.10',\n",
       "  'accepted_name': 'adenylyl-sulfate reductase (thioredoxin)',\n",
       "  'reaction': 'AMP + sulfite + thioredoxin disulfide = 5′-adenylyl sulfate + thioredoxin',\n",
       "  'other_names': 'thioredoxin-dependent 5′-adenylylsulfate reductase',\n",
       "  'sys_name': 'AMP,sulfite:thioredoxin-disulfide oxidoreductase (adenosine-5′-phosphosulfate-forming)',\n",
       "  'comments': 'Uses adenylyl sulfate, not phosphoadenylyl sulfate, distinguishing this enzyme from EC 1.8.4.8, phosphoadenylyl-sulfate reductase (thioredoxin). Uses thioredoxin as electron donor, not glutathione or other donors, distinguishing it from EC 1.8.4.9 [adenylyl-sulfate reductase (glutathione)] and EC 1.8.99.2 (adenylyl-sulfate reductase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.5.1': {'ec_num': '1.8.5.1',\n",
       "  'accepted_name': 'glutathione dehydrogenase (ascorbate)',\n",
       "  'reaction': '2 glutathione + dehydroascorbate = glutathione disulfide + ascorbate',\n",
       "  'other_names': 'dehydroascorbic reductase; dehydroascorbic acid reductase; glutathione dehydroascorbate reductase; DHA reductase ; dehydroascorbate reductase; GDOR; glutathione:dehydroascorbic acid oxidoreductase',\n",
       "  'sys_name': 'glutathione:dehydroascorbate oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.5.2': {'ec_num': '1.8.5.2',\n",
       "  'accepted_name': 'thiosulfate dehydrogenase (quinone)',\n",
       "  'reaction': '2 thiosulfate + 6-decylubiquinone = tetrathionate + 6-decylubiquinol',\n",
       "  'other_names': 'thiosulfate:quinone oxidoreductase; thiosulphate:quinone oxidoreductase; thiosulfate oxidoreductase, tetrathionate-forming; TQO',\n",
       "  'sys_name': 'thiosulfate:6-decylubiquinone oxidoreductase',\n",
       "  'comments': 'The reaction can also proceed with ferricyanide as the electron acceptor, but more slowly. Unlike EC 1.8.2.2, thiosulfate dehydrogenase, this enzyme cannot utilize cytochrome c as an acceptor.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.7.1': {'ec_num': '1.8.7.1',\n",
       "  'accepted_name': 'assimilatory sulfite reductase (ferredoxin)',\n",
       "  'reaction': 'hydrogen sulfide + 6 oxidized ferredoxin [iron-sulfur] cluster + 3 H2O = sulfite + 6 reduced ferredoxin [iron-sulfur] cluster + 6 H+',\n",
       "  'other_names': 'ferredoxin-sulfite reductase; SIR (gene name); sulfite reductase (ferredoxin)',\n",
       "  'sys_name': 'hydrogen-sulfide:ferredoxin oxidoreductase',\n",
       "  'comments': 'An iron protein. The enzyme participates in sulfate assimilation. While it is usually found in cyanobacteria, plants and algae, it has also been reported in bacteria [4]. Different from EC 1.8.99.5, dissimilatory sulfite reductase, which is involved in prokaryotic sulfur-based energy metabolism. cf. EC 1.8.1.2, assimilatory sulfite reductase (NADPH). ',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.98.1': {'ec_num': '1.8.98.1',\n",
       "  'accepted_name': 'dihydromethanophenazine:CoB-CoM heterodisulfide reductase',\n",
       "  'reaction': 'CoB + CoM + methanophenazine = CoM-S-S-CoB + dihydromethanophenazine',\n",
       "  'other_names': 'hdrDE (gene names); CoB—CoM heterodisulfide reductase (ambiguous); heterodisulfide reductase (ambiguous); coenzyme B:coenzyme M:methanophenazine oxidoreductase',\n",
       "  'sys_name': 'CoB:CoM:methanophenazine oxidoreductase',\n",
       "  'comments': 'This enzyme, found in methanogenic archaea that belong to the Methanosarcinales order, regenerates CoM and CoB after the action of EC 2.8.4.1, coenzyme-B sulfoethylthiotransferase. It is a membrane-bound enzyme that contains (per heterodimeric unit) two distinct b-type hemes and two [4Fe-4S] clusters. cf. EC 1.8.7.3, ferredoxin:CoB-CoM heterodisulfide reductase, EC 1.8.98.5, H2:CoB-CoM heterodisulfide,ferredoxin reductase, EC 1.8.98.6, formate:CoB-CoM heterodisulfide,ferredoxin reductase and EC 1.8.98.4, coenzyme F420:CoB-CoM heterodisulfide,ferredoxin reductase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'CoB = coenzyme B = N-(7-sulfanylheptanoyl)threonine = N-(7-mercaptoheptanoyl)threonine 3-O-phosphate (deprecated)\\nCoM = coenzyme M = 2-sulfanylethane-1-sulfonate = 2-mercaptoethanesulfonate (deprecated)\\nmethanophenazine = 2-{[(6E,10E,14E)-3,7,11,15,19-pentamethylicosa-6,10,14,18-tetraen-1-yl]oxy}phenazine\\nCoM-S-S-CoB = CoB-CoM heterodisulfide = N-{7-[(2-sulfoethyl)dithio]heptanoyl}-O3-phospho-L-threonine = O3-phospho-N-{7-[2-(2-sulfoethyl)disulfan-1-yl]heptanoyl}-L-threonine',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.98.2': {'ec_num': '1.8.98.2',\n",
       "  'accepted_name': 'sulfiredoxin',\n",
       "  'reaction': 'peroxiredoxin-(S-hydroxy-S-oxocysteine) + ATP + 2 R-SH = peroxiredoxin-(S-hydroxycysteine) + ADP + phosphate + R-S-S-R',\n",
       "  'other_names': 'Srx1; sulphiredoxin; peroxiredoxin-(S-hydroxy-S-oxocysteine) reductase',\n",
       "  'sys_name': 'peroxiredoxin-(S-hydroxy-S-oxocysteine):thiol oxidoreductase [ATP-hydrolysing; peroxiredoxin-(S-hydroxycysteine)-forming]',\n",
       "  'comments': 'In the course of the reaction of EC 1.11.1.15, peroxiredoxin, its cysteine residue is alternately oxidized to the sulfenic acid, S-hydroxycysteine, and reduced back to cysteine. Occasionally the S-hydroxycysteine residue is further oxidized to the sulfinic acid S-hydroxy-S-oxocysteine, thereby inactivating the enzyme. The reductase provides a mechanism for regenerating the active form of peroxiredoxin, i.e. the peroxiredoxin-(S-hydroxycysteine) form. Apparently the reductase first catalyses the phosphorylation of the -S(O)-OH group by ATP to give -S(O)-O-P, which is attached to the peroxiredoxin by a cysteine residue, forming an -S(O)-S- link between the two enzymes. Attack by a thiol splits this bond, leaving the peroxiredoxin as the sulfenic acid and the reductase as the thiol.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.99.1': {'ec_num': '1.8.99.1',\n",
       "  'accepted_name': 'assimilatory sulfite reductase',\n",
       "  'reaction': 'hydrogen sulfide + acceptor + 3 H2O = sulfite + reduced acceptor',\n",
       "  'other_names': 'sulfite reductase (ambiguous); assimilatory-type sulfite reductase',\n",
       "  'sys_name': 'hydrogen-sulfide:acceptor oxidoreductase',\n",
       "  'comments': 'An iron-protein. A stoichiometry of six molecules of reduced methyl viologen per molecule of sulfide formed was found.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.99.2': {'ec_num': '1.8.99.2',\n",
       "  'accepted_name': 'adenylyl-sulfate reductase',\n",
       "  'reaction': 'AMP + sulfite + acceptor = adenylyl sulfate + reduced acceptor',\n",
       "  'other_names': 'adenosine phosphosulfate reductase; adenosine 5′-phosphosulfate reductase; APS-reductase; APS reductase; AMP, sulfite:(acceptor) oxidoreductase (adenosine-5′-phosphosulfate-forming)',\n",
       "  'sys_name': 'AMP,sulfite:acceptor oxidoreductase (adenosine-5′-phosphosulfate-forming)',\n",
       "  'comments': 'An iron flavoprotein (FAD). Methyl viologen can act as acceptor.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.8.99.3': {'ec_num': '1.8.99.3',\n",
       "  'accepted_name': 'hydrogensulfite reductase',\n",
       "  'reaction': 'S(SO3-)2 + acceptor + 2 H2O + OH- = 3 HSO3- + reduced acceptor',\n",
       "  'other_names': 'bisulfite reductase; dissimilatory sulfite reductase; desulfoviridin; desulforubidin; desulfofuscidin; dissimilatory-type sulfite reductase; trithionate:(acceptor) oxidoreductase',\n",
       "  'sys_name': 'trithionate:acceptor oxidoreductase',\n",
       "  'comments': 'Methylviologen can act as acceptor. A group of sirohemoproteins containing iron-sulfur centres (P-582).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.8.99.3\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.8.99.3\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.8.99.3\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.8.99.3\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.8.99.3%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': 'HSO3- = bisulfite\\nS(SO3-)2 = trithionate',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.9.3.1': {'ec_num': '1.9.3.1',\n",
       "  'accepted_name': 'cytochrome-c oxidase',\n",
       "  'reaction': '4 ferrocytochrome c + O2 + 4 H+ = 4 ferricytochrome c + 2 H2O',\n",
       "  'other_names': \"cytochrome aa3; cytochrome caa3; cytochrome bb3; cytochrome cbb3; cytochrome ba3; cytochrome a3; Warburg's respiratory enzyme; indophenol oxidase; indophenolase; complex IV (mitochondrial electron transport); ferrocytochrome c oxidase; cytochrome oxidase (ambiguous); NADH cytochrome c oxidase (incorrect)\",\n",
       "  'sys_name': 'ferrocytochrome-c:oxygen oxidoreductase',\n",
       "  'comments': 'An oligomeric membrane heme-Cu: O2 reductase-type enzyme that terminates the respiratory chains of aerobic and facultative aerobic organisms. The reduction of O2 to water is accompanied by the extrusion of four protons. The aa3-type enzymes of mitochondria and many bacterial species are the most abundant group, but other variations, such as the bacterial cbb3 enzymes, also exist. All of the variants have a conserved catalytic core subunit (subunit I) that contains a low spin heme (of a- or b-type), a binuclear metal center composed of a high spin heme (of a-, o-, or b-type heme, referred to as a3, o3 or b3)-iron, and a Cu atom (CuB). Besides subunit I, the enzyme usually has at least two other core subunits: subunit II is the primary electron acceptor; subunit III usually does not contain any cofactors, but in the case of cbb3-type enzymes it is a diheme c-type cytochrome. While most bacterial enzymes consist of only 3-4 subunits, the mitochondrial enzyme is much more complex and contains 14 subunits.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-12 17:58:49'},\n",
       " '1.9.6.1': {'ec_num': '1.9.6.1',\n",
       "  'accepted_name': 'nitrate reductase (cytochrome)',\n",
       "  'reaction': '2 ferrocytochrome + 2 H+ + nitrate = 2 ferricytochrome + nitrite',\n",
       "  'other_names': 'respiratory nitrate reductase; benzyl viologen-nitrate reductase',\n",
       "  'sys_name': 'ferrocytochrome:nitrate oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.9.6.1\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.9.6.1\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.9.6.1\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.9.6.1\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.9.6.1%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.9.99.1': {'ec_num': '1.9.99.1',\n",
       "  'accepted_name': 'iron—cytochrome-c reductase',\n",
       "  'reaction': 'ferrocytochrome c + Fe3+ = ferricytochrome c + Fe2+',\n",
       "  'other_names': 'iron-cytochrome c reductase',\n",
       "  'sys_name': 'ferrocytochrome-c:Fe3+ oxidoreductase',\n",
       "  'comments': 'An iron protein.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.10.1.1': {'ec_num': '1.10.1.1',\n",
       "  'accepted_name': 'trans-acenaphthene-1,2-diol dehydrogenase',\n",
       "  'reaction': '(±)-trans-acenaphthene-1,2-diol + 2 NADP+ = acenaphthenequinone + 2 NADPH + 2 H+',\n",
       "  'other_names': 'trans-1,2-acenaphthenediol dehydrogenase',\n",
       "  'sys_name': '(±)-trans-acenaphthene-1,2-diol:NADP+ oxidoreductase',\n",
       "  'comments': 'Some preparations also utilize NAD+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.10.2.1': {'ec_num': '1.10.2.1',\n",
       "  'accepted_name': 'L-ascorbate—cytochrome-b5 reductase',\n",
       "  'reaction': 'L-ascorbate + ferricytochrome b5 = monodehydroascorbate + ferrocytochrome b5 + H+',\n",
       "  'other_names': 'ascorbate-cytochrome b5 reductase',\n",
       "  'sys_name': 'L-ascorbate:ferricytochrome-b5 oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.10.2.2': {'ec_num': '1.10.2.2',\n",
       "  'accepted_name': 'quinol—cytochrome-c reductase',\n",
       "  'reaction': 'quinol + 2 ferricytochrome c = quinone + 2 ferrocytochrome c + 2 H+',\n",
       "  'other_names': 'ubiquinol—cytochrome-c reductase; coenzyme Q-cytochrome c reductase; dihydrocoenzyme Q-cytochrome c reductase; reduced ubiquinone-cytochrome c reductase; complex III (mitochondrial electron transport); ubiquinone-cytochrome c reductase; ubiquinol-cytochrome c oxidoreductase; reduced coenzyme Q-cytochrome c reductase; ubiquinone-cytochrome c oxidoreductase; reduced ubiquinone-cytochrome c oxidoreductase; mitochondrial electron transport complex III; ubiquinol-cytochrome c-2 oxidoreductase; ubiquinone-cytochrome b-c1 oxidoreductase; ubiquinol-cytochrome c2 reductase; ubiquinol-cytochrome c1 oxidoreductase; CoQH2-cytochrome c oxidoreductase; ubihydroquinol:cytochrome c oxidoreductase; coenzyme QH2-cytochrome c reductase; QH2:cytochrome c oxidoreductase; ubiquinol:ferricytochrome-c oxidoreductase',\n",
       "  'sys_name': 'quinol:ferricytochrome-c oxidoreductase',\n",
       "  'comments': 'The enzyme, often referred to as the cytochrome bc1 complex or complex III, is the third complex in the electron transport chain. It is present in the mitochondria of all aerobic eukaryotes and in the inner membranes of most bacteria. The mammalian enzyme contains cytochromes b-562, b-566 and c1, and a 2-iron ferredoxin. Depending on the organism and physiological conditions, the enzyme extrudes either two or four protons from the cytoplasmic to the non-cytoplasmic compartment (cf. EC 1.6.99.3, NADH dehydrogenase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.10.3.1': {'ec_num': '1.10.3.1',\n",
       "  'accepted_name': 'catechol oxidase',\n",
       "  'reaction': '2 catechol + O2 = 2 1,2-benzoquinone + 2 H2O',\n",
       "  'other_names': 'diphenol oxidase; o-diphenolase; polyphenol oxidase; pyrocatechol oxidase; dopa oxidase; catecholase; o-diphenol:oxygen oxidoreductase; o-diphenol oxidoreductase',\n",
       "  'sys_name': '1,2-benzenediol:oxygen oxidoreductase',\n",
       "  'comments': 'A type 3 copper protein that catalyses exclusively the oxidation of catechol (i.e., o-diphenol) to the corresponding o-quinone. The enzyme also acts on a variety of substituted catechols.\\nIt is different from tyrosinase, EC 1.14.18.1, which can catalyse both the monooxygenation of monophenols and the oxidation of catechols.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'catechol = 1,2-benzenediol',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.10.3.2': {'ec_num': '1.10.3.2',\n",
       "  'accepted_name': 'laccase',\n",
       "  'reaction': '4 benzenediol + O2 = 4 benzosemiquinone + 2 H2O',\n",
       "  'other_names': 'urishiol oxidase; urushiol oxidase; p-diphenol oxidase',\n",
       "  'sys_name': 'benzenediol:oxygen oxidoreductase',\n",
       "  'comments': 'A group of multi-copper proteins of low specificity acting on both o- and p-quinols, and often acting also on aminophenols and phenylenediamine. The semiquinone may react further either enzymically or non-enzymically.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.10.3.2\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.10.3.2\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.10.3.2\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.10.3.2\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.10.3.2%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.10.3.3': {'ec_num': '1.10.3.3',\n",
       "  'accepted_name': 'L-ascorbate oxidase',\n",
       "  'reaction': '4 L-ascorbate + O2 = 4 monodehydroascorbate + 2 H2O',\n",
       "  'other_names': 'ascorbase; ascorbic acid oxidase; ascorbate oxidase; ascorbic oxidase; ascorbate dehydrogenase; L-ascorbic acid oxidase; AAO; L-ascorbate:O2 oxidoreductase; AA oxidase',\n",
       "  'sys_name': 'L-ascorbate:oxygen oxidoreductase',\n",
       "  'comments': 'A multicopper protein.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.10.3.4': {'ec_num': '1.10.3.4',\n",
       "  'accepted_name': 'o-aminophenol oxidase',\n",
       "  'reaction': '4 2-aminophenol + 3 O2 = 2 2-aminophenoxazin-3-one + 6 H2O',\n",
       "  'other_names': 'isophenoxazine synthase; o-aminophenol:O2 oxidoreductase; 2-aminophenol:O2 oxidoreductase',\n",
       "  'sys_name': '2-aminophenol:oxygen oxidoreductase',\n",
       "  'comments': 'A flavoprotein which catalyses a 6-electron oxidation. The enzyme from the plant Tecoma stans requires Mn2+ and FAD [1] whereas the fungus Pycnoporus coccineus requires Mn2+ and riboflavin 5′-phosphate [2], the bacteria Streptomyces antibioticus requires Cu2+ [4] and the plant Bauhenia monandra does not require any co-factors [3].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '2-aminophenoxazin-3-one = isophenoxazine',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.10.3.5': {'ec_num': '1.10.3.5',\n",
       "  'accepted_name': '3-hydroxyanthranilate oxidase',\n",
       "  'reaction': '3-hydroxyanthranilate + O2 = 6-imino-5-oxocyclohexa-1,3-dienecarboxylate + H2O2',\n",
       "  'other_names': '3-hydroxyanthranilic acid oxidase',\n",
       "  'sys_name': '3-hydroxyanthranilate:oxygen oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.10.3.6': {'ec_num': '1.10.3.6',\n",
       "  'accepted_name': 'rifamycin-B oxidase',\n",
       "  'reaction': 'rifamycin B + O2 = rifamycin O + H2O2',\n",
       "  'other_names': 'rifamycin B oxidase',\n",
       "  'sys_name': 'rifamycin-B:oxygen oxidoreductase',\n",
       "  'comments': 'Acts also on benzene-1,4-diol and, more slowly, on some other p-quinols. Not identical with EC 1.10.3.1 (catechol oxidase), EC 1.10.3.2 (laccase), EC 1.10.3.4 (o-aminophenol oxidase) or EC 1.10.3.5 (3-hydroxyanthranilate oxidase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.10.99.1': {'ec_num': '1.10.99.1',\n",
       "  'accepted_name': 'plastoquinol—plastocyanin reductase',\n",
       "  'reaction': 'plastoquinol-1 + 2 oxidized plastocyanin = plastoquinone + 2 reduced plastocyanin',\n",
       "  'other_names': 'plastoquinol/plastocyanin oxidoreductase; cytochrome b6/f complex; cytochrome b6/ complex',\n",
       "  'sys_name': 'plastoquinol:oxidized-plastocyanin oxidoreductase',\n",
       "  'comments': 'A cytochrome f,b6 complex separated from chloroplasts. Also acts, more slowly, on plastoquinol-9 and ubiquinols. Cytochrome c-552 can act instead of plastocyanin, but more slowly.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.10.99.2': {'ec_num': '1.10.99.2',\n",
       "  'accepted_name': 'ribosyldihydronicotinamide dehydrogenase (quinone)',\n",
       "  'reaction': '1-(β-D-ribofuranosyl)-1,4-dihydronicotinamide + a quinone = 1-(β-D-ribofuranosyl)nicotinamide + a hydroquinone',\n",
       "  'other_names': 'NRH:quinone oxidoreductase 2; NQO2; NAD(P)H:quinone oxidoreductase-2 (misleading); QR2; quinone reductase 2; N-ribosyldihydronicotinamide dehydrogenase (quinone); NAD(P)H:quinone oxidoreductase2 (misleading)',\n",
       "  'sys_name': '1-(β-D-ribofuranosyl)-1,4-dihydronicotinamide:quinone oxidoreductase',\n",
       "  'comments': 'A flavoprotein. Unlike EC 1.6.5.2, NAD(P)H dehydrogenase (quinone), this quinone reductase cannot use NADH or NADPH; instead it uses N-ribosyl- and N-alkyldihydronicotinamides. Polycyclic aromatic hydrocarbons, such as benz[a]anthracene, and the oestrogens 17β-estradiol and diethylstilbestrol are potent inhibitors, but dicoumarol is only a very weak inhibitor [2]. This enzyme can catalyse both 2-electron and 4-electron reductions, but one-electron acceptors, such as potassium ferricyanide, cannot be reduced [3].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.10.99.3': {'ec_num': '1.10.99.3',\n",
       "  'accepted_name': 'violaxanthin de-epoxidase',\n",
       "  'reaction': 'violaxanthin + 2 L-ascorbate = zeaxanthin + 2 L-dehydroascorbate + 2 H2O (overall reaction);;(1a) violaxanthin + L-ascorbate = antheraxanthin + L-dehydroascorbate + H2O;;(1b) antheraxanthin + L-ascorbate = zeaxanthin + L-dehydroascorbate + H2O',\n",
       "  'other_names': 'VDE',\n",
       "  'sys_name': 'violaxanthin:ascorbate oxidoreductase',\n",
       "  'comments': 'Along with EC 1.14.13.90, zeaxanthin epoxidase, this enzyme forms part of the xanthophyll (or violaxanthin) cycle for controlling the concentration of zeaxanthin in chloroplasts. It is activated by a low pH of the thylakoid lumen (produced by high light intensity). Zeaxanthin induces the dissipation of excitation energy in the chlorophyll of the light-harvesting protein complex of photosystem II. In higher plants the enzyme reacts with all-trans-diepoxides, such as violaxanthin, and all-trans-monoepoxides, but in the alga Mantoniella squamata, only the diepoxides are good substrates.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'violaxanthin = (3S,3′S,5R,5′R,6S,6′S)-5,5′,6,6′-tetrahydro-5,6:5′,6′-diepoxy-β,β-carotene-3,3′-diol\\nantheraxanthin = (3R,3′S,5′R,6′S)-5′,6′-dihydro-5′,6′-epoxy-β,β-carotene-3,3′-diol\\nzeaxanthin = (3R,3′R)-β,β-carotene-3,3′-diol',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.11.1.1': {'ec_num': '1.11.1.1',\n",
       "  'accepted_name': 'NADH peroxidase',\n",
       "  'reaction': 'NADH + H+ + H2O2 = NAD+ + 2 H2O',\n",
       "  'other_names': 'DPNH peroxidase; NAD peroxidase; diphosphopyridine nucleotide peroxidase; NADH-peroxidase; nicotinamide adenine dinucleotide peroxidase; NADH2 peroxidase',\n",
       "  'sys_name': 'NADH:hydrogen-peroxide oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD). Ferricyanide, quinones, etc., can replace H2O2.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.11.1.2': {'ec_num': '1.11.1.2',\n",
       "  'accepted_name': 'NADPH peroxidase',\n",
       "  'reaction': 'NADPH + H+ + H2O2 = NADP+ + 2 H2O',\n",
       "  'other_names': 'TPNH peroxidase; NADP peroxidase; nicotinamide adenine dinucleotide phosphate peroxidase; TPN peroxidase; triphosphopyridine nucleotide peroxidase; NADPH2 peroxidase',\n",
       "  'sys_name': 'NADPH:hydrogen-peroxide oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.11.1.3': {'ec_num': '1.11.1.3',\n",
       "  'accepted_name': 'fatty-acid peroxidase',\n",
       "  'reaction': 'palmitate + 2 H2O2 = pentadecanal + CO2 + 3 H2O',\n",
       "  'other_names': 'long chain fatty acid peroxidase',\n",
       "  'sys_name': 'hexadecanoate:hydrogen-peroxide oxidoreductase',\n",
       "  'comments': 'Acts on long-chain fatty acids from dodecanoic to octadecanoic acid.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.11.1.5': {'ec_num': '1.11.1.5',\n",
       "  'accepted_name': 'cytochrome-c peroxidase',\n",
       "  'reaction': '2 ferrocytochrome c + H2O2 = 2 ferricytochrome c + 2 H2O',\n",
       "  'other_names': 'cytochrome peroxidase; cytochrome c-551 peroxidase; apocytochrome c peroxidase; mesocytochrome c peroxidase azide; mesocytochrome c peroxidase cyanide; mesocytochrome c peroxidase cyanate; cytochrome c-H2O oxidoreductase; cytochrome c peroxidase',\n",
       "  'sys_name': 'ferrocytochrome-c:hydrogen-peroxide oxidoreductase',\n",
       "  'comments': 'A hemoprotein.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.11.1.6': {'ec_num': '1.11.1.6',\n",
       "  'accepted_name': 'catalase',\n",
       "  'reaction': '2 H2O2 = O2 + 2 H2O',\n",
       "  'other_names': 'equilase; caperase; optidase; catalase-peroxidase; CAT',\n",
       "  'sys_name': 'hydrogen-peroxide:hydrogen-peroxide oxidoreductase',\n",
       "  'comments': 'A hemoprotein. A manganese protein containing MnIII in the resting state, which also belongs here, is often called pseudocatalase. The enzymes from some organisms, such as Penicillium simplicissimum, can also act as a peroxidase (EC 1.11.1.7) for which several organic substances, especially ethanol, can act as a hydrogen donor. Enzymes that exhibit both catalase and peroxidase activity belong under EC 1.11.1.21, catalase-peroxidase.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.11.1.6\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.11.1.6\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.11.1.6\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.11.1.6\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.11.1.6%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.11.1.7': {'ec_num': '1.11.1.7',\n",
       "  'accepted_name': 'peroxidase',\n",
       "  'reaction': '2 phenolic donor + H2O2 = 2 phenoxyl radical of the donor + 2 H2O',\n",
       "  'other_names': 'lactoperoxidase; guaiacol peroxidase; plant peroxidase; Japanese radish peroxidase; horseradish peroxidase (HRP); soybean peroxidase (SBP); extensin peroxidase; heme peroxidase; oxyperoxidase; protoheme peroxidase; pyrocatechol peroxidase; scopoletin peroxidase; Coprinus cinereus peroxidase; Arthromyces ramosus peroxidase',\n",
       "  'sys_name': 'phenolic donor:hydrogen-peroxide oxidoreductase',\n",
       "  'comments': 'Heme proteins with histidine as proximal ligand. The iron in the resting enzyme is Fe(III). They also peroxidize non-phenolic substrates such as 3,3′,5,5′-tetramethylbenzidine (TMB) and 2,2′-azinobis(3-ethylbenzthiazoline-6-sulfonic acid) (ABTS). Certain peroxidases (e.g. lactoperoxidase, SBP) oxidize bromide, iodide and thiocyanate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.11.1.8': {'ec_num': '1.11.1.8',\n",
       "  'accepted_name': 'iodide peroxidase',\n",
       "  'reaction': '(1) 2 iodide + H2O2 + 2 H+ = diiodine + 2 H2O;;(2) [thyroglobulin]-L-tyrosine + iodide + H2O2 = [thyroglobulin]-3-iodo-L-tyrosine + 2 H2O;;(3) [thyroglobulin]-3-iodo-L-tyrosine + iodide + H2O2 = [thyroglobulin]-3,5-diiodo-L-tyrosine + 2 H2O;;(4) 2 [thyroglobulin]-3,5-diiodo-L-tyrosine + H2O2 = [thyroglobulin]-L-thyroxine + [thyroglobulin]-aminoacrylate + 2 H2O;;(5) [thyroglobulin]-3-iodo-L-tyrosine + [thyroglobulin]-3,5-diiodo-L-tyrosine + H2O2 = [thyroglobulin]-3,5,3′-triiodo-L-thyronine + [thyroglobulin]-aminoacrylate + 2 H2O',\n",
       "  'other_names': 'thyroid peroxidase; iodoperoxidase (heme type); iodide peroxidase-tyrosine iodinase; thyroperoxidase; tyrosine iodinase; TPO; iodinase',\n",
       "  'sys_name': 'iodide:hydrogen-peroxide oxidoreductase',\n",
       "  'comments': 'Thyroid peroxidase catalyses the biosynthesis of the thyroid hormones L-thyroxine and triiodo-L-thyronine. It catalyses both the iodination of tyrosine residues in thyroglobulin (forming mono- and di-iodinated forms) and their coupling to form either L-thyroxine or triiodo-L-thyronine.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '3,5,3′-triiodo-L-thyronine = triiodo-L-thyronine',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.11.1.9': {'ec_num': '1.11.1.9',\n",
       "  'accepted_name': 'glutathione peroxidase',\n",
       "  'reaction': '2 glutathione + H2O2 = glutathione disulfide + 2 H2O',\n",
       "  'other_names': 'GSH peroxidase; selenium-glutathione peroxidase; reduced glutathione peroxidase',\n",
       "  'sys_name': 'glutathione:hydrogen-peroxide oxidoreductase',\n",
       "  'comments': 'A protein containing a selenocysteine residue. Steroid and lipid hydroperoxides, but not the product of reaction of EC 1.13.11.12 lipoxygenase on phospholipids, can act as acceptor, but more slowly than H2O2 (cf. EC 1.11.1.12 phospholipid-hydroperoxide glutathione peroxidase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.11.1.10': {'ec_num': '1.11.1.10',\n",
       "  'accepted_name': 'chloride peroxidase',\n",
       "  'reaction': 'RH + chloride + H2O2 = RCl + 2 H2O',\n",
       "  'other_names': 'chloroperoxidase; CPO; vanadium haloperoxidase',\n",
       "  'sys_name': 'chloride:hydrogen-peroxide oxidoreductase',\n",
       "  'comments': 'Brings about the chlorination of a range of organic molecules, forming stable C-Cl bonds. Also oxidizes bromide and iodide. Enzymes of this type are either heme-thiolate proteins, or contain vanadate. A secreted enzyme produced by the ascomycetous fungus Caldariomyces fumago (Leptoxyphium fumago) is an example of the heme-thiolate type. It catalyses the production of hypochlorous acid by transferring one oxygen atom from H2O2 to chloride. At a separate site it catalyses the chlorination of activated aliphatic and aromatic substrates, via HClO and derived chlorine species. In the absence of halides, it shows peroxidase (e.g. phenol oxidation) and peroxygenase activities. The latter inserts oxygen from H2O2 into, for example, styrene (side chain epoxidation) and toluene (benzylic hydroxylation), however, these activities are less pronounced than its activity with halides. Has little activity with non-activated substrates such as aromatic rings, ethers or saturated alkanes. The chlorinating peroxidase produced by ascomycetous fungi (e.g. Curvularia inaequalis) is an example of a vanadium chloroperoxidase, and is related to bromide peroxidase (EC 1.11.1.18). It contains vanadate and oxidizes chloride, bromide and iodide into hypohalous acids. In the absence of halides, it peroxygenates organic sulfides and oxidizes ABTS [2,2′-azinobis(3-ethylbenzthiazoline-6-sulfonic acid)] but no phenols.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.11.1.11': {'ec_num': '1.11.1.11',\n",
       "  'accepted_name': 'L-ascorbate peroxidase',\n",
       "  'reaction': '2 L-ascorbate + H2O2 + 2 H+ = L-ascorbate + L-dehydroascorbate + 2 H2O (overall reaction);;(1a) 2 L-ascorbate + H2O2 + 2 H+ = 2 monodehydroascorbate + 2 H2O;;(1b) 2 monodehydroascorbate = L-ascorbate + L-dehydroascorbate (spontaneous) ',\n",
       "  'other_names': 'L-ascorbic acid peroxidase; L-ascorbic acid-specific peroxidase; ascorbate peroxidase; ascorbic acid peroxidase',\n",
       "  'sys_name': 'L-ascorbate:hydrogen-peroxide oxidoreductase',\n",
       "  'comments': 'A heme protein. Oxidizes ascorbate and low molecular weight aromatic substrates. The monodehydroascorbate radical produced is either directly reduced back to ascorbate by EC 1.6.5.4 [monodehydroascorbate reductase (NADH)] or undergoes non-enzymic disproportionation to ascorbate and dehydroascorbate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'monodehydroascorbate = ascorbate radical',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.11.1.12': {'ec_num': '1.11.1.12',\n",
       "  'accepted_name': 'phospholipid-hydroperoxide glutathione peroxidase',\n",
       "  'reaction': '2 glutathione + a hydroperoxy-fatty-acyl-[lipid] = glutathione disulfide + a hydroxy-fatty-acyl-[lipid] + H2O ',\n",
       "  'other_names': 'peroxidation-inhibiting protein; PHGPX; peroxidation-inhibiting protein:peroxidase,glutathione (phospholipid hydroperoxide-reducing); phospholipid hydroperoxide glutathione peroxidase; hydroperoxide glutathione peroxidase',\n",
       "  'sys_name': 'glutathione:lipid-hydroperoxide oxidoreductase',\n",
       "  'comments': 'A protein containing a selenocysteine residue. The products of action of EC 1.13.11.12 lipoxygenase on phospholipids can act as acceptors; H2O2 can also act, but much more slowly (cf. EC 1.11.1.9 glutathione peroxidase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.11.1.13': {'ec_num': '1.11.1.13',\n",
       "  'accepted_name': 'manganese peroxidase',\n",
       "  'reaction': '2 Mn(II) + 2 H+ + H2O2 = 2 Mn(III) + 2 H2O',\n",
       "  'other_names': 'peroxidase-M2; Mn-dependent (NADH-oxidizing) peroxidase',\n",
       "  'sys_name': 'Mn(II):hydrogen-peroxide oxidoreductase',\n",
       "  'comments': 'A hemoprotein. The enzyme from white rot basidiomycetes is involved in the oxidative degradation of lignin. The enzyme oxidizes a bound Mn2+ ion to Mn3+ in the presence of hydrogen peroxide. The product, Mn3+, is released from the active site in the presence of a chelator (mostly oxalate and malate) that stabilizes it against disproportionation to Mn2+ and insoluble Mn4+ [4]. The complexed Mn3+ ion can diffuse into the lignified cell wall, where it oxidizes phenolic components of lignin and other organic substrates [1]. It is inactive with veratryl alcohol or nonphenolic substrates.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.11.1.13\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.11.1.13\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.11.1.13\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.11.1.13\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.11.1.13%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.11.1.14': {'ec_num': '1.11.1.14',\n",
       "  'accepted_name': 'lignin peroxidase',\n",
       "  'reaction': '(1) 1-(3,4-dimethoxyphenyl)-2-(2-methoxyphenoxy)propane-1,3-diol + H2O2 = 3,4-dimethoxybenzaldehyde + 2-methoxyphenol + glycolaldehyde + H2O;;(2) 2 (3,4-dimethoxyphenyl)methanol + H2O2 = 2 (3,4-dimethoxyphenyl)methanol radical + 2 H2O',\n",
       "  'other_names': 'diarylpropane oxygenase; ligninase I; diarylpropane peroxidase; LiP; diarylpropane:oxygen,hydrogen-peroxide oxidoreductase (C-C-bond-cleaving); 1,2-bis(3,4-dimethoxyphenyl)propane-1,3-diol:hydrogen-peroxide oxidoreductase (incorrect); (3,4-dimethoxyphenyl)methanol:hydrogen-peroxide oxidoreductase',\n",
       "  'sys_name': '1-(3,4-dimethoxyphenyl)-2-(2-methoxyphenoxy)propane-1,3-diol:hydrogen-peroxide oxidoreductase',\n",
       "  'comments': 'A hemoprotein, involved in the oxidative breakdown of lignin by white-rot basidiomycete fungi. The reaction involves an initial oxidation of the heme iron by hydrogen peroxide, forming compound I (FeIV=O radical cation) at the active site. A single one-electron reduction of compound I by an electron derived from a substrate molecule yields compound II (FeIV=O non-radical cation), followed by a second one-electron transfer that returns the enzyme to the ferric oxidation state. The electron transfer events convert the substrate molecule into a transient cation radical intermediate that fragments spontaneously. The enzyme can act on a wide range of aromatic compounds, including methoxybenzenes and nonphenolic β-O-4 linked arylglycerol β-aryl ethers, but cannot act directly on the lignin molecule, which is too large to fit into the active site. However larger lignin molecules can be degraded in the presence of veratryl alcohol. It has been suggested that the free radical that is formed when the enzyme acts on veratryl alcohol can diffuse into the lignified cell wall, where it oxidizes lignin and other organic substrates. In the presence of high concentration of hydrogen peroxide and lack of substrate, the enzyme forms a catalytically inactive form (compound III). This form can be rescued by interaction with two molecules of the free radical products. In the case of veratryl alcohol, such an interaction yields two molecules of veratryl aldehyde.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.11.1.14\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.11.1.14\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.11.1.14\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.11.1.14\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.11.1.14%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': 'veratryl alcohol = (3,4-dimethoxyphenyl)methanol\\nveratraldehyde = 3,4-dimethoxybenzaldehyde\\n2-methoxyphenol = guaiacol',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.11.1.15': {'ec_num': '1.11.1.15',\n",
       "  'accepted_name': 'peroxiredoxin',\n",
       "  'reaction': '2 R′-SH + ROOH = R′-S-S-R′ + H2O + ROH',\n",
       "  'other_names': 'thioredoxin peroxidase; tryparedoxin peroxidase; alkyl hydroperoxide reductase C22; AhpC; TrxPx; TXNPx; Prx; PRDX',\n",
       "  'sys_name': 'thiol-containing-reductant:hydroperoxide oxidoreductase',\n",
       "  'comments': 'Peroxiredoxins (Prxs) are a ubiquitous family of antioxidant proteins. They can be divided into three classes: typical 2-Cys, atypical 2-Cys and 1-Cys peroxiredoxins [1]. The peroxidase reaction comprises two steps centred around a redox-active cysteine called the peroxidatic cysteine. All three peroxiredoxin classes have the first step in common, in which the peroxidatic cysteine attacks the peroxide substrate and is oxidized to S-hydroxycysteine (a sulfenic acid) (see mechanism). The second step of the peroxidase reaction, the regeneration of cysteine from S-hydroxycysteine, distinguishes the three peroxiredoxin classes. For typical 2-Cys Prxs, in the second step, the peroxidatic S-hydroxycysteine from one subunit is attacked by the ‘resolving’ cysteine located in the C-terminus of the second subunit, to form an intersubunit disulfide bond, which is then reduced by one of several cell-specific thiol-containing reductants (R′-SH) (e.g. thioredoxin, AhpF, tryparedoxin or AhpD), completing the catalytic cycle. In the atypical 2-Cys Prxs, both the peroxidatic cysteine and its resolving cysteine are in the same polypeptide, so their reaction forms an intrachain disulfide bond [1]. To recycle the disulfide, known atypical 2-Cys Prxs appear to use thioredoxin as an electron donor [3]. The 1-Cys Prxs conserve only the peroxidatic cysteine, so that its oxidized form is directly reduced to cysteine by the reductant molecule [4].',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.11.1.15\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.11.1.15\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.11.1.15\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.11.1.15\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.11.1.15%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.11.1.24': {'ec_num': '1.11.1.24',\n",
       "  'accepted_name': 'thioredoxin-dependent peroxiredoxin',\n",
       "  'reaction': 'thioredoxin + ROOH = thioredoxin disulfide + H2O + ROH',\n",
       "  'other_names': 'thioredoxin peroxidase; bcp (gene name); tpx (gene name); PrxQ',\n",
       "  'sys_name': 'thioredoxin:hydroperoxide oxidoreductase',\n",
       "  'comments': 'Peroxiredoxins (Prxs) are a ubiquitous family of antioxidant proteins. They can be divided into three classes: typical 2-Cys, atypical 2-Cys and 1-Cys peroxiredoxins [4]. The peroxidase reaction comprises two steps centred around a redox-active cysteine called the peroxidatic cysteine. All three peroxiredoxin classes have the first step in common, in which the peroxidatic cysteine attacks the peroxide substrate and is oxidized to S-hydroxycysteine (a sulfenic acid) (see mechanism). The second step of the peroxidase reaction, the regeneration of cysteine from S-hydroxycysteine, distinguishes the three peroxiredoxin classes. For typical 2-Cys Prxs, in the second step, the peroxidatic S-hydroxycysteine from one subunit is attacked by the ‘resolving’ cysteine located in the C-terminus of the second subunit, to form an intersubunit disulfide bond, which is then reduced by one of several cell-specific thiol-containing reductants completing the catalytic cycle. In the atypical 2-Cys Prxs, both the peroxidatic cysteine and its resolving cysteine are in the same polypeptide, so their reaction forms an intrachain disulfide bond. The 1-Cys Prxs conserve only the peroxidatic cysteine, so its regeneration involves direct interaction with a reductant molecule. Thioredoxin-dependent peroxiredoxins are the most common. They have been reported from archaea, bacteria, fungi, plants, and animals.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.11.1.24\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.11.1.24\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.11.1.24\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.11.1.24\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.11.1.24%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.12.1.2': {'ec_num': '1.12.1.2',\n",
       "  'accepted_name': 'hydrogen dehydrogenase',\n",
       "  'reaction': 'H2 + NAD+ = H+ + NADH',\n",
       "  'other_names': 'H2:NAD+ oxidoreductase; NAD-linked hydrogenase; bidirectional hydrogenase; hydrogenase',\n",
       "  'sys_name': 'hydrogen:NAD+ oxidoreductase',\n",
       "  'comments': 'An iron-sulfur flavoprotein (FMN or FAD). Some forms of this enzyme contain nickel.',\n",
       "  'links': 'BRENDA, EXPASY, GTD, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.12.1.3': {'ec_num': '1.12.1.3',\n",
       "  'accepted_name': 'hydrogen dehydrogenase (NADP+)',\n",
       "  'reaction': 'H2 + NADP+ = H+ + NADPH',\n",
       "  'other_names': 'NADP+-linked hydrogenase; NADP+-reducing hydrogenase; hydrogenase (ambiguous); hydrogenase I (ambiguous)',\n",
       "  'sys_name': 'hydrogen:NADP+ oxidoreductase',\n",
       "  'comments': 'The protein from the bacterium Desulfovibrio fructosovorans is an iron-sulfur protein that exclusively functions as a hydrogen dehydrogenase [1], while the enzyme from the archaeon Pyrococcus furiosus is a nickel, iron, iron-sulfur protein, that is part of a heterotetrameric complex where the α and δ subunits function as a hydrogenase while the β and γ subunits function as sulfur reductase (EC 1.12.98.4, sulfhydrogenase). Different from EC 1.12.1.5, hydrogen dehydrogenase [NAD(P)+]. ',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.12.2.1': {'ec_num': '1.12.2.1',\n",
       "  'accepted_name': 'cytochrome-c3 hydrogenase',\n",
       "  'reaction': 'H2 + 2 ferricytochrome c3 = 2 H+ + 2 ferrocytochrome c3',\n",
       "  'other_names': 'H2:ferricytochrome c3 oxidoreductase; cytochrome c3 reductase; cytochrome hydrogenase; hydrogenase [ambiguous]',\n",
       "  'sys_name': 'hydrogen:ferricytochrome-c3 oxidoreductase',\n",
       "  'comments': 'An iron-sulfur protein. Some forms of the enzyme contain nickel ([NiFe]-hydrogenases) and, of these, some contain selenocysteine ([NiFeSe]-hydrogenases). Methylene blue and other acceptors can also be reduced.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.12.2.1\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.12.2.1\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.12.2.1\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.12.2.1\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.12.2.1%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.12.5.1': {'ec_num': '1.12.5.1',\n",
       "  'accepted_name': 'hydrogen:quinone oxidoreductase',\n",
       "  'reaction': 'H2 + menaquinone = menaquinol',\n",
       "  'other_names': 'hydrogen-ubiquinone oxidoreductase; hydrogen:menaquinone oxidoreductase; membrane-bound hydrogenase; quinone-reactive Ni/Fe-hydrogenase',\n",
       "  'sys_name': 'hydrogen:quinone oxidoreductase',\n",
       "  'comments': 'Contains nickel, iron-sulfur clusters and cytochrome b. Also catalyses the reduction of water-soluble quinones (e.g. 2,3-dimethylnaphthoquinone) or viologen dyes (benzyl viologen or methyl viologen).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'methyl viologen = 1,1′-dimethyl-4,4′-bipyridine-1,1′-diium\\nbenzyl viologen = 1,1′-dibenzyl-4,4′-bipyridine-1,1′-diium\\n',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.12.7.2': {'ec_num': '1.12.7.2',\n",
       "  'accepted_name': 'ferredoxin hydrogenase',\n",
       "  'reaction': 'H2 + 2 oxidized ferredoxin = 2 reduced ferredoxin + 2 H+',\n",
       "  'other_names': 'H2 oxidizing hydrogenase; H2 producing hydrogenase [ambiguous]; bidirectional hydrogenase; hydrogen-lyase [ambiguous]; hydrogenase (ferredoxin); hydrogenase I; hydrogenase II; hydrogenlyase [ambiguous]; uptake hydrogenase [ambiguous]',\n",
       "  'sys_name': 'hydrogen:ferredoxin oxidoreductase',\n",
       "  'comments': 'Contains iron-sulfur clusters. The enzymes from some sources contains nickel. Can use molecular hydrogen for the reduction of a variety of substances.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.12.7.2\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.12.7.2\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.12.7.2\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.12.7.2\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.12.7.2%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.12.98.1': {'ec_num': '1.12.98.1',\n",
       "  'accepted_name': 'coenzyme F420 hydrogenase',\n",
       "  'reaction': 'H2 + oxidized coenzyme F420 = reduced coenzyme F420',\n",
       "  'other_names': '8-hydroxy-5-deazaflavin-reducing hydrogenase; F420-reducing hydrogenase; coenzyme F420-dependent hydrogenase',\n",
       "  'sys_name': 'hydrogen:coenzyme F420 oxidoreductase',\n",
       "  'comments': 'An iron-sulfur flavoprotein (FAD) containing nickel. The enzyme from some sources contains selenocysteine. The enzyme also reduces the riboflavin analogue of F420, flavins and methyl viologen, but to a lesser extent. The hydrogen acceptor coenzyme F420 is a deazaflavin derivative.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'coenzyme F420 = N-(N-{O-[5-(8-hydroxy-2,4-dioxo-2,3,4,10-tetrahydropyrimido[4,5-b]quinolin-10-yl)-5-deoxy-L-ribityl-1-phospho]-(S)-lactyl}-γ-L-glutamyl)-L-glutamate',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '2.1.1.167': {'ec_num': '2.1.1.167',\n",
       "  'accepted_name': '27S pre-rRNA (guanosine2922-2′-O)-methyltransferase',\n",
       "  'reaction': 'S-adenosyl-L-methionine + guanosine2922 in 27S pre-rRNA = S-adenosyl-L-homocysteine + 2′-O-methylguanosine2922 in 27S pre-rRNA ',\n",
       "  'other_names': 'Spb1p (gene name); YCL054W (gene name)',\n",
       "  'sys_name': 'S-adenosyl-L-methionine:27S pre-rRNA (guanosine2922-2′-O-)-methyltransferase',\n",
       "  'comments': 'Spb1p is a site-specific 2′-O-ribose RNA methyltransferase that catalyses the formation of 2′-O-methylguanosine2922, a universally conserved position of the catalytic center of the ribosome that is essential for translation. 2′-O-Methylguanosine2922 is formed at a later stage of the processing, during the maturation of of the 27S pre-rRNA. In absence of snR52, Spb1p can also catalyse the formation of uridine2921 [1]. ',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.12.98.2': {'ec_num': '1.12.98.2',\n",
       "  'accepted_name': '5,10-methenyltetrahydromethanopterin hydrogenase',\n",
       "  'reaction': 'H2 + 5,10-methenyltetrahydromethanopterin = H+ + 5,10-methylenetetrahydromethanopterin',\n",
       "  'other_names': 'H2-forming N5,N10-methylenetetrahydromethanopterin dehydrogenase; nonmetal hydrogenase; N5,N10-methenyltetrahydromethanopterin hydrogenase; hydrogen:N5,N10-methenyltetrahydromethanopterin oxidoreductase',\n",
       "  'sys_name': 'hydrogen:5,10-methenyltetrahydromethanopterin oxidoreductase',\n",
       "  'comments': 'Does not catalyse the reduction of artificial dyes. Does not by itself catalyse a H2/H+ exchange reaction. Does not contain nickel or iron-sulfur clusters.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.12.98.3': {'ec_num': '1.12.98.3',\n",
       "  'accepted_name': 'Methanosarcina-phenazine hydrogenase',\n",
       "  'reaction': 'H2 + 2-(2,3-dihydropentaprenyloxy)phenazine = 2-dihydropentaprenyloxyphenazine',\n",
       "  'other_names': 'methanophenazine hydrogenase; methylviologen-reducing hydrogenase ',\n",
       "  'sys_name': 'hydrogen:2-(2,3-dihydropentaprenyloxy)phenazine oxidoreductase',\n",
       "  'comments': 'Contains nickel, iron-sulfur clusters and cytochrome b. The enzyme from some sources contains selenocysteine.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.12.99.6': {'ec_num': '1.12.99.6',\n",
       "  'accepted_name': 'hydrogenase (acceptor)',\n",
       "  'reaction': 'H2 + acceptor = reduced acceptor',\n",
       "  'other_names': 'H2 producing hydrogenase (ambiguous); hydrogen-lyase (ambiguous); hydrogenlyase (ambiguous); uptake hydrogenase (ambiguous); hydrogen:(acceptor) oxidoreductase',\n",
       "  'sys_name': 'hydrogen:acceptor oxidoreductase',\n",
       "  'comments': 'Uses molecular hydrogen for the reduction of a variety of substances. Contains iron-sulfur clusters. The enzyme from some sources contains nickel.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.1': {'ec_num': '1.13.11.1',\n",
       "  'accepted_name': 'catechol 1,2-dioxygenase',\n",
       "  'reaction': 'catechol + O2 = cis,cis-muconate',\n",
       "  'other_names': 'catechol-oxygen 1,2-oxidoreductase; 1,2-pyrocatechase; catechase; catechol 1,2-oxygenase; catechol dioxygenase; pyrocatechase; pyrocatechol 1,2-dioxygenase; CD I; CD II',\n",
       "  'sys_name': 'catechol:oxygen 1,2-oxidoreductase',\n",
       "  'comments': 'Requires Fe3+. Involved in the metabolism of nitro-aromatic compounds by a strain of Pseudomonas putida.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.1\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.1\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.1\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.1\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.13.11.1%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.2': {'ec_num': '1.13.11.2',\n",
       "  'accepted_name': 'catechol 2,3-dioxygenase',\n",
       "  'reaction': 'catechol + O2 = 2-hydroxymuconate-6-semialdehyde',\n",
       "  'other_names': '2,3-pyrocatechase; catechol 2,3-oxygenase; catechol oxygenase; metapyrocatechase; pyrocatechol 2,3-dioxygenase; xylE (gene name); catechol:oxygen 2,3-oxidoreductase (decyclizing)',\n",
       "  'sys_name': 'catechol:oxygen 2,3-oxidoreductase (ring-opening)',\n",
       "  'comments': 'Requires FeII. The enzyme initiates the meta-cleavage pathway of catechol degradation.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.2\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.2\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.2\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.2\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.13.11.2%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '2-hydroxymuconate-6-semialdehyde = (2Z,4E)-2-hydroxy-6-oxohexa-2,4-dienoate',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.75': {'ec_num': '1.13.11.75',\n",
       "  'accepted_name': 'all-trans-8′-apo-β-carotenal 15,15′-oxygenase',\n",
       "  'reaction': ' all-trans-8′-apo-β-carotenal + O2 = all-trans-retinal + (2E,4E,6E)-2,6-dimethylocta-2,4,6-trienedial\\n ',\n",
       "  'other_names': 'Diox1; ACO; 8′-apo-β-carotenal 15,15′-oxygenase',\n",
       "  'sys_name': 'all-trans-8′-apo-β-carotenal:oxygen 15,15′-oxidoreductase (bond-cleaving)',\n",
       "  'comments': 'Contains an Fe2+-4His arrangement. The enzyme is involved in retinal biosynthesis in bacteria [2]. ',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.3': {'ec_num': '1.13.11.3',\n",
       "  'accepted_name': 'protocatechuate 3,4-dioxygenase',\n",
       "  'reaction': '3,4-dihydroxybenzoate + O2 = 3-carboxy-cis,cis-muconate',\n",
       "  'other_names': 'protocatechuate oxygenase; protocatechuic acid oxidase; protocatechuic 3,4-dioxygenase; protocatechuic 3,4-oxygenase; protocatechuate:oxygen 3,4-oxidoreductase (decyclizing) ',\n",
       "  'sys_name': 'protocatechuate:oxygen 3,4-oxidoreductase (ring-opening) ',\n",
       "  'comments': 'Requires Fe3+. The enzyme, which participates in the degradation of aromatic compounds, catalyses the intradiol addition of both oxygen atoms from molecular oxygen, resulting in ortho-cleavage of the aromatic ring. The type of cleavage leads to mineralization via the intermediate 3-oxoadipate.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.3\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.3\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.3\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.3\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.13.11.3%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '3,4-dihydroxybenzoate = protocatechuate',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.4': {'ec_num': '1.13.11.4',\n",
       "  'accepted_name': 'gentisate 1,2-dioxygenase',\n",
       "  'reaction': '2,5-dihydroxybenzoate + O2 = maleylpyruvate',\n",
       "  'other_names': 'gentisate oxygenase; 2,5-dihydroxybenzoate dioxygenase; gentisate dioxygenase; gentisic acid oxidase; gentisate:oxygen 1,2-oxidoreductase (decyclizing)',\n",
       "  'sys_name': 'gentisate:oxygen 1,2-oxidoreductase (ring-opening)',\n",
       "  'comments': 'Requires Fe2+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.5': {'ec_num': '1.13.11.5',\n",
       "  'accepted_name': 'homogentisate 1,2-dioxygenase',\n",
       "  'reaction': 'homogentisate + O2 = 4-maleylacetoacetate',\n",
       "  'other_names': 'homogentisicase; homogentisate oxygenase; homogentisate dioxygenase; homogentisate oxidase; homogentisic acid oxidase; homogentisic acid oxygenase; homogentisic oxygenase; homogentisate:oxygen 1,2-oxidoreductase (decyclizing)',\n",
       "  'sys_name': 'homogentisate:oxygen 1,2-oxidoreductase (ring-opening)',\n",
       "  'comments': 'Requires Fe2+.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.5\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.5\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.5\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.5\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.13.11.5%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.6': {'ec_num': '1.13.11.6',\n",
       "  'accepted_name': '3-hydroxyanthranilate 3,4-dioxygenase',\n",
       "  'reaction': '3-hydroxyanthranilate + O2 = 2-amino-3-carboxymuconate semialdehyde',\n",
       "  'other_names': '3-hydroxyanthranilate oxygenase; 3-hydroxyanthranilic acid oxygenase; 3-hydroxyanthranilic oxygenase; 3-hydroxyanthranilic acid oxidase; 3HAO; 3-hydroxyanthranilate:oxygen 3,4-oxidoreductase (decyclizing)',\n",
       "  'sys_name': '3-hydroxyanthranilate:oxygen 3,4-oxidoreductase (ring-opening)',\n",
       "  'comments': 'Requires Fe2+.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.6\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.6\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.6\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.6\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.13.11.6%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.8': {'ec_num': '1.13.11.8',\n",
       "  'accepted_name': 'protocatechuate 4,5-dioxygenase',\n",
       "  'reaction': '3,4-dihydroxybenzoate + O2 = 4-carboxy-2-hydroxymuconate semialdehyde',\n",
       "  'other_names': 'protocatechuate 4,5-oxygenase; protocatechuic 4,5-dioxygenase; protocatechuic 4,5-oxygenase; protocatechuate:oxygen 4,5-oxidoreductase (decyclizing); protocatechuate:oxygen 4,5-oxidoreductase (ring-opening)',\n",
       "  'sys_name': '3,4-dihydroxybenzoate:oxygen 4,5-oxidoreductase (ring-opening)',\n",
       "  'comments': 'Requires Fe2+.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.8\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.8\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.8\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.8\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.13.11.8%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '3,4-dihydroxybenzoate = protocatechuate',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.9': {'ec_num': '1.13.11.9',\n",
       "  'accepted_name': '2,5-dihydroxypyridine 5,6-dioxygenase',\n",
       "  'reaction': '2,5-dihydroxypyridine + O2  = N-formylmaleamic acid',\n",
       "  'other_names': '2,5-dihydroxypyridine oxygenase; pyridine-2,5-diol dioxygenase; NicX',\n",
       "  'sys_name': '2,5-dihydroxypyridine:oxygen 5,6-oxidoreductase',\n",
       "  'comments': 'Requires Fe2+.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.9\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.9\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.9\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.9\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.13.11.9%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.10': {'ec_num': '1.13.11.10',\n",
       "  'accepted_name': '7,8-dihydroxykynurenate 8,8a-dioxygenase',\n",
       "  'reaction': '7,8-dihydroxykynurenate + O2 = 5-(3-carboxy-3-oxopropenyl)-4,6-dihydroxypyridine-2-carboxylate',\n",
       "  'other_names': '7,8-dihydroxykynurenate oxygenase; 7,8-dihydroxykynurenate 8,8α-dioxygenase; 7,8-dihydroxykynurenate:oxygen 8,8a-oxidoreductase (decyclizing)',\n",
       "  'sys_name': '7,8-dihydroxykynurenate:oxygen 8,8a-oxidoreductase (ring-opening)',\n",
       "  'comments': 'Requires Fe2+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.11': {'ec_num': '1.13.11.11',\n",
       "  'accepted_name': 'tryptophan 2,3-dioxygenase',\n",
       "  'reaction': 'L-tryptophan + O2 = N-formyl-L-kynurenine',\n",
       "  'other_names': 'tryptophan pyrrolase (ambiguous); tryptophanase; tryptophan oxygenase; tryptamine 2,3-dioxygenase; tryptophan peroxidase; indoleamine 2,3-dioxygenase (ambiguous); indolamine 2,3-dioxygenase (ambiguous); L-tryptophan pyrrolase; TDO; L-tryptophan 2,3-dioxygenase; L-tryptophan:oxygen 2,3-oxidoreductase (decyclizing)',\n",
       "  'sys_name': 'L-tryptophan:oxygen 2,3-oxidoreductase (ring-opening)',\n",
       "  'comments': 'A protohemoprotein. In mammals, the enzyme appears to be located only in the liver. This enzyme, together with EC 1.13.11.52, indoleamine 2,3-dioxygenase, catalyses the first and rate-limiting step in the kynurenine pathway, the major pathway of tryptophan metabolism [5]. The enzyme is specific for tryptophan as substrate, but is far more active with L-tryptophan than with D-tryptophan [2]. ',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.12': {'ec_num': '1.13.11.12',\n",
       "  'accepted_name': 'linoleate 13S-lipoxygenase',\n",
       "  'reaction': '(1) linoleate + O2 = (9Z,11E,13S)-13-hydroperoxyoctadeca-9,11-dienoate;;(2) α-linolenate + O2 = (9Z,11E,13S,15Z)-13-hydroperoxyoctadeca-9,11,15-trienoate',\n",
       "  'other_names': '13-lipoxidase; carotene oxidase; 13-lipoperoxidase; fat oxidase; 13-lipoxydase; lionoleate:O2 13-oxidoreductase',\n",
       "  'sys_name': 'linoleate:oxygen 13-oxidoreductase',\n",
       "  'comments': 'Contains nonheme iron. A common plant lipoxygenase that oxidizes linoleate and α-linolenate, the two most common polyunsaturated fatty acids in plants, by inserting molecular oxygen at the C-13 position with (S)-configuration. This enzyme produces precursors for several important compounds, including the plant hormone jasmonic acid. EC 1.13.11.58, linoleate 9S-lipoxygenase, catalyses a similar reaction at the second available position of these fatty acids.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'linoleate = (9Z,12Z)-octadeca-9,12-dienoate\\nα-linolenate = (9Z,12Z,15Z)-octadeca-9,12,15-trienoate',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.13': {'ec_num': '1.13.11.13',\n",
       "  'accepted_name': 'ascorbate 2,3-dioxygenase',\n",
       "  'reaction': 'ascorbate + O2 + H2O = oxalate + threonate',\n",
       "  'other_names': 'AAoxygenase',\n",
       "  'sys_name': 'ascorbate:oxygen 2,3-oxidoreductase (bond-cleaving)',\n",
       "  'comments': 'Requires iron(II).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.14': {'ec_num': '1.13.11.14',\n",
       "  'accepted_name': '2,3-dihydroxybenzoate 3,4-dioxygenase',\n",
       "  'reaction': '2,3-dihydroxybenzoate + O2 = 3-carboxy-2-hydroxymuconate semialdehyde',\n",
       "  'other_names': 'o-pyrocatechuate oxygenase; 2,3-dihydroxybenzoate 1,2-dioxygenase; 2,3-dihydroxybenzoic oxygenase; 2,3-dihydroxybenzoate oxygenase; 2,3-dihydroxybenzoate:oxygen 3,4-oxidoreductase (decyclizing)',\n",
       "  'sys_name': '2,3-dihydroxybenzoate:oxygen 3,4-oxidoreductase (ring-opening)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.15': {'ec_num': '1.13.11.15',\n",
       "  'accepted_name': '3,4-dihydroxyphenylacetate 2,3-dioxygenase',\n",
       "  'reaction': '3,4-dihydroxyphenylacetate + O2 = 2-hydroxy-5-carboxymethylmuconate semialdehyde',\n",
       "  'other_names': '3,4-dihydroxyphenylacetic acid 2,3-dioxygenase; HPC dioxygenase; homoprotocatechuate 2,3-dioxygenase; 3,4-dihydroxyphenylacetate:oxygen 2,3-oxidoreductase (decyclizing)',\n",
       "  'sys_name': '3,4-dihydroxyphenylacetate:oxygen 2,3-oxidoreductase (ring-opening)',\n",
       "  'comments': 'An iron protein.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.15\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.15\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.15\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.15\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.13.11.15%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.16': {'ec_num': '1.13.11.16',\n",
       "  'accepted_name': '3-carboxyethylcatechol 2,3-dioxygenase',\n",
       "  'reaction': '(1) 3-(2,3-dihydroxyphenyl)propanoate + O2 = (2Z,4E)-2-hydroxy-6-oxonona-2,4-diene-1,9-dioate;;(2) (2E)-3-(2,3-dihydroxyphenyl)prop-2-enoate + O2 = (2Z,4E,7E)-2-hydroxy-6-oxonona-2,4,7-triene-1,9-dioate',\n",
       "  'other_names': '2,3-dihydroxy-β-phenylpropionic dioxygenase; 2,3-dihydroxy-β-phenylpropionate oxygenase; 3-(2,3-dihydroxyphenyl)propanoate:oxygen 1,2-oxidoreductase; 3-(2,3-dihydroxyphenyl)propanoate:oxygen 1,2-oxidoreductase (decyclizing)',\n",
       "  'sys_name': '3-(2,3-dihydroxyphenyl)propanoate:oxygen 1,2-oxidoreductase (ring-opening)',\n",
       "  'comments': 'An iron protein. This enzyme catalyses a step in the pathway of phenylpropanoid compounds degradation.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.16\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.16\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.16\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.16\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '(2E)-3-(2,3-dihydroxyphenyl)prop-2-enoate = trans-2,3-dihydroxycinnamate',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.17': {'ec_num': '1.13.11.17',\n",
       "  'accepted_name': 'indole 2,3-dioxygenase',\n",
       "  'reaction': 'indole + O2 = 2-formylaminobenzaldehyde',\n",
       "  'other_names': 'indole oxidase; indoleamine 2,3-dioxygenase (ambiguous); indole:O2 oxidoreductase; indole-oxygen 2,3-oxidoreductase (decyclizing); IDO (ambiguous); indole:oxygen 2,3-oxidoreductase (decyclizing)',\n",
       "  'sys_name': 'indole:oxygen 2,3-oxidoreductase (ring-opening)',\n",
       "  'comments': 'Enzymes from the plants Tecoma stans, Jasminum grandiflorum and Zea mays are flavoproteins containing copper. They are part of enzyme systems that form either anthranil (2,1-benzoisoxazole) (Tecoma stans), anthranilate (Jasminum grandiflorum) or both (Zea mays) as the final product. A second enzyme from Tecoma stans is not a flavoprotein, does not require copper, and is part of a system that forms anthranilate as the final product.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.18': {'ec_num': '1.13.11.18',\n",
       "  'accepted_name': 'persulfide dioxygenase',\n",
       "  'reaction': 'S-sulfanylglutathione + O2 + H2O = glutathione + sulfite + 2 H+ (overall reaction);;(1a) S-sulfanylglutathione + O2 = S-sulfinatoglutathione + H+;;(1b) S-sulfinatoglutathione + H2O = glutathione + sulfite + H+ (spontaneous)',\n",
       "  'other_names': 'sulfur oxygenase (incorrect); sulfur:oxygen oxidoreductase (incorrect); sulfur dioxygenase (incorrect)',\n",
       "  'sys_name': 'S-sulfanylglutathione:oxygen oxidoreductase',\n",
       "  'comments': 'An iron protein. Perthiols, formed spontaneously by interactions between thiols and elemental sulfur or sulfide, are the only acceptable substrate to the enzyme. The sulfite that is formed by the enzyme can be further converted into sulfate, thiosulfate or S-sulfoglutathione (GSSO3-) non-enzymically [2].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.19': {'ec_num': '1.13.11.19',\n",
       "  'accepted_name': 'cysteamine dioxygenase',\n",
       "  'reaction': ' cysteamine + O2 = hypotaurine',\n",
       "  'other_names': 'ADO (gene name); persulfurase; cysteamine oxygenase; cysteamine:oxygen oxidoreductase',\n",
       "  'sys_name': '2-aminoethanethiol:oxygen oxidoreductase',\n",
       "  'comments': 'A non-heme iron protein that is involved in the biosynthesis of taurine. 3-Aminopropanethiol (homocysteamine) and 2-sulfanylethan-1-ol (2-mercaptoethanol) can also act as substrates, but glutathione, cysteine, and cysteine ethyl- and methyl esters are not good substrates [1,3].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'cysteamine = 2-aminoethanethiol',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.20': {'ec_num': '1.13.11.20',\n",
       "  'accepted_name': 'cysteine dioxygenase',\n",
       "  'reaction': 'L-cysteine + O2 = 3-sulfinoalanine',\n",
       "  'other_names': 'cysteine oxidase',\n",
       "  'sys_name': 'L-cysteine:oxygen oxidoreductase',\n",
       "  'comments': 'Requires Fe2+ and NAD(P)H.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.22': {'ec_num': '1.13.11.22',\n",
       "  'accepted_name': 'caffeate 3,4-dioxygenase',\n",
       "  'reaction': '3,4-dihydroxy-trans-cinnamate + O2 = 3-(2-carboxyethenyl)-cis,cis-muconate',\n",
       "  'other_names': '3,4-dihydroxy-trans-cinnamate:oxygen 3,4-oxidoreductase (decyclizing)',\n",
       "  'sys_name': '3,4-dihydroxy-trans-cinnamate:oxygen 3,4-oxidoreductase (ring-opening)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.23': {'ec_num': '1.13.11.23',\n",
       "  'accepted_name': '2,3-dihydroxyindole 2,3-dioxygenase',\n",
       "  'reaction': '2,3-dihydroxyindole + O2 = anthranilate + CO2',\n",
       "  'other_names': '2,3-dihydroxyindole:oxygen 2,3-oxidoreductase (decyclizing)',\n",
       "  'sys_name': '2,3-dihydroxyindole:oxygen 2,3-oxidoreductase (ring-opening)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.24': {'ec_num': '1.13.11.24',\n",
       "  'accepted_name': 'quercetin 2,3-dioxygenase',\n",
       "  'reaction': 'quercetin + O2 = 2-(3,4-dihydroxybenzoyloxy)-4,6-dihydroxybenzoate + CO + H+',\n",
       "  'other_names': 'quercetinase; flavonol 2,4-oxygenase; quercetin:oxygen 2,3-oxidoreductase (decyclizing)',\n",
       "  'sys_name': 'quercetin:oxygen 2,3-oxidoreductase (ring-opening)',\n",
       "  'comments': 'The enzyme from Aspergillus sp. is a copper protein whereas that from Bacillus subtilis contains iron. Quercetin is a flavonol (5,7,3′,4′-tetrahydroxyflavonol).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.24\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.24\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.24\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.24\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.13.11.24%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.25': {'ec_num': '1.13.11.25',\n",
       "  'accepted_name': '3,4-dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione 4,5-dioxygenase',\n",
       "  'reaction': '3,4-dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione + O2 = 3-hydroxy-5,9,17-trioxo-4,5:9,10-disecoandrosta-1(10),2-dien-4-oate',\n",
       "  'other_names': 'steroid 4,5-dioxygenase; 3-alkylcatechol 2,3-dioxygenase; 3,4-dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione:oxygen 4,5-oxidoreductase (decyclizing)',\n",
       "  'sys_name': '3,4-dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione:oxygen 4,5-oxidoreductase (ring-opening)',\n",
       "  'comments': 'Requires Fe2+. Also acts on 3-isopropylcatechol and 3-tert-butyl-5-methylcatechol.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.25\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.25\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.25\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.25\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.13.11.25%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.26': {'ec_num': '1.13.11.26',\n",
       "  'accepted_name': 'peptide-tryptophan 2,3-dioxygenase',\n",
       "  'reaction': '[protein]-L-tryptophan + O2 = [protein]-N-formyl-L-kynurenine',\n",
       "  'other_names': 'pyrrolooxygenase; peptidyltryptophan 2,3-dioxygenase; tryptophan pyrrolooxygenase; [protein]-L-tryptophan:oxygen 2,3-oxidoreductase (decyclizing)',\n",
       "  'sys_name': '[protein]-L-tryptophan:oxygen 2,3-oxidoreductase (ring-opening)',\n",
       "  'comments': 'Also acts on tryptophan.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'N-formyl-L-kynurenine = (2S)-2-amino-4-[2-(formamido)phenyl]-4-oxobutanoic acid',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.27': {'ec_num': '1.13.11.27',\n",
       "  'accepted_name': '4-hydroxyphenylpyruvate dioxygenase',\n",
       "  'reaction': '4-hydroxyphenylpyruvate + O2 = homogentisate + CO2',\n",
       "  'other_names': 'p-hydroxyphenylpyruvic hydroxylase; p-hydroxyphenylpyruvate hydroxylase; p-hydroxyphenylpyruvate oxidase; p-hydroxyphenylpyruvic oxidase; p-hydroxyphenylpyruvate dioxygenase; p-hydroxyphenylpyruvic acid hydroxylase; 4-hydroxyphenylpyruvic acid dioxygenase',\n",
       "  'sys_name': '4-hydroxyphenylpyruvate:oxygen oxidoreductase (hydroxylating, decarboxylating)',\n",
       "  'comments': 'The Pseudomonas enzyme contains one Fe3+ per mole of enzyme; the enzymes from other sources may contain essential iron or copper.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.27\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.27\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.27\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.27\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.13.11.27%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.28': {'ec_num': '1.13.11.28',\n",
       "  'accepted_name': '2,3-dihydroxybenzoate 2,3-dioxygenase',\n",
       "  'reaction': '2,3-dihydroxybenzoate + O2 = 2-carboxy-cis,cis-muconate',\n",
       "  'other_names': '2,3-dihydroxybenzoate 2,3-oxygenase; 2,3-dihydroxybenzoate:oxygen 2,3-oxidoreductase (decyclizing)',\n",
       "  'sys_name': '2,3-dihydroxybenzoate:oxygen 2,3-oxidoreductase (ring-opening)',\n",
       "  'comments': 'Also acts, more slowly, with 2,3-dihydroxy-4-methylbenzoate and 2,3-dihydroxy-4-isopropylbenzoate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.29': {'ec_num': '1.13.11.29',\n",
       "  'accepted_name': 'stizolobate synthase',\n",
       "  'reaction': 'L-dopa + O2 = 4-(L-alanin-3-yl)-2-hydroxy-cis,cis-muconate 6-semialdehyde',\n",
       "  'other_names': '',\n",
       "  'sys_name': '3,4-dihydroxy-L-phenylalanine:oxygen 4,5-oxidoreductase (recyclizing)',\n",
       "  'comments': 'The intermediate product undergoes ring closure and oxidation, with NAD(P)+ as acceptor, to stizolobic acid. The enzyme requires Zn2+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'L-dopa = 3,4-dihydroxy-L-phenylalanine',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.30': {'ec_num': '1.13.11.30',\n",
       "  'accepted_name': 'stizolobinate synthase',\n",
       "  'reaction': 'L-dopa + O2 = 5-(L-alanin-3-yl)-2-hydroxy-cis,cis-muconate 6-semialdehyde',\n",
       "  'other_names': '',\n",
       "  'sys_name': '3,4-dihydroxy-L-phenylalanine:oxygen 2,3-oxidoreductase (recyclizing)',\n",
       "  'comments': 'The intermediate product undergoes ring closure and oxidation, with NAD(P)+ as acceptor, to stizolobinic acid. The enzyme requires Zn2+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'L-dopa = 3,4-dihydroxy-L-phenylalanine',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.31': {'ec_num': '1.13.11.31',\n",
       "  'accepted_name': 'arachidonate 12-lipoxygenase',\n",
       "  'reaction': 'arachidonate + O2 = (5Z,8Z,10E,14Z)-(12S)-12-hydroperoxyicosa-5,8,10,14-tetraenoate',\n",
       "  'other_names': 'Δ12-lipoxygenase; 12-lipoxygenase; 12Δ-lipoxygenase; C-12 lipoxygenase; 12S-lipoxygenase; leukotriene A4 synthase; LTA4 synthase',\n",
       "  'sys_name': 'arachidonate:oxygen 12-oxidoreductase',\n",
       "  'comments': 'The product is rapidly reduced to the corresponding 12S-hydroxy compound.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.32': {'ec_num': '1.13.11.32',\n",
       "  'accepted_name': '2-nitropropane dioxygenase',\n",
       "  'reaction': '2 2-nitropropane + O2 = 2 acetone + 2 nitrite',\n",
       "  'other_names': '',\n",
       "  'sys_name': '2-nitropropane:oxygen 2-oxidoreductase',\n",
       "  'comments': 'While the enzyme from the fungus Neurospora crassa contains non-covalently bound FMN as the cofactor, that from the yeast Williopsis mrakii contains FAD [2]. This enzyme differs from EC 1.7.3.1, nitroalkane oxidase, in that the preferred substrates are anionic nitronates rather than neutral nitroalkanes [2,3]. The enzyme has broad substrate specificity that is independent of substrate size [2,3]. Some other nitroalkanes, including nitroethane, 1-nitropropane and 3-nitropentan-2-ol, can act as donors, but more slowly.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.32\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.32\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.32\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.32\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.33': {'ec_num': '1.13.11.33',\n",
       "  'accepted_name': 'arachidonate 15-lipoxygenase',\n",
       "  'reaction': 'arachidonate + O2 = (5Z,8Z,11Z,13E)-(15S)-15-hydroperoxyicosa-5,8,11,13-tetraenoate',\n",
       "  'other_names': '15-lipoxygenase; linoleic acid ω6-lipoxygenase; ω6 lipoxygenase',\n",
       "  'sys_name': 'arachidonate:oxygen 15-oxidoreductase',\n",
       "  'comments': 'The product is rapidly reduced to the corresponding 15S-hydroxy compound.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.34': {'ec_num': '1.13.11.34',\n",
       "  'accepted_name': 'arachidonate 5-lipoxygenase',\n",
       "  'reaction': 'arachidonate + O2 = leukotriene A4 + H2O (overall reaction);;(1a) arachidonate + O2 = (6E,8Z,11Z,14Z)-(5S)-5-hydroperoxyicosa-6,8,11,14-tetraenoate;;(1b) (6E,8Z,11Z,14Z)-(5S)-5-hydroperoxyicosa-6,8,11,14-tetraenoate = leukotriene A4 + H2O ',\n",
       "  'other_names': 'leukotriene-A4 synthase; Δ5-lipoxygenase; 5Δ-lipoxygenase; arachidonic 5-lipoxygenase; arachidonic acid 5-lipoxygenase; C-5-lipoxygenase; LTA synthase; leukotriene A4 synthase',\n",
       "  'sys_name': 'arachidonate:oxygen 5-oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'leukotriene A4 = (7E,9E,11Z,14Z)-(5S,6S)-5,6-epoxyicosa-7,9,11,14-tetraenoate',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.35': {'ec_num': '1.13.11.35',\n",
       "  'accepted_name': 'pyrogallol 1,2-oxygenase',\n",
       "  'reaction': '1,2,3-trihydroxybenzene + O2 = (2Z,4E)-2-hydroxyhexa-2,4-dienedioate',\n",
       "  'other_names': 'pyrogallol 1,2-dioxygenase; 1,2,3-trihydroxybenzene:oxygen 1,2-oxidoreductase (decyclizing)',\n",
       "  'sys_name': '1,2,3-trihydroxybenzene:oxygen 1,2-oxidoreductase (ring-opening)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '(2Z,4E)-2-hydroxyhexa-2,4-dienedioate = (2Z,4E)-2-hydroxymuconate\\n1,2,3-trihydroxybenzene = pyrogallol',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.36': {'ec_num': '1.13.11.36',\n",
       "  'accepted_name': 'chloridazon-catechol dioxygenase',\n",
       "  'reaction': '5-amino-4-chloro-2-(2,3-dihydroxyphenyl)-3(2H)-pyridazinone + O2 = 5-amino-4-chloro-2-(2-hydroxymuconoyl)-3(2H)-pyridazinone',\n",
       "  'other_names': '5-amino-4-chloro-2-(2,3-dihydroxyphenyl)-3(2H)-pyridazinone 1,2-oxidoreductase (decyclizing)',\n",
       "  'sys_name': '5-amino-4-chloro-2-(2,3-dihydroxyphenyl)-3(2H)-pyridazinone 1,2-oxidoreductase (ring-opening)',\n",
       "  'comments': 'An iron protein, requiring additional Fe2+. Not identical with EC 1.13.11.1 (catechol 1,2-dioxygenase), EC 1.13.11.2 (catechol 2,3-dioxygenase) or EC 1.13.11.5 (homogentisate 1,2-dioxygenase). Involved in the breakdown of the herbicide chloridazon.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.37': {'ec_num': '1.13.11.37',\n",
       "  'accepted_name': 'hydroxyquinol 1,2-dioxygenase',\n",
       "  'reaction': 'hydroxyquinol + O2 = maleylacetate',\n",
       "  'other_names': 'hydroxyquinol dioxygenase; benzene-1,2,4-triol:oxygen 1,2-oxidoreductase (decyclizing); benzene-1,2,4-triol:oxygen 1,2-oxidoreductase (ring-opening)',\n",
       "  'sys_name': 'hydroxyquinol:oxygen 1,2-oxidoreductase (ring-opening)',\n",
       "  'comments': 'An iron protein. Highly specific; catechol and pyrogallol are acted on at less than 1% of the rate at which hydroxyquinol is oxidized.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.37\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.37\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.37\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.37\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.13.11.37%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': 'hydroxyquinol = 1,2,4-trihydroxybenzene\\nmaleylacetate = (2Z)-4-oxohex-2-enedioate\\n',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.38': {'ec_num': '1.13.11.38',\n",
       "  'accepted_name': '1-hydroxy-2-naphthoate 1,2-dioxygenase',\n",
       "  'reaction': '1-hydroxy-2-naphthoate + O2 = (3Z)-4-(2-carboxyphenyl)-2-oxobut-3-enoate',\n",
       "  'other_names': '1-hydroxy-2-naphthoate dioxygenase; 1-hydroxy-2-naphthoate-degrading enzyme; 1-hydroxy-2-naphthoic acid dioxygenase; 1-hydroxy-2-naphthoate:oxygen 1,2-oxidoreductase (decyclizing)',\n",
       "  'sys_name': '1-hydroxy-2-naphthoate:oxygen 1,2-oxidoreductase (ring-opening)',\n",
       "  'comments': 'Requires Fe2+. Involved, with EC 4.1.2.34 4-(2-carboxyphenyl)-2-oxobut-3-enoate aldolase, in the metabolism of phenanthrene in bacteria.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.38\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.38\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.38\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.38\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.39': {'ec_num': '1.13.11.39',\n",
       "  'accepted_name': 'biphenyl-2,3-diol 1,2-dioxygenase',\n",
       "  'reaction': 'biphenyl-2,3-diol + O2 = 2-hydroxy-6-oxo-6-phenylhexa-2,4-dienoate',\n",
       "  'other_names': '2,3-dihydroxybiphenyl dioxygenase; biphenyl-2,3-diol dioxygenase; bphC (gene name); biphenyl-2,3-diol:oxygen 1,2-oxidoreductase (decyclizing)',\n",
       "  'sys_name': 'biphenyl-2,3-diol:oxygen 1,2-oxidoreductase (ring-opening)',\n",
       "  'comments': 'Contains Fe2+ or Mn2+ [3]. This enzyme participates in the degradation pathway of biphenyl and PCB (poly chlorinated biphenyls), and catalyses the first ring cleavage step by incorporating two oxygen atoms into the catechol ring formed by EC 1.3.1.56, cis-2,3-dihydrobiphenyl-2,3-diol dehydrogenase.The enzyme from the bacterium Burkholderia xenovorans LB400 can also process catechol, 3-methylcatechol, and 4-methylcatechol, but less efficiently [1]. The enzyme from the carbazole-degrader Pseudomonas resinovorans strain CA10 also accepts 2′-aminobiphenyl-2,3-diol [5]. The enzyme from Ralstonia sp. SBUG 290 can also accept 1,2-dihydroxydibenzofuran and 1,2-dihydroxynaphthalene [4]. The enzyme is strongly inhibited by the substrate [1].Not identical with EC 1.13.11.2 catechol 2,3-dioxygenase.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.39\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.39\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.39\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.39\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.13.11.39%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.40': {'ec_num': '1.13.11.40',\n",
       "  'accepted_name': 'arachidonate 8-lipoxygenase',\n",
       "  'reaction': 'arachidonate + O2 = (5Z,9E,11Z,14Z)-(8R)-8-hydroperoxyicosa-5,9,11,14-tetraenoate',\n",
       "  'other_names': '8-lipoxygenase; 8(R)-lipoxygenase',\n",
       "  'sys_name': 'arachidonate:oxygen 8-oxidoreductase',\n",
       "  'comments': 'From the coral Pseudoplexaura porosa.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.41': {'ec_num': '1.13.11.41',\n",
       "  'accepted_name': '2,4′-dihydroxyacetophenone dioxygenase',\n",
       "  'reaction': '2,4′-dihydroxyacetophenone + O2 = 4-hydroxybenzoate + formate',\n",
       "  'other_names': '(4-hydroxybenzoyl)methanol oxygenase',\n",
       "  'sys_name': '2,4′-dihydroxyacetophenone oxidoreductase (C-C-bond-cleaving)',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.41\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.41\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.41\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.41\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.13.11.41%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.42': {'ec_num': '1.13.11.42',\n",
       "  'accepted_name': 'indoleamine-pyrrole 2,3-dioxygenase',\n",
       "  'reaction': 'L-tryptophan + O2 = N-formylkynurenine',\n",
       "  'other_names': 'indoleamine 2,3-dioxygenase',\n",
       "  'sys_name': 'indoleamine:oxygen 2,3-dioxygenase (indole-decyclizing)',\n",
       "  'comments': 'Acts on many substituted and unsubstituted indoleamines, including melatonin. Involved in the degradation of melatonin.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.43': {'ec_num': '1.13.11.43',\n",
       "  'accepted_name': 'lignostilbene αβ-dioxygenase',\n",
       "  'reaction': '1,2-bis(4-hydroxy-3-methoxyphenyl)ethylene + O2 = 2 vanillin',\n",
       "  'other_names': '',\n",
       "  'sys_name': '1,2-bis(4-hydroxy-3-methoxyphenyl)ethylene:oxygen oxidoreductase (αβ-bond-cleaving)',\n",
       "  'comments': 'An iron protein. The enzyme catalyses oxidative cleavage of the interphenyl double bond in the synthetic substrate and lignin-derived stilbenes. It is responsible for the degradation of a diarylpropane-type structure in lignin.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.44': {'ec_num': '1.13.11.44',\n",
       "  'accepted_name': 'linoleate diol synthase',\n",
       "  'reaction': 'linoleate + O2 = (9Z,12Z)-(7S,8S)-dihydroxyoctadeca-9,12-dienoate',\n",
       "  'other_names': 'linoleate (8R)-dioxygenase',\n",
       "  'sys_name': 'linoleate:oxygen 7S,8S-oxidoreductase',\n",
       "  'comments': 'Oleate and linolenate are also substrates, whereas stearate, elaidate, γ-linolenate, arachidonate and icosapentaenate are not. This enzyme differs from lipoxygenase (EC 1.13.11.12) because it catalyses the formation of a hydroperoxide without affecting the double bonds of the substrate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.45': {'ec_num': '1.13.11.45',\n",
       "  'accepted_name': 'linoleate 11-lipoxygenase',\n",
       "  'reaction': 'linoleate + O2 = (9Z,12Z)-(11S)-11-hydroperoxyoctadeca-9,12-dienoate',\n",
       "  'other_names': 'linoleate dioxygenase; manganese lipoxygenase',\n",
       "  'sys_name': 'linoleate:oxygen 11S-oxidoreductase',\n",
       "  'comments': 'The product (9Z,12Z)-(11S)-11-hydroperoxyoctadeca-9,12-dienoate, is converted, more slowly, into (9Z,11E)-(13R)-13-hydroperoxyoctadeca-9,11-dienoate. The enzyme from the fungus Gaeumannomyces graminis requires Mn2+. It also acts on α-linolenate, whereas γ-linolenate is a poor substrate. Oleate and arachidonate are not substrates.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'arachidonate = (all-Z)-icosa-5,8,11,14-tetraenoate\\nlinoleate = (9Z,12Z)-octadeca-9,12-dienoate\\nα-linolenate = (9Z,12Z,15Z)-octadeca-9,12,15-trienoate\\nγ-linolenate = (6Z,9Z,12Z)-octadeca-6,9,12-trienoate\\noleate = (Z)-octadec-9-enoate',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.46': {'ec_num': '1.13.11.46',\n",
       "  'accepted_name': '4-hydroxymandelate synthase',\n",
       "  'reaction': '4-hydroxyphenylpyruvate + O2 = (S)-4-hydroxymandelate + CO2',\n",
       "  'other_names': '4-hydroxyphenylpyruvate dioxygenase II',\n",
       "  'sys_name': '(S)-4-hydroxyphenylpyruvate:oxygen oxidoreductase (decarboxylating)',\n",
       "  'comments': 'Requires Fe2+. Involved in the biosynthesis of the vancomycin group of glycopeptide antibiotics.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '(S)-4-hydroxymandelate = (S)-2-hydroxy-2-(4-hydroxyphenyl)acetate \\n',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.47': {'ec_num': '1.13.11.47',\n",
       "  'accepted_name': '3-hydroxy-4-oxoquinoline 2,4-dioxygenase',\n",
       "  'reaction': '3-hydroxy-1H-quinolin-4-one + O2 = N-formylanthranilate + CO',\n",
       "  'other_names': '(1H)-3-hydroxy-4-oxoquinoline 2,4-dioxygenase; 3-hydroxy-4-oxo-1,4-dihydroquinoline 2,4-dioxygenase; 3-hydroxy-4(1H)-one, 2,4-dioxygenase; quinoline-3,4-diol 2,4-dioxygenase',\n",
       "  'sys_name': '3-hydroxy-1H-quinolin-4-one 2,4-dioxygenase (CO-forming)',\n",
       "  'comments': 'Does not contain a metal centre or organic cofactor. Fission of two C-C bonds: 2,4-dioxygenolytic cleavage with concomitant release of carbon monoxide. The enzyme from Pseudomonas putida is highly specific for this substrate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.48': {'ec_num': '1.13.11.48',\n",
       "  'accepted_name': '3-hydroxy-2-methylquinolin-4-one 2,4-dioxygenase',\n",
       "  'reaction': '3-hydroxy-2-methyl-1H-quinolin-4-one + O2 = N-acetylanthranilate + CO',\n",
       "  'other_names': '(1H)-3-hydroxy-4-oxoquinaldine 2,4-dioxygenase',\n",
       "  'sys_name': '3-hydroxy-2-methyl-1H-quinolin-4-one 2,4-dioxygenase (CO-forming)',\n",
       "  'comments': 'Does not contain a metal centre or organic cofactor. Fission of two C-C bonds: 2,4-dioxygenolytic cleavage with concomitant release of carbon monoxide. The enzyme from Arthrobacter sp. can also act on 3-hydroxy-4-oxoquinoline, forming N-formylanthranilate and CO (cf. EC 1.13.11.47, 3-hydroxy-4-oxoquinoline 2,4-dioxygenase), but more slowly.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.48\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.48\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.48\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.48\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.13.11.48%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.49': {'ec_num': '1.13.11.49',\n",
       "  'accepted_name': 'chlorite O2-lyase',\n",
       "  'reaction': 'chloride + O2 = chlorite',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'chloride:oxygen oxidoreductase',\n",
       "  'comments': \"Reaction occurs in the reverse direction in chlorate- and perchlorate-reducing bacteria. There is no activity when chlorite is replaced by hydrogen peroxide, perchlorate, chlorate or nitrite. The term 'chlorite dismutase' is misleading as the reaction does not involve dismutation/disproportionation. Contains iron and protoheme IX.\",\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.49\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.13.11.49\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.13.11.49\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.13.11.49\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.13.11.49%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.50': {'ec_num': '1.13.11.50',\n",
       "  'accepted_name': 'acetylacetone-cleaving enzyme',\n",
       "  'reaction': 'pentane-2,4-dione + O2 = acetate + 2-oxopropanal',\n",
       "  'other_names': 'Dke1; acetylacetone dioxygenase; diketone cleaving dioxygenase; diketone cleaving enzyme',\n",
       "  'sys_name': 'acetylacetone:oxygen oxidoreductase',\n",
       "  'comments': 'An iron(II)-dependent enzyme. Forms the first step in the acetylacetone degradation pathway of Acinetobacter johnsonii. While acetylacetone is by far the best substrate, heptane-3,5-dione, octane-2,4-dione, 2-acetylcyclohexanone and ethyl acetoacetate can also act as substrates.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '2-oxopropanal = methylglyoxal\\n',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.11.51': {'ec_num': '1.13.11.51',\n",
       "  'accepted_name': '9-cis-epoxycarotenoid dioxygenase',\n",
       "  'reaction': '(1) a 9-cis-epoxycarotenoid + O2 = 2-cis,4-trans-xanthoxin + a 12′-apo-carotenal;;(2) 9-cis-violaxanthin + O2 = 2-cis,4-trans-xanthoxin + (3S,5R,6S)-5,6-epoxy-3-hydroxy-5,6-dihydro-12′-apo-β-caroten-12′-al;;(3) 9′-cis-neoxanthin + O2 = 2-cis,4-trans-xanthoxin + (3S,5R,6R)-5,6-dihydroxy-6,7-didehydro-5,6-dihydro-12′-apo-β-caroten-12′-al',\n",
       "  'other_names': 'nine-cis-epoxycarotenoid dioxygenase; NCED; AtNCED3; PvNCED1; VP14',\n",
       "  'sys_name': '9-cis-epoxycarotenoid 11,12-dioxygenase',\n",
       "  'comments': 'Requires iron(II). Acts on 9-cis-violaxanthin and 9′-cis-neoxanthin but not on the all-trans isomers [2,3]. In vitro, it will cleave 9-cis-zeaxanthin. Catalyses the first step of abscisic-acid biosynthesis from carotenoids in chloroplasts, in response to water stress. The other enzymes involved in the abscisic-acid biosynthesis pathway are EC 1.1.1.288 (xanthoxin dehydrogenase), EC 1.2.3.14 (abscisic-aldehyde oxidase) and EC 1.14.13.93 [(+)-abscisic acid 8′-hydroxylase].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.12.1': {'ec_num': '1.13.12.1',\n",
       "  'accepted_name': 'arginine 2-monooxygenase',\n",
       "  'reaction': 'L-arginine + O2 = 4-guanidinobutanamide + CO2 + H2O',\n",
       "  'other_names': 'arginine monooxygenase; arginine decarboxylase (incorrect); arginine oxygenase (decarboxylating); arginine decarboxy-oxidase',\n",
       "  'sys_name': 'L-arginine:oxygen 2-oxidoreductase (decarboxylating)',\n",
       "  'comments': 'A flavoprotein. Also acts on canavanine and homoarginine.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.12.2': {'ec_num': '1.13.12.2',\n",
       "  'accepted_name': 'lysine 2-monooxygenase',\n",
       "  'reaction': 'L-lysine + O2 = 5-aminopentanamide + CO2 + H2O',\n",
       "  'other_names': 'lysine oxygenase; lysine monooxygenase; L-lysine-2-monooxygenase',\n",
       "  'sys_name': 'L-lysine:oxygen 2-oxidoreductase (decarboxylating)',\n",
       "  'comments': 'A flavoprotein (FAD). Also acts on other diamino acids.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.12.3': {'ec_num': '1.13.12.3',\n",
       "  'accepted_name': 'tryptophan 2-monooxygenase',\n",
       "  'reaction': 'L-tryptophan + O2 = (indol-3-yl)acetamide + CO2 + H2O',\n",
       "  'other_names': 'tms1 (gene name); iaaM (gene name)',\n",
       "  'sys_name': 'L-tryptophan:oxygen 2-oxidoreductase (decarboxylating)',\n",
       "  'comments': 'The enzyme, studied from phytopathogenic bacteria such as Pseudomonas savastanoi, is involved in a pathway for the production of (indol-3-yl)acetate (IAA), the main auxin hormone in plants.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.12.4': {'ec_num': '1.13.12.4',\n",
       "  'accepted_name': 'lactate 2-monooxygenase',\n",
       "  'reaction': '(S)-lactate + O2 = acetate + CO2 + H2O',\n",
       "  'other_names': 'lactate oxidative decarboxylase; lactate oxidase; lactic oxygenase; lactate oxygenase; lactic oxidase; L-lactate monooxygenase; lactate monooxygenase; L-lactate-2-monooxygenase',\n",
       "  'sys_name': '(S)-lactate:oxygen 2-oxidoreductase (decarboxylating)',\n",
       "  'comments': 'A flavoprotein (FMN).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.12.5': {'ec_num': '1.13.12.5',\n",
       "  'accepted_name': 'Renilla-type luciferase',\n",
       "  'reaction': 'coelenterazine h + O2 = excited coelenteramide h monoanion + CO2 (over-all reaction);;(1a) coelenterazine h + O2 = coelenterazine h dioxetanone;;(1b) coelenterazine h dioxetanone = excited coelenteramide h monoanion + CO2',\n",
       "  'other_names': 'Renilla-luciferin 2-monooxygenase; luciferase (Renilla luciferin); Renilla-luciferin:oxygen 2-oxidoreductase (decarboxylating)',\n",
       "  'sys_name': 'coelenterazine h:oxygen 2-oxidoreductase (decarboxylating)',\n",
       "  'comments': 'This enzyme has been studied from the soft coral Renilla reniformis. Before the reaction occurs the substrate is sequestered by a coelenterazine-binding protein. Elevation in the concentration of calcium ions releases the substrate, which then interacts with the luciferase. Upon binding the substrate, the enzyme catalyses an oxygenation, producing a very short-lived hydroperoxide that cyclizes into a dioxetanone structure, which collapses, releasing a CO2 molecule. The spontaneous breakdown of the dioxetanone releases the energy (about 50 kcal/mole) that is necessary to generate the excited state of the coelenteramide product, which is the singlet form of the monoanion. In vivo the product undergoes the process of nonradiative energy transfer to an accessory protein, a green fluorescent protein (GFP), which results in green bioluminescence. In vitro, in the absence of GFP, the product emits blue light.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'coelenterazine h = Renilla luciferin = 2,8-dibenzyl-6-(4-hydroxyphenyl)imidazo[1,2-a]pyrazin-3(7H)-one\\ncoelenteramide h = Renilla oxyluciferin = N-[3-benzyl-5-(4-hydroxyphenyl)pyrazin-2-yl]-2-phenylacetamide',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.12.6': {'ec_num': '1.13.12.6',\n",
       "  'accepted_name': 'Cypridina-luciferin 2-monooxygenase',\n",
       "  'reaction': 'Cypridina luciferin + O2 = oxidized Cypridina luciferin + CO2 + hν',\n",
       "  'other_names': 'Cypridina-type luciferase; luciferase (Cypridina luciferin); Cypridina luciferase',\n",
       "  'sys_name': 'Cypridina-luciferin:oxygen 2-oxidoreductase (decarboxylating)',\n",
       "  'comments': 'Cypridina is a bioluminescent crustacea. The luciferins (and presumably the luciferases, since they cross-react) of some luminous fish (e.g. Apogon, Parapriacanthus, Porichthys) are apparently similar. The enzyme may be assayed by measurement of light emission.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'Cypridina-luciferin = (3-{3,7-dihydro-6-(1H-indol-3-yl)-2-[(S)-1-methylpropyl]-3-oxoimidazo[1,2-a]pyrazin-8-yl}propyl)guanidine ',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.12.7': {'ec_num': '1.13.12.7',\n",
       "  'accepted_name': 'firefly luciferase',\n",
       "  'reaction': 'D-firefly luciferin + O2 + ATP = firefly oxyluciferin + CO2 + AMP + diphosphate + hν',\n",
       "  'other_names': 'Photinus-luciferin 4-monooxygenase (ATP-hydrolysing); luciferase (firefly luciferin); Photinus luciferin 4-monooxygenase (adenosine triphosphate-hydrolyzing); firefly luciferin luciferase; Photinus pyralis luciferase; Photinus-luciferin:oxygen 4-oxidoreductase (decarboxylating, ATP-hydrolysing)',\n",
       "  'sys_name': 'D-firefly luciferin:oxygen 4-oxidoreductase (decarboxylating, ATP-hydrolysing)',\n",
       "  'comments': 'The enzyme, which is found in fireflies (Lampyridae), is responsible for their biolouminescence. The reaction begins with the formation of an acid anhydride between the carboxylic group of D-firefly luciferin and AMP, with the release of diphosphate. An oxygenation follows, with release of the AMP group and formation of a very short-lived peroxide that cyclizes into a dioxetanone structure, which collapses, releasing a CO2 molecule. The spontaneous breakdown of the dioxetanone (rather than the hydrolysis of the adenylate) releases the energy (about 50 kcal/mole) that is necessary to generate the excited state of oxyluciferin. The excited luciferin then emits a photon, returning to its ground state. The enzyme has a secondary acyl-CoA ligase activity when acting on L-firefly luciferin (see EC 6.2.1.52).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'D-firefly luciferin = Photinus-luciferin = (S)-4,5-dihydro-2-(6-hydroxy-1,3-benzothiazol-2-yl)thiazole-4-carboxylate\\nfirefly oxyluciferin = 4,5-dihydro-2-(6-hydroxy-1,3-benzothiazol-2-yl)thiazol-4-one',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.12.8': {'ec_num': '1.13.12.8',\n",
       "  'accepted_name': 'Watasenia-luciferin 2-monooxygenase',\n",
       "  'reaction': 'Watasenia luciferin + O2 = oxidized Watasenia luciferin + CO2 + hν',\n",
       "  'other_names': 'Watasenia-type luciferase',\n",
       "  'sys_name': 'Watasenia-luciferin:oxygen 2-oxidoreductase (decarboxylating)',\n",
       "  'comments': 'The enzyme from the luminous squid Watasenia may be assayed by measurement of light emission.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.12.9': {'ec_num': '1.13.12.9',\n",
       "  'accepted_name': 'phenylalanine 2-monooxygenase',\n",
       "  'reaction': 'L-phenylalanine + O2 = 2-phenylacetamide + CO2 + H2O',\n",
       "  'other_names': 'L-phenylalanine oxidase (deaminating and decarboxylating); phenylalanine (deaminating, decarboxylating)oxidase',\n",
       "  'sys_name': 'L-phenylalanine:oxygen 2-oxidoreductase (decarboxylating)',\n",
       "  'comments': 'The reaction shown above is about 80% of the reaction catalysed; the remaining 20% is:<p> \\xa0\\xa0\\xa0\\xa0<small>L</small>-phenylalanine + O<small><sub>2</sub></small> + H<small><sub>2</sub></small>O = 3-phenylpyruvic acid + ammonia + H<small><sub>2</sub></small>O<small><sub>2</sub></small><p> a reaction similar to that of <a href=\"./query.php?ec=1.4.3.2\" target=\"new\">EC 1.4.3.2</a>, <small>L</small>-amino-acid oxidase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.12.11': {'ec_num': '1.13.12.11',\n",
       "  'accepted_name': 'methylphenyltetrahydropyridine N-monooxygenase',\n",
       "  'reaction': '1-methyl-4-phenyl-1,2,3,6-tetrahydropyridine + O2 = 1-methyl-4-phenyl-1,2,3,6-tetrahydropyridine N-oxide + methanol',\n",
       "  'other_names': '',\n",
       "  'sys_name': '1-methyl-4-phenyl-1,2,3,6-tetrahydropyridine:oxygen N-oxidoreductase',\n",
       "  'comments': 'A flavoprotein.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.12.12': {'ec_num': '1.13.12.12',\n",
       "  'accepted_name': 'apo-β-carotenoid-14′,13′-dioxygenase',\n",
       "  'reaction': '8′-apo-β-carotenol + O2 = 14′-apo-β-carotenal + 7-hydroxy-6-methylhepta-3,5-dienal',\n",
       "  'other_names': '',\n",
       "  'sys_name': '8′-apo-β-carotenol:O2 oxidoreductase',\n",
       "  'comments': 'A thiol-dependent enzyme. Unlike EC 1.14.99.36, β-carotene-15,15′-monooxygenase, it is not active towards β-carotene. Presumably, 2-methyl-6-oxohepta-2,4-dienal is also formed in this reaction.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.12.13': {'ec_num': '1.13.12.13',\n",
       "  'accepted_name': 'Oplophorus-luciferin 2-monooxygenase',\n",
       "  'reaction': 'Oplophorus luciferin + O2 = oxidized Oplophorus luciferin + CO2 + hν',\n",
       "  'other_names': 'Oplophorus luciferase',\n",
       "  'sys_name': 'Oplophorus-luciferin:oxygen 2-oxidoreductase (decarboxylating)',\n",
       "  'comments': 'The luciferase from the deep sea shrimp Oplophorus gracilirostris is a complex composed of more than one protein. The enzyme’s specificity is quite broad, with both coelenterazine and bisdeoxycoelenterazine being good substrates.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'Oplophorus luciferin = 8-benzyl-2-(4-hydroxybenzyl)-6-(4-hydroxyphenyl)imidazo[1,2-a]pyrazin-3(7H)-one',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.99.1': {'ec_num': '1.13.99.1',\n",
       "  'accepted_name': 'inositol oxygenase',\n",
       "  'reaction': 'myo-inositol + O2 = D-glucuronate + H2O',\n",
       "  'other_names': 'meso-inositol oxygenase; myo-inositol oxygenase; MOO',\n",
       "  'sys_name': 'myo-inositol:oxygen oxidoreductase',\n",
       "  'comments': 'An iron protein.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.13.99.3': {'ec_num': '1.13.99.3',\n",
       "  'accepted_name': 'tryptophan 2′-dioxygenase',\n",
       "  'reaction': 'L-tryptophan + O2 = (indol-3-yl)glycolaldehyde + CO2 + NH3',\n",
       "  'other_names': 'indole-3-alkane α-hydroxylase; tryptophan side-chain α,β-oxidase; tryptophan side chain oxidase II; tryptophan side-chain oxidase; TSO; indolyl-3-alkan α-hydroxylase; tryptophan side chain oxidase type I; TSO I ; TSO II; tryptophan side chain oxidase',\n",
       "  'sys_name': 'L-tryptophan:oxygen 2′-oxidoreductase (side-chain-cleaving)',\n",
       "  'comments': 'A hemoprotein. Acts on a number of indole-3-alkane derivatives, oxidizing the 3-side-chain in the 2′-position. Best substrates were L-tryptophan and 5-hydroxy-L-tryptophan.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.1': {'ec_num': '1.14.11.1',\n",
       "  'accepted_name': 'γ-butyrobetaine dioxygenase',\n",
       "  'reaction': '4-trimethylammoniobutanoate + 2-oxoglutarate + O2 = 3-hydroxy-4-trimethylammoniobutanoate + succinate + CO2',\n",
       "  'other_names': 'α-butyrobetaine hydroxylase; γ-butyrobetaine hydroxylase; butyrobetaine hydroxylase',\n",
       "  'sys_name': '4-trimethylammoniobutanoate,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'comments': 'Requires Fe2+ and ascorbate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.2': {'ec_num': '1.14.11.2',\n",
       "  'accepted_name': 'procollagen-proline 4-dioxygenase',\n",
       "  'reaction': 'procollagen L-proline + 2-oxoglutarate + O2 = procollagen trans-4-hydroxy-L-proline + succinate + CO2',\n",
       "  'other_names': 'P4HA (gene name); P4HB (gene name); protocollagen hydroxylase; proline hydroxylase; proline,2-oxoglutarate 4-dioxygenase; collagen proline hydroxylase; hydroxylase, collagen proline; peptidyl proline hydroxylase; proline protocollagen hydroxylase; proline, 2-oxoglutarate dioxygenase; prolyl hydroxylase; prolylprotocollagen dioxygenase; prolylprotocollagen hydroxylase; protocollagen proline 4-hydroxylase; protocollagen proline dioxygenase; protocollagen proline hydroxylase; protocollagen prolyl hydroxylase; prolyl 4-hydroxylase; prolyl-glycyl-peptide, 2-oxoglutarate:oxygen oxidoreductase, 4-hydroxylating; procollagen-proline 4-dioxygenase (ambiguous)',\n",
       "  'sys_name': 'procollagen-L-proline,2-oxoglutarate:oxygen oxidoreductase (4-hydroxylating)',\n",
       "  'comments': 'Requires Fe2+ and ascorbate.The enzyme, which is located within the lumen of the endoplasmic reticulum, catalyses the 4-hydroxylation of prolines in -X-Pro-Gly- sequences. The 4-hydroxyproline residues are essential for the formation of the collagen triple helix. The enzyme forms a complex with protein disulfide isomerase and acts not only on procollagen but also on more than 15 other proteins that have collagen-like domains.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.3': {'ec_num': '1.14.11.3',\n",
       "  'accepted_name': 'pyrimidine-deoxynucleoside 2′-dioxygenase',\n",
       "  'reaction': '2′-deoxyuridine + 2-oxoglutarate + O2 = uridine + succinate + CO2',\n",
       "  'other_names': 'deoxyuridine 2′-dioxygenase; deoxyuridine 2′-hydroxylase; pyrimidine deoxyribonucleoside 2′-hydroxylase; thymidine 2′-dioxygenase; thymidine 2′-hydroxylase; thymidine 2-oxoglutarate dioxygenase; thymidine dioxygenase ',\n",
       "  'sys_name': '2′-deoxyuridine,2-oxoglutarate:oxygen oxidoreductase (2′-hydroxylating)',\n",
       "  'comments': 'Requires iron(II) and ascorbate. Also acts on thymidine. cf. EC 1.14.11.10, pyrimidine-deoxynucleoside 1′-dioxygenase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.4': {'ec_num': '1.14.11.4',\n",
       "  'accepted_name': 'procollagen-lysine 5-dioxygenase',\n",
       "  'reaction': '[procollagen]-L-lysine + 2-oxoglutarate + O2 = [procollagen]-(2S,5R)-5-hydroxy-L-lysine + succinate + CO2',\n",
       "  'other_names': 'lysine hydroxylase; lysine,2-oxoglutarate 5-dioxygenase; protocollagen lysine dioxygenase; collagen lysine hydroxylase; lysine-2-oxoglutarate dioxygenase; lysyl hydroxylase; lysylprotocollagen dioxygenase; protocollagen lysyl hydroxylase; peptidyl-lysine, 2-oxoglutarate: oxygen oxidoreductase; peptidyllysine, 2-oxoglutarate:oxygen 5-oxidoreductase; protocollagen lysine hydroxylase; procollagen-L-lysine,2-oxoglutarate:oxygen oxidoreductase (5-hydroxylating); L-lysine-[procollagen],2-oxoglutarate:oxygen oxidoreductase (5-hydroxylating)',\n",
       "  'sys_name': '[procollagen]-L-lysine,2-oxoglutarate:oxygen oxidoreductase (5-hydroxylating)',\n",
       "  'comments': 'Requires Fe2+ and ascorbate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.6': {'ec_num': '1.14.11.6',\n",
       "  'accepted_name': 'thymine dioxygenase',\n",
       "  'reaction': 'thymine + 2-oxoglutarate + O2 = 5-hydroxymethyluracil + succinate + CO2',\n",
       "  'other_names': 'thymine 7-hydroxylase; 5-hydroxy-methyluracil dioxygenase; 5-hydroxymethyluracil oxygenase',\n",
       "  'sys_name': 'thymine,2-oxoglutarate:oxygen oxidoreductase (7-hydroxylating)',\n",
       "  'comments': 'Requires Fe2+ and ascorbate. Also acts on 5-hydroxymethyluracil to oxidize its -CH2OH group first to -CHO and then to -COOH.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.7': {'ec_num': '1.14.11.7',\n",
       "  'accepted_name': 'procollagen-proline 3-dioxygenase',\n",
       "  'reaction': '[procollagen]-L-proline + 2-oxoglutarate + O2 = [procollagen]-trans-3-hydroxy-L-proline + succinate + CO2',\n",
       "  'other_names': 'proline,2-oxoglutarate 3-dioxygenase; prolyl 3-hydroxylase; protocollagen proline 3-hydroxylase; prolyl-4-hydroxyprolyl-glycyl-peptide,2-oxoglutarate:oxygen oxidoreductase, 3-hydroxylating',\n",
       "  'sys_name': '[procollagen]-L-proline,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'comments': 'Requires Fe2+ and ascorbate. The enzyme forms a complex with protein disulfide isomerase, and is located in the endoplasmic reticulum. It modifies proline residues within the procollagen peptide of certain collagen types. The modification is essential for proper collagen triple helix formation. ',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.8': {'ec_num': '1.14.11.8',\n",
       "  'accepted_name': 'trimethyllysine dioxygenase',\n",
       "  'reaction': 'N6,N6,N6-trimethyl-L-lysine + 2-oxoglutarate + O2 = (3S)-3-hydroxy-N6,N6,N6-trimethyl-L-lysine + succinate + CO2',\n",
       "  'other_names': 'trimethyllysine α-ketoglutarate dioxygenase; TML-α-ketoglutarate dioxygenase; TML hydroxylase; 6-N,6-N,6-N-trimethyl-L-lysine,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'sys_name': 'N6,N6,N6-trimethyl-L-lysine,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'comments': 'Requires Fe2+ and ascorbate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.9': {'ec_num': '1.14.11.9',\n",
       "  'accepted_name': 'flavanone 3-dioxygenase',\n",
       "  'reaction': 'a (2S)-flavan-4-one + 2-oxoglutarate + O2 = a (2R,3R)-dihydroflavonol + succinate + CO2',\n",
       "  'other_names': 'naringenin 3-hydroxylase; flavanone 3-hydroxylase; flavanone 3β-hydroxylase; flavanone synthase I; (2S)-flavanone 3-hydroxylase; naringenin,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating); F3H; flavanone,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'sys_name': '(2S)-flavan-4-one,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'comments': 'Requires Fe2+ and ascorbate. This plant enzyme catalyses an early step in the flavonoid biosynthesis pathway, leading to the production of flavanols and anthocyanins. Substrates include (2S)-naringenin, (2S)-eriodictyol, (2S)-dihydrotricetin and (2S)-pinocembrin. Some enzymes are bifuctional and also catalyse EC 1.14.20.6, flavonol synthase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.10': {'ec_num': '1.14.11.10',\n",
       "  'accepted_name': 'pyrimidine-deoxynucleoside 1′-dioxygenase',\n",
       "  'reaction': '2′-deoxyuridine + 2-oxoglutarate + O2 = uracil + 2-deoxyribonolactone + succinate + CO2',\n",
       "  'other_names': 'deoxyuridine-uridine 1′-dioxygenase',\n",
       "  'sys_name': '2′-deoxyuridine,2-oxoglutarate:oxygen oxidoreductase (1′-hydroxylating)',\n",
       "  'comments': 'Requires iron(II) and ascorbate. cf. EC 1.14.11.3, pyrimidine-deoxynucleoside 2′-dioxygenase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.11': {'ec_num': '1.14.11.11',\n",
       "  'accepted_name': 'hyoscyamine (6S)-dioxygenase',\n",
       "  'reaction': 'L-hyoscyamine + 2-oxoglutarate + O2 = (6S)-hydroxyhyoscyamine + succinate + CO2',\n",
       "  'other_names': 'hyoscyamine 6β-hydroxylase; hyoscyamine 6β-dioxygenase; hyoscyamine 6-hydroxylase',\n",
       "  'sys_name': 'L-hyoscyamine,2-oxoglutarate:oxygen oxidoreductase [(6S)-hydroxylating]',\n",
       "  'comments': 'Requires Fe2+ and ascorbate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.12': {'ec_num': '1.14.11.12',\n",
       "  'accepted_name': 'gibberellin-44 dioxygenase',\n",
       "  'reaction': 'gibberellin 44 + 2-oxoglutarate + O2 = gibberellin 19 + succinate + CO2',\n",
       "  'other_names': 'oxygenase, gibberellin A44 oxidase; (gibberellin-44), 2-oxoglutarate:oxygen oxidoreductase',\n",
       "  'sys_name': '(gibberellin-44),2-oxoglutarate:oxygen oxidoreductase',\n",
       "  'comments': 'Requires Fe2+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.13': {'ec_num': '1.14.11.13',\n",
       "  'accepted_name': 'gibberellin 2β-dioxygenase',\n",
       "  'reaction': 'gibberellin 1 + 2-oxoglutarate + O2 = 2β-hydroxygibberellin 1 + succinate + CO2',\n",
       "  'other_names': 'gibberellin 2β-hydroxylase',\n",
       "  'sys_name': '(gibberellin-1),2-oxoglutarate:oxygen oxidoreductase (2β-hydroxylating)',\n",
       "  'comments': 'Also acts on a number of other gibberellins.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.14': {'ec_num': '1.14.11.14',\n",
       "  'accepted_name': '6β-hydroxyhyoscyamine epoxidase',\n",
       "  'reaction': '(6S)-6β-hydroxyhyoscyamine + 2-oxoglutarate + O2 = scopolamine + succinate + CO2 + H2O',\n",
       "  'other_names': 'hydroxyhyoscyamine dioxygenase; (6S)-6-hydroxyhyoscyamine,2-oxoglutarate oxidoreductase (epoxide-forming)',\n",
       "  'sys_name': '(6S)-6β-hydroxyhyoscyamine,2-oxoglutarate:oxygen oxidoreductase (epoxide-forming)',\n",
       "  'comments': 'Requires Fe2+ and ascorbate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'scopolamine = hyoscine',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.15': {'ec_num': '1.14.11.15',\n",
       "  'accepted_name': 'gibberellin 3β-dioxygenase',\n",
       "  'reaction': 'gibberellin 20 + 2-oxoglutarate + O2 = gibberellin 1 + succinate + CO2',\n",
       "  'other_names': 'gibberellin 3β-hydroxylase; (gibberrellin-20),2-oxoglutarate: oxygen oxidoreductase (3β-hydroxylating)',\n",
       "  'sys_name': '(gibberellin-20),2-oxoglutarate:oxygen oxidoreductase (3β-hydroxylating)',\n",
       "  'comments': 'Requires Fe2+ and ascorbate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.16': {'ec_num': '1.14.11.16',\n",
       "  'accepted_name': 'peptide-aspartate β-dioxygenase',\n",
       "  'reaction': 'peptide-L-aspartate + 2-oxoglutarate + O2 = peptide-3-hydroxy-L-aspartate + succinate + CO2',\n",
       "  'other_names': 'aspartate β-hydroxylase; aspartylpeptide β-dioxygenase',\n",
       "  'sys_name': 'peptide-L-aspartate,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'comments': 'Requires Fe2+. Some vitamin K-dependent coagulation factors, as well as synthetic peptides based on the structure of the first epidermal growth factor domain of human coagulation factor IX or X, can act as acceptors.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.17': {'ec_num': '1.14.11.17',\n",
       "  'accepted_name': 'taurine dioxygenase',\n",
       "  'reaction': 'taurine + 2-oxoglutarate + O2 = sulfite + aminoacetaldehyde + succinate + CO2',\n",
       "  'other_names': '2-aminoethanesulfonate dioxygenase; α-ketoglutarate-dependent taurine dioxygenase',\n",
       "  'sys_name': 'taurine, 2-oxoglutarate:oxygen oxidoreductase (sulfite-forming)',\n",
       "  'comments': 'Requires FeII. The enzyme from Escherichia coli also acts on pentanesulfonate, 3-(N-morpholino)propanesulfonate and 2-(1,3-dioxoisoindolin-2-yl)ethanesulfonate, but at lower rates.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.18': {'ec_num': '1.14.11.18',\n",
       "  'accepted_name': 'phytanoyl-CoA dioxygenase',\n",
       "  'reaction': 'phytanoyl-CoA + 2-oxoglutarate + O2 = 2-hydroxyphytanoyl-CoA + succinate + CO2',\n",
       "  'other_names': 'phytanoyl-CoA hydroxylase',\n",
       "  'sys_name': 'phytanoyl-CoA, 2-oxoglutarate:oxygen oxidoreductase (2-hydroxylating)',\n",
       "  'comments': 'Part of the peroxisomal phytanic acid α-oxidation pathway. Requires Fe2+ and ascorbate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'phytanate = 3,7,11,15-tetramethylhexadecanoate',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.19': {'ec_num': '1.14.11.19',\n",
       "  'accepted_name': 'anthocyanidin synthase',\n",
       "  'reaction': '  a (2R,3S,4S)-leucoanthocyanidin + 2-oxoglutarate + O2 = an anthocyanidin + succinate + CO2 + 2 H2O (overall reaction);;(1a) a (2R,3S,4S)-leucoanthocyanidin + 2-oxoglutarate + O2 = a (4S)- 2,3-dehydroflavan-3,4-diol + succinate + CO2 + H2O;;(1b) a (4S)- 2,3-dehydroflavan-3,4-diol = an anthocyanidin + H2O\\n  ',\n",
       "  'other_names': 'leucocyanidin oxygenase; leucocyanidin,2-oxoglutarate:oxygen oxidoreductase; ANS (gene name)',\n",
       "  'sys_name': '(2R,3S,4S)-leucoanthocyanidin,2-oxoglutarate:oxygen oxidoreductase',\n",
       "  'comments': 'The enzyme requires Fe(II) and ascorbate. It is involved in the pathway by which many flowering plants make anthocyanin flower pigments (glycosylated anthocyandins). The enzyme hydroxylates the C-3 carbon, followed by a trans diaxial elimination, forming a C-2,C-3 enol. The product loses a second water molecule to form anthocyanidins. When assayed in vitro, non-enzymic epimerization of the product can lead to formation of dihydroflavanols. Thus when the substrate is leucocyanidin, a mixture of (+)-taxifolin and (+)-epitaxifolin are formed. The enzyme can also oxidize the formed (+)-taxifolin to quercetin (cf. EC 1.14.11.23, flavonol synthase) [2,3].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'taxifolin = 3,4-dihydroquercitin',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.20': {'ec_num': '1.14.11.20',\n",
       "  'accepted_name': 'deacetoxyvindoline 4-hydroxylase',\n",
       "  'reaction': 'deacetoxyvindoline + 2-oxoglutarate + O2 = deacetylvindoline + succinate + CO2',\n",
       "  'other_names': 'desacetoxyvindoline 4-hydroxylase; desacetyoxyvindoline-17-hydroxylase; D17H; desacetoxyvindoline,2-oxoglutarate:oxygen oxidoreductase (4β-hydroxylating)',\n",
       "  'sys_name': 'deacetoxyvindoline,2-oxoglutarate:oxygen oxidoreductase (4β-hydroxylating)',\n",
       "  'comments': 'Requires Fe2+ and ascorbate. Also acts on 3-hydroxy-16-methoxy-2,3-dihydrotabersonine and to a lesser extent on 16-methoxy-2,3-dihydrotabersonine.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.21': {'ec_num': '1.14.11.21',\n",
       "  'accepted_name': 'clavaminate synthase',\n",
       "  'reaction': '(1) deoxyamidinoproclavaminate + 2-oxoglutarate + O2 = amidinoproclavaminate + succinate + CO2;;(2) proclavaminate + 2-oxoglutarate + O2 = dihydroclavaminate + succinate + CO2 + H2O;;(3) dihydroclavaminate + 2-oxoglutarate + O2 = clavaminate + succinate + CO2 + H2O',\n",
       "  'other_names': 'clavaminate synthase 2; clavaminic acid synthase',\n",
       "  'sys_name': 'deoxyamidinoproclavaminate,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'comments': 'Contains nonheme iron. Catalyses three separate oxidative reactions in the pathway for the biosythesis of the β-lactamase inhibitor clavulanate in Streptomyces clavuligerus. The first step (hydroxylation) is separated from the latter two (oxidative cyclization and desaturation) by the action of EC 3.5.3.22, proclavaminate amidinohydrolase. The three reactions are all catalysed at the same nonheme iron site.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.22': {'ec_num': '1.14.11.22',\n",
       "  'accepted_name': 'flavone synthase',\n",
       "  'reaction': 'a flavanone + 2-oxoglutarate + O2 = a flavone + succinate + CO2 + H2O',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'flavanone,2-oxoglutarate:oxygen oxidoreductase (dehydrating)',\n",
       "  'comments': 'Requires ascorbate for full activity and Fe2+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.23': {'ec_num': '1.14.11.23',\n",
       "  'accepted_name': 'flavonol synthase',\n",
       "  'reaction': 'a dihydroflavonol + 2-oxoglutarate + O2 = a flavonol + succinate + CO2 + H2O',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'dihydroflavonol,2-oxoglutarate:oxygen oxidoreductase',\n",
       "  'comments': 'In addition to the desaturation of (2R,3R)-dihydroflavonols to flavonols, the enzyme from the satsuma Citrus unshiu also has a non-specific activity that trans-hydroxylates the flavanones (2S)-naringenin and the unnatural (2R)-naringenin at C-3 to kaempferol and (2R,3R)-dihydrokaempferol, respectively [2]. Requires Fe2+.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.24': {'ec_num': '1.14.11.24',\n",
       "  'accepted_name': '2′-deoxymugineic-acid 2′-dioxygenase',\n",
       "  'reaction': '2′-deoxymugineic acid + 2-oxoglutarate + O2 = mugineic acid + succinate + CO2',\n",
       "  'other_names': 'IDS3',\n",
       "  'sys_name': '2′-deoxymugineic acid,2-oxoglutarate:oxygen oxidoreductase (2-hydroxylating)',\n",
       "  'comments': 'Requires iron(II). It is also likely that this enzyme can catalyse the hydroxylation of 3-epihydroxy-2′-deoxymugineic acid to form 3-epihydroxymugineic acid.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.25': {'ec_num': '1.14.11.25',\n",
       "  'accepted_name': 'mugineic-acid 3-dioxygenase',\n",
       "  'reaction': '(1) mugineic acid + 2-oxoglutarate + O2 = 3-epihydroxymugineic acid + succinate + CO2;;(2) 2′-deoxymugineic acid + 2-oxoglutarate + O2 = 3-epihydroxy-2′-deoxymugineic acid + succinate + CO2',\n",
       "  'other_names': 'IDS2',\n",
       "  'sys_name': 'mugineic acid,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'comments': 'Requires iron(II).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.11.26': {'ec_num': '1.14.11.26',\n",
       "  'accepted_name': 'deacetoxycephalosporin-C hydroxylase',\n",
       "  'reaction': 'deacetoxycephalosporin C + 2-oxoglutarate + O2 = deacetylcephalosporin C + succinate + CO2',\n",
       "  'other_names': 'deacetylcephalosporin C synthase; 3′-methylcephem hydroxylase; DACS; DAOC hydroxylase; deacetoxycephalosporin C hydroxylase',\n",
       "  'sys_name': 'deacetoxycephalosporin-C,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'comments': 'Requires iron(II). The enzyme can also use 3-exomethylenecephalosporin C as a substrate to form deacetoxycephalosporin C, although more slowly [2]. In Acremonium chrysogenum, the enzyme forms part of a bifunctional protein along with EC 1.14.20.1, deactoxycephalosporin-C synthase. It is a separate enzyme in Streptomyces clavuligerus. ',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:22'},\n",
       " '1.14.12.1': {'ec_num': '1.14.12.1',\n",
       "  'accepted_name': 'anthranilate 1,2-dioxygenase (deaminating, decarboxylating)',\n",
       "  'reaction': 'anthranilate + NAD(P)H + 2 H+ + O2 = catechol + CO2 + NAD(P)+ + NH3',\n",
       "  'other_names': 'anthranilate hydroxylase; anthranilic hydroxylase; anthranilic acid hydroxylase',\n",
       "  'sys_name': 'anthranilate,NAD(P)H:oxygen oxidoreductase (1,2-hydroxylating, deaminating, decarboxylating)',\n",
       "  'comments': 'Requires Fe2+.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.12.1\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.12.1\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.12.1\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.12.1\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.12.3': {'ec_num': '1.14.12.3',\n",
       "  'accepted_name': 'benzene 1,2-dioxygenase',\n",
       "  'reaction': 'benzene + NADH + H+ + O2 = cis-cyclohexa-3,5-diene-1,2-diol + NAD+',\n",
       "  'other_names': 'benzene hydroxylase; benzene dioxygenase',\n",
       "  'sys_name': 'benzene,NADH:oxygen oxidoreductase (1,2-hydroxylating)',\n",
       "  'comments': 'A system, containing a reductase which is an iron-sulfur flavoprotein (FAD), an iron-sulfur oxygenase and ferredoxin. Requires Fe2+.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.12.3\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.12.3\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.12.3\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.12.3\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.12.3%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.12.4': {'ec_num': '1.14.12.4',\n",
       "  'accepted_name': '3-hydroxy-2-methylpyridinecarboxylate dioxygenase',\n",
       "  'reaction': '3-hydroxy-2-methylpyridine-5-carboxylate + NAD(P)H + H+ + O2 = 2-(acetamidomethylene)succinate + NAD(P)+',\n",
       "  'other_names': 'methylhydroxypyridinecarboxylate oxidase; 2-methyl-3-hydroxypyridine 5-carboxylic acid dioxygenase; methylhydroxypyridine carboxylate dioxygenase; 3-hydroxy-3-methylpyridinecarboxylate dioxygenase [incorrect]; 3-hydroxy-2-methylpyridine-5-carboxylate,NAD(P)H:oxygen oxidoreductase (decyclizing)',\n",
       "  'sys_name': '3-hydroxy-2-methylpyridine-5-carboxylate,NAD(P)H:oxygen oxidoreductase (ring-opening)',\n",
       "  'comments': 'A flavoprotein (FAD).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.12.4\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.12.4\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.12.4\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.12.4\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.12.5': {'ec_num': '1.14.12.5',\n",
       "  'accepted_name': '5-pyridoxate dioxygenase',\n",
       "  'reaction': '3-hydroxy-4-hydroxymethyl-2-methylpyridine-5-carboxylate + NADPH + H+ + O2 = 2-(acetamidomethylene)-3-(hydroxymethyl)succinate + NADP+',\n",
       "  'other_names': '5-pyridoxate oxidase; 5-pyridoxate,NADPH:oxygen oxidoreductase (decyclizing)',\n",
       "  'sys_name': '5-pyridoxate,NADPH:oxygen oxidoreductase (ring-opening)',\n",
       "  'comments': 'A flavoprotein.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.12.5\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.12.5\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.12.5\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.12.5\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.12.7': {'ec_num': '1.14.12.7',\n",
       "  'accepted_name': 'phthalate 4,5-dioxygenase',\n",
       "  'reaction': 'phthalate + NADH + H+ + O2 = cis-4,5-dihydroxycyclohexa-1(6),2-diene-1,2-dicarboxylate + NAD+',\n",
       "  'other_names': 'PDO ; phthalate dioxygenase',\n",
       "  'sys_name': 'phthalate,NADH:oxygen oxidoreductase (4,5-hydroxylating)',\n",
       "  'comments': 'A system, containing a reductase which is an iron-sulfur flavoprotein (FMN), an iron-sulfur oxygenase, and no independent ferredoxin. Requires Fe2+.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.12.7\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.12.7\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.12.7\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.12.7\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.12.7%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.12.8': {'ec_num': '1.14.12.8',\n",
       "  'accepted_name': '4-sulfobenzoate 3,4-dioxygenase',\n",
       "  'reaction': '4-sulfobenzoate + NADH + H+ + O2 = 3,4-dihydroxybenzoate + sulfite + NAD+',\n",
       "  'other_names': '4-sulfobenzoate dioxygenase; 4-sulfobenzoate 3,4-dioxygenase system',\n",
       "  'sys_name': '4-sulfobenzoate,NADH:oxygen oxidoreductase (3,4-hydroxylating, sulfite-forming) ',\n",
       "  'comments': 'A system, containing a reductase which is an iron-sulfur flavoprotein (FMN), an iron-sulfur oxygenase, and no independent ferredoxin. Requires Fe2+.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.12.8\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.12.8\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.12.8\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.12.8\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.12.9': {'ec_num': '1.14.12.9',\n",
       "  'accepted_name': '4-chlorophenylacetate 3,4-dioxygenase',\n",
       "  'reaction': '4-chlorophenylacetate + NADH + H+ + O2 = 3,4-dihydroxyphenylacetate + chloride + NAD+',\n",
       "  'other_names': '',\n",
       "  'sys_name': '4-chlorophenylacetate,NADH:oxygen oxidoreductase (3,4-hydroxylating, dechlorinating)',\n",
       "  'comments': 'A system, containing a reductase and an iron-sulfur oxygenase, and no independent ferredoxin. Requires Fe2+. Also acts on 4-bromophenyl acetate.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.12.9\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.12.9\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.12.9\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.12.9\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.12.10': {'ec_num': '1.14.12.10',\n",
       "  'accepted_name': 'benzoate 1,2-dioxygenase',\n",
       "  'reaction': 'benzoate + NADH + H+ + O2 = (1R,6S)-1,6-dihydroxycyclohexa-2,4-diene-1-carboxylate + NAD+',\n",
       "  'other_names': 'benzoate hydroxylase; benzoate hydroxylase; benzoic hydroxylase; benzoate dioxygenase; benzoate,NADH:oxygen oxidoreductase (1,2-hydroxylating, decarboxylating) [incorrect]',\n",
       "  'sys_name': 'benzoate,NADH:oxygen oxidoreductase (1,2-hydroxylating)',\n",
       "  'comments': 'A system, containing a reductase which is an iron-sulfur flavoprotein (FAD), and an iron-sulfur oxygenase. Requires Fe2+.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.12.10\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.12.10\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.12.10\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.12.10\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.12.10%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.12.11': {'ec_num': '1.14.12.11',\n",
       "  'accepted_name': 'toluene dioxygenase',\n",
       "  'reaction': 'toluene + NADH + H+ + O2 = (1S,2R)-3-methylcyclohexa-3,5-diene-1,2-diol + NAD+',\n",
       "  'other_names': 'toluene 2,3-dioxygenase',\n",
       "  'sys_name': 'toluene,NADH:oxygen oxidoreductase (1,2-hydroxylating)',\n",
       "  'comments': 'A system, containing a reductase which is an iron-sulfur flavoprotein (FAD), an iron-sulfur oxygenase, and a ferredoxin. Some other aromatic compounds, including ethylbenzene, 4-xylene and some halogenated toluenes, are converted into the corresponding cis-dihydrodiols.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.12.11\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.12.11\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.12.11\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.12.11\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.12.11%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.12.12': {'ec_num': '1.14.12.12',\n",
       "  'accepted_name': 'naphthalene 1,2-dioxygenase',\n",
       "  'reaction': 'naphthalene + NADH + H+ + O2 = (1R,2S)-1,2-dihydronaphthalene-1,2-diol + NAD+',\n",
       "  'other_names': 'naphthalene dioxygenase; naphthalene oxygenase; NDO',\n",
       "  'sys_name': 'naphthalene,NADH:oxygen oxidoreductase (1,2-hydroxylating)',\n",
       "  'comments': 'This enzyme is a member of the ring-hydroxylating dioxygenase (RHD) family of bacterial enzymes that play a critical role in the degradation of aromatic compounds, such as polycyclic aromatic hydrocarbons [5]. This enzyme comprises a multicomponent system, containing a reductase that is an iron-sulfur flavoprotein (FAD; EC 1.18.1.3, ferredoxin—NAD+ reductase), an iron-sulfur oxygenase, and ferredoxin. Requires Fe2+.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.12.12\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.12.12\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.12.12\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.12.12\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.12.12%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.12.13': {'ec_num': '1.14.12.13',\n",
       "  'accepted_name': '2-halobenzoate 1,2-dioxygenase',\n",
       "  'reaction': 'a 2-halobenzoate + NADH + H+ + O2 = catechol + a halide anion + NAD+ + CO2',\n",
       "  'other_names': '2-chlorobenzoate 1,2-dioxygenase',\n",
       "  'sys_name': '2-halobenzoate,NADH:oxygen oxidoreductase (1,2-hydroxylating, dehalogenating, decarboxylating)',\n",
       "  'comments': 'A multicomponent enzyme system composed of a dioxygenase component and an electron transfer component. The latter contains FAD. The enzyme, characterized from the bacterium Burkholderia cepacia 2CBS, has a broad substrate specificity. Substrates include 2-fluorobenzoate, 2-chlorobenzoate, 2-bromobenzoate, and 2-iodobenzoate, which are processed in this order of preference.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.12.13\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.12.13\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.12.13\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.12.13\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.12.14': {'ec_num': '1.14.12.14',\n",
       "  'accepted_name': '2-aminobenzenesulfonate 2,3-dioxygenase',\n",
       "  'reaction': '2-aminobenzenesulfonate + NADH + H+ + O2 = 2,3-dihydroxybenzenesulfonate + NH3 + NAD+',\n",
       "  'other_names': '2-aminosulfobenzene 2,3-dioxygenase',\n",
       "  'sys_name': '2-aminobenzenesulfonate,NADH:oxygen oxidoreductase (2,3-hydroxylating, ammonia-forming)',\n",
       "  'comments': '',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.12.14\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.12.14\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.12.14\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.12.14\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.12.15': {'ec_num': '1.14.12.15',\n",
       "  'accepted_name': 'terephthalate 1,2-dioxygenase',\n",
       "  'reaction': 'terephthalate + NADH + H+ + O2 = (1R,6S)-dihydroxycyclohexa-2,4-diene-1,4-dicarboxylate + NAD+',\n",
       "  'other_names': 'benzene-1,4-dicarboxylate 1,2-dioxygenase; 1,4-dicarboxybenzoate 1,2-dioxygenase',\n",
       "  'sys_name': 'benzene-1,4-dicarboxylate,NADH:oxygen oxidoreductase (1,2-hydroxylating)',\n",
       "  'comments': 'Has been shown to contain a Rieske [2Fe-2S] cluster',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.12.15\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.12.15\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.12.15\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.12.15\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.12.15%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.12.16': {'ec_num': '1.14.12.16',\n",
       "  'accepted_name': '2-hydroxyquinoline 5,6-dioxygenase',\n",
       "  'reaction': 'quinolin-2-ol + NADH + H+ + O2 = 2,5,6-trihydroxy-5,6-dihydroquinoline + NAD+',\n",
       "  'other_names': '2-oxo-1,2-dihydroquinoline 5,6-dioxygenase; quinolin-2-ol 5,6-dioxygenase; quinolin-2(1H)-one 5,6-dioxygenase',\n",
       "  'sys_name': 'quinolin-2-ol,NADH:oxygen oxidoreductase (5,6-hydroxylating) ',\n",
       "  'comments': '3-Methylquinolin-2-ol, quinolin-8-ol and quinolin-2,8-diol are also substrates. Quinolin-2-ols exist largely as their quinolin-2(1H)-one tautomers',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.12.16\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.12.16\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.12.16\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.12.16\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.12.17': {'ec_num': '1.14.12.17',\n",
       "  'accepted_name': 'nitric oxide dioxygenase',\n",
       "  'reaction': '2 nitric oxide + 2 O2 + NAD(P)H = 2 nitrate + NAD(P)+ + H+',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'nitric oxide,NAD(P)H:oxygen oxidoreductase',\n",
       "  'comments': 'A flavohemoglobin (FAD). It has been proposed that FAD functions as the electron carrier from NADPH to the ferric heme cofactor',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'nitric oxide = NO = nitrogen(II) oxide',\n",
       "  'last_change': '2023-05-19 09:09:05'},\n",
       " '1.14.12.18': {'ec_num': '1.14.12.18',\n",
       "  'accepted_name': 'biphenyl 2,3-dioxygenase',\n",
       "  'reaction': 'biphenyl + NADH + H+ + O2 = (1S,2R)-3-phenylcyclohexa-3,5-diene-1,2-diol + NAD+',\n",
       "  'other_names': 'biphenyl dioxygenase',\n",
       "  'sys_name': 'biphenyl,NADH:oxygen oxidoreductase (2,3-hydroxylating)',\n",
       "  'comments': 'Requires Fe2+. The enzyme from Burkholderia fungorum LB400 (previously Pseudomonas sp.) is part of a multicomponent system composed of an NADH:ferredoxin oxidoreductase (FAD cofactor), a [2Fe-2S] Rieske-type ferredoxin, and a terminal oxygenase that contains a [2Fe-2S] Rieske-type iron-sulfur cluster and a catalytic mononuclear nonheme iron centre. Chlorine-substituted biphenyls can also act as substrates. Similar to the three-component enzyme systems EC 1.14.12.3 (benzene 1,2-dioxygenase) and EC 1.14.12.11 (toluene dioxygenase).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.12.18\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.12.18\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.12.18\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.12.18\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.12.18%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.12.19': {'ec_num': '1.14.12.19',\n",
       "  'accepted_name': '3-phenylpropanoate dioxygenase',\n",
       "  'reaction': '(1) 3-phenylpropanoate + NADH + H+ + O2 = 3-(cis-5,6-dihydroxycyclohexa-1,3-dien-1-yl)propanoate + NAD+;;(2) (2E)-3-phenylprop-2-enoate + NADH + H+ + O2 = (2E)-3-(2,3-dihydroxyphenyl)prop-2-enoate + NAD+',\n",
       "  'other_names': 'HcaA1A2CD; Hca dioxygenase; 3-phenylpropionate dioxygenase',\n",
       "  'sys_name': '3-phenylpropanoate,NADH:oxygen oxidoreductase (2,3-hydroxylating)',\n",
       "  'comments': 'This enzyme catalyses a step in the pathway of phenylpropanoid compounds degradation. It catalyses the insertion of both atoms of molecular oxygen into positions 2 and 3 of the phenyl ring of 3-phenylpropanoate or (2E)-3-phenylprop-2-enoate.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.12.19\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.12.19\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.12.19\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.12.19\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '(2E)-3-phenylprop-2-enoate = trans-cinnamate\\n(2E)-3-(2,3-dihydroxyphenyl)prop-2-enoate = trans-2,3-dihydroxycinnamate',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.1': {'ec_num': '1.14.13.1',\n",
       "  'accepted_name': 'salicylate 1-monooxygenase',\n",
       "  'reaction': 'salicylate + NADH + 2 H+ + O2 = catechol + NAD+ + H2O + CO2',\n",
       "  'other_names': 'salicylate hydroxylase; salicylate 1-hydroxylase; salicylate monooxygenase; salicylate hydroxylase (decarboxylating)',\n",
       "  'sys_name': 'salicylate,NADH:oxygen oxidoreductase (1-hydroxylating, decarboxylating)',\n",
       "  'comments': 'A flavoprotein (FAD).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.1\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.1\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.1\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.1\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.13.1%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.2': {'ec_num': '1.14.13.2',\n",
       "  'accepted_name': '4-hydroxybenzoate 3-monooxygenase',\n",
       "  'reaction': '4-hydroxybenzoate + NADPH + H+ + O2 = 3,4-dihydroxybenzoate + NADP+ + H2O',\n",
       "  'other_names': 'p-hydroxybenzoate hydrolyase; p-hydroxybenzoate hydroxylase; 4-hydroxybenzoate 3-hydroxylase; 4-hydroxybenzoate monooxygenase; 4-hydroxybenzoic hydroxylase; p-hydroxybenzoate-3-hydroxylase; p-hydroxybenzoic acid hydrolase; p-hydroxybenzoic acid hydroxylase; p-hydroxybenzoic hydroxylase',\n",
       "  'sys_name': '4-hydroxybenzoate,NADPH:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'comments': 'A flavoprotein (FAD). Most enzymes from Pseudomonas are highly specific for NADPH (cf. EC 1.14.13.33 4-hydroxybenzoate 3-monooxygenase [NAD(P)H]).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.2\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.2\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.2\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.2\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.13.2%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '3,4-dihydroxybenzoate = protocatechuate ',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.3': {'ec_num': '1.14.13.3',\n",
       "  'accepted_name': '4-hydroxyphenylacetate 3-monooxygenase',\n",
       "  'reaction': '4-hydroxyphenylacetate + NADH + H+ + O2 = 3,4-dihydroxyphenylacetate + NAD+ + H2O',\n",
       "  'other_names': 'p-hydroxyphenylacetate 3-hydroxylase; 4-hydroxyphenylacetic acid-3-hydroxylase; p-hydroxyphenylacetate hydroxylase; 4 HPA 3-hydroxylase',\n",
       "  'sys_name': '4-hydroxyphenylacetate,NADH:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'comments': 'A flavoprotein.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.3\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.3\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.3\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.3\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.4': {'ec_num': '1.14.13.4',\n",
       "  'accepted_name': 'melilotate 3-monooxygenase',\n",
       "  'reaction': '3-(2-hydroxyphenyl)propanoate + NADH + H+ + O2 = 3-(2,3-dihydroxyphenyl)propanoate + NAD+ + H2O',\n",
       "  'other_names': '2-hydroxyphenylpropionate hydroxylase; melilotate hydroxylase; 2-hydroxyphenylpropionic hydroxylase; melilotic hydroxylase',\n",
       "  'sys_name': '3-(2-hydroxyphenyl)propanoate,NADH:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'comments': 'A flavoprotein (FAD).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.5': {'ec_num': '1.14.13.5',\n",
       "  'accepted_name': 'imidazoleacetate 4-monooxygenase',\n",
       "  'reaction': '4-imidazoleacetate + NADH + H+ + O2 = 5-hydroxy-4-imidazoleacetate + NAD+ + H2O',\n",
       "  'other_names': 'imidazoleacetic hydroxylase; imidazoleacetate hydroxylase; imidazoleacetic monooxygenase',\n",
       "  'sys_name': '4-imidazoleacetate,NADH:oxygen oxidoreductase (5-hydroxylating)',\n",
       "  'comments': 'A flavoprotein (FAD).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.6': {'ec_num': '1.14.13.6',\n",
       "  'accepted_name': 'orcinol 2-monooxygenase',\n",
       "  'reaction': 'orcinol + NADH + H+ + O2 = 2,3,5-trihydroxytoluene + NAD+ + H2O',\n",
       "  'other_names': 'orcinol hydroxylase',\n",
       "  'sys_name': 'orcinol,NADH:oxygen oxidoreductase (2-hydroxylating)',\n",
       "  'comments': 'A flavoprotein (FAD).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.6\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.6\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.6\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.6\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.7': {'ec_num': '1.14.13.7',\n",
       "  'accepted_name': 'phenol 2-monooxygenase (NADPH)',\n",
       "  'reaction': 'phenol + NADPH + H+ + O2 = catechol + NADP+ + H2O',\n",
       "  'other_names': 'phenol hydroxylase; phenol o-hydroxylase',\n",
       "  'sys_name': 'phenol,NADPH:oxygen oxidoreductase (2-hydroxylating)',\n",
       "  'comments': 'A flavoprotein (FAD). The enzyme from the fungus Trichosporon cutaneum has a broad substrate specificity, and has been reported to catalyse the hydroxylation of a variety of substituted phenols, such as fluoro-, chloro-, amino- and methyl-phenols and also dihydroxybenzenes. cf. EC 1.14.14.20, phenol 2-monooxygenase (FADH2).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.7\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.7\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.7\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.7\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.13.7%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': 'o-cresol = 2-cresol = 2-methylphenol',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.8': {'ec_num': '1.14.13.8',\n",
       "  'accepted_name': 'flavin-containing monooxygenase',\n",
       "  'reaction': 'N,N-dimethylaniline + NADPH + H+ + O2 = N,N-dimethylaniline N-oxide + NADP+ + H2O',\n",
       "  'other_names': 'dimethylaniline oxidase; dimethylaniline N-oxidase; FAD-containing monooxygenase; N,N-dimethylaniline monooxygenase; DMA oxidase; flavin mixed function oxidase; Ziegler’s enzyme; mixed-function amine oxidase; FMO; FMO-I; FMO-II; FMO1; FMO2; FMO3; FMO4; FMO5; flavin monooxygenase; methylphenyltetrahydropyridine N-monooxygenase; 1-methyl-4-phenyl-1,2,3,6-tetrahydropyridine:oxygen N-oxidoreductase; dimethylaniline monooxygenase (N-oxide-forming)',\n",
       "  'sys_name': 'N,N-dimethylaniline,NADPH:oxygen oxidoreductase (N-oxide-forming)',\n",
       "  'comments': 'A flavoprotein. A broad spectrum monooxygenase that accepts substrates as diverse as hydrazines, phosphines, boron-containing compounds, sulfides, selenides, iodide, as well as primary, secondary and tertiary amines [3,4]. This enzyme is distinct from other monooxygenases in that the enzyme forms a relatively stable hydroperoxy flavin intermediate [4,5]. This microsomal enzyme generally converts nucleophilic heteroatom-containing chemicals and drugs into harmless, readily excreted metabolites. For example, N-oxygenation is largely responsible for the detoxification of the dopaminergic neurotoxin 1-methyl-4-phenyl-1,2,3,6-tetrahydropyridine (MPTP)  [2,6]',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.9': {'ec_num': '1.14.13.9',\n",
       "  'accepted_name': 'kynurenine 3-monooxygenase',\n",
       "  'reaction': 'L-kynurenine + NADPH + H+ + O2 = 3-hydroxy-L-kynurenine + NADP+ + H2O',\n",
       "  'other_names': 'kynurenine 3-hydroxylase; kynurenine hydroxylase; L-kynurenine-3-hydroxylase',\n",
       "  'sys_name': 'L-kynurenine,NADPH:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'comments': 'A flavoprotein (FAD).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.10': {'ec_num': '1.14.13.10',\n",
       "  'accepted_name': '2,6-dihydroxypyridine 3-monooxygenase',\n",
       "  'reaction': '2,6-dihydroxypyridine + NADH + H+ + O2 = 2,3,6-trihydroxypyridine + NAD+ + H2O',\n",
       "  'other_names': '2,6-dihydroxypyridine oxidase',\n",
       "  'sys_name': '2,6-dihydroxypyridine,NADH:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'comments': 'A flavoprotein.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.10\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.10\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.10\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.10\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.13.10%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.11': {'ec_num': '1.14.13.11',\n",
       "  'accepted_name': 'trans-cinnamate 4-monooxygenase',\n",
       "  'reaction': 'trans-cinnamate + NADPH + H+ + O2 = 4-hydroxycinnamate + NADP+ + H2O',\n",
       "  'other_names': 'cinnamic acid 4-hydroxylase; oxygenase, cinnamate 4-mono-; CA4H; cytochrome P450 cinnamate 4-hydroxylase; cinnamate 4-hydroxylase; cinnamate 4-monooxygenase; cinnamate hydroxylase; cinnamic 4-hydroxylase; cinnamic acid 4-monooxygenase; cinnamic acid p-hydroxylase; hydroxylase, cinnamate 4-; t-cinnamic acid hydroxylase; trans-cinnamate 4-hydroxylase; trans-cinnamic acid 4-hydroxylase',\n",
       "  'sys_name': 'trans-cinnamate,NADPH:oxygen oxidoreductase (4-hydroxylating)',\n",
       "  'comments': 'NADH also acts, more slowly. Involves a heme-thiolate protein (P-450).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.12': {'ec_num': '1.14.13.12',\n",
       "  'accepted_name': 'benzoate 4-monooxygenase',\n",
       "  'reaction': 'benzoate + NADPH + H+ + O2 = 4-hydroxybenzoate + NADP+ + H2O',\n",
       "  'other_names': 'benzoic acid 4-hydroxylase; benzoate 4-hydroxylase; benzoic 4-hydroxylase; benzoate-p-hydroxylase; p-hydroxybenzoate hydroxylase',\n",
       "  'sys_name': 'benzoate,NADPH:oxygen oxidoreductase (4-hydroxylating)',\n",
       "  'comments': 'Requires Fe2+ and tetrahydropteridine.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.12\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.12\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.12\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.12\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.13': {'ec_num': '1.14.13.13',\n",
       "  'accepted_name': 'calcidiol 1-monooxygenase',\n",
       "  'reaction': 'calcidiol + NADPH + H+ + O2 = calcitriol + NADP+ + H2O',\n",
       "  'other_names': '25-hydroxycholecalciferol 1-hydroxylase; 25-hydroxycholecalciferol 1-monooxygenase; 1-hydroxylase-25-hydroxyvitamin D3; 25-hydroxy D3-1α-hydroxylase; 25-hydroxycholecalciferol 1α-hydroxylase; 25-hydroxyvitamin D3 1α-hydroxylase',\n",
       "  'sys_name': 'calcidiol,NADPH:oxygen oxidoreductase (1-hydroxylating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'calcidiol = 25-hydroxyvitamin D3 = (3S,5Z,7E)-9,10-seco-5,7,10(19)-cholestatriene-3,25-diol\\ncalcitriol = 1α,25-dihydroxyvitamin D3 = (1S,3R,5Z,7E)-9,10-seco-5,7,10(19)-cholestatriene-1,3,25-triol\\n',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.14': {'ec_num': '1.14.13.14',\n",
       "  'accepted_name': 'trans-cinnamate 2-monooxygenase',\n",
       "  'reaction': 'trans-cinnamate + NADPH + H+ + O2 = 2-hydroxycinnamate + NADP+ + H2O',\n",
       "  'other_names': 'cinnamic acid 2-hydroxylase; cinnamate 2-monooxygenase; cinnamic 2-hydroxylase; cinnamate 2-hydroxylase; trans-cinnamic acid 2-hydroxylase',\n",
       "  'sys_name': 'trans-cinnamate,NADPH:oxygen oxidoreductase (2-hydroxylating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.15': {'ec_num': '1.14.13.15',\n",
       "  'accepted_name': 'cholestanetriol 26-monooxygenase',\n",
       "  'reaction': '5β-cholestane-3α,7α,12α-triol + 3 NADPH + 3 H+ + 3 O2 = (25R)-3α,7α,12α-trihydroxy-5β-cholestan-26-oate + 3 NADP+ + 4 H2O (overall reaction);;(1a) 5β-cholestane-3α,7α,12α-triol + NADPH + H+ + O2 = (25R)-5β-cholestane-3α,7α,12α,26-tetraol + NADP+ + H2O;;(1b) (25R)-5β-cholestane-3α,7α,12α,26-tetraol + NADPH + H+ + O2 = (25R)-3α,7α,12α-trihydroxy-5β-cholestan-26-al + NADP+ + 2 H2O;;(1c) (25R)-3α,7α,12α-trihydroxy-5β-cholestan-26-al + NADPH + H+ + O2 = (25R)-3α,7α,12α-trihydroxy-5β-cholestan-26-oate + NADP+ + H2O',\n",
       "  'other_names': '5β-cholestane-3α,7α,12α-triol 26-hydroxylase; 5β-cholestane-3α,7α,12α-triol hydroxylase; cholestanetriol 26-hydroxylase; sterol 27-hydroxylase; sterol 26-hydroxylase; cholesterol 27-hydroxylase; CYP27A; CYP27A1; cytochrome P450 27A1′',\n",
       "  'sys_name': '5β-cholestane-3α,7α,12α-triol,NADPH:oxygen oxidoreductase (26-hydroxylating)',\n",
       "  'comments': 'The enzyme requires ferredoxin and ferredoxin reductase. It catalyses the first three sterol side chain oxidations in bile acid biosynthesis via the neutral (classic) pathway. Can also act on cholesterol, cholest-5-en-3β,7α-diol, 7α-hydroxycholest-4-en-3-one, and 5β-cholestane-3α,7α-diol. The enzyme can also hydroxylate cholesterol at positions 24 and 25.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.16': {'ec_num': '1.14.13.16',\n",
       "  'accepted_name': 'cyclopentanone monooxygenase',\n",
       "  'reaction': 'cyclopentanone + NADPH + H+ + O2 = 5-valerolactone + NADP+ + H2O',\n",
       "  'other_names': 'cyclopentanone oxygenase',\n",
       "  'sys_name': 'cyclopentanone,NADPH:oxygen oxidoreductase (5-hydroxylating, lactonizing)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.17': {'ec_num': '1.14.13.17',\n",
       "  'accepted_name': 'cholesterol 7α-monooxygenase',\n",
       "  'reaction': 'cholesterol + NADPH + H+ + O2 = 7α-hydroxycholesterol + NADP+ + H2O',\n",
       "  'other_names': 'cholesterol 7α-hydroxylase; CYP7A1',\n",
       "  'sys_name': 'cholesterol,NADPH:oxygen oxidoreductase (7α-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.18': {'ec_num': '1.14.13.18',\n",
       "  'accepted_name': '4-hydroxyphenylacetate 1-monooxygenase',\n",
       "  'reaction': '4-hydroxyphenylacetate + NAD(P)H + H+ + O2 = homogentisate + NAD(P)+ + H2O',\n",
       "  'other_names': '4-hydroxyphenylacetate 1-hydroxylase; 4-hydroxyphenylacetic 1-hydroxylase; 4-HPA 1-hydroxylase',\n",
       "  'sys_name': '4-hydroxyphenylacetate,NAD(P)H:oxygen oxidoreductase (1-hydroxylating)',\n",
       "  'comments': 'A flavoprotein (FAD). Also acts on 4-hydroxyhydratropate (forming 2-methylhomogentisate) and on 4-hydroxyphenoxyacetate (forming hydroquinone and glycolate).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.18\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.18\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.18\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.18\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.19': {'ec_num': '1.14.13.19',\n",
       "  'accepted_name': 'taxifolin 8-monooxygenase',\n",
       "  'reaction': 'taxifolin + NAD(P)H + H+ + O2 = 2,3-dihydrogossypetin + NAD(P)+ + H2O',\n",
       "  'other_names': 'taxifolin hydroxylase',\n",
       "  'sys_name': 'taxifolin,NAD(P)H:oxygen oxidoreductase (8-hydroxylating)',\n",
       "  'comments': 'A flavoprotein, converting a flavanol into a flavanone. Also acts on fustin, but not on catechin, quercetin or mollisacidin.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.20': {'ec_num': '1.14.13.20',\n",
       "  'accepted_name': '2,4-dichlorophenol 6-monooxygenase',\n",
       "  'reaction': '2,4-dichlorophenol + NADPH + H+ + O2 = 3,5-dichlorocatechol + NADP+ + H2O',\n",
       "  'other_names': '2,4-dichlorophenol hydroxylase; 2,4-dichlorophenol monooxygenase',\n",
       "  'sys_name': '2,4-dichlorophenol,NADPH:oxygen oxidoreductase (6-hydroxylating)',\n",
       "  'comments': 'A flavoprotein (FAD). Also acts, more slowly, on 4-chlorophenol and 4-chloro-2-methylphenol; NADH can act instead of NADPH, but more slowly.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.20\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.20\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.20\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.20\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.21': {'ec_num': '1.14.13.21',\n",
       "  'accepted_name': 'flavonoid 3′-monooxygenase',\n",
       "  'reaction': 'a flavonoid + NADPH + H+ + O2 = a 3′-hydroxyflavonoid + NADP+ + H2O',\n",
       "  'other_names': 'flavonoid 3′-hydroxylase; flavonoid 3-hydroxylase (erroneous); NADPH:flavonoid-3′-hydroxylase; flavonoid 3-monooxygenase (erroneous)',\n",
       "  'sys_name': 'flavonoid,NADPH:oxygen oxidoreductase (3′-hydroxylating)',\n",
       "  'comments': 'Acts on a number of flavonoids, including naringenin and dihydrokaempferol. Does not act on 4-coumarate or 4-coumaroyl-CoA.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.22': {'ec_num': '1.14.13.22',\n",
       "  'accepted_name': 'cyclohexanone monooxygenase',\n",
       "  'reaction': 'cyclohexanone + NADPH + H+ + O2 = hexano-6-lactone + NADP+ + H2O',\n",
       "  'other_names': 'cyclohexanone 1,2-monooxygenase; cyclohexanone oxygenase; cyclohexanone:NADPH:oxygen oxidoreductase (6-hydroxylating, 1,2-lactonizing)',\n",
       "  'sys_name': 'cyclohexanone,NADPH:oxygen oxidoreductase (lactone-forming)',\n",
       "  'comments': 'A flavoprotein (FAD). In the catalytic mechanism of this enzyme, the nucleophilic species that attacks the carbonyl group is a peroxyflavin intermediate that is generated by reaction of the enzyme-bound flavin cofactor with NAD(P)H and oxygen [2]. This enzyme is able to catalyse a wide range of oxidative reactions, including enantioselective Baeyer-Villiger reactions [3], sulfoxidations [4], amine oxidations [5] and epoxidations [6].',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.22\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.22\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.22\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.22\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.13.22%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.23': {'ec_num': '1.14.13.23',\n",
       "  'accepted_name': '3-hydroxybenzoate 4-monooxygenase',\n",
       "  'reaction': '3-hydroxybenzoate + NADPH + H+ + O2 = 3,4-dihydroxybenzoate + NADP+ + H2O',\n",
       "  'other_names': '3-hydroxybenzoate 4-hydroxylase',\n",
       "  'sys_name': '3-hydroxybenzoate,NADPH:oxygen oxidoreductase (4-hydroxylating)',\n",
       "  'comments': 'A flavoprotein (FAD). Acts also on a number of analogues of 3-hydroxybenzoate substituted in the 2, 4, 5 and 6 positions.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.23\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.23\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.23\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.23\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.13.23%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.24': {'ec_num': '1.14.13.24',\n",
       "  'accepted_name': '3-hydroxybenzoate 6-monooxygenase',\n",
       "  'reaction': '3-hydroxybenzoate + NADH + H+ + O2 = 2,5-dihydroxybenzoate + NAD+ + H2O',\n",
       "  'other_names': '3-hydroxybenzoate 6-hydroxylase; m-hydroxybenzoate 6-hydroxylase; 3-hydroxybenzoic acid-6-hydroxylase',\n",
       "  'sys_name': '3-hydroxybenzoate,NADH:oxygen oxidoreductase (6-hydroxylating)',\n",
       "  'comments': 'A flavoprotein (FAD). Acts also on a number of analogues of 3-hydroxybenzoate substituted in the 2, 4, 5 and 6 positions; NADPH can act instead of NADH, but more slowly.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.24\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.24\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.24\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.24\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.13.24%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.25': {'ec_num': '1.14.13.25',\n",
       "  'accepted_name': 'methane monooxygenase (soluble)',\n",
       "  'reaction': 'methane + NAD(P)H + H+ + O2 = methanol + NAD(P)+ + H2O',\n",
       "  'other_names': 'methane hydroxylase',\n",
       "  'sys_name': 'methane,NAD(P)H:oxygen oxidoreductase (hydroxylating)',\n",
       "  'comments': 'The enzyme is soluble, in contrast to the particulate enzyme, EC 1.14.18.3. Broad specificity; many alkanes can be hydroxylated, and alkenes are converted into the corresponding epoxides; CO is oxidized to CO2, ammonia is oxidized to hydroxylamine, and some aromatic compounds and cyclic alkanes can also be hydroxylated, but more slowly.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.25\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.25\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.25\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.25\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.13.25%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.26': {'ec_num': '1.14.13.26',\n",
       "  'accepted_name': 'phosphatidylcholine 12-monooxygenase',\n",
       "  'reaction': '1-acyl-2-oleoyl-sn-glycero-3-phosphocholine + NADH + H+ + O2 = 1-acyl-2-[(S)-12-hydroxyoleoyl]-sn-glycero-3-phosphocholine + NAD+ + H2O',\n",
       "  'other_names': 'ricinoleic acid synthase; oleate Δ12-hydroxylase; oleate Δ12-monooxygenase',\n",
       "  'sys_name': '1-acyl-2-oleoyl-sn-glycero-3-phosphocholine,NADH:oxygen oxidoreductase (12-hydroxylating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.27': {'ec_num': '1.14.13.27',\n",
       "  'accepted_name': '4-aminobenzoate 1-monooxygenase',\n",
       "  'reaction': '4-aminobenzoate + NAD(P)H + 2 H+ + O2 = 4-hydroxyaniline + NAD(P)+ + H2O + CO2',\n",
       "  'other_names': '4-aminobenzoate hydroxylase; 4-aminobenzoate monooxygenase',\n",
       "  'sys_name': '4-aminobenzoate,NAD(P)H:oxygen oxidoreductase (1-hydroxylating, decarboxylating)',\n",
       "  'comments': 'A flavoprotein (FAD). Acts on anthranilate and 4-aminosalicylate but not on salicylate (cf. EC 1.14.13.1 salicylate 1-monooxygenase).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.27\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.27\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.27\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.27\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.28': {'ec_num': '1.14.13.28',\n",
       "  'accepted_name': '3,9-dihydroxypterocarpan 6a-monooxygenase',\n",
       "  'reaction': '(6aR,11aR)-3,9-dihydroxypterocarpan + NADPH + H+ + O2 = (6aS,11aS)-3,6a,9-trihydroxypterocarpan + NADP+ + H2O',\n",
       "  'other_names': '3,9-dihydroxypterocarpan 6a-hydroxylase; 3,9-dihydroxypterocarpan 6α-monooxygenase (erroneous)',\n",
       "  'sys_name': '(6aR,11aR)-3,9-dihydroxypterocarpan,NADPH:oxygen oxidoreductase (6a-hydroxylating)',\n",
       "  'comments': 'Possibly a heme-thiolate protein (P-450). The product of the reaction is the biosynthetic precursor of the phytoalexin glyceollin in soybean.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.29': {'ec_num': '1.14.13.29',\n",
       "  'accepted_name': '4-nitrophenol 2-monooxygenase',\n",
       "  'reaction': '4-nitrophenol + NADH + H+ + O2 = 4-nitrocatechol + NAD+ + H2O',\n",
       "  'other_names': '4-nitrophenol hydroxylase; 4-nitrophenol-2-hydroxylase',\n",
       "  'sys_name': '4-nitrophenol,NADH:oxygen oxidoreductase (2-hydroxylating)',\n",
       "  'comments': 'A flavoprotein (FAD).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.29\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.29\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.29\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.29\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.30': {'ec_num': '1.14.13.30',\n",
       "  'accepted_name': 'leukotriene-B4 20-monooxygenase',\n",
       "  'reaction': '(6Z,8E,10E,14Z)-(5S,12R)-5,12-dihydroxyicosa-6,8,10,14-tetraenoate + NADPH + H+ + O2 = (6Z,8E,10E,14Z)-(5S,12R)-5,12,20-trihydroxyicosa-6,8,10,14-tetraenoate + NADP+ + H2O',\n",
       "  'other_names': 'leukotriene-B4 20-hydroxylase; leucotriene-B4 ω-hydroxylase; LTB4 20-hydroxylase; LTB4 ω-hydroxylase',\n",
       "  'sys_name': '(6Z,8E,10E,14Z)-(5S,12R)-5,12-dihydroxyicosa-6,8,10,14-tetraenoate,NADPH:oxygen oxidoreductase (20-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450). Not identical with EC 1.14.13.34, leukotriene-E4 20-monooxygenase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.31': {'ec_num': '1.14.13.31',\n",
       "  'accepted_name': '2-nitrophenol 2-monooxygenase',\n",
       "  'reaction': '2-nitrophenol + 2 NADPH + 2 H+ + O2 = catechol + nitrite + 2 NADP+ + H2O',\n",
       "  'other_names': '2-nitrophenol oxygenase; nitrophenol oxygenase',\n",
       "  'sys_name': '2-nitrophenol,NADPH:oxygen 2-oxidoreductase (2-hydroxylating, nitrite-forming)',\n",
       "  'comments': 'Involved in the metabolism of nitro-aromatic compounds by a strain of Pseudomonas putida.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.31\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.31\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.31\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.31\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.32': {'ec_num': '1.14.13.32',\n",
       "  'accepted_name': 'albendazole monooxygenase',\n",
       "  'reaction': 'albendazole + NADPH + H+ + O2 = albendazole S-oxide + NADP+ + H2O',\n",
       "  'other_names': 'albendazole oxidase (misleading); albendazole sulfoxidase (ambiguous); FMO3 (gene name); albendazole monooxygenase (flavin-containing)',\n",
       "  'sys_name': 'albendazole,NADPH:oxygen oxidoreductase (sulfoxide-forming)',\n",
       "  'comments': 'A microsomal flavin-containing monooxygenase. A similar conversion is also carried out by some microsomal cytochrome P-450 enzymes [EC 1.14.14.73, albendazole monooxygenase (sulfoxide-forming)]. It is estimated that cytochrome P-450s are responsible for 70% of the activity.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'albendazole = methyl [5-(propylsulfanyl)-1H-benzimidazol-2-yl]carbamate',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.33': {'ec_num': '1.14.13.33',\n",
       "  'accepted_name': '4-hydroxybenzoate 3-monooxygenase [NAD(P)H]',\n",
       "  'reaction': '4-hydroxybenzoate + NAD(P)H + H+ + O2 = 3,4-dihydroxybenzoate + NAD(P)+ + H2O',\n",
       "  'other_names': '4-hydroxybenzoate 3-monooxygenase (reduced nicotinamide adenine dinucleotide (phosphate)); 4-hydroxybenzoate-3-hydroxylase; 4-hydroxybenzoate 3-hydroxylase',\n",
       "  'sys_name': '4-hydroxybenzoate,NAD(P)H:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'comments': 'A flavoprotein (FAD). The enzyme from Corynebacterium cyclohexanicum is highly specific for 4-hydroxybenzoate, but uses NADH and NADPH at approximately equal rates (cf. EC 1.14.13.2 4-hydroxybenzoate 3-monooxygenase). It is less specific for NADPH than EC 1.14.13.2.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.34': {'ec_num': '1.14.13.34',\n",
       "  'accepted_name': 'leukotriene-E4 20-monooxygenase',\n",
       "  'reaction': '(7E,9E,11Z,14Z)-(5S,6R)-6-(cystein-S-yl)-5-hydroxyicosa-7,9,11,14-tetraenoate + NADPH + H+ + O2 = 20-hydroxyleukotriene E4 + NADP+ + H2O',\n",
       "  'other_names': 'leukotriene-E4 ω-hydroxylase',\n",
       "  'sys_name': '(7E,9E,11Z,14Z)-(5S,6R)-6-(cystein-S-yl)-5-hydroxyicosa-7,9,11,14-tetraenoate,NADPH:oxygen oxidoreductase (20-hydroxylating)',\n",
       "  'comments': 'Also acts on N-acetyl-leukotriene E4, but more slowly. Not identical with EC 1.14.13.30 leukotriene-B4 20-monooxygenase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.35': {'ec_num': '1.14.13.35',\n",
       "  'accepted_name': 'anthranilate 3-monooxygenase (deaminating)',\n",
       "  'reaction': 'anthranilate + NADPH + H+ + O2 = 2,3-dihydroxybenzoate + NADP+ + NH3',\n",
       "  'other_names': 'anthranilate hydroxylase; anthranilate 2,3-dioxygenase (deaminating); anthranilate hydroxylase (deaminating); anthranilic hydroxylase; anthranilate 2,3-hydroxylase (deaminating)',\n",
       "  'sys_name': 'anthranilate,NADPH:oxygen oxidoreductase (3-hydroxylating, deaminating)',\n",
       "  'comments': 'The enzyme from Aspergillus niger is an iron protein; that from the yeast Trichosporon cutaneum is a flavoprotein (FAD).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.35\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.35\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.35\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.35\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.36': {'ec_num': '1.14.13.36',\n",
       "  'accepted_name': '5-O-(4-coumaroyl)-D-quinate 3′-monooxygenase',\n",
       "  'reaction': 'trans-5-O-(4-coumaroyl)-D-quinate + NADPH + H+ + O2 = trans-5-O-caffeoyl-D-quinate + NADP+ + H2O',\n",
       "  'other_names': '5-O-(4-coumaroyl)-D-quinate/shikimate 3′-hydroxylase; coumaroylquinate(coumaroylshikimate) 3′-monooxygenase',\n",
       "  'sys_name': 'trans-5-O-(4-coumaroyl)-D-quinate,NADPH:oxygen oxidoreductase (3′-hydroxylating)',\n",
       "  'comments': 'Also acts on trans-5-O-(4-coumaroyl)shikimate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.37': {'ec_num': '1.14.13.37',\n",
       "  'accepted_name': 'methyltetrahydroprotoberberine 14-monooxygenase',\n",
       "  'reaction': '(S)-N-methylcanadine + NADPH + H+ + O2 = allocryptopine + NADP+ + H2O',\n",
       "  'other_names': 'methyltetrahydroprotoberberine 14-hydroxylase; (S)-cis-N-methyltetrahydroberberine 14-monooxygenase; (S)-cis-N-methyltetrahydroprotoberberine-14-hydroxylase',\n",
       "  'sys_name': '(S)-N-methylcanadine,NADPH:oxygen oxidoreductase (14-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.38': {'ec_num': '1.14.13.38',\n",
       "  'accepted_name': 'anhydrotetracycline 6-monooxygenase',\n",
       "  'reaction': 'anhydrotetracycline + NADPH + H+ + O2 = 12-dehydrotetracycline + NADP+ + H2O',\n",
       "  'other_names': 'ATC oxygenase; anhydrotetracycline oxygenase; oxyS (gene name); anhydrotetracycline monooxygenase',\n",
       "  'sys_name': 'anhydrotetracycline,NADPH:oxygen oxidoreductase (6-hydroxylating)',\n",
       "  'comments': 'The enzyme, characterized from the bacterium Streptomyces rimosus, participates in the biosynthesis of tetracycline antibiotics. It can also catalyse EC 1.14.13.234, 12-dehydrotetracycline 5-monooxygenase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'anhydrotetracycline = (4S,4aS,12aS)-4-(dimethylamino)-3,10,11,12a-tetrahydroxy-6-methyl-1,12-dioxo-1,4,4a,5,12,12a-hexahydrotetracene-2-carboxamide\\n12-dehydrotetracycline = (4S,4aS,6S,12aS)-4-(dimethylamino)-3,6,10,12a-tetrahydroxy-6-methyl-1,11,12-trioxo-1,4,4a,5,6,11,12,12a-octahydrotetracene-2-carboxamide',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.39': {'ec_num': '1.14.13.39',\n",
       "  'accepted_name': 'nitric-oxide synthase (NADPH)',\n",
       "  'reaction': '2 L-arginine + 3 NADPH + 3 H+ + 4 O2 = 2 L-citrulline + 2 nitric oxide + 3 NADP+ + 4 H2O (overall reaction);;(1a) 2 L-arginine + 2 NADPH + 2 H+ + 2 O2 = 2 Nω-hydroxy-L-arginine + 2 NADP+ + 2 H2O ;;(1b) 2 Nω-hydroxy-L-arginine + NADPH + H+ + 2 O2 = 2 L-citrulline + 2 nitric oxide + NADP+ + 2 H2O',\n",
       "  'other_names': 'NOS (gene name); nitric oxide synthetase (ambiguous); endothelium-derived relaxation factor-forming enzyme; endothelium-derived relaxing factor synthase; NO synthase (ambiguous); NADPH-diaphorase (ambiguous)',\n",
       "  'sys_name': 'L-arginine,NADPH:oxygen oxidoreductase (nitric-oxide-forming)',\n",
       "  'comments': 'The enzyme consists of linked oxygenase and reductase domains. The eukaryotic enzyme binds FAD, FMN, heme (iron protoporphyrin IX) and tetrahydrobiopterin, and its two domains are linked via a regulatory calmodulin-binding domain. Upon calcium-induced calmodulin binding, the reductase and oxygenase domains form a complex, allowing electrons to flow from NADPH via FAD and FMN to the active center. The reductase domain of the enzyme from the bacterium Sorangium cellulosum utilizes a [2Fe-2S] cluster to transfer the electrons from NADPH to the active center. cf. EC 1.14.14.47, nitric-oxide synthase (flavodoxin).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'nitric oxide = NO = nitrogen(II) oxide',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.40': {'ec_num': '1.14.13.40',\n",
       "  'accepted_name': 'anthraniloyl-CoA monooxygenase',\n",
       "  'reaction': 'anthraniloyl-CoA + 2 NAD(P)H + 2 H+ + O2 = 2-amino-5-oxocyclohex-1-enecarboxyl-CoA + H2O + 2 NAD(P)+',\n",
       "  'other_names': 'anthraniloyl coenzyme A reductase; 2-aminobenzoyl-CoA monooxygenase/reductase',\n",
       "  'sys_name': 'anthraniloyl-CoA,NAD(P)H:oxygen oxidoreductase (de-aromatizing) ',\n",
       "  'comments': 'A flavoprotein (FAD). The non-aromatic product is unstable and releases CO2 and NH3, forming 1,4-cyclohexanedione.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.40\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.40\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.40\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.40\" target=\"new\">KEGG</a>',\n",
       "  'glossary': 'anthraniloyl-CoA = 2-aminobenzoyl-CoA',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.41': {'ec_num': '1.14.13.41',\n",
       "  'accepted_name': 'tyrosine N-monooxygenase',\n",
       "  'reaction': ' L-tyrosine + 2 O2 + 2 NADPH + 2 H+ = (E)-[4-hydroxyphenylacetaldehyde oxime] + 2 NADP+ + CO2 + 3 H2O (overall reaction);;(1a) L-tyrosine + O2 + NADPH + H+ = N-hydroxy-L-tyrosine + NADP+ + H2O;;(1b) N-hydroxy-L-tyrosine + O2 + NADPH + H+ = N,N-dihydroxy-L-tyrosine + NADP+ + H2O;;(1c) N,N-dihydroxy-L-tyrosine = (E)-[4-hydroxyphenylacetaldehyde oxime] + CO2 + H2O (spontaneous)',\n",
       "  'other_names': 'tyrosine N-hydroxylase; CYP79A1',\n",
       "  'sys_name': 'L-tyrosine,NADPH:oxygen oxidoreductase (N-hydroxylating)',\n",
       "  'comments': 'A P-450 (heme-thiolate) protein. This enzyme is involved in the biosynthesis of the cyanogenic glucoside dhurrin in sorghum, along with EC 1.14.13.68, 4-hydroxyphenylacetaldehyde oxime monooxygenase and EC 2.4.1.85, cyanohydrin β-glucosyltransferase. Some 2-(4-hydroxyphenyl)-1-nitroethane is formed as a side product.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.42': {'ec_num': '1.14.13.42',\n",
       "  'accepted_name': 'hydroxyphenylacetonitrile 2-monooxygenase',\n",
       "  'reaction': '4-hydroxyphenylacetonitrile + NADPH + H+ + O2 = 4-hydroxymandelonitrile + NADP+ + H2O',\n",
       "  'other_names': '4-hydroxyphenylacetonitrile monooxygenase; 4-hydroxyphenylacetonitrile hydroxylase',\n",
       "  'sys_name': '4-hydroxyphenylacetonitrile,NADPH:oxygen oxidoreductase (2-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.43': {'ec_num': '1.14.13.43',\n",
       "  'accepted_name': 'questin monooxygenase',\n",
       "  'reaction': 'questin + NADPH + H+ + O2 = demethylsulochrin + NADP+',\n",
       "  'other_names': 'questin oxygenase',\n",
       "  'sys_name': 'questin,NADPH:oxygen oxidoreductase (hydroxylating, anthraquinone-ring-opening)',\n",
       "  'comments': 'The enzyme cleaves the anthraquinone ring of questin to form a benzophenone. Involved in the biosynthesis of the seco-anthraquinone (+)-geodin.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'questin = 3,8-dihydroxy-1-methoxy-6-methylanthracene-9,10-dione<br>\\ndemethylsulochrin = 2-(2,6-dihydroxy-4-methylbenzoyl)-5-hydroxy-3-methoxybenzoic acid',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.44': {'ec_num': '1.14.13.44',\n",
       "  'accepted_name': '2-hydroxybiphenyl 3-monooxygenase',\n",
       "  'reaction': '2-hydroxybiphenyl + NADH + H+ + O2 = 2,3-dihydroxybiphenyl + NAD+ + H2O',\n",
       "  'other_names': '',\n",
       "  'sys_name': '2-hydroxybiphenyl,NADH:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'comments': 'Also converts 2,2′-dihydroxybiphenyl into 2,2′,3-trihydroxy-biphenyl.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.44\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.44\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.44\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.44\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.13.44%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.46': {'ec_num': '1.14.13.46',\n",
       "  'accepted_name': '(-)-menthol monooxygenase',\n",
       "  'reaction': '(-)-menthol + NADPH + H+ + O2 = p-menthane-3,8-diol + NADP+ + H2O',\n",
       "  'other_names': 'l-menthol monooxygenase',\n",
       "  'sys_name': '(-)-menthol,NADPH:oxygen oxidoreductase (8-hydroxylating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.47': {'ec_num': '1.14.13.47',\n",
       "  'accepted_name': '(S)-limonene 3-monooxygenase',\n",
       "  'reaction': '(S)-limonene + NADPH + H+ + O2 = (–)-trans-isopiperitenol + NADP+ + H2O',\n",
       "  'other_names': '(–)-limonene 3-hydroxylase; (–)-limonene 3-monooxygenase; (–)-limonene,NADPH:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'sys_name': '(S)-limonene,NADPH:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'comments': 'High specificity, but NADH can act instead of NADPH, although more slowly. A heme-thiolate protein (P-450).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.47\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.47\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.47\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.47\" target=\"new\">KEGG</a>',\n",
       "  'glossary': 'limonene = a monoterpenoid\\n(S)-limonene = (–)-limonene',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.48': {'ec_num': '1.14.13.48',\n",
       "  'accepted_name': '(S)-limonene 6-monooxygenase',\n",
       "  'reaction': '(S)-limonene + NADPH + H+ + O2 = (–)-trans-carveol + NADP+ + H2O',\n",
       "  'other_names': '(–)-limonene 6-hydroxylase; (–)-limonene 6-monooxygenase; (–)-limonene,NADPH:oxygen oxidoreductase (6-hydroxylating)',\n",
       "  'sys_name': '(S)-limonene,NADPH:oxygen oxidoreductase (6-hydroxylating)',\n",
       "  'comments': 'High specificity, but NADH can act instead of NADPH, but more slowly. A heme-thiolate protein (P-450).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.48\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.48\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.48\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.48\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.13.48%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': 'limonene = a monoterpenoid\\n(S)-limonene = (–)-limonene',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.49': {'ec_num': '1.14.13.49',\n",
       "  'accepted_name': '(S)-limonene 7-monooxygenase',\n",
       "  'reaction': '(S)-limonene + NADPH + H+ + O2 = (–)-perillyl alcohol + NADP+ + H2O',\n",
       "  'other_names': '(–)-limonene 7-monooxygenase; (–)-limonene hydroxylase; (–)-limonene monooxygenase; (–)-limonene,NADPH:oxygen oxidoreductase (7-hydroxylating)',\n",
       "  'sys_name': '(S)-limonene,NADPH:oxygen oxidoreductase (7-hydroxylating)',\n",
       "  'comments': 'High specificity, but NADH can act instead of NADPH, although more slowly. A heme-thiolate protein (P-450).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.49\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.49\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.49\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.49\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.13.49%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': 'limonene = a monoterpenoid\\n(S)-limonene = (–)-limonene',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.50': {'ec_num': '1.14.13.50',\n",
       "  'accepted_name': 'pentachlorophenol monooxygenase',\n",
       "  'reaction': '(1) pentachlorophenol + NADPH + H+ + O2 = 2,3,5,6-tetrachloro-1,4-benzoquinone + NADP+ + chloride + H2O;;(2) 2,3,5,6-tetrachlorophenol + NADPH + H+ + O2 = 2,3,5,6-tetrachlorohydroquinone + NADP+ + H2O',\n",
       "  'other_names': 'pcpB (gene name); pentachlorophenol dechlorinase; pentachlorophenol dehalogenase; pentachlorophenol 4-monooxygenase; PCP hydroxylase; pentachlorophenol hydroxylase; PCB 4-monooxygenase; PCB4MO',\n",
       "  'sys_name': 'pentachlorophenol,NADPH:oxygen oxidoreductase (hydroxylating, dechlorinating)',\n",
       "  'comments': 'A flavoprotein (FAD). The enzyme displaces a diverse range of substituents from the 4-position of polyhalogenated phenols but requires that a halogen substituent be present at the 2-position [2]. If C-4 carries a halogen substituent, reaction 1 is catalysed; if C-4 is unsubstituted, reaction 2 is catalysed.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.50\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.50\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.50\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.50\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.13.50%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.51': {'ec_num': '1.14.13.51',\n",
       "  'accepted_name': '6-oxocineole dehydrogenase',\n",
       "  'reaction': '6-oxocineole + NADPH + H+ + O2 = 1,6,6-trimethyl-2,7-dioxabicyclo[3.2.2]nonan-3-one + NADP+ + H2O',\n",
       "  'other_names': '6-oxocineole oxygenase',\n",
       "  'sys_name': '6-oxocineole,NADPH:oxygen oxidoreductase',\n",
       "  'comments': 'The product undergoes non-enzymic cleavage and subsequent ring closure to form the lactone 4,5-dihydro-5,5-dimethyl-4-(3-oxobutyl)furan-2(3H)-one.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.52': {'ec_num': '1.14.13.52',\n",
       "  'accepted_name': 'isoflavone 3′-hydroxylase',\n",
       "  'reaction': 'formononetin + NADPH + H+ + O2 = calycosin + NADP+ + H2O',\n",
       "  'other_names': 'isoflavone 3′-monooxygenase',\n",
       "  'sys_name': 'formononetin,NADPH:oxygen oxidoreductase (3′-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450). Also acts on biochanin A and other isoflavones with a 4′-methoxy group. Involved in the biosynthesis of the pterocarpin phytoalexins medicarpin and maackiain.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'calycosin = 3′-hydroxyformononetin',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.53': {'ec_num': '1.14.13.53',\n",
       "  'accepted_name': '4′-methoxyisoflavone 2′-hydroxylase',\n",
       "  'reaction': 'formononetin + NADPH + H+ + O2 = 2′-hydroxyformononetin + NADP+ + H2O',\n",
       "  'other_names': 'isoflavone 2′-monooxygenase (ambiguous); isoflavone 2′-hydroxylase (ambiguous)',\n",
       "  'sys_name': 'formononetin,NADPH:oxygen oxidoreductase (2′-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450). Acts on isoflavones with a 4′-methoxy group, such as formononetin and biochanin A. Involved in the biosynthesis of the pterocarpin phytoalexins medicarpin and maackiain. EC 1.14.13.89, isoflavone 2′-hydroxylase, is less specific and acts on other isoflavones as well as 4′-methoxyisoflavones.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.54': {'ec_num': '1.14.13.54',\n",
       "  'accepted_name': 'ketosteroid monooxygenase',\n",
       "  'reaction': 'a ketosteroid + NADPH + H+ + O2 = a steroid ester/lactone + NADP+ + H2O (general reaction);;(1) progesterone + NADPH + H+ + O2 = testosterone acetate + NADP+ + H2O;;(2) androstenedione + NADPH + H+ + O2 = testololactone + NADP+ + H2O;;(3) 17α-hydroxyprogesterone + NADPH + H+ + O2 = androstenedione + acetate + NADP+ + H2O',\n",
       "  'other_names': 'steroid-ketone monooxygenase; progesterone, NADPH2:oxygen oxidoreductase (20-hydroxylating, ester-producing); 17α-hydroxyprogesterone, NADPH2:oxygen oxidoreductase (20-hydroxylating, side-chain cleaving); androstenedione, NADPH2:oxygen oxidoreductase (17-hydroxylating, lactonizing)',\n",
       "  'sys_name': 'ketosteroid,NADPH:oxygen oxidoreductase (20-hydroxylating, ester-producing/20-hydroxylating, side-chain cleaving/17-hydroxylating, lactonizing)',\n",
       "  'comments': 'A single FAD-containing enzyme catalyses three types of monooxygenase (Baeyer-Villiger oxidation) reaction. The oxidative esterification of a number of derivatives of progesterone to produce the corresponding 17α-hydroxysteroid 17-acetate ester, such as testosterone acetate, is shown in Reaction (1). The oxidative lactonization of a number of derivatives of androstenedione to produce the 13,17-secoandrosteno-17,13α-lactone, such as testololactone, is shown in Reaction (2). The oxidative cleavage of the 17β-side-chain of 17α-hydroxyprogesterone to produce androstenedione and acetate is shown in Reaction (3). Reaction (1) is also catalysed by EC 1.14.99.4 (progesterone monooxygenase), and Reactions (2) and (3) correspond to that catalysed by EC 1.14.99.12 (androst-4-ene-3,17-dione monooxygenase). The possibility that a single enzyme is responsible for the reactions ascribed to EC 1.14.99.4 and EC 1.14.99.12 in other tissues cannot be excluded.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'progesterone = pregn-4-ene-3,20-dione<br>\\ntestosterone acetate = 3-oxoandrost-4-en-17β-yl acetate<br>\\nandrostenedione = androst-4-ene-3,17-dione<br>\\ntestololactone = 3-oxo-13,17-secoandrost-4-eno-17,13α-lactone<br>\\n17α-hydroxyprogesterone = 17α-hydroxypregn-4-ene-3,20-dione',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.55': {'ec_num': '1.14.13.55',\n",
       "  'accepted_name': 'protopine 6-monooxygenase',\n",
       "  'reaction': 'protopine + NADPH + H+ + O2 = 6-hydroxyprotopine + NADP+ + H2O',\n",
       "  'other_names': 'protopine 6-hydroxylase',\n",
       "  'sys_name': 'protopine,NADPH:oxygen oxidoreductase (6-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450) involved in benzophenanthridine alkaloid synthesis in higher plants.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.56': {'ec_num': '1.14.13.56',\n",
       "  'accepted_name': 'dihydrosanguinarine 10-monooxygenase',\n",
       "  'reaction': 'dihydrosanguinarine + NADPH + H+ + O2 = 10-hydroxydihydrosanguinarine + NADP+ + H2O',\n",
       "  'other_names': 'dihydrosanguinarine 10-hydroxylase',\n",
       "  'sys_name': 'dihydrosanguinarine,NADPH:oxygen oxidoreductase (10-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450) involved in benzophenanthridine alkaloid synthesis in higher plants.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.57': {'ec_num': '1.14.13.57',\n",
       "  'accepted_name': 'dihydrochelirubine 12-monooxygenase',\n",
       "  'reaction': 'dihydrochelirubine + NADPH + H+ + O2 = 12-hydroxydihydrochelirubine + NADP+ + H2O',\n",
       "  'other_names': 'dihydrochelirubine 12-hydroxylase',\n",
       "  'sys_name': 'dihydrochelirubine,NADPH:oxygen oxidoreductase (12-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450)',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.58': {'ec_num': '1.14.13.58',\n",
       "  'accepted_name': 'benzoyl-CoA 3-monooxygenase',\n",
       "  'reaction': 'benzoyl-CoA + NADPH + H+ + O2 = 3-hydroxybenzoyl-CoA + NADP+ + H2O',\n",
       "  'other_names': 'benzoyl-CoA 3-hydroxylase',\n",
       "  'sys_name': 'benzoyl-CoA,NADPH:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'comments': 'The enzyme from the denitrifying bacterium Pseudomonas KB740 catalyses a flavin-requiring reaction (FAD or FMN). Benzoate is not a substrate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.59': {'ec_num': '1.14.13.59',\n",
       "  'accepted_name': 'L-lysine N6-monooxygenase (NADPH)',\n",
       "  'reaction': 'L-lysine + NADPH + H+ + O2 = N6-hydroxy-L-lysine + NADP+ + H2O',\n",
       "  'other_names': 'lysine N6-hydroxylase; L-lysine 6-monooxygenase (NADPH) (ambiguous)',\n",
       "  'sys_name': 'L-lysine,NADPH:oxygen oxidoreductase (6-hydroxylating)',\n",
       "  'comments': 'A flavoprotein (FAD). The enzyme from strain EN 222 of Escherichia coli is highly specific for L-lysine; L-ornithine and L-homolysine are, for example, not substrates.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.60': {'ec_num': '1.14.13.60',\n",
       "  'accepted_name': '27-hydroxycholesterol 7α-monooxygenase',\n",
       "  'reaction': 'cholest-5-ene-3β,26-diol + NADPH + H+ + O2 = cholest-5-ene-3β,7α,27-triol + NADP+ + H2O',\n",
       "  'other_names': '27-hydroxycholesterol 7α-hydroxylase',\n",
       "  'sys_name': '27-hydroxycholesterol,NADPH:oxygen oxidoreductase (7α-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450). The enzyme from mammalian liver differs from cholesterol 7α-monooxygenase (EC 1.14.13.17) in having no activity towards cholesterol.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-14 11:40:50'},\n",
       " '1.14.13.61': {'ec_num': '1.14.13.61',\n",
       "  'accepted_name': '2-hydroxyquinoline 8-monooxygenase',\n",
       "  'reaction': 'quinolin-2-ol + NADH + H+ + O2 = quinolin-2,8-diol + NAD+ + H2O',\n",
       "  'other_names': '2-oxo-1,2-dihydroquinoline 8-monooxygenase',\n",
       "  'sys_name': 'quinolin-2(1H)-one,NADH:oxygen oxidoreductase (8-oxygenating)',\n",
       "  'comments': 'Requires iron. Quinolin-2-ol exists largely as the quinolin-2(1H)-one tautomer.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.62': {'ec_num': '1.14.13.62',\n",
       "  'accepted_name': '4-hydroxyquinoline 3-monooxygenase',\n",
       "  'reaction': 'quinolin-4-ol + NADH + H+ + O2 = quinolin-3,4-diol + NAD+ + H2O',\n",
       "  'other_names': 'quinolin-4(1H)-one 3-monooxygenase',\n",
       "  'sys_name': 'quinolin-4(1H)-one,NADH:oxygen oxidoreductase (3-oxygenating)',\n",
       "  'comments': 'Quinolin-4-ol exists largely as the quinolin-4(1H)-one tautomer.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.63': {'ec_num': '1.14.13.63',\n",
       "  'accepted_name': '3-hydroxyphenylacetate 6-hydroxylase',\n",
       "  'reaction': '3-hydroxyphenylacetate + NAD(P)H + H+ + O2 = 2,5-dihydroxyphenylacetate + NAD(P)+ + H2O',\n",
       "  'other_names': '3-hydroxyphenylacetate 6-monooxygenase',\n",
       "  'sys_name': '3-hydroxyphenylacetate,NAD(P)H:oxygen oxidoreductase (6-hydroxylating)',\n",
       "  'comments': '3-hydroxyphenylacetate 6-hydroxylase from Flavobacterium sp. is highly specific for 3-hydroxyphenylacetate and uses NADH and NADPH as electron donors with similar efficiency.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'homogentisate = 2,5-dihydroxyphenylacetate',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.64': {'ec_num': '1.14.13.64',\n",
       "  'accepted_name': '4-hydroxybenzoate 1-hydroxylase',\n",
       "  'reaction': '4-hydroxybenzoate + NAD(P)H + 2 H+ + O2 = hydroquinone + NAD(P)+ + H2O + CO2',\n",
       "  'other_names': '4-hydroxybenzoate 1-monooxygenase',\n",
       "  'sys_name': '4-hydroxybenzoate,NAD(P)H:oxygen oxidoreductase (1-hydroxylating, decarboxylating)',\n",
       "  'comments': 'Requires FAD. The enzyme from Candida parapsilosis is specific for 4-hydroxybenzoate derivatives and prefers NADH to NADPH as electron donor.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.64\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.64\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.64\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.64\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.65': {'ec_num': '1.14.13.65',\n",
       "  'accepted_name': '2-hydroxyquinoline 8-monooxygenase',\n",
       "  'reaction': 'quinolin-2-ol + NADH + H+ + O2 = quinolin-2,8-diol + NAD+',\n",
       "  'other_names': '2-oxo-1,2-dihydroquinoline 5,6-dioxygenase',\n",
       "  'sys_name': 'quinolin-2-ol,NADH:oxygen oxidoreductase (8-hydroxylating) ',\n",
       "  'comments': '3-Methyl-quinolin-2-ol is also a substrate. Quinolin-2-ols exist largely as the quinolin-2(1H)-one tautomers.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.66': {'ec_num': '1.14.13.66',\n",
       "  'accepted_name': '2-hydroxycyclohexanone 2-monooxygenase',\n",
       "  'reaction': '2-hydroxycyclohexan-1-one + NADPH + H+ + O2 = 6-hydroxyhexan-6-olide + NADP+ + H2O',\n",
       "  'other_names': '',\n",
       "  'sys_name': '2-hydroxycyclohexan-1-one,NADPH:oxygen 2-oxidoreductase (1,2-lactonizing)',\n",
       "  'comments': 'The product decomposes spontaneously to 6-oxohexanoic acid (adipic semialdehyde).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.67': {'ec_num': '1.14.13.67',\n",
       "  'accepted_name': 'quinine 3-monooxygenase',\n",
       "  'reaction': 'quinine + NADPH + H+ + O2 = 3-hydroxyquinine + NADP+ + H2O',\n",
       "  'other_names': 'quinine 3-hydroxylase',\n",
       "  'sys_name': 'quinine,NADPH:oxygen oxidoreductase',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'quinine = a quinoline alkaloid',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.68': {'ec_num': '1.14.13.68',\n",
       "  'accepted_name': '4-hydroxyphenylacetaldehyde oxime monooxygenase',\n",
       "  'reaction': '(Z)-4-hydroxyphenylacetaldehyde oxime + NADPH + H+ + O2 = (S)-4-hydroxymandelonitrile + NADP+ + 2 H2O',\n",
       "  'other_names': '4-hydroxybenzeneacetaldehyde oxime monooxygenase; cytochrome P450II-dependent monooxygenase; CYP71E1; 4-hydroxyphenylacetaldehyde oxime,NADPH:oxygen oxidoreductase',\n",
       "  'sys_name': '(Z)-4-hydroxyphenylacetaldehyde oxime,NADPH:oxygen oxidoreductase',\n",
       "  'comments': 'This enzyme is involved in the biosynthesis of the cyanogenic glucoside dhurrin in sorghum, along with EC 1.14.13.41, tyrosine N-monooxygenase and EC 2.4.1.85, cyanohydrin β-glucosyltransferase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.69': {'ec_num': '1.14.13.69',\n",
       "  'accepted_name': 'alkene monooxygenase',\n",
       "  'reaction': 'propene + NADH + H+ + O2 = 1,2-epoxypropane + NAD+ + H2O',\n",
       "  'other_names': 'alkene epoxygenase; etnABCD (gene names); amoABCDE (gene names)',\n",
       "  'sys_name': 'alkene,NADH:oxygen oxidoreductase',\n",
       "  'comments': 'This bacterial binuclear non-heme iron enzyme is a multicomponent enzyme complex comprising an oxygenase, a reductase, and a Rieske-type ferredoxin. The enzyme from the bacterium Xanthobacter sp. strain Py2 contains an additional small protein of unknown function that is essential for activity. In general, the enzyme oxygenates C2 to C6 aliphatic alkenes, although enzymes from different organisms show different substrate range. With propene as substrate, the stereospecificity of the epoxypropane formed is 95% (R) and 5% (S).',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.69\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.69\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.69\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.69\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.70': {'ec_num': '1.14.13.70',\n",
       "  'accepted_name': 'sterol 14α-demethylase',\n",
       "  'reaction': 'a 14α-methylsteroid + 3 O2 + 3 NADPH + 3 H+ = a Δ14-steroid + formate + 3 NADP+ + 4 H2O',\n",
       "  'other_names': 'obtusufoliol 14-demethylase; lanosterol 14-demethylase; lanosterol 14α-demethylase; sterol 14-demethylase',\n",
       "  'sys_name': 'sterol,NADPH:oxygen oxidoreductase (14-methyl cleaving)',\n",
       "  'comments': 'This heme-thiolate (P-450) enzyme acts on a range of steroids with a 14α-methyl group, such as obtusifoliol and lanosterol. The enzyme catalyses two successive hydroxylations of the 14α-methyl group, followed by elimination as formate, leaving a 14(15) double bond.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'obtusifoliol = 4α,14α-dimethyl-5α-ergosta-8,24(28)-dien-3β-ol or 4α,14α-dimethyl-24-methylene-5α-cholesta-8-en-3β-ol',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.71': {'ec_num': '1.14.13.71',\n",
       "  'accepted_name': 'N-methylcoclaurine 3′-monooxygenase',\n",
       "  'reaction': '(S)-N-methylcoclaurine + NADPH + H+ + O2 = (S)-3′-hydroxy-N-methylcoclaurine + NADP+ + H2O',\n",
       "  'other_names': 'N-methylcoclaurine 3′-hydroxylase',\n",
       "  'sys_name': '(S)-N-methylcoclaurine,NADPH:oxygen oxidoreductase (3′-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450) involved in benzylisoquinoline alkaloid synthesis in higher plants.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.72': {'ec_num': '1.14.13.72',\n",
       "  'accepted_name': 'methylsterol monooxygenase',\n",
       "  'reaction': '4,4-dimethyl-5α-cholest-7-en-3β-ol + 3 NAD(P)H + 3 H+ + 3 O2 = 3β-hydroxy-4β-methyl-5α-cholest-7-ene-4α-carboxylate + 3 NAD(P)+ + 4 H2O (overall reaction);;(1a) 4,4-dimethyl-5α-cholest-7-en-3β-ol + NAD(P)H + H+ + O2 = 4β-hydroxymethyl-4α-methyl-5α-cholest-7-en-3β-ol + NAD(P)+ + H2O;;(1b) 4β-hydroxymethyl-4α-methyl-5α-cholest-7-en-3β-ol + NAD(P)H + H+ + O2 = 3β-hydroxy-4β-methyl-5α-cholest-7-ene-4α-carbaldehyde + NAD(P)+ + 2 H2O;;(1c) 3β-hydroxy-4β-methyl-5α-cholest-7-ene-4α-carbaldehyde + NAD(P)H + H+ + O2 = 3β-hydroxy-4β-methyl-5α-cholest-7-ene-4α-carboxylate + NAD(P)+ + H2O',\n",
       "  'other_names': 'methylsterol hydroxylase; 4-methylsterol oxidase; 4,4-dimethyl-5α-cholest-7-en-3β-ol,hydrogen-donor:oxygen oxidoreductase (hydroxylating)',\n",
       "  'sys_name': '4,4-dimethyl-5α-cholest-7-en-3β-ol,NAD(P)H:oxygen oxidoreductase (hydroxylating)',\n",
       "  'comments': 'Requires cytochrome b5. Also acts on 4α-methyl-5α-cholest-7-en-3β-ol. The sterol can be based on cycloartenol as well as lanosterol.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.73': {'ec_num': '1.14.13.73',\n",
       "  'accepted_name': 'tabersonine 16-hydroxylase',\n",
       "  'reaction': 'tabersonine + NADPH + H+ + O2 = 16-hydroxytabersonine + NADP+ + H2O',\n",
       "  'other_names': 'tabersonine-11-hydroxylase; T11H',\n",
       "  'sys_name': 'tabersonine,NADPH:oxygen oxidoreductase (16-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.74': {'ec_num': '1.14.13.74',\n",
       "  'accepted_name': '7-deoxyloganin 7-hydroxylase',\n",
       "  'reaction': '7-deoxyloganin + NADPH + H+ + O2 = loganin + NADP+ + H2O',\n",
       "  'other_names': '',\n",
       "  'sys_name': '7-deoxyloganin,NADPH:oxygen oxidoreductase (7α-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.75': {'ec_num': '1.14.13.75',\n",
       "  'accepted_name': 'vinorine hydroxylase',\n",
       "  'reaction': 'vinorine + NADPH + H+ + O2 = vomilenine + NADP+ + H2O',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'vinorine,NADPH:oxygen oxidoreductase (21α-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450). Forms a stage in the biosynthesis of the indole alkaloid ajmaline.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.76': {'ec_num': '1.14.13.76',\n",
       "  'accepted_name': 'taxane 10β-hydroxylase',\n",
       "  'reaction': 'taxa-4(20),11-dien-5α-yl acetate + NADPH + H+ + O2 = 10β-hydroxytaxa-4(20),11-dien-5α-yl acetate + NADP+ + H2O',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'taxa-4(20),11-dien-5α-yl acetate,NADPH:oxygen oxidoreductase (10β-hydroxylating)',\n",
       "  'comments': 'This microsomal cytochrome-P-450-dependent enzyme is involved in the biosynthesis of the diterpenoid antineoplastic drug taxol (paclitaxel).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.77': {'ec_num': '1.14.13.77',\n",
       "  'accepted_name': 'taxane 13α-hydroxylase',\n",
       "  'reaction': 'taxa-4(20),11-dien-5α-ol + NADPH + H+ + O2 = taxa-4(20),11-dien-5α,13α-diol + NADP+ + H2O',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'taxa-4(20),11-dien-5α-ol,NADPH:oxygen oxidoreductase (13α-hydroxylating)',\n",
       "  'comments': 'This cytochrome-P-450-dependent enzyme is involved in the biosynthesis of the diterpenoid antineoplastic drug taxol (paclitaxel).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.78': {'ec_num': '1.14.13.78',\n",
       "  'accepted_name': 'ent-kaurene oxidase',\n",
       "  'reaction': 'ent-kaur-16-ene + 3 NADPH + 3 H+ + 3 O2 = ent-kaur-16-en-19-oate + 3 NADP+ + 4 H2O (overall reaction);;(1a) ent-kaur-16-ene + NADPH + H+ + O2 = ent-kaur-16-en-19-ol + NADP+ + H2O;;(1b) ent-kaur-16-en-19-ol + NADPH + H+ + O2 = ent-kaur-16-en-19-al + NADP+ + 2 H2O;;(1c) ent-kaur-16-en-19-al + NADPH + O2 = ent-kaur-16-en-19-oate + NADP+ + H2O',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'ent-kaur-16-ene,NADPH:oxygen oxidoreductase (hydroxylating)',\n",
       "  'comments': 'Requires cytochrome P-450. Catalyses three sucessive oxidations of the 4-methyl group of ent-kaurene giving kaurenoic acid.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.79': {'ec_num': '1.14.13.79',\n",
       "  'accepted_name': 'ent-kaurenoic acid oxidase',\n",
       "  'reaction': 'ent-kaur-16-en-19-oate + 3 NADPH + 3 H+ + 3 O2 = gibberellin A12 + 3 NADP+ + 4 H2O (overall reaction);;(1a) ent-kaur-16-en-19-oate + NADPH + H+ + O2 = ent-7α-hydroxykaur-16-en-19-oate + NADP+ + H2O;;(1b) ent-7α-hydroxykaur-16-en-19-oate + NADPH + H+ + O2 = gibberellin A12 aldehyde + NADP+ + 2 H2O;;(1c) gibberellin A12 aldehyde + NADPH + H+ + O2 = gibberellin A12 + NADP+ + H2O',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'ent-kaur-16-en-19-oate,NADPH:oxygen oxidoreductase (hydroxylating)',\n",
       "  'comments': 'Requires cytochrome P-450. Catalyses three sucessive oxidations of ent-kaurenoic acid. The second step includes a ring-B contraction giving the gibbane skeleton. In pumpkin (Cucurbita maxima) ent-6α,7α-dihydroxykaur-16-en-19-oate is also formed.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.80': {'ec_num': '1.14.13.80',\n",
       "  'accepted_name': '(R)-limonene 6-monooxygenase',\n",
       "  'reaction': '(R)-limonene + NADPH + H+ + O2 = (+)-trans-carveol + NADP+ + H2O',\n",
       "  'other_names': '(+)-limonene-6-hydroxylase; (+)-limonene 6-monooxygenase',\n",
       "  'sys_name': '(R)-limonene,NADPH:oxygen oxidoreductase (6-hydroxylating)',\n",
       "  'comments': 'The reaction is stereospecific with over 95% yield of (+)-trans-carveol from (R)-limonene. (S)-Limonene, the substrate for EC 1.14.13.48, (S)-limonene 6-monooxygenase, is not a substrate. Forms part of the carvone biosynthesis pathway in Carum carvi (caraway) seeds.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'limonene = a monoterpenoid\\n(R)-limonene = (+)-limonene',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.81': {'ec_num': '1.14.13.81',\n",
       "  'accepted_name': 'magnesium-protoporphyrin IX monomethyl ester (oxidative) cyclase',\n",
       "  'reaction': 'magnesium-protoporphyrin IX 13-monomethyl ester + 3 NADPH + 3 H+ + 3 O2 = 3,8-divinyl protochlorophyllide a + 3 NADP+ + 5 H2O (overall reaction);;(1a) magnesium-protoporphyrin IX 13-monomethyl ester + NADPH + H+ + O2 = 131-hydroxy-magnesium-protoporphyrin IX 13-monomethyl ester + NADP+ + H2O;;(1b) 131-hydroxy-magnesium-protoporphyrin IX 13-monomethyl ester + NADPH + H+ + O2 = 131-oxo-magnesium-protoporphyrin IX 13-monomethyl ester + NADP+ + 2 H2O;;(1c) 131-oxo-magnesium-protoporphyrin IX 13-monomethyl ester + NADPH + H+ + O2 = 3,8-divinyl protochlorophyllide a + NADP+ + 2 H2O',\n",
       "  'other_names': 'Mg-protoporphyrin IX monomethyl ester (oxidative) cyclase',\n",
       "  'sys_name': 'magnesium-protoporphyrin-IX 13-monomethyl ester,NADPH:oxygen oxidoreductase (hydroxylating)',\n",
       "  'comments': 'Requires iron(II) for activity. The enzyme participates in the biosynthesis of chlorophyllide a in aerobic organisms. The same transformation is achieved in anaerobic organisms by EC 1.21.98.3, anaerobic magnesium-protoporphyrin IX monomethyl ester cyclase. Some facultative phototrophic bacteria, such as Rubrivivax gelatinosus, possess both enzymes.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.82': {'ec_num': '1.14.13.82',\n",
       "  'accepted_name': 'vanillate monooxygenase',\n",
       "  'reaction': 'vanillate + O2 + NADH + H+ = 3,4-dihydroxybenzoate + NAD+ + H2O + formaldehyde',\n",
       "  'other_names': '4-hydroxy-3-methoxybenzoate demethylase; vanillate demethylase',\n",
       "  'sys_name': 'vanillate:oxygen oxidoreductase (demethylating)',\n",
       "  'comments': 'Forms part of the vanillin degradation pathway in Arthrobacter sp.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.13.82\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.13.82\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.13.82\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.13.82\" target=\"new\">KEGG</a>',\n",
       "  'glossary': 'vanillate = 4-hydroxy-3-methoxybenzoate',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.83': {'ec_num': '1.14.13.83',\n",
       "  'accepted_name': 'precorrin-3B synthase',\n",
       "  'reaction': 'precorrin-3A + NADH + H+ + O2 = precorrin-3B + NAD+ + H2O',\n",
       "  'other_names': 'precorrin-3X synthase; CobG',\n",
       "  'sys_name': 'precorrin-3A,NADH:oxygen oxidoreductase (20-hydroxylating)',\n",
       "  'comments': 'An iron-sulfur protein. An oxygen atom from dioxygen is incorporated into the macrocycle at C-20. In the aerobic cobalamin biosythesis pathway, four enzymes are involved in the conversion of precorrin-3A to precorrin-6A. The first of the four steps is carried out by EC 1.14.13.83, precorrin-3B synthase (CobG), yielding precorrin-3B as the product. This is followed by three methylation reactions, which introduce a methyl group at C-17 (CobJ; EC 2.1.1.131), C-11 (CobM; EC 2.1.1.133) and C-1 (CobF; EC 2.1.1.152) of the macrocycle, giving rise to precorrin-4, precorrin-5 and precorrin-6A, respectively.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.84': {'ec_num': '1.14.13.84',\n",
       "  'accepted_name': '4-hydroxyacetophenone monooxygenase',\n",
       "  'reaction': '(4-hydroxyphenyl)ethan-1-one + NADPH + H+ + O2 = 4-hydroxyphenyl acetate + NADP+ + H2O',\n",
       "  'other_names': 'HAPMO',\n",
       "  'sys_name': '(4-hydroxyphenyl)ethan-1-one,NADPH:oxygen oxidoreductase (ester-forming)',\n",
       "  'comments': 'Contains FAD. The enzyme from Pseudomonas fluorescens ACB catalyses the conversion of a wide range of acetophenone derivatives. Highest activity occurs with compounds bearing an electron-donating substituent at the para position of the aromatic ring [1]. In the absence of substrate, the enzyme can act as an NAD(P)H oxidase (EC 1.6.3.1).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.85': {'ec_num': '1.14.13.85',\n",
       "  'accepted_name': 'glyceollin synthase',\n",
       "  'reaction': '2-(or 4-)dimethylallyl-(6aS,11aS)-3,6a,9-trihydroxypterocarpan + NADPH + H+ + O2 = glyceollin + NADP+ + 2 H2O',\n",
       "  'other_names': '',\n",
       "  'sys_name': '2-(or 4-)dimethylallyl-(6aS,11aS)-3,6a,9-trihydroxypterocarpan,NADPH:oxygen oxidoreductase (cyclizing)',\n",
       "  'comments': 'A heme-thiolate protein (P-450). Glyceollins II and III are formed from 2-dimethylallyl-(6aS,11aS)-3,6a,9-trihydroxypterocarpan whereas glyceollin I is formed from the 4-isomer.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.86': {'ec_num': '1.14.13.86',\n",
       "  'accepted_name': '2-hydroxyisoflavanone synthase',\n",
       "  'reaction': 'apigenin + 2 NADPH + 2 H+ + O2 = 2-hydroxy-2,3-dihydrogenistein + 2 NADP+ + H2O',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'apigenin,NADPH:oxygen oxidoreductase (isoflavanone-forming)',\n",
       "  'comments': 'A heme-thiolate protein (P-450). EC 4.2.1.105, 2-hydroxyisoflavanone dehydratase, acts on 2-hydroxy-2,3-dihydrogenistein with loss of water and formation of genistein. This may occur spontaneously.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'apigenin = 4′,5,7-trihydroxyflavone',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.87': {'ec_num': '1.14.13.87',\n",
       "  'accepted_name': 'licodione synthase',\n",
       "  'reaction': 'liquiritigenin + NADPH + H+ + O2 = licodione + NADP+ + H2O',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'liquiritigenin,NADPH:oxygen oxidoreductase (licodione-forming)',\n",
       "  'comments': 'A heme-thiolate protein (P-450). It probably forms 2-hydroxyliquiritigenin which spontaneously forms licodione. NADH can act instead of NADPH, but more slowly.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'licodione = 1-(2,4-dihydroxyphenyl)-3-(4-hydroxyphenyl)propane-1,3-dione',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.88': {'ec_num': '1.14.13.88',\n",
       "  'accepted_name': 'flavanoid 3′,5′-hydroxylase',\n",
       "  'reaction': 'a flavanone + 2 NADPH + 2 H+ + 2 O2 = a 3′,5′-dihydroxyflavanone + 2 NADP+ + 2 H2O (overall reaction);;(1a) a flavanone + NADPH + H+ + O2 = a 3′-hydroxyflavanone + NADP+ + H2O;;(1b) a 3′-hydroxyflavanone + NADPH + H+ + O2 = a 3′,5′-dihydroxyflavanone + NADP+ + H2O',\n",
       "  'other_names': 'flavonoid 3′,5′-hydroxylase',\n",
       "  'sys_name': 'flavanone,NADPH:oxygen oxidoreductase',\n",
       "  'comments': 'A heme-thiolate protein (P-450). The 3′,5′-dihydroxyflavanone is formed via the 3′-hydroxyflavanone. In Petunia hybrida the enzyme acts on naringenin, eriodictyol, dihydroquercetin (taxifolin) and dihydrokaempferol (aromadendrin). The enzyme catalyses the hydroxylation of 5,7,4′-trihydroxyflavanone (naringenin) at either the 3′ position to form eriodictyol or at both the 3′ and 5′ positions to form 5,7,3′,4′,5′-pentahydroxyflavanone (dihydrotricetin). The enzyme also catalyses the hydroxylation of 3,5,7,3′,4′-pentahydroxyflavanone (taxifolin) at the 5′ position, forming ampelopsin. NADH is not a good substitute for NADPH.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.89': {'ec_num': '1.14.13.89',\n",
       "  'accepted_name': 'isoflavone 2′-hydroxylase',\n",
       "  'reaction': 'an isoflavone + NADPH + H+ + O2 = a 2′-hydroxyisoflavone + NADP+ + H2O',\n",
       "  'other_names': 'isoflavone 2′-monooxygenase; CYP81E1; CYP Ge-3',\n",
       "  'sys_name': 'isoflavone,NADPH:oxygen oxidoreductase (2′-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450). Acts on daidzein, formononetin and genistein. EC 1.14.13.53, 4′-methoxyisoflavone 2′-hydroxylase, has the same reaction but is more specific as it requires a 4′-methoxyisoflavone.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.90': {'ec_num': '1.14.13.90',\n",
       "  'accepted_name': 'zeaxanthin epoxidase',\n",
       "  'reaction': 'zeaxanthin + 2 NAD(P)H + 2 H+ + 2 O2 = violaxanthin + 2 NAD(P)+ + 2 H2O (overall reaction);;(1a) zeaxanthin + NAD(P)H + H+ + O2 = antheraxanthin + NAD(P)+ + H2O;;(1b) antheraxanthin + NAD(P)H + H+ + O2 = violaxanthin + NAD(P)+ + H2O',\n",
       "  'other_names': 'Zea-epoxidase',\n",
       "  'sys_name': 'zeaxanthin,NAD(P)H:oxygen oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD) that is active under conditions of low light. Along with EC 1.10.99.3, violaxanthin de-epoxidase, this enzyme forms part of the xanthophyll (or violaxanthin) cycle, which is involved in protecting the plant against damage by excess light. It will also epoxidize lutein in some higher-plant species.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.91': {'ec_num': '1.14.13.91',\n",
       "  'accepted_name': 'deoxysarpagine hydroxylase',\n",
       "  'reaction': '10-deoxysarpagine + NADPH + H+ + O2 = sarpagine + NADP+ + H2O',\n",
       "  'other_names': 'DOSH',\n",
       "  'sys_name': '10-deoxysarpagine,NADPH:oxygen oxidoreductase (10-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.92': {'ec_num': '1.14.13.92',\n",
       "  'accepted_name': 'phenylacetone monooxygenase',\n",
       "  'reaction': 'phenylacetone + NADPH + H+ + O2 = benzyl acetate + NADP+ + H2O',\n",
       "  'other_names': 'PAMO',\n",
       "  'sys_name': 'phenylacetone,NADPH:oxygen oxidoreductase',\n",
       "  'comments': 'A flavoprotein (FAD). NADH cannot replace NADPH as coenzyme. In addition to phenylacetone, which is the best substrate found to date, this Baeyer-Villiger monooxygenase can oxidize other aromatic ketones [1-(4-hydroxyphenyl)propan-2-one, 1-(4-hydroxyphenyl)propan-2-one and 3-phenylbutan-2-one], some alipatic ketones (e.g. dodecan-2-one) and sulfides (e.g. 1-methyl-4-(methylsulfanyl)benzene).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.93': {'ec_num': '1.14.13.93',\n",
       "  'accepted_name': '(+)-abscisic acid 8′-hydroxylase',\n",
       "  'reaction': '(+)-abscisate + NADPH + H+ + O2 = 8′-hydroxyabscisate + NADP+ + H2O',\n",
       "  'other_names': '(+)-ABA 8′-hydroxylase; ABA 8′-hydroxylase',\n",
       "  'sys_name': 'abscisate,NADPH:oxygen oxidoreductase (8′-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450). Catalyses the first step in the oxidative degradation of abscisic acid and is considered to be the pivotal enzyme in controlling the rate of degradation of this plant hormone [1]. CO inhibits the reaction, but its effects can be reversed by the presence of blue light [1]. The 8′-hydroxyabscisate formed can be converted into (–)-phaseic acid, most probably spontaneously. Other enzymes involved in the abscisic-acid biosynthesis pathway are EC 1.1.1.288 (xanthoxin dehydrogenase), EC 1.2.3.14 (abscisic-aldehyde oxidase) and EC 1.13.11.51 (9-cis-epoxycarotenoid dioxygenase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.94': {'ec_num': '1.14.13.94',\n",
       "  'accepted_name': 'lithocholate 6β-hydroxylase',\n",
       "  'reaction': 'lithocholate + NADPH + H+ + O2 = 6β-hydroxylithocholate + NADP+ + H2O',\n",
       "  'other_names': 'lithocholate 6β-monooxygenase; CYP3A10; 6β-hydroxylase; cytochrome P450 3A10/lithocholic acid 6β-hydroxylase',\n",
       "  'sys_name': 'lithocholate,NADPH:oxygen oxidoreductase (6β-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450). Expression of the gene for this enzyme is 50-fold higher in male compared to female hamsters [1].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'lithocholate = 3α-hydroxy-5β-cholan-24-oate\\n6β-hydroxylithocholate = murideoxycholate = 3α,6β-dihydroxy-5β-cholan-24-oate',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.95': {'ec_num': '1.14.13.95',\n",
       "  'accepted_name': '7α-hydroxycholest-4-en-3-one 12α-hydroxylase',\n",
       "  'reaction': '7α-hydroxycholest-4-en-3-one + NADPH + H+ + O2 = 7α,12α-dihydroxycholest-4-en-3-one + NADP+ + H2O',\n",
       "  'other_names': '7α-hydroxy-4-cholesten-3-one 12α-monooxygenase; CYP12; sterol 12α-hydroxylase (ambiguous); HCO 12α-hydroxylase',\n",
       "  'sys_name': '7α-hydroxycholest-4-en-3-one,NADPH:oxygen oxidoreductase (12α-hydroxylating)',\n",
       "  'comments': 'A P-450 heme-thiolate protein. Requires EC 1.6.2.4, NADPH—hemoprotein reductase, which reduced cytochrome b5 at the expense of NADPH. This enzyme is important in bile acid biosynthesis, being responsible for the balance between the formation of cholic acid and chenodeoxycholic acid [2].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.96': {'ec_num': '1.14.13.96',\n",
       "  'accepted_name': '5β-cholestane-3α,7α-diol 12α-hydroxylase',\n",
       "  'reaction': '5β-cholestane-3α,7α-diol + NADPH + H+ + O2 = 5β-cholestane-3α,7α,12α-triol + NADP+ + H2O',\n",
       "  'other_names': '5β-cholestane-3α,7α-diol 12α-monooxygenase; sterol 12α-hydroxylase (ambiguous); CYP8B1; cytochrome P450 8B1',\n",
       "  'sys_name': '5β-cholestane-3α,7α-diol,NADPH:oxygen oxidoreductase (12α-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450). This is the key enzyme in the biosynthesis of the bile acid cholic acid (3α,7α,12α-trihydroxy-5β-cholanoic acid). The activity of this enzyme determines the biosynthetic ratio between cholic acid and chenodeoxycholic acid [3]. The enzyme can also hydroxylate the substrate at the 25 and 26 position, but to a lesser extent [1].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.97': {'ec_num': '1.14.13.97',\n",
       "  'accepted_name': 'taurochenodeoxycholate 6α-hydroxylase',\n",
       "  'reaction': '(1) taurochenodeoxycholate + NADPH + H+ + O2 = taurohyocholate + NADP+ + H2O;;(2) lithocholate + NADPH + H+ + O2 = hyodeoxycholate + NADP+ + H2O',\n",
       "  'other_names': 'CYP3A4; CYP4A21; taurochenodeoxycholate 6α-monooxygenase',\n",
       "  'sys_name': 'taurochenodeoxycholate,NADPH:oxygen oxidoreductase (6α-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450). Requires cytochrome b5 for maximal activity. Acts on taurochenodeoxycholate, taurodeoxycholate and less readily on lithocholate and chenodeoxycholate. In adult pig (Sus scrofa), hyocholic acid replaces cholic acid as a primary bile acid [5].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'taurochenodeoxycholic acid = N-(3α,7α-dihydroxy-5β-cholan-24-oyl)taurine\\ntaurohyocholic acid = N-(3α,6α,7α-trihydroxy-5β-cholan-24-oyl)taurine\\nhyodeoxycholate = 3α,6α-dihydroxy-5β-cholan-24-oate\\nlithocholate = 3α-hydroxy-5β-cholan-24-oate',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.98': {'ec_num': '1.14.13.98',\n",
       "  'accepted_name': 'cholesterol 24-hydroxylase',\n",
       "  'reaction': 'cholesterol + NADPH + H+ + O2 = (24S)-24-hydroxycholesterol + NADP+ + H2O',\n",
       "  'other_names': 'cholesterol 24-monooxygenase; CYP46; CYP46A1; cholesterol 24S-hydroxylase; cytochrome P450 46A1',\n",
       "  'sys_name': 'cholesterol,NADPH:oxygen oxidoreductase (24-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450). The enzyme can also produce 25-hydroxycholesterol. In addition, it can further hydroxylate the product to 24,25-dihydroxycholesterol and 24,27-dihydroxycholesterol [2]. This reaction is the first step in the enzymic degradation of cholesterol in the brain as hydroxycholesterol can pass the blood—brain barrier whereas cholesterol cannot [3].',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'cholesterol = cholest-5-en-3β-ol',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.99': {'ec_num': '1.14.13.99',\n",
       "  'accepted_name': '24-hydroxycholesterol 7α-hydroxylase',\n",
       "  'reaction': '(24R)-cholest-5-ene-3β,24-diol + NADPH + H+ + O2 = (24R)-cholest-5-ene-3β,7α,24-triol + NADP+ + H2O',\n",
       "  'other_names': '24-hydroxycholesterol 7α-monooxygenase; CYP39A1; CYP39A1 oxysterol 7α-hydroxylase',\n",
       "  'sys_name': '(24R)-cholest-5-ene-3β,24-diol,NADPH:oxygen oxidoreductase (7α-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450) that is found in liver microsomes and in ciliary non-pigmented epithelium [2]. The enzyme is specific for (24R)-cholest-5-ene-3β,24-diol as substrate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.13.100': {'ec_num': '1.14.13.100',\n",
       "  'accepted_name': '25/26-hydroxycholesterol 7α-hydroxylase',\n",
       "  'reaction': '(1) cholest-5-ene-3β,25-diol + NADPH + H+ + O2 = cholest-5-ene-3β,7α,25-triol + NADP+ + H2O;;(2) (25R)-cholest-5-ene-3β,26-diol + NADPH + H+ + O2 = (25R)-cholest-5-ene-3β,7α,26-triol + NADP+ + H2O',\n",
       "  'other_names': '25-hydroxycholesterol 7α-monooxygenase; CYP7B1; CYP7B1 oxysterol 7α-hydroxylase; 27-hydroxycholesterol 7-monooxygenase; 27-hydroxycholesterol 7α-hydroxylase; cholest-5-ene-3β,25-diol,NADPH:oxygen oxidoreductase (7α-hydroxylating); 25-hydroxycholesterol 7α-hydroxylase',\n",
       "  'sys_name': 'cholest-5-ene-3β,25/26-diol,NADPH:oxygen oxidoreductase (7α-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450). Unlike EC 1.14.13.99, 24-hydroxycholesterol 7α-monooxygenase, which is specific for its oxysterol substrate, this enzyme can also metabolize the oxysterols 24,25-epoxycholesterol, 22-hydroxycholesterol and 24-hydroxycholesterol, but to a lesser extent [2]. ',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.14.1': {'ec_num': '1.14.14.1',\n",
       "  'accepted_name': 'unspecific monooxygenase',\n",
       "  'reaction': 'RH + [reduced NADPH—hemoprotein reductase] + O2 = ROH + [oxidized NADPH—hemoprotein reductase] + H2O',\n",
       "  'other_names': 'microsomal monooxygenase; xenobiotic monooxygenase; aryl-4-monooxygenase; aryl hydrocarbon hydroxylase; microsomal P-450; flavoprotein-linked monooxygenase; flavoprotein monooxygenase; substrate,reduced-flavoprotein:oxygen oxidoreductase (RH-hydroxylating or -epoxidizing)',\n",
       "  'sys_name': 'substrate,NADPH—hemoprotein reductase:oxygen oxidoreductase (RH-hydroxylating or -epoxidizing)',\n",
       "  'comments': 'A group of P-450 heme-thiolate proteins, acting on a wide range of substrates including many xenobiotics, steroids, fatty acids, vitamins and prostaglandins; reactions catalysed include hydroxylation, epoxidation, N-oxidation, sulfooxidation, N-, S- and O-dealkylations, desulfation, deamination, and reduction of azo, nitro and N-oxide groups. Together with EC 1.6.2.4, NADPH—hemoprotein reductase, it forms a system in which two reducing equivalents are supplied by NADPH. Some of the reactions attributed to EC 1.14.15.3, alkane 1-monooxygenase, belong here.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.14.1\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.14.1\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.14.1\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.14.1\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.14.1%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.14.3': {'ec_num': '1.14.14.3',\n",
       "  'accepted_name': 'bacterial luciferase',\n",
       "  'reaction': 'a long-chain aldehyde + FMNH2 + O2 = a long-chain fatty acid + FMN + H2O + hν',\n",
       "  'other_names': 'aldehyde monooxygenase; luciferase; Vibrio fischeri luciferase; alkanal,reduced-FMN:oxygen oxidoreductase (1-hydroxylating, luminescing); alkanal,FMNH2:oxygen oxidoreductase (1-hydroxylating, luminescing); alkanal monooxygenase (FMN); aldehyde,FMNH2:oxygen oxidoreductase (1-hydroxylating, luminescing)',\n",
       "  'sys_name': 'long-chain-aldehyde,FMNH2:oxygen oxidoreductase (1-hydroxylating, luminescing)',\n",
       "  'comments': 'The reaction sequence starts with the incorporation of a molecule of oxygen into reduced FMN bound to the enzyme, forming luciferase peroxyflavin. The peroxyflavin interacts with an aliphatic long-chain aldehyde, producing a highly fluorescent species believed to be luciferase hydroxyflavin. The enzyme is highly specific for reduced FMN and for long-chain aliphatic aldehydes with eight carbons or more. The highest efficiency is achieved with tetradecanal. cf. EC 1.13.12.18, dinoflagellate luciferase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.14.5': {'ec_num': '1.14.14.5',\n",
       "  'accepted_name': 'alkanesulfonate monooxygenase',\n",
       "  'reaction': 'an alkanesulfonate + FMNH2 + O2 = an aldehyde + FMN + sulfite + H2O',\n",
       "  'other_names': 'SsuD; sulfate starvation-induced protein 6; alkanesulfonate,reduced-FMN:oxygen oxidoreductase',\n",
       "  'sys_name': 'alkanesulfonate,FMNH2:oxygen oxidoreductase',\n",
       "  'comments': 'The enzyme from Escherichia coli catalyses the desulfonation of a wide range of aliphatic sulfonates (unsubstituted C1- to C14-sulfonates as well as substituted C2-sulfonates). Does not desulfonate taurine (2-aminoethanesulfonate) or aromatic sulfonates. Does not use FMN as a bound cofactor. Instead, it uses reduced FMN (i.e., FMNH2) as a substrate. FMNH2 is provided by SsuE, the associated FMN reductase (EC 1.5.1.38).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'an alkanesulfonate = R-CH2-SO3-\\nan aldehyde = R-CHO',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.15.1': {'ec_num': '1.14.15.1',\n",
       "  'accepted_name': 'camphor 5-monooxygenase',\n",
       "  'reaction': '(+)-camphor + reduced putidaredoxin + O2 = (+)-exo-5-hydroxycamphor + oxidized putidaredoxin + H2O',\n",
       "  'other_names': 'camphor 5-exo-methylene hydroxylase; 2-bornanone 5-exo-hydroxylase; bornanone 5-exo-hydroxylase; camphor 5-exo-hydroxylase; camphor 5-exohydroxylase; camphor hydroxylase; d-camphor monooxygenase; methylene hydroxylase; methylene monooxygenase; D-camphor-exo-hydroxylase; camphor methylene hydroxylase',\n",
       "  'sys_name': '(+)-camphor,reduced putidaredoxin:oxygen oxidoreductase (5-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450). Also acts on (-)-camphor and 1,2-campholide, forming 5-exo-hydroxy-1,2-campholide.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.15.1\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.15.1\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.15.1\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.15.1\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.15.1%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.15.2': {'ec_num': '1.14.15.2',\n",
       "  'accepted_name': 'camphor 1,2-monooxygenase',\n",
       "  'reaction': '(+)-bornane-2,5-dione + reduced rubredoxin + O2 = 5-oxo-1,2-campholide + oxidized rubredoxin + H2O',\n",
       "  'other_names': '2,5-diketocamphane lactonizing enzyme; camphor ketolactonase I; ketolactonase I',\n",
       "  'sys_name': '(+)-camphor,reduced-rubredoxin:oxygen oxidoreductase (1,2-lactonizing)',\n",
       "  'comments': 'A flavoprotein which requires Fe2+. Isolated from Pseudomonas putida. The product is spontaneously converted to [(1R)-5-oxo-2,2,3-trimethylcyclopent-3-enyl]acetate. The enzyme slowly oxidized (-)-camphor.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.15.2\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.15.2\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.15.2\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.15.2\" target=\"new\">KEGG</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.15.3': {'ec_num': '1.14.15.3',\n",
       "  'accepted_name': 'alkane 1-monooxygenase',\n",
       "  'reaction': 'octane + 2 reduced rubredoxin + O2 + 2 H+ = 1-octanol + 2 oxidized rubredoxin + H2O',\n",
       "  'other_names': 'alkane 1-hydroxylase; ω-hydroxylase; fatty acid ω-hydroxylase; alkane monooxygenase; 1-hydroxylase; alkane hydroxylase',\n",
       "  'sys_name': 'alkane,reduced-rubredoxin:oxygen 1-oxidoreductase',\n",
       "  'comments': 'Some enzymes in this group are heme-thiolate proteins (P-450). Also hydroxylates fatty acids in the ω-position.',\n",
       "  'links': '<a href=\"https://www.brenda-enzymes.org/enzyme.php?ecno=1.14.15.3\" target=\"new\">BRENDA</a>, <a href=\"http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=1.14.15.3\" target=\"new\">EAWAG-BBD</a>, <a href=\"https://enzyme.expasy.org/EC/1.14.15.3\" target=\"new\">EXPASY</a>, <a href=\"https://www.kegg.jp/dbget-bin/www_bget?ec:1.14.15.3\" target=\"new\">KEGG</a>, <a href=\"https://www.rcsb.org/search?request=%7B%22query%22%3A%7B%22type%22%3A%22terminal%22%2C%22service%22%3A%22text%22%2C%22parameters%22%3A%7B%22attribute%22%3A%22rcsb_polymer_entity.rcsb_ec_lineage.id%22%2C%22operator%22%3A%22in%22%2C%22value%22%3A%5B%221.14.15.3%22%5D%7D%7D%2C%22return_type%22%3A%22polymer_entity%22%7D\" target=\"new\">PDB</a>',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.15.4': {'ec_num': '1.14.15.4',\n",
       "  'accepted_name': 'steroid 11β-monooxygenase',\n",
       "  'reaction': 'a steroid + 2 reduced adrenodoxin + O2 + 2 H+ = an 11β-hydroxysteroid + 2 oxidized adrenodoxin + H2O',\n",
       "  'other_names': 'steroid 11β-hydroxylase; steroid 11β/18-hydroxylase',\n",
       "  'sys_name': 'steroid,reduced-adrenodoxin:oxygen oxidoreductase (11β-hydroxylating)',\n",
       "  'comments': 'A heme-thiolate protein (P-450). Also hydroxylates steroids at the 18-position, and converts 18-hydroxycorticosterone into aldosterone.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.15.5': {'ec_num': '1.14.15.5',\n",
       "  'accepted_name': 'corticosterone 18-monooxygenase',\n",
       "  'reaction': 'corticosterone + 2 reduced adrenodoxin + O2 + 2 H+ = 18-hydroxycorticosterone + 2 oxidized adrenodoxin + H2O',\n",
       "  'other_names': 'corticosterone 18-hydroxylase; corticosterone methyl oxidase',\n",
       "  'sys_name': 'corticosterone,reduced-adrenodoxin:oxygen oxidoreductase (18-hydroxylating)',\n",
       "  'comments': '',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.15.6': {'ec_num': '1.14.15.6',\n",
       "  'accepted_name': 'cholesterol monooxygenase (side-chain-cleaving)',\n",
       "  'reaction': 'cholesterol + 6 reduced adrenodoxin + 3 O2 + 6 H+ = pregnenolone + 4-methylpentanal + 6 oxidized adrenodoxin + 4 H2O (overall reaction);;(1a) cholesterol + 2 reduced adrenodoxin + O2 + 2 H+ = (22R)-22-hydroxycholesterol + 2 oxidized adrenodoxin + H2O;;(1b) (22R)-22-hydroxycholesterol + 2 reduced adrenodoxin + O2 + 2 H+ = (20R,22R)-20,22-dihydroxycholesterol + 2 oxidized adrenodoxin + H2O;;(1c) (20R,22R)-20,22-dihydroxy-cholesterol + 2 reduced adrenodoxin + O2 + 2 H+ = pregnenolone + 4-methylpentanal + 2 oxidized adrenodoxin + 2 H2O',\n",
       "  'other_names': 'cholesterol desmolase; cytochrome P-450scc; C27-side chain cleavage enzyme; cholesterol 20-22-desmolase; cholesterol C20-22 desmolase; cholesterol side-chain cleavage enzyme; cholesterol side-chain-cleaving enzyme; steroid 20-22 desmolase; steroid 20-22-lyase; CYP11A1 (gene name)',\n",
       "  'sys_name': 'cholesterol,reduced-adrenodoxin:oxygen oxidoreductase (side-chain-cleaving)',\n",
       "  'comments': 'A heme-thiolate protein (cytochrome P-450). The reaction proceeds in three stages, with two hydroxylations at C-22 and C-20 preceding scission of the side-chain between carbons 20 and 22. The initial source of the electrons is NADPH, which transfers the electrons to the adrenodoxin via EC 1.18.1.6, adrenodoxin-NADP+ reductase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.15.7': {'ec_num': '1.14.15.7',\n",
       "  'accepted_name': 'choline monooxygenase',\n",
       "  'reaction': 'choline + O2 + 2 reduced ferredoxin + 2 H+ = betaine aldehyde hydrate + H2O + 2 oxidized ferredoxin',\n",
       "  'other_names': '',\n",
       "  'sys_name': 'choline,reduced-ferredoxin:oxygen oxidoreductase',\n",
       "  'comments': 'The spinach enzyme, which is located in the chloroplast, contains a Rieske-type [2Fe-2S] cluster, and probably also a mononuclear Fe centre. Requires Mg2+. Catalyses the first step of glycine betaine synthesis. In many bacteria, plants and animals, betaine is synthesized in two steps: (1) choline to betaine aldehyde and (2) betaine aldehyde to betaine. Different enzymes are involved in the first reaction. In plants, the reaction is catalysed by this enzyme whereas in animals and many bacteria it is catalysed by either membrane-bound EC 1.1.99.1 (choline dehydrogenase) or soluble EC 1.1.3.17 (choline oxidase) [7]. The enzyme involved in the second step, EC 1.2.1.8 (betaine-aldehyde dehydrogenase), appears to be the same in plants, animals and bacteria. In some bacteria, betaine is synthesized from glycine through the actions of EC 2.1.1.156 (glycine/sarcosine N-methyltransferase) and EC 2.1.1.157 (sarcosine/dimethylglycine N-methyltransferase).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': 'betaine = glycine betaine = N,N,N-trimethylglycine = N,N,N-trimethylammonioacetate\\nbetaine aldehyde = N,N,N-trimethyl-2-oxoethylammonium\\ncholine = (2-hydroxyethyl)trimethylammonium',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.16.1': {'ec_num': '1.14.16.1',\n",
       "  'accepted_name': 'phenylalanine 4-monooxygenase',\n",
       "  'reaction': 'L-phenylalanine + a 5,6,7,8-tetrahydropteridine + O2 = L-tyrosine + a 4a-hydroxy-5,6,7,8-tetrahydropteridine',\n",
       "  'other_names': 'phenylalaninase; phenylalanine 4-hydroxylase; phenylalanine hydroxylase',\n",
       "  'sys_name': 'L-phenylalanine,tetrahydropteridine:oxygen oxidoreductase (4-hydroxylating)',\n",
       "  'comments': 'The active centre contains mononuclear iron(II). The reaction involves an arene oxide that rearranges to give the phenolic hydroxy group. This results in the hydrogen at C-4 migrating to C-3 and in part being retained. This process is known as the NIH-shift. The 4a-hydroxytetrahydropteridine formed can dehydrate to 6,7-dihydropteridine, both spontaneously and by the action of EC 4.2.1.96, 4a-hydroxytetrahydrobiopterin dehydratase. The 6,7-dihydropteridine must be enzymically reduced back to tetrahydropteridine, by EC 1.5.1.34, 6,7-dihydropteridine reductase, before it slowly rearranges into the more stable but inactive compound 7,8-dihydropteridine.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.16.2': {'ec_num': '1.14.16.2',\n",
       "  'accepted_name': 'tyrosine 3-monooxygenase',\n",
       "  'reaction': 'L-tyrosine + a 5,6,7,8-tetrahydropteridine + O2 = L-dopa + a 4a-hydroxy-5,6,7,8-tetrahydropteridine',\n",
       "  'other_names': 'L-tyrosine hydroxylase; tyrosine 3-hydroxylase; tyrosine hydroxylase',\n",
       "  'sys_name': 'L-tyrosine,tetrahydropteridine:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'comments': 'The active centre contains mononuclear iron(II). The enzyme is activated by phosphorylation, catalysed by EC 2.7.11.27, [acetyl-CoA carboxylase] kinase. The 4a-hydroxytetrahydropteridine formed can dehydrate to 6,7-dihydropteridine, both spontaneously and by the action of EC 4.2.1.96, 4a-hydroxytetrahydrobiopterin dehydratase. The 6,7-dihydropteridine must be enzymically reduced back to tetrahydropteridine, by EC 1.5.1.34, 6,7-dihydropteridine reductase, before it slowly rearranges into the more stable but inactive compound 7,8-dihydropteridine.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'L-dopa = 3,4-dihydroxy-L-phenylalanine',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.16.3': {'ec_num': '1.14.16.3',\n",
       "  'accepted_name': 'anthranilate 3-monooxygenase',\n",
       "  'reaction': 'anthranilate + a 5,6,7,8-tetrahydropteridine + O2 = 3-hydroxyanthranilate + a 4a-hydroxy-5,6,7,8-tetrahydropteridine',\n",
       "  'other_names': 'anthranilate 3-hydroxylase; anthranilate hydroxylase; anthranilic hydroxylase; anthranilic acid hydroxylase; anthranilate,tetrahydrobiopterin:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'sys_name': 'anthranilate,tetrahydropteridine:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'comments': 'Requires Fe2+. The enzyme has been characterized from the leaves of the plant Tecoma stans (Yellow Bells).',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.16.4': {'ec_num': '1.14.16.4',\n",
       "  'accepted_name': 'tryptophan 5-monooxygenase',\n",
       "  'reaction': 'L-tryptophan + a 5,6,7,8-tetrahydropteridine + O2 = 5-hydroxy-L-tryptophan + a 4a-hydroxy-5,6,7,8-tetrahydropteridine',\n",
       "  'other_names': 'L-tryptophan hydroxylase; indoleacetic acid-5-hydroxylase; tryptophan 5-hydroxylase; tryptophan hydroxylase',\n",
       "  'sys_name': 'L-tryptophan,tetrahydropteridine:oxygen oxidoreductase (5-hydroxylating)',\n",
       "  'comments': 'The active centre contains mononuclear iron(II). The enzyme is activated by phosphorylation, catalysed by a Ca2+-activated protein kinase. The 4a-hydroxytetrahydropteridine formed can dehydrate to 6,7-dihydropteridine, both spontaneously and by the action of EC 4.2.1.96, 4a-hydroxytetrahydrobiopterin dehydratase. The 6,7-dihydropteridine must be enzymically reduced back to tetrahydropteridine, by EC 1.5.1.34, 6,7-dihydropteridine reductase, before it slowly rearranges into the more stable but inactive compound 7,8-dihydropteridine.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.16.5': {'ec_num': '1.14.16.5',\n",
       "  'accepted_name': 'alkylglycerol monooxygenase',\n",
       "  'reaction': '1-O-alkyl-sn-glycerol + a 5,6,7,8-tetrahydropteridine + O2 = 1-O-(1-hydroxyalkyl)-sn-glycerol + a 4a-hydroxy-5,6,7,8-tetrahydropteridine',\n",
       "  'other_names': 'glyceryl-ether monooxygenase; glyceryl-ether cleaving enzyme; glyceryl ether oxygenase; glyceryl etherase; O-alkylglycerol monooxygenase',\n",
       "  'sys_name': '1-alkyl-sn-glycerol,tetrahydrobiopteridine:oxygen oxidoreductase',\n",
       "  'comments': 'The enzyme cleaves alkylglycerols, but does not cleave alkenylglycerols (plasmalogens). Requires non-heme iron [7], reduced glutathione and phospholipids for full activity. The product spontaneously breaks down to form a fatty aldehyde and glycerol. The co-product, 4a-hydroxytetrahydropteridine, is rapidly dehydrated to 6,7-dihydropteridine, either spontaneously or by EC 4.2.1.96, 4a-hydroxytetrahydrobiopterin dehydratase.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.16.6': {'ec_num': '1.14.16.6',\n",
       "  'accepted_name': 'mandelate 4-monooxygenase',\n",
       "  'reaction': '(S)-2-hydroxy-2-phenylacetate + a 5,6,7,8-tetrahydropteridine + O2 = (S)-4-hydroxymandelate + a 4a-hydroxy-5,6,7,8-tetrahydropteridine',\n",
       "  'other_names': 'L-mandelate 4-hydroxylase; mandelic acid 4-hydroxylase',\n",
       "  'sys_name': '(S)-2-hydroxy-2-phenylacetate,tetrahydropteridine:oxygen oxidoreductase (4-hydroxylating)',\n",
       "  'comments': 'Requires Fe2+. The enzyme has been characterized from the bacterium Pseudomonas putida. The 4a-hydroxytetrahydropteridine formed can dehydrate to 6,7-dihydropteridine, both spontaneously and by the action of EC 4.2.1.96, 4a-hydroxytetrahydrobiopterin dehydratase. The 6,7-dihydropteridine must be enzymically reduced back to tetrahydropteridine, by EC 1.5.1.34, 6,7-dihydropteridine reductase, before it slowly rearranges into the more stable but inactive compound 7,8-dihydropteridine.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG',\n",
       "  'glossary': '(S)-4-hydroxymandelate = (S)-2-hydroxy-2-(4-hydroxyphenyl)acetate',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.17.1': {'ec_num': '1.14.17.1',\n",
       "  'accepted_name': 'dopamine β-monooxygenase',\n",
       "  'reaction': 'dopamine + 2 ascorbate + O2 = noradrenaline + 2 monodehydroascorbate + H2O',\n",
       "  'other_names': 'dopamine β-hydroxylase; MDBH (membrane-associated dopamine β-monooxygenase); SDBH (soluble dopamine β-monooxygenase); dopamine-B-hydroxylase; 3,4-dihydroxyphenethylamine β-oxidase; 4-(2-aminoethyl)pyrocatechol β-oxidase; dopa β-hydroxylase; dopamine β-oxidase; dopamine hydroxylase; phenylamine β-hydroxylase; (3,4-dihydroxyphenethylamine)β-mono-oxygenase; DβM (gene name)',\n",
       "  'sys_name': 'dopamine,ascorbate:oxygen oxidoreductase (β-hydroxylating)',\n",
       "  'comments': 'A copper protein. The enzyme, found in animals, binds two copper ions with distinct roles during catalysis. Stimulated by fumarate.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': 'dopamine = 4-(2-aminoethyl)benzene-1,2-diol',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " '1.14.17.3': {'ec_num': '1.14.17.3',\n",
       "  'accepted_name': 'peptidylglycine monooxygenase',\n",
       "  'reaction': '[peptide]-glycine + 2 ascorbate + O2 = [peptide]-(2S)-2-hydroxyglycine + 2 monodehydroascorbate + H2O',\n",
       "  'other_names': 'peptidylglycine 2-hydroxylase; peptidyl α-amidating enzyme; peptide-α-amide synthetase; peptide α-amidating enzyme; peptide α-amide synthase; peptidylglycine α-hydroxylase; peptidylglycine α-amidating monooxygenase; PAM-A; PAM-B; PAM; peptidylglycine,ascorbate:oxygen oxidoreductase (2-hydroxylating)',\n",
       "  'sys_name': '[peptide]-glycine,ascorbate:oxygen oxidoreductase (2-hydroxylating)',\n",
       "  'comments': 'A copper protein. The enzyme binds two copper ions with distinct roles during catalysis. Peptidylglycines with a neutral amino acid residue in the penultimate position are the best substrates for the enzyme. The product is unstable and dismutates to glyoxylate and the corresponding desglycine peptide amide, a reaction catalysed by EC 4.3.2.5 peptidylamidoglycolate lyase. In mammals, the two activities are part of a bifunctional protein. Involved in the final step of biosynthesis of α-melanotropin and related biologically active peptides.',\n",
       "  'links': 'BRENDA, EXPASY, KEGG, PDB',\n",
       "  'glossary': '',\n",
       "  'last_change': '2023-02-11 18:31:23'},\n",
       " ...}"
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from src.iubmb import extract_ec_numbers\n",
    "\n",
    "xml_file_path = '/home/robaina/Documents/NewAtlantis/enzyme_activity/notebooks/data/annotations/iubmb/enzyme-data.xml'\n",
    "ec_dict = extract_ec_numbers(xml_file_path)\n",
    "\n",
    "ec_dict"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "neo4j",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
